
Download and run validation tests on all the curated models from BioModels https://www.ebi.ac.uk/biomodels.
The final step is to run the model in tellurium,
only models specified in SBML with a matching SEDML file are run in tellurium.
Errors or validation failures are reported at each step.
Outputs to the Markdown Table below.

'valid-sbml-units' enforces strict unit checking, 'broken-ref' indicates that the SEDML file contained
a broken source='model.xml' reference which was corrected to the name of the model's provided SBML file.
|Model|valid-sbml|valid-sbml-units|valid-sedml|broken-ref|tellurium|copasi|
|---|---|---|---|---|---|---|
|n=1073|<details><summary>pass=1030 FAIL=43</summary>n_FAIL=33 n_NonSBML=10</details>|pass=785 NA=10 FAIL=278|<details><summary>pass=297 NA=10 FAIL=766</summary>n_NoSEDML=679 n_FAIL=62 n_MultipleSEDMLs=25</details>|pass=67 NA=714 FAIL=292|pass=9 NA=714 other=350|pass=9 NA=714 other=350|
|[BIOMD0000000001](https://www.ebi.ac.uk/biomodels/BIOMD0000000001)<br/><sup>Edelstein1996 - EPSP ACh event</sup>|<details><summary>pass</summary>[BIOMD0000000001_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000001#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000002](https://www.ebi.ac.uk/biomodels/BIOMD0000000002)<br/><sup>Edelstein1996 - EPSP ACh species</sup>|<details><summary>pass</summary>[BIOMD0000000002_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000002#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000003](https://www.ebi.ac.uk/biomodels/BIOMD0000000003)<br/><sup>Goldbeter1991 - Min Mit Oscil</sup>|<details><summary>pass</summary>[BIOMD0000000003_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000003#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000004](https://www.ebi.ac.uk/biomodels/BIOMD0000000004)<br/><sup>Goldbeter1991 - Min Mit Oscil,</sup></summary><sup>Goldbeter1991 - Min Mit Oscil, Expl Inact</sup></details>|<details><summary>pass</summary>[BIOMD0000000004_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000004#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000005](https://www.ebi.ac.uk/biomodels/BIOMD0000000005)<br/><sup>Tyson1991 - Cell Cycle 6 var</sup>|<details><summary>pass</summary>[BIOMD0000000005_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000005#Files)</details>|pass|<details><summary>FAIL</summary>[BIOMD0000000005_SED-ML.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000005#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/BIOMD0000000005_SED-ML.xml494445.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model1` has warnings.\r\n - Model resolution via URNs will likely be deprecated in a future version of SED-ML. Instead models can be resolved by URLs.\r\n - URN model source `urn:miriam:biomodels.db:BIOMD0000000005` was not validated.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./BIOMD0000000005_SED-ML.xml` has warnings.\r\n - Model `model1` has warnings.\r\n - Model resolution via URNs will likely be deprecated in a future version of SED-ML. Instead models can be resolved by URLs.\r\n - URN model source `urn:miriam:biomodels.db:BIOMD0000000005` was not validated.\x1b[0m\r\n\r\n`/root/in/BIOMD0000000005_SED-ML.xml494445.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./BIOMD0000000005_SED-ML.xml` is invalid.\n - Data generator `total_cyclin` is invalid.\n - Variable `c1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='YT']` are not defined. No namespaces are defined for the target.\n - Data generator `pcyclin_cdc2` is invalid.\n - Variable `c2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/BIOMD0000000005_SED-ML.xml619010.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model1` has warnings.\r\n - Model resolution via URNs will likely be deprecated in a future version of SED-ML. Instead models can be resolved by URLs.\r\n - URN model source `urn:miriam:biomodels.db:BIOMD0000000005` was not validated.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./BIOMD0000000005_SED-ML.xml` has warnings.\r\n - Model `model1` has warnings.\r\n - Model resolution via URNs will likely be deprecated in a future version of SED-ML. Instead models can be resolved by URLs.\r\n - URN model source `urn:miriam:biomodels.db:BIOMD0000000005` was not validated.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/BIOMD0000000005_SED-ML.xml619010.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./BIOMD0000000005_SED-ML.xml` is invalid.\n - Data generator `total_cyclin` is invalid.\n - Variable `c1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='YT']` are not defined. No namespaces are defined for the target.\n - Data generator `pcyclin_cdc2` is invalid.\n - Variable `c2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M']` are not defined. No namespaces are defined for the target.\n"```</details>|
|[BIOMD0000000006](https://www.ebi.ac.uk/biomodels/BIOMD0000000006)<br/><sup>Tyson1991 - Cell Cycle 2 var</sup>|<details><summary>pass</summary>[BIOMD0000000006_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000006#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000007](https://www.ebi.ac.uk/biomodels/BIOMD0000000007)<br/><sup>Novak1997 - Cell Cycle</sup>|<details><summary>pass</summary>[BIOMD0000000007_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000007#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000008](https://www.ebi.ac.uk/biomodels/BIOMD0000000008)<br/><sup>Gardner1998 - Cell Cycle Goldbeter</sup>|<details><summary>pass</summary>[BIOMD0000000008_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000008#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000009](https://www.ebi.ac.uk/biomodels/BIOMD0000000009)<br/><sup>Huang1996 - Ultrasensitivity i</sup></summary><sup>Huang1996 - Ultrasensitivity in MAPK cascade</sup></details>|<details><summary>pass</summary>[BIOMD0000000009_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000009#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000010](https://www.ebi.ac.uk/biomodels/BIOMD0000000010)<br/><sup>Kholodenko2000 - Ultrasensitiv</sup></summary><sup>Kholodenko2000 - Ultrasensitivity and negative feedback bring oscillations in MAPK cascade</sup></details>|<details><summary>pass</summary>[BIOMD0000000010_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000010#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000011](https://www.ebi.ac.uk/biomodels/BIOMD0000000011)<br/><sup>Levchenko2000_MAPK_noScaffold</sup>|<details><summary>pass</summary>[BIOMD0000000011_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000011#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000012](https://www.ebi.ac.uk/biomodels/BIOMD0000000012)<br/><sup>Elowitz2000 - Repressilator</sup>|<details><summary>pass</summary>[BIOMD0000000012_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000012#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000013](https://www.ebi.ac.uk/biomodels/BIOMD0000000013)<br/><sup>Poolman2004_CalvinCycle</sup>|<details><summary>pass</summary>[BIOMD0000000013_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000013#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000014](https://www.ebi.ac.uk/biomodels/BIOMD0000000014)<br/><sup>Levchenko2000_MAPK_Scaffold</sup>|<details><summary>pass</summary>[BIOMD0000000014_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000014#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000015](https://www.ebi.ac.uk/biomodels/BIOMD0000000015)<br/><sup>Curto1998 - purine metabolism</sup>|<details><summary>pass</summary>[BIOMD0000000015_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000015#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000016](https://www.ebi.ac.uk/biomodels/BIOMD0000000016)<br/><sup>Goldbeter1995_CircClock</sup>|<details><summary>pass</summary>[BIOMD0000000016_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000016#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000017](https://www.ebi.ac.uk/biomodels/BIOMD0000000017)<br/><sup>Hoefnagel2002_PyruvateBranches</sup>|<details><summary>pass</summary>[BIOMD0000000017_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000017#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000018](https://www.ebi.ac.uk/biomodels/BIOMD0000000018)<br/><sup>Morrison1989 - Folate Cycle</sup>|<details><summary>pass</summary>[BIOMD0000000018_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000018#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000019](https://www.ebi.ac.uk/biomodels/BIOMD0000000019)<br/><sup>Schoeberl2002 - EGF MAPK</sup>|<details><summary>pass</summary>[BIOMD0000000019_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000019#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000020](https://www.ebi.ac.uk/biomodels/BIOMD0000000020)<br/><sup>hodgkin-huxley squid-axon 1952</sup>|<details><summary>pass</summary>[BIOMD0000000020_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000020#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000021](https://www.ebi.ac.uk/biomodels/BIOMD0000000021)<br/><sup>Leloup1999_CircClock</sup>|<details><summary>pass</summary>[BIOMD0000000021_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000021#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000022](https://www.ebi.ac.uk/biomodels/BIOMD0000000022)<br/><sup>Ueda2001_CircClock</sup>|<details><summary>pass</summary>[BIOMD0000000022_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000022#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000023](https://www.ebi.ac.uk/biomodels/BIOMD0000000023)<br/><sup>Rohwer2001_Sucrose</sup>|<details><summary>pass</summary>[BIOMD0000000023_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000023#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000024](https://www.ebi.ac.uk/biomodels/BIOMD0000000024)<br/><sup>Scheper1999_CircClock</sup>|<details><summary>pass</summary>[BIOMD0000000024_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000024#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000025](https://www.ebi.ac.uk/biomodels/BIOMD0000000025)<br/><sup>Smolen2002_CircClock</sup>|<details><summary>pass</summary>[BIOMD0000000025_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000025#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000026](https://www.ebi.ac.uk/biomodels/BIOMD0000000026)<br/><sup>Markevich2004_MAPK_orderedElementary</sup>|<details><summary>pass</summary>[BIOMD0000000026_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000026#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000027](https://www.ebi.ac.uk/biomodels/BIOMD0000000027)<br/><sup>Markevich2004 - MAPK double ph</sup></summary><sup>Markevich2004 - MAPK double phosphorylation,  ordered Michaelis-Menton</sup></details>|<details><summary>pass</summary>[BIOMD0000000027_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000027#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000028](https://www.ebi.ac.uk/biomodels/BIOMD0000000028)<br/><sup>Markevich2004_MAPK_phosphoRand</sup></summary><sup>Markevich2004_MAPK_phosphoRandomElementary</sup></details>|<details><summary>pass</summary>[BIOMD0000000028_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000028#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000029](https://www.ebi.ac.uk/biomodels/BIOMD0000000029)<br/><sup>Markevich2004_MAPK_phosphoRandomMM</sup>|<details><summary>pass</summary>[BIOMD0000000029_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000029#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000030](https://www.ebi.ac.uk/biomodels/BIOMD0000000030)<br/><sup>Markevich2004_MAPK_AllRandomEl</sup></summary><sup>Markevich2004_MAPK_AllRandomElementary</sup></details>|<details><summary>pass</summary>[BIOMD0000000030_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000030#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000031](https://www.ebi.ac.uk/biomodels/BIOMD0000000031)<br/><sup>Markevich2004_MAPK_orderedMM2kinases</sup>|<details><summary>pass</summary>[BIOMD0000000031_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000031#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000032](https://www.ebi.ac.uk/biomodels/BIOMD0000000032)<br/><sup>Kofahl2004_PheromonePathway</sup>|<details><summary>pass</summary>[BIOMD0000000032_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000032#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000033](https://www.ebi.ac.uk/biomodels/BIOMD0000000033)<br/><sup>Brown2004 - NGF and EGF signaling</sup>|<details><summary>pass</summary>[BIOMD0000000033_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000033#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000034](https://www.ebi.ac.uk/biomodels/BIOMD0000000034)<br/><sup>Smolen2004_CircClock</sup>|<details><summary>pass</summary>[BIOMD0000000034_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000034#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000035](https://www.ebi.ac.uk/biomodels/BIOMD0000000035)<br/><sup>Vilar2002_Oscillator</sup>|<details><summary>pass</summary>[BIOMD0000000035_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000035#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000036](https://www.ebi.ac.uk/biomodels/BIOMD0000000036)<br/><sup>Tyson1999_CircClock</sup>|<details><summary>pass</summary>[BIOMD0000000036_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000036#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000037](https://www.ebi.ac.uk/biomodels/BIOMD0000000037)<br/><sup>Marwan2003 - Genetics, regulat</sup></summary><sup>Marwan2003 - Genetics, regulatory hierarchy between genes</sup></details>|<details><summary>pass</summary>[BIOMD0000000037_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000037#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000038](https://www.ebi.ac.uk/biomodels/BIOMD0000000038)<br/><sup>Rohwer2000_Phosphotransferase_System</sup>|<details><summary>pass</summary>[BIOMD0000000038_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000038#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000039](https://www.ebi.ac.uk/biomodels/BIOMD0000000039)<br/><sup>Marhl2000_CaOscillations</sup>|<details><summary>pass</summary>[BIOMD0000000039_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000039#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000040](https://www.ebi.ac.uk/biomodels/BIOMD0000000040)<br/><sup>Field1974_Oregonator</sup>|<details><summary>pass</summary>[BIOMD0000000040_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000040#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000041](https://www.ebi.ac.uk/biomodels/BIOMD0000000041)<br/><sup>Kongas2007 - Creatine Kinase i</sup></summary><sup>Kongas2007 - Creatine Kinase in energy metabolic signaling in muscle</sup></details>|<details><summary>pass</summary>[BIOMD0000000041_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000041#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000042](https://www.ebi.ac.uk/biomodels/BIOMD0000000042)<br/><sup>Nielsen1998_Glycolysis</sup>|<details><summary>pass</summary>[BIOMD0000000042_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000042#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000043](https://www.ebi.ac.uk/biomodels/BIOMD0000000043)<br/><sup>Borghans1997 - Calcium Oscilla</sup></summary><sup>Borghans1997 - Calcium Oscillation - Model 1</sup></details>|<details><summary>pass</summary>[BIOMD0000000043_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000043#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000044](https://www.ebi.ac.uk/biomodels/BIOMD0000000044)<br/><sup>Borghans1997 - Calcium Oscilla</sup></summary><sup>Borghans1997 - Calcium Oscillation - Model 2</sup></details>|<details><summary>pass</summary>[BIOMD0000000044_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000044#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000045](https://www.ebi.ac.uk/biomodels/BIOMD0000000045)<br/><sup>Borghans1997 - Calcium Oscilla</sup></summary><sup>Borghans1997 - Calcium Oscillation - Model 3</sup></details>|<details><summary>pass</summary>[BIOMD0000000045_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000045#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000046](https://www.ebi.ac.uk/biomodels/BIOMD0000000046)<br/><sup>Olsen2003_peroxidase</sup>|<details><summary>pass</summary>[BIOMD0000000046_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000046#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000047](https://www.ebi.ac.uk/biomodels/BIOMD0000000047)<br/><sup>Oxhamre2005_Ca_oscillation</sup>|<details><summary>pass</summary>[BIOMD0000000047_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000047#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000048](https://www.ebi.ac.uk/biomodels/BIOMD0000000048)<br/><sup>Kholodenko1999 - EGFR signaling</sup>|<details><summary>pass</summary>[BIOMD0000000048_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000048#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000049](https://www.ebi.ac.uk/biomodels/BIOMD0000000049)<br/><sup>Sasagawa2005_MAPK</sup>|<details><summary>pass</summary>[BIOMD0000000049_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000049#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000050](https://www.ebi.ac.uk/biomodels/BIOMD0000000050)<br/><sup>Martins2003_AmadoriDegradation</sup>|<details><summary>pass</summary>[BIOMD0000000050_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000050#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000051](https://www.ebi.ac.uk/biomodels/BIOMD0000000051)<br/><sup>Chassagnole2002_Carbon_Metabolism</sup>|<details><summary>FAIL</summary>[BIOMD0000000051_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000051#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000052](https://www.ebi.ac.uk/biomodels/BIOMD0000000052)<br/><sup>Brands2002 - Monosaccharide-ca</sup></summary><sup>Brands2002 - Monosaccharide-casein systems</sup></details>|<details><summary>pass</summary>[BIOMD0000000052_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000052#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000053](https://www.ebi.ac.uk/biomodels/BIOMD0000000053)<br/><sup>Ferreira2003_CML_generation2</sup>|<details><summary>pass</summary>[BIOMD0000000053_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000053#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000054](https://www.ebi.ac.uk/biomodels/BIOMD0000000054)<br/><sup>Ataullahkhanov1996_Adenylate</sup>|<details><summary>pass</summary>[BIOMD0000000054_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000054#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000055](https://www.ebi.ac.uk/biomodels/BIOMD0000000055)<br/><sup>Locke2005 - Circadian Clock</sup>|<details><summary>pass</summary>[BIOMD0000000055_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000055#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000056](https://www.ebi.ac.uk/biomodels/BIOMD0000000056)<br/><sup>Chen2004 - Cell Cycle Regulation</sup>|<details><summary>pass</summary>[BIOMD0000000056_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000056#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000057](https://www.ebi.ac.uk/biomodels/BIOMD0000000057)<br/><sup>Sneyd2002_IP3_Receptor</sup>|<details><summary>pass</summary>[BIOMD0000000057_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000057#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000058](https://www.ebi.ac.uk/biomodels/BIOMD0000000058)<br/><sup>Bindschadler2001_coupled_Ca_os</sup></summary><sup>Bindschadler2001_coupled_Ca_oscillators</sup></details>|<details><summary>pass</summary>[BIOMD0000000058_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000058#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000059](https://www.ebi.ac.uk/biomodels/BIOMD0000000059)<br/><sup>Fridlyand2003_Calcium_flux</sup>|<details><summary>pass</summary>[BIOMD0000000059_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000059#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000060](https://www.ebi.ac.uk/biomodels/BIOMD0000000060)<br/><sup>Keizer1996_Ryanodine_receptor_</sup></summary><sup>Keizer1996_Ryanodine_receptor_adaptation</sup></details>|<details><summary>FAIL</summary>[BIOMD0000000060_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000060#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000061](https://www.ebi.ac.uk/biomodels/BIOMD0000000061)<br/><sup>Hynne2001_Glycolysis</sup>|<details><summary>pass</summary>[BIOMD0000000061_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000061#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000062](https://www.ebi.ac.uk/biomodels/BIOMD0000000062)<br/><sup>Bhartiya2003_Tryptophan_operon</sup>|<details><summary>pass</summary>[BIOMD0000000062_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000062#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000063](https://www.ebi.ac.uk/biomodels/BIOMD0000000063)<br/><sup>Galazzo1990_FermentationPathwa</sup></summary><sup>Galazzo1990_FermentationPathwayKinetics</sup></details>|<details><summary>pass</summary>[BIOMD0000000063_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000063#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000064](https://www.ebi.ac.uk/biomodels/BIOMD0000000064)<br/><sup>Teusink2000_Glycolysis</sup>|<details><summary>FAIL</summary>[BIOMD0000000064_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000064#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000065](https://www.ebi.ac.uk/biomodels/BIOMD0000000065)<br/><sup>Yildirim2003_Lac_Operon</sup>|<details><summary>pass</summary>[BIOMD0000000065_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000065#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000066](https://www.ebi.ac.uk/biomodels/BIOMD0000000066)<br/><sup>Chassagnole2001_Threonine Synthesis</sup>|<details><summary>pass</summary>[BIOMD0000000066_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000066#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000067](https://www.ebi.ac.uk/biomodels/BIOMD0000000067)<br/><sup>Fung2005_Metabolic_Oscillator</sup>|<details><summary>pass</summary>[BIOMD0000000067_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000067#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000068](https://www.ebi.ac.uk/biomodels/BIOMD0000000068)<br/><sup>Curien2003_MetThr_synthesis</sup>|<details><summary>pass</summary>[BIOMD0000000068_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000068#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000069](https://www.ebi.ac.uk/biomodels/BIOMD0000000069)<br/><sup>Fuss2006_MitoticActivation</sup>|<details><summary>pass</summary>[BIOMD0000000069_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000069#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000070](https://www.ebi.ac.uk/biomodels/BIOMD0000000070)<br/><sup>Holzhutter2004_Erythrocyte_Met</sup></summary><sup>Holzhutter2004_Erythrocyte_Metabolism</sup></details>|<details><summary>pass</summary>[BIOMD0000000070_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000070#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000071](https://www.ebi.ac.uk/biomodels/BIOMD0000000071)<br/><sup>Bakker2001_Glycolysis</sup>|<details><summary>pass</summary>[BIOMD0000000071_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000071#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000072](https://www.ebi.ac.uk/biomodels/BIOMD0000000072)<br/><sup>Yi2003_GproteinCycle</sup>|<details><summary>pass</summary>[BIOMD0000000072_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000072#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000073](https://www.ebi.ac.uk/biomodels/BIOMD0000000073)<br/><sup>Leloup2003_CircClock_DD</sup>|<details><summary>pass</summary>[BIOMD0000000073_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000073#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000074](https://www.ebi.ac.uk/biomodels/BIOMD0000000074)<br/><sup>Leloup2003_CircClock_DD_REV-ERBalpha</sup>|<details><summary>pass</summary>[BIOMD0000000074_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000074#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000075](https://www.ebi.ac.uk/biomodels/BIOMD0000000075)<br/><sup>Xu2003 - Phosphoinositide turnover</sup>|<details><summary>pass</summary>[BIOMD0000000075_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000075#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000076](https://www.ebi.ac.uk/biomodels/BIOMD0000000076)<br/><sup>Cronwright2002_Glycerol_Synthesis</sup>|<details><summary>pass</summary>[BIOMD0000000076_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000076#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000077](https://www.ebi.ac.uk/biomodels/BIOMD0000000077)<br/><sup>Blum2000_LHsecretion_1</sup>|<details><summary>pass</summary>[BIOMD0000000077_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000077#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000078](https://www.ebi.ac.uk/biomodels/BIOMD0000000078)<br/><sup>Leloup2003_CircClock_LD</sup>|<details><summary>pass</summary>[BIOMD0000000078_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000078#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000079](https://www.ebi.ac.uk/biomodels/BIOMD0000000079)<br/><sup>Goldbeter2006_weightCycling</sup>|<details><summary>pass</summary>[BIOMD0000000079_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000079#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000080](https://www.ebi.ac.uk/biomodels/BIOMD0000000080)<br/><sup>Thomsen1989_AdenylateCyclase</sup>|<details><summary>pass</summary>[BIOMD0000000080_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000080#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000081](https://www.ebi.ac.uk/biomodels/BIOMD0000000081)<br/><sup>Suh2004_KCNQ_Regulation</sup>|<details><summary>pass</summary>[BIOMD0000000081_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000081#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000082](https://www.ebi.ac.uk/biomodels/BIOMD0000000082)<br/><sup>Thomsen1988_AdenylateCyclase_I</sup></summary><sup>Thomsen1988_AdenylateCyclase_Inhibition</sup></details>|<details><summary>pass</summary>[BIOMD0000000082_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000082#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000083](https://www.ebi.ac.uk/biomodels/BIOMD0000000083)<br/><sup>Leloup2003_CircClock_LD_REV-ERBalpha</sup>|<details><summary>pass</summary>[BIOMD0000000083_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000083#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000084](https://www.ebi.ac.uk/biomodels/BIOMD0000000084)<br/><sup>Hornberg2005_ERKcascade</sup>|<details><summary>pass</summary>[BIOMD0000000084_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000084#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000085](https://www.ebi.ac.uk/biomodels/BIOMD0000000085)<br/><sup>Maurya2005_GTPaseCycle_reducedOrder</sup>|<details><summary>pass</summary>[BIOMD0000000085_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000085#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000086](https://www.ebi.ac.uk/biomodels/BIOMD0000000086)<br/><sup>Bornheimer2004_GTPaseCycle</sup>|<details><summary>pass</summary>[BIOMD0000000086_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000086#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000087](https://www.ebi.ac.uk/biomodels/BIOMD0000000087)<br/><sup>Proctor2006_telomere</sup>|<details><summary>pass</summary>[BIOMD0000000087_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000087#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000088](https://www.ebi.ac.uk/biomodels/BIOMD0000000088)<br/><sup>Maeda2006_MyosinPhosphorylation</sup>|<details><summary>pass</summary>[BIOMD0000000088_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000088#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000089](https://www.ebi.ac.uk/biomodels/BIOMD0000000089)<br/><sup>Locke2006_CircClock_LL</sup>|<details><summary>pass</summary>[BIOMD0000000089_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000089#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000090](https://www.ebi.ac.uk/biomodels/BIOMD0000000090)<br/><sup>Wolf2001_Respiratory_Oscillations</sup>|<details><summary>pass</summary>[BIOMD0000000090_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000090#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000091](https://www.ebi.ac.uk/biomodels/BIOMD0000000091)<br/><sup>Proctor2005 - Actions of chape</sup></summary><sup>Proctor2005 - Actions of chaperones and their role in ageing</sup></details>|<details><summary>pass</summary>[BIOMD0000000091_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000091#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000092](https://www.ebi.ac.uk/biomodels/BIOMD0000000092)<br/><sup>Fuentes2005_ZymogenActivation</sup>|<details><summary>pass</summary>[BIOMD0000000092_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000092#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000093](https://www.ebi.ac.uk/biomodels/BIOMD0000000093)<br/><sup>Yamada2003_JAK_STAT_pathway</sup>|<details><summary>pass</summary>[BIOMD0000000093_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000093#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000094](https://www.ebi.ac.uk/biomodels/BIOMD0000000094)<br/><sup>Yamada2003_JAK_STAT_SOCS1_knockout</sup>|<details><summary>pass</summary>[BIOMD0000000094_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000094#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000095](https://www.ebi.ac.uk/biomodels/BIOMD0000000095)<br/><sup>Zeilinger2006_PRR7-PRR9-Y</sup>|<details><summary>pass</summary>[BIOMD0000000095_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000095#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000096](https://www.ebi.ac.uk/biomodels/BIOMD0000000096)<br/><sup>Zeilinger2006_PRR7-PRR9light-Y</sup>|<details><summary>FAIL</summary>[BIOMD0000000096_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000096#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000097](https://www.ebi.ac.uk/biomodels/BIOMD0000000097)<br/><sup>Zeilinger2006_PRR7-PRR9light-Yprime</sup>|<details><summary>FAIL</summary>[BIOMD0000000097_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000097#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000098](https://www.ebi.ac.uk/biomodels/BIOMD0000000098)<br/><sup>Goldbeter1990_CalciumSpike_CICR</sup>|<details><summary>pass</summary>[BIOMD0000000098_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000098#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000099](https://www.ebi.ac.uk/biomodels/BIOMD0000000099)<br/><sup>Laub1998_SpontaneousOscillations</sup>|<details><summary>pass</summary>[BIOMD0000000099_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000099#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000100](https://www.ebi.ac.uk/biomodels/BIOMD0000000100)<br/><sup>Rozi2003_GlycogenPhosphorylase</sup></summary><sup>Rozi2003_GlycogenPhosphorylase_Activation</sup></details>|<details><summary>pass</summary>[BIOMD0000000100_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000100#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000101](https://www.ebi.ac.uk/biomodels/BIOMD0000000101)<br/><sup>Vilar2006_TGFbeta</sup>|<details><summary>pass</summary>[BIOMD0000000101_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000101#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000102](https://www.ebi.ac.uk/biomodels/BIOMD0000000102)<br/><sup>Legewie2006_apoptosis_WT</sup>|<details><summary>pass</summary>[BIOMD0000000102_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000102#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000103](https://www.ebi.ac.uk/biomodels/BIOMD0000000103)<br/><sup>Legewie2006_apoptosis_NC</sup>|<details><summary>pass</summary>[BIOMD0000000103_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000103#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000104](https://www.ebi.ac.uk/biomodels/BIOMD0000000104)<br/><sup>Klipp2002_MetabolicOptimizatio</sup></summary><sup>Klipp2002_MetabolicOptimization_linearPathway(n=2)</sup></details>|<details><summary>pass</summary>[BIOMD0000000104_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000104#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000105](https://www.ebi.ac.uk/biomodels/BIOMD0000000105)<br/><sup>Proctor2007 - Age related decl</sup></summary><sup>Proctor2007 - Age related decline of proteolysis, ubiquitin-proteome system</sup></details>|<details><summary>pass</summary>[BIOMD0000000105_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000105#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000106](https://www.ebi.ac.uk/biomodels/BIOMD0000000106)<br/><sup>Yang2007_ArachidonicAcid</sup>|<details><summary>pass</summary>[BIOMD0000000106_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000106#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000107](https://www.ebi.ac.uk/biomodels/BIOMD0000000107)<br/><sup>Novak1993 - Cell cycle M-phase</sup></summary><sup>Novak1993 - Cell cycle M-phase control</sup></details>|<details><summary>pass</summary>[BIOMD0000000107_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000107#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000108](https://www.ebi.ac.uk/biomodels/BIOMD0000000108)<br/><sup>Kowald2006_SOD</sup>|<details><summary>pass</summary>[BIOMD0000000108_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000108#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000109](https://www.ebi.ac.uk/biomodels/BIOMD0000000109)<br/><sup>Haberichter2007_cellcycle</sup>|<details><summary>pass</summary>[BIOMD0000000109_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000109#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000110](https://www.ebi.ac.uk/biomodels/BIOMD0000000110)<br/><sup>Qu2003_CellCycle</sup>|<details><summary>pass</summary>[BIOMD0000000110_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000110#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000111](https://www.ebi.ac.uk/biomodels/BIOMD0000000111)<br/><sup>Novak2001_FissionYeast_CellCycle</sup>|<details><summary>pass</summary>[BIOMD0000000111_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000111#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000112](https://www.ebi.ac.uk/biomodels/BIOMD0000000112)<br/><sup>Clarke2006_Smad_signalling</sup>|<details><summary>pass</summary>[BIOMD0000000112_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000112#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000113](https://www.ebi.ac.uk/biomodels/BIOMD0000000113)<br/><sup>Dupont1992_Ca_dpt_protein_phospho</sup>|<details><summary>pass</summary>[BIOMD0000000113_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000113#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000114](https://www.ebi.ac.uk/biomodels/BIOMD0000000114)<br/><sup>Somogyi1990_CaOscillations</sup>|<details><summary>pass</summary>[BIOMD0000000114_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000114#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000115](https://www.ebi.ac.uk/biomodels/BIOMD0000000115)<br/><sup>Somogyi1990_CaOscillations_Sin</sup></summary><sup>Somogyi1990_CaOscillations_SingleCaSpike</sup></details>|<details><summary>pass</summary>[BIOMD0000000115_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000115#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000116](https://www.ebi.ac.uk/biomodels/BIOMD0000000116)<br/><sup>McClean2007_CrossTalk</sup>|<details><summary>pass</summary>[BIOMD0000000116_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000116#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000117](https://www.ebi.ac.uk/biomodels/BIOMD0000000117)<br/><sup>Dupont1991_CaOscillation</sup>|<details><summary>pass</summary>[BIOMD0000000117_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000117#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000118](https://www.ebi.ac.uk/biomodels/BIOMD0000000118)<br/><sup>Golomb2006_SomaticBursting</sup>|<details><summary>pass</summary>[BIOMD0000000118_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000118#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000119](https://www.ebi.ac.uk/biomodels/BIOMD0000000119)<br/><sup>Golomb2006_SomaticBursting_non</sup></summary><sup>Golomb2006_SomaticBursting_nonzero[Ca]</sup></details>|<details><summary>pass</summary>[BIOMD0000000119_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000119#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000120](https://www.ebi.ac.uk/biomodels/BIOMD0000000120)<br/><sup>Chan2004_TCell_receptor_activation</sup>|<details><summary>pass</summary>[BIOMD0000000120_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000120#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000121](https://www.ebi.ac.uk/biomodels/BIOMD0000000121)<br/><sup>Clancy2001_Kchannel</sup>|<details><summary>pass</summary>[BIOMD0000000121_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000121#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000122](https://www.ebi.ac.uk/biomodels/BIOMD0000000122)<br/><sup>Fisher2006_Ca_Oscillation_dpdn</sup></summary><sup>Fisher2006_Ca_Oscillation_dpdnt_NFAT_dynamics</sup></details>|<details><summary>pass</summary>[BIOMD0000000122_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000122#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000123](https://www.ebi.ac.uk/biomodels/BIOMD0000000123)<br/><sup>Fisher2006_NFAT_Activation</sup>|<details><summary>pass</summary>[BIOMD0000000123_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000123#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000124](https://www.ebi.ac.uk/biomodels/BIOMD0000000124)<br/><sup>Wu2006_K+Channel</sup>|<details><summary>pass</summary>[BIOMD0000000124_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000124#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000125](https://www.ebi.ac.uk/biomodels/BIOMD0000000125)<br/><sup>Komarova2005_TheoreticalFramew</sup></summary><sup>Komarova2005_TheoreticalFramework_BasicArchitecture</sup></details>|<details><summary>pass</summary>[BIOMD0000000125_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000125#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000126](https://www.ebi.ac.uk/biomodels/BIOMD0000000126)<br/><sup>Clancy2002_CardiacSodiumChannel_WT</sup>|<details><summary>pass</summary>[BIOMD0000000126_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000126#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000127](https://www.ebi.ac.uk/biomodels/BIOMD0000000127)<br/><sup>Izhikevich2003_SpikingNeuron</sup>|<details><summary>pass</summary>[BIOMD0000000127_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000127#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000128](https://www.ebi.ac.uk/biomodels/BIOMD0000000128)<br/><sup>Bertram2006_Endothelin</sup>|<details><summary>pass</summary>[BIOMD0000000128_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000128#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000129](https://www.ebi.ac.uk/biomodels/BIOMD0000000129)<br/><sup>Izhikevich2004_SpikingNeurons_</sup></summary><sup>Izhikevich2004_SpikingNeurons_inhibitionInducedSpiking</sup></details>|<details><summary>pass</summary>[BIOMD0000000129_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000129#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000130](https://www.ebi.ac.uk/biomodels/BIOMD0000000130)<br/><sup>Izhikevich2004_SpikingNeurons_</sup></summary><sup>Izhikevich2004_SpikingNeurons_integrator</sup></details>|<details><summary>pass</summary>[BIOMD0000000130_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000130#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000131](https://www.ebi.ac.uk/biomodels/BIOMD0000000131)<br/><sup>Izhikevich2004_SpikingNeurons_</sup></summary><sup>Izhikevich2004_SpikingNeurons_reboundBurst</sup></details>|<details><summary>pass</summary>[BIOMD0000000131_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000131#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000132](https://www.ebi.ac.uk/biomodels/BIOMD0000000132)<br/><sup>Izhikevich2004_SpikingNeurons_</sup></summary><sup>Izhikevich2004_SpikingNeurons_reboundSpike</sup></details>|<details><summary>pass</summary>[BIOMD0000000132_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000132#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000133](https://www.ebi.ac.uk/biomodels/BIOMD0000000133)<br/><sup>Izhikevich2004_SpikingNeurons_</sup></summary><sup>Izhikevich2004_SpikingNeurons_resonator</sup></details>|<details><summary>pass</summary>[BIOMD0000000133_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000133#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000134](https://www.ebi.ac.uk/biomodels/BIOMD0000000134)<br/><sup>Izhikevich2004_SpikingNeurons_</sup></summary><sup>Izhikevich2004_SpikingNeurons_SpikeLatency</sup></details>|<details><summary>pass</summary>[BIOMD0000000134_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000134#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000135](https://www.ebi.ac.uk/biomodels/BIOMD0000000135)<br/><sup>Izhikevich2004_SpikingNeurons_</sup></summary><sup>Izhikevich2004_SpikingNeurons_subthresholdOscillations</sup></details>|<details><summary>pass</summary>[BIOMD0000000135_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000135#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000136](https://www.ebi.ac.uk/biomodels/BIOMD0000000136)<br/><sup>Izhikevich2004_SpikingNeurons_</sup></summary><sup>Izhikevich2004_SpikingNeurons_thresholdVariability</sup></details>|<details><summary>pass</summary>[BIOMD0000000136_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000136#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000137](https://www.ebi.ac.uk/biomodels/BIOMD0000000137)<br/><sup>Sedaghat2002_InsulinSignalling</sup></summary><sup>Sedaghat2002_InsulinSignalling_noFeedback</sup></details>|<details><summary>pass</summary>[BIOMD0000000137_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000137#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000138](https://www.ebi.ac.uk/biomodels/BIOMD0000000138)<br/><sup>Tabak2007_dopamine</sup>|<details><summary>pass</summary>[BIOMD0000000138_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000138#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000139](https://www.ebi.ac.uk/biomodels/BIOMD0000000139)<br/><sup>Hoffmann2002_KnockOut_IkBNFkB_</sup></summary><sup>Hoffmann2002_KnockOut_IkBNFkB_Signaling</sup></details>|<details><summary>pass</summary>[BIOMD0000000139_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000139#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000140](https://www.ebi.ac.uk/biomodels/BIOMD0000000140)<br/><sup>Hoffmann2002_WT_IkBNFkB_Signaling</sup>|<details><summary>pass</summary>[BIOMD0000000140_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000140#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000141](https://www.ebi.ac.uk/biomodels/BIOMD0000000141)<br/><sup>Izhikevich2004_SpikingNeurons_</sup></summary><sup>Izhikevich2004_SpikingNeurons_Class1Excitable</sup></details>|<details><summary>pass</summary>[BIOMD0000000141_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000141#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000142](https://www.ebi.ac.uk/biomodels/BIOMD0000000142)<br/><sup>Izhikevich2004_SpikingNeurons_</sup></summary><sup>Izhikevich2004_SpikingNeurons_Class2Excitable</sup></details>|<details><summary>pass</summary>[BIOMD0000000142_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000142#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000143](https://www.ebi.ac.uk/biomodels/BIOMD0000000143)<br/><sup>Olsen2003_neutrophil_oscillato</sup></summary><sup>Olsen2003_neutrophil_oscillatory_metabolism</sup></details>|<details><summary>pass</summary>[BIOMD0000000143_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000143#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000144](https://www.ebi.ac.uk/biomodels/BIOMD0000000144)<br/><sup>Calzone2007_CellCycle</sup>|<details><summary>pass</summary>[BIOMD0000000144_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000144#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000145](https://www.ebi.ac.uk/biomodels/BIOMD0000000145)<br/><sup>Wang2007 - ATP induced intrace</sup></summary><sup>Wang2007 - ATP induced intracellular Calcium Oscillation</sup></details>|<details><summary>pass</summary>[BIOMD0000000145_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000145#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000146](https://www.ebi.ac.uk/biomodels/BIOMD0000000146)<br/><sup>Hatakeyama2003_MAPK</sup>|<details><summary>pass</summary>[BIOMD0000000146_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000146#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000147](https://www.ebi.ac.uk/biomodels/BIOMD0000000147)<br/><sup>ODea2007_IkappaB</sup>|<details><summary>pass</summary>[BIOMD0000000147_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000147#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000148](https://www.ebi.ac.uk/biomodels/BIOMD0000000148)<br/><sup>Komarova2003_BoneRemodeling</sup>|<details><summary>pass</summary>[BIOMD0000000148_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000148#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000149](https://www.ebi.ac.uk/biomodels/BIOMD0000000149)<br/><sup>Kim2007 - Crosstalk between Wn</sup></summary><sup>Kim2007 - Crosstalk between Wnt and ERK pathways</sup></details>|<details><summary>pass</summary>[BIOMD0000000149_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000149#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000150](https://www.ebi.ac.uk/biomodels/BIOMD0000000150)<br/><sup>Morris2002_CellCycle_CDK2Cyclin</sup>|<details><summary>pass</summary>[BIOMD0000000150_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000150#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000151](https://www.ebi.ac.uk/biomodels/BIOMD0000000151)<br/><sup>Singh2006_IL6_Signal_Transduction</sup>|<details><summary>pass</summary>[BIOMD0000000151_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000151#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000152](https://www.ebi.ac.uk/biomodels/BIOMD0000000152)<br/><sup>Fernandez2006_ModelA</sup>|<details><summary>pass</summary>[BIOMD0000000152_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000152#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000153](https://www.ebi.ac.uk/biomodels/BIOMD0000000153)<br/><sup>Fernandez2006_ModelB</sup>|<details><summary>pass</summary>[BIOMD0000000153_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000153#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000154](https://www.ebi.ac.uk/biomodels/BIOMD0000000154)<br/><sup>Zatorsky2006_p53_Model3</sup>|<details><summary>pass</summary>[BIOMD0000000154_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000154#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000155](https://www.ebi.ac.uk/biomodels/BIOMD0000000155)<br/><sup>Zatorsky2006_p53_Model6</sup>|<details><summary>pass</summary>[BIOMD0000000155_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000155#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000156](https://www.ebi.ac.uk/biomodels/BIOMD0000000156)<br/><sup>Zatorsky2006_p53_Model5</sup>|<details><summary>pass</summary>[BIOMD0000000156_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000156#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000157](https://www.ebi.ac.uk/biomodels/BIOMD0000000157)<br/><sup>Zatorsky2006_p53_Model4</sup>|<details><summary>pass</summary>[BIOMD0000000157_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000157#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000158](https://www.ebi.ac.uk/biomodels/BIOMD0000000158)<br/><sup>Zatorsky2006_p53_Model2</sup>|<details><summary>pass</summary>[BIOMD0000000158_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000158#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000159](https://www.ebi.ac.uk/biomodels/BIOMD0000000159)<br/><sup>Zatorsky2006_p53_Model1</sup>|<details><summary>pass</summary>[BIOMD0000000159_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000159#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000160](https://www.ebi.ac.uk/biomodels/BIOMD0000000160)<br/><sup>Xie2007_CircClock</sup>|<details><summary>pass</summary>[BIOMD0000000160_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000160#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000161](https://www.ebi.ac.uk/biomodels/BIOMD0000000161)<br/><sup>Eungdamrong2007_Ras_Activation</sup>|<details><summary>pass</summary>[BIOMD0000000161_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000161#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000162](https://www.ebi.ac.uk/biomodels/BIOMD0000000162)<br/><sup>Hernjak2005_Calcium_Signaling</sup>|<details><summary>pass</summary>[BIOMD0000000162_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000162#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000163](https://www.ebi.ac.uk/biomodels/BIOMD0000000163)<br/><sup>Zi2007_TGFbeta_signaling</sup>|<details><summary>pass</summary>[BIOMD0000000163_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000163#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000164](https://www.ebi.ac.uk/biomodels/BIOMD0000000164)<br/><sup>SmithAE2002_RanTransport</sup>|<details><summary>FAIL</summary>[BIOMD0000000164_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000164#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000165](https://www.ebi.ac.uk/biomodels/BIOMD0000000165)<br/><sup>Saucerman2006_PKA</sup>|<details><summary>pass</summary>[BIOMD0000000165_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000165#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000166](https://www.ebi.ac.uk/biomodels/BIOMD0000000166)<br/><sup>Zhu2007_TF_modulated_by_Calcium</sup>|<details><summary>pass</summary>[BIOMD0000000166_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000166#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000167](https://www.ebi.ac.uk/biomodels/BIOMD0000000167)<br/><sup>Mayya2005_STATmodule</sup>|<details><summary>pass</summary>[BIOMD0000000167_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000167#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000168](https://www.ebi.ac.uk/biomodels/BIOMD0000000168)<br/><sup>Obeyesekere1999_CellCycle</sup>|<details><summary>pass</summary>[BIOMD0000000168_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000168#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000169](https://www.ebi.ac.uk/biomodels/BIOMD0000000169)<br/><sup>Aguda1999_CellCycle</sup>|<details><summary>pass</summary>[BIOMD0000000169_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000169#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000170](https://www.ebi.ac.uk/biomodels/BIOMD0000000170)<br/><sup>Weimann2004_CircadianOscillator</sup>|<details><summary>pass</summary>[BIOMD0000000170_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000170#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000171](https://www.ebi.ac.uk/biomodels/BIOMD0000000171)<br/><sup>Leloup1998_CircClock_LD</sup>|<details><summary>pass</summary>[BIOMD0000000171_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000171#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000172](https://www.ebi.ac.uk/biomodels/BIOMD0000000172)<br/><sup>Pritchard2002_glycolysis</sup>|<details><summary>pass</summary>[BIOMD0000000172_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000172#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000173](https://www.ebi.ac.uk/biomodels/BIOMD0000000173)<br/><sup>Schmierer_2008_Smad_Tgfb</sup>|<details><summary>pass</summary>[BIOMD0000000173_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000173#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000174](https://www.ebi.ac.uk/biomodels/BIOMD0000000174)<br/><sup>Del_Conte_Zerial2008_Rab5_Rab7</sup></summary><sup>Del_Conte_Zerial2008_Rab5_Rab7_cut_out_switch</sup></details>|<details><summary>pass</summary>[BIOMD0000000174_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000174#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000175](https://www.ebi.ac.uk/biomodels/BIOMD0000000175)<br/><sup>Birtwistle2007_ErbB_Signalling</sup>|<details><summary>pass</summary>[BIOMD0000000175_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000175#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000176](https://www.ebi.ac.uk/biomodels/BIOMD0000000176)<br/><sup>Conant2007_WGD_glycolysis_2A3AB</sup>|<details><summary>pass</summary>[BIOMD0000000176_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000176#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000177](https://www.ebi.ac.uk/biomodels/BIOMD0000000177)<br/><sup>Conant2007_glycolysis_2C</sup>|<details><summary>pass</summary>[BIOMD0000000177_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000177#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000178](https://www.ebi.ac.uk/biomodels/BIOMD0000000178)<br/><sup>Lebeda2008 - BoTN Paralysis (4</sup></summary><sup>Lebeda2008 - BoTN Paralysis (4 step model)</sup></details>|<details><summary>pass</summary>[BIOMD0000000178_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000178#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000179](https://www.ebi.ac.uk/biomodels/BIOMD0000000179)<br/><sup>Kim2007_CellularMemory_Asymmet</sup></summary><sup>Kim2007_CellularMemory_AsymmetricModel</sup></details>|<details><summary>pass</summary>[BIOMD0000000179_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000179#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000180](https://www.ebi.ac.uk/biomodels/BIOMD0000000180)<br/><sup>Kim2007_CellularMemory_Symmetr</sup></summary><sup>Kim2007_CellularMemory_SymmetricModel</sup></details>|<details><summary>pass</summary>[BIOMD0000000180_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000180#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000181](https://www.ebi.ac.uk/biomodels/BIOMD0000000181)<br/><sup>Sriram2007_CellCycle</sup>|<details><summary>pass</summary>[BIOMD0000000181_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000181#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000182](https://www.ebi.ac.uk/biomodels/BIOMD0000000182)<br/><sup>Neves2008 - Role of cell shape</sup></summary><sup>Neves2008 - Role of cell shape and size in controlling intracellular signalling</sup></details>|<details><summary>pass</summary>[BIOMD0000000182_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000182#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000183](https://www.ebi.ac.uk/biomodels/BIOMD0000000183)<br/><sup>Stefan2008 - calmodulin allostery</sup>|<details><summary>pass</summary>[BIOMD0000000183_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000183#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000184](https://www.ebi.ac.uk/biomodels/BIOMD0000000184)<br/><sup>Lavrentovich2008_Ca_Oscillations</sup>|<details><summary>pass</summary>[BIOMD0000000184_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000184#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000185](https://www.ebi.ac.uk/biomodels/BIOMD0000000185)<br/><sup>Locke2008_Circadian_Clock</sup>|<details><summary>pass</summary>[BIOMD0000000185_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000185#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000186](https://www.ebi.ac.uk/biomodels/BIOMD0000000186)<br/><sup>Ibrahim2008 - Mitotic Spindle </sup></summary><sup>Ibrahim2008 - Mitotic Spindle Assembly Checkpoint - Dissociation variant</sup></details>|<details><summary>pass</summary>[BIOMD0000000186_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000186#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000187](https://www.ebi.ac.uk/biomodels/BIOMD0000000187)<br/><sup>Ibrahim2008 - Mitotic Spindle </sup></summary><sup>Ibrahim2008 - Mitotic Spindle Assembly Checkpoint - Convey variant</sup></details>|<details><summary>pass</summary>[BIOMD0000000187_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000187#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000188](https://www.ebi.ac.uk/biomodels/BIOMD0000000188)<br/><sup>Proctor2008 - p53/Mdm2 circuit</sup></summary><sup>Proctor2008 - p53/Mdm2 circuit - p53 stabilisation by ATM</sup></details>|<details><summary>pass</summary>[BIOMD0000000188_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000188#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000189](https://www.ebi.ac.uk/biomodels/BIOMD0000000189)<br/><sup>Proctor2008 - p53/Mdm2 circuit</sup></summary><sup>Proctor2008 - p53/Mdm2 circuit - p53 stablisation by p14ARF</sup></details>|<details><summary>pass</summary>[BIOMD0000000189_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000189#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000190](https://www.ebi.ac.uk/biomodels/BIOMD0000000190)<br/><sup>Rodriguez-Caso2006_Polyamine_M</sup></summary><sup>Rodriguez-Caso2006_Polyamine_Metabolism</sup></details>|<details><summary>pass</summary>[BIOMD0000000190_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000190#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000191](https://www.ebi.ac.uk/biomodels/BIOMD0000000191)<br/><sup>Montaez2008_Arginine_catabolism</sup>|<details><summary>pass</summary>[BIOMD0000000191_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000191#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000192](https://www.ebi.ac.uk/biomodels/BIOMD0000000192)<br/><sup>Grlich2003_RanGTP_gradient</sup>|<details><summary>pass</summary>[BIOMD0000000192_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000192#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000193](https://www.ebi.ac.uk/biomodels/BIOMD0000000193)<br/><sup>Ibrahim2008_MCC_assembly_model_KDM</sup>|<details><summary>pass</summary>[BIOMD0000000193_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000193#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000194](https://www.ebi.ac.uk/biomodels/BIOMD0000000194)<br/><sup>Ibrahim2008_Cdc20_Sequestring_</sup></summary><sup>Ibrahim2008_Cdc20_Sequestring_Template_Model</sup></details>|<details><summary>pass</summary>[BIOMD0000000194_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000194#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000195](https://www.ebi.ac.uk/biomodels/BIOMD0000000195)<br/><sup>Tyson2001_Cell_Cycle_Regulation</sup>|<details><summary>pass</summary>[BIOMD0000000195_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000195#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000196](https://www.ebi.ac.uk/biomodels/BIOMD0000000196)<br/><sup>Srividhya2006_CellCycle</sup>|<details><summary>pass</summary>[BIOMD0000000196_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000196#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000197](https://www.ebi.ac.uk/biomodels/BIOMD0000000197)<br/><sup>Bartholome2007_MDCKII</sup>|<details><summary>pass</summary>[BIOMD0000000197_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000197#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000198](https://www.ebi.ac.uk/biomodels/BIOMD0000000198)<br/><sup>Stone1996 - activation of solu</sup></summary><sup>Stone1996 - activation of soluble guanylate cyclase by nitric oxide</sup></details>|<details><summary>pass</summary>[BIOMD0000000198_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000198#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000199](https://www.ebi.ac.uk/biomodels/BIOMD0000000199)<br/><sup>Santolini2001_nNOS_Mechanism_R</sup></summary><sup>Santolini2001_nNOS_Mechanism_Regulation</sup></details>|<details><summary>pass</summary>[BIOMD0000000199_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000199#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000200](https://www.ebi.ac.uk/biomodels/BIOMD0000000200)<br/><sup>Bray1995_chemotaxis_receptorli</sup></summary><sup>Bray1995_chemotaxis_receptorlinkedcomplex</sup></details>|<details><summary>FAIL</summary>[BIOMD0000000200_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000200#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000201](https://www.ebi.ac.uk/biomodels/BIOMD0000000201)<br/><sup>Goldbeter2008_Somite_Segmentat</sup></summary><sup>Goldbeter2008_Somite_Segmentation_Clock_Notch_Wnt_FGF</sup></details>|<details><summary>pass</summary>[BIOMD0000000201_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000201#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000202](https://www.ebi.ac.uk/biomodels/BIOMD0000000202)<br/><sup>ChenXF2008_CICR</sup>|<details><summary>pass</summary>[BIOMD0000000202_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000202#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000203](https://www.ebi.ac.uk/biomodels/BIOMD0000000203)<br/><sup>Chickarmane2006 - Stem cell sw</sup></summary><sup>Chickarmane2006 - Stem cell switch reversible</sup></details>|<details><summary>pass</summary>[BIOMD0000000203_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000203#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000204](https://www.ebi.ac.uk/biomodels/BIOMD0000000204)<br/><sup>Chickarmane2006 - Stem cell sw</sup></summary><sup>Chickarmane2006 - Stem cell switch irreversible</sup></details>|<details><summary>pass</summary>[BIOMD0000000204_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000204#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000205](https://www.ebi.ac.uk/biomodels/BIOMD0000000205)<br/><sup>Ung2008_EGFR_Endocytosis</sup>|<details><summary>pass</summary>[BIOMD0000000205_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000205#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000206](https://www.ebi.ac.uk/biomodels/BIOMD0000000206)<br/><sup>Wolf2000_Glycolytic_Oscillations</sup>|<details><summary>pass</summary>[BIOMD0000000206_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000206#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000207](https://www.ebi.ac.uk/biomodels/BIOMD0000000207)<br/><sup>Romond1999_CellCycle</sup>|<details><summary>FAIL</summary>[BIOMD0000000207_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000207#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000208](https://www.ebi.ac.uk/biomodels/BIOMD0000000208)<br/><sup>Deineko2003_CellCycle</sup>|<details><summary>pass</summary>[BIOMD0000000208_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000208#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000209](https://www.ebi.ac.uk/biomodels/BIOMD0000000209)<br/><sup>Chickarmane2008 - Stem cell li</sup></summary><sup>Chickarmane2008 - Stem cell lineage determination</sup></details>|<details><summary>pass</summary>[BIOMD0000000209_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000209#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000210](https://www.ebi.ac.uk/biomodels/BIOMD0000000210)<br/><sup>Chickarmane2008 - Stem cell li</sup></summary><sup>Chickarmane2008 - Stem cell lineage - NANOG GATA-6 switch</sup></details>|<details><summary>pass</summary>[BIOMD0000000210_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000210#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000211](https://www.ebi.ac.uk/biomodels/BIOMD0000000211)<br/><sup>Albert2005_Glycolysis</sup>|<details><summary>pass</summary>[BIOMD0000000211_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000211#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000212](https://www.ebi.ac.uk/biomodels/BIOMD0000000212)<br/><sup>Curien2009_Aspartate_Metabolism</sup>|<details><summary>pass</summary>[BIOMD0000000212_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000212#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000213](https://www.ebi.ac.uk/biomodels/BIOMD0000000213)<br/><sup>Nijhout2004_Folate_Cycle</sup>|<details><summary>pass</summary>[BIOMD0000000213_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000213#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000214](https://www.ebi.ac.uk/biomodels/BIOMD0000000214)<br/><sup>Akman2008_Circadian_Clock_Model2</sup>|<details><summary>pass</summary>[BIOMD0000000214_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000214#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000215](https://www.ebi.ac.uk/biomodels/BIOMD0000000215)<br/><sup>Schulz2009_Th1_differentiation</sup>|<details><summary>pass</summary>[BIOMD0000000215_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000215#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000216](https://www.ebi.ac.uk/biomodels/BIOMD0000000216)<br/><sup>Hong2009_CircadianClock</sup>|<details><summary>pass</summary>[BIOMD0000000216_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000216#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000217](https://www.ebi.ac.uk/biomodels/BIOMD0000000217)<br/><sup>Bruggeman2005_AmmoniumAssimilation</sup>|<details><summary>pass</summary>[BIOMD0000000217_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000217#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000218](https://www.ebi.ac.uk/biomodels/BIOMD0000000218)<br/><sup>Singh2006_TCA_mtu_model2</sup>|<details><summary>pass</summary>[BIOMD0000000218_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000218#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000219](https://www.ebi.ac.uk/biomodels/BIOMD0000000219)<br/><sup>Singh2006_TCA_mtu_model1</sup>|<details><summary>pass</summary>[BIOMD0000000219_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000219#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000220](https://www.ebi.ac.uk/biomodels/BIOMD0000000220)<br/><sup>Albeck2008_extrinsic_apoptosis</sup>|<details><summary>pass</summary>[BIOMD0000000220_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000220#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000221](https://www.ebi.ac.uk/biomodels/BIOMD0000000221)<br/><sup>Singh2006_TCA_Ecoli_acetate</sup>|<details><summary>pass</summary>[BIOMD0000000221_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000221#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000222](https://www.ebi.ac.uk/biomodels/BIOMD0000000222)<br/><sup>Singh2006_TCA_Ecoli_glucose</sup>|<details><summary>pass</summary>[BIOMD0000000222_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000222#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000223](https://www.ebi.ac.uk/biomodels/BIOMD0000000223)<br/><sup>Borisov2009_EGF_Insulin_Crosstalk</sup>|<details><summary>pass</summary>[BIOMD0000000223_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000223#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000224](https://www.ebi.ac.uk/biomodels/BIOMD0000000224)<br/><sup>Meyer1991_CalciumSpike_ICC</sup>|<details><summary>pass</summary>[BIOMD0000000224_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000224#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000225](https://www.ebi.ac.uk/biomodels/BIOMD0000000225)<br/><sup>Westermark2003_Pancreatic_Glyc</sup></summary><sup>Westermark2003_Pancreatic_GlycOsc_basic</sup></details>|<details><summary>pass</summary>[BIOMD0000000225_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000225#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000226](https://www.ebi.ac.uk/biomodels/BIOMD0000000226)<br/><sup>Radulescu2008_NFkB_hierarchy_M</sup></summary><sup>Radulescu2008_NFkB_hierarchy_M_14_25_28_Lipniacky</sup></details>|<details><summary>pass</summary>[BIOMD0000000226_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000226#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000227](https://www.ebi.ac.uk/biomodels/BIOMD0000000227)<br/><sup>Radulescu2008_NFkB_hierarchy_M</sup></summary><sup>Radulescu2008_NFkB_hierarchy_M_39_65_90</sup></details>|<details><summary>pass</summary>[BIOMD0000000227_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000227#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000228](https://www.ebi.ac.uk/biomodels/BIOMD0000000228)<br/><sup>Swat2004_Mammalian_G1_S_Transition</sup>|<details><summary>pass</summary>[BIOMD0000000228_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000228#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000229](https://www.ebi.ac.uk/biomodels/BIOMD0000000229)<br/><sup>Ma2002_cAMP_oscillations</sup>|<details><summary>pass</summary>[BIOMD0000000229_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000229#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000230](https://www.ebi.ac.uk/biomodels/BIOMD0000000230)<br/><sup>Ihekwaba2004_NFkB_Sensitivity</sup>|<details><summary>pass</summary>[BIOMD0000000230_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000230#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000231](https://www.ebi.ac.uk/biomodels/BIOMD0000000231)<br/><sup>Valero2006_Adenine_TernaryCycle</sup>|<details><summary>pass</summary>[BIOMD0000000231_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000231#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000232](https://www.ebi.ac.uk/biomodels/BIOMD0000000232)<br/><sup>Nazaret2009_TCA_RC_ATP</sup>|<details><summary>pass</summary>[BIOMD0000000232_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000232#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000233](https://www.ebi.ac.uk/biomodels/BIOMD0000000233)<br/><sup>Wilhelm2009_BistableReaction</sup>|<details><summary>pass</summary>[BIOMD0000000233_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000233#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000234](https://www.ebi.ac.uk/biomodels/BIOMD0000000234)<br/><sup>Tham2008 - PDmodel, Tumour shr</sup></summary><sup>Tham2008 - PDmodel, Tumour shrinkage by gemcitabine and carboplatin</sup></details>|<details><summary>pass</summary>[BIOMD0000000234_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000234#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000235](https://www.ebi.ac.uk/biomodels/BIOMD0000000235)<br/><sup>Kuhn2009_EndoMesodermNetwork</sup>|<details><summary>pass</summary>[BIOMD0000000235_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000235#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000236](https://www.ebi.ac.uk/biomodels/BIOMD0000000236)<br/><sup>Westermark2003_Pancreatic_Glyc</sup></summary><sup>Westermark2003_Pancreatic_GlycOsc_extended</sup></details>|<details><summary>pass</summary>[BIOMD0000000236_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000236#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000237](https://www.ebi.ac.uk/biomodels/BIOMD0000000237)<br/><sup>Schaber2006_Pheromone_Starvati</sup></summary><sup>Schaber2006_Pheromone_Starvation_Crosstalk</sup></details>|<details><summary>pass</summary>[BIOMD0000000237_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000237#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000238](https://www.ebi.ac.uk/biomodels/BIOMD0000000238)<br/><sup>Overgaard2007_PDmodel_IL21</sup>|<details><summary>pass</summary>[BIOMD0000000238_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000238#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000239](https://www.ebi.ac.uk/biomodels/BIOMD0000000239)<br/><sup>Jiang2007 - GSIS system, Pancr</sup></summary><sup>Jiang2007 - GSIS system, Pancreatic Beta Cells</sup></details>|<details><summary>pass</summary>[BIOMD0000000239_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000239#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000240](https://www.ebi.ac.uk/biomodels/BIOMD0000000240)<br/><sup>Veening2008_DegU_Regulation</sup>|<details><summary>pass</summary>[BIOMD0000000240_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000240#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000241](https://www.ebi.ac.uk/biomodels/BIOMD0000000241)<br/><sup>Shi1993_Caffeine_pressor_tolerance</sup>|<details><summary>pass</summary>[BIOMD0000000241_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000241#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000242](https://www.ebi.ac.uk/biomodels/BIOMD0000000242)<br/><sup>Bai2003_G1phaseRegulation</sup>|<details><summary>pass</summary>[BIOMD0000000242_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000242#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000243](https://www.ebi.ac.uk/biomodels/BIOMD0000000243)<br/><sup>Neumann2010_CD95Stimulation_NF</sup></summary><sup>Neumann2010_CD95Stimulation_NFkB_Apoptosis</sup></details>|<details><summary>pass</summary>[BIOMD0000000243_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000243#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000244](https://www.ebi.ac.uk/biomodels/BIOMD0000000244)<br/><sup>Kotte2010_Ecoli_Metabolic_Adaption</sup>|<details><summary>pass</summary>[BIOMD0000000244_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000244#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000245](https://www.ebi.ac.uk/biomodels/BIOMD0000000245)<br/><sup>Lei2001_Yeast_Aerobic_Metabolism</sup>|<details><summary>pass</summary>[BIOMD0000000245_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000245#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000246](https://www.ebi.ac.uk/biomodels/BIOMD0000000246)<br/><sup>Vasalou2010_Pacemaker_Neuron_SCN</sup>|<details><summary>pass</summary>[BIOMD0000000246_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000246#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000247](https://www.ebi.ac.uk/biomodels/BIOMD0000000247)<br/><sup>Ralser2007_Carbohydrate_Rerout</sup></summary><sup>Ralser2007_Carbohydrate_Rerouting_ROS</sup></details>|<details><summary>pass</summary>[BIOMD0000000247_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000247#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000248](https://www.ebi.ac.uk/biomodels/BIOMD0000000248)<br/><sup>Lai2007_O2_Transport_Metabolism</sup>|<details><summary>pass</summary>[BIOMD0000000248_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000248#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000249](https://www.ebi.ac.uk/biomodels/BIOMD0000000249)<br/><sup>Restif2006 - Whooping cough</sup>|<details><summary>pass</summary>[BIOMD0000000249_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000249#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000250](https://www.ebi.ac.uk/biomodels/BIOMD0000000250)<br/><sup>Nakakuki2010_CellFateDecision_</sup></summary><sup>Nakakuki2010_CellFateDecision_Mechanistic</sup></details>|<details><summary>pass</summary>[BIOMD0000000250_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000250#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000251](https://www.ebi.ac.uk/biomodels/BIOMD0000000251)<br/><sup>Nakakuki2010_CellFateDecision_Core</sup>|<details><summary>pass</summary>[BIOMD0000000251_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000251#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000252](https://www.ebi.ac.uk/biomodels/BIOMD0000000252)<br/><sup>Hunziker2010_p53_StressSpecifi</sup></summary><sup>Hunziker2010_p53_StressSpecificResponse</sup></details>|<details><summary>pass</summary>[BIOMD0000000252_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000252#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000253](https://www.ebi.ac.uk/biomodels/BIOMD0000000253)<br/><sup>Teusink1998_Glycolysis_TurboDesign</sup>|<details><summary>pass</summary>[BIOMD0000000253_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000253#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000254](https://www.ebi.ac.uk/biomodels/BIOMD0000000254)<br/><sup>Bier2000_GlycolyticOscillation</sup>|<details><summary>pass</summary>[BIOMD0000000254_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000254#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000255](https://www.ebi.ac.uk/biomodels/BIOMD0000000255)<br/><sup>Chen2009 - ErbB Signaling</sup>|<details><summary>pass</summary>[BIOMD0000000255_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000255#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000256](https://www.ebi.ac.uk/biomodels/BIOMD0000000256)<br/><sup>Rehm2006_Caspase</sup>|<details><summary>pass</summary>[BIOMD0000000256_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000256#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000257](https://www.ebi.ac.uk/biomodels/BIOMD0000000257)<br/><sup>Piedrafita2010_MR_System</sup>|<details><summary>pass</summary>[BIOMD0000000257_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000257#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000258](https://www.ebi.ac.uk/biomodels/BIOMD0000000258)<br/><sup>Ortega2006 - bistability from </sup></summary><sup>Ortega2006 - bistability from double phosphorylation in signal transduction</sup></details>|<details><summary>pass</summary>[BIOMD0000000258_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000258#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000259](https://www.ebi.ac.uk/biomodels/BIOMD0000000259)<br/><sup>Tiago2010_FeMetabolism_FeDeficient</sup>|<details><summary>pass</summary>[BIOMD0000000259_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000259#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000260](https://www.ebi.ac.uk/biomodels/BIOMD0000000260)<br/><sup>Tiago2010_FeMetabolism_FeAdequate</sup>|<details><summary>pass</summary>[BIOMD0000000260_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000260#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000261](https://www.ebi.ac.uk/biomodels/BIOMD0000000261)<br/><sup>Tiago2010_FeMetabolism_FeLoaded</sup>|<details><summary>pass</summary>[BIOMD0000000261_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000261#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000262](https://www.ebi.ac.uk/biomodels/BIOMD0000000262)<br/><sup>Fujita2010_Akt_Signalling_EGF</sup>|<details><summary>pass</summary>[BIOMD0000000262_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000262#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000263](https://www.ebi.ac.uk/biomodels/BIOMD0000000263)<br/><sup>Fujita2010_Akt_Signalling_NGF</sup>|<details><summary>pass</summary>[BIOMD0000000263_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000263#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000264](https://www.ebi.ac.uk/biomodels/BIOMD0000000264)<br/><sup>Fujita2010_Akt_Signalling_EGFRinhib</sup>|<details><summary>pass</summary>[BIOMD0000000264_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000264#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000265](https://www.ebi.ac.uk/biomodels/BIOMD0000000265)<br/><sup>Conradie2010_RPControl_CellCycle</sup>|<details><summary>pass</summary>[BIOMD0000000265_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000265#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000266](https://www.ebi.ac.uk/biomodels/BIOMD0000000266)<br/><sup>Voit2003 - Trehalose Cycle</sup>|<details><summary>pass</summary>[BIOMD0000000266_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000266#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000267](https://www.ebi.ac.uk/biomodels/BIOMD0000000267)<br/><sup>Lebeda2008 - BoNT paralysis (3</sup></summary><sup>Lebeda2008 - BoNT paralysis (3 step model)</sup></details>|<details><summary>pass</summary>[BIOMD0000000267_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000267#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000268](https://www.ebi.ac.uk/biomodels/BIOMD0000000268)<br/><sup>Reed2008_Glutathione_Metabolism</sup>|<details><summary>pass</summary>[BIOMD0000000268_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000268#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000269](https://www.ebi.ac.uk/biomodels/BIOMD0000000269)<br/><sup>Liu2010_Hormonal_Crosstalk_Ara</sup></summary><sup>Liu2010_Hormonal_Crosstalk_Arabidopsis</sup></details>|<details><summary>pass</summary>[BIOMD0000000269_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000269#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000270](https://www.ebi.ac.uk/biomodels/BIOMD0000000270)<br/><sup>Schilling2009 - ERK distributive</sup>|<details><summary>pass</summary>[BIOMD0000000270_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000270#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000271](https://www.ebi.ac.uk/biomodels/BIOMD0000000271)<br/><sup>Becker2010_EpoR_CoreModel</sup>|<details><summary>pass</summary>[BIOMD0000000271_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000271#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000272](https://www.ebi.ac.uk/biomodels/BIOMD0000000272)<br/><sup>Becker2010_EpoR_AuxiliaryModel</sup>|<details><summary>pass</summary>[BIOMD0000000272_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000272#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000273](https://www.ebi.ac.uk/biomodels/BIOMD0000000273)<br/><sup>Pokhilko2010_CircClock</sup>|<details><summary>pass</summary>[BIOMD0000000273_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000273#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000274](https://www.ebi.ac.uk/biomodels/BIOMD0000000274)<br/><sup>Rattanakul2003_BoneFormationModel</sup>|<details><summary>pass</summary>[BIOMD0000000274_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000274#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000275](https://www.ebi.ac.uk/biomodels/BIOMD0000000275)<br/><sup>Goldbeter2007_Somitogenesis_Switch</sup>|<details><summary>pass</summary>[BIOMD0000000275_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000275#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000276](https://www.ebi.ac.uk/biomodels/BIOMD0000000276)<br/><sup>Shrestha2010_HypoCalcemia_PTHr</sup></summary><sup>Shrestha2010_HypoCalcemia_PTHresponse</sup></details>|<details><summary>pass</summary>[BIOMD0000000276_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000276#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000277](https://www.ebi.ac.uk/biomodels/BIOMD0000000277)<br/><sup>Shrestha2010_HyperCalcemia_PTH</sup></summary><sup>Shrestha2010_HyperCalcemia_PTHresponse</sup></details>|<details><summary>pass</summary>[BIOMD0000000277_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000277#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000278](https://www.ebi.ac.uk/biomodels/BIOMD0000000278)<br/><sup>Lemaire2004 - Role of RANK/RAN</sup></summary><sup>Lemaire2004 - Role of RANK/RANKL/OPG pathway in bone remodelling process</sup></details>|<details><summary>pass</summary>[BIOMD0000000278_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000278#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000279](https://www.ebi.ac.uk/biomodels/BIOMD0000000279)<br/><sup>Komarova2005_PTHaction_Osteocl</sup></summary><sup>Komarova2005_PTHaction_OsteoclastOsteoblastCoupling</sup></details>|<details><summary>pass</summary>[BIOMD0000000279_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000279#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000280](https://www.ebi.ac.uk/biomodels/BIOMD0000000280)<br/><sup>Morris1981_MuscleFibre_Voltage</sup></summary><sup>Morris1981_MuscleFibre_Voltage_reduced</sup></details>|<details><summary>pass</summary>[BIOMD0000000280_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000280#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000281](https://www.ebi.ac.uk/biomodels/BIOMD0000000281)<br/><sup>Chance1960_Glycolysis_Respiration</sup>|<details><summary>pass</summary>[BIOMD0000000281_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000281#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000282](https://www.ebi.ac.uk/biomodels/BIOMD0000000282)<br/><sup>Chance1952_Catalase_Mechanism</sup>|<details><summary>pass</summary>[BIOMD0000000282_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000282#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000283](https://www.ebi.ac.uk/biomodels/BIOMD0000000283)<br/><sup>Chance1943_Peroxidase_ES_Kinetics</sup>|<details><summary>pass</summary>[BIOMD0000000283_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000283#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000284](https://www.ebi.ac.uk/biomodels/BIOMD0000000284)<br/><sup>Hofmeyer1986_SeqFb_Proc_AA_Synthesis</sup>|<details><summary>pass</summary>[BIOMD0000000284_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000284#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000285](https://www.ebi.ac.uk/biomodels/BIOMD0000000285)<br/><sup>Tang2010_PolyGlutamate</sup>|<details><summary>pass</summary>[BIOMD0000000285_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000285#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000286](https://www.ebi.ac.uk/biomodels/BIOMD0000000286)<br/><sup>Proctor2010 - a link between G</sup></summary><sup>Proctor2010 - a link between GSK3 and p53 in Alzheimer's Disease</sup></details>|<details><summary>pass</summary>[BIOMD0000000286_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000286#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000287](https://www.ebi.ac.uk/biomodels/BIOMD0000000287)<br/><sup>Passos2010_DNAdamage_CellularS</sup></summary><sup>Passos2010_DNAdamage_CellularSenescence</sup></details>|<details><summary>pass</summary>[BIOMD0000000287_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000287#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000288](https://www.ebi.ac.uk/biomodels/BIOMD0000000288)<br/><sup>Wang2009 - PI3K Ras Crosstalk</sup>|<details><summary>pass</summary>[BIOMD0000000288_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000288#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000289](https://www.ebi.ac.uk/biomodels/BIOMD0000000289)<br/><sup>Alexander2010_Tcell_Regulation_Sys1</sup>|<details><summary>pass</summary>[BIOMD0000000289_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000289#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000290](https://www.ebi.ac.uk/biomodels/BIOMD0000000290)<br/><sup>Alexander2010_Tcell_Regulation_Sys2</sup>|<details><summary>pass</summary>[BIOMD0000000290_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000290#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000291](https://www.ebi.ac.uk/biomodels/BIOMD0000000291)<br/><sup>Nikolaev2005_AlbuminBilirubinA</sup></summary><sup>Nikolaev2005_AlbuminBilirubinAdsorption</sup></details>|<details><summary>pass</summary>[BIOMD0000000291_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000291#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000292](https://www.ebi.ac.uk/biomodels/BIOMD0000000292)<br/><sup>Rovers1995_Photsynthetic_Oscil</sup></summary><sup>Rovers1995_Photsynthetic_Oscillations</sup></details>|<details><summary>pass</summary>[BIOMD0000000292_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000292#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000293](https://www.ebi.ac.uk/biomodels/BIOMD0000000293)<br/><sup>Proctor2010 - UCHL1 Protein Ag</sup></summary><sup>Proctor2010 - UCHL1 Protein Aggregation</sup></details>|<details><summary>pass</summary>[BIOMD0000000293_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000293#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000294](https://www.ebi.ac.uk/biomodels/BIOMD0000000294)<br/><sup>Restif2007 - Vaccination invasion</sup>|<details><summary>pass</summary>[BIOMD0000000294_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000294#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000295](https://www.ebi.ac.uk/biomodels/BIOMD0000000295)<br/><sup>Akman2008_Circadian_Clock_Model1</sup>|<details><summary>pass</summary>[BIOMD0000000295_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000295#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000296](https://www.ebi.ac.uk/biomodels/BIOMD0000000296)<br/><sup>Balagadd2008_E_coli_Predator_Prey</sup>|<details><summary>pass</summary>[BIOMD0000000296_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000296#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000297](https://www.ebi.ac.uk/biomodels/BIOMD0000000297)<br/><sup>Ciliberto2003_Morphogenesis_Ch</sup></summary><sup>Ciliberto2003_Morphogenesis_Checkpoint</sup></details>|<details><summary>pass</summary>[BIOMD0000000297_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000297#Files)</details>|pass|<details><summary>pass</summary>[f3.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000297#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/f3.sedml332751.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/f3.sedml332751.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./f3.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - YDataReference from Curve `p1_curve_3_task1` cannot be resolved to a data generator with id `Clb2_3_task1` because multiple data generators have this id.\n - YDataReference from Curve `p2_curve_3_task1` cannot be resolved to a data generator with id `Clb2_3_task1` because multiple data generators have this id.\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/f3.sedml420581.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/f3.sedml420581.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./f3.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - YDataReference from Curve `p1_curve_3_task1` cannot be resolved to a data generator with id `Clb2_3_task1` because multiple data generators have this id.\n - YDataReference from Curve `p2_curve_3_task1` cannot be resolved to a data generator with id `Clb2_3_task1` because multiple data generators have this id.\n'```</details>|
|<details><summary>[BIOMD0000000298](https://www.ebi.ac.uk/biomodels/BIOMD0000000298)<br/><sup>Leloup1999_CircadianRhythms_Dr</sup></summary><sup>Leloup1999_CircadianRhythms_Drosophila</sup></details>|<details><summary>pass</summary>[BIOMD0000000298_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000298#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000299](https://www.ebi.ac.uk/biomodels/BIOMD0000000299)<br/><sup>Leloup1999_CircadianRhythms_Ne</sup></summary><sup>Leloup1999_CircadianRhythms_Neurospora</sup></details>|<details><summary>pass</summary>[BIOMD0000000299_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000299#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000300](https://www.ebi.ac.uk/biomodels/BIOMD0000000300)<br/><sup>Schmierer2010_FIH_Ankyrins</sup>|<details><summary>pass</summary>[BIOMD0000000300_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000300#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000301](https://www.ebi.ac.uk/biomodels/BIOMD0000000301)<br/><sup>Friedland2009_Ara_RTC3_counter</sup>|<details><summary>pass</summary>[BIOMD0000000301_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000301#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000302](https://www.ebi.ac.uk/biomodels/BIOMD0000000302)<br/><sup>Wang1996_Synaptic_Inhibition_T</sup></summary><sup>Wang1996_Synaptic_Inhibition_Two_Neuron</sup></details>|<details><summary>pass</summary>[BIOMD0000000302_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000302#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000303](https://www.ebi.ac.uk/biomodels/BIOMD0000000303)<br/><sup>Liu2011_Complement_System</sup>|<details><summary>pass</summary>[BIOMD0000000303_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000303#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000304](https://www.ebi.ac.uk/biomodels/BIOMD0000000304)<br/><sup>Plant1981_BurstingNerveCells</sup>|<details><summary>pass</summary>[BIOMD0000000304_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000304#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000305](https://www.ebi.ac.uk/biomodels/BIOMD0000000305)<br/><sup>Kolomeisky2003_MyosinV_Processivity</sup>|<details><summary>pass</summary>[BIOMD0000000305_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000305#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000306](https://www.ebi.ac.uk/biomodels/BIOMD0000000306)<br/><sup>Tyson2003_Activator_Inhibitor</sup>|<details><summary>pass</summary>[BIOMD0000000306_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000306#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000307](https://www.ebi.ac.uk/biomodels/BIOMD0000000307)<br/><sup>Tyson2003_Substrate_Depletion_Osc</sup>|<details><summary>pass</summary>[BIOMD0000000307_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000307#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000308](https://www.ebi.ac.uk/biomodels/BIOMD0000000308)<br/><sup>Tyson2003_NegFB_Oscillator</sup>|<details><summary>pass</summary>[BIOMD0000000308_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000308#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000309](https://www.ebi.ac.uk/biomodels/BIOMD0000000309)<br/><sup>Tyson2003_NegFB_Homeostasis</sup>|<details><summary>pass</summary>[BIOMD0000000309_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000309#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000310](https://www.ebi.ac.uk/biomodels/BIOMD0000000310)<br/><sup>Tyson2003_Mutual_Inhibition</sup>|<details><summary>pass</summary>[BIOMD0000000310_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000310#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000311](https://www.ebi.ac.uk/biomodels/BIOMD0000000311)<br/><sup>Tyson2003_Mutual_Activation</sup>|<details><summary>pass</summary>[BIOMD0000000311_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000311#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000312](https://www.ebi.ac.uk/biomodels/BIOMD0000000312)<br/><sup>Tyson2003_Perfect_Adaption</sup>|<details><summary>pass</summary>[BIOMD0000000312_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000312#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000313](https://www.ebi.ac.uk/biomodels/BIOMD0000000313)<br/><sup>Raia2010 - IL13 Signalling MedB1</sup>|<details><summary>pass</summary>[BIOMD0000000313_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000313#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000314](https://www.ebi.ac.uk/biomodels/BIOMD0000000314)<br/><sup>Raia2011 - IL13 L1236</sup>|<details><summary>pass</summary>[BIOMD0000000314_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000314#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000315](https://www.ebi.ac.uk/biomodels/BIOMD0000000315)<br/><sup>Montagne2011_Oligator_optimised</sup>|<details><summary>pass</summary>[BIOMD0000000315_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000315#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000316](https://www.ebi.ac.uk/biomodels/BIOMD0000000316)<br/><sup>Shen-Orr2002_FeedForward_AND_gate</sup>|<details><summary>pass</summary>[BIOMD0000000316_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000316#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000317](https://www.ebi.ac.uk/biomodels/BIOMD0000000317)<br/><sup>Shen-Orr2002_Single_Input_Module</sup>|<details><summary>pass</summary>[BIOMD0000000317_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000317#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000318](https://www.ebi.ac.uk/biomodels/BIOMD0000000318)<br/><sup>Yao2008_Rb_E2F_Switch</sup>|<details><summary>pass</summary>[BIOMD0000000318_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000318#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000319](https://www.ebi.ac.uk/biomodels/BIOMD0000000319)<br/><sup>Decroly1982_Enzymatic_Oscillator</sup>|<details><summary>pass</summary>[BIOMD0000000319_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000319#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000320](https://www.ebi.ac.uk/biomodels/BIOMD0000000320)<br/><sup>Grange2001 - PK interaction of</sup></summary><sup>Grange2001 - PK interaction of L-dopa and benserazide</sup></details>|<details><summary>pass</summary>[BIOMD0000000320_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000320#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000321](https://www.ebi.ac.uk/biomodels/BIOMD0000000321)<br/><sup>Grange2001 - L Dopa PK model</sup>|<details><summary>pass</summary>[BIOMD0000000321_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000321#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000322](https://www.ebi.ac.uk/biomodels/BIOMD0000000322)<br/><sup>Kim2011_Oscillator_SimpleI</sup>|<details><summary>pass</summary>[BIOMD0000000322_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000322#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000323](https://www.ebi.ac.uk/biomodels/BIOMD0000000323)<br/><sup>Kim2011_Oscillator_SimpleIII</sup>|<details><summary>pass</summary>[BIOMD0000000323_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000323#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000324](https://www.ebi.ac.uk/biomodels/BIOMD0000000324)<br/><sup>Morris1981_MuscleFibre_Voltage_full</sup>|<details><summary>pass</summary>[BIOMD0000000324_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000324#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000325](https://www.ebi.ac.uk/biomodels/BIOMD0000000325)<br/><sup>Palini2011_Minimal_2_Feedback_Model</sup>|<details><summary>pass</summary>[BIOMD0000000325_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000325#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000326](https://www.ebi.ac.uk/biomodels/BIOMD0000000326)<br/><sup>DellOrco2009_phototransduction</sup>|<details><summary>pass</summary>[BIOMD0000000326_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000326#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000327](https://www.ebi.ac.uk/biomodels/BIOMD0000000327)<br/><sup>Whitcomb2004_Bicarbonate_Pancreas</sup>|<details><summary>pass</summary>[BIOMD0000000327_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000327#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000328](https://www.ebi.ac.uk/biomodels/BIOMD0000000328)<br/><sup>Bucher2011_Atorvastatin_Metabolism</sup>|<details><summary>pass</summary>[BIOMD0000000328_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000328#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000329](https://www.ebi.ac.uk/biomodels/BIOMD0000000329)<br/><sup>Kummer2000 - Oscillations in C</sup></summary><sup>Kummer2000 - Oscillations in Calcium Signalling</sup></details>|<details><summary>pass</summary>[BIOMD0000000329_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000329#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000330](https://www.ebi.ac.uk/biomodels/BIOMD0000000330)<br/><sup>Larsen2004_CalciumSpiking</sup>|<details><summary>pass</summary>[BIOMD0000000330_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000330#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000331](https://www.ebi.ac.uk/biomodels/BIOMD0000000331)<br/><sup>Larsen2004_CalciumSpiking_Enzy</sup></summary><sup>Larsen2004_CalciumSpiking_EnzymeBinding</sup></details>|<details><summary>pass</summary>[BIOMD0000000331_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000331#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000332](https://www.ebi.ac.uk/biomodels/BIOMD0000000332)<br/><sup>Bungay2006_Plasma</sup>|<details><summary>pass</summary>[BIOMD0000000332_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000332#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000333](https://www.ebi.ac.uk/biomodels/BIOMD0000000333)<br/><sup>Bungay2006_FollicularFluid</sup>|<details><summary>pass</summary>[BIOMD0000000333_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000333#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000334](https://www.ebi.ac.uk/biomodels/BIOMD0000000334)<br/><sup>Bungay2003_Thrombin_Generation</sup>|<details><summary>pass</summary>[BIOMD0000000334_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000334#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000335](https://www.ebi.ac.uk/biomodels/BIOMD0000000335)<br/><sup>Hockin2002_BloodCoagulation</sup>|<details><summary>pass</summary>[BIOMD0000000335_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000335#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000336](https://www.ebi.ac.uk/biomodels/BIOMD0000000336)<br/><sup>Jones1994_BloodCoagulation</sup>|<details><summary>pass</summary>[BIOMD0000000336_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000336#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000337](https://www.ebi.ac.uk/biomodels/BIOMD0000000337)<br/><sup>Pfeiffer2001_ATP-ProducingPath</sup></summary><sup>Pfeiffer2001_ATP-ProducingPathways_CooperationCompetition</sup></details>|<details><summary>pass</summary>[BIOMD0000000337_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000337#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000338](https://www.ebi.ac.uk/biomodels/BIOMD0000000338)<br/><sup>Wajima2009_BloodCoagulation_aPTTtest</sup>|<details><summary>pass</summary>[BIOMD0000000338_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000338#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000339](https://www.ebi.ac.uk/biomodels/BIOMD0000000339)<br/><sup>Wajima2009_BloodCoagulation_PTtest</sup>|<details><summary>pass</summary>[BIOMD0000000339_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000339#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000340](https://www.ebi.ac.uk/biomodels/BIOMD0000000340)<br/><sup>Wajima2009_BloodCoagulation_wa</sup></summary><sup>Wajima2009_BloodCoagulation_warfarin_heparin</sup></details>|<details><summary>pass</summary>[BIOMD0000000340_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000340#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000341](https://www.ebi.ac.uk/biomodels/BIOMD0000000341)<br/><sup>Topp2000_BetaCellMass_Diabetes</sup>|<details><summary>pass</summary>[BIOMD0000000341_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000341#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000342](https://www.ebi.ac.uk/biomodels/BIOMD0000000342)<br/><sup>Zi2011_TGF-beta_Pathway</sup>|<details><summary>pass</summary>[BIOMD0000000342_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000342#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000343](https://www.ebi.ac.uk/biomodels/BIOMD0000000343)<br/><sup>Brannmark2010_InsulinSignallin</sup></summary><sup>Brannmark2010_InsulinSignalling_Mifamodel</sup></details>|<details><summary>pass</summary>[BIOMD0000000343_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000343#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000344](https://www.ebi.ac.uk/biomodels/BIOMD0000000344)<br/><sup>Proctor2011_ProteinHomeostasis</sup></summary><sup>Proctor2011_ProteinHomeostasis_NormalCondition</sup></details>|<details><summary>pass</summary>[BIOMD0000000344_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000344#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000345](https://www.ebi.ac.uk/biomodels/BIOMD0000000345)<br/><sup>Koschorreck2008_InsulinClearance</sup>|<details><summary>pass</summary>[BIOMD0000000345_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000345#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000346](https://www.ebi.ac.uk/biomodels/BIOMD0000000346)<br/><sup>FitzHugh1961_NerveMembrane</sup>|<details><summary>pass</summary>[BIOMD0000000346_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000346#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000347](https://www.ebi.ac.uk/biomodels/BIOMD0000000347)<br/><sup>Bachmann2011_JAK2-STAT5_Feedba</sup></summary><sup>Bachmann2011_JAK2-STAT5_FeedbackControl</sup></details>|<details><summary>pass</summary>[BIOMD0000000347_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000347#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000348](https://www.ebi.ac.uk/biomodels/BIOMD0000000348)<br/><sup>Fridlyand2010_GlucoseSensitivity_A</sup>|<details><summary>pass</summary>[BIOMD0000000348_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000348#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000349](https://www.ebi.ac.uk/biomodels/BIOMD0000000349)<br/><sup>Fridlyand2010_GlucoseSensitivity_B</sup>|<details><summary>pass</summary>[BIOMD0000000349_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000349#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000350](https://www.ebi.ac.uk/biomodels/BIOMD0000000350)<br/><sup>Troein2011_ClockCircuit_Ostreo</sup></summary><sup>Troein2011_ClockCircuit_OstreococcusTauri</sup></details>|<details><summary>pass</summary>[BIOMD0000000350_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000350#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000351](https://www.ebi.ac.uk/biomodels/BIOMD0000000351)<br/><sup>Vernoux2011_AuxinSignaling_Aux</sup></summary><sup>Vernoux2011_AuxinSignaling_AuxinSingleStepInput</sup></details>|<details><summary>pass</summary>[BIOMD0000000351_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000351#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000352](https://www.ebi.ac.uk/biomodels/BIOMD0000000352)<br/><sup>Vernoux2011_AuxinSignaling_Aux</sup></summary><sup>Vernoux2011_AuxinSignaling_AuxinFluctuating</sup></details>|<details><summary>pass</summary>[BIOMD0000000352_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000352#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000353](https://www.ebi.ac.uk/biomodels/BIOMD0000000353)<br/><sup>Nag2011_ChloroplasticStarchDeg</sup></summary><sup>Nag2011_ChloroplasticStarchDegradation</sup></details>|<details><summary>pass</summary>[BIOMD0000000353_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000353#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000354](https://www.ebi.ac.uk/biomodels/BIOMD0000000354)<br/><sup>Abell2011_CalciumSignaling_Wit</sup></summary><sup>Abell2011_CalciumSignaling_WithoutAdaptation</sup></details>|<details><summary>pass</summary>[BIOMD0000000354_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000354#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000355](https://www.ebi.ac.uk/biomodels/BIOMD0000000355)<br/><sup>Abell2011_CalciumSignaling_Wit</sup></summary><sup>Abell2011_CalciumSignaling_WithAdaptation</sup></details>|<details><summary>pass</summary>[BIOMD0000000355_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000355#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000356](https://www.ebi.ac.uk/biomodels/BIOMD0000000356)<br/><sup>Nyman2011_M3Hierarachical_Insu</sup></summary><sup>Nyman2011_M3Hierarachical_InsulinGlucosedynamics</sup></details>|<details><summary>pass</summary>[BIOMD0000000356_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000356#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000357](https://www.ebi.ac.uk/biomodels/BIOMD0000000357)<br/><sup>Lee2010_ThrombinActivation_One</sup></summary><sup>Lee2010_ThrombinActivation_OneForm_reduced</sup></details>|<details><summary>pass</summary>[BIOMD0000000357_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000357#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000358](https://www.ebi.ac.uk/biomodels/BIOMD0000000358)<br/><sup>Stortelder1997 - Thrombin Gene</sup></summary><sup>Stortelder1997 - Thrombin Generation Amidolytic Activity</sup></details>|<details><summary>pass</summary>[BIOMD0000000358_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000358#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000359](https://www.ebi.ac.uk/biomodels/BIOMD0000000359)<br/><sup>Panteleev2002_TFPImechanism_schmema3</sup>|<details><summary>pass</summary>[BIOMD0000000359_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000359#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000360](https://www.ebi.ac.uk/biomodels/BIOMD0000000360)<br/><sup>Panteleev2002_TFPImechanism_schmema2</sup>|<details><summary>pass</summary>[BIOMD0000000360_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000360#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000361](https://www.ebi.ac.uk/biomodels/BIOMD0000000361)<br/><sup>Panteleev2002_TFPImechanism_schmema1</sup>|<details><summary>pass</summary>[BIOMD0000000361_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000361#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000362](https://www.ebi.ac.uk/biomodels/BIOMD0000000362)<br/><sup>Butenas2004_BloodCoagulation</sup>|<details><summary>pass</summary>[BIOMD0000000362_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000362#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000363](https://www.ebi.ac.uk/biomodels/BIOMD0000000363)<br/><sup>Lee2010_ThrombinActivation_One</sup></summary><sup>Lee2010_ThrombinActivation_OneForm_minimal</sup></details>|<details><summary>pass</summary>[BIOMD0000000363_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000363#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000364](https://www.ebi.ac.uk/biomodels/BIOMD0000000364)<br/><sup>Lee2010_ThrombinActivation_OneForm</sup>|<details><summary>pass</summary>[BIOMD0000000364_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000364#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000365](https://www.ebi.ac.uk/biomodels/BIOMD0000000365)<br/><sup>Hockin1999_BloodCoagulation_Va</sup></summary><sup>Hockin1999_BloodCoagulation_VaInactivation</sup></details>|<details><summary>pass</summary>[BIOMD0000000365_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000365#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000366](https://www.ebi.ac.uk/biomodels/BIOMD0000000366)<br/><sup>Orfao2008_ThrombinGeneration_A</sup></summary><sup>Orfao2008_ThrombinGeneration_AmidolyticActivity</sup></details>|<details><summary>pass</summary>[BIOMD0000000366_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000366#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000367](https://www.ebi.ac.uk/biomodels/BIOMD0000000367)<br/><sup>Mueller2008_ThrombinGeneration</sup></summary><sup>Mueller2008_ThrombinGeneration_minimal</sup></details>|<details><summary>pass</summary>[BIOMD0000000367_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000367#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000368](https://www.ebi.ac.uk/biomodels/BIOMD0000000368)<br/><sup>Beltrami1995_ThrombinGeneration_C</sup>|<details><summary>pass</summary>[BIOMD0000000368_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000368#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000369](https://www.ebi.ac.uk/biomodels/BIOMD0000000369)<br/><sup>Beltrami1995_ThrombinGeneration_D</sup>|<details><summary>pass</summary>[BIOMD0000000369_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000369#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000370](https://www.ebi.ac.uk/biomodels/BIOMD0000000370)<br/><sup>Vinod2011_MitoticExit</sup>|<details><summary>pass</summary>[BIOMD0000000370_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000370#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000371](https://www.ebi.ac.uk/biomodels/BIOMD0000000371)<br/><sup>DeVries2000_PancreaticBetaCell</sup></summary><sup>DeVries2000_PancreaticBetaCells_InsulinSecretion</sup></details>|<details><summary>pass</summary>[BIOMD0000000371_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000371#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000372](https://www.ebi.ac.uk/biomodels/BIOMD0000000372)<br/><sup>Tolic2000_InsulinGlucoseFeedback</sup>|<details><summary>pass</summary>[BIOMD0000000372_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000372#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000373](https://www.ebi.ac.uk/biomodels/BIOMD0000000373)<br/><sup>Bertram2004_PancreaticBetaCell</sup></summary><sup>Bertram2004_PancreaticBetaCell_modelB</sup></details>|<details><summary>pass</summary>[BIOMD0000000373_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000373#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000374](https://www.ebi.ac.uk/biomodels/BIOMD0000000374)<br/><sup>Bertram1995_PancreaticBetaCell_CRAC</sup>|<details><summary>pass</summary>[BIOMD0000000374_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000374#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000375](https://www.ebi.ac.uk/biomodels/BIOMD0000000375)<br/><sup>Mears1997_CRAC_PancreaticBetaCells</sup>|<details><summary>pass</summary>[BIOMD0000000375_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000375#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000376](https://www.ebi.ac.uk/biomodels/BIOMD0000000376)<br/><sup>Bertram2007_IsletCell_Oscillations</sup>|<details><summary>pass</summary>[BIOMD0000000376_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000376#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000377](https://www.ebi.ac.uk/biomodels/BIOMD0000000377)<br/><sup>Bertram2000_PancreaticBetaCell</sup></summary><sup>Bertram2000_PancreaticBetaCells_Oscillations</sup></details>|<details><summary>pass</summary>[BIOMD0000000377_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000377#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000378](https://www.ebi.ac.uk/biomodels/BIOMD0000000378)<br/><sup>Chay1997_CalciumConcentration</sup>|<details><summary>pass</summary>[BIOMD0000000378_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000378#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000379](https://www.ebi.ac.uk/biomodels/BIOMD0000000379)<br/><sup>DallaMan2007_MealModel_Glucose</sup></summary><sup>DallaMan2007_MealModel_GlucoseInsulinSystem</sup></details>|<details><summary>pass</summary>[BIOMD0000000379_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000379#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000380](https://www.ebi.ac.uk/biomodels/BIOMD0000000380)<br/><sup>Smallbone2011_TrehaloseBiosynthesis</sup>|<details><summary>pass</summary>[BIOMD0000000380_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000380#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000381](https://www.ebi.ac.uk/biomodels/BIOMD0000000381)<br/><sup>Maree2006_DuCa_Type1DiabetesModel</sup>|<details><summary>pass</summary>[BIOMD0000000381_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000381#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000382](https://www.ebi.ac.uk/biomodels/BIOMD0000000382)<br/><sup>Sturis1991_InsulinGlucoseModel</sup></summary><sup>Sturis1991_InsulinGlucoseModel_UltradianOscillation</sup></details>|<details><summary>pass</summary>[BIOMD0000000382_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000382#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000383](https://www.ebi.ac.uk/biomodels/BIOMD0000000383)<br/><sup>Arnold2011_Farquhar1980_RuBisC</sup></summary><sup>Arnold2011_Farquhar1980_RuBisCO-CalvinCycle</sup></details>|<details><summary>pass</summary>[BIOMD0000000383_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000383#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000384](https://www.ebi.ac.uk/biomodels/BIOMD0000000384)<br/><sup>Arnold2011_Medlyn2002_RuBisCO-</sup></summary><sup>Arnold2011_Medlyn2002_RuBisCO-CalvinCycle</sup></details>|<details><summary>pass</summary>[BIOMD0000000384_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000384#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000385](https://www.ebi.ac.uk/biomodels/BIOMD0000000385)<br/><sup>Arnold2011_Schultz2003_RuBisCO</sup></summary><sup>Arnold2011_Schultz2003_RuBisCO-CalvinCycle</sup></details>|<details><summary>pass</summary>[BIOMD0000000385_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000385#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000386](https://www.ebi.ac.uk/biomodels/BIOMD0000000386)<br/><sup>Arnold2011_Sharkey2007_RuBisCO</sup></summary><sup>Arnold2011_Sharkey2007_RuBisCO-CalvinCycle</sup></details>|<details><summary>pass</summary>[BIOMD0000000386_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000386#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000387](https://www.ebi.ac.uk/biomodels/BIOMD0000000387)<br/><sup>Arnold2011_Damour2007_RuBisCO-</sup></summary><sup>Arnold2011_Damour2007_RuBisCO-CalvinCycle</sup></details>|<details><summary>pass</summary>[BIOMD0000000387_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000387#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000388](https://www.ebi.ac.uk/biomodels/BIOMD0000000388)<br/><sup>Arnold2011_Zhu2009_CalvinCycle</sup>|<details><summary>pass</summary>[BIOMD0000000388_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000388#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000389](https://www.ebi.ac.uk/biomodels/BIOMD0000000389)<br/><sup>Arnold2011_Hahn1986_CalvinCycl</sup></summary><sup>Arnold2011_Hahn1986_CalvinCycle_Starch_Sucrose</sup></details>|<details><summary>pass</summary>[BIOMD0000000389_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000389#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000390](https://www.ebi.ac.uk/biomodels/BIOMD0000000390)<br/><sup>Arnold2011_Giersch1990_CalvinCycle</sup>|<details><summary>pass</summary>[BIOMD0000000390_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000390#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000391](https://www.ebi.ac.uk/biomodels/BIOMD0000000391)<br/><sup>Arnold2011_Poolman2000_CalvinC</sup></summary><sup>Arnold2011_Poolman2000_CalvinCycle_Starch</sup></details>|<details><summary>pass</summary>[BIOMD0000000391_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000391#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000392](https://www.ebi.ac.uk/biomodels/BIOMD0000000392)<br/><sup>Arnold2011_Laisk2006_CalvinCyc</sup></summary><sup>Arnold2011_Laisk2006_CalvinCycle_Starch_Sucrose</sup></details>|<details><summary>pass</summary>[BIOMD0000000392_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000392#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000393](https://www.ebi.ac.uk/biomodels/BIOMD0000000393)<br/><sup>Arnold2011_Zhu2007_CalvinCycle</sup></summary><sup>Arnold2011_Zhu2007_CalvinCycle_Starch_Sucrose_Photorespiration</sup></details>|<details><summary>pass</summary>[BIOMD0000000393_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000393#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000394](https://www.ebi.ac.uk/biomodels/BIOMD0000000394)<br/><sup>Sivakumar2011 - EGF Receptor S</sup></summary><sup>Sivakumar2011 - EGF Receptor Signaling Pathway</sup></details>|<details><summary>pass</summary>[BIOMD0000000394_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000394#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000395](https://www.ebi.ac.uk/biomodels/BIOMD0000000395)<br/><sup>Sivakumar2011 - Hedgehog Signa</sup></summary><sup>Sivakumar2011 - Hedgehog Signaling Pathway</sup></details>|<details><summary>pass</summary>[BIOMD0000000395_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000395#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000396](https://www.ebi.ac.uk/biomodels/BIOMD0000000396)<br/><sup>Sivakumar2011 - Notch Signalin</sup></summary><sup>Sivakumar2011 - Notch Signaling Pathway</sup></details>|<details><summary>pass</summary>[BIOMD0000000396_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000396#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000397](https://www.ebi.ac.uk/biomodels/BIOMD0000000397)<br/><sup>Sivakumar2011_WntSignalingPathway</sup>|<details><summary>pass</summary>[BIOMD0000000397_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000397#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000398](https://www.ebi.ac.uk/biomodels/BIOMD0000000398)<br/><sup>Sivakumar2011_NeuralStemCellDi</sup></summary><sup>Sivakumar2011_NeuralStemCellDifferentiation_Crosstalk</sup></details>|<details><summary>pass</summary>[BIOMD0000000398_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000398#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000399](https://www.ebi.ac.uk/biomodels/BIOMD0000000399)<br/><sup>Jenkinson2011_EGF_MAPK</sup>|<details><summary>pass</summary>[BIOMD0000000399_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000399#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000400](https://www.ebi.ac.uk/biomodels/BIOMD0000000400)<br/><sup>Cooling2007_IP3transients_Card</sup></summary><sup>Cooling2007_IP3transients_CardiacMyocyte</sup></details>|<details><summary>pass</summary>[BIOMD0000000400_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000400#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000401](https://www.ebi.ac.uk/biomodels/BIOMD0000000401)<br/><sup>Ayati2010_BoneRemodelingDynami</sup></summary><sup>Ayati2010_BoneRemodelingDynamics_NormalCondition</sup></details>|<details><summary>pass</summary>[BIOMD0000000401_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000401#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000402](https://www.ebi.ac.uk/biomodels/BIOMD0000000402)<br/><sup>Ayati2010_BoneRemodelingDynami</sup></summary><sup>Ayati2010_BoneRemodelingDynamics_WithTumour</sup></details>|<details><summary>pass</summary>[BIOMD0000000402_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000402#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000403](https://www.ebi.ac.uk/biomodels/BIOMD0000000403)<br/><sup>Ayati2010_BoneRemodelingDynami</sup></summary><sup>Ayati2010_BoneRemodelingDynamics_WithTumour+DrugTreatment</sup></details>|<details><summary>pass</summary>[BIOMD0000000403_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000403#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000404](https://www.ebi.ac.uk/biomodels/BIOMD0000000404)<br/><sup>Bray1993_chemotaxis</sup>|<details><summary>pass</summary>[BIOMD0000000404_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000404#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000405](https://www.ebi.ac.uk/biomodels/BIOMD0000000405)<br/><sup>Cookson2011_EnzymaticQueueingC</sup></summary><sup>Cookson2011_EnzymaticQueueingCoupling</sup></details>|<details><summary>pass</summary>[BIOMD0000000405_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000405#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000406](https://www.ebi.ac.uk/biomodels/BIOMD0000000406)<br/><sup>Moriya2011_CellCycle_FissionYeast</sup>|<details><summary>pass</summary>[BIOMD0000000406_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000406#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000407](https://www.ebi.ac.uk/biomodels/BIOMD0000000407)<br/><sup>Schliemann2011_TNF_ProAntiApoptosis</sup>|<details><summary>pass</summary>[BIOMD0000000407_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000407#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000408](https://www.ebi.ac.uk/biomodels/BIOMD0000000408)<br/><sup>Hettling2011_CreatineKinase</sup>|<details><summary>pass</summary>[BIOMD0000000408_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000408#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000409](https://www.ebi.ac.uk/biomodels/BIOMD0000000409)<br/><sup>Queralt2006_MitoticExit_Cdc55D</sup></summary><sup>Queralt2006_MitoticExit_Cdc55DownregulationBySeparase</sup></details>|<details><summary>pass</summary>[BIOMD0000000409_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000409#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000410](https://www.ebi.ac.uk/biomodels/BIOMD0000000410)<br/><sup>Wegner2012_TGFbetaSignalling_F</sup></summary><sup>Wegner2012_TGFbetaSignalling_FeedbackLoops</sup></details>|<details><summary>pass</summary>[BIOMD0000000410_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000410#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000411](https://www.ebi.ac.uk/biomodels/BIOMD0000000411)<br/><sup>Heiland2012_CircadianClock_C.r</sup></summary><sup>Heiland2012_CircadianClock_C.reinhardtii</sup></details>|<details><summary>pass</summary>[BIOMD0000000411_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000411#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000412](https://www.ebi.ac.uk/biomodels/BIOMD0000000412)<br/><sup>Pokhilko2012_CircClock_Repress</sup></summary><sup>Pokhilko2012_CircClock_RepressilatorFeedbackloop</sup></details>|<details><summary>pass</summary>[BIOMD0000000412_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000412#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000413](https://www.ebi.ac.uk/biomodels/BIOMD0000000413)<br/><sup>Band2012_DII-Venus_FullModel</sup>|<details><summary>pass</summary>[BIOMD0000000413_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000413#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000414](https://www.ebi.ac.uk/biomodels/BIOMD0000000414)<br/><sup>Band2012_DII-Venus_ReducedModel</sup>|<details><summary>pass</summary>[BIOMD0000000414_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000414#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000415](https://www.ebi.ac.uk/biomodels/BIOMD0000000415)<br/><sup>Mellor2012_LipooxygenasePathway</sup>|<details><summary>pass</summary>[BIOMD0000000415_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000415#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000416](https://www.ebi.ac.uk/biomodels/BIOMD0000000416)<br/><sup>Muraro2011_Cytokinin-Auxin_Cro</sup></summary><sup>Muraro2011_Cytokinin-Auxin_CrossRegulation</sup></details>|<details><summary>pass</summary>[BIOMD0000000416_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000416#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000417](https://www.ebi.ac.uk/biomodels/BIOMD0000000417)<br/><sup>Ratushny2012_NF</sup>|<details><summary>pass</summary>[BIOMD0000000417_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000417#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000418](https://www.ebi.ac.uk/biomodels/BIOMD0000000418)<br/><sup>Ratushny2012_SPF</sup>|<details><summary>pass</summary>[BIOMD0000000418_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000418#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000419](https://www.ebi.ac.uk/biomodels/BIOMD0000000419)<br/><sup>Ratushny2012_SPF_I</sup>|<details><summary>pass</summary>[BIOMD0000000419_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000419#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000420](https://www.ebi.ac.uk/biomodels/BIOMD0000000420)<br/><sup>Ratushny2012_ASSURE_I</sup>|<details><summary>pass</summary>[BIOMD0000000420_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000420#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000421](https://www.ebi.ac.uk/biomodels/BIOMD0000000421)<br/><sup>Ratushny2012_ASSURE_II</sup>|<details><summary>pass</summary>[BIOMD0000000421_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000421#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000422](https://www.ebi.ac.uk/biomodels/BIOMD0000000422)<br/><sup>Middleton2012_GibberellinSignalling</sup>|<details><summary>pass</summary>[BIOMD0000000422_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000422#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000423](https://www.ebi.ac.uk/biomodels/BIOMD0000000423)<br/><sup>Nyman2012_InsulinSignalling</sup>|<details><summary>pass</summary>[BIOMD0000000423_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000423#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000424](https://www.ebi.ac.uk/biomodels/BIOMD0000000424)<br/><sup>Faratian2009 - Role of PTEN in</sup></summary><sup>Faratian2009 - Role of PTEN in Trastuzumab resistance</sup></details>|<details><summary>pass</summary>[BIOMD0000000424_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000424#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000425](https://www.ebi.ac.uk/biomodels/BIOMD0000000425)<br/><sup>Tan2012 - Antibiotic Treatment</sup></summary><sup>Tan2012 - Antibiotic Treatment, Inoculum Effect</sup></details>|<details><summary>FAIL</summary>[BIOMD0000000425_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000425#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000426](https://www.ebi.ac.uk/biomodels/BIOMD0000000426)<br/><sup>Mosca2012 - Central Carbon Met</sup></summary><sup>Mosca2012 - Central Carbon Metabolism Regulated by AKT</sup></details>|<details><summary>pass</summary>[BIOMD0000000426_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000426#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000427](https://www.ebi.ac.uk/biomodels/BIOMD0000000427)<br/><sup>Bianconi2012 - EGFR and IGF1R </sup></summary><sup>Bianconi2012 - EGFR and IGF1R pathway in lung cancer</sup></details>|<details><summary>pass</summary>[BIOMD0000000427_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000427#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000428](https://www.ebi.ac.uk/biomodels/BIOMD0000000428)<br/><sup>Achcar2012 - Glycolysis in blo</sup></summary><sup>Achcar2012 - Glycolysis in bloodstream form T. brucei</sup></details>|<details><summary>pass</summary>[BIOMD0000000428_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000428#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000429](https://www.ebi.ac.uk/biomodels/BIOMD0000000429)<br/><sup>Schaber2012 - Hog pathway in yeast</sup>|<details><summary>pass</summary>[BIOMD0000000429_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000429#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000430](https://www.ebi.ac.uk/biomodels/BIOMD0000000430)<br/><sup>Sarma2012 - Interaction topolo</sup></summary><sup>Sarma2012 - Interaction topologies of MAPK cascade (M4_K2_USEQ)</sup></details>|<details><summary>pass</summary>[BIOMD0000000430_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000430#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000431](https://www.ebi.ac.uk/biomodels/BIOMD0000000431)<br/><sup>Sarma2012 - Interaction topolo</sup></summary><sup>Sarma2012 - Interaction topologies of MAPK cascade (M4_K2_PSEQ)</sup></details>|<details><summary>pass</summary>[BIOMD0000000431_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000431#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000432](https://www.ebi.ac.uk/biomodels/BIOMD0000000432)<br/><sup>Sarma2012 - Interaction topolo</sup></summary><sup>Sarma2012 - Interaction topologies of MAPK cascade (M4_K2_QSS_USEQ)</sup></details>|<details><summary>pass</summary>[BIOMD0000000432_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000432#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000433](https://www.ebi.ac.uk/biomodels/BIOMD0000000433)<br/><sup>Sarma2012 - Interaction topolo</sup></summary><sup>Sarma2012 - Interaction topologies of MAPK cascade (M4_K2_QSS_PSEQ)</sup></details>|<details><summary>pass</summary>[BIOMD0000000433_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000433#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000434](https://www.ebi.ac.uk/biomodels/BIOMD0000000434)<br/><sup>McAuley2012 - Whole-body Chole</sup></summary><sup>McAuley2012 - Whole-body Cholesterol Metabolism</sup></details>|<details><summary>pass</summary>[BIOMD0000000434_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000434#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000435](https://www.ebi.ac.uk/biomodels/BIOMD0000000435)<br/><sup>deBack2012 - Lineage Specifica</sup></summary><sup>deBack2012 - Lineage Specification in Pancreas Development</sup></details>|<details><summary>pass</summary>[BIOMD0000000435_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000435#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000436](https://www.ebi.ac.uk/biomodels/BIOMD0000000436)<br/><sup>Gupta2009 - Eicosanoid Metabolism</sup>|<details><summary>pass</summary>[BIOMD0000000436_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000436#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000437](https://www.ebi.ac.uk/biomodels/BIOMD0000000437)<br/><sup>Tseng2012 - Circadian clock of</sup></summary><sup>Tseng2012 - Circadian clock of N.crassa</sup></details>|<details><summary>FAIL</summary>[BIOMD0000000437_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000437#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000438](https://www.ebi.ac.uk/biomodels/BIOMD0000000438)<br/><sup>Saeidi2012 - Quorum sensing de</sup></summary><sup>Saeidi2012 - Quorum sensing device that produces GFP</sup></details>|<details><summary>pass</summary>[BIOMD0000000438_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000438#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000439](https://www.ebi.ac.uk/biomodels/BIOMD0000000439)<br/><sup>Smith2009 - RGS mediated GTP h</sup></summary><sup>Smith2009 - RGS mediated GTP hydrolysis</sup></details>|<details><summary>FAIL</summary>[BIOMD0000000439_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000439#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000440](https://www.ebi.ac.uk/biomodels/BIOMD0000000440)<br/><sup>Sarma2012 - Oscillations in MA</sup></summary><sup>Sarma2012 - Oscillations in MAPK cascade (S1)</sup></details>|<details><summary>pass</summary>[BIOMD0000000440_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000440#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000441](https://www.ebi.ac.uk/biomodels/BIOMD0000000441)<br/><sup>Sarma2012 - Oscillations in MA</sup></summary><sup>Sarma2012 - Oscillations in MAPK cascade (S2)</sup></details>|<details><summary>pass</summary>[BIOMD0000000441_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000441#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000442](https://www.ebi.ac.uk/biomodels/BIOMD0000000442)<br/><sup>Sarma2012 - Oscillations in MA</sup></summary><sup>Sarma2012 - Oscillations in MAPK cascade (S2), inclusion of external signalling module</sup></details>|<details><summary>pass</summary>[BIOMD0000000442_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000442#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000443](https://www.ebi.ac.uk/biomodels/BIOMD0000000443)<br/><sup>Sarma2012 - Oscillations in MA</sup></summary><sup>Sarma2012 - Oscillations in MAPK cascade (S1n)</sup></details>|<details><summary>pass</summary>[BIOMD0000000443_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000443#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000444](https://www.ebi.ac.uk/biomodels/BIOMD0000000444)<br/><sup>Sarma2012 - Oscillations in MA</sup></summary><sup>Sarma2012 - Oscillations in MAPK cascade (S2n)</sup></details>|<details><summary>pass</summary>[BIOMD0000000444_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000444#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000445](https://www.ebi.ac.uk/biomodels/BIOMD0000000445)<br/><sup>Pokhilko2013 - TOC1 signalling</sup></summary><sup>Pokhilko2013 - TOC1 signalling in Arabidopsis circadian clock</sup></details>|<details><summary>pass</summary>[BIOMD0000000445_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000445#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000446](https://www.ebi.ac.uk/biomodels/BIOMD0000000446)<br/><sup>Erguler2013 - Unfolded protein</sup></summary><sup>Erguler2013 - Unfolded protein stress response</sup></details>|<details><summary>pass</summary>[BIOMD0000000446_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000446#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000447](https://www.ebi.ac.uk/biomodels/BIOMD0000000447)<br/><sup>Venkatraman2012 - Interplay be</sup></summary><sup>Venkatraman2012 - Interplay between PLS and TSP1 in TGF-1 activation</sup></details>|<details><summary>pass</summary>[BIOMD0000000447_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000447#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000448](https://www.ebi.ac.uk/biomodels/BIOMD0000000448)<br/><sup>Brnnmark2013 - Insulin signal</sup></summary><sup>Brnnmark2013 - Insulin signalling in human adipocytes (normal condition)</sup></details>|<details><summary>pass</summary>[BIOMD0000000448_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000448#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000449](https://www.ebi.ac.uk/biomodels/BIOMD0000000449)<br/><sup>Brnnmark2013 - Insulin signal</sup></summary><sup>Brnnmark2013 - Insulin signalling in human adipocytes (diabetic condition)</sup></details>|<details><summary>pass</summary>[BIOMD0000000449_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000449#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000450](https://www.ebi.ac.uk/biomodels/BIOMD0000000450)<br/><sup>Reyes-Palomares2012 -  a combi</sup></summary><sup>Reyes-Palomares2012 -  a combined model hepatic polyamine and sulfur aminoacid metabolism - version2</sup></details>|<details><summary>pass</summary>[BIOMD0000000450_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000450#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000451](https://www.ebi.ac.uk/biomodels/BIOMD0000000451)<br/><sup>Carbo2013 - Cytokine driven CD</sup></summary><sup>Carbo2013 - Cytokine driven CD4+ T Cell differentiation and phenotype plasticity</sup></details>|<details><summary>pass</summary>[BIOMD0000000451_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000451#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000452](https://www.ebi.ac.uk/biomodels/BIOMD0000000452)<br/><sup>Bidkhori2012 - normal EGFR sig</sup></summary><sup>Bidkhori2012 - normal EGFR signalling</sup></details>|<details><summary>pass</summary>[BIOMD0000000452_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000452#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000453](https://www.ebi.ac.uk/biomodels/BIOMD0000000453)<br/><sup>Bidkhori2012 - EGFR signalling</sup></summary><sup>Bidkhori2012 - EGFR signalling in NSCLC</sup></details>|<details><summary>pass</summary>[BIOMD0000000453_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000453#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000454](https://www.ebi.ac.uk/biomodels/BIOMD0000000454)<br/><sup>Smallbone2013 - Metabolic Cont</sup></summary><sup>Smallbone2013 - Metabolic Control Analysis - Example 1</sup></details>|<details><summary>pass</summary>[BIOMD0000000454_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000454#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000455](https://www.ebi.ac.uk/biomodels/BIOMD0000000455)<br/><sup>Smallbone2013 - Metabolic Cont</sup></summary><sup>Smallbone2013 - Metabolic Control Analysis - Example 2</sup></details>|<details><summary>pass</summary>[BIOMD0000000455_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000455#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000456](https://www.ebi.ac.uk/biomodels/BIOMD0000000456)<br/><sup>Smallbone2013 - Metabolic Cont</sup></summary><sup>Smallbone2013 - Metabolic Control Analysis - Example 3</sup></details>|<details><summary>pass</summary>[BIOMD0000000456_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000456#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000457](https://www.ebi.ac.uk/biomodels/BIOMD0000000457)<br/><sup>Firczuk2013 - Eukaryotic mRNA </sup></summary><sup>Firczuk2013 - Eukaryotic mRNA translation machinery</sup></details>|<details><summary>pass</summary>[BIOMD0000000457_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000457#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000458](https://www.ebi.ac.uk/biomodels/BIOMD0000000458)<br/><sup>Smallbone2013 - Serine biosynthesis</sup>|<details><summary>pass</summary>[BIOMD0000000458_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000458#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000459](https://www.ebi.ac.uk/biomodels/BIOMD0000000459)<br/><sup>Liebal2012 - B.subtilis post-t</sup></summary><sup>Liebal2012 - B.subtilis post-transcriptional instability model</sup></details>|<details><summary>pass</summary>[BIOMD0000000459_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000459#Files)</details>|pass|<details><summary>FAIL</summary>[BIOMD0000000459-SEDML.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000459#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/BIOMD0000000459-SEDML.xml984908.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/BIOMD0000000459-SEDML.xml984908.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./BIOMD0000000459-SEDML.xml` is invalid.\n - A <model> object may contain one and only one instance of the <listOfChanges> element. No other elements from the SBML Level 3 SED-ML namespaces are permitted on a <model> object. \n Reference: L3V1 Sedml V1 Section\n \n - A <model> object may contain one and only one instance of the <listOfChanges> element. No other elements from the SBML Level 3 SED-ML namespaces are permitted on a <model> object. \n Reference: L3V1 Sedml V1 Section\n \n'```</details>|<details><summary>other</summary>```Command '-i /root/in/BIOMD0000000459-SEDML.xml397621.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/BIOMD0000000459-SEDML.xml397621.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./BIOMD0000000459-SEDML.xml` is invalid.\n - A <model> object may contain one and only one instance of the <listOfChanges> element. No other elements from the SBML Level 3 SED-ML namespaces are permitted on a <model> object. \n Reference: L3V1 Sedml V1 Section\n \n - A <model> object may contain one and only one instance of the <listOfChanges> element. No other elements from the SBML Level 3 SED-ML namespaces are permitted on a <model> object. \n Reference: L3V1 Sedml V1 Section\n \n'```</details>|
|<details><summary>[BIOMD0000000460](https://www.ebi.ac.uk/biomodels/BIOMD0000000460)<br/><sup>Liebal2012 - B.subtilis sigB p</sup></summary><sup>Liebal2012 - B.subtilis sigB proteolysis model</sup></details>|<details><summary>pass</summary>[BIOMD0000000460_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000460#Files)</details>|pass|<details><summary>FAIL</summary>[BIOMD0000000460-SEDML.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000460#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/BIOMD0000000460-SEDML.xml17104.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/BIOMD0000000460-SEDML.xml17104.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./BIOMD0000000460-SEDML.xml` is invalid.\n - A <model> object may contain one and only one instance of the <listOfChanges> element. No other elements from the SBML Level 3 SED-ML namespaces are permitted on a <model> object. \n Reference: L3V1 Sedml V1 Section\n \n - A <model> object may contain one and only one instance of the <listOfChanges> element. No other elements from the SBML Level 3 SED-ML namespaces are permitted on a <model> object. \n Reference: L3V1 Sedml V1 Section\n \n'```</details>|<details><summary>other</summary>```Command '-i /root/in/BIOMD0000000460-SEDML.xml240020.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/BIOMD0000000460-SEDML.xml240020.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./BIOMD0000000460-SEDML.xml` is invalid.\n - A <model> object may contain one and only one instance of the <listOfChanges> element. No other elements from the SBML Level 3 SED-ML namespaces are permitted on a <model> object. \n Reference: L3V1 Sedml V1 Section\n \n - A <model> object may contain one and only one instance of the <listOfChanges> element. No other elements from the SBML Level 3 SED-ML namespaces are permitted on a <model> object. \n Reference: L3V1 Sedml V1 Section\n \n'```</details>|
|<details><summary>[BIOMD0000000461](https://www.ebi.ac.uk/biomodels/BIOMD0000000461)<br/><sup>Liebal2012 - B.subtilis transc</sup></summary><sup>Liebal2012 - B.subtilis transcription inhibition model</sup></details>|<details><summary>pass</summary>[BIOMD0000000461_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000461#Files)</details>|pass|<details><summary>FAIL</summary>[BIOMD0000000461-SEDML.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000461#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/BIOMD0000000461-SEDML.xml487462.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/BIOMD0000000461-SEDML.xml487462.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./BIOMD0000000461-SEDML.xml` is invalid.\n - A <model> object may contain one and only one instance of the <listOfChanges> element. No other elements from the SBML Level 3 SED-ML namespaces are permitted on a <model> object. \n Reference: L3V1 Sedml V1 Section\n \n - A <model> object may contain one and only one instance of the <listOfChanges> element. No other elements from the SBML Level 3 SED-ML namespaces are permitted on a <model> object. \n Reference: L3V1 Sedml V1 Section\n \n'```</details>|<details><summary>other</summary>```Command '-i /root/in/BIOMD0000000461-SEDML.xml147278.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/BIOMD0000000461-SEDML.xml147278.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./BIOMD0000000461-SEDML.xml` is invalid.\n - A <model> object may contain one and only one instance of the <listOfChanges> element. No other elements from the SBML Level 3 SED-ML namespaces are permitted on a <model> object. \n Reference: L3V1 Sedml V1 Section\n \n - A <model> object may contain one and only one instance of the <listOfChanges> element. No other elements from the SBML Level 3 SED-ML namespaces are permitted on a <model> object. \n Reference: L3V1 Sedml V1 Section\n \n'```</details>|
|<details><summary>[BIOMD0000000462](https://www.ebi.ac.uk/biomodels/BIOMD0000000462)<br/><sup>Proctor2012 - Role of Amyloid-</sup></summary><sup>Proctor2012 - Role of Amyloid-beta dimers in aggregation formation</sup></details>|<details><summary>pass</summary>[BIOMD0000000462_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000462#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000463](https://www.ebi.ac.uk/biomodels/BIOMD0000000463)<br/><sup>Heldt2012 - Influenza Virus Re</sup></summary><sup>Heldt2012 - Influenza Virus Replication</sup></details>|<details><summary>pass</summary>[BIOMD0000000463_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000463#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000464](https://www.ebi.ac.uk/biomodels/BIOMD0000000464)<br/><sup>Koo2013 - Shear stress induced</sup></summary><sup>Koo2013 - Shear stress induced calcium influx and eNOS activation - Model 1</sup></details>|<details><summary>pass</summary>[BIOMD0000000464_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000464#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000465](https://www.ebi.ac.uk/biomodels/BIOMD0000000465)<br/><sup>Koo2013 - Shear stress induced</sup></summary><sup>Koo2013 - Shear stress induced AKT and eNOS phosphorylation - Model 2</sup></details>|<details><summary>pass</summary>[BIOMD0000000465_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000465#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000466](https://www.ebi.ac.uk/biomodels/BIOMD0000000466)<br/><sup>Koo2013 - Shear stress induced</sup></summary><sup>Koo2013 - Shear stress induced eNOS expression - Model 3</sup></details>|<details><summary>pass</summary>[BIOMD0000000466_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000466#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000467](https://www.ebi.ac.uk/biomodels/BIOMD0000000467)<br/><sup>Koo2013 - Shear stress induced</sup></summary><sup>Koo2013 - Shear stress induced NO production - Model 4</sup></details>|<details><summary>pass</summary>[BIOMD0000000467_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000467#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000468](https://www.ebi.ac.uk/biomodels/BIOMD0000000468)<br/><sup>Koo2013 - Integrated shear str</sup></summary><sup>Koo2013 - Integrated shear stress induced NO production model</sup></details>|<details><summary>pass</summary>[BIOMD0000000468_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000468#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000469](https://www.ebi.ac.uk/biomodels/BIOMD0000000469)<br/><sup>Smallbone2013 - E.coli metabol</sup></summary><sup>Smallbone2013 - E.coli metabolic model with linlog rate law</sup></details>|<details><summary>pass</summary>[BIOMD0000000469_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000469#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000470](https://www.ebi.ac.uk/biomodels/BIOMD0000000470)<br/><sup>Smallbone2013 - E.coli metabol</sup></summary><sup>Smallbone2013 - E.coli metabolic model with modular rate law</sup></details>|<details><summary>pass</summary>[BIOMD0000000470_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000470#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000471](https://www.ebi.ac.uk/biomodels/BIOMD0000000471)<br/><sup>Smallbone2013 - Yeast metaboli</sup></summary><sup>Smallbone2013 - Yeast metabolic model with linlog rate law</sup></details>|<details><summary>pass</summary>[BIOMD0000000471_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000471#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000472](https://www.ebi.ac.uk/biomodels/BIOMD0000000472)<br/><sup>Smallbone2013 - Yeast metaboli</sup></summary><sup>Smallbone2013 - Yeast metabolic model with modular rate law</sup></details>|<details><summary>pass</summary>[BIOMD0000000472_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000472#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000473](https://www.ebi.ac.uk/biomodels/BIOMD0000000473)<br/><sup>Smallbone2013 - Yeast metaboli</sup></summary><sup>Smallbone2013 - Yeast metabolic model with modular rate law, merged with Pritchard 2002</sup></details>|<details><summary>pass</summary>[BIOMD0000000473_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000473#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000474](https://www.ebi.ac.uk/biomodels/BIOMD0000000474)<br/><sup>Smith2013 - Regulation of Insu</sup></summary><sup>Smith2013 - Regulation of Insulin Signalling by Oxidative Stress</sup></details>|<details><summary>pass</summary>[BIOMD0000000474_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000474#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000475](https://www.ebi.ac.uk/biomodels/BIOMD0000000475)<br/><sup>Amara2013 - PCNA ubiquitylatio</sup></summary><sup>Amara2013 - PCNA ubiquitylation in the activation of PRR pathway</sup></details>|<details><summary>pass</summary>[BIOMD0000000475_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000475#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000476](https://www.ebi.ac.uk/biomodels/BIOMD0000000476)<br/><sup>Adams2012 - Locke2006 Circadia</sup></summary><sup>Adams2012 - Locke2006 Circadian Rhythm model refined with Input Signal Light Function</sup></details>|<details><summary>pass</summary>[BIOMD0000000476_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000476#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000477](https://www.ebi.ac.uk/biomodels/BIOMD0000000477)<br/><sup>Mol2013 - Immune Signal Transd</sup></summary><sup>Mol2013 - Immune Signal Transduction in Leishmaniasis</sup></details>|<details><summary>pass</summary>[BIOMD0000000477_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000477#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000478](https://www.ebi.ac.uk/biomodels/BIOMD0000000478)<br/><sup>Besozzi2012 - Oscillatory regi</sup></summary><sup>Besozzi2012 - Oscillatory regimes in the Ras/cAMP/PKA pathway in S.cerevisiae</sup></details>|<details><summary>pass</summary>[BIOMD0000000478_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000478#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000479](https://www.ebi.ac.uk/biomodels/BIOMD0000000479)<br/><sup>Croft2013 - GPCR-RGS interacti</sup></summary><sup>Croft2013 - GPCR-RGS interaction that compartmentalizes RGS activity</sup></details>|<details><summary>FAIL</summary>[BIOMD0000000479_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000479#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000480](https://www.ebi.ac.uk/biomodels/BIOMD0000000480)<br/><sup>Carbo2013 - Mucosal Immune Res</sup></summary><sup>Carbo2013 - Mucosal Immune Response during H.pylori Infection</sup></details>|<details><summary>pass</summary>[BIOMD0000000480_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000480#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000481](https://www.ebi.ac.uk/biomodels/BIOMD0000000481)<br/><sup>Sttzel2012 - Bovine estrous c</sup></summary><sup>Sttzel2012 - Bovine estrous cycle, synchronization with prostaglandin F2</sup></details>|<details><summary>pass</summary>[BIOMD0000000481_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000481#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000482](https://www.ebi.ac.uk/biomodels/BIOMD0000000482)<br/><sup>Noguchi2013 - Insulin dependen</sup></summary><sup>Noguchi2013 - Insulin dependent glucose metabolism</sup></details>|<details><summary>pass</summary>[BIOMD0000000482_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000482#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000483](https://www.ebi.ac.uk/biomodels/BIOMD0000000483)<br/><sup>Cao2008 - Network of a toggle switch</sup>|<details><summary>pass</summary>[BIOMD0000000483_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000483#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000484](https://www.ebi.ac.uk/biomodels/BIOMD0000000484)<br/><sup>Cao2013 - Application of ABSIS</sup></summary><sup>Cao2013 - Application of ABSIS method in birth-death process</sup></details>|<details><summary>pass</summary>[BIOMD0000000484_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000484#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000485](https://www.ebi.ac.uk/biomodels/BIOMD0000000485)<br/><sup>Cao2013 - Application of ABSIS</sup></summary><sup>Cao2013 - Application of ABSIS method in the bistable Schlgl model</sup></details>|<details><summary>pass</summary>[BIOMD0000000485_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000485#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000486](https://www.ebi.ac.uk/biomodels/BIOMD0000000486)<br/><sup>Cao2013 - Application of ABSIS</sup></summary><sup>Cao2013 - Application of ABSIS method in the reversible isomerization model</sup></details>|<details><summary>pass</summary>[BIOMD0000000486_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000486#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000487](https://www.ebi.ac.uk/biomodels/BIOMD0000000487)<br/><sup>Cao2013 - Application of ABSIS</sup></summary><sup>Cao2013 - Application of ABSIS in the the enzymatic futile cycle</sup></details>|<details><summary>pass</summary>[BIOMD0000000487_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000487#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000488](https://www.ebi.ac.uk/biomodels/BIOMD0000000488)<br/><sup>Proctor2013 - Effect of A imm</sup></summary><sup>Proctor2013 - Effect of A immunisation in Alzheimer's disease (deterministic version)</sup></details>|<details><summary>pass</summary>[BIOMD0000000488_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000488#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000489](https://www.ebi.ac.uk/biomodels/BIOMD0000000489)<br/><sup>Sharp2013 - Lipopolysaccharide</sup></summary><sup>Sharp2013 - Lipopolysaccharide induced NFkB activation</sup></details>|<details><summary>pass</summary>[BIOMD0000000489_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000489#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000490](https://www.ebi.ac.uk/biomodels/BIOMD0000000490)<br/><sup>Demin2013 - PKPD behaviour - 5</sup></summary><sup>Demin2013 - PKPD behaviour - 5-Lipoxygenase inhibitors</sup></details>|<details><summary>pass</summary>[BIOMD0000000490_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000490#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000491](https://www.ebi.ac.uk/biomodels/BIOMD0000000491)<br/><sup>Pathak2013 - MAPK activation i</sup></summary><sup>Pathak2013 - MAPK activation in response to various abiotic stresses</sup></details>|<details><summary>pass</summary>[BIOMD0000000491_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000491#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000492](https://www.ebi.ac.uk/biomodels/BIOMD0000000492)<br/><sup>Pathak2013 - MAPK activation i</sup></summary><sup>Pathak2013 - MAPK activation in response to various biotic stresses</sup></details>|<details><summary>pass</summary>[BIOMD0000000492_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000492#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000493](https://www.ebi.ac.uk/biomodels/BIOMD0000000493)<br/><sup>Schittler2010 - Cell fate of p</sup></summary><sup>Schittler2010 - Cell fate of progenitor cells, osteoblasts or chondrocytes</sup></details>|<details><summary>pass</summary>[BIOMD0000000493_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000493#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000494](https://www.ebi.ac.uk/biomodels/BIOMD0000000494)<br/><sup>Roblitz2013 - Menstrual Cycle </sup></summary><sup>Roblitz2013 - Menstrual Cycle following GnRH analogue administration</sup></details>|<details><summary>pass</summary>[BIOMD0000000494_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000494#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000495](https://www.ebi.ac.uk/biomodels/BIOMD0000000495)<br/><sup>Sen2013 - Phospholipid Synthes</sup></summary><sup>Sen2013 - Phospholipid Synthesis in P.knowlesi</sup></details>|<details><summary>pass</summary>[BIOMD0000000495_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000495#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000496](https://www.ebi.ac.uk/biomodels/BIOMD0000000496)<br/><sup>Stanford2013 - Kinetic model o</sup></summary><sup>Stanford2013 - Kinetic model of yeast metabolic network (standard)</sup></details>|<details><summary>pass</summary>[BIOMD0000000496_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000496#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000497](https://www.ebi.ac.uk/biomodels/BIOMD0000000497)<br/><sup>Stanford2013 - Kinetic model o</sup></summary><sup>Stanford2013 - Kinetic model of yeast metabolic network (regulation)</sup></details>|<details><summary>pass</summary>[BIOMD0000000497_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000497#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000498](https://www.ebi.ac.uk/biomodels/BIOMD0000000498)<br/><sup>Mitchell2013 - Liver Iron Metabolism</sup>|<details><summary>pass</summary>[BIOMD0000000498_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000498#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000499](https://www.ebi.ac.uk/biomodels/BIOMD0000000499)<br/><sup>Vizan2013 - TGF pathway long t</sup></summary><sup>Vizan2013 - TGF pathway long term signaling</sup></details>|<details><summary>pass</summary>[BIOMD0000000499_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000499#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000500](https://www.ebi.ac.uk/biomodels/BIOMD0000000500)<br/><sup>Begitt2014 - STAT1 cooperative</sup></summary><sup>Begitt2014 - STAT1 cooperative DNA binding - single GAS polymer model</sup></details>|<details><summary>pass</summary>[BIOMD0000000500_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000500#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000501](https://www.ebi.ac.uk/biomodels/BIOMD0000000501)<br/><sup>Begitt2014 - STAT1 cooperative</sup></summary><sup>Begitt2014 - STAT1 cooperative DNA binding - double GAS polymer model</sup></details>|<details><summary>pass</summary>[BIOMD0000000501_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000501#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000502](https://www.ebi.ac.uk/biomodels/BIOMD0000000502)<br/><sup>Messiha2013 - Pentose phosphat</sup></summary><sup>Messiha2013 - Pentose phosphate pathway model</sup></details>|<details><summary>pass</summary>[BIOMD0000000502_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000502#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000503](https://www.ebi.ac.uk/biomodels/BIOMD0000000503)<br/><sup>Messiha2013 - combined glycoly</sup></summary><sup>Messiha2013 - combined glycolysis and pentose phosphate pathway model</sup></details>|<details><summary>pass</summary>[BIOMD0000000503_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000503#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000504](https://www.ebi.ac.uk/biomodels/BIOMD0000000504)<br/><sup>Proctor2013 - Cartilage breakd</sup></summary><sup>Proctor2013 - Cartilage breakdown, interventions to reduce collagen release</sup></details>|<details><summary>pass</summary>[BIOMD0000000504_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000504#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000505](https://www.ebi.ac.uk/biomodels/BIOMD0000000505)<br/><sup>vanEunen2013 - Network dynamic</sup></summary><sup>vanEunen2013 - Network dynamics of fatty acid -oxidation (steady-state model)</sup></details>|<details><summary>pass</summary>[BIOMD0000000505_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000505#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000506](https://www.ebi.ac.uk/biomodels/BIOMD0000000506)<br/><sup>vanEunen2013 - Network dynamic</sup></summary><sup>vanEunen2013 - Network dynamics of fatty acid -oxidation (time-course model)</sup></details>|<details><summary>pass</summary>[BIOMD0000000506_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000506#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000507](https://www.ebi.ac.uk/biomodels/BIOMD0000000507)<br/><sup>Gardner2000 - genetic toggle s</sup></summary><sup>Gardner2000 - genetic toggle switch in E.coli</sup></details>|<details><summary>pass</summary>[BIOMD0000000507_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000507#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000508](https://www.ebi.ac.uk/biomodels/BIOMD0000000508)<br/><sup>Barrack2014 - Calcium/cell cyc</sup></summary><sup>Barrack2014 - Calcium/cell cycle coupling - Cyclin D dependent ATP release</sup></details>|<details><summary>FAIL</summary>[BIOMD0000000508_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000508#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000509](https://www.ebi.ac.uk/biomodels/BIOMD0000000509)<br/><sup>Barrack2014 - Calcium/cell cyc</sup></summary><sup>Barrack2014 - Calcium/cell cycle coupling - Rs dependent ATP release</sup></details>|<details><summary>FAIL</summary>[BIOMD0000000509_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000509#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000510](https://www.ebi.ac.uk/biomodels/BIOMD0000000510)<br/><sup>Kerkhoven2013 - Glycolysis and</sup></summary><sup>Kerkhoven2013 - Glycolysis and Pentose Phosphate Pathway in T.brucei - MODEL C (with glucosomal ribokinase)</sup></details>|<details><summary>pass</summary>[BIOMD0000000510_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000510#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000511](https://www.ebi.ac.uk/biomodels/BIOMD0000000511)<br/><sup>Kerkhoven2013 - Glycolysis and</sup></summary><sup>Kerkhoven2013 - Glycolysis and Pentose Phosphate Pathway in T.brucei - MODEL D (with ATP:ADP antiporter)</sup></details>|<details><summary>pass</summary>[BIOMD0000000511_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000511#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000512](https://www.ebi.ac.uk/biomodels/BIOMD0000000512)<br/><sup>Benson2014 - FAAH inhibitors f</sup></summary><sup>Benson2014 - FAAH inhibitors for the treatment of osteoarthritic pain</sup></details>|<details><summary>pass</summary>[BIOMD0000000512_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000512#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000513](https://www.ebi.ac.uk/biomodels/BIOMD0000000513)<br/><sup>Kerkhoven2013 - Glycolysis in </sup></summary><sup>Kerkhoven2013 - Glycolysis in T.brucei - MODEL A</sup></details>|<details><summary>pass</summary>[BIOMD0000000513_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000513#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000514](https://www.ebi.ac.uk/biomodels/BIOMD0000000514)<br/><sup>Kerkhoven2013 - Glycolysis and</sup></summary><sup>Kerkhoven2013 - Glycolysis and Pentose Phosphate Pathway in T.brucei - MODEL B</sup></details>|<details><summary>pass</summary>[BIOMD0000000514_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000514#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000515](https://www.ebi.ac.uk/biomodels/BIOMD0000000515)<br/><sup>Kerkhoven2013 - Glycolysis and</sup></summary><sup>Kerkhoven2013 - Glycolysis and Pentose Phosphate Pathway in T.brucei - MODEL C in fructose medium (with glucosomal ribokinase)</sup></details>|<details><summary>pass</summary>[BIOMD0000000515_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000515#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000516](https://www.ebi.ac.uk/biomodels/BIOMD0000000516)<br/><sup>Kerkhoven2013 - Glycolysis and</sup></summary><sup>Kerkhoven2013 - Glycolysis and Pentose Phosphate Pathway in T.brucei - MODEL D in fructose medium (with ATP:ADP antiporter)</sup></details>|<details><summary>pass</summary>[BIOMD0000000516_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000516#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000517](https://www.ebi.ac.uk/biomodels/BIOMD0000000517)<br/><sup>Smallbone2013 - Colon Crypt cy</sup></summary><sup>Smallbone2013 - Colon Crypt cycle - Version 3</sup></details>|<details><summary>pass</summary>[BIOMD0000000517_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000517#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000518](https://www.ebi.ac.uk/biomodels/BIOMD0000000518)<br/><sup>Smallbone2013 - Colon Crypt cy</sup></summary><sup>Smallbone2013 - Colon Crypt cycle - Version 2</sup></details>|<details><summary>pass</summary>[BIOMD0000000518_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000518#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000519](https://www.ebi.ac.uk/biomodels/BIOMD0000000519)<br/><sup>Smallbone2013 - Colon Crypt cy</sup></summary><sup>Smallbone2013 - Colon Crypt cycle - Version 1</sup></details>|<details><summary>pass</summary>[BIOMD0000000519_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000519#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000520](https://www.ebi.ac.uk/biomodels/BIOMD0000000520)<br/><sup>Smallbone2013 - Colon Crypt cy</sup></summary><sup>Smallbone2013 - Colon Crypt cycle - Version 0</sup></details>|<details><summary>pass</summary>[BIOMD0000000520_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000520#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000521](https://www.ebi.ac.uk/biomodels/BIOMD0000000521)<br/><sup>Ribba2012 - Low-grade gliomas,</sup></summary><sup>Ribba2012 - Low-grade gliomas, tumour growth inhibition model</sup></details>|<details><summary>pass</summary>[BIOMD0000000521_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000521#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000522](https://www.ebi.ac.uk/biomodels/BIOMD0000000522)<br/><sup>Muraro2014 - Vascular patterni</sup></summary><sup>Muraro2014 - Vascular patterning in Arabidopsis roots</sup></details>|<details><summary>pass</summary>[BIOMD0000000522_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000522#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000523](https://www.ebi.ac.uk/biomodels/BIOMD0000000523)<br/><sup>Kallenberger2014 - CD95L induc</sup></summary><sup>Kallenberger2014 - CD95L induced apoptosis initiated by caspase-8, CD95 HeLa cells (cis/trans variant)</sup></details>|<details><summary>pass</summary>[BIOMD0000000523_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000523#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000524](https://www.ebi.ac.uk/biomodels/BIOMD0000000524)<br/><sup>Kallenberger2014 - CD95L induc</sup></summary><sup>Kallenberger2014 - CD95L induced apoptosis initiated by caspase-8, wild-type HeLa cells (cis/trans variant)</sup></details>|<details><summary>pass</summary>[BIOMD0000000524_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000524#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000525](https://www.ebi.ac.uk/biomodels/BIOMD0000000525)<br/><sup>Kallenberger2014 - CD95L induc</sup></summary><sup>Kallenberger2014 - CD95L induced apoptosis initiated by caspase-8, CD95 HeLa cells (cis/trans-cis/trans variant)</sup></details>|<details><summary>pass</summary>[BIOMD0000000525_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000525#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000526](https://www.ebi.ac.uk/biomodels/BIOMD0000000526)<br/><sup>Kallenberger2014 - CD95L induc</sup></summary><sup>Kallenberger2014 - CD95L induced apoptosis initiated by caspase-8, wild-type HeLa cells (cis/trans-cis/trans variant)</sup></details>|<details><summary>pass</summary>[BIOMD0000000526_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000526#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000527](https://www.ebi.ac.uk/biomodels/BIOMD0000000527)<br/><sup>Kaiser2014 - Salmonella persis</sup></summary><sup>Kaiser2014 - Salmonella persistence after ciprofloxacin treatment</sup></details>|<details><summary>pass</summary>[BIOMD0000000527_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000527#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000528](https://www.ebi.ac.uk/biomodels/BIOMD0000000528)<br/><sup>Fribourg2014 - Dynamics of vir</sup></summary><sup>Fribourg2014 - Dynamics of viral antagonism and innate immune response (H1N1 influenza A virus - Cal/09)</sup></details>|<details><summary>pass</summary>[BIOMD0000000528_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000528#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000529](https://www.ebi.ac.uk/biomodels/BIOMD0000000529)<br/><sup>Fribourg2014 - Dynamics of vir</sup></summary><sup>Fribourg2014 - Dynamics of viral antagonism and innate immune response (H1N1 influenza A virus - NC/99)</sup></details>|<details><summary>pass</summary>[BIOMD0000000529_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000529#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000530](https://www.ebi.ac.uk/biomodels/BIOMD0000000530)<br/><sup>Schmitz2014 - RNA triplex formation</sup>|<details><summary>pass</summary>[BIOMD0000000530_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000530#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000531](https://www.ebi.ac.uk/biomodels/BIOMD0000000531)<br/><sup>Crespo2012 - Kinetics of Amylo</sup></summary><sup>Crespo2012 - Kinetics of Amyloid Fibril Formation</sup></details>|<details><summary>pass</summary>[BIOMD0000000531_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000531#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000532](https://www.ebi.ac.uk/biomodels/BIOMD0000000532)<br/><sup>Vazquez2014 - Chemical inhibit</sup></summary><sup>Vazquez2014 - Chemical inhibition from amyloid protein aggregation kinetics</sup></details>|<details><summary>pass</summary>[BIOMD0000000532_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000532#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000533](https://www.ebi.ac.uk/biomodels/BIOMD0000000533)<br/><sup>Steckmann2012 - Amyloid beta-p</sup></summary><sup>Steckmann2012 - Amyloid beta-protein fibrillogenesis (kinetics of secondary structure conversion)</sup></details>|<details><summary>pass</summary>[BIOMD0000000533_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000533#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000534](https://www.ebi.ac.uk/biomodels/BIOMD0000000534)<br/><sup>Dwivedi2014 - Healthy Voluntee</sup></summary><sup>Dwivedi2014 - Healthy Volunteer IL6 Model</sup></details>|<details><summary>pass</summary>[BIOMD0000000534_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000534#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000535](https://www.ebi.ac.uk/biomodels/BIOMD0000000535)<br/><sup>Dwivedi2014 - Crohns IL6 Disea</sup></summary><sup>Dwivedi2014 - Crohns IL6 Disease model - Anti-IL6 Antibody</sup></details>|<details><summary>pass</summary>[BIOMD0000000535_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000535#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000536](https://www.ebi.ac.uk/biomodels/BIOMD0000000536)<br/><sup>Dwivedi2014 - Crohns IL6 Disea</sup></summary><sup>Dwivedi2014 - Crohns IL6 Disease model - sgp130 activity</sup></details>|<details><summary>pass</summary>[BIOMD0000000536_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000536#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000537](https://www.ebi.ac.uk/biomodels/BIOMD0000000537)<br/><sup>Dwivedi2014 - Crohns IL6 Disea</sup></summary><sup>Dwivedi2014 - Crohns IL6 Disease model - Anti-IL6R Antibody</sup></details>|<details><summary>pass</summary>[BIOMD0000000537_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000537#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000538](https://www.ebi.ac.uk/biomodels/BIOMD0000000538)<br/><sup>Clarke2000 - One-hit model of </sup></summary><sup>Clarke2000 - One-hit model of cell death in neuronal degenerations</sup></details>|<details><summary>pass</summary>[BIOMD0000000538_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000538#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000539](https://www.ebi.ac.uk/biomodels/BIOMD0000000539)<br/><sup>Franois2005 - Mixed Feedback </sup></summary><sup>Franois2005 - Mixed Feedback Loop (two-gene network)</sup></details>|<details><summary>pass</summary>[BIOMD0000000539_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000539#Files)</details>|pass|<details><summary>FAIL</summary>[BIOMD0000000539.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000539#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/BIOMD0000000539.sedml687407.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - Model resolution via URNs will likely be deprecated in a future version of SED-ML. Instead models can be resolved by URLs.\r\n - URN model source `urn:miriam:biomodels.db:BIOMD0000000539` was not validated.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 700.7\r\n - Number of steps: 1001\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./BIOMD0000000539.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - Model resolution via URNs will likely be deprecated in a future version of SED-ML. Instead models can be resolved by URLs.\r\n - URN model source `urn:miriam:biomodels.db:BIOMD0000000539` was not validated.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 700.7\r\n - Number of steps: 1001\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time`\x1b[0m\r\n\r\n`/root/in/BIOMD0000000539.sedml687407.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./BIOMD0000000539.sedml` is invalid.\n - Output `plot1` is invalid.\n - Plot must have at least one curve.\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/BIOMD0000000539.sedml38333.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - Model resolution via URNs will likely be deprecated in a future version of SED-ML. Instead models can be resolved by URLs.\r\n - URN model source `urn:miriam:biomodels.db:BIOMD0000000539` was not validated.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 700.7\r\n - Number of steps: 1001\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./BIOMD0000000539.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - Model resolution via URNs will likely be deprecated in a future version of SED-ML. Instead models can be resolved by URLs.\r\n - URN model source `urn:miriam:biomodels.db:BIOMD0000000539` was not validated.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 700.7\r\n - Number of steps: 1001\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/BIOMD0000000539.sedml38333.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./BIOMD0000000539.sedml` is invalid.\n - Output `plot1` is invalid.\n - Plot must have at least one curve.\n'```</details>|
|<details><summary>[BIOMD0000000540](https://www.ebi.ac.uk/biomodels/BIOMD0000000540)<br/><sup>Yugi2014 - Insulin induced sig</sup></summary><sup>Yugi2014 - Insulin induced signalling (PFKL phosphorylation) - model 1</sup></details>|<details><summary>pass</summary>[BIOMD0000000540_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000540#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000541](https://www.ebi.ac.uk/biomodels/BIOMD0000000541)<br/><sup>Yugi2014 - Insulin induced sig</sup></summary><sup>Yugi2014 - Insulin induced signalling (PFKL phosphorylation) - model 2</sup></details>|<details><summary>pass</summary>[BIOMD0000000541_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000541#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000542](https://www.ebi.ac.uk/biomodels/BIOMD0000000542)<br/><sup>Yuraszeck2010 - Vulnerabilitie</sup></summary><sup>Yuraszeck2010 - Vulnerabilities in the Tau Network in Tau Pathophysiology</sup></details>|<details><summary>pass</summary>[BIOMD0000000542_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000542#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000543](https://www.ebi.ac.uk/biomodels/BIOMD0000000543)<br/><sup>Qi2013 - IL-6 and IFN crosstal</sup></summary><sup>Qi2013 - IL-6 and IFN crosstalk model (non-competitive)</sup></details>|<details><summary>pass</summary>[BIOMD0000000543_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000543#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000544](https://www.ebi.ac.uk/biomodels/BIOMD0000000544)<br/><sup>Qi2013 - IL-6 and IFN crosstal</sup></summary><sup>Qi2013 - IL-6 and IFN crosstalk model</sup></details>|<details><summary>pass</summary>[BIOMD0000000544_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000544#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000545](https://www.ebi.ac.uk/biomodels/BIOMD0000000545)<br/><sup>Ouyang2014 - photomorphogenic </sup></summary><sup>Ouyang2014 - photomorphogenic UV-B signalling network</sup></details>|<details><summary>pass</summary>[BIOMD0000000545_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000545#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000546](https://www.ebi.ac.uk/biomodels/BIOMD0000000546)<br/><sup>Miao2010 - Innate and adaptive</sup></summary><sup>Miao2010 - Innate and adaptive immune responses to primary Influenza A Virus infection_1_1</sup></details>|<details><summary>FAIL</summary>[BIOMD0000000546.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000546#Files)</details>|FAIL|<details><summary>pass</summary>[BIOMD0000000546.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000546#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/BIOMD0000000546.sedml892122.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/BIOMD0000000546.sedml892122.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./BIOMD0000000546.sedml` is invalid.\n - Model `model` is invalid.\n - The model file `BIOMD0000000546.xml` is invalid.\n - General SBML conformance (10102) at line 422, column 6: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L3V1 Section 4.1\n Element 'globalRenderInformation' is not part of the definition of 'listOfGlobalRenderInformation' in SBML Level 3 Version 1 Package render Version 1.\n \n - General SBML conformance (10102) at line 422, column 6: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L3V1 Section 4.1\n Element 'globalRenderInformation' is not part of the definition of 'listOfGlobalRenderInformation' in SBML Level 3 Version 1 Package render Version 1.\n \n - General SBML conformance (10102) at line 422, column 6: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L3V1 Section 4.1\n Element 'globalRenderInformation' is not part of the definition of 'listOfGlobalRenderInformation' in SBML Level 3 Version 1 Package render Version 1.\n \n - General SBML conformance (10102) at line 422, column 6: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L3V1 Section 4.1\n Element 'globalRenderInformation' is not part of the definition of 'listOfGlobalRenderInformation' in SBML Level 3 Version 1 Package render Version 1.\n \n - Data generator `s1_1_task1` is invalid.\n - Variable `p1_s1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s1']` are not defined. No namespaces are defined for the target.\n - Data generator `s2_2_task1` is invalid.\n - Variable `p1_s2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s2']` are not defined. No namespaces are defined for the target.\n - Data generator `s3_3_task1` is invalid.\n - Variable `p1_s3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s3']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/BIOMD0000000546.sedml873809.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/BIOMD0000000546.sedml873809.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./BIOMD0000000546.sedml` is invalid.\n - Model `model` is invalid.\n - The model file `BIOMD0000000546.xml` is invalid.\n - General SBML conformance (10102) at line 422, column 6: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L3V1 Section 4.1\n Element 'globalRenderInformation' is not part of the definition of 'listOfGlobalRenderInformation' in SBML Level 3 Version 1 Package render Version 1.\n \n - General SBML conformance (10102) at line 422, column 6: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L3V1 Section 4.1\n Element 'globalRenderInformation' is not part of the definition of 'listOfGlobalRenderInformation' in SBML Level 3 Version 1 Package render Version 1.\n \n - General SBML conformance (10102) at line 422, column 6: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L3V1 Section 4.1\n Element 'globalRenderInformation' is not part of the definition of 'listOfGlobalRenderInformation' in SBML Level 3 Version 1 Package render Version 1.\n \n - General SBML conformance (10102) at line 422, column 6: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L3V1 Section 4.1\n Element 'globalRenderInformation' is not part of the definition of 'listOfGlobalRenderInformation' in SBML Level 3 Version 1 Package render Version 1.\n \n - Data generator `s1_1_task1` is invalid.\n - Variable `p1_s1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s1']` are not defined. No namespaces are defined for the target.\n - Data generator `s2_2_task1` is invalid.\n - Variable `p1_s2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s2']` are not defined. No namespaces are defined for the target.\n - Data generator `s3_3_task1` is invalid.\n - Variable `p1_s3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s3']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000547](https://www.ebi.ac.uk/biomodels/BIOMD0000000547)<br/><sup>Talemi2014 - Arsenic toxicity </sup></summary><sup>Talemi2014 - Arsenic toxicity and detoxification mechanisms in yeast</sup></details>|<details><summary>pass</summary>[BIOMD0000000547_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000547#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000548](https://www.ebi.ac.uk/biomodels/BIOMD0000000548)<br/><sup>Sneppen2009 - Modeling proteas</sup></summary><sup>Sneppen2009 - Modeling proteasome dynamics in Parkinson's disease</sup></details>|<details><summary>pass</summary>[BIOMD0000000548_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000548#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000549](https://www.ebi.ac.uk/biomodels/BIOMD0000000549)<br/><sup>Baker2013 - Cytokine Mediated </sup></summary><sup>Baker2013 - Cytokine Mediated Inflammation in Rheumatoid Arthritis - Age Dependent</sup></details>|<details><summary>pass</summary>[BIOMD0000000549_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000549#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000550](https://www.ebi.ac.uk/biomodels/BIOMD0000000550)<br/><sup>Baker2013 - Cytokine Mediated </sup></summary><sup>Baker2013 - Cytokine Mediated Inflammation in Rheumatoid Arthritis</sup></details>|<details><summary>pass</summary>[BIOMD0000000550_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000550#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000551](https://www.ebi.ac.uk/biomodels/BIOMD0000000551)<br/><sup>Das2010 - Effect of a gamma-se</sup></summary><sup>Das2010 - Effect of a gamma-secretase inhibitor on Amyloid-beta dynamics</sup></details>|<details><summary>pass</summary>[BIOMD0000000551_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000551#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000552](https://www.ebi.ac.uk/biomodels/BIOMD0000000552)<br/><sup>Ehrenstein2000 - Positive-Feed</sup></summary><sup>Ehrenstein2000 - Positive-Feedback model for the loss of acetylcholine in Alzheimer's disease</sup></details>|<details><summary>pass</summary>[BIOMD0000000552_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000552#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000553](https://www.ebi.ac.uk/biomodels/BIOMD0000000553)<br/><sup>Ehrenstein1997 - The choline-l</sup></summary><sup>Ehrenstein1997 - The choline-leakage hypothesis in Alzheimer's disease</sup></details>|<details><summary>pass</summary>[BIOMD0000000553_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000553#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000554](https://www.ebi.ac.uk/biomodels/BIOMD0000000554)<br/><sup>Cloutier2009 - Brain Energy Me</sup></summary><sup>Cloutier2009 - Brain Energy Metabolism</sup></details>|<details><summary>pass</summary>[BIOMD0000000554_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000554#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000555](https://www.ebi.ac.uk/biomodels/BIOMD0000000555)<br/><sup>Auer2010 - Correlation between</sup></summary><sup>Auer2010 - Correlation between lag time and aggregation rate in protein aggregation</sup></details>|<details><summary>pass</summary>[BIOMD0000000555_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000555#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000556](https://www.ebi.ac.uk/biomodels/BIOMD0000000556)<br/><sup>Ortega2013 - Interplay between</sup></summary><sup>Ortega2013 - Interplay between secretases determines biphasic amyloid-beta level</sup></details>|<details><summary>pass</summary>[BIOMD0000000556_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000556#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000557](https://www.ebi.ac.uk/biomodels/BIOMD0000000557)<br/><sup>Reiterer2013 - pseudophosphata</sup></summary><sup>Reiterer2013 - pseudophosphatase STYX role in ERK signalling</sup></details>|<details><summary>pass</summary>[BIOMD0000000557_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000557#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000558](https://www.ebi.ac.uk/biomodels/BIOMD0000000558)<br/><sup>Cloutier2012 - Feedback motif </sup></summary><sup>Cloutier2012 - Feedback motif for Parkinson's disease</sup></details>|<details><summary>pass</summary>[BIOMD0000000558_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000558#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000559](https://www.ebi.ac.uk/biomodels/BIOMD0000000559)<br/><sup>Ouzounoglou2014 - Modeling of </sup></summary><sup>Ouzounoglou2014 - Modeling of alpha-synuclein effects on neuronal homeostasis</sup></details>|<details><summary>pass</summary>[BIOMD0000000559_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000559#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000560](https://www.ebi.ac.uk/biomodels/BIOMD0000000560)<br/><sup>Hui2016 - Age-related changes </sup></summary><sup>Hui2016 - Age-related changes in articular cartilage</sup></details>|<details><summary>pass</summary>[BIOMD0000000560_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000560#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000561](https://www.ebi.ac.uk/biomodels/BIOMD0000000561)<br/><sup>Martins2013 - True and apparen</sup></summary><sup>Martins2013 - True and apparent inhibition of amyloid fribril formation</sup></details>|<details><summary>pass</summary>[BIOMD0000000561_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000561#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000562](https://www.ebi.ac.uk/biomodels/BIOMD0000000562)<br/><sup>Chaouiya2013 - EGF and TNFalph</sup></summary><sup>Chaouiya2013 - EGF and TNFalpha mediated signalling pathway</sup></details>|<details><summary>pass</summary>[BIOMD0000000562_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000562#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000563](https://www.ebi.ac.uk/biomodels/BIOMD0000000563)<br/><sup>Pritchard2014 - plant-microbe </sup></summary><sup>Pritchard2014 - plant-microbe interaction</sup></details>|<details><summary>pass</summary>[BIOMD0000000563_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000563#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000564](https://www.ebi.ac.uk/biomodels/BIOMD0000000564)<br/><sup>Gould2013 - Temperature Sensit</sup></summary><sup>Gould2013 - Temperature Sensitive Circadian Clock</sup></details>|<details><summary>pass</summary>[BIOMD0000000564_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000564#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000565](https://www.ebi.ac.uk/biomodels/BIOMD0000000565)<br/><sup>Machado2014 - Curcumin product</sup></summary><sup>Machado2014 - Curcumin production pathway in Escherichia coli</sup></details>|<details><summary>pass</summary>[BIOMD0000000565_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000565#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000566](https://www.ebi.ac.uk/biomodels/BIOMD0000000566)<br/><sup>Morris2009 - -Synuclein aggre</sup></summary><sup>Morris2009 - -Synuclein aggregation variable temperature and pH</sup></details>|<details><summary>pass</summary>[BIOMD0000000566_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000566#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000567](https://www.ebi.ac.uk/biomodels/BIOMD0000000567)<br/><sup>Morris2008 - Fitting protein a</sup></summary><sup>Morris2008 - Fitting protein aggregation data via F-W 2-step mechanism</sup></details>|<details><summary>pass</summary>[BIOMD0000000567_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000567#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000568](https://www.ebi.ac.uk/biomodels/BIOMD0000000568)<br/><sup>Mueller2015 - Hepatocyte proli</sup></summary><sup>Mueller2015 - Hepatocyte proliferation, T160 phosphorylation of CDK2</sup></details>|<details><summary>pass</summary>[BIOMD0000000568_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000568#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000569](https://www.ebi.ac.uk/biomodels/BIOMD0000000569)<br/><sup>Dutta-Roy2015 - Opening of the</sup></summary><sup>Dutta-Roy2015 - Opening of the multiple AMPA receptor conductance states</sup></details>|<details><summary>pass</summary>[BIOMD0000000569_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000569#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000570](https://www.ebi.ac.uk/biomodels/BIOMD0000000570)<br/><sup>Aubert2002 - Coupling between </sup></summary><sup>Aubert2002 - Coupling between Brain electrical activity, Metabolism and Hemodynamics</sup></details>|<details><summary>pass</summary>[BIOMD0000000570_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000570#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000571](https://www.ebi.ac.uk/biomodels/BIOMD0000000571)<br/><sup>Nishio2008 - Design of the pho</sup></summary><sup>Nishio2008 - Design of the phosphotransferase system for enhanced glucose uptake in E. coli.</sup></details>|<details><summary>pass</summary>[BIOMD0000000571_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000571#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000572](https://www.ebi.ac.uk/biomodels/BIOMD0000000572)<br/><sup>Costa2014 - Computational Mode</sup></summary><sup>Costa2014 - Computational Model of L. lactis Metabolism</sup></details>|<details><summary>pass</summary>[BIOMD0000000572_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000572#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000573](https://www.ebi.ac.uk/biomodels/BIOMD0000000573)<br/><sup>Aguilera 2014 - HIV latency. I</sup></summary><sup>Aguilera 2014 - HIV latency. Interaction between HIV proteins and immune response</sup></details>|<details><summary>pass</summary>[BIOMD0000000573_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000573#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000574](https://www.ebi.ac.uk/biomodels/BIOMD0000000574)<br/><sup>Lai2014 - Hemiconcerted MWC mo</sup></summary><sup>Lai2014 - Hemiconcerted MWC model of intact calmodulin with two targets</sup></details>|<details><summary>pass</summary>[BIOMD0000000574_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000574#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000575](https://www.ebi.ac.uk/biomodels/BIOMD0000000575)<br/><sup>Sass2009 - Approach to an -sy</sup></summary><sup>Sass2009 - Approach to an -synuclein-based BST model of Parkinson's disease</sup></details>|<details><summary>pass</summary>[BIOMD0000000575_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000575#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000576](https://www.ebi.ac.uk/biomodels/BIOMD0000000576)<br/><sup>Kolodkin2013 - Nuclear recepto</sup></summary><sup>Kolodkin2013 - Nuclear receptor-mediated cortisol signalling network</sup></details>|<details><summary>pass</summary>[BIOMD0000000576_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000576#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000577](https://www.ebi.ac.uk/biomodels/BIOMD0000000577)<br/><sup>Zhou2015 - Circadian clock wit</sup></summary><sup>Zhou2015 - Circadian clock with immune regulator NPR1</sup></details>|<details><summary>pass</summary>[BIOMD0000000577_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000577#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000578](https://www.ebi.ac.uk/biomodels/BIOMD0000000578)<br/><sup>Invergo2014 - Phototransductio</sup></summary><sup>Invergo2014 - Phototransduction cascade in mouse rod cells</sup></details>|<details><summary>pass</summary>[BIOMD0000000578_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000578#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000579](https://www.ebi.ac.uk/biomodels/BIOMD0000000579)<br/><sup>Sengupta2015 - Knowledge base </sup></summary><sup>Sengupta2015 - Knowledge base model of human energy pool network (HEPNet)</sup></details>|<details><summary>pass</summary>[BIOMD0000000579_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000579#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000580](https://www.ebi.ac.uk/biomodels/BIOMD0000000580)<br/><sup>Sonntag2012 - mTOR model - IRS</sup></summary><sup>Sonntag2012 - mTOR model - IRS dependent regulation of AMPK by insulin</sup></details>|<details><summary>pass</summary>[BIOMD0000000580_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000580#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000581](https://www.ebi.ac.uk/biomodels/BIOMD0000000581)<br/><sup>DallePezze2012 - TSC-independe</sup></summary><sup>DallePezze2012 - TSC-independent mTORC2 regulation</sup></details>|<details><summary>pass</summary>[BIOMD0000000581_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000581#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000582](https://www.ebi.ac.uk/biomodels/BIOMD0000000582)<br/><sup>DallePezze2014 -  Cellular sen</sup></summary><sup>DallePezze2014 -  Cellular senescene-induced mitochondrial dysfunction</sup></details>|<details><summary>pass</summary>[BIOMD0000000582_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000582#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000583](https://www.ebi.ac.uk/biomodels/BIOMD0000000583)<br/><sup>Leber2015 - Mucosal immunity a</sup></summary><sup>Leber2015 - Mucosal immunity and gut microbiome interaction during C. difficile infection</sup></details>|<details><summary>pass</summary>[BIOMD0000000583_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000583#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000584](https://www.ebi.ac.uk/biomodels/BIOMD0000000584)<br/><sup>Mandlik2015 - Tristable geneti</sup></summary><sup>Mandlik2015 - Tristable genetic circuit of Leishmania</sup></details>|<details><summary>FAIL</summary>[BIOMD0000000584_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000584#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000585](https://www.ebi.ac.uk/biomodels/BIOMD0000000585)<br/><sup>Rateitschak2012 - Interferon-g</sup></summary><sup>Rateitschak2012 - Interferon-gamma (IFN) induced STAT1 signalling (PC_IFNg100)</sup></details>|<details><summary>FAIL</summary>[BIOMD0000000585_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000585#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000586](https://www.ebi.ac.uk/biomodels/BIOMD0000000586)<br/><sup>Karapetyan2016 - Genetic oscil</sup></summary><sup>Karapetyan2016 - Genetic oscillatory network - Activator Titration Circuit (ATC)</sup></details>|<details><summary>pass</summary>[BIOMD0000000586_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000586#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000587](https://www.ebi.ac.uk/biomodels/BIOMD0000000587)<br/><sup>Karapetyan2016 - Genetic oscil</sup></summary><sup>Karapetyan2016 - Genetic oscillatory network - Repressor Titration Circuit (RTC)</sup></details>|<details><summary>pass</summary>[BIOMD0000000587_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000587#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000588](https://www.ebi.ac.uk/biomodels/BIOMD0000000588)<br/><sup>Benson2013 - Identification of</sup></summary><sup>Benson2013 - Identification of key drug targets in nerve growth factor pathway</sup></details>|<details><summary>pass</summary>[BIOMD0000000588_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000588#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000589](https://www.ebi.ac.uk/biomodels/BIOMD0000000589)<br/><sup>Valero2016 - Ascorbate-Glutath</sup></summary><sup>Valero2016 - Ascorbate-Glutathione cycle in chloroplasts under light/dark conditions</sup></details>|<details><summary>pass</summary>[BIOMD0000000589_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000589#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000590](https://www.ebi.ac.uk/biomodels/BIOMD0000000590)<br/><sup>Hermansen2015 - denovo biosynt</sup></summary><sup>Hermansen2015 - denovo biosynthesis of pyrimidines in yeast</sup></details>|<details><summary>pass</summary>[BIOMD0000000590_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000590#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000591](https://www.ebi.ac.uk/biomodels/BIOMD0000000591)<br/><sup>Boehm2014 - isoform-specific d</sup></summary><sup>Boehm2014 - isoform-specific dimerization of pSTAT5A and pSTAT5B</sup></details>|<details><summary>pass</summary>[BIOMD0000000591_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000591#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000592](https://www.ebi.ac.uk/biomodels/BIOMD0000000592)<br/><sup>Martinez-Sanchez2015 - T CD4+ </sup></summary><sup>Martinez-Sanchez2015 - T CD4+ lymphocyte transcriptional regulatory network</sup></details>|<details><summary>pass</summary>[BIOMD0000000592_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000592#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000593](https://www.ebi.ac.uk/biomodels/BIOMD0000000593)<br/><sup>Martinez-Sanchez2015 - T CD4+ </sup></summary><sup>Martinez-Sanchez2015 - T CD4+ lymphocyte transcriptional-signaling regulatory network</sup></details>|<details><summary>pass</summary>[BIOMD0000000593_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000593#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000594](https://www.ebi.ac.uk/biomodels/BIOMD0000000594)<br/><sup>Capuani2015 - Binding of Cbl a</sup></summary><sup>Capuani2015 - Binding of Cbl and Gbr2 to EGFR (Multisite Phosphorylation Model - MPM)</sup></details>|<details><summary>pass</summary>[BIOMD0000000594_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000594#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000595](https://www.ebi.ac.uk/biomodels/BIOMD0000000595)<br/><sup>Capuani2015 - Binding of Cbl a</sup></summary><sup>Capuani2015 - Binding of Cbl and Grb2 to EGFR (Early Activation Model - EAM)</sup></details>|<details><summary>pass</summary>[BIOMD0000000595_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000595#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000596](https://www.ebi.ac.uk/biomodels/BIOMD0000000596)<br/><sup>Philipson2015 - Innate immune </sup></summary><sup>Philipson2015 - Innate immune response modulated by NLRX1</sup></details>|<details><summary>FAIL</summary>[Philipson2015.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000596#Files)</details>|FAIL|<details><summary>pass</summary>[Philipson2015.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000596#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Philipson2015.sedml705798.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/Philipson2015.sedml705798.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Philipson2015.sedml` is invalid.\n - Model `model` is invalid.\n - The model file `Philipson2015.xml` is invalid.\n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout' cannot be used in this document.\n \n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout_L2' cannot be used in this document.\n \n - General SBML conformance (10102) at line 3, column 2: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L2V4 Section 4.1\n Element 'listOfLayouts' is not part of the definition of SBML Level 2 Version 4.\n \n - Data generator `CytoE_1_task1` is invalid.\n - Variable `p1_CytoE_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CytoE']` are not defined. No namespaces are defined for the target.\n - Data generator `CytoL_2_task1` is invalid.\n - Variable `p1_CytoL_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CytoL']` are not defined. No namespaces are defined for the target.\n - Data generator `NFkB_3_task1` is invalid.\n - Variable `p1_NFkB_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='NFkB']` are not defined. No namespaces are defined for the target.\n - Data generator `TRAF6_4_task1` is invalid.\n - Variable `p1_TRAF6_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='TRAF6']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Philipson2015.sedml281024.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Philipson2015.sedml281024.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Philipson2015.sedml` is invalid.\n - Model `model` is invalid.\n - The model file `Philipson2015.xml` is invalid.\n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout' cannot be used in this document.\n \n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout_L2' cannot be used in this document.\n \n - General SBML conformance (10102) at line 3, column 2: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L2V4 Section 4.1\n Element 'listOfLayouts' is not part of the definition of SBML Level 2 Version 4.\n \n - Data generator `CytoE_1_task1` is invalid.\n - Variable `p1_CytoE_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CytoE']` are not defined. No namespaces are defined for the target.\n - Data generator `CytoL_2_task1` is invalid.\n - Variable `p1_CytoL_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CytoL']` are not defined. No namespaces are defined for the target.\n - Data generator `NFkB_3_task1` is invalid.\n - Variable `p1_NFkB_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='NFkB']` are not defined. No namespaces are defined for the target.\n - Data generator `TRAF6_4_task1` is invalid.\n - Variable `p1_TRAF6_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='TRAF6']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000597](https://www.ebi.ac.uk/biomodels/BIOMD0000000597)<br/><sup>Flis2015 - Plant clock gene ci</sup></summary><sup>Flis2015 - Plant clock gene circuit (P2011.1.2 PLM_71 ver 1)</sup></details>|<details><summary>pass</summary>[BIOMD0000000597_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000597#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000598](https://www.ebi.ac.uk/biomodels/BIOMD0000000598)<br/><sup>Flis2015 - Plant clock gene ci</sup></summary><sup>Flis2015 - Plant clock gene circuit (P2011.2.1 PLM_71 ver 2)</sup></details>|<details><summary>pass</summary>[BIOMD0000000598_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000598#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000599](https://www.ebi.ac.uk/biomodels/BIOMD0000000599)<br/><sup>Coggins2014 - CXCL12 dependent</sup></summary><sup>Coggins2014 - CXCL12 dependent recruitment of beta arrestin</sup></details>|<details><summary>pass</summary>[BIOMD0000000599_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000599#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000600](https://www.ebi.ac.uk/biomodels/BIOMD0000000600)<br/><sup>Cellire2011 - Plasticity of T</sup></summary><sup>Cellire2011 - Plasticity of TGF- Signalling</sup></details>|<details><summary>pass</summary>[BIOMD0000000600_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000600#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000601](https://www.ebi.ac.uk/biomodels/BIOMD0000000601)<br/><sup>Rosas2015 - Caffeine-induced l</sup></summary><sup>Rosas2015 - Caffeine-induced luminal SR calcium changes</sup></details>|<details><summary>pass</summary>[BIOMD0000000601_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000601#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000602](https://www.ebi.ac.uk/biomodels/BIOMD0000000602)<br/><sup>Stavrum2013 - Tryptophan Metab</sup></summary><sup>Stavrum2013 - Tryptophan Metabolism in Liver</sup></details>|<details><summary>pass</summary>[BIOMD0000000602_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000602#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000603](https://www.ebi.ac.uk/biomodels/BIOMD0000000603)<br/><sup>PetelenzKuehn_osmoadaptation_WT</sup>|<details><summary>pass</summary>[BIOMD0000000603_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000603#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000604](https://www.ebi.ac.uk/biomodels/BIOMD0000000604)<br/><sup>PetelenzKuehn_osmoadaptation_p</sup></summary><sup>PetelenzKuehn_osmoadaptation_pfk2627D</sup></details>|<details><summary>pass</summary>[BIOMD0000000604_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000604#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000605](https://www.ebi.ac.uk/biomodels/BIOMD0000000605)<br/><sup>PetelenzKuehn_osmoadaptation_HOG1att</sup>|<details><summary>pass</summary>[BIOMD0000000605_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000605#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000606](https://www.ebi.ac.uk/biomodels/BIOMD0000000606)<br/><sup>PetelenzKuehn_osmoadaptation_hog1D</sup>|<details><summary>pass</summary>[BIOMD0000000606_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000606#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000607](https://www.ebi.ac.uk/biomodels/BIOMD0000000607)<br/><sup>PetelenzKuehn_osmoadaptation_fps1D1</sup>|<details><summary>pass</summary>[BIOMD0000000607_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000607#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000608](https://www.ebi.ac.uk/biomodels/BIOMD0000000608)<br/><sup>Palsson2013 - Fully-integrated</sup></summary><sup>Palsson2013 - Fully-integrated immune response model (FIRM)</sup></details>|<details><summary>pass</summary>[BIOMD0000000608_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000608#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000609](https://www.ebi.ac.uk/biomodels/BIOMD0000000609)<br/><sup>Reddyhoff2015 - Acetaminophen </sup></summary><sup>Reddyhoff2015 - Acetaminophen metabolism and toxicity</sup></details>|<details><summary>pass</summary>[BIOMD0000000609_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000609#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000610](https://www.ebi.ac.uk/biomodels/BIOMD0000000610)<br/><sup>PetelenzKuehn_osmoadaptation_gpd1D</sup>|<details><summary>pass</summary>[BIOMD0000000610_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000610#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000611](https://www.ebi.ac.uk/biomodels/BIOMD0000000611)<br/><sup>Nayak2015 - Blood Coagulation </sup></summary><sup>Nayak2015 - Blood Coagulation Network - Predicting the Effects of Various Therapies on Biomarkers</sup></details>|<details><summary>pass</summary>[BIOMD0000000611_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000611#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000612](https://www.ebi.ac.uk/biomodels/BIOMD0000000612)<br/><sup>Proctor2016 - Circadian rhythm</sup></summary><sup>Proctor2016 - Circadian rhythm of PTH and the dynamics of signaling molecules on bone remodeling</sup></details>|<details><summary>pass</summary>[BIOMD0000000612_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000612#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000613](https://www.ebi.ac.uk/biomodels/BIOMD0000000613)<br/><sup>Peterson2010 - Integrated calc</sup></summary><sup>Peterson2010 - Integrated calcium homeostasis and bone remodelling</sup></details>|<details><summary>pass</summary>[BIOMD0000000613_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000613#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000614](https://www.ebi.ac.uk/biomodels/BIOMD0000000614)<br/><sup>Kamihira2000 - calcitonin fibr</sup></summary><sup>Kamihira2000 - calcitonin fibrillation kinetics</sup></details>|<details><summary>pass</summary>[BIOMD0000000614_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000614#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000615](https://www.ebi.ac.uk/biomodels/BIOMD0000000615)<br/><sup>Kuznetsov2016(II) - -syn aggr</sup></summary><sup>Kuznetsov2016(II) - -syn aggregation kinetics in Parkinson's Disease</sup></details>|<details><summary>pass</summary>[BIOMD0000000615_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000615#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000616](https://www.ebi.ac.uk/biomodels/BIOMD0000000616)<br/><sup>Dunster2014 - WBC Interactions</sup></summary><sup>Dunster2014 - WBC Interactions (Model1)</sup></details>|<details><summary>pass</summary>[BIOMD0000000616_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000616#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000617](https://www.ebi.ac.uk/biomodels/BIOMD0000000617)<br/><sup>Walsh2014 - Inhibition kinetic</sup></summary><sup>Walsh2014 - Inhibition kinetics of DAPT on APP Cleavage</sup></details>|<details><summary>pass</summary>[BIOMD0000000617_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000617#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000618](https://www.ebi.ac.uk/biomodels/BIOMD0000000618)<br/><sup>Krohn2011 - Cerebral amyloid-</sup></summary><sup>Krohn2011 - Cerebral amyloid- proteostasis regulated by membrane transport protein ABCC1</sup></details>|<details><summary>pass</summary>[BIOMD0000000618_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000618#Files)</details>|pass|<details><summary>FAIL</summary>[BIOMD0000000618_SED-ML.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000618#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/BIOMD0000000618_SED-ML.xml648655.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/BIOMD0000000618_SED-ML.xml648655.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./BIOMD0000000618_SED-ML.xml` is invalid.\n - A <model> object may contain one and only one instance of the <listOfChanges> element. No other elements from the SBML Level 3 SED-ML namespaces are permitted on a <model> object. \n Reference: L3V1 Sedml V1 Section\n \n'```</details>|<details><summary>other</summary>```Command '-i /root/in/BIOMD0000000618_SED-ML.xml142580.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/BIOMD0000000618_SED-ML.xml142580.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./BIOMD0000000618_SED-ML.xml` is invalid.\n - A <model> object may contain one and only one instance of the <listOfChanges> element. No other elements from the SBML Level 3 SED-ML namespaces are permitted on a <model> object. \n Reference: L3V1 Sedml V1 Section\n \n'```</details>|
|[BIOMD0000000619](https://www.ebi.ac.uk/biomodels/BIOMD0000000619)<br/><sup>Sluka2016  - Acetaminophen PBPK</sup>|<details><summary>pass</summary>[BIOMD0000000619_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000619#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000620](https://www.ebi.ac.uk/biomodels/BIOMD0000000620)<br/><sup>Palmer2014 - Effect of IL-1-B</sup></summary><sup>Palmer2014 - Effect of IL-1-Blocking therapies in T2DM - Disease Condition</sup></details>|<details><summary>pass</summary>[BIOMD0000000620_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000620#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000621](https://www.ebi.ac.uk/biomodels/BIOMD0000000621)<br/><sup>Palmer2014 - Effect of IL-1-B</sup></summary><sup>Palmer2014 - Effect of IL-1-Blocking therapies in T2DM - Healthy Condition</sup></details>|<details><summary>pass</summary>[BIOMD0000000621_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000621#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000622](https://www.ebi.ac.uk/biomodels/BIOMD0000000622)<br/><sup>NguyenLK2011 - Ubiquitination </sup></summary><sup>NguyenLK2011 - Ubiquitination dynamics in Ring1B/Bmi1 system</sup></details>|<details><summary>pass</summary>[BIOMD0000000622_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000622#Files)</details>|pass|<details><summary>MultipleSEDMLs</summary>['NguyenLK2011_High_Free_Bmi1_Levels.sedml', 'NguyenLK2011_High_Free_Bmi1ubd_Levels.sedml']</details>|NA|NA|NA|
|<details><summary>[BIOMD0000000623](https://www.ebi.ac.uk/biomodels/BIOMD0000000623)<br/><sup>Orton2009 - Modelling cancerou</sup></summary><sup>Orton2009 - Modelling cancerous mutations in the EGFR/ERK pathway - EGF Model</sup></details>|<details><summary>pass</summary>[BIOMD0000000623_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000623#Files)</details>|pass|<details><summary>pass</summary>[Orton2009.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000623#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Orton2009.sedml73314.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000623_url.xml` has warnings.\r\n - 31 warnings of type SBML unit consistency (99505). The following is the first warning at line 2085, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment_0 * (k1 * species_25 * species_1 - k2 * species_0)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 54 warnings of type Modeling practice (80701). The following is the first warning at line 2107, column 12:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Orton2009.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000623_url.xml` has warnings.\r\n - 31 warnings of type SBML unit consistency (99505). The following is the first warning at line 2085, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment_0 * (k1 * species_25 * species_1 - k2 * species_0)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 54 warnings of type Modeling practice (80701). The following is the first warning at line 2107, column 12:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Orton2009.sedml73314.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Orton2009.sedml` is invalid.\n - Data generator `species_0_1` is invalid.\n - Variable `p1_species_0` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_0']` are not defined. No namespaces are defined for the target.\n - Data generator `species_1_2` is invalid.\n - Variable `p1_species_1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_1']` are not defined. No namespaces are defined for the target.\n - Data generator `species_2_3` is invalid.\n - Variable `p1_species_2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_2']` are not defined. No namespaces are defined for the target.\n - Data generator `species_3_4` is invalid.\n - Variable `p1_species_3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_3']` are not defined. No namespaces are defined for the target.\n - Data generator `species_4_5` is invalid.\n - Variable `p1_species_4` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_4']` are not defined. No namespaces are defined for the target.\n - Data generator `species_5_6` is invalid.\n - Variable `p1_species_5` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_5']` are not defined. No namespaces are defined for the target.\n - Data generator `species_6_7` is invalid.\n - Variable `p1_species_6` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_6']` are not defined. No namespaces are defined for the target.\n - Data generator `species_7_8` is invalid.\n - Variable `p1_species_7` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_7']` are not defined. No namespaces are defined for the target.\n - Data generator `species_8_9` is invalid.\n - Variable `p1_species_8` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_8']` are not defined. No namespaces are defined for the target.\n - Data generator `species_9_10` is invalid.\n - Variable `p1_species_9` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_9']` are not defined. No namespaces are defined for the target.\n - Data generator `species_10_11` is invalid.\n - Variable `p1_species_10` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_10']` are not defined. No namespaces are defined for the target.\n - Data generator `species_11_12` is invalid.\n - Variable `p1_species_11` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_11']` are not defined. No namespaces are defined for the target.\n - Data generator `species_12_13` is invalid.\n - Variable `p1_species_12` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_12']` are not defined. No namespaces are defined for the target.\n - Data generator `species_13_14` is invalid.\n - Variable `p1_species_13` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_13']` are not defined. No namespaces are defined for the target.\n - Data generator `species_14_15` is invalid.\n - Variable `p1_species_14` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_14']` are not defined. No namespaces are defined for the target.\n - Data generator `species_15_16` is invalid.\n - Variable `p1_species_15` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_15']` are not defined. No namespaces are defined for the target.\n - Data generator `species_16_17` is invalid.\n - Variable `p1_species_16` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_16']` are not defined. No namespaces are defined for the target.\n - Data generator `species_17_18` is invalid.\n - Variable `p1_species_17` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_17']` are not defined. No namespaces are defined for the target.\n - Data generator `species_18_19` is invalid.\n - Variable `p1_species_18` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_18']` are not defined. No namespaces are defined for the target.\n - Data generator `species_19_20` is invalid.\n - Variable `p1_species_19` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_19']` are not defined. No namespaces are defined for the target.\n - Data generator `species_20_21` is invalid.\n - Variable `p1_species_20` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_20']` are not defined. No namespaces are defined for the target.\n - Data generator `species_21_22` is invalid.\n - Variable `p1_species_21` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_21']` are not defined. No namespaces are defined for the target.\n - Data generator `species_22_23` is invalid.\n - Variable `p1_species_22` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_22']` are not defined. No namespaces are defined for the target.\n - Data generator `species_23_24` is invalid.\n - Variable `p1_species_23` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_23']` are not defined. No namespaces are defined for the target.\n - Data generator `species_24_25` is invalid.\n - Variable `p1_species_24` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_24']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Orton2009.sedml771483.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000623_url.xml` has warnings.\r\n - 31 warnings of type SBML unit consistency (99505). The following is the first warning at line 2085, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment_0 * (k1 * species_25 * species_1 - k2 * species_0)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 54 warnings of type Modeling practice (80701). The following is the first warning at line 2107, column 12:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Orton2009.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000623_url.xml` has warnings.\r\n - 31 warnings of type SBML unit consistency (99505). The following is the first warning at line 2085, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment_0 * (k1 * species_25 * species_1 - k2 * species_0)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 54 warnings of type Modeling practice (80701). The following is the first warning at line 2107, column 12:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Orton2009.sedml771483.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Orton2009.sedml` is invalid.\n - Data generator `species_0_1` is invalid.\n - Variable `p1_species_0` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_0']` are not defined. No namespaces are defined for the target.\n - Data generator `species_1_2` is invalid.\n - Variable `p1_species_1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_1']` are not defined. No namespaces are defined for the target.\n - Data generator `species_2_3` is invalid.\n - Variable `p1_species_2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_2']` are not defined. No namespaces are defined for the target.\n - Data generator `species_3_4` is invalid.\n - Variable `p1_species_3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_3']` are not defined. No namespaces are defined for the target.\n - Data generator `species_4_5` is invalid.\n - Variable `p1_species_4` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_4']` are not defined. No namespaces are defined for the target.\n - Data generator `species_5_6` is invalid.\n - Variable `p1_species_5` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_5']` are not defined. No namespaces are defined for the target.\n - Data generator `species_6_7` is invalid.\n - Variable `p1_species_6` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_6']` are not defined. No namespaces are defined for the target.\n - Data generator `species_7_8` is invalid.\n - Variable `p1_species_7` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_7']` are not defined. No namespaces are defined for the target.\n - Data generator `species_8_9` is invalid.\n - Variable `p1_species_8` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_8']` are not defined. No namespaces are defined for the target.\n - Data generator `species_9_10` is invalid.\n - Variable `p1_species_9` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_9']` are not defined. No namespaces are defined for the target.\n - Data generator `species_10_11` is invalid.\n - Variable `p1_species_10` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_10']` are not defined. No namespaces are defined for the target.\n - Data generator `species_11_12` is invalid.\n - Variable `p1_species_11` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_11']` are not defined. No namespaces are defined for the target.\n - Data generator `species_12_13` is invalid.\n - Variable `p1_species_12` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_12']` are not defined. No namespaces are defined for the target.\n - Data generator `species_13_14` is invalid.\n - Variable `p1_species_13` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_13']` are not defined. No namespaces are defined for the target.\n - Data generator `species_14_15` is invalid.\n - Variable `p1_species_14` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_14']` are not defined. No namespaces are defined for the target.\n - Data generator `species_15_16` is invalid.\n - Variable `p1_species_15` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_15']` are not defined. No namespaces are defined for the target.\n - Data generator `species_16_17` is invalid.\n - Variable `p1_species_16` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_16']` are not defined. No namespaces are defined for the target.\n - Data generator `species_17_18` is invalid.\n - Variable `p1_species_17` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_17']` are not defined. No namespaces are defined for the target.\n - Data generator `species_18_19` is invalid.\n - Variable `p1_species_18` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_18']` are not defined. No namespaces are defined for the target.\n - Data generator `species_19_20` is invalid.\n - Variable `p1_species_19` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_19']` are not defined. No namespaces are defined for the target.\n - Data generator `species_20_21` is invalid.\n - Variable `p1_species_20` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_20']` are not defined. No namespaces are defined for the target.\n - Data generator `species_21_22` is invalid.\n - Variable `p1_species_21` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_21']` are not defined. No namespaces are defined for the target.\n - Data generator `species_22_23` is invalid.\n - Variable `p1_species_22` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_22']` are not defined. No namespaces are defined for the target.\n - Data generator `species_23_24` is invalid.\n - Variable `p1_species_23` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_23']` are not defined. No namespaces are defined for the target.\n - Data generator `species_24_25` is invalid.\n - Variable `p1_species_24` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_24']` are not defined. No namespaces are defined for the target.\n"```</details>|
|[BIOMD0000000624](https://www.ebi.ac.uk/biomodels/BIOMD0000000624)<br/><sup>Sluka2016 - Acetaminophen metabolism</sup>|<details><summary>pass</summary>[BIOMD0000000624_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000624#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000625](https://www.ebi.ac.uk/biomodels/BIOMD0000000625)<br/><sup>Leber2016 - Expanded model of </sup></summary><sup>Leber2016 - Expanded model of Tfh-Tfr differentiation - Helicobacter pylori infection</sup></details>|<details><summary>pass</summary>[BIOMD0000000625_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000625#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000626](https://www.ebi.ac.uk/biomodels/BIOMD0000000626)<br/><sup>Ray2013 - Meiotic initiation i</sup></summary><sup>Ray2013 - Meiotic initiation in S. cerevisiae</sup></details>|<details><summary>pass</summary>[BIOMD0000000626_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000626#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000627](https://www.ebi.ac.uk/biomodels/BIOMD0000000627)<br/><sup>Winter2017 - Brain Energy Meta</sup></summary><sup>Winter2017 - Brain Energy Metabolism with PPP</sup></details>|<details><summary>pass</summary>[BIOMD0000000627_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000627#Files)</details>|pass|<details><summary>FAIL</summary>[BIOMD0000000627_SED-ML.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000627#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/BIOMD0000000627_SED-ML.xml492187.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `MODEL1603240000` has warnings.\r\n - Model resolution via URNs will likely be deprecated in a future version of SED-ML. Instead models can be resolved by URLs.\r\n - URN model source `urn:miriam:biomodels.db:BIOMD0000000627.xml` was not validated.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./BIOMD0000000627_SED-ML.xml` has warnings.\r\n - Model `MODEL1603240000` has warnings.\r\n - Model resolution via URNs will likely be deprecated in a future version of SED-ML. Instead models can be resolved by URLs.\r\n - URN model source `urn:miriam:biomodels.db:BIOMD0000000627.xml` was not validated.\x1b[0m\r\n\r\n`/root/in/BIOMD0000000627_SED-ML.xml492187.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./BIOMD0000000627_SED-ML.xml` is invalid.\n - Data generator `BOLD_signal_1` is invalid.\n - Variable `p1_BOLD_signal` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='BOLD_signal']` are not defined. No namespaces are defined for the target.\n - Data generator `vSTIM_1` is invalid.\n - Variable `p2_vSTIM` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='vSTIM']` are not defined. No namespaces are defined for the target.\n - Data generator `venous_balloon_1` is invalid.\n - Variable `p3_venous_balloon` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfCompartments/sbml:compartment[@id='venous_balloon']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/BIOMD0000000627_SED-ML.xml751379.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `MODEL1603240000` has warnings.\r\n - Model resolution via URNs will likely be deprecated in a future version of SED-ML. Instead models can be resolved by URLs.\r\n - URN model source `urn:miriam:biomodels.db:BIOMD0000000627.xml` was not validated.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./BIOMD0000000627_SED-ML.xml` has warnings.\r\n - Model `MODEL1603240000` has warnings.\r\n - Model resolution via URNs will likely be deprecated in a future version of SED-ML. Instead models can be resolved by URLs.\r\n - URN model source `urn:miriam:biomodels.db:BIOMD0000000627.xml` was not validated.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/BIOMD0000000627_SED-ML.xml751379.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./BIOMD0000000627_SED-ML.xml` is invalid.\n - Data generator `BOLD_signal_1` is invalid.\n - Variable `p1_BOLD_signal` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='BOLD_signal']` are not defined. No namespaces are defined for the target.\n - Data generator `vSTIM_1` is invalid.\n - Variable `p2_vSTIM` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='vSTIM']` are not defined. No namespaces are defined for the target.\n - Data generator `venous_balloon_1` is invalid.\n - Variable `p3_venous_balloon` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfCompartments/sbml:compartment[@id='venous_balloon']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000628](https://www.ebi.ac.uk/biomodels/BIOMD0000000628)<br/><sup>Li2012 Calcium mediated synapt</sup></summary><sup>Li2012 Calcium mediated synaptic plasticity</sup></details>|<details><summary>pass</summary>[BIOMD0000000628_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000628#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000629](https://www.ebi.ac.uk/biomodels/BIOMD0000000629)<br/><sup>Haffez2017 - RAR interaction w</sup></summary><sup>Haffez2017 - RAR interaction with synthetic analogues</sup></details>|<details><summary>pass</summary>[BIOMD0000000629_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000629#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000630](https://www.ebi.ac.uk/biomodels/BIOMD0000000630)<br/><sup>Venkatraman2011 - PLS-UPA beha</sup></summary><sup>Venkatraman2011 - PLS-UPA behaviour in the presence of substrate competition_1_1_1_1</sup></details>|<details><summary>pass</summary>[BIOMD0000000630_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000630#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000631](https://www.ebi.ac.uk/biomodels/BIOMD0000000631)<br/><sup>DeCaluwe2016 - Circadian Clock</sup>|<details><summary>FAIL</summary>[BIOMD0000000631_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000631#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000632](https://www.ebi.ac.uk/biomodels/BIOMD0000000632)<br/><sup>Kollarovic2016 - Cell fate dec</sup></summary><sup>Kollarovic2016 - Cell fate decision at G1-S transition</sup></details>|<details><summary>pass</summary>[BIOMD0000000632_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000632#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000633](https://www.ebi.ac.uk/biomodels/BIOMD0000000633)<br/><sup>Bulik2016 - Regulation of hepa</sup></summary><sup>Bulik2016 - Regulation of hepatic glucose metabolism</sup></details>|<details><summary>pass</summary>[BIOMD0000000633_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000633#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000634](https://www.ebi.ac.uk/biomodels/BIOMD0000000634)<br/><sup>Proctor2013 - Effect of A imm</sup></summary><sup>Proctor2013 - Effect of A immunisation in Alzheimer's disease (stochastic version)</sup></details>|<details><summary>pass</summary>[BIOMD0000000634_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000634#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000635](https://www.ebi.ac.uk/biomodels/BIOMD0000000635)<br/><sup>Nair2015 - Interaction between</sup></summary><sup>Nair2015 - Interaction between neuromodulators via GPCRs - Effect on cAMP/PKA signaling (D1 Neuron)</sup></details>|<details><summary>pass</summary>[BIOMD0000000635_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000635#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000636](https://www.ebi.ac.uk/biomodels/BIOMD0000000636)<br/><sup>Nair2015 - Interaction between</sup></summary><sup>Nair2015 - Interaction between neuromodulators via GPCRs - Effect on cAMP/PKA signaling (D2 Neuron)</sup></details>|<details><summary>pass</summary>[BIOMD0000000636_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000636#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000637](https://www.ebi.ac.uk/biomodels/BIOMD0000000637)<br/><sup>Bush2016 - Simplified Carrouse</sup></summary><sup>Bush2016 - Simplified Carrousel model of GPCR</sup></details>|<details><summary>pass</summary>[BIOMD0000000637_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000637#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000638](https://www.ebi.ac.uk/biomodels/BIOMD0000000638)<br/><sup>Bush2016 - Extended Carrousel </sup></summary><sup>Bush2016 - Extended Carrousel model of GPCR-RGS</sup></details>|<details><summary>pass</summary>[BIOMD0000000638_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000638#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000639](https://www.ebi.ac.uk/biomodels/BIOMD0000000639)<br/><sup>Thiaville2016 - Wild type fola</sup></summary><sup>Thiaville2016 - Wild type folate pathway model with proposed PanB reaction</sup></details>|<details><summary>pass</summary>[BIOMD0000000639_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000639#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000640](https://www.ebi.ac.uk/biomodels/BIOMD0000000640)<br/><sup>DallePezze2016 - Activation of</sup></summary><sup>DallePezze2016 - Activation of AMPK and mTOR by amino acids</sup></details>|<details><summary>pass</summary>[BIOMD0000000640_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000640#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000641](https://www.ebi.ac.uk/biomodels/BIOMD0000000641)<br/><sup>Jaiswal2017 - Cell cycle arrest</sup>|<details><summary>pass</summary>[BIOMD0000000641_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000641#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000642](https://www.ebi.ac.uk/biomodels/BIOMD0000000642)<br/><sup>Mufudza2012 - Estrogen effect </sup></summary><sup>Mufudza2012 - Estrogen effect on the dynamics of breast cancer</sup></details>|<details><summary>pass</summary>[BIOMD0000000642_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000642#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000643](https://www.ebi.ac.uk/biomodels/BIOMD0000000643)<br/><sup>Musante2017 - Switching behavi</sup></summary><sup>Musante2017 - Switching behaviour of PP2A inhibition by ARPP-16 - mutual inhibitions</sup></details>|<details><summary>pass</summary>[BIOMD0000000643_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000643#Files)</details>|pass|<details><summary>FAIL</summary>[ARPP-16_Layer1_mutualInhibitions.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000643#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/ARPP-16_Layer1_mutualInhibitions.sedml343908.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/ARPP-16_Layer1_mutualInhibitions.sedml343908.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./ARPP-16_Layer1_mutualInhibitions.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - XDataReference from Curve `p1_curve_1` cannot be resolved to a data generator with id `cAMP_1` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_1` cannot be resolved to a data generator with id `A46bar_1` because multiple data generators have this id.\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/ARPP-16_Layer1_mutualInhibitions.sedml426551.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/ARPP-16_Layer1_mutualInhibitions.sedml426551.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./ARPP-16_Layer1_mutualInhibitions.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - XDataReference from Curve `p1_curve_1` cannot be resolved to a data generator with id `cAMP_1` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_1` cannot be resolved to a data generator with id `A46bar_1` because multiple data generators have this id.\n'```</details>|
|<details><summary>[BIOMD0000000644](https://www.ebi.ac.uk/biomodels/BIOMD0000000644)<br/><sup>Musante2017 - Switching behavi</sup></summary><sup>Musante2017 - Switching behaviour of PP2A inhibition by ARPP-16 - mutual inhibitions and PKA inhibits MAST3</sup></details>|<details><summary>pass</summary>[BIOMD0000000644_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000644#Files)</details>|pass|<details><summary>FAIL</summary>[ARPP-16_Layer1and2_mutualInhibitions_PKAinhibitsMAST3.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000644#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/ARPP-16_Layer1and2_mutualInhibitions_PKAinhibitsMAST3.sedml211739.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/ARPP-16_Layer1and2_mutualInhibitions_PKAinhibitsMAST3.sedml211739.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./ARPP-16_Layer1and2_mutualInhibitions_PKAinhibitsMAST3.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - XDataReference from Curve `p1_curve_1` cannot be resolved to a data generator with id `time` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_1` cannot be resolved to a data generator with id `ARPPtot_1` because multiple data generators have this id.\n - XDataReference from Curve `p1_curve_2` cannot be resolved to a data generator with id `time` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_2` cannot be resolved to a data generator with id `A46_2` because multiple data generators have this id.\n - XDataReference from Curve `p1_curve_3` cannot be resolved to a data generator with id `time` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_3` cannot be resolved to a data generator with id `A88_3` because multiple data generators have this id.\n - XDataReference from Curve `p1_curve_4` cannot be resolved to a data generator with id `time` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_4` cannot be resolved to a data generator with id `M_4` because multiple data generators have this id.\n - XDataReference from Curve `p1_curve_5` cannot be resolved to a data generator with id `time` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_5` cannot be resolved to a data generator with id `PKA_5` because multiple data generators have this id.\n - XDataReference from Curve `p2_curve_1` cannot be resolved to a data generator with id `cAMP_1` because multiple data generators have this id.\n - YDataReference from Curve `p2_curve_1` cannot be resolved to a data generator with id `A46bar_1` because multiple data generators have this id.\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/ARPP-16_Layer1and2_mutualInhibitions_PKAinhibitsMAST3.sedml35976.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/ARPP-16_Layer1and2_mutualInhibitions_PKAinhibitsMAST3.sedml35976.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./ARPP-16_Layer1and2_mutualInhibitions_PKAinhibitsMAST3.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - XDataReference from Curve `p1_curve_1` cannot be resolved to a data generator with id `time` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_1` cannot be resolved to a data generator with id `ARPPtot_1` because multiple data generators have this id.\n - XDataReference from Curve `p1_curve_2` cannot be resolved to a data generator with id `time` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_2` cannot be resolved to a data generator with id `A46_2` because multiple data generators have this id.\n - XDataReference from Curve `p1_curve_3` cannot be resolved to a data generator with id `time` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_3` cannot be resolved to a data generator with id `A88_3` because multiple data generators have this id.\n - XDataReference from Curve `p1_curve_4` cannot be resolved to a data generator with id `time` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_4` cannot be resolved to a data generator with id `M_4` because multiple data generators have this id.\n - XDataReference from Curve `p1_curve_5` cannot be resolved to a data generator with id `time` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_5` cannot be resolved to a data generator with id `PKA_5` because multiple data generators have this id.\n - XDataReference from Curve `p2_curve_1` cannot be resolved to a data generator with id `cAMP_1` because multiple data generators have this id.\n - YDataReference from Curve `p2_curve_1` cannot be resolved to a data generator with id `A46bar_1` because multiple data generators have this id.\n'```</details>|
|<details><summary>[BIOMD0000000645](https://www.ebi.ac.uk/biomodels/BIOMD0000000645)<br/><sup>Musante2017 - Switching behavi</sup></summary><sup>Musante2017 - Switching behaviour of PP2A inhibition by ARPP-16 - mutual inhibitions and PKA inhibits MAST3 and dominant negative effect</sup></details>|<details><summary>pass</summary>[BIOMD0000000645_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000645#Files)</details>|pass|<details><summary>FAIL</summary>[ARPP-16_Layer1and2and3_mutualInhibitions_PKAinhibitsMAST3_dominantNegative.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000645#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/ARPP-16_Layer1and2and3_mutualInhibitions_PKAinhibitsMAST3_dominantNegative.sedml518492.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/ARPP-16_Layer1and2and3_mutualInhibitions_PKAinhibitsMAST3_dominantNegative.sedml518492.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./ARPP-16_Layer1and2and3_mutualInhibitions_PKAinhibitsMAST3_dominantNegative.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - XDataReference from Curve `p1_curve_1` cannot be resolved to a data generator with id `cAMP_1` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_1` cannot be resolved to a data generator with id `A46bar_1` because multiple data generators have this id.\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/ARPP-16_Layer1and2and3_mutualInhibitions_PKAinhibitsMAST3_dominantNegative.sedml548696.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/ARPP-16_Layer1and2and3_mutualInhibitions_PKAinhibitsMAST3_dominantNegative.sedml548696.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./ARPP-16_Layer1and2and3_mutualInhibitions_PKAinhibitsMAST3_dominantNegative.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - XDataReference from Curve `p1_curve_1` cannot be resolved to a data generator with id `cAMP_1` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_1` cannot be resolved to a data generator with id `A46bar_1` because multiple data generators have this id.\n'```</details>|
|<details><summary>[BIOMD0000000646](https://www.ebi.ac.uk/biomodels/BIOMD0000000646)<br/><sup>Barr2016 - All-or-nothing G1/S</sup></summary><sup>Barr2016 - All-or-nothing G1/S transition</sup></details>|<details><summary>pass</summary>[BIOMD0000000646_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000646#Files)</details>|pass|<details><summary>FAIL</summary>[Barr2016.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000646#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Barr2016.sedml53238.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/Barr2016.sedml53238.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Barr2016.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - YDataReference from Curve `p1_curve_2_task1` cannot be resolved to a data generator with id `CycA_2_task1` because multiple data generators have this id.\n - YDataReference from Curve `p2_curve_2_task1` cannot be resolved to a data generator with id `CycA_2_task1` because multiple data generators have this id.\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Barr2016.sedml167088.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Barr2016.sedml167088.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Barr2016.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - YDataReference from Curve `p1_curve_2_task1` cannot be resolved to a data generator with id `CycA_2_task1` because multiple data generators have this id.\n - YDataReference from Curve `p2_curve_2_task1` cannot be resolved to a data generator with id `CycA_2_task1` because multiple data generators have this id.\n'```</details>|
|<details><summary>[BIOMD0000000647](https://www.ebi.ac.uk/biomodels/BIOMD0000000647)<br/><sup>Kwang2003 - The influence of R</sup></summary><sup>Kwang2003 - The influence of RKIP on the ERK signaling pathway</sup></details>|<details><summary>pass</summary>[BIOMD0000000647_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000647#Files)</details>|pass|<details><summary>FAIL</summary>[BIOMD0000000647.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000647#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/BIOMD0000000647.sedml58913.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/BIOMD0000000647.sedml58913.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./BIOMD0000000647.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmp_r3tps6r/./model1.xml` is not a file.\n - Data generator `RKIP_1_task1` is invalid.\n - Variable `p1_RKIP_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='RKIP']` are not defined. No namespaces are defined for the target.\n - Data generator `Raf1_2_task1` is invalid.\n - Variable `p1_Raf1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Raf1']` are not defined. No namespaces are defined for the target.\n - Data generator `Raf1_RKIP_3_task1` is invalid.\n - Variable `p1_Raf1_RKIP_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Raf1_RKIP']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/BIOMD0000000647.sedml225673.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/BIOMD0000000647.sedml225673.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./BIOMD0000000647.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmpekx_8lte/./model1.xml` is not a file.\n - Data generator `RKIP_1_task1` is invalid.\n - Variable `p1_RKIP_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='RKIP']` are not defined. No namespaces are defined for the target.\n - Data generator `Raf1_2_task1` is invalid.\n - Variable `p1_Raf1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Raf1']` are not defined. No namespaces are defined for the target.\n - Data generator `Raf1_RKIP_3_task1` is invalid.\n - Variable `p1_Raf1_RKIP_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Raf1_RKIP']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000648](https://www.ebi.ac.uk/biomodels/BIOMD0000000648)<br/><sup>Padala2017- ERK, PI3K/Akt and </sup></summary><sup>Padala2017- ERK, PI3K/Akt and Wnt signalling network (normal)</sup></details>|<details><summary>pass</summary>[BIOMD0000000648_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000648#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000650](https://www.ebi.ac.uk/biomodels/BIOMD0000000650)<br/><sup>Owen1998 - Tumour  treatment model</sup>|<details><summary>pass</summary>[BIOMD0000000650_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000650#Files)</details>|FAIL|<details><summary>pass</summary>[MODEL1708250002_treatment_start10.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000650#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1708250002_treatment_start10.sedml849012.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000650_url.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 493, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <rateRule> <math> expression 'A * l * (m + F) * (N + 1) / (N + l + m + n) + I * (1 + S * (m + F)) - K_l * l * m * (m + F) - delta_l * l' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10532). The following is the first warning at line 606, column 6:\r\n - When the 'variable' in a <rateRule> definition refers to a <species>, the units of the rule's right-hand side are expected to be of the form _x per time_, where _x_ is the units of that species' quantity, and _time_ refers to the units of time for the model.\r\n Reference: L2V4 Section 4.11.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <rateRule> with variable 'n' are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0).\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 310, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'A' does not have a 'units' attribute.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1708250002_treatment_start10.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000650_url.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 493, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <rateRule> <math> expression 'A * l * (m + F) * (N + 1) / (N + l + m + n) + I * (1 + S * (m + F)) - K_l * l * m * (m + F) - delta_l * l' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10532). The following is the first warning at line 606, column 6:\r\n - When the 'variable' in a <rateRule> definition refers to a <species>, the units of the rule's right-hand side are expected to be of the form _x per time_, where _x_ is the units of that species' quantity, and _time_ refers to the units of time for the model.\r\n Reference: L2V4 Section 4.11.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <rateRule> with variable 'n' are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0).\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 310, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'A' does not have a 'units' attribute.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/MODEL1708250002_treatment_start10.sedml849012.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1708250002_treatment_start10.sedml` is invalid.\n - Task `task3` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='S']` are not defined. No namespaces are defined for the target.\n - Data generator `l_1_task1` is invalid.\n - Variable `p1_l_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='l']` are not defined. No namespaces are defined for the target.\n - Data generator `m_2_task1` is invalid.\n - Variable `p1_m_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='m']` are not defined. No namespaces are defined for the target.\n - Data generator `n_3_task1` is invalid.\n - Variable `p1_n_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='n']` are not defined. No namespaces are defined for the target.\n - Data generator `l_1_task3` is invalid.\n - Variable `p1_l_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='l']` are not defined. No namespaces are defined for the target.\n - Data generator `m_2_task3` is invalid.\n - Variable `p1_m_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='m']` are not defined. No namespaces are defined for the target.\n - Data generator `n_3_task3` is invalid.\n - Variable `p1_n_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='n']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1708250002_treatment_start10.sedml107062.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000650_url.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 493, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <rateRule> <math> expression 'A * l * (m + F) * (N + 1) / (N + l + m + n) + I * (1 + S * (m + F)) - K_l * l * m * (m + F) - delta_l * l' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10532). The following is the first warning at line 606, column 6:\r\n - When the 'variable' in a <rateRule> definition refers to a <species>, the units of the rule's right-hand side are expected to be of the form _x per time_, where _x_ is the units of that species' quantity, and _time_ refers to the units of time for the model.\r\n Reference: L2V4 Section 4.11.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <rateRule> with variable 'n' are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0).\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 310, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'A' does not have a 'units' attribute.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1708250002_treatment_start10.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000650_url.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 493, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <rateRule> <math> expression 'A * l * (m + F) * (N + 1) / (N + l + m + n) + I * (1 + S * (m + F)) - K_l * l * m * (m + F) - delta_l * l' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10532). The following is the first warning at line 606, column 6:\r\n - When the 'variable' in a <rateRule> definition refers to a <species>, the units of the rule's right-hand side are expected to be of the form _x per time_, where _x_ is the units of that species' quantity, and _time_ refers to the units of time for the model.\r\n Reference: L2V4 Section 4.11.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <rateRule> with variable 'n' are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0).\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 310, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'A' does not have a 'units' attribute.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1708250002_treatment_start10.sedml107062.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1708250002_treatment_start10.sedml` is invalid.\n - Task `task3` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='S']` are not defined. No namespaces are defined for the target.\n - Data generator `l_1_task1` is invalid.\n - Variable `p1_l_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='l']` are not defined. No namespaces are defined for the target.\n - Data generator `m_2_task1` is invalid.\n - Variable `p1_m_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='m']` are not defined. No namespaces are defined for the target.\n - Data generator `n_3_task1` is invalid.\n - Variable `p1_n_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='n']` are not defined. No namespaces are defined for the target.\n - Data generator `l_1_task3` is invalid.\n - Variable `p1_l_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='l']` are not defined. No namespaces are defined for the target.\n - Data generator `m_2_task3` is invalid.\n - Variable `p1_m_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='m']` are not defined. No namespaces are defined for the target.\n - Data generator `n_3_task3` is invalid.\n - Variable `p1_n_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='n']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000651](https://www.ebi.ac.uk/biomodels/BIOMD0000000651)<br/><sup>Nguyen2016 - Feedback regulati</sup></summary><sup>Nguyen2016 - Feedback regulation in cell signalling: Lessons for cancer therapeutics</sup></details>|<details><summary>pass</summary>[BIOMD0000000651_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000651#Files)</details>|pass|<details><summary>FAIL</summary>[BIOMD0000000651.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000651#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/BIOMD0000000651.sedml577382.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./BIOMD0000000651.sedml` has warnings.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/BIOMD0000000651.sedml577382.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./BIOMD0000000651.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmp9b5_p0k9/./model1.xml` is not a file.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='MEKI']` are not defined. No namespaces are defined for the target.\n - Data generator `aERK_1_task1` is invalid.\n - Variable `p1_aERK_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='aERK']` are not defined. No namespaces are defined for the target.\n - Data generator `aPI3K_2_task1` is invalid.\n - Variable `p1_aPI3K_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='aPI3K']` are not defined. No namespaces are defined for the target.\n - Data generator `aS6K_3_task1` is invalid.\n - Variable `p1_aS6K_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='aS6K']` are not defined. No namespaces are defined for the target.\n - Data generator `aERK_1_task2` is invalid.\n - Variable `p1_aERK_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='aERK']` are not defined. No namespaces are defined for the target.\n - Data generator `aPI3K_2_task2` is invalid.\n - Variable `p1_aPI3K_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='aPI3K']` are not defined. No namespaces are defined for the target.\n - Data generator `aS6K_3_task2` is invalid.\n - Variable `p1_aS6K_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='aS6K']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/BIOMD0000000651.sedml13075.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./BIOMD0000000651.sedml` has warnings.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/BIOMD0000000651.sedml13075.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./BIOMD0000000651.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmpkuzd46e1/./model1.xml` is not a file.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='MEKI']` are not defined. No namespaces are defined for the target.\n - Data generator `aERK_1_task1` is invalid.\n - Variable `p1_aERK_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='aERK']` are not defined. No namespaces are defined for the target.\n - Data generator `aPI3K_2_task1` is invalid.\n - Variable `p1_aPI3K_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='aPI3K']` are not defined. No namespaces are defined for the target.\n - Data generator `aS6K_3_task1` is invalid.\n - Variable `p1_aS6K_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='aS6K']` are not defined. No namespaces are defined for the target.\n - Data generator `aERK_1_task2` is invalid.\n - Variable `p1_aERK_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='aERK']` are not defined. No namespaces are defined for the target.\n - Data generator `aPI3K_2_task2` is invalid.\n - Variable `p1_aPI3K_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='aPI3K']` are not defined. No namespaces are defined for the target.\n - Data generator `aS6K_3_task2` is invalid.\n - Variable `p1_aS6K_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='aS6K']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000652](https://www.ebi.ac.uk/biomodels/BIOMD0000000652)<br/><sup>Padala2017- ERK, PI3K/Akt and </sup></summary><sup>Padala2017- ERK, PI3K/Akt and Wnt signalling network (PI3K mutated)</sup></details>|<details><summary>pass</summary>[BIOMD0000000652_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000652#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000653](https://www.ebi.ac.uk/biomodels/BIOMD0000000653)<br/><sup>Padala2017- ERK, PI3K/Akt and </sup></summary><sup>Padala2017- ERK, PI3K/Akt and Wnt signalling network (bRaf mutated)</sup></details>|<details><summary>pass</summary>[BIOMD0000000653_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000653#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000654](https://www.ebi.ac.uk/biomodels/BIOMD0000000654)<br/><sup>Padala2017- ERK, PI3K/Akt and </sup></summary><sup>Padala2017- ERK, PI3K/Akt and Wnt signalling network (Ras mutated)</sup></details>|<details><summary>pass</summary>[BIOMD0000000654_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000654#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000655](https://www.ebi.ac.uk/biomodels/BIOMD0000000655)<br/><sup>Padala2017- ERK, PI3K/Akt and </sup></summary><sup>Padala2017- ERK, PI3K/Akt and Wnt signalling network (PTEN mutation)</sup></details>|<details><summary>pass</summary>[BIOMD0000000655_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000655#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000656](https://www.ebi.ac.uk/biomodels/BIOMD0000000656)<br/><sup>Padala2017- ERK, PI3K/Akt and </sup></summary><sup>Padala2017- ERK, PI3K/Akt and Wnt signalling network (EGFR overexpression)</sup></details>|<details><summary>pass</summary>[BIOMD0000000656_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000656#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000657](https://www.ebi.ac.uk/biomodels/BIOMD0000000657)<br/><sup>Araujo2016 - Positive feedback</sup></summary><sup>Araujo2016 - Positive feedback in Cdk1 signalling keeps mitotic duration short and constant</sup></details>|<details><summary>pass</summary>[BIOMD0000000657.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000657#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000658](https://www.ebi.ac.uk/biomodels/BIOMD0000000658)<br/><sup>Lee2003 - Roles of APC and Axi</sup></summary><sup>Lee2003 - Roles of APC and Axin in Wnt Pathway (without regulatory loop)</sup></details>|<details><summary>pass</summary>[BIOMD0000000658_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000658#Files)</details>|pass|<details><summary>pass</summary>[MODEL1708310000_edited.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000658#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1708310000_edited.sedml447194.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000658_url.xml` has warnings.\r\n - 23 warnings of type SBML unit consistency (99505). The following is the first warning at line 1296, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Dsh0 - Dsh_a' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 31 warnings of type Modeling practice (80701). The following is the first warning at line 890, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1708310000_edited.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000658_url.xml` has warnings.\r\n - 23 warnings of type SBML unit consistency (99505). The following is the first warning at line 1296, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Dsh0 - Dsh_a' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 31 warnings of type Modeling practice (80701). The following is the first warning at line 890, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/MODEL1708310000_edited.sedml447194.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1708310000_edited.sedml` is invalid.\n - Data generator `Total_Axin_1_task1` is invalid.\n - Variable `p1_Total_Axin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Axin']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_B_Catenin_2_task1` is invalid.\n - Variable `p1_Total_B_Catenin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_B_Catenin']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1708310000_edited.sedml409819.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000658_url.xml` has warnings.\r\n - 23 warnings of type SBML unit consistency (99505). The following is the first warning at line 1296, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Dsh0 - Dsh_a' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 31 warnings of type Modeling practice (80701). The following is the first warning at line 890, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1708310000_edited.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000658_url.xml` has warnings.\r\n - 23 warnings of type SBML unit consistency (99505). The following is the first warning at line 1296, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Dsh0 - Dsh_a' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 31 warnings of type Modeling practice (80701). The following is the first warning at line 890, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1708310000_edited.sedml409819.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1708310000_edited.sedml` is invalid.\n - Data generator `Total_Axin_1_task1` is invalid.\n - Variable `p1_Total_Axin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Axin']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_B_Catenin_2_task1` is invalid.\n - Variable `p1_Total_B_Catenin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_B_Catenin']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000659](https://www.ebi.ac.uk/biomodels/BIOMD0000000659)<br/><sup>Cursons2015 - Regulation of ER</sup></summary><sup>Cursons2015 - Regulation of ERK-MAPK signaling in human epidermis</sup></details>|<details><summary>pass</summary>[BIOMD0000000659_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000659#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000660](https://www.ebi.ac.uk/biomodels/BIOMD0000000660)<br/><sup>Barr2017 - Dynamics of p21 in </sup></summary><sup>Barr2017 - Dynamics of p21 in hTert-RPE1 cells</sup></details>|<details><summary>pass</summary>[BIOMD0000000660_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000660#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000661](https://www.ebi.ac.uk/biomodels/BIOMD0000000661)<br/><sup>Webb2002 - Fas/FasL mediated t</sup></summary><sup>Webb2002 - Fas/FasL mediated tumor T-cell interaction</sup></details>|<details><summary>pass</summary>[BIOMD0000000661_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000661#Files)</details>|pass|<details><summary>FAIL</summary>[MODEL1006230046.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000661#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/MODEL1006230046.sedml719335.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/MODEL1006230046.sedml719335.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1006230046.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmp6wvxa9ot/./model1.xml` is not a file.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `LT_2_task1` is invalid.\n - Variable `p1_LT_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LT']` are not defined. No namespaces are defined for the target.\n - Data generator `RT_3_task1` is invalid.\n - Variable `p1_RT_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='RT']` are not defined. No namespaces are defined for the target.\n - Data generator `m_4_task1` is invalid.\n - Variable `p1_m_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='m']` are not defined. No namespaces are defined for the target.\n - Data generator `Lm_5_task1` is invalid.\n - Variable `p1_Lm_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Lm']` are not defined. No namespaces are defined for the target.\n - Data generator `Rm_6_task1` is invalid.\n - Variable `p1_Rm_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Rm']` are not defined. No namespaces are defined for the target.\n - Data generator `SL_7_task1` is invalid.\n - Variable `p1_SL_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SL']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1006230046.sedml794658.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1006230046.sedml794658.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1006230046.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmpfhvqfw_s/./model1.xml` is not a file.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `LT_2_task1` is invalid.\n - Variable `p1_LT_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LT']` are not defined. No namespaces are defined for the target.\n - Data generator `RT_3_task1` is invalid.\n - Variable `p1_RT_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='RT']` are not defined. No namespaces are defined for the target.\n - Data generator `m_4_task1` is invalid.\n - Variable `p1_m_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='m']` are not defined. No namespaces are defined for the target.\n - Data generator `Lm_5_task1` is invalid.\n - Variable `p1_Lm_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Lm']` are not defined. No namespaces are defined for the target.\n - Data generator `Rm_6_task1` is invalid.\n - Variable `p1_Rm_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Rm']` are not defined. No namespaces are defined for the target.\n - Data generator `SL_7_task1` is invalid.\n - Variable `p1_SL_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SL']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000662](https://www.ebi.ac.uk/biomodels/BIOMD0000000662)<br/><sup>Moore2004 - Chronic Myeloid Le</sup></summary><sup>Moore2004 - Chronic Myeloid Leukemic cells and T-lymphocyte interaction</sup></details>|<details><summary>pass</summary>[BIOMD0000000662_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000662#Files)</details>|FAIL|<details><summary>pass</summary>[Moore2004.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000662#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Moore2004.sedml941043.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000662_url.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (10501). The following is the first warning at line 1115, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'CML + eta' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 10 warnings of type SBML unit consistency (10541). The following is the first warning at line 1022, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'T_cell_naive_Source') are second (exponent = -1, multiplier = 11.5741, scale = 0), item (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 100.142406\r\n - Number of steps: 702\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Moore2004.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000662_url.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (10501). The following is the first warning at line 1115, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'CML + eta' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 10 warnings of type SBML unit consistency (10541). The following is the first warning at line 1022, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'T_cell_naive_Source') are second (exponent = -1, multiplier = 11.5741, scale = 0), item (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 100.142406\r\n - Number of steps: 702\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n`/root/in/Moore2004.sedml941043.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Moore2004.sedml` is invalid.\n - Output `plot_1_task1` is invalid.\n - Plot must have at least one curve.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Moore2004.sedml429106.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000662_url.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (10501). The following is the first warning at line 1115, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'CML + eta' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 10 warnings of type SBML unit consistency (10541). The following is the first warning at line 1022, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'T_cell_naive_Source') are second (exponent = -1, multiplier = 11.5741, scale = 0), item (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 100.142406\r\n - Number of steps: 702\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Moore2004.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000662_url.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (10501). The following is the first warning at line 1115, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'CML + eta' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 10 warnings of type SBML unit consistency (10541). The following is the first warning at line 1022, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'T_cell_naive_Source') are second (exponent = -1, multiplier = 11.5741, scale = 0), item (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 100.142406\r\n - Number of steps: 702\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Moore2004.sedml429106.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Moore2004.sedml` is invalid.\n - Output `plot_1_task1` is invalid.\n - Plot must have at least one curve.\n"```</details>|
|<details><summary>[BIOMD0000000663](https://www.ebi.ac.uk/biomodels/BIOMD0000000663)<br/><sup>Wodarz2007 - HIV/CD4 T-cell in</sup></summary><sup>Wodarz2007 - HIV/CD4 T-cell interaction</sup></details>|<details><summary>pass</summary>[BIOMD0000000663_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000663#Files)</details>|pass|<details><summary>pass</summary>[Wodarz2007_model_2.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000663#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Wodarz2007_model_2.sedml83646.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000663_url.xml` has warnings.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 669, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * generic_modifier_function_2_1(r, v, x)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type Modeling practice (80701). The following is the first warning at line 528, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Wodarz2007_model_2.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000663_url.xml` has warnings.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 669, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * generic_modifier_function_2_1(r, v, x)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type Modeling practice (80701). The following is the first warning at line 528, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Wodarz2007_model_2.sedml83646.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Wodarz2007_model_2.sedml` is invalid.\n - Data generator `v_1_task1` is invalid.\n - Variable `p1_v_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='v']` are not defined. No namespaces are defined for the target.\n - Data generator `x_2_task1` is invalid.\n - Variable `p1_x_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x']` are not defined. No namespaces are defined for the target.\n - Data generator `y_3_task1` is invalid.\n - Variable `p1_y_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='y']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Wodarz2007_model_2.sedml53319.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000663_url.xml` has warnings.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 669, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * generic_modifier_function_2_1(r, v, x)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type Modeling practice (80701). The following is the first warning at line 528, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Wodarz2007_model_2.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000663_url.xml` has warnings.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 669, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * generic_modifier_function_2_1(r, v, x)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type Modeling practice (80701). The following is the first warning at line 528, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Wodarz2007_model_2.sedml53319.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Wodarz2007_model_2.sedml` is invalid.\n - Data generator `v_1_task1` is invalid.\n - Variable `p1_v_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='v']` are not defined. No namespaces are defined for the target.\n - Data generator `x_2_task1` is invalid.\n - Variable `p1_x_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x']` are not defined. No namespaces are defined for the target.\n - Data generator `y_3_task1` is invalid.\n - Variable `p1_y_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='y']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000664](https://www.ebi.ac.uk/biomodels/BIOMD0000000664)<br/><sup>Muller2008 - Simplified MAPK a</sup></summary><sup>Muller2008 - Simplified MAPK activation Dynamics (Model B)</sup></details>|<details><summary>pass</summary>[BIOMD0000000664_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000664#Files)</details>|FAIL|<details><summary>pass</summary>[MODEL8687732743.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000664#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL8687732743.sedml791375.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000664_url.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 1016, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(0, and(geq(time, 0), lt(time, 5)), piecewise(1, and(geq(time, 5), lt(time, 10)), 0))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type SBML unit consistency (10541). The following is the first warning at line 1178, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 60, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'FGFR_Degradation') are second (exponent = -1, multiplier = 59.9999, scale = 0), mole (exponent = 1, multiplier = 1, scale = 0).\r\n - 5 warnings of type Modeling practice (80701). The following is the first warning at line 729, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'g1' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL8687732743.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000664_url.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 1016, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(0, and(geq(time, 0), lt(time, 5)), piecewise(1, and(geq(time, 5), lt(time, 10)), 0))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type SBML unit consistency (10541). The following is the first warning at line 1178, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 60, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'FGFR_Degradation') are second (exponent = -1, multiplier = 59.9999, scale = 0), mole (exponent = 1, multiplier = 1, scale = 0).\r\n - 5 warnings of type Modeling practice (80701). The following is the first warning at line 729, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'g1' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/MODEL8687732743.sedml791375.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL8687732743.sedml` is invalid.\n - Data generator `MAPK_1_task1` is invalid.\n - Variable `p1_MAPK_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='MAPK']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL8687732743.sedml551540.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000664_url.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 1016, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(0, and(geq(time, 0), lt(time, 5)), piecewise(1, and(geq(time, 5), lt(time, 10)), 0))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type SBML unit consistency (10541). The following is the first warning at line 1178, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 60, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'FGFR_Degradation') are second (exponent = -1, multiplier = 59.9999, scale = 0), mole (exponent = 1, multiplier = 1, scale = 0).\r\n - 5 warnings of type Modeling practice (80701). The following is the first warning at line 729, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'g1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL8687732743.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000664_url.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 1016, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(0, and(geq(time, 0), lt(time, 5)), piecewise(1, and(geq(time, 5), lt(time, 10)), 0))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type SBML unit consistency (10541). The following is the first warning at line 1178, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 60, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'FGFR_Degradation') are second (exponent = -1, multiplier = 59.9999, scale = 0), mole (exponent = 1, multiplier = 1, scale = 0).\r\n - 5 warnings of type Modeling practice (80701). The following is the first warning at line 729, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'g1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL8687732743.sedml551540.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL8687732743.sedml` is invalid.\n - Data generator `MAPK_1_task1` is invalid.\n - Variable `p1_MAPK_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='MAPK']` are not defined. No namespaces are defined for the target.\n"```</details>|
|[BIOMD0000000665](https://www.ebi.ac.uk/biomodels/BIOMD0000000665)<br/><sup>Fallon2000 - Interleukin-2 dynamics</sup>|<details><summary>pass</summary>[BIOMD0000000665_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000665#Files)</details>|FAIL|<details><summary>pass</summary>[MODEL1006230001.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000665#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1006230001.sedml203307.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000665_url.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 1640, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'kf * Li_0 * Ri_0 - kre * Ci_0' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 1036, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(kr / 11.1, eq(IL2, 1), kr / 8.2)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type SBML unit consistency (10541). The following is the first warning at line 1202, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 60, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Receptor_binding_to_Ligand') are second (exponent = -1, multiplier = 1.66667e+10, scale = 0), mole (exponent = 1, multiplier = 1, scale = 0).\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1006230001.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000665_url.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 1640, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'kf * Li_0 * Ri_0 - kre * Ci_0' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 1036, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(kr / 11.1, eq(IL2, 1), kr / 8.2)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type SBML unit consistency (10541). The following is the first warning at line 1202, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 60, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Receptor_binding_to_Ligand') are second (exponent = -1, multiplier = 1.66667e+10, scale = 0), mole (exponent = 1, multiplier = 1, scale = 0).\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/MODEL1006230001.sedml203307.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1006230001.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='kr']` are not defined. No namespaces are defined for the target.\n - Data generator `Cs_0_1_task1` is invalid.\n - Variable `p1_Cs_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cs_0']` are not defined. No namespaces are defined for the target.\n - Data generator `L_0_2_task1` is invalid.\n - Variable `p1_L_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L_0']` are not defined. No namespaces are defined for the target.\n - Data generator `Cs_0_1_task2` is invalid.\n - Variable `p1_Cs_0_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cs_0']` are not defined. No namespaces are defined for the target.\n - Data generator `L_0_2_task2` is invalid.\n - Variable `p1_L_0_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L_0']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1006230001.sedml20107.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000665_url.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 1640, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'kf * Li_0 * Ri_0 - kre * Ci_0' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 1036, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(kr / 11.1, eq(IL2, 1), kr / 8.2)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type SBML unit consistency (10541). The following is the first warning at line 1202, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 60, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Receptor_binding_to_Ligand') are second (exponent = -1, multiplier = 1.66667e+10, scale = 0), mole (exponent = 1, multiplier = 1, scale = 0).\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1006230001.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000665_url.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 1640, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'kf * Li_0 * Ri_0 - kre * Ci_0' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 1036, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(kr / 11.1, eq(IL2, 1), kr / 8.2)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type SBML unit consistency (10541). The following is the first warning at line 1202, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 60, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Receptor_binding_to_Ligand') are second (exponent = -1, multiplier = 1.66667e+10, scale = 0), mole (exponent = 1, multiplier = 1, scale = 0).\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1006230001.sedml20107.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1006230001.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='kr']` are not defined. No namespaces are defined for the target.\n - Data generator `Cs_0_1_task1` is invalid.\n - Variable `p1_Cs_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cs_0']` are not defined. No namespaces are defined for the target.\n - Data generator `L_0_2_task1` is invalid.\n - Variable `p1_L_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L_0']` are not defined. No namespaces are defined for the target.\n - Data generator `Cs_0_1_task2` is invalid.\n - Variable `p1_Cs_0_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cs_0']` are not defined. No namespaces are defined for the target.\n - Data generator `L_0_2_task2` is invalid.\n - Variable `p1_L_0_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L_0']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000666](https://www.ebi.ac.uk/biomodels/BIOMD0000000666)<br/><sup>Pappalardo2016 - PI3K/AKT and </sup></summary><sup>Pappalardo2016 - PI3K/AKT and MAPK Signaling Pathways in Melanoma Cancer</sup></details>|<details><summary>pass</summary>[BIOMD0000000666_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000666#Files)</details>|pass|<details><summary>pass</summary>[BIOMD0000000666.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000666#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/BIOMD0000000666.sedml607519.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000666_url.xml` has warnings.\r\n - 48 warnings of type SBML unit consistency (99505). The following is the first warning at line 3103, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment_0 * (k1 * species_25 * species_1 - k2 * species_0)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 87 warnings of type Modeling practice (80701). The following is the first warning at line 3125, column 12:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./BIOMD0000000666.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000666_url.xml` has warnings.\r\n - 48 warnings of type SBML unit consistency (99505). The following is the first warning at line 3103, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment_0 * (k1 * species_25 * species_1 - k2 * species_0)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 87 warnings of type Modeling practice (80701). The following is the first warning at line 3125, column 12:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/BIOMD0000000666.sedml607519.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./BIOMD0000000666.sedml` is invalid.\n - Data generator `Dabrafenib_1_task1` is invalid.\n - Variable `p1_Dabrafenib_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Dabrafenib']` are not defined. No namespaces are defined for the target.\n - Data generator `species_10_2_task1` is invalid.\n - Variable `p1_species_10_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_10']` are not defined. No namespaces are defined for the target.\n - Data generator `bRafMutated_3_task1` is invalid.\n - Variable `p1_bRafMutated_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='bRafMutated']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/BIOMD0000000666.sedml431622.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000666_url.xml` has warnings.\r\n - 48 warnings of type SBML unit consistency (99505). The following is the first warning at line 3103, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment_0 * (k1 * species_25 * species_1 - k2 * species_0)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 87 warnings of type Modeling practice (80701). The following is the first warning at line 3125, column 12:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./BIOMD0000000666.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000666_url.xml` has warnings.\r\n - 48 warnings of type SBML unit consistency (99505). The following is the first warning at line 3103, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment_0 * (k1 * species_25 * species_1 - k2 * species_0)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 87 warnings of type Modeling practice (80701). The following is the first warning at line 3125, column 12:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/BIOMD0000000666.sedml431622.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./BIOMD0000000666.sedml` is invalid.\n - Data generator `Dabrafenib_1_task1` is invalid.\n - Variable `p1_Dabrafenib_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Dabrafenib']` are not defined. No namespaces are defined for the target.\n - Data generator `species_10_2_task1` is invalid.\n - Variable `p1_species_10_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='species_10']` are not defined. No namespaces are defined for the target.\n - Data generator `bRafMutated_3_task1` is invalid.\n - Variable `p1_bRafMutated_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='bRafMutated']` are not defined. No namespaces are defined for the target.\n"```</details>|
|[BIOMD0000000667](https://www.ebi.ac.uk/biomodels/BIOMD0000000667)<br/><sup>Hornberg2005 - MAPKsignalling</sup>|<details><summary>pass</summary>[BIOMD0000000667_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000667#Files)</details>|pass|<details><summary>pass</summary>[MODEL0848279215.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000667#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL0848279215.sedml893633.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000667_url.xml` has warnings.\r\n - 148 warnings of type SBML unit consistency (99505). The following is the first warning at line 5034, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Compartment * function_for_v1_1(EGF, EGFR, EGF_EGFR, k1, kd1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 83 warnings of type Modeling practice (80701). The following is the first warning at line 4865, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL0848279215.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000667_url.xml` has warnings.\r\n - 148 warnings of type SBML unit consistency (99505). The following is the first warning at line 5034, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Compartment * function_for_v1_1(EGF, EGFR, EGF_EGFR, k1, kd1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 83 warnings of type Modeling practice (80701). The following is the first warning at line 4865, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/MODEL0848279215.sedml893633.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL0848279215.sedml` is invalid.\n - Data generator `ERK_PP_1_task1` is invalid.\n - Variable `p1_ERK_PP_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ERK_PP']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL0848279215.sedml618750.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000667_url.xml` has warnings.\r\n - 148 warnings of type SBML unit consistency (99505). The following is the first warning at line 5034, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Compartment * function_for_v1_1(EGF, EGFR, EGF_EGFR, k1, kd1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 83 warnings of type Modeling practice (80701). The following is the first warning at line 4865, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL0848279215.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000667_url.xml` has warnings.\r\n - 148 warnings of type SBML unit consistency (99505). The following is the first warning at line 5034, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Compartment * function_for_v1_1(EGF, EGFR, EGF_EGFR, k1, kd1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 83 warnings of type Modeling practice (80701). The following is the first warning at line 4865, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL0848279215.sedml618750.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL0848279215.sedml` is invalid.\n - Data generator `ERK_PP_1_task1` is invalid.\n - Variable `p1_ERK_PP_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ERK_PP']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000668](https://www.ebi.ac.uk/biomodels/BIOMD0000000668)<br/><sup>Zhu2015 - Combined gemcitabine</sup></summary><sup>Zhu2015 - Combined gemcitabine and birinapant in pancreatic cancer cells - basic PD model</sup></details>|<details><summary>pass</summary>[BIOMD0000000668_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000668#Files)</details>|pass|<details><summary>pass</summary>[MODEL1604270000_basic.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000668#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1604270000_basic.sedml8363.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000668_url.xml` has warnings.\r\n - 23 warnings of type SBML unit consistency (99505). The following is the first warning at line 1438, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Ra_0' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 26 warnings of type Modeling practice (80701). The following is the first warning at line 711, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Ra_0' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1604270000_basic.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000668_url.xml` has warnings.\r\n - 23 warnings of type SBML unit consistency (99505). The following is the first warning at line 1438, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Ra_0' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 26 warnings of type Modeling practice (80701). The following is the first warning at line 711, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Ra_0' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/MODEL1604270000_basic.sedml8363.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1604270000_basic.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='C_g']` are not defined. No namespaces are defined for the target.\n - Data generator `Ra_1_task1` is invalid.\n - Variable `p1_Ra_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Ra']` are not defined. No namespaces are defined for the target.\n - Data generator `Rd_2_task1` is invalid.\n - Variable `p1_Rd_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Rd']` are not defined. No namespaces are defined for the target.\n - Data generator `Ra_1_task2` is invalid.\n - Variable `p1_Ra_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Ra']` are not defined. No namespaces are defined for the target.\n - Data generator `Rd_2_task2` is invalid.\n - Variable `p1_Rd_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Rd']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1604270000_basic.sedml128716.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000668_url.xml` has warnings.\r\n - 23 warnings of type SBML unit consistency (99505). The following is the first warning at line 1438, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Ra_0' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 26 warnings of type Modeling practice (80701). The following is the first warning at line 711, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Ra_0' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1604270000_basic.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000668_url.xml` has warnings.\r\n - 23 warnings of type SBML unit consistency (99505). The following is the first warning at line 1438, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Ra_0' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 26 warnings of type Modeling practice (80701). The following is the first warning at line 711, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Ra_0' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1604270000_basic.sedml128716.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1604270000_basic.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='C_g']` are not defined. No namespaces are defined for the target.\n - Data generator `Ra_1_task1` is invalid.\n - Variable `p1_Ra_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Ra']` are not defined. No namespaces are defined for the target.\n - Data generator `Rd_2_task1` is invalid.\n - Variable `p1_Rd_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Rd']` are not defined. No namespaces are defined for the target.\n - Data generator `Ra_1_task2` is invalid.\n - Variable `p1_Ra_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Ra']` are not defined. No namespaces are defined for the target.\n - Data generator `Rd_2_task2` is invalid.\n - Variable `p1_Rd_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Rd']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000669](https://www.ebi.ac.uk/biomodels/BIOMD0000000669)<br/><sup>Zhu2015 - Combined gemcitabine</sup></summary><sup>Zhu2015 - Combined gemcitabine and birinapant in pancreatic cancer cells - mechanistic PD model</sup></details>|<details><summary>pass</summary>[BIOMD0000000669_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000669#Files)</details>|pass|<details><summary>pass</summary>[MODEL1604270001_mechanistic.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000669#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1604270001_mechanistic.sedml28526.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000669_url.xml` has warnings.\r\n - 24 warnings of type SBML unit consistency (99505). The following is the first warning at line 2265, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'f_G1__0 * R_0 / 100' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 58 warnings of type Modeling practice (80701). The following is the first warning at line 506, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Inh_1' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1604270001_mechanistic.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000669_url.xml` has warnings.\r\n - 24 warnings of type SBML unit consistency (99505). The following is the first warning at line 2265, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'f_G1__0 * R_0 / 100' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 58 warnings of type Modeling practice (80701). The following is the first warning at line 506, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Inh_1' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/MODEL1604270001_mechanistic.sedml28526.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1604270001_mechanistic.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='C_g']` are not defined. No namespaces are defined for the target.\n - Data generator `CURVE_G1_1_task1` is invalid.\n - Variable `p1_CURVE_G1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='CURVE_G1']` are not defined. No namespaces are defined for the target.\n - Data generator `CURVE_G2_2_task1` is invalid.\n - Variable `p1_CURVE_G2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='CURVE_G2']` are not defined. No namespaces are defined for the target.\n - Data generator `CURVE_S_3_task1` is invalid.\n - Variable `p1_CURVE_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='CURVE_S']` are not defined. No namespaces are defined for the target.\n - Data generator `CURVE_G1_1_task2` is invalid.\n - Variable `p1_CURVE_G1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='CURVE_G1']` are not defined. No namespaces are defined for the target.\n - Data generator `CURVE_G2_2_task2` is invalid.\n - Variable `p1_CURVE_G2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='CURVE_G2']` are not defined. No namespaces are defined for the target.\n - Data generator `CURVE_S_3_task2` is invalid.\n - Variable `p1_CURVE_S_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='CURVE_S']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1604270001_mechanistic.sedml154671.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000669_url.xml` has warnings.\r\n - 24 warnings of type SBML unit consistency (99505). The following is the first warning at line 2265, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'f_G1__0 * R_0 / 100' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 58 warnings of type Modeling practice (80701). The following is the first warning at line 506, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Inh_1' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1604270001_mechanistic.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000669_url.xml` has warnings.\r\n - 24 warnings of type SBML unit consistency (99505). The following is the first warning at line 2265, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'f_G1__0 * R_0 / 100' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 58 warnings of type Modeling practice (80701). The following is the first warning at line 506, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Inh_1' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1604270001_mechanistic.sedml154671.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1604270001_mechanistic.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='C_g']` are not defined. No namespaces are defined for the target.\n - Data generator `CURVE_G1_1_task1` is invalid.\n - Variable `p1_CURVE_G1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='CURVE_G1']` are not defined. No namespaces are defined for the target.\n - Data generator `CURVE_G2_2_task1` is invalid.\n - Variable `p1_CURVE_G2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='CURVE_G2']` are not defined. No namespaces are defined for the target.\n - Data generator `CURVE_S_3_task1` is invalid.\n - Variable `p1_CURVE_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='CURVE_S']` are not defined. No namespaces are defined for the target.\n - Data generator `CURVE_G1_1_task2` is invalid.\n - Variable `p1_CURVE_G1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='CURVE_G1']` are not defined. No namespaces are defined for the target.\n - Data generator `CURVE_G2_2_task2` is invalid.\n - Variable `p1_CURVE_G2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='CURVE_G2']` are not defined. No namespaces are defined for the target.\n - Data generator `CURVE_S_3_task2` is invalid.\n - Variable `p1_CURVE_S_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='CURVE_S']` are not defined. No namespaces are defined for the target.\n"```</details>|
|[BIOMD0000000670](https://www.ebi.ac.uk/biomodels/BIOMD0000000670)<br/><sup>Owen1998 - tumour growth model</sup>|<details><summary>pass</summary>[BIOMD0000000670_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000670#Files)</details>|FAIL|<details><summary>pass</summary>[MODEL1708240000.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000670#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1708240000.sedml152661.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000670_url.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 351, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <rateRule> <math> expression 'A * l * m * (N + 1) / (N + l + m + n) + I * (1 + S * m) - K_l * l * m * m - delta_l * l' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10532). The following is the first warning at line 410, column 6:\r\n - When the 'variable' in a <rateRule> definition refers to a <species>, the units of the rule's right-hand side are expected to be of the form _x per time_, where _x_ is the units of that species' quantity, and _time_ refers to the units of time for the model.\r\n Reference: L2V4 Section 4.11.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <rateRule> with variable 'n' are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0).\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 271, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'A' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1708240000.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000670_url.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 351, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <rateRule> <math> expression 'A * l * m * (N + 1) / (N + l + m + n) + I * (1 + S * m) - K_l * l * m * m - delta_l * l' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10532). The following is the first warning at line 410, column 6:\r\n - When the 'variable' in a <rateRule> definition refers to a <species>, the units of the rule's right-hand side are expected to be of the form _x per time_, where _x_ is the units of that species' quantity, and _time_ refers to the units of time for the model.\r\n Reference: L2V4 Section 4.11.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <rateRule> with variable 'n' are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0).\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 271, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'A' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/MODEL1708240000.sedml152661.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1708240000.sedml` is invalid.\n - Data generator `l_1_task1` is invalid.\n - Variable `p1_l_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='l']` are not defined. No namespaces are defined for the target.\n - Data generator `m_2_task1` is invalid.\n - Variable `p1_m_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='m']` are not defined. No namespaces are defined for the target.\n - Data generator `n_3_task1` is invalid.\n - Variable `p1_n_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='n']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1708240000.sedml235396.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000670_url.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 351, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <rateRule> <math> expression 'A * l * m * (N + 1) / (N + l + m + n) + I * (1 + S * m) - K_l * l * m * m - delta_l * l' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10532). The following is the first warning at line 410, column 6:\r\n - When the 'variable' in a <rateRule> definition refers to a <species>, the units of the rule's right-hand side are expected to be of the form _x per time_, where _x_ is the units of that species' quantity, and _time_ refers to the units of time for the model.\r\n Reference: L2V4 Section 4.11.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <rateRule> with variable 'n' are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0).\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 271, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'A' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1708240000.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000670_url.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 351, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <rateRule> <math> expression 'A * l * m * (N + 1) / (N + l + m + n) + I * (1 + S * m) - K_l * l * m * m - delta_l * l' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10532). The following is the first warning at line 410, column 6:\r\n - When the 'variable' in a <rateRule> definition refers to a <species>, the units of the rule's right-hand side are expected to be of the form _x per time_, where _x_ is the units of that species' quantity, and _time_ refers to the units of time for the model.\r\n Reference: L2V4 Section 4.11.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <rateRule> with variable 'n' are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0).\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 271, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'A' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1708240000.sedml235396.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1708240000.sedml` is invalid.\n - Data generator `l_1_task1` is invalid.\n - Variable `p1_l_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='l']` are not defined. No namespaces are defined for the target.\n - Data generator `m_2_task1` is invalid.\n - Variable `p1_m_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='m']` are not defined. No namespaces are defined for the target.\n - Data generator `n_3_task1` is invalid.\n - Variable `p1_n_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='n']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000671](https://www.ebi.ac.uk/biomodels/BIOMD0000000671)<br/><sup>Murphy2016 - Differences in pr</sup></summary><sup>Murphy2016 - Differences in predictions of ODE models of tumor growth</sup></details>|<details><summary>pass</summary>[BIOMD0000000671_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000671#Files)</details>|FAIL|<details><summary>pass</summary>[MODEL1708250001.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000671#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1708250001.sedml772613.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000671_url.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 561, column 6:\r\n - The formula 'pow(V_mend, b_mend)' in the math element of the <rateRule> contains a power that is not an integer and thus may produce invalid units.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 552, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <rateRule> <math> expression 'a_exp * V_exp' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 14 warnings of type Modeling practice (80701). The following is the first warning at line 340, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a_exp' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1708250001.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000671_url.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 561, column 6:\r\n - The formula 'pow(V_mend, b_mend)' in the math element of the <rateRule> contains a power that is not an integer and thus may produce invalid units.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 552, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <rateRule> <math> expression 'a_exp * V_exp' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 14 warnings of type Modeling practice (80701). The following is the first warning at line 340, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a_exp' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/MODEL1708250001.sedml772613.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1708250001.sedml` is invalid.\n - Data generator `V_bert_1_task1` is invalid.\n - Variable `p1_V_bert_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V_bert']` are not defined. No namespaces are defined for the target.\n - Data generator `V_exp_2_task1` is invalid.\n - Variable `p1_V_exp_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V_exp']` are not defined. No namespaces are defined for the target.\n - Data generator `V_gomp_3_task1` is invalid.\n - Variable `p1_V_gomp_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V_gomp']` are not defined. No namespaces are defined for the target.\n - Data generator `V_lin_4_task1` is invalid.\n - Variable `p1_V_lin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V_lin']` are not defined. No namespaces are defined for the target.\n - Data generator `V_log_5_task1` is invalid.\n - Variable `p1_V_log_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V_log']` are not defined. No namespaces are defined for the target.\n - Data generator `V_mend_6_task1` is invalid.\n - Variable `p1_V_mend_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V_mend']` are not defined. No namespaces are defined for the target.\n - Data generator `V_surf_7_task1` is invalid.\n - Variable `p1_V_surf_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V_surf']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1708250001.sedml888907.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000671_url.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 561, column 6:\r\n - The formula 'pow(V_mend, b_mend)' in the math element of the <rateRule> contains a power that is not an integer and thus may produce invalid units.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 552, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <rateRule> <math> expression 'a_exp * V_exp' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 14 warnings of type Modeling practice (80701). The following is the first warning at line 340, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a_exp' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1708250001.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000671_url.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 561, column 6:\r\n - The formula 'pow(V_mend, b_mend)' in the math element of the <rateRule> contains a power that is not an integer and thus may produce invalid units.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 552, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <rateRule> <math> expression 'a_exp * V_exp' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 14 warnings of type Modeling practice (80701). The following is the first warning at line 340, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a_exp' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1708250001.sedml888907.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1708250001.sedml` is invalid.\n - Data generator `V_bert_1_task1` is invalid.\n - Variable `p1_V_bert_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V_bert']` are not defined. No namespaces are defined for the target.\n - Data generator `V_exp_2_task1` is invalid.\n - Variable `p1_V_exp_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V_exp']` are not defined. No namespaces are defined for the target.\n - Data generator `V_gomp_3_task1` is invalid.\n - Variable `p1_V_gomp_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V_gomp']` are not defined. No namespaces are defined for the target.\n - Data generator `V_lin_4_task1` is invalid.\n - Variable `p1_V_lin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V_lin']` are not defined. No namespaces are defined for the target.\n - Data generator `V_log_5_task1` is invalid.\n - Variable `p1_V_log_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V_log']` are not defined. No namespaces are defined for the target.\n - Data generator `V_mend_6_task1` is invalid.\n - Variable `p1_V_mend_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V_mend']` are not defined. No namespaces are defined for the target.\n - Data generator `V_surf_7_task1` is invalid.\n - Variable `p1_V_surf_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V_surf']` are not defined. No namespaces are defined for the target.\n"```</details>|
|[BIOMD0000000672](https://www.ebi.ac.uk/biomodels/BIOMD0000000672)<br/><sup>Brown1997 - Plasma Melatonin Levels</sup>|<details><summary>pass</summary>[BIOMD0000000672_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000672#Files)</details>|pass|<details><summary>pass</summary>[MODEL1006230019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000672#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1006230019.sedml896693.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000672_url.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 660, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'log(2) / tau_I' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 14 warnings of type Modeling practice (80701). The following is the first warning at line 238, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'H1' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1006230019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000672_url.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 660, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'log(2) / tau_I' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 14 warnings of type Modeling practice (80701). The following is the first warning at line 238, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'H1' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/MODEL1006230019.sedml896693.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1006230019.sedml` is invalid.\n - Data generator `H2_1_task1` is invalid.\n - Variable `p1_H2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='H2']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1006230019.sedml498464.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000672_url.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 660, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'log(2) / tau_I' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 14 warnings of type Modeling practice (80701). The following is the first warning at line 238, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'H1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1006230019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000672_url.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 660, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'log(2) / tau_I' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 14 warnings of type Modeling practice (80701). The following is the first warning at line 238, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'H1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1006230019.sedml498464.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1006230019.sedml` is invalid.\n - Data generator `H2_1_task1` is invalid.\n - Variable `p1_H2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='H2']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000673](https://www.ebi.ac.uk/biomodels/BIOMD0000000673)<br/><sup>Lockwood2006 - Alzheimer's Dis</sup></summary><sup>Lockwood2006 - Alzheimer's Disease PBPK model</sup></details>|<details><summary>pass</summary>[BIOMD0000000673_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000673#Files)</details>|pass|<details><summary>FAIL</summary>[MODEL1006230054_edited.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000673#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/MODEL1006230054_edited.sedml237324.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task2`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1006230054_edited.sedml` has warnings.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task2`\x1b[0m\r\n\r\n`/root/in/MODEL1006230054_edited.sedml237324.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1006230054_edited.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmpg0y6bb1c/./model1.xml` is not a file.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='CeA']` are not defined. No namespaces are defined for the target.\n - Data generator `S_1_task1` is invalid.\n - Variable `p1_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='S']` are not defined. No namespaces are defined for the target.\n - Data generator `CeA_1_task1` is invalid.\n - Variable `p1_CeA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='CeA']` are not defined. No namespaces are defined for the target.\n - Data generator `S_1_task2` is invalid.\n - Variable `p1_S_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='S']` are not defined. No namespaces are defined for the target.\n - Data generator `CeA_1_task2` is invalid.\n - Variable `p1_CeA_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='CeA']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1006230054_edited.sedml703166.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task2`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1006230054_edited.sedml` has warnings.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task2`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1006230054_edited.sedml703166.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1006230054_edited.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmppas83o_w/./model1.xml` is not a file.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='CeA']` are not defined. No namespaces are defined for the target.\n - Data generator `S_1_task1` is invalid.\n - Variable `p1_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='S']` are not defined. No namespaces are defined for the target.\n - Data generator `CeA_1_task1` is invalid.\n - Variable `p1_CeA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='CeA']` are not defined. No namespaces are defined for the target.\n - Data generator `S_1_task2` is invalid.\n - Variable `p1_S_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='S']` are not defined. No namespaces are defined for the target.\n - Data generator `CeA_1_task2` is invalid.\n - Variable `p1_CeA_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='CeA']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000674](https://www.ebi.ac.uk/biomodels/BIOMD0000000674)<br/><sup>Reyes-Palomares2012 - a combin</sup></summary><sup>Reyes-Palomares2012 - a combined model hepatic polyamine and sulfur aminoacid metabolism - version1</sup></details>|<details><summary>pass</summary>[BIOMD0000000674_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000674#Files)</details>|FAIL|<details><summary>pass</summary>[MODEL1305060000_edited.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000674#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1305060000_edited.sedml162356.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000674_url.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 8795, column 8:\r\n - The formula 'pow(met, 1.21)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 89 warnings of type SBML unit consistency (99505). The following is the first warning at line 6010, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Fol / (2 * 3 / 4) - (c_5mf + c_2cf + c_1cf + c_10f + c_dhf)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 8028, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -6), second (exponent = -1, multiplier = 3600, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'VmGLYc') are litre (exponent = 1, multiplier = 1, scale = 0).\r\n - 214 warnings of type Modeling practice (80701). The following is the first warning at line 5467, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'tot_cfol' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1305060000_edited.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000674_url.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 8795, column 8:\r\n - The formula 'pow(met, 1.21)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 89 warnings of type SBML unit consistency (99505). The following is the first warning at line 6010, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Fol / (2 * 3 / 4) - (c_5mf + c_2cf + c_1cf + c_10f + c_dhf)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 8028, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -6), second (exponent = -1, multiplier = 3600, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'VmGLYc') are litre (exponent = 1, multiplier = 1, scale = 0).\r\n - 214 warnings of type Modeling practice (80701). The following is the first warning at line 5467, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'tot_cfol' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/MODEL1305060000_edited.sedml162356.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1305060000_edited.sedml` is invalid.\n - Data generator `sam_1_task1` is invalid.\n - Variable `p1_sam_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='sam']` are not defined. No namespaces are defined for the target.\n - Data generator `c_gsh_2_task1` is invalid.\n - Variable `p1_c_gsh_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='c_gsh']` are not defined. No namespaces are defined for the target.\n - Data generator `met_3_task1` is invalid.\n - Variable `p1_met_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='met']` are not defined. No namespaces are defined for the target.\n - Data generator `V_CBS_1_task1` is invalid.\n - Variable `p2_V_CBS_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='V_CBS']` are not defined. No namespaces are defined for the target.\n - Data generator `V_DNMT_2_task1` is invalid.\n - Variable `p2_V_DNMT_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='V_DNMT']` are not defined. No namespaces are defined for the target.\n - Data generator `V_GS_3_task1` is invalid.\n - Variable `p2_V_GS_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='V_GS']` are not defined. No namespaces are defined for the target.\n - Data generator `hcy_1_task1` is invalid.\n - Variable `p3_hcy_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='hcy']` are not defined. No namespaces are defined for the target.\n - Data generator `b_gsh_2_task1` is invalid.\n - Variable `p3_b_gsh_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='b_gsh']` are not defined. No namespaces are defined for the target.\n - Data generator `c_5mf_3_task1` is invalid.\n - Variable `p3_c_5mf_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='c_5mf']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1305060000_edited.sedml627275.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000674_url.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 8795, column 8:\r\n - The formula 'pow(met, 1.21)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 89 warnings of type SBML unit consistency (99505). The following is the first warning at line 6010, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Fol / (2 * 3 / 4) - (c_5mf + c_2cf + c_1cf + c_10f + c_dhf)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 8028, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -6), second (exponent = -1, multiplier = 3600, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'VmGLYc') are litre (exponent = 1, multiplier = 1, scale = 0).\r\n - 214 warnings of type Modeling practice (80701). The following is the first warning at line 5467, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'tot_cfol' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1305060000_edited.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000674_url.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 8795, column 8:\r\n - The formula 'pow(met, 1.21)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 89 warnings of type SBML unit consistency (99505). The following is the first warning at line 6010, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Fol / (2 * 3 / 4) - (c_5mf + c_2cf + c_1cf + c_10f + c_dhf)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 8028, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -6), second (exponent = -1, multiplier = 3600, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'VmGLYc') are litre (exponent = 1, multiplier = 1, scale = 0).\r\n - 214 warnings of type Modeling practice (80701). The following is the first warning at line 5467, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'tot_cfol' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1305060000_edited.sedml627275.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1305060000_edited.sedml` is invalid.\n - Data generator `sam_1_task1` is invalid.\n - Variable `p1_sam_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='sam']` are not defined. No namespaces are defined for the target.\n - Data generator `c_gsh_2_task1` is invalid.\n - Variable `p1_c_gsh_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='c_gsh']` are not defined. No namespaces are defined for the target.\n - Data generator `met_3_task1` is invalid.\n - Variable `p1_met_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='met']` are not defined. No namespaces are defined for the target.\n - Data generator `V_CBS_1_task1` is invalid.\n - Variable `p2_V_CBS_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='V_CBS']` are not defined. No namespaces are defined for the target.\n - Data generator `V_DNMT_2_task1` is invalid.\n - Variable `p2_V_DNMT_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='V_DNMT']` are not defined. No namespaces are defined for the target.\n - Data generator `V_GS_3_task1` is invalid.\n - Variable `p2_V_GS_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='V_GS']` are not defined. No namespaces are defined for the target.\n - Data generator `hcy_1_task1` is invalid.\n - Variable `p3_hcy_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='hcy']` are not defined. No namespaces are defined for the target.\n - Data generator `b_gsh_2_task1` is invalid.\n - Variable `p3_b_gsh_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='b_gsh']` are not defined. No namespaces are defined for the target.\n - Data generator `c_5mf_3_task1` is invalid.\n - Variable `p3_c_5mf_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='c_5mf']` are not defined. No namespaces are defined for the target.\n"```</details>|
|[BIOMD0000000675](https://www.ebi.ac.uk/biomodels/BIOMD0000000675)<br/><sup>Chen2000_CellCycle</sup>|<details><summary>pass</summary>[BIOMD0000000675.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000675#Files)</details>|pass|<details><summary>FAIL</summary>[MODEL1006230004.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000675#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/MODEL1006230004.sedml357740.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/MODEL1006230004.sedml357740.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1006230004.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - YDataReference from Curve `p2_curve_2_task1` cannot be resolved to a data generator with id `Hct1_2_task1` because multiple data generators have this id.\n - YDataReference from Curve `p2_curve_5_task1` cannot be resolved to a data generator with id `Sic1_T_5_task1` because multiple data generators have this id.\n - YDataReference from Curve `p3_curve_2_task1` cannot be resolved to a data generator with id `Hct1_2_task1` because multiple data generators have this id.\n - YDataReference from Curve `p3_curve_5_task1` cannot be resolved to a data generator with id `Sic1_T_5_task1` because multiple data generators have this id.\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1006230004.sedml959165.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1006230004.sedml959165.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1006230004.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - YDataReference from Curve `p2_curve_2_task1` cannot be resolved to a data generator with id `Hct1_2_task1` because multiple data generators have this id.\n - YDataReference from Curve `p2_curve_5_task1` cannot be resolved to a data generator with id `Sic1_T_5_task1` because multiple data generators have this id.\n - YDataReference from Curve `p3_curve_2_task1` cannot be resolved to a data generator with id `Hct1_2_task1` because multiple data generators have this id.\n - YDataReference from Curve `p3_curve_5_task1` cannot be resolved to a data generator with id `Sic1_T_5_task1` because multiple data generators have this id.\n'```</details>|
|<details><summary>[BIOMD0000000676](https://www.ebi.ac.uk/biomodels/BIOMD0000000676)<br/><sup>Chen2006 - Nitric Oxide Releas</sup></summary><sup>Chen2006 - Nitric Oxide Release from Endothelial Cells</sup></details>|<details><summary>pass</summary>[BIOMD0000000676_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000676#Files)</details>|pass|<details><summary>pass</summary>[MODEL1006230005.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000676#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1006230005.sedml494313.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000676_url.xml` has warnings.\r\n - 17 warnings of type SBML unit consistency (99505). The following is the first warning at line 1124, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Endothelium * (k1 * Arg * Fe3__enos - k1_prime * Fe3__Arg)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 21 warnings of type Modeling practice (80701). The following is the first warning at line 786, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - `task2`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task2`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1006230005.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000676_url.xml` has warnings.\r\n - 17 warnings of type SBML unit consistency (99505). The following is the first warning at line 1124, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Endothelium * (k1 * Arg * Fe3__enos - k1_prime * Fe3__Arg)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 21 warnings of type Modeling practice (80701). The following is the first warning at line 786, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - `task2`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task2`\x1b[0m\r\n\r\n`/root/in/MODEL1006230005.sedml494313.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1006230005.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Fe3__enos']` are not defined. No namespaces are defined for the target.\n - Output `plot_1_task1` is invalid.\n - Plot must have at least one curve.\n - Output `plot_2_task2` is invalid.\n - Plot must have at least one curve.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1006230005.sedml905973.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000676_url.xml` has warnings.\r\n - 17 warnings of type SBML unit consistency (99505). The following is the first warning at line 1124, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Endothelium * (k1 * Arg * Fe3__enos - k1_prime * Fe3__Arg)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 21 warnings of type Modeling practice (80701). The following is the first warning at line 786, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - `task2`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task2`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1006230005.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000676_url.xml` has warnings.\r\n - 17 warnings of type SBML unit consistency (99505). The following is the first warning at line 1124, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Endothelium * (k1 * Arg * Fe3__enos - k1_prime * Fe3__Arg)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 21 warnings of type Modeling practice (80701). The following is the first warning at line 786, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - `task2`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task2`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1006230005.sedml905973.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1006230005.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Fe3__enos']` are not defined. No namespaces are defined for the target.\n - Output `plot_1_task1` is invalid.\n - Plot must have at least one curve.\n - Output `plot_2_task2` is invalid.\n - Plot must have at least one curve.\n"```</details>|
|<details><summary>[BIOMD0000000677](https://www.ebi.ac.uk/biomodels/BIOMD0000000677)<br/><sup>Holmes2006 - Hill's model of m</sup></summary><sup>Holmes2006 - Hill's model of muscle contraction</sup></details>|<details><summary>pass</summary>[BIOMD0000000677_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000677#Files)</details>|pass|<details><summary>pass</summary>[MODEL1006230007.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000677#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1006230007.sedml611356.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000677_url.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 332, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'L_se' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 190, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'L_ce' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1006230007.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000677_url.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 332, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'L_se' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 190, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'L_ce' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/MODEL1006230007.sedml611356.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1006230007.sedml` is invalid.\n - Data generator `L_1_task1` is invalid.\n - Variable `p1_L_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='L']` are not defined. No namespaces are defined for the target.\n - Data generator `L_ce_2_task1` is invalid.\n - Variable `p1_L_ce_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='L_ce']` are not defined. No namespaces are defined for the target.\n - Data generator `L_se_3_task1` is invalid.\n - Variable `p1_L_se_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='L_se']` are not defined. No namespaces are defined for the target.\n - Data generator `P_4_task1` is invalid.\n - Variable `p1_P_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='P']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1006230007.sedml819463.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000677_url.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 332, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'L_se' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 190, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'L_ce' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1006230007.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000677_url.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 332, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'L_se' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 190, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'L_ce' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1006230007.sedml819463.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1006230007.sedml` is invalid.\n - Data generator `L_1_task1` is invalid.\n - Variable `p1_L_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='L']` are not defined. No namespaces are defined for the target.\n - Data generator `L_ce_2_task1` is invalid.\n - Variable `p1_L_ce_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='L_ce']` are not defined. No namespaces are defined for the target.\n - Data generator `L_se_3_task1` is invalid.\n - Variable `p1_L_se_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='L_se']` are not defined. No namespaces are defined for the target.\n - Data generator `P_4_task1` is invalid.\n - Variable `p1_P_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='P']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000678](https://www.ebi.ac.uk/biomodels/BIOMD0000000678)<br/><sup>Tomida2003 - Calcium Oscillato</sup></summary><sup>Tomida2003 - Calcium Oscillatory-induced translocation of nuclear factor of activated T cells</sup></details>|<details><summary>pass</summary>[BIOMD0000000678_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000678#Files)</details>|pass|<details><summary>FAIL</summary>[MODEL0912160005.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000678#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/MODEL0912160005.sedml515962.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/MODEL0912160005.sedml515962.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL0912160005.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmpooagjgis/./model1.xml` is not a file.\n - Data generator `dNFAT_1_task1` is invalid.\n - Variable `p1_dNFAT_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='dNFAT']` are not defined. No namespaces are defined for the target.\n - Data generator `tNFAT_2_task1` is invalid.\n - Variable `p1_tNFAT_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='tNFAT']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL0912160005.sedml622242.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL0912160005.sedml622242.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL0912160005.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmpqhruqey8/./model1.xml` is not a file.\n - Data generator `dNFAT_1_task1` is invalid.\n - Variable `p1_dNFAT_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='dNFAT']` are not defined. No namespaces are defined for the target.\n - Data generator `tNFAT_2_task1` is invalid.\n - Variable `p1_tNFAT_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='tNFAT']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000679](https://www.ebi.ac.uk/biomodels/BIOMD0000000679)<br/><sup>Waugh2006 - Diabetic Wound Hea</sup></summary><sup>Waugh2006 - Diabetic Wound Healing - Macrophage Dynamics</sup></details>|<details><summary>pass</summary>[Waugh2006_1.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000679#Files)</details>|FAIL|<details><summary>pass</summary>[Waugh2006_1.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000679#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Waugh2006_1.sedml990207.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Waugh2006_1.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 821, column 6:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'tau1 * pow(T, 3) + tau2 * pow(T, 2)' in the math element of the <assignmentRule> can only act on variables with the same units.\r\n - 1 warning of type SBML unit consistency (10512). The following is the first warning at line 821, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <species>, the units of the rule's right-hand side are expected to be consistent with the units of the species' quantity.\r\n Reference: L2V4 Section 4.11.3\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'K_T' are indeterminable.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 852, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <rateRule> <math> expression 'alpha * K_T + k1 * k2 * phi_I * (1 - k3 * (phi_I + phi_R)) - d1 * phi_I' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10532). The following is the first warning at line 934, column 6:\r\n - When the 'variable' in a <rateRule> definition refers to a <species>, the units of the rule's right-hand side are expected to be of the form _x per time_, where _x_ is the units of that species' quantity, and _time_ refers to the units of time for the model.\r\n Reference: L2V4 Section 4.11.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression of the <rateRule> with variable 'T' are indeterminable.\r\n - 5 warnings of type SBML unit consistency (10541). The following is the first warning at line 973, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Migration_of_monocytes_to_the_wound_site_phi_I') are mole (exponent = 1, multiplier = 1, scale = 0).\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Waugh2006_1.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Waugh2006_1.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 821, column 6:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'tau1 * pow(T, 3) + tau2 * pow(T, 2)' in the math element of the <assignmentRule> can only act on variables with the same units.\r\n - 1 warning of type SBML unit consistency (10512). The following is the first warning at line 821, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <species>, the units of the rule's right-hand side are expected to be consistent with the units of the species' quantity.\r\n Reference: L2V4 Section 4.11.3\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'K_T' are indeterminable.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 852, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <rateRule> <math> expression 'alpha * K_T + k1 * k2 * phi_I * (1 - k3 * (phi_I + phi_R)) - d1 * phi_I' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10532). The following is the first warning at line 934, column 6:\r\n - When the 'variable' in a <rateRule> definition refers to a <species>, the units of the rule's right-hand side are expected to be of the form _x per time_, where _x_ is the units of that species' quantity, and _time_ refers to the units of time for the model.\r\n Reference: L2V4 Section 4.11.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression of the <rateRule> with variable 'T' are indeterminable.\r\n - 5 warnings of type SBML unit consistency (10541). The following is the first warning at line 973, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Migration_of_monocytes_to_the_wound_site_phi_I') are mole (exponent = 1, multiplier = 1, scale = 0).\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/Waugh2006_1.sedml990207.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Waugh2006_1.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n - Data generator `phi_I_1_task1` is invalid.\n - Variable `p1_phi_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='phi_I']` are not defined. No namespaces are defined for the target.\n - Data generator `phi_R_1_task1` is invalid.\n - Variable `p2_phi_R_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='phi_R']` are not defined. No namespaces are defined for the target.\n - Data generator `phi_I_1_task2` is invalid.\n - Variable `p1_phi_I_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='phi_I']` are not defined. No namespaces are defined for the target.\n - Data generator `phi_R_1_task2` is invalid.\n - Variable `p2_phi_R_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='phi_R']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Waugh2006_1.sedml728384.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Waugh2006_1.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 821, column 6:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'tau1 * pow(T, 3) + tau2 * pow(T, 2)' in the math element of the <assignmentRule> can only act on variables with the same units.\r\n - 1 warning of type SBML unit consistency (10512). The following is the first warning at line 821, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <species>, the units of the rule's right-hand side are expected to be consistent with the units of the species' quantity.\r\n Reference: L2V4 Section 4.11.3\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'K_T' are indeterminable.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 852, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <rateRule> <math> expression 'alpha * K_T + k1 * k2 * phi_I * (1 - k3 * (phi_I + phi_R)) - d1 * phi_I' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10532). The following is the first warning at line 934, column 6:\r\n - When the 'variable' in a <rateRule> definition refers to a <species>, the units of the rule's right-hand side are expected to be of the form _x per time_, where _x_ is the units of that species' quantity, and _time_ refers to the units of time for the model.\r\n Reference: L2V4 Section 4.11.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression of the <rateRule> with variable 'T' are indeterminable.\r\n - 5 warnings of type SBML unit consistency (10541). The following is the first warning at line 973, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Migration_of_monocytes_to_the_wound_site_phi_I') are mole (exponent = 1, multiplier = 1, scale = 0).\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Waugh2006_1.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Waugh2006_1.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 821, column 6:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'tau1 * pow(T, 3) + tau2 * pow(T, 2)' in the math element of the <assignmentRule> can only act on variables with the same units.\r\n - 1 warning of type SBML unit consistency (10512). The following is the first warning at line 821, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <species>, the units of the rule's right-hand side are expected to be consistent with the units of the species' quantity.\r\n Reference: L2V4 Section 4.11.3\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'K_T' are indeterminable.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 852, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <rateRule> <math> expression 'alpha * K_T + k1 * k2 * phi_I * (1 - k3 * (phi_I + phi_R)) - d1 * phi_I' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10532). The following is the first warning at line 934, column 6:\r\n - When the 'variable' in a <rateRule> definition refers to a <species>, the units of the rule's right-hand side are expected to be of the form _x per time_, where _x_ is the units of that species' quantity, and _time_ refers to the units of time for the model.\r\n Reference: L2V4 Section 4.11.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression of the <rateRule> with variable 'T' are indeterminable.\r\n - 5 warnings of type SBML unit consistency (10541). The following is the first warning at line 973, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Migration_of_monocytes_to_the_wound_site_phi_I') are mole (exponent = 1, multiplier = 1, scale = 0).\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Waugh2006_1.sedml728384.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Waugh2006_1.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n - Data generator `phi_I_1_task1` is invalid.\n - Variable `p1_phi_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='phi_I']` are not defined. No namespaces are defined for the target.\n - Data generator `phi_R_1_task1` is invalid.\n - Variable `p2_phi_R_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='phi_R']` are not defined. No namespaces are defined for the target.\n - Data generator `phi_I_1_task2` is invalid.\n - Variable `p1_phi_I_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='phi_I']` are not defined. No namespaces are defined for the target.\n - Data generator `phi_R_1_task2` is invalid.\n - Variable `p2_phi_R_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='phi_R']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000680](https://www.ebi.ac.uk/biomodels/BIOMD0000000680)<br/><sup>Waugh2006 - Diabetic Wound Hea</sup></summary><sup>Waugh2006 - Diabetic Wound Healing - TGF-B Dynamics</sup></details>|<details><summary>pass</summary>[BIOMD0000000680_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000680#Files)</details>|FAIL|<details><summary>pass</summary>[Waugh2006_2.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000680#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Waugh2006_2.sedml669901.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000680_url.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 628, column 6:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'tau1 * pow(T, 3) + tau2 * pow(T, 2)' in the math element of the <assignmentRule> can only act on variables with the same units.\r\n - 1 warning of type SBML unit consistency (10512). The following is the first warning at line 628, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <species>, the units of the rule's right-hand side are expected to be consistent with the units of the species' quantity.\r\n Reference: L2V4 Section 4.11.3\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'K_T' are indeterminable.\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 659, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <rateRule> <math> expression 'alpha * K_T + k1 * k2 * phi_I * (1 - k3 * (phi_I + phi_R)) - d1 * phi_I' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10532). The following is the first warning at line 741, column 6:\r\n - When the 'variable' in a <rateRule> definition refers to a <species>, the units of the rule's right-hand side are expected to be of the form _x per time_, where _x_ is the units of that species' quantity, and _time_ refers to the units of time for the model.\r\n Reference: L2V4 Section 4.11.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression of the <rateRule> with variable 'T' are indeterminable.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Waugh2006_2.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000680_url.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 628, column 6:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'tau1 * pow(T, 3) + tau2 * pow(T, 2)' in the math element of the <assignmentRule> can only act on variables with the same units.\r\n - 1 warning of type SBML unit consistency (10512). The following is the first warning at line 628, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <species>, the units of the rule's right-hand side are expected to be consistent with the units of the species' quantity.\r\n Reference: L2V4 Section 4.11.3\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'K_T' are indeterminable.\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 659, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <rateRule> <math> expression 'alpha * K_T + k1 * k2 * phi_I * (1 - k3 * (phi_I + phi_R)) - d1 * phi_I' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10532). The following is the first warning at line 741, column 6:\r\n - When the 'variable' in a <rateRule> definition refers to a <species>, the units of the rule's right-hand side are expected to be of the form _x per time_, where _x_ is the units of that species' quantity, and _time_ refers to the units of time for the model.\r\n Reference: L2V4 Section 4.11.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression of the <rateRule> with variable 'T' are indeterminable.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/Waugh2006_2.sedml669901.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Waugh2006_2.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `T_1_task2` is invalid.\n - Variable `p1_T_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Waugh2006_2.sedml111037.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000680_url.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 628, column 6:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'tau1 * pow(T, 3) + tau2 * pow(T, 2)' in the math element of the <assignmentRule> can only act on variables with the same units.\r\n - 1 warning of type SBML unit consistency (10512). The following is the first warning at line 628, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <species>, the units of the rule's right-hand side are expected to be consistent with the units of the species' quantity.\r\n Reference: L2V4 Section 4.11.3\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'K_T' are indeterminable.\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 659, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <rateRule> <math> expression 'alpha * K_T + k1 * k2 * phi_I * (1 - k3 * (phi_I + phi_R)) - d1 * phi_I' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10532). The following is the first warning at line 741, column 6:\r\n - When the 'variable' in a <rateRule> definition refers to a <species>, the units of the rule's right-hand side are expected to be of the form _x per time_, where _x_ is the units of that species' quantity, and _time_ refers to the units of time for the model.\r\n Reference: L2V4 Section 4.11.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression of the <rateRule> with variable 'T' are indeterminable.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Waugh2006_2.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000680_url.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 628, column 6:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'tau1 * pow(T, 3) + tau2 * pow(T, 2)' in the math element of the <assignmentRule> can only act on variables with the same units.\r\n - 1 warning of type SBML unit consistency (10512). The following is the first warning at line 628, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <species>, the units of the rule's right-hand side are expected to be consistent with the units of the species' quantity.\r\n Reference: L2V4 Section 4.11.3\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'K_T' are indeterminable.\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 659, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <rateRule> <math> expression 'alpha * K_T + k1 * k2 * phi_I * (1 - k3 * (phi_I + phi_R)) - d1 * phi_I' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10532). The following is the first warning at line 741, column 6:\r\n - When the 'variable' in a <rateRule> definition refers to a <species>, the units of the rule's right-hand side are expected to be of the form _x per time_, where _x_ is the units of that species' quantity, and _time_ refers to the units of time for the model.\r\n Reference: L2V4 Section 4.11.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression of the <rateRule> with variable 'T' are indeterminable.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Waugh2006_2.sedml111037.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Waugh2006_2.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `T_1_task2` is invalid.\n - Variable `p1_T_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000681](https://www.ebi.ac.uk/biomodels/BIOMD0000000681)<br/><sup>Waugh2006 - Diabetic Wound Hea</sup></summary><sup>Waugh2006 - Diabetic Wound Healing - Treated and Untreated Macrophage Dynamics</sup></details>|<details><summary>pass</summary>[BIOMD0000000681_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000681#Files)</details>|FAIL|<details><summary>pass</summary>[Waugh2006_3.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000681#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Waugh2006_3.sedml994253.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000681_url.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 658, column 6:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'tau1 * pow(T, 3) + tau2 * pow(T, 2)' in the math element of the <assignmentRule> can only act on variables with the same units.\r\n - 1 warning of type SBML unit consistency (10512). The following is the first warning at line 658, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <species>, the units of the rule's right-hand side are expected to be consistent with the units of the species' quantity.\r\n Reference: L2V4 Section 4.11.3\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'K_T' are indeterminable.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 689, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '0 * K_T + phi_I + phi_R' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10532). The following is the first warning at line 785, column 6:\r\n - When the 'variable' in a <rateRule> definition refers to a <species>, the units of the rule's right-hand side are expected to be of the form _x per time_, where _x_ is the units of that species' quantity, and _time_ refers to the units of time for the model.\r\n Reference: L2V4 Section 4.11.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression of the <rateRule> with variable 'T' are indeterminable.\r\n - 2 warnings of type Modeling practice (80701). The following is the first warning at line 626, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'total_M' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Waugh2006_3.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000681_url.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 658, column 6:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'tau1 * pow(T, 3) + tau2 * pow(T, 2)' in the math element of the <assignmentRule> can only act on variables with the same units.\r\n - 1 warning of type SBML unit consistency (10512). The following is the first warning at line 658, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <species>, the units of the rule's right-hand side are expected to be consistent with the units of the species' quantity.\r\n Reference: L2V4 Section 4.11.3\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'K_T' are indeterminable.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 689, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '0 * K_T + phi_I + phi_R' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10532). The following is the first warning at line 785, column 6:\r\n - When the 'variable' in a <rateRule> definition refers to a <species>, the units of the rule's right-hand side are expected to be of the form _x per time_, where _x_ is the units of that species' quantity, and _time_ refers to the units of time for the model.\r\n Reference: L2V4 Section 4.11.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression of the <rateRule> with variable 'T' are indeterminable.\r\n - 2 warnings of type Modeling practice (80701). The following is the first warning at line 626, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'total_M' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/Waugh2006_3.sedml994253.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Waugh2006_3.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='dummy_variable']` are not defined. No namespaces are defined for the target.\n - Data generator `total_M_1_task1` is invalid.\n - Variable `p1_total_M_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='total_M']` are not defined. No namespaces are defined for the target.\n - Data generator `total_M_1_task2` is invalid.\n - Variable `p1_total_M_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='total_M']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Waugh2006_3.sedml478645.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000681_url.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 658, column 6:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'tau1 * pow(T, 3) + tau2 * pow(T, 2)' in the math element of the <assignmentRule> can only act on variables with the same units.\r\n - 1 warning of type SBML unit consistency (10512). The following is the first warning at line 658, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <species>, the units of the rule's right-hand side are expected to be consistent with the units of the species' quantity.\r\n Reference: L2V4 Section 4.11.3\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'K_T' are indeterminable.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 689, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '0 * K_T + phi_I + phi_R' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10532). The following is the first warning at line 785, column 6:\r\n - When the 'variable' in a <rateRule> definition refers to a <species>, the units of the rule's right-hand side are expected to be of the form _x per time_, where _x_ is the units of that species' quantity, and _time_ refers to the units of time for the model.\r\n Reference: L2V4 Section 4.11.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression of the <rateRule> with variable 'T' are indeterminable.\r\n - 2 warnings of type Modeling practice (80701). The following is the first warning at line 626, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'total_M' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Waugh2006_3.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000681_url.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 658, column 6:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'tau1 * pow(T, 3) + tau2 * pow(T, 2)' in the math element of the <assignmentRule> can only act on variables with the same units.\r\n - 1 warning of type SBML unit consistency (10512). The following is the first warning at line 658, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <species>, the units of the rule's right-hand side are expected to be consistent with the units of the species' quantity.\r\n Reference: L2V4 Section 4.11.3\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'K_T' are indeterminable.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 689, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '0 * K_T + phi_I + phi_R' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10532). The following is the first warning at line 785, column 6:\r\n - When the 'variable' in a <rateRule> definition refers to a <species>, the units of the rule's right-hand side are expected to be of the form _x per time_, where _x_ is the units of that species' quantity, and _time_ refers to the units of time for the model.\r\n Reference: L2V4 Section 4.11.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), litre (exponent = -1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression of the <rateRule> with variable 'T' are indeterminable.\r\n - 2 warnings of type Modeling practice (80701). The following is the first warning at line 626, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'total_M' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Waugh2006_3.sedml478645.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Waugh2006_3.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='dummy_variable']` are not defined. No namespaces are defined for the target.\n - Data generator `total_M_1_task1` is invalid.\n - Variable `p1_total_M_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='total_M']` are not defined. No namespaces are defined for the target.\n - Data generator `total_M_1_task2` is invalid.\n - Variable `p1_total_M_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='total_M']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000682](https://www.ebi.ac.uk/biomodels/BIOMD0000000682)<br/><sup>Wierschem2004 - Electrical bur</sup></summary><sup>Wierschem2004 - Electrical bursting activity in Pancreatic Islets</sup></details>|<details><summary>pass</summary>[BIOMD0000000682_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000682#Files)</details>|pass|<details><summary>pass</summary>[MODEL1006230006.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000682#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1006230006.sedml95565.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000682_url.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 1241, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '(V_membrane - V_K) * g_KATP_ / ATP' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 34 warnings of type Modeling practice (80701). The following is the first warning at line 322, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'tau_c' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1006230006.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000682_url.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 1241, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '(V_membrane - V_K) * g_KATP_ / ATP' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 34 warnings of type Modeling practice (80701). The following is the first warning at line 322, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'tau_c' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/MODEL1006230006.sedml95565.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1006230006.sedml` is invalid.\n - Data generator `ATP_1_task1` is invalid.\n - Variable `p1_ATP_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='ATP']` are not defined. No namespaces are defined for the target.\n - Data generator `V_membrane_1_task1` is invalid.\n - Variable `p2_V_membrane_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='V_membrane']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1006230006.sedml770037.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000682_url.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 1241, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '(V_membrane - V_K) * g_KATP_ / ATP' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 34 warnings of type Modeling practice (80701). The following is the first warning at line 322, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'tau_c' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1006230006.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000682_url.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 1241, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '(V_membrane - V_K) * g_KATP_ / ATP' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 34 warnings of type Modeling practice (80701). The following is the first warning at line 322, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'tau_c' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1006230006.sedml770037.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1006230006.sedml` is invalid.\n - Data generator `ATP_1_task1` is invalid.\n - Variable `p1_ATP_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='ATP']` are not defined. No namespaces are defined for the target.\n - Data generator `V_membrane_1_task1` is invalid.\n - Variable `p2_V_membrane_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='V_membrane']` are not defined. No namespaces are defined for the target.\n"```</details>|
|[BIOMD0000000683](https://www.ebi.ac.uk/biomodels/BIOMD0000000683)<br/><sup>Wodarz1999 CTL memory response HIV</sup>|<details><summary>pass</summary>[MODEL1006230062.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000683#Files)</details>|pass|<details><summary>pass</summary>[MODEL1006230062.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000683#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1006230062.sedml631394.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1006230062.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 829, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'log10(y)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 537, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lamda' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1006230062.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1006230062.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 829, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'log10(y)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 537, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lamda' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/MODEL1006230062.sedml631394.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1006230062.sedml` is invalid.\n - Data generator `log_w_1_task1` is invalid.\n - Variable `p1_log_w_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_w']` are not defined. No namespaces are defined for the target.\n - Data generator `log_y_2_task1` is invalid.\n - Variable `p1_log_y_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_y']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1006230062.sedml264442.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1006230062.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 829, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'log10(y)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 537, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lamda' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1006230062.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1006230062.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 829, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'log10(y)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 537, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lamda' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1006230062.sedml264442.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1006230062.sedml` is invalid.\n - Data generator `log_w_1_task1` is invalid.\n - Variable `p1_log_w_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_w']` are not defined. No namespaces are defined for the target.\n - Data generator `log_y_2_task1` is invalid.\n - Variable `p1_log_y_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_y']` are not defined. No namespaces are defined for the target.\n"```</details>|
|[BIOMD0000000684](https://www.ebi.ac.uk/biomodels/BIOMD0000000684)<br/><sup>Wodarz2003 - Immunological Memory</sup>|<details><summary>pass</summary>[BIOMD0000000684_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000684#Files)</details>|pass|<details><summary>pass</summary>[MODEL1006230061.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000684#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1006230061.sedml499606.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000684_url.xml` has warnings.\r\n - 15 warnings of type SBML unit consistency (99505). The following is the first warning at line 987, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'S + I_1 + R_1 + I_2 + R_2 + I_12 + I_21 + R_12' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 28 warnings of type Modeling practice (80701). The following is the first warning at line 246, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'S' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1006230061.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000684_url.xml` has warnings.\r\n - 15 warnings of type SBML unit consistency (99505). The following is the first warning at line 987, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'S + I_1 + R_1 + I_2 + R_2 + I_12 + I_21 + R_12' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 28 warnings of type Modeling practice (80701). The following is the first warning at line 246, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'S' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/MODEL1006230061.sedml499606.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1006230061.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='g']` are not defined. No namespaces are defined for the target.\n - Data generator `log_P1_1_task1` is invalid.\n - Variable `p1_log_P1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_P1']` are not defined. No namespaces are defined for the target.\n - Data generator `log_P2_2_task1` is invalid.\n - Variable `p1_log_P2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_P2']` are not defined. No namespaces are defined for the target.\n - Data generator `log_P1_1_task2` is invalid.\n - Variable `p1_log_P1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_P1']` are not defined. No namespaces are defined for the target.\n - Data generator `log_P2_2_task2` is invalid.\n - Variable `p1_log_P2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_P2']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1006230061.sedml39118.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000684_url.xml` has warnings.\r\n - 15 warnings of type SBML unit consistency (99505). The following is the first warning at line 987, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'S + I_1 + R_1 + I_2 + R_2 + I_12 + I_21 + R_12' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 28 warnings of type Modeling practice (80701). The following is the first warning at line 246, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'S' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1006230061.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000684_url.xml` has warnings.\r\n - 15 warnings of type SBML unit consistency (99505). The following is the first warning at line 987, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'S + I_1 + R_1 + I_2 + R_2 + I_12 + I_21 + R_12' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 28 warnings of type Modeling practice (80701). The following is the first warning at line 246, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'S' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1006230061.sedml39118.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1006230061.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='g']` are not defined. No namespaces are defined for the target.\n - Data generator `log_P1_1_task1` is invalid.\n - Variable `p1_log_P1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_P1']` are not defined. No namespaces are defined for the target.\n - Data generator `log_P2_2_task1` is invalid.\n - Variable `p1_log_P2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_P2']` are not defined. No namespaces are defined for the target.\n - Data generator `log_P1_1_task2` is invalid.\n - Variable `p1_log_P1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_P1']` are not defined. No namespaces are defined for the target.\n - Data generator `log_P2_2_task2` is invalid.\n - Variable `p1_log_P2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_P2']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000685](https://www.ebi.ac.uk/biomodels/BIOMD0000000685)<br/><sup>Wodarz2003 - Cytotoxic T lymph</sup></summary><sup>Wodarz2003 - Cytotoxic T lymphocyte cross-priming</sup></details>|<details><summary>pass</summary>[BIOMD0000000685_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000685#Files)</details>|pass|<details><summary>pass</summary>[MODEL1006230094.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000685#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1006230094.sedml431298.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000685_url.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 791, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'C * A_star / (q * T)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 17 warnings of type Modeling practice (80701). The following is the first warning at line 231, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'T' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1006230094.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000685_url.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 791, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'C * A_star / (q * T)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 17 warnings of type Modeling practice (80701). The following is the first warning at line 231, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'T' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/MODEL1006230094.sedml431298.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1006230094.sedml` is invalid.\n - Data generator `C_1_task1` is invalid.\n - Variable `p1_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='C']` are not defined. No namespaces are defined for the target.\n - Data generator `T_2_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1006230094.sedml740055.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000685_url.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 791, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'C * A_star / (q * T)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 17 warnings of type Modeling practice (80701). The following is the first warning at line 231, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'T' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1006230094.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000685_url.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 791, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'C * A_star / (q * T)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 17 warnings of type Modeling practice (80701). The following is the first warning at line 231, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'T' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1006230094.sedml740055.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1006230094.sedml` is invalid.\n - Data generator `C_1_task1` is invalid.\n - Variable `p1_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='C']` are not defined. No namespaces are defined for the target.\n - Data generator `T_2_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000686](https://www.ebi.ac.uk/biomodels/BIOMD0000000686)<br/><sup>Wodarz2007 - Basic Model of Cy</sup></summary><sup>Wodarz2007 - Basic Model of Cytomegalovirus Infection</sup></details>|<details><summary>pass</summary>[BIOMD0000000686_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000686#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000687](https://www.ebi.ac.uk/biomodels/BIOMD0000000687)<br/><sup>Wodarz2007 - Cytomegalovirus i</sup></summary><sup>Wodarz2007 - Cytomegalovirus infection model with cytotoxic T lymphocyte response</sup></details>|<details><summary>pass</summary>[BIOMD0000000687_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000687#Files)</details>|pass|<details><summary>pass</summary>[MODEL1006230052.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000687#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1006230052.sedml761733.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000687_url.xml` has warnings.\r\n - 26 warnings of type SBML unit consistency (99505). The following is the first warning at line 1199, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'lambda * eta / (d * a1 * (a0 + eta)) * (beta + gamma * phi / (phi + d))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 16 warnings of type Modeling practice (80701). The following is the first warning at line 957, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'R0' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1006230052.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000687_url.xml` has warnings.\r\n - 26 warnings of type SBML unit consistency (99505). The following is the first warning at line 1199, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'lambda * eta / (d * a1 * (a0 + eta)) * (beta + gamma * phi / (phi + d))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 16 warnings of type Modeling practice (80701). The following is the first warning at line 957, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'R0' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/MODEL1006230052.sedml761733.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1006230052.sedml` is invalid.\n - Data generator `z_a_1_task1` is invalid.\n - Variable `p1_z_a_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='z_a']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1006230052.sedml823417.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000687_url.xml` has warnings.\r\n - 26 warnings of type SBML unit consistency (99505). The following is the first warning at line 1199, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'lambda * eta / (d * a1 * (a0 + eta)) * (beta + gamma * phi / (phi + d))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 16 warnings of type Modeling practice (80701). The following is the first warning at line 957, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'R0' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1006230052.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000687_url.xml` has warnings.\r\n - 26 warnings of type SBML unit consistency (99505). The following is the first warning at line 1199, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'lambda * eta / (d * a1 * (a0 + eta)) * (beta + gamma * phi / (phi + d))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 16 warnings of type Modeling practice (80701). The following is the first warning at line 957, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'R0' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1006230052.sedml823417.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1006230052.sedml` is invalid.\n - Data generator `z_a_1_task1` is invalid.\n - Variable `p1_z_a_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='z_a']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000688](https://www.ebi.ac.uk/biomodels/BIOMD0000000688)<br/><sup>Wodarz2007 - Cytomegalovirus i</sup></summary><sup>Wodarz2007 - Cytomegalovirus infection model with cytotoxic T lymphocyte and natural killer cell response</sup></details>|<details><summary>pass</summary>[BIOMD0000000688_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000688#Files)</details>|pass|<details><summary>pass</summary>[MODEL1006230097.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000688#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1006230097.sedml73536.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000688_url.xml` has warnings.\r\n - 31 warnings of type SBML unit consistency (99505). The following is the first warning at line 1246, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'lambda * eta / (d * a1 * (a0 + eta)) * (beta + gamma * phi / (phi + d))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 20 warnings of type Modeling practice (80701). The following is the first warning at line 1000, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'R0' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1006230097.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000688_url.xml` has warnings.\r\n - 31 warnings of type SBML unit consistency (99505). The following is the first warning at line 1246, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'lambda * eta / (d * a1 * (a0 + eta)) * (beta + gamma * phi / (phi + d))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 20 warnings of type Modeling practice (80701). The following is the first warning at line 1000, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'R0' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/MODEL1006230097.sedml73536.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1006230097.sedml` is invalid.\n - Data generator `z_i_1_task1` is invalid.\n - Variable `p1_z_i_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='z_i']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1006230097.sedml405512.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000688_url.xml` has warnings.\r\n - 31 warnings of type SBML unit consistency (99505). The following is the first warning at line 1246, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'lambda * eta / (d * a1 * (a0 + eta)) * (beta + gamma * phi / (phi + d))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 20 warnings of type Modeling practice (80701). The following is the first warning at line 1000, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'R0' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1006230097.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000688_url.xml` has warnings.\r\n - 31 warnings of type SBML unit consistency (99505). The following is the first warning at line 1246, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'lambda * eta / (d * a1 * (a0 + eta)) * (beta + gamma * phi / (phi + d))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 20 warnings of type Modeling practice (80701). The following is the first warning at line 1000, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'R0' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1006230097.sedml405512.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1006230097.sedml` is invalid.\n - Data generator `z_i_1_task1` is invalid.\n - Variable `p1_z_i_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='z_i']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000689](https://www.ebi.ac.uk/biomodels/BIOMD0000000689)<br/><sup>Thiaville2016 - Folate pathway</sup></summary><sup>Thiaville2016 - Folate pathway model (PanB overexpression)</sup></details>|<details><summary>pass</summary>[BIOMD0000000689_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000689#Files)</details>|pass|<details><summary>pass</summary>[MODEL1602280002.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000689#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1602280002.sedml892695.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000689_url.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 1023, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * k1 * ATP * H2_HMPt' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 1034, column 12:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1602280002.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000689_url.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 1023, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * k1 * ATP * H2_HMPt' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 1034, column 12:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/MODEL1602280002.sedml892695.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1602280002.sedml` is invalid.\n - Data generator `CH2_THF_1_task1` is invalid.\n - Variable `p1_CH2_THF_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CH2_THF']` are not defined. No namespaces are defined for the target.\n - Data generator `THF_2_task1` is invalid.\n - Variable `p1_THF_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='THF']` are not defined. No namespaces are defined for the target.\n - Data generator `DHF_1_task1` is invalid.\n - Variable `p2_DHF_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='DHF']` are not defined. No namespaces are defined for the target.\n - Data generator `H2_HMPt_2_task1` is invalid.\n - Variable `p2_H2_HMPt_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H2_HMPt']` are not defined. No namespaces are defined for the target.\n - Data generator `H2_HMPterinPP_3_task1` is invalid.\n - Variable `p2_H2_HMPterinPP_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H2_HMPterinPP']` are not defined. No namespaces are defined for the target.\n - Data generator `H2_Pteroate_4_task1` is invalid.\n - Variable `p2_H2_Pteroate_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H2_Pteroate']` are not defined. No namespaces are defined for the target.\n - Data generator `p_ABA_5_task1` is invalid.\n - Variable `p2_p_ABA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='p_ABA']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1602280002.sedml78027.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000689_url.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 1023, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * k1 * ATP * H2_HMPt' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 1034, column 12:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1602280002.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000689_url.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 1023, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * k1 * ATP * H2_HMPt' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 1034, column 12:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1602280002.sedml78027.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1602280002.sedml` is invalid.\n - Data generator `CH2_THF_1_task1` is invalid.\n - Variable `p1_CH2_THF_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CH2_THF']` are not defined. No namespaces are defined for the target.\n - Data generator `THF_2_task1` is invalid.\n - Variable `p1_THF_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='THF']` are not defined. No namespaces are defined for the target.\n - Data generator `DHF_1_task1` is invalid.\n - Variable `p2_DHF_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='DHF']` are not defined. No namespaces are defined for the target.\n - Data generator `H2_HMPt_2_task1` is invalid.\n - Variable `p2_H2_HMPt_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H2_HMPt']` are not defined. No namespaces are defined for the target.\n - Data generator `H2_HMPterinPP_3_task1` is invalid.\n - Variable `p2_H2_HMPterinPP_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H2_HMPterinPP']` are not defined. No namespaces are defined for the target.\n - Data generator `H2_Pteroate_4_task1` is invalid.\n - Variable `p2_H2_Pteroate_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H2_Pteroate']` are not defined. No namespaces are defined for the target.\n - Data generator `p_ABA_5_task1` is invalid.\n - Variable `p2_p_ABA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='p_ABA']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000690](https://www.ebi.ac.uk/biomodels/BIOMD0000000690)<br/><sup>Thiaville2016 - Folate pathway</sup></summary><sup>Thiaville2016 - Folate pathway model (PanB overexpression and THF regulation)</sup></details>|<details><summary>pass</summary>[BIOMD0000000690_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000690#Files)</details>|pass|<details><summary>pass</summary>[MODEL1602280003.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000690#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1602280003.sedml556981.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000690_url.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 1023, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * k1 * ATP * H2_HMPt' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 1034, column 12:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1602280003.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000690_url.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 1023, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * k1 * ATP * H2_HMPt' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 1034, column 12:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/MODEL1602280003.sedml556981.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1602280003.sedml` is invalid.\n - Data generator `CH2_THF_1_task1` is invalid.\n - Variable `p1_CH2_THF_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CH2_THF']` are not defined. No namespaces are defined for the target.\n - Data generator `THF_2_task1` is invalid.\n - Variable `p1_THF_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='THF']` are not defined. No namespaces are defined for the target.\n - Data generator `DHF_1_task1` is invalid.\n - Variable `p2_DHF_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='DHF']` are not defined. No namespaces are defined for the target.\n - Data generator `H2_HMPt_2_task1` is invalid.\n - Variable `p2_H2_HMPt_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H2_HMPt']` are not defined. No namespaces are defined for the target.\n - Data generator `H2_HMPterinPP_3_task1` is invalid.\n - Variable `p2_H2_HMPterinPP_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H2_HMPterinPP']` are not defined. No namespaces are defined for the target.\n - Data generator `H2_Pteroate_4_task1` is invalid.\n - Variable `p2_H2_Pteroate_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H2_Pteroate']` are not defined. No namespaces are defined for the target.\n - Data generator `p_ABA_5_task1` is invalid.\n - Variable `p2_p_ABA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='p_ABA']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1602280003.sedml634617.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000690_url.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 1023, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * k1 * ATP * H2_HMPt' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 1034, column 12:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1602280003.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000690_url.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 1023, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * k1 * ATP * H2_HMPt' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 1034, column 12:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1602280003.sedml634617.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1602280003.sedml` is invalid.\n - Data generator `CH2_THF_1_task1` is invalid.\n - Variable `p1_CH2_THF_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CH2_THF']` are not defined. No namespaces are defined for the target.\n - Data generator `THF_2_task1` is invalid.\n - Variable `p1_THF_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='THF']` are not defined. No namespaces are defined for the target.\n - Data generator `DHF_1_task1` is invalid.\n - Variable `p2_DHF_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='DHF']` are not defined. No namespaces are defined for the target.\n - Data generator `H2_HMPt_2_task1` is invalid.\n - Variable `p2_H2_HMPt_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H2_HMPt']` are not defined. No namespaces are defined for the target.\n - Data generator `H2_HMPterinPP_3_task1` is invalid.\n - Variable `p2_H2_HMPterinPP_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H2_HMPterinPP']` are not defined. No namespaces are defined for the target.\n - Data generator `H2_Pteroate_4_task1` is invalid.\n - Variable `p2_H2_Pteroate_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H2_Pteroate']` are not defined. No namespaces are defined for the target.\n - Data generator `p_ABA_5_task1` is invalid.\n - Variable `p2_p_ABA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='p_ABA']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000691](https://www.ebi.ac.uk/biomodels/BIOMD0000000691)<br/><sup>Wolf2000 - Cellular interactio</sup></summary><sup>Wolf2000 - Cellular interaction on glycolytic oscillations in yeast</sup></details>|<details><summary>pass</summary>[BIOMD0000000691_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000691#Files)</details>|pass|<details><summary>pass</summary>[MODEL1006230022.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000691#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1006230022.sedml153232.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/MODEL1006230022.sedml153232.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1006230022.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - YDataReference from Curve `p1_curve_1_task1` cannot be resolved to a data generator with id `A3__Cell_1__1_task1` because multiple data generators have this id.\n - XDataReference from Curve `p2_curve_1_task1` cannot be resolved to a data generator with id `A3__Cell_1__1_task1` because multiple data generators have this id.\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1006230022.sedml227935.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1006230022.sedml227935.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1006230022.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - YDataReference from Curve `p1_curve_1_task1` cannot be resolved to a data generator with id `A3__Cell_1__1_task1` because multiple data generators have this id.\n - XDataReference from Curve `p2_curve_1_task1` cannot be resolved to a data generator with id `A3__Cell_1__1_task1` because multiple data generators have this id.\n'```</details>|
|<details><summary>[BIOMD0000000692](https://www.ebi.ac.uk/biomodels/BIOMD0000000692)<br/><sup>Phillips2003 - The Mechanism o</sup></summary><sup>Phillips2003 - The Mechanism of Ras GTPase Activation by Neurofibromin</sup></details>|<details><summary>pass</summary>[BIOMD0000000692_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000692#Files)</details>|pass|<details><summary>pass</summary>[MODEL4968912141.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000692#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL4968912141.sedml666830.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000692_url.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 777, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '100 * (Pi - 1) / 1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 735, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Pi_curve' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL4968912141.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000692_url.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 777, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '100 * (Pi - 1) / 1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 735, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Pi_curve' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/MODEL4968912141.sedml666830.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL4968912141.sedml` is invalid.\n - Data generator `Pi_curve_1_task1` is invalid.\n - Variable `p1_Pi_curve_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Pi_curve']` are not defined. No namespaces are defined for the target.\n - Data generator `hplc_curve_2_task1` is invalid.\n - Variable `p1_hplc_curve_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='hplc_curve']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL4968912141.sedml233280.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000692_url.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 777, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '100 * (Pi - 1) / 1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 735, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Pi_curve' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL4968912141.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000692_url.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 777, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '100 * (Pi - 1) / 1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 735, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Pi_curve' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL4968912141.sedml233280.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL4968912141.sedml` is invalid.\n - Data generator `Pi_curve_1_task1` is invalid.\n - Variable `p1_Pi_curve_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Pi_curve']` are not defined. No namespaces are defined for the target.\n - Data generator `hplc_curve_2_task1` is invalid.\n - Variable `p1_hplc_curve_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='hplc_curve']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000693](https://www.ebi.ac.uk/biomodels/BIOMD0000000693)<br/><sup>Wang2008 - Mimicking the inhib</sup></summary><sup>Wang2008 - Mimicking the inhibitory effect of riluzole on membrane conductance in skeletal fibres</sup></details>|<details><summary>pass</summary>[BIOMD0000000693_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000693#Files)</details>|FAIL|<details><summary>pass</summary>[MODEL7817907010.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000693#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL7817907010.sedml574481.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000693_url.xml` has warnings.\r\n - 35 warnings of type SBML unit consistency (99505). The following is the first warning at line 1101, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'gk_max * pow(n, 4) * (Vm - EK)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10512). The following is the first warning at line 1360, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <species>, the units of the rule's right-hand side are expected to be consistent with the units of the species' quantity.\r\n Reference: L2V4 Section 4.11.3\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3) but the units returned by the <math> expression of the <assignmentRule> with variable 'w' are dimensionless (exponent = 1, multiplier = 1, scale = 0).\r\n - 3 warnings of type SBML unit consistency (10532). The following is the first warning at line 1692, column 6:\r\n - When the 'variable' in a <rateRule> definition refers to a <species>, the units of the rule's right-hand side are expected to be of the form _x per time_, where _x_ is the units of that species' quantity, and _time_ refers to the units of time for the model.\r\n Reference: L2V4 Section 4.11.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = -3) but the units returned by the <math> expression of the <rateRule> with variable 'm' are mole (exponent = 2, multiplier = 0.001, scale = 0), litre (exponent = -2, multiplier = 0.001, scale = 0).\r\n - 55 warnings of type Modeling practice (80701). The following is the first warning at line 708, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'gNa_max' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL7817907010.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000693_url.xml` has warnings.\r\n - 35 warnings of type SBML unit consistency (99505). The following is the first warning at line 1101, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'gk_max * pow(n, 4) * (Vm - EK)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10512). The following is the first warning at line 1360, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <species>, the units of the rule's right-hand side are expected to be consistent with the units of the species' quantity.\r\n Reference: L2V4 Section 4.11.3\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3) but the units returned by the <math> expression of the <assignmentRule> with variable 'w' are dimensionless (exponent = 1, multiplier = 1, scale = 0).\r\n - 3 warnings of type SBML unit consistency (10532). The following is the first warning at line 1692, column 6:\r\n - When the 'variable' in a <rateRule> definition refers to a <species>, the units of the rule's right-hand side are expected to be of the form _x per time_, where _x_ is the units of that species' quantity, and _time_ refers to the units of time for the model.\r\n Reference: L2V4 Section 4.11.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = -3) but the units returned by the <math> expression of the <rateRule> with variable 'm' are mole (exponent = 2, multiplier = 0.001, scale = 0), litre (exponent = -2, multiplier = 0.001, scale = 0).\r\n - 55 warnings of type Modeling practice (80701). The following is the first warning at line 708, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'gNa_max' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/MODEL7817907010.sedml574481.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL7817907010.sedml` is invalid.\n - Data generator `Vm_1_task1` is invalid.\n - Variable `p1_Vm_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Vm']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL7817907010.sedml244374.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000693_url.xml` has warnings.\r\n - 35 warnings of type SBML unit consistency (99505). The following is the first warning at line 1101, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'gk_max * pow(n, 4) * (Vm - EK)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10512). The following is the first warning at line 1360, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <species>, the units of the rule's right-hand side are expected to be consistent with the units of the species' quantity.\r\n Reference: L2V4 Section 4.11.3\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3) but the units returned by the <math> expression of the <assignmentRule> with variable 'w' are dimensionless (exponent = 1, multiplier = 1, scale = 0).\r\n - 3 warnings of type SBML unit consistency (10532). The following is the first warning at line 1692, column 6:\r\n - When the 'variable' in a <rateRule> definition refers to a <species>, the units of the rule's right-hand side are expected to be of the form _x per time_, where _x_ is the units of that species' quantity, and _time_ refers to the units of time for the model.\r\n Reference: L2V4 Section 4.11.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = -3) but the units returned by the <math> expression of the <rateRule> with variable 'm' are mole (exponent = 2, multiplier = 0.001, scale = 0), litre (exponent = -2, multiplier = 0.001, scale = 0).\r\n - 55 warnings of type Modeling practice (80701). The following is the first warning at line 708, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'gNa_max' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL7817907010.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000693_url.xml` has warnings.\r\n - 35 warnings of type SBML unit consistency (99505). The following is the first warning at line 1101, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'gk_max * pow(n, 4) * (Vm - EK)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10512). The following is the first warning at line 1360, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <species>, the units of the rule's right-hand side are expected to be consistent with the units of the species' quantity.\r\n Reference: L2V4 Section 4.11.3\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3) but the units returned by the <math> expression of the <assignmentRule> with variable 'w' are dimensionless (exponent = 1, multiplier = 1, scale = 0).\r\n - 3 warnings of type SBML unit consistency (10532). The following is the first warning at line 1692, column 6:\r\n - When the 'variable' in a <rateRule> definition refers to a <species>, the units of the rule's right-hand side are expected to be of the form _x per time_, where _x_ is the units of that species' quantity, and _time_ refers to the units of time for the model.\r\n Reference: L2V4 Section 4.11.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = -3) but the units returned by the <math> expression of the <rateRule> with variable 'm' are mole (exponent = 2, multiplier = 0.001, scale = 0), litre (exponent = -2, multiplier = 0.001, scale = 0).\r\n - 55 warnings of type Modeling practice (80701). The following is the first warning at line 708, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'gNa_max' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL7817907010.sedml244374.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL7817907010.sedml` is invalid.\n - Data generator `Vm_1_task1` is invalid.\n - Variable `p1_Vm_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Vm']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000695](https://www.ebi.ac.uk/biomodels/BIOMD0000000695)<br/><sup>FelixGarza2017 - Blue Light Tr</sup></summary><sup>FelixGarza2017 - Blue Light Treatment of Psoriasis (simplified)</sup></details>|<details><summary>pass</summary>[BIOMD0000000695_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000695#Files)</details>|pass|<details><summary>pass</summary>[MODEL1701090001.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000695#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1701090001.sedml3810.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000695_url.xml` has warnings.\r\n - 66 warnings of type SBML unit consistency (99505). The following is the first warning at line 3731, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'rhoDe * alpha' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 55 warnings of type Modeling practice (80701). The following is the first warning at line 2331, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'AId' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1701090001.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000695_url.xml` has warnings.\r\n - 66 warnings of type SBML unit consistency (99505). The following is the first warning at line 3731, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'rhoDe * alpha' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 55 warnings of type Modeling practice (80701). The following is the first warning at line 2331, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'AId' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/MODEL1701090001.sedml3810.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1701090001.sedml` is invalid.\n - Data generator `xFinal_6_1_task1` is invalid.\n - Variable `p1_xFinal_6_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='xFinal_6']` are not defined. No namespaces are defined for the target.\n - Data generator `xFinal_5_2_task1` is invalid.\n - Variable `p1_xFinal_5_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='xFinal_5']` are not defined. No namespaces are defined for the target.\n - Data generator `xFinal_3_3_task1` is invalid.\n - Variable `p1_xFinal_3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='xFinal_3']` are not defined. No namespaces are defined for the target.\n - Data generator `xFinal_4_4_task1` is invalid.\n - Variable `p1_xFinal_4_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='xFinal_4']` are not defined. No namespaces are defined for the target.\n - Data generator `xFinal_1_5_task1` is invalid.\n - Variable `p1_xFinal_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='xFinal_1']` are not defined. No namespaces are defined for the target.\n - Data generator `xFinal_2_6_task1` is invalid.\n - Variable `p1_xFinal_2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='xFinal_2']` are not defined. No namespaces are defined for the target.\n - Data generator `xFinal_12_1_task1` is invalid.\n - Variable `p2_xFinal_12_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='xFinal_12']` are not defined. No namespaces are defined for the target.\n - Data generator `xFinal_11_2_task1` is invalid.\n - Variable `p2_xFinal_11_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='xFinal_11']` are not defined. No namespaces are defined for the target.\n - Data generator `xFinal_9_3_task1` is invalid.\n - Variable `p2_xFinal_9_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='xFinal_9']` are not defined. No namespaces are defined for the target.\n - Data generator `xFinal_10_4_task1` is invalid.\n - Variable `p2_xFinal_10_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='xFinal_10']` are not defined. No namespaces are defined for the target.\n - Data generator `xFinal_7_5_task1` is invalid.\n - Variable `p2_xFinal_7_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='xFinal_7']` are not defined. No namespaces are defined for the target.\n - Data generator `xFinal_8_6_task1` is invalid.\n - Variable `p2_xFinal_8_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='xFinal_8']` are not defined. No namespaces are defined for the target.\n - Data generator `doseBL_1_task1` is invalid.\n - Variable `p3_doseBL_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='doseBL']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1701090001.sedml3321.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000695_url.xml` has warnings.\r\n - 66 warnings of type SBML unit consistency (99505). The following is the first warning at line 3731, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'rhoDe * alpha' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 55 warnings of type Modeling practice (80701). The following is the first warning at line 2331, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'AId' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1701090001.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000695_url.xml` has warnings.\r\n - 66 warnings of type SBML unit consistency (99505). The following is the first warning at line 3731, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'rhoDe * alpha' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 55 warnings of type Modeling practice (80701). The following is the first warning at line 2331, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'AId' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1701090001.sedml3321.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1701090001.sedml` is invalid.\n - Data generator `xFinal_6_1_task1` is invalid.\n - Variable `p1_xFinal_6_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='xFinal_6']` are not defined. No namespaces are defined for the target.\n - Data generator `xFinal_5_2_task1` is invalid.\n - Variable `p1_xFinal_5_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='xFinal_5']` are not defined. No namespaces are defined for the target.\n - Data generator `xFinal_3_3_task1` is invalid.\n - Variable `p1_xFinal_3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='xFinal_3']` are not defined. No namespaces are defined for the target.\n - Data generator `xFinal_4_4_task1` is invalid.\n - Variable `p1_xFinal_4_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='xFinal_4']` are not defined. No namespaces are defined for the target.\n - Data generator `xFinal_1_5_task1` is invalid.\n - Variable `p1_xFinal_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='xFinal_1']` are not defined. No namespaces are defined for the target.\n - Data generator `xFinal_2_6_task1` is invalid.\n - Variable `p1_xFinal_2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='xFinal_2']` are not defined. No namespaces are defined for the target.\n - Data generator `xFinal_12_1_task1` is invalid.\n - Variable `p2_xFinal_12_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='xFinal_12']` are not defined. No namespaces are defined for the target.\n - Data generator `xFinal_11_2_task1` is invalid.\n - Variable `p2_xFinal_11_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='xFinal_11']` are not defined. No namespaces are defined for the target.\n - Data generator `xFinal_9_3_task1` is invalid.\n - Variable `p2_xFinal_9_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='xFinal_9']` are not defined. No namespaces are defined for the target.\n - Data generator `xFinal_10_4_task1` is invalid.\n - Variable `p2_xFinal_10_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='xFinal_10']` are not defined. No namespaces are defined for the target.\n - Data generator `xFinal_7_5_task1` is invalid.\n - Variable `p2_xFinal_7_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='xFinal_7']` are not defined. No namespaces are defined for the target.\n - Data generator `xFinal_8_6_task1` is invalid.\n - Variable `p2_xFinal_8_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='xFinal_8']` are not defined. No namespaces are defined for the target.\n - Data generator `doseBL_1_task1` is invalid.\n - Variable `p3_doseBL_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='doseBL']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000696](https://www.ebi.ac.uk/biomodels/BIOMD0000000696)<br/><sup>Boada2016 - Incoherent type 1 </sup></summary><sup>Boada2016 - Incoherent type 1 feed-forward loop (I1-FFL)</sup></details>|<details><summary>pass</summary>[BIOMD0000000696_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000696#Files)</details>|pass|<details><summary>pass</summary>[MODEL1511290000.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000696#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1511290000.sedml212116.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000696_url.xml` has warnings.\r\n - 34 warnings of type SBML unit consistency (99505). The following is the first warning at line 1967, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression '0.0432 * 104 / 0.3624' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 41 warnings of type Modeling practice (80701). The following is the first warning at line 864, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k_mA_C_gA' does not have a 'units' attribute.\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1511290000.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000696_url.xml` has warnings.\r\n - 34 warnings of type SBML unit consistency (99505). The following is the first warning at line 1967, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression '0.0432 * 104 / 0.3624' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 41 warnings of type Modeling practice (80701). The following is the first warning at line 864, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k_mA_C_gA' does not have a 'units' attribute.\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/MODEL1511290000.sedml212116.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1511290000.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='gamma_1']` are not defined. No namespaces are defined for the target.\n - Change 2 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='k_mC_C_gC']` are not defined. No namespaces are defined for the target.\n - Data generator `J1_1_task1` is invalid.\n - Variable `p1_J1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='J1']` are not defined. No namespaces are defined for the target.\n - Data generator `x8_1_task1` is invalid.\n - Variable `p2_x8_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x8']` are not defined. No namespaces are defined for the target.\n - Data generator `J1_1_task2` is invalid.\n - Variable `p1_J1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='J1']` are not defined. No namespaces are defined for the target.\n - Data generator `x8_1_task2` is invalid.\n - Variable `p2_x8_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x8']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1511290000.sedml620793.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000696_url.xml` has warnings.\r\n - 34 warnings of type SBML unit consistency (99505). The following is the first warning at line 1967, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression '0.0432 * 104 / 0.3624' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 41 warnings of type Modeling practice (80701). The following is the first warning at line 864, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k_mA_C_gA' does not have a 'units' attribute.\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1511290000.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000696_url.xml` has warnings.\r\n - 34 warnings of type SBML unit consistency (99505). The following is the first warning at line 1967, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression '0.0432 * 104 / 0.3624' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 41 warnings of type Modeling practice (80701). The following is the first warning at line 864, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k_mA_C_gA' does not have a 'units' attribute.\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1511290000.sedml620793.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1511290000.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='gamma_1']` are not defined. No namespaces are defined for the target.\n - Change 2 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='k_mC_C_gC']` are not defined. No namespaces are defined for the target.\n - Data generator `J1_1_task1` is invalid.\n - Variable `p1_J1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='J1']` are not defined. No namespaces are defined for the target.\n - Data generator `x8_1_task1` is invalid.\n - Variable `p2_x8_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x8']` are not defined. No namespaces are defined for the target.\n - Data generator `J1_1_task2` is invalid.\n - Variable `p1_J1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='J1']` are not defined. No namespaces are defined for the target.\n - Data generator `x8_1_task2` is invalid.\n - Variable `p2_x8_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x8']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000697](https://www.ebi.ac.uk/biomodels/BIOMD0000000697)<br/><sup>Ciliberto2003 - CyclinE / Cdk2</sup></summary><sup>Ciliberto2003 - CyclinE / Cdk2 timer in the cell cycle of Xenopus laevis embryo</sup></details>|<details><summary>pass</summary>[BIOMD0000000697_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000697#Files)</details>|FAIL|<details><summary>MultipleSEDMLs</summary>['MODEL0913242281_fig6.sedml', 'MODEL0913242281_fig5.sedml']</details>|NA|NA|NA|
|[BIOMD0000000698](https://www.ebi.ac.uk/biomodels/BIOMD0000000698)<br/><sup>Reed2004 - Methionine Cycle</sup>|<details><summary>pass</summary>[BIOMD0000000698_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000698#Files)</details>|pass|<details><summary>pass</summary>[MODEL1006230091.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000698#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1006230091.sedml661779.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000698_url.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 999, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(200, or(lt(time, 2), geq(time, 8)), piecewise(300, and(geq(time, 2), lt(time, 5)), piecewise(100, and(geq(time, 5), lt(time, 8)), 200)))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 23 warnings of type Modeling practice (80701). The following is the first warning at line 905, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'V_max_MATI' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1006230091.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000698_url.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 999, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(200, or(lt(time, 2), geq(time, 8)), piecewise(300, and(geq(time, 2), lt(time, 5)), piecewise(100, and(geq(time, 5), lt(time, 8)), 200)))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 23 warnings of type Modeling practice (80701). The following is the first warning at line 905, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'V_max_MATI' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/MODEL1006230091.sedml661779.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1006230091.sedml` is invalid.\n - Data generator `AdoMet_1_task1` is invalid.\n - Variable `p1_AdoMet_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='AdoMet']` are not defined. No namespaces are defined for the target.\n - Data generator `Homocysteine_2_task1` is invalid.\n - Variable `p1_Homocysteine_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Homocysteine']` are not defined. No namespaces are defined for the target.\n - Data generator `Methionine_3_task1` is invalid.\n - Variable `p1_Methionine_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Methionine']` are not defined. No namespaces are defined for the target.\n - Data generator `Metin_4_task1` is invalid.\n - Variable `p1_Metin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Metin']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1006230091.sedml824137.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000698_url.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 999, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(200, or(lt(time, 2), geq(time, 8)), piecewise(300, and(geq(time, 2), lt(time, 5)), piecewise(100, and(geq(time, 5), lt(time, 8)), 200)))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 23 warnings of type Modeling practice (80701). The following is the first warning at line 905, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'V_max_MATI' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1006230091.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000698_url.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 999, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(200, or(lt(time, 2), geq(time, 8)), piecewise(300, and(geq(time, 2), lt(time, 5)), piecewise(100, and(geq(time, 5), lt(time, 8)), 200)))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 23 warnings of type Modeling practice (80701). The following is the first warning at line 905, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'V_max_MATI' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1006230091.sedml824137.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1006230091.sedml` is invalid.\n - Data generator `AdoMet_1_task1` is invalid.\n - Variable `p1_AdoMet_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='AdoMet']` are not defined. No namespaces are defined for the target.\n - Data generator `Homocysteine_2_task1` is invalid.\n - Variable `p1_Homocysteine_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Homocysteine']` are not defined. No namespaces are defined for the target.\n - Data generator `Methionine_3_task1` is invalid.\n - Variable `p1_Methionine_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Methionine']` are not defined. No namespaces are defined for the target.\n - Data generator `Metin_4_task1` is invalid.\n - Variable `p1_Metin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Metin']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000699](https://www.ebi.ac.uk/biomodels/BIOMD0000000699)<br/><sup>Caydasi2012 - Regulation of Te</sup></summary><sup>Caydasi2012 - Regulation of Tem1 by the GAP complex in spindle position cell cycle checkpoint - Ubiquitous association model</sup></details>|<details><summary>pass</summary>[BIOMD0000000699_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000699#Files)</details>|FAIL|<details><summary>FAIL</summary>[MODEL1202090002_Fig5 i.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000699#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/MODEL1202090002_Fig5 i.sedml275758.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 2: b'usage: biosimulators-tellurium [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-tellurium: error: unrecognized arguments: i.sedml275758.omex\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1202090002_Fig5 i.sedml744405.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 2: b'usage: biosimulators-copasi [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-copasi: error: unrecognized arguments: i.sedml744405.omex\n'```</details>|
|<details><summary>[BIOMD0000000700](https://www.ebi.ac.uk/biomodels/BIOMD0000000700)<br/><sup>Heldt2018 - Proliferation-quie</sup></summary><sup>Heldt2018 - Proliferation-quiescence decision in response to DNA damage</sup></details>|<details><summary>pass</summary>[BIOMD0000000700_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000700#Files)</details>|pass|<details><summary>pass</summary>[figure1.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000700#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/figure1.sedml538435.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000700_url.xml` has warnings.\r\n - 49 warnings of type SBML unit consistency (99505). The following is the first warning at line 3788, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Cell * rPhRb_1(Ca, Cd, Ce, Rb, kPhRbCa, kPhRbCd, kPhRbCe)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 56 warnings of type Modeling practice (80701). The following is the first warning at line 2556, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Cd' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./figure1.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000700_url.xml` has warnings.\r\n - 49 warnings of type SBML unit consistency (99505). The following is the first warning at line 3788, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Cell * rPhRb_1(Ca, Cd, Ce, Rb, kPhRbCa, kPhRbCd, kPhRbCe)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 56 warnings of type Modeling practice (80701). The following is the first warning at line 2556, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Cd' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/figure1.sedml538435.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./figure1.sedml` is invalid.\n - Data generator `tCa_1_task1` is invalid.\n - Variable `p1_tCa_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='tCa']` are not defined. No namespaces are defined for the target.\n - Data generator `tCe_2_task1` is invalid.\n - Variable `p1_tCe_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='tCe']` are not defined. No namespaces are defined for the target.\n - Data generator `tP21_3_task1` is invalid.\n - Variable `p1_tP21_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='tP21']` are not defined. No namespaces are defined for the target.\n - Data generator `aRc_4_task1` is invalid.\n - Variable `p1_aRc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='aRc']` are not defined. No namespaces are defined for the target.\n - Data generator `C1_1_task1` is invalid.\n - Variable `p2_C1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C1']` are not defined. No namespaces are defined for the target.\n - Data generator `tE1_2_task1` is invalid.\n - Variable `p2_tE1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='tE1']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/figure1.sedml709358.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000700_url.xml` has warnings.\r\n - 49 warnings of type SBML unit consistency (99505). The following is the first warning at line 3788, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Cell * rPhRb_1(Ca, Cd, Ce, Rb, kPhRbCa, kPhRbCd, kPhRbCe)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 56 warnings of type Modeling practice (80701). The following is the first warning at line 2556, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Cd' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./figure1.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000700_url.xml` has warnings.\r\n - 49 warnings of type SBML unit consistency (99505). The following is the first warning at line 3788, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Cell * rPhRb_1(Ca, Cd, Ce, Rb, kPhRbCa, kPhRbCd, kPhRbCe)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 56 warnings of type Modeling practice (80701). The following is the first warning at line 2556, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Cd' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/figure1.sedml709358.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./figure1.sedml` is invalid.\n - Data generator `tCa_1_task1` is invalid.\n - Variable `p1_tCa_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='tCa']` are not defined. No namespaces are defined for the target.\n - Data generator `tCe_2_task1` is invalid.\n - Variable `p1_tCe_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='tCe']` are not defined. No namespaces are defined for the target.\n - Data generator `tP21_3_task1` is invalid.\n - Variable `p1_tP21_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='tP21']` are not defined. No namespaces are defined for the target.\n - Data generator `aRc_4_task1` is invalid.\n - Variable `p1_aRc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='aRc']` are not defined. No namespaces are defined for the target.\n - Data generator `C1_1_task1` is invalid.\n - Variable `p2_C1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C1']` are not defined. No namespaces are defined for the target.\n - Data generator `tE1_2_task1` is invalid.\n - Variable `p2_tE1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='tE1']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000701](https://www.ebi.ac.uk/biomodels/BIOMD0000000701)<br/><sup>Caydasi2012 - Inhibition of Te</sup></summary><sup>Caydasi2012 - Inhibition of Tem1 by the GAP complex in Spindle Position Checkpoint</sup></details>|<details><summary>pass</summary>[MODEL1202090001_FINAL.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000701#Files)</details>|FAIL|<details><summary>FAIL</summary>[MODEL1202090001_FINAL.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000701#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/MODEL1202090001_FINAL.sedml144313.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/MODEL1202090001_FINAL.sedml144313.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1202090001_FINAL.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmp1rnttsdt/./model1.xml` is not a file.\n - Data generator `Active_Tem1_in_the_Cytosol_1_task1` is invalid.\n - Variable `p1_Active_Tem1_in_the_Cytosol_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Active_Tem1_in_the_Cytosol']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Tem1_in_the_Cytosol_2_task1` is invalid.\n - Variable `p1_Total_Tem1_in_the_Cytosol_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Total_Tem1_in_the_Cytosol']` are not defined. No namespaces are defined for the target.\n - Data generator `Active_Bfa1_at_the_Cytosol_1_task1` is invalid.\n - Variable `p2_Active_Bfa1_at_the_Cytosol_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Active_Bfa1_at_the_Cytosol']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Bfa1_in_the_Cytosol_2_task1` is invalid.\n - Variable `p2_Total_Bfa1_in_the_Cytosol_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Total_Bfa1_in_the_Cytosol']` are not defined. No namespaces are defined for the target.\n - Data generator `Active_Bfa1_at_the_SPB_1_task1` is invalid.\n - Variable `p3_Active_Bfa1_at_the_SPB_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Active_Bfa1_at_the_SPB']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Bfa1_at_the_SPB_2_task1` is invalid.\n - Variable `p3_Total_Bfa1_at_the_SPB_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Total_Bfa1_at_the_SPB']` are not defined. No namespaces are defined for the target.\n - Data generator `Active_Tem1_at_the_SPB_1_task1` is invalid.\n - Variable `p4_Active_Tem1_at_the_SPB_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Active_Tem1_at_the_SPB']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Tem1_at_the_SPB_2_task1` is invalid.\n - Variable `p4_Total_Tem1_at_the_SPB_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Total_Tem1_at_the_SPB']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1202090001_FINAL.sedml454642.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1202090001_FINAL.sedml454642.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1202090001_FINAL.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmp3sfyjbjp/./model1.xml` is not a file.\n - Data generator `Active_Tem1_in_the_Cytosol_1_task1` is invalid.\n - Variable `p1_Active_Tem1_in_the_Cytosol_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Active_Tem1_in_the_Cytosol']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Tem1_in_the_Cytosol_2_task1` is invalid.\n - Variable `p1_Total_Tem1_in_the_Cytosol_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Total_Tem1_in_the_Cytosol']` are not defined. No namespaces are defined for the target.\n - Data generator `Active_Bfa1_at_the_Cytosol_1_task1` is invalid.\n - Variable `p2_Active_Bfa1_at_the_Cytosol_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Active_Bfa1_at_the_Cytosol']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Bfa1_in_the_Cytosol_2_task1` is invalid.\n - Variable `p2_Total_Bfa1_in_the_Cytosol_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Total_Bfa1_in_the_Cytosol']` are not defined. No namespaces are defined for the target.\n - Data generator `Active_Bfa1_at_the_SPB_1_task1` is invalid.\n - Variable `p3_Active_Bfa1_at_the_SPB_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Active_Bfa1_at_the_SPB']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Bfa1_at_the_SPB_2_task1` is invalid.\n - Variable `p3_Total_Bfa1_at_the_SPB_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Total_Bfa1_at_the_SPB']` are not defined. No namespaces are defined for the target.\n - Data generator `Active_Tem1_at_the_SPB_1_task1` is invalid.\n - Variable `p4_Active_Tem1_at_the_SPB_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Active_Tem1_at_the_SPB']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Tem1_at_the_SPB_2_task1` is invalid.\n - Variable `p4_Total_Tem1_at_the_SPB_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Total_Tem1_at_the_SPB']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000702](https://www.ebi.ac.uk/biomodels/BIOMD0000000702)<br/><sup>Caydasi2012 - Regulation of Te</sup></summary><sup>Caydasi2012 - Regulation of Tem1 by the GAP complex in Spindle Position Checkpoint - Ubiquitous inactive model</sup></details>|<details><summary>pass</summary>[BIOMD0000000702_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000702#Files)</details>|FAIL|<details><summary>pass</summary>[MODEL1202090003_Fig5 iii.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000702#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1202090003_Fig5 iii.sedml557921.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 2: b'usage: biosimulators-tellurium [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-tellurium: error: unrecognized arguments: iii.sedml557921.omex\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1202090003_Fig5 iii.sedml495793.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 2: b'usage: biosimulators-copasi [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-copasi: error: unrecognized arguments: iii.sedml495793.omex\n'```</details>|
|<details><summary>[BIOMD0000000703](https://www.ebi.ac.uk/biomodels/BIOMD0000000703)<br/><sup>Diedrichs2018 - A data-entrain</sup></summary><sup>Diedrichs2018 - A data-entrained computational model for testing the regulatory logic of the vertebrate unfolded protein response</sup></details>|<details><summary>pass</summary>[BIOMD0000000703_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000703#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|[BIOMD0000000704](https://www.ebi.ac.uk/biomodels/BIOMD0000000704)<br/><sup>Aguda1999 - G2 DNA damage checkpoint</sup>|<details><summary>pass</summary>[BIOMD0000000704_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000704#Files)</details>|pass|<details><summary>pass</summary>[figure8.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000704#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/figure8.sedml579885.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000704_url.xml` has warnings.\r\n - 33 warnings of type SBML unit consistency (99505). The following is the first warning at line 1726, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'nucleus * function_for_R18(k2, Chk1P, Cdc25_active, k_ctak1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 40 warnings of type Modeling practice (80701). The following is the first warning at line 1076, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./figure8.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000704_url.xml` has warnings.\r\n - 33 warnings of type SBML unit consistency (99505). The following is the first warning at line 1726, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'nucleus * function_for_R18(k2, Chk1P, Cdc25_active, k_ctak1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 40 warnings of type Modeling practice (80701). The following is the first warning at line 1076, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/figure8.sedml579885.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./figure8.sedml` is invalid.\n - Data generator `MPF_1_task1` is invalid.\n - Variable `p1_MPF_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='MPF']` are not defined. No namespaces are defined for the target.\n - Data generator `Wee1_2_task1` is invalid.\n - Variable `p1_Wee1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Wee1']` are not defined. No namespaces are defined for the target.\n - Data generator `Cdc25_active_total_3_task1` is invalid.\n - Variable `p1_Cdc25_active_total_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cdc25_active_total']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/figure8.sedml764609.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000704_url.xml` has warnings.\r\n - 33 warnings of type SBML unit consistency (99505). The following is the first warning at line 1726, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'nucleus * function_for_R18(k2, Chk1P, Cdc25_active, k_ctak1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 40 warnings of type Modeling practice (80701). The following is the first warning at line 1076, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./figure8.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000704_url.xml` has warnings.\r\n - 33 warnings of type SBML unit consistency (99505). The following is the first warning at line 1726, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'nucleus * function_for_R18(k2, Chk1P, Cdc25_active, k_ctak1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 40 warnings of type Modeling practice (80701). The following is the first warning at line 1076, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/figure8.sedml764609.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./figure8.sedml` is invalid.\n - Data generator `MPF_1_task1` is invalid.\n - Variable `p1_MPF_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='MPF']` are not defined. No namespaces are defined for the target.\n - Data generator `Wee1_2_task1` is invalid.\n - Variable `p1_Wee1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Wee1']` are not defined. No namespaces are defined for the target.\n - Data generator `Cdc25_active_total_3_task1` is invalid.\n - Variable `p1_Cdc25_active_total_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cdc25_active_total']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000705](https://www.ebi.ac.uk/biomodels/BIOMD0000000705)<br/><sup>Smith2010 - Response of FOXO T</sup></summary><sup>Smith2010 - Response of FOXO Transcription Factors to Post-Translational Modifications Made by Ageing-Related Signalling Pathways</sup></details>|<details><summary>pass</summary>[BIOMD0000000705_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000705#Files)</details>|pass|<details><summary>MultipleSEDMLs</summary>['Smith2010_figure_4C_red_curve.sedml', 'Smith2010_figure_4C_blue_curve.sedml']</details>|NA|NA|NA|
|<details><summary>[BIOMD0000000706](https://www.ebi.ac.uk/biomodels/BIOMD0000000706)<br/><sup>Smith2010 - Response of FOXO T</sup></summary><sup>Smith2010 - Response of FOXO Transcription Factors to Post-Translational Modifications (with acetylation pathway)</sup></details>|<details><summary>pass</summary>[BIOMD0000000706_url.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000706#Files)</details>|pass|<details><summary>MultipleSEDMLs</summary>['Smith2010_figure_4D_red_curve.sedml', 'Smith2010_figure_4D_blue_curve.sedml']</details>|NA|NA|NA|
|<details><summary>[BIOMD0000000707](https://www.ebi.ac.uk/biomodels/BIOMD0000000707)<br/><sup>Revilla2003 - Controlling HIV </sup></summary><sup>Revilla2003 - Controlling HIV infection using recombinant viruses</sup></details>|<details><summary>pass</summary>[MODEL1006230047.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000707#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000708](https://www.ebi.ac.uk/biomodels/BIOMD0000000708)<br/><sup>Liu2017 - Dynamics of Avian In</sup></summary><sup>Liu2017 - Dynamics of Avian Influenza with Logistic Growth</sup></details>|<details><summary>pass</summary>[Liu2017 - Dynamics of Avian Influenza with Logistic Growth.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000708#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000709](https://www.ebi.ac.uk/biomodels/BIOMD0000000709)<br/><sup>Liu2017 - Dynamics of Avian In</sup></summary><sup>Liu2017 - Dynamics of Avian Influenza with Allee Growth Effect</sup></details>|<details><summary>pass</summary>[Liu2017 - Dynamics of Avian Influenza with Allee Growth Effect.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000709#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000710](https://www.ebi.ac.uk/biomodels/BIOMD0000000710)<br/><sup>Hernandez-Vargas2012 - Innate </sup></summary><sup>Hernandez-Vargas2012 - Innate immune system dynamics to Influenza virus</sup></details>|<details><summary>pass</summary>[Hernandez-Vargas2012 - Innate immune system dynamics to Influenza virus.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000710#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000711](https://www.ebi.ac.uk/biomodels/BIOMD0000000711)<br/><sup>Hancioglu2007 - Human Immune R</sup></summary><sup>Hancioglu2007 - Human Immune Response to Influenza A virus Infection</sup></details>|<details><summary>pass</summary>[Hancioglu2007 - Human Immune Response to Influenza A virus.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000711#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000712](https://www.ebi.ac.uk/biomodels/BIOMD0000000712)<br/><sup>Manchanda2014 - Effect on Immu</sup></summary><sup>Manchanda2014 - Effect on Immune System by 4 different Influenza A virus strains</sup></details>|<details><summary>pass</summary>[Jena5258.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000712#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000713](https://www.ebi.ac.uk/biomodels/BIOMD0000000713)<br/><sup>Aston2018 - Dynamics of Hepati</sup></summary><sup>Aston2018 - Dynamics of Hepatitis C Infection</sup></details>|<details><summary>pass</summary>[PVR.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000713#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000714](https://www.ebi.ac.uk/biomodels/BIOMD0000000714)<br/><sup>Reynolds2006 - Reduced model o</sup></summary><sup>Reynolds2006 - Reduced model of the acute inflammatory response</sup></details>|<details><summary>pass</summary>[HealthyOutcome.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000714#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000715](https://www.ebi.ac.uk/biomodels/BIOMD0000000715)<br/><sup>Huo2017 - SEIS epidemic model </sup></summary><sup>Huo2017 - SEIS epidemic model with the impact of media</sup></details>|<details><summary>pass</summary>[Epidemic Model with the impact of Media Reactions.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000715#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000716](https://www.ebi.ac.uk/biomodels/BIOMD0000000716)<br/><sup>Lee2018 - Avian human bilinear</sup></summary><sup>Lee2018 - Avian human bilinear incidence (BI) model</sup></details>|<details><summary>pass</summary>[BI reactions.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000716#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000717](https://www.ebi.ac.uk/biomodels/BIOMD0000000717)<br/><sup>Lee2018 - Avian human half-sat</sup></summary><sup>Lee2018 - Avian human half-saturated incidence (HSI) model</sup></details>|<details><summary>pass</summary>[HSI reactions.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000717#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000718](https://www.ebi.ac.uk/biomodels/BIOMD0000000718)<br/><sup>Li2008 - Caulobacter Cell Cycle</sup>|<details><summary>pass</summary>[Li2008.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000718#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000719](https://www.ebi.ac.uk/biomodels/BIOMD0000000719)<br/><sup>Tsai2014 - Cell cycle duration</sup></summary><sup>Tsai2014 - Cell cycle duration control by oscillatory Dynamics  in Early Xenopus laevis Embryos</sup></details>|<details><summary>pass</summary>[Tsai2014.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000719#Files)</details>|pass|<details><summary>pass</summary>[tsai2014.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000719#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/tsai2014.sedml360601.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/tsai2014.sedml360601.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./tsai2014.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - YDataReference from Curve `p1_curve_1_task1` cannot be resolved to a data generator with id `r_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p6_curve_1_task1` cannot be resolved to a data generator with id `r_1_task1` because multiple data generators have this id.\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/tsai2014.sedml408732.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/tsai2014.sedml408732.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./tsai2014.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - YDataReference from Curve `p1_curve_1_task1` cannot be resolved to a data generator with id `r_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p6_curve_1_task1` cannot be resolved to a data generator with id `r_1_task1` because multiple data generators have this id.\n'```</details>|
|<details><summary>[BIOMD0000000720](https://www.ebi.ac.uk/biomodels/BIOMD0000000720)<br/><sup>Yan2012 - Rb-E2F pathway dynam</sup></summary><sup>Yan2012 - Rb-E2F pathway dynamics with miR449</sup></details>|<details><summary>pass</summary>[Yan2012.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000720#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000721](https://www.ebi.ac.uk/biomodels/BIOMD0000000721)<br/><sup>Graham2013 - Role of osteocyte</sup></summary><sup>Graham2013 - Role of osteocytes in targeted bone remodeling</sup></details>|<details><summary>pass</summary>[Model.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000721#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000722](https://www.ebi.ac.uk/biomodels/BIOMD0000000722)<br/><sup>Bianchi2015 -Model for lymphan</sup></summary><sup>Bianchi2015 -Model for lymphangiogenesis in normal and diabetic wounds</sup></details>|<details><summary>pass</summary>[Wound Healing with Normal health condition.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000722#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000723](https://www.ebi.ac.uk/biomodels/BIOMD0000000723)<br/><sup>Weis2014 - Data driven Mammali</sup></summary><sup>Weis2014 - Data driven Mammalian Cell Cycle Model</sup></details>|<details><summary>pass</summary>[Weis2014.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000723#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000724](https://www.ebi.ac.uk/biomodels/BIOMD0000000724)<br/><sup>Theinmozhi2018 - Mechanism of </sup></summary><sup>Theinmozhi2018 - Mechanism of PD1 inhibiting TCR signaling in Tumor immune regulation</sup></details>|<details><summary>pass</summary>[Theinmozhi_2018.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000724#Files)</details>|pass|<details><summary>pass</summary>[Theinmozhi_2018.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000724#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Theinmozhi_2018.sedml743126.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Theinmozhi_2018.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (99505). The following is the first warning at line 2492, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'LCK_switch' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 15 warnings of type Modeling practice (80701). The following is the first warning at line 1870, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k' does not have a 'units' attribute.\r\n - Output `plot_7_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_8_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_9_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_10_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_11_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_12_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Theinmozhi_2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Theinmozhi_2018.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (99505). The following is the first warning at line 2492, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'LCK_switch' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 15 warnings of type Modeling practice (80701). The following is the first warning at line 1870, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k' does not have a 'units' attribute.\r\n - Output `plot_7_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_8_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_9_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_10_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_11_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_12_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/Theinmozhi_2018.sedml743126.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Theinmozhi_2018.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='LCK_switch']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_PI3K_free_1_task1` is invalid.\n - Variable `p1_perc_PI3K_free_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_PI3K_free']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_ZAP70_free_1_task1` is invalid.\n - Variable `p2_perc_ZAP70_free_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_ZAP70_free']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_LCK_active_1_task1` is invalid.\n - Variable `p3_perc_LCK_active_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_LCK_active']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_LCK_inactive_2_task1` is invalid.\n - Variable `p3_perc_LCK_inactive_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_LCK_inactive']` are not defined. No namespaces are defined for the target.\n - Data generator `SLP76a_1_task1` is invalid.\n - Variable `p4_SLP76a_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SLP76a']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70a2_1_task1` is invalid.\n - Variable `p5_ZAP70a2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ZAP70a2']` are not defined. No namespaces are defined for the target.\n - Data generator `CD28_phosphorylated_1_task1` is invalid.\n - Variable `p6_CD28_phosphorylated_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='CD28_phosphorylated']` are not defined. No namespaces are defined for the target.\n - Data generator `CD3z_phosphorylated_2_task1` is invalid.\n - Variable `p6_CD3z_phosphorylated_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='CD3z_phosphorylated']` are not defined. No namespaces are defined for the target.\n - Data generator `Kdpa_pi_3_task1` is invalid.\n - Variable `p6_Kdpa_pi_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Kdpa_pi']` are not defined. No namespaces are defined for the target.\n - Data generator `Kdpa_ya_4_task1` is invalid.\n - Variable `p6_Kdpa_ya_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Kdpa_ya']` are not defined. No namespaces are defined for the target.\n - Data generator `Kdpa_yiya_5_task1` is invalid.\n - Variable `p6_Kdpa_yiya_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Kdpa_yiya']` are not defined. No namespaces are defined for the target.\n - Data generator `Kdpi_yi_6_task1` is invalid.\n - Variable `p6_Kdpi_yi_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Kdpi_yi']` are not defined. No namespaces are defined for the target.\n - Data generator `Kdpi_yiya_7_task1` is invalid.\n - Variable `p6_Kdpi_yiya_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Kdpi_yiya']` are not defined. No namespaces are defined for the target.\n - Data generator `LAT_phosphorylated_8_task1` is invalid.\n - Variable `p6_LAT_phosphorylated_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='LAT_phosphorylated']` are not defined. No namespaces are defined for the target.\n - Data generator `LCK_phos_Y394_9_task1` is invalid.\n - Variable `p6_LCK_phos_Y394_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='LCK_phos_Y394']` are not defined. No namespaces are defined for the target.\n - Data generator `LCK_phos_Y505_10_task1` is invalid.\n - Variable `p6_LCK_phos_Y505_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='LCK_phos_Y505']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70_phos_Y315_11_task1` is invalid.\n - Variable `p6_ZAP70_phos_Y315_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='ZAP70_phos_Y315']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70_phos_Y93_12_task1` is invalid.\n - Variable `p6_ZAP70_phos_Y93_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='ZAP70_phos_Y93']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_LCK_active_13_task1` is invalid.\n - Variable `p6_perc_LCK_active_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_LCK_active']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_LCK_inactive_14_task1` is invalid.\n - Variable `p6_perc_LCK_inactive_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_LCK_inactive']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_PI3K_free_15_task1` is invalid.\n - Variable `p6_perc_PI3K_free_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_PI3K_free']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_SHP2_bound_16_task1` is invalid.\n - Variable `p6_perc_SHP2_bound_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_SHP2_bound']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_SHP2_free_17_task1` is invalid.\n - Variable `p6_perc_SHP2_free_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_SHP2_free']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_ZAP70_free_18_task1` is invalid.\n - Variable `p6_perc_ZAP70_free_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_ZAP70_free']` are not defined. No namespaces are defined for the target.\n - Data generator `CD28a_19_task1` is invalid.\n - Variable `p6_CD28a_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CD28a']` are not defined. No namespaces are defined for the target.\n - Data generator `CD28i_20_task1` is invalid.\n - Variable `p6_CD28i_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CD28i']` are not defined. No namespaces are defined for the target.\n - Data generator `CD28t_21_task1` is invalid.\n - Variable `p6_CD28t_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CD28t']` are not defined. No namespaces are defined for the target.\n - Data generator `CD3a_22_task1` is invalid.\n - Variable `p6_CD3a_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CD3a']` are not defined. No namespaces are defined for the target.\n - Data generator `CD3i_23_task1` is invalid.\n - Variable `p6_CD3i_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CD3i']` are not defined. No namespaces are defined for the target.\n - Data generator `CD3t_24_task1` is invalid.\n - Variable `p6_CD3t_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CD3t']` are not defined. No namespaces are defined for the target.\n - Data generator `CP1_25_task1` is invalid.\n - Variable `p6_CP1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CP1']` are not defined. No namespaces are defined for the target.\n - Data generator `CP2_26_task1` is invalid.\n - Variable `p6_CP2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CP2']` are not defined. No namespaces are defined for the target.\n - Data generator `CPactive_27_task1` is invalid.\n - Variable `p6_CPactive_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CPactive']` are not defined. No namespaces are defined for the target.\n - Data generator `GADS_28_task1` is invalid.\n - Variable `p6_GADS_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='GADS']` are not defined. No namespaces are defined for the target.\n - Data generator `GADSa_29_task1` is invalid.\n - Variable `p6_GADSa_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='GADSa']` are not defined. No namespaces are defined for the target.\n - Data generator `GADSt_30_task1` is invalid.\n - Variable `p6_GADSt_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='GADSt']` are not defined. No namespaces are defined for the target.\n - Data generator `LATa_31_task1` is invalid.\n - Variable `p6_LATa_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LATa']` are not defined. No namespaces are defined for the target.\n - Data generator `LATi_32_task1` is invalid.\n - Variable `p6_LATi_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LATi']` are not defined. No namespaces are defined for the target.\n - Data generator `LATt_33_task1` is invalid.\n - Variable `p6_LATt_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LATt']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKactive_34_task1` is invalid.\n - Variable `p6_LCKactive_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKactive']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKi_35_task1` is invalid.\n - Variable `p6_LCKi_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKi']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKinactive_36_task1` is invalid.\n - Variable `p6_LCKinactive_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKinactive']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKpi_37_task1` is invalid.\n - Variable `p6_LCKpi_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKpi']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKt_38_task1` is invalid.\n - Variable `p6_LCKt_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKt']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKya_39_task1` is invalid.\n - Variable `p6_LCKya_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKya']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKyi_40_task1` is invalid.\n - Variable `p6_LCKyi_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKyi']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKyiya_41_task1` is invalid.\n - Variable `p6_LCKyiya_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKyiya']` are not defined. No namespaces are defined for the target.\n - Data generator `PD1_42_task1` is invalid.\n - Variable `p6_PD1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PD1']` are not defined. No namespaces are defined for the target.\n - Data generator `PD1p1_43_task1` is invalid.\n - Variable `p6_PD1p1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PD1p1']` are not defined. No namespaces are defined for the target.\n - Data generator `PD1p2_44_task1` is invalid.\n - Variable `p6_PD1p2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PD1p2']` are not defined. No namespaces are defined for the target.\n - Data generator `PI3K_45_task1` is invalid.\n - Variable `p6_PI3K_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PI3K']` are not defined. No namespaces are defined for the target.\n - Data generator `PI3Kb_46_task1` is invalid.\n - Variable `p6_PI3Kb_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PI3Kb']` are not defined. No namespaces are defined for the target.\n - Data generator `PI3Kt_47_task1` is invalid.\n - Variable `p6_PI3Kt_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PI3Kt']` are not defined. No namespaces are defined for the target.\n - Data generator `SHP2_48_task1` is invalid.\n - Variable `p6_SHP2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SHP2']` are not defined. No namespaces are defined for the target.\n - Data generator `SLP76_49_task1` is invalid.\n - Variable `p6_SLP76_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SLP76']` are not defined. No namespaces are defined for the target.\n - Data generator `SLP76a_50_task1` is invalid.\n - Variable `p6_SLP76a_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SLP76a']` are not defined. No namespaces are defined for the target.\n - Data generator `SLP76i_51_task1` is invalid.\n - Variable `p6_SLP76i_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SLP76i']` are not defined. No namespaces are defined for the target.\n - Data generator `SLP76t_52_task1` is invalid.\n - Variable `p6_SLP76t_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SLP76t']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70_53_task1` is invalid.\n - Variable `p6_ZAP70_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ZAP70']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70a1_54_task1` is invalid.\n - Variable `p6_ZAP70a1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ZAP70a1']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70a2_55_task1` is invalid.\n - Variable `p6_ZAP70a2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ZAP70a2']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70i_56_task1` is invalid.\n - Variable `p6_ZAP70i_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ZAP70i']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70t_57_task1` is invalid.\n - Variable `p6_ZAP70t_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ZAP70t']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_PI3K_free_1_task2` is invalid.\n - Variable `p1_perc_PI3K_free_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_PI3K_free']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_ZAP70_free_1_task2` is invalid.\n - Variable `p2_perc_ZAP70_free_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_ZAP70_free']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_LCK_active_1_task2` is invalid.\n - Variable `p3_perc_LCK_active_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_LCK_active']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_LCK_inactive_2_task2` is invalid.\n - Variable `p3_perc_LCK_inactive_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_LCK_inactive']` are not defined. No namespaces are defined for the target.\n - Data generator `SLP76a_1_task2` is invalid.\n - Variable `p4_SLP76a_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SLP76a']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70a2_1_task2` is invalid.\n - Variable `p5_ZAP70a2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ZAP70a2']` are not defined. No namespaces are defined for the target.\n - Data generator `CD28_phosphorylated_1_task2` is invalid.\n - Variable `p6_CD28_phosphorylated_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='CD28_phosphorylated']` are not defined. No namespaces are defined for the target.\n - Data generator `CD3z_phosphorylated_2_task2` is invalid.\n - Variable `p6_CD3z_phosphorylated_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='CD3z_phosphorylated']` are not defined. No namespaces are defined for the target.\n - Data generator `Kdpa_pi_3_task2` is invalid.\n - Variable `p6_Kdpa_pi_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Kdpa_pi']` are not defined. No namespaces are defined for the target.\n - Data generator `Kdpa_ya_4_task2` is invalid.\n - Variable `p6_Kdpa_ya_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Kdpa_ya']` are not defined. No namespaces are defined for the target.\n - Data generator `Kdpa_yiya_5_task2` is invalid.\n - Variable `p6_Kdpa_yiya_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Kdpa_yiya']` are not defined. No namespaces are defined for the target.\n - Data generator `Kdpi_yi_6_task2` is invalid.\n - Variable `p6_Kdpi_yi_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Kdpi_yi']` are not defined. No namespaces are defined for the target.\n - Data generator `Kdpi_yiya_7_task2` is invalid.\n - Variable `p6_Kdpi_yiya_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Kdpi_yiya']` are not defined. No namespaces are defined for the target.\n - Data generator `LAT_phosphorylated_8_task2` is invalid.\n - Variable `p6_LAT_phosphorylated_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='LAT_phosphorylated']` are not defined. No namespaces are defined for the target.\n - Data generator `LCK_phos_Y394_9_task2` is invalid.\n - Variable `p6_LCK_phos_Y394_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='LCK_phos_Y394']` are not defined. No namespaces are defined for the target.\n - Data generator `LCK_phos_Y505_10_task2` is invalid.\n - Variable `p6_LCK_phos_Y505_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='LCK_phos_Y505']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70_phos_Y315_11_task2` is invalid.\n - Variable `p6_ZAP70_phos_Y315_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='ZAP70_phos_Y315']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70_phos_Y93_12_task2` is invalid.\n - Variable `p6_ZAP70_phos_Y93_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='ZAP70_phos_Y93']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_LCK_active_13_task2` is invalid.\n - Variable `p6_perc_LCK_active_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_LCK_active']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_LCK_inactive_14_task2` is invalid.\n - Variable `p6_perc_LCK_inactive_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_LCK_inactive']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_PI3K_free_15_task2` is invalid.\n - Variable `p6_perc_PI3K_free_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_PI3K_free']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_SHP2_bound_16_task2` is invalid.\n - Variable `p6_perc_SHP2_bound_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_SHP2_bound']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_SHP2_free_17_task2` is invalid.\n - Variable `p6_perc_SHP2_free_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_SHP2_free']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_ZAP70_free_18_task2` is invalid.\n - Variable `p6_perc_ZAP70_free_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_ZAP70_free']` are not defined. No namespaces are defined for the target.\n - Data generator `CD28a_19_task2` is invalid.\n - Variable `p6_CD28a_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CD28a']` are not defined. No namespaces are defined for the target.\n - Data generator `CD28i_20_task2` is invalid.\n - Variable `p6_CD28i_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CD28i']` are not defined. No namespaces are defined for the target.\n - Data generator `CD28t_21_task2` is invalid.\n - Variable `p6_CD28t_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CD28t']` are not defined. No namespaces are defined for the target.\n - Data generator `CD3a_22_task2` is invalid.\n - Variable `p6_CD3a_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CD3a']` are not defined. No namespaces are defined for the target.\n - Data generator `CD3i_23_task2` is invalid.\n - Variable `p6_CD3i_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CD3i']` are not defined. No namespaces are defined for the target.\n - Data generator `CD3t_24_task2` is invalid.\n - Variable `p6_CD3t_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CD3t']` are not defined. No namespaces are defined for the target.\n - Data generator `CP1_25_task2` is invalid.\n - Variable `p6_CP1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CP1']` are not defined. No namespaces are defined for the target.\n - Data generator `CP2_26_task2` is invalid.\n - Variable `p6_CP2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CP2']` are not defined. No namespaces are defined for the target.\n - Data generator `CPactive_27_task2` is invalid.\n - Variable `p6_CPactive_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CPactive']` are not defined. No namespaces are defined for the target.\n - Data generator `GADS_28_task2` is invalid.\n - Variable `p6_GADS_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='GADS']` are not defined. No namespaces are defined for the target.\n - Data generator `GADSa_29_task2` is invalid.\n - Variable `p6_GADSa_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='GADSa']` are not defined. No namespaces are defined for the target.\n - Data generator `GADSt_30_task2` is invalid.\n - Variable `p6_GADSt_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='GADSt']` are not defined. No namespaces are defined for the target.\n - Data generator `LATa_31_task2` is invalid.\n - Variable `p6_LATa_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LATa']` are not defined. No namespaces are defined for the target.\n - Data generator `LATi_32_task2` is invalid.\n - Variable `p6_LATi_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LATi']` are not defined. No namespaces are defined for the target.\n - Data generator `LATt_33_task2` is invalid.\n - Variable `p6_LATt_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LATt']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKactive_34_task2` is invalid.\n - Variable `p6_LCKactive_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKactive']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKi_35_task2` is invalid.\n - Variable `p6_LCKi_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKi']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKinactive_36_task2` is invalid.\n - Variable `p6_LCKinactive_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKinactive']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKpi_37_task2` is invalid.\n - Variable `p6_LCKpi_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKpi']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKt_38_task2` is invalid.\n - Variable `p6_LCKt_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKt']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKya_39_task2` is invalid.\n - Variable `p6_LCKya_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKya']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKyi_40_task2` is invalid.\n - Variable `p6_LCKyi_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKyi']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKyiya_41_task2` is invalid.\n - Variable `p6_LCKyiya_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKyiya']` are not defined. No namespaces are defined for the target.\n - Data generator `PD1_42_task2` is invalid.\n - Variable `p6_PD1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PD1']` are not defined. No namespaces are defined for the target.\n - Data generator `PD1p1_43_task2` is invalid.\n - Variable `p6_PD1p1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PD1p1']` are not defined. No namespaces are defined for the target.\n - Data generator `PD1p2_44_task2` is invalid.\n - Variable `p6_PD1p2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PD1p2']` are not defined. No namespaces are defined for the target.\n - Data generator `PI3K_45_task2` is invalid.\n - Variable `p6_PI3K_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PI3K']` are not defined. No namespaces are defined for the target.\n - Data generator `PI3Kb_46_task2` is invalid.\n - Variable `p6_PI3Kb_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PI3Kb']` are not defined. No namespaces are defined for the target.\n - Data generator `PI3Kt_47_task2` is invalid.\n - Variable `p6_PI3Kt_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PI3Kt']` are not defined. No namespaces are defined for the target.\n - Data generator `SHP2_48_task2` is invalid.\n - Variable `p6_SHP2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SHP2']` are not defined. No namespaces are defined for the target.\n - Data generator `SLP76_49_task2` is invalid.\n - Variable `p6_SLP76_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SLP76']` are not defined. No namespaces are defined for the target.\n - Data generator `SLP76a_50_task2` is invalid.\n - Variable `p6_SLP76a_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SLP76a']` are not defined. No namespaces are defined for the target.\n - Data generator `SLP76i_51_task2` is invalid.\n - Variable `p6_SLP76i_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SLP76i']` are not defined. No namespaces are defined for the target.\n - Data generator `SLP76t_52_task2` is invalid.\n - Variable `p6_SLP76t_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SLP76t']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70_53_task2` is invalid.\n - Variable `p6_ZAP70_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ZAP70']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70a1_54_task2` is invalid.\n - Variable `p6_ZAP70a1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ZAP70a1']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70a2_55_task2` is invalid.\n - Variable `p6_ZAP70a2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ZAP70a2']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70i_56_task2` is invalid.\n - Variable `p6_ZAP70i_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ZAP70i']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70t_57_task2` is invalid.\n - Variable `p6_ZAP70t_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ZAP70t']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Theinmozhi_2018.sedml177249.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Theinmozhi_2018.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (99505). The following is the first warning at line 2492, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'LCK_switch' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 15 warnings of type Modeling practice (80701). The following is the first warning at line 1870, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k' does not have a 'units' attribute.\r\n - Output `plot_7_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_8_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_9_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_10_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_11_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_12_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Theinmozhi_2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Theinmozhi_2018.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (99505). The following is the first warning at line 2492, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'LCK_switch' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 15 warnings of type Modeling practice (80701). The following is the first warning at line 1870, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k' does not have a 'units' attribute.\r\n - Output `plot_7_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_8_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_9_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_10_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_11_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_12_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Theinmozhi_2018.sedml177249.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Theinmozhi_2018.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='LCK_switch']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_PI3K_free_1_task1` is invalid.\n - Variable `p1_perc_PI3K_free_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_PI3K_free']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_ZAP70_free_1_task1` is invalid.\n - Variable `p2_perc_ZAP70_free_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_ZAP70_free']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_LCK_active_1_task1` is invalid.\n - Variable `p3_perc_LCK_active_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_LCK_active']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_LCK_inactive_2_task1` is invalid.\n - Variable `p3_perc_LCK_inactive_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_LCK_inactive']` are not defined. No namespaces are defined for the target.\n - Data generator `SLP76a_1_task1` is invalid.\n - Variable `p4_SLP76a_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SLP76a']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70a2_1_task1` is invalid.\n - Variable `p5_ZAP70a2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ZAP70a2']` are not defined. No namespaces are defined for the target.\n - Data generator `CD28_phosphorylated_1_task1` is invalid.\n - Variable `p6_CD28_phosphorylated_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='CD28_phosphorylated']` are not defined. No namespaces are defined for the target.\n - Data generator `CD3z_phosphorylated_2_task1` is invalid.\n - Variable `p6_CD3z_phosphorylated_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='CD3z_phosphorylated']` are not defined. No namespaces are defined for the target.\n - Data generator `Kdpa_pi_3_task1` is invalid.\n - Variable `p6_Kdpa_pi_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Kdpa_pi']` are not defined. No namespaces are defined for the target.\n - Data generator `Kdpa_ya_4_task1` is invalid.\n - Variable `p6_Kdpa_ya_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Kdpa_ya']` are not defined. No namespaces are defined for the target.\n - Data generator `Kdpa_yiya_5_task1` is invalid.\n - Variable `p6_Kdpa_yiya_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Kdpa_yiya']` are not defined. No namespaces are defined for the target.\n - Data generator `Kdpi_yi_6_task1` is invalid.\n - Variable `p6_Kdpi_yi_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Kdpi_yi']` are not defined. No namespaces are defined for the target.\n - Data generator `Kdpi_yiya_7_task1` is invalid.\n - Variable `p6_Kdpi_yiya_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Kdpi_yiya']` are not defined. No namespaces are defined for the target.\n - Data generator `LAT_phosphorylated_8_task1` is invalid.\n - Variable `p6_LAT_phosphorylated_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='LAT_phosphorylated']` are not defined. No namespaces are defined for the target.\n - Data generator `LCK_phos_Y394_9_task1` is invalid.\n - Variable `p6_LCK_phos_Y394_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='LCK_phos_Y394']` are not defined. No namespaces are defined for the target.\n - Data generator `LCK_phos_Y505_10_task1` is invalid.\n - Variable `p6_LCK_phos_Y505_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='LCK_phos_Y505']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70_phos_Y315_11_task1` is invalid.\n - Variable `p6_ZAP70_phos_Y315_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='ZAP70_phos_Y315']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70_phos_Y93_12_task1` is invalid.\n - Variable `p6_ZAP70_phos_Y93_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='ZAP70_phos_Y93']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_LCK_active_13_task1` is invalid.\n - Variable `p6_perc_LCK_active_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_LCK_active']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_LCK_inactive_14_task1` is invalid.\n - Variable `p6_perc_LCK_inactive_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_LCK_inactive']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_PI3K_free_15_task1` is invalid.\n - Variable `p6_perc_PI3K_free_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_PI3K_free']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_SHP2_bound_16_task1` is invalid.\n - Variable `p6_perc_SHP2_bound_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_SHP2_bound']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_SHP2_free_17_task1` is invalid.\n - Variable `p6_perc_SHP2_free_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_SHP2_free']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_ZAP70_free_18_task1` is invalid.\n - Variable `p6_perc_ZAP70_free_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_ZAP70_free']` are not defined. No namespaces are defined for the target.\n - Data generator `CD28a_19_task1` is invalid.\n - Variable `p6_CD28a_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CD28a']` are not defined. No namespaces are defined for the target.\n - Data generator `CD28i_20_task1` is invalid.\n - Variable `p6_CD28i_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CD28i']` are not defined. No namespaces are defined for the target.\n - Data generator `CD28t_21_task1` is invalid.\n - Variable `p6_CD28t_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CD28t']` are not defined. No namespaces are defined for the target.\n - Data generator `CD3a_22_task1` is invalid.\n - Variable `p6_CD3a_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CD3a']` are not defined. No namespaces are defined for the target.\n - Data generator `CD3i_23_task1` is invalid.\n - Variable `p6_CD3i_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CD3i']` are not defined. No namespaces are defined for the target.\n - Data generator `CD3t_24_task1` is invalid.\n - Variable `p6_CD3t_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CD3t']` are not defined. No namespaces are defined for the target.\n - Data generator `CP1_25_task1` is invalid.\n - Variable `p6_CP1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CP1']` are not defined. No namespaces are defined for the target.\n - Data generator `CP2_26_task1` is invalid.\n - Variable `p6_CP2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CP2']` are not defined. No namespaces are defined for the target.\n - Data generator `CPactive_27_task1` is invalid.\n - Variable `p6_CPactive_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CPactive']` are not defined. No namespaces are defined for the target.\n - Data generator `GADS_28_task1` is invalid.\n - Variable `p6_GADS_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='GADS']` are not defined. No namespaces are defined for the target.\n - Data generator `GADSa_29_task1` is invalid.\n - Variable `p6_GADSa_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='GADSa']` are not defined. No namespaces are defined for the target.\n - Data generator `GADSt_30_task1` is invalid.\n - Variable `p6_GADSt_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='GADSt']` are not defined. No namespaces are defined for the target.\n - Data generator `LATa_31_task1` is invalid.\n - Variable `p6_LATa_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LATa']` are not defined. No namespaces are defined for the target.\n - Data generator `LATi_32_task1` is invalid.\n - Variable `p6_LATi_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LATi']` are not defined. No namespaces are defined for the target.\n - Data generator `LATt_33_task1` is invalid.\n - Variable `p6_LATt_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LATt']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKactive_34_task1` is invalid.\n - Variable `p6_LCKactive_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKactive']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKi_35_task1` is invalid.\n - Variable `p6_LCKi_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKi']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKinactive_36_task1` is invalid.\n - Variable `p6_LCKinactive_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKinactive']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKpi_37_task1` is invalid.\n - Variable `p6_LCKpi_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKpi']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKt_38_task1` is invalid.\n - Variable `p6_LCKt_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKt']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKya_39_task1` is invalid.\n - Variable `p6_LCKya_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKya']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKyi_40_task1` is invalid.\n - Variable `p6_LCKyi_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKyi']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKyiya_41_task1` is invalid.\n - Variable `p6_LCKyiya_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKyiya']` are not defined. No namespaces are defined for the target.\n - Data generator `PD1_42_task1` is invalid.\n - Variable `p6_PD1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PD1']` are not defined. No namespaces are defined for the target.\n - Data generator `PD1p1_43_task1` is invalid.\n - Variable `p6_PD1p1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PD1p1']` are not defined. No namespaces are defined for the target.\n - Data generator `PD1p2_44_task1` is invalid.\n - Variable `p6_PD1p2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PD1p2']` are not defined. No namespaces are defined for the target.\n - Data generator `PI3K_45_task1` is invalid.\n - Variable `p6_PI3K_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PI3K']` are not defined. No namespaces are defined for the target.\n - Data generator `PI3Kb_46_task1` is invalid.\n - Variable `p6_PI3Kb_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PI3Kb']` are not defined. No namespaces are defined for the target.\n - Data generator `PI3Kt_47_task1` is invalid.\n - Variable `p6_PI3Kt_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PI3Kt']` are not defined. No namespaces are defined for the target.\n - Data generator `SHP2_48_task1` is invalid.\n - Variable `p6_SHP2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SHP2']` are not defined. No namespaces are defined for the target.\n - Data generator `SLP76_49_task1` is invalid.\n - Variable `p6_SLP76_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SLP76']` are not defined. No namespaces are defined for the target.\n - Data generator `SLP76a_50_task1` is invalid.\n - Variable `p6_SLP76a_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SLP76a']` are not defined. No namespaces are defined for the target.\n - Data generator `SLP76i_51_task1` is invalid.\n - Variable `p6_SLP76i_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SLP76i']` are not defined. No namespaces are defined for the target.\n - Data generator `SLP76t_52_task1` is invalid.\n - Variable `p6_SLP76t_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SLP76t']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70_53_task1` is invalid.\n - Variable `p6_ZAP70_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ZAP70']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70a1_54_task1` is invalid.\n - Variable `p6_ZAP70a1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ZAP70a1']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70a2_55_task1` is invalid.\n - Variable `p6_ZAP70a2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ZAP70a2']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70i_56_task1` is invalid.\n - Variable `p6_ZAP70i_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ZAP70i']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70t_57_task1` is invalid.\n - Variable `p6_ZAP70t_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ZAP70t']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_PI3K_free_1_task2` is invalid.\n - Variable `p1_perc_PI3K_free_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_PI3K_free']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_ZAP70_free_1_task2` is invalid.\n - Variable `p2_perc_ZAP70_free_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_ZAP70_free']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_LCK_active_1_task2` is invalid.\n - Variable `p3_perc_LCK_active_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_LCK_active']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_LCK_inactive_2_task2` is invalid.\n - Variable `p3_perc_LCK_inactive_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_LCK_inactive']` are not defined. No namespaces are defined for the target.\n - Data generator `SLP76a_1_task2` is invalid.\n - Variable `p4_SLP76a_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SLP76a']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70a2_1_task2` is invalid.\n - Variable `p5_ZAP70a2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ZAP70a2']` are not defined. No namespaces are defined for the target.\n - Data generator `CD28_phosphorylated_1_task2` is invalid.\n - Variable `p6_CD28_phosphorylated_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='CD28_phosphorylated']` are not defined. No namespaces are defined for the target.\n - Data generator `CD3z_phosphorylated_2_task2` is invalid.\n - Variable `p6_CD3z_phosphorylated_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='CD3z_phosphorylated']` are not defined. No namespaces are defined for the target.\n - Data generator `Kdpa_pi_3_task2` is invalid.\n - Variable `p6_Kdpa_pi_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Kdpa_pi']` are not defined. No namespaces are defined for the target.\n - Data generator `Kdpa_ya_4_task2` is invalid.\n - Variable `p6_Kdpa_ya_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Kdpa_ya']` are not defined. No namespaces are defined for the target.\n - Data generator `Kdpa_yiya_5_task2` is invalid.\n - Variable `p6_Kdpa_yiya_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Kdpa_yiya']` are not defined. No namespaces are defined for the target.\n - Data generator `Kdpi_yi_6_task2` is invalid.\n - Variable `p6_Kdpi_yi_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Kdpi_yi']` are not defined. No namespaces are defined for the target.\n - Data generator `Kdpi_yiya_7_task2` is invalid.\n - Variable `p6_Kdpi_yiya_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Kdpi_yiya']` are not defined. No namespaces are defined for the target.\n - Data generator `LAT_phosphorylated_8_task2` is invalid.\n - Variable `p6_LAT_phosphorylated_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='LAT_phosphorylated']` are not defined. No namespaces are defined for the target.\n - Data generator `LCK_phos_Y394_9_task2` is invalid.\n - Variable `p6_LCK_phos_Y394_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='LCK_phos_Y394']` are not defined. No namespaces are defined for the target.\n - Data generator `LCK_phos_Y505_10_task2` is invalid.\n - Variable `p6_LCK_phos_Y505_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='LCK_phos_Y505']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70_phos_Y315_11_task2` is invalid.\n - Variable `p6_ZAP70_phos_Y315_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='ZAP70_phos_Y315']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70_phos_Y93_12_task2` is invalid.\n - Variable `p6_ZAP70_phos_Y93_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='ZAP70_phos_Y93']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_LCK_active_13_task2` is invalid.\n - Variable `p6_perc_LCK_active_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_LCK_active']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_LCK_inactive_14_task2` is invalid.\n - Variable `p6_perc_LCK_inactive_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_LCK_inactive']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_PI3K_free_15_task2` is invalid.\n - Variable `p6_perc_PI3K_free_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_PI3K_free']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_SHP2_bound_16_task2` is invalid.\n - Variable `p6_perc_SHP2_bound_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_SHP2_bound']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_SHP2_free_17_task2` is invalid.\n - Variable `p6_perc_SHP2_free_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_SHP2_free']` are not defined. No namespaces are defined for the target.\n - Data generator `perc_ZAP70_free_18_task2` is invalid.\n - Variable `p6_perc_ZAP70_free_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='perc_ZAP70_free']` are not defined. No namespaces are defined for the target.\n - Data generator `CD28a_19_task2` is invalid.\n - Variable `p6_CD28a_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CD28a']` are not defined. No namespaces are defined for the target.\n - Data generator `CD28i_20_task2` is invalid.\n - Variable `p6_CD28i_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CD28i']` are not defined. No namespaces are defined for the target.\n - Data generator `CD28t_21_task2` is invalid.\n - Variable `p6_CD28t_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CD28t']` are not defined. No namespaces are defined for the target.\n - Data generator `CD3a_22_task2` is invalid.\n - Variable `p6_CD3a_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CD3a']` are not defined. No namespaces are defined for the target.\n - Data generator `CD3i_23_task2` is invalid.\n - Variable `p6_CD3i_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CD3i']` are not defined. No namespaces are defined for the target.\n - Data generator `CD3t_24_task2` is invalid.\n - Variable `p6_CD3t_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CD3t']` are not defined. No namespaces are defined for the target.\n - Data generator `CP1_25_task2` is invalid.\n - Variable `p6_CP1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CP1']` are not defined. No namespaces are defined for the target.\n - Data generator `CP2_26_task2` is invalid.\n - Variable `p6_CP2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CP2']` are not defined. No namespaces are defined for the target.\n - Data generator `CPactive_27_task2` is invalid.\n - Variable `p6_CPactive_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CPactive']` are not defined. No namespaces are defined for the target.\n - Data generator `GADS_28_task2` is invalid.\n - Variable `p6_GADS_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='GADS']` are not defined. No namespaces are defined for the target.\n - Data generator `GADSa_29_task2` is invalid.\n - Variable `p6_GADSa_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='GADSa']` are not defined. No namespaces are defined for the target.\n - Data generator `GADSt_30_task2` is invalid.\n - Variable `p6_GADSt_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='GADSt']` are not defined. No namespaces are defined for the target.\n - Data generator `LATa_31_task2` is invalid.\n - Variable `p6_LATa_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LATa']` are not defined. No namespaces are defined for the target.\n - Data generator `LATi_32_task2` is invalid.\n - Variable `p6_LATi_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LATi']` are not defined. No namespaces are defined for the target.\n - Data generator `LATt_33_task2` is invalid.\n - Variable `p6_LATt_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LATt']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKactive_34_task2` is invalid.\n - Variable `p6_LCKactive_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKactive']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKi_35_task2` is invalid.\n - Variable `p6_LCKi_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKi']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKinactive_36_task2` is invalid.\n - Variable `p6_LCKinactive_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKinactive']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKpi_37_task2` is invalid.\n - Variable `p6_LCKpi_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKpi']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKt_38_task2` is invalid.\n - Variable `p6_LCKt_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKt']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKya_39_task2` is invalid.\n - Variable `p6_LCKya_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKya']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKyi_40_task2` is invalid.\n - Variable `p6_LCKyi_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKyi']` are not defined. No namespaces are defined for the target.\n - Data generator `LCKyiya_41_task2` is invalid.\n - Variable `p6_LCKyiya_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='LCKyiya']` are not defined. No namespaces are defined for the target.\n - Data generator `PD1_42_task2` is invalid.\n - Variable `p6_PD1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PD1']` are not defined. No namespaces are defined for the target.\n - Data generator `PD1p1_43_task2` is invalid.\n - Variable `p6_PD1p1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PD1p1']` are not defined. No namespaces are defined for the target.\n - Data generator `PD1p2_44_task2` is invalid.\n - Variable `p6_PD1p2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PD1p2']` are not defined. No namespaces are defined for the target.\n - Data generator `PI3K_45_task2` is invalid.\n - Variable `p6_PI3K_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PI3K']` are not defined. No namespaces are defined for the target.\n - Data generator `PI3Kb_46_task2` is invalid.\n - Variable `p6_PI3Kb_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PI3Kb']` are not defined. No namespaces are defined for the target.\n - Data generator `PI3Kt_47_task2` is invalid.\n - Variable `p6_PI3Kt_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PI3Kt']` are not defined. No namespaces are defined for the target.\n - Data generator `SHP2_48_task2` is invalid.\n - Variable `p6_SHP2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SHP2']` are not defined. No namespaces are defined for the target.\n - Data generator `SLP76_49_task2` is invalid.\n - Variable `p6_SLP76_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SLP76']` are not defined. No namespaces are defined for the target.\n - Data generator `SLP76a_50_task2` is invalid.\n - Variable `p6_SLP76a_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SLP76a']` are not defined. No namespaces are defined for the target.\n - Data generator `SLP76i_51_task2` is invalid.\n - Variable `p6_SLP76i_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SLP76i']` are not defined. No namespaces are defined for the target.\n - Data generator `SLP76t_52_task2` is invalid.\n - Variable `p6_SLP76t_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SLP76t']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70_53_task2` is invalid.\n - Variable `p6_ZAP70_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ZAP70']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70a1_54_task2` is invalid.\n - Variable `p6_ZAP70a1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ZAP70a1']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70a2_55_task2` is invalid.\n - Variable `p6_ZAP70a2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ZAP70a2']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70i_56_task2` is invalid.\n - Variable `p6_ZAP70i_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ZAP70i']` are not defined. No namespaces are defined for the target.\n - Data generator `ZAP70t_57_task2` is invalid.\n - Variable `p6_ZAP70t_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ZAP70t']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000725](https://www.ebi.ac.uk/biomodels/BIOMD0000000725)<br/><sup>Salcedo-Sora2016 - Microbial f</sup></summary><sup>Salcedo-Sora2016 - Microbial folate biosynthesis and utilisation</sup></details>|<details><summary>pass</summary>[Model.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000725#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000726](https://www.ebi.ac.uk/biomodels/BIOMD0000000726)<br/><sup>Ruan2017 - Transmission dynami</sup></summary><sup>Ruan2017 - Transmission dynamics and control of rabies in China</sup></details>|<details><summary>pass</summary>[Model.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000726#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000727](https://www.ebi.ac.uk/biomodels/BIOMD0000000727)<br/><sup>Li2009- Assymetric Caulobacter</sup></summary><sup>Li2009- Assymetric Caulobacter cell cycle</sup></details>|<details><summary>pass</summary>[Li2009.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000727#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000728](https://www.ebi.ac.uk/biomodels/BIOMD0000000728)<br/><sup>Norel1990 - MPF and Cyclin Osc</sup></summary><sup>Norel1990 - MPF and Cyclin Oscillations</sup></details>|<details><summary>pass</summary>[Norell1990.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000728#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000729](https://www.ebi.ac.uk/biomodels/BIOMD0000000729)<br/><sup>Goldbeter1996 - Cyclin Cdc2 ki</sup></summary><sup>Goldbeter1996 - Cyclin Cdc2 kinase Oscillations</sup></details>|<details><summary>pass</summary>[Goldbeter1996.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000729#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000730](https://www.ebi.ac.uk/biomodels/BIOMD0000000730)<br/><sup>Gerard2009 - An Integrated Mam</sup></summary><sup>Gerard2009 - An Integrated Mammalian Cell Cycle Model</sup></details>|<details><summary>pass</summary>[Gerard2009.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000730#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000731](https://www.ebi.ac.uk/biomodels/BIOMD0000000731)<br/><sup>Robertson-Tessi M 2012 A model</sup></summary><sup>Robertson-Tessi M 2012 A model of tumor Immune interaction</sup></details>|<details><summary>pass</summary>[22051568_Tessi.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000731#Files)</details>|FAIL|<details><summary>FAIL</summary>[22051568_Tessi.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000731#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/22051568_Tessi.sedml498496.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./22051568_Tessi.sedml` has warnings.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n`/root/in/22051568_Tessi.sedml498496.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./22051568_Tessi.sedml` is invalid.\n - Model `model2` is invalid.\n - The model file `model2.xml` is invalid.\n - `/tmp/tmpwg88_fyz/./model2.xml` is not a file.\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/22051568_Tessi.sedml734510.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./22051568_Tessi.sedml` has warnings.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/22051568_Tessi.sedml734510.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./22051568_Tessi.sedml` is invalid.\n - Model `model2` is invalid.\n - The model file `model2.xml` is invalid.\n - `/tmp/tmpdc05vava/./model2.xml` is not a file.\n'```</details>|
|[BIOMD0000000732](https://www.ebi.ac.uk/biomodels/BIOMD0000000732)<br/><sup>Kirschner1998_Immunotherapy_Tumour</sup>|<details><summary>pass</summary>[Kirschner_1998.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000732#Files)</details>|FAIL|<details><summary>FAIL</summary>[Kirschner_1998.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000732#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Kirschner_1998.sedml145021.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Kirschner_1998.sedml` has warnings.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/Kirschner_1998.sedml145021.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Kirschner_1998.sedml` is invalid.\n - Model `model2` is invalid.\n - The model file `model2.xml` is invalid.\n - `/tmp/tmp_r1wubhv/./model2.xml` is not a file.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='c']` are not defined. No namespaces are defined for the target.\n - Data generator `IL2_1_task1` is invalid.\n - Variable `p1_IL2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IL2']` are not defined. No namespaces are defined for the target.\n - Data generator `Immune_cells_2_task1` is invalid.\n - Variable `p1_Immune_cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Immune_cells']` are not defined. No namespaces are defined for the target.\n - Data generator `Tumor_3_task1` is invalid.\n - Variable `p1_Tumor_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Tumor']` are not defined. No namespaces are defined for the target.\n - Data generator `IL2_1_task2` is invalid.\n - Variable `p1_IL2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IL2']` are not defined. No namespaces are defined for the target.\n - Data generator `Immune_cells_2_task2` is invalid.\n - Variable `p1_Immune_cells_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Immune_cells']` are not defined. No namespaces are defined for the target.\n - Data generator `Tumor_3_task2` is invalid.\n - Variable `p1_Tumor_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Tumor']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Kirschner_1998.sedml145155.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Kirschner_1998.sedml` has warnings.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Kirschner_1998.sedml145155.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Kirschner_1998.sedml` is invalid.\n - Model `model2` is invalid.\n - The model file `model2.xml` is invalid.\n - `/tmp/tmpsk5gwee2/./model2.xml` is not a file.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='c']` are not defined. No namespaces are defined for the target.\n - Data generator `IL2_1_task1` is invalid.\n - Variable `p1_IL2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IL2']` are not defined. No namespaces are defined for the target.\n - Data generator `Immune_cells_2_task1` is invalid.\n - Variable `p1_Immune_cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Immune_cells']` are not defined. No namespaces are defined for the target.\n - Data generator `Tumor_3_task1` is invalid.\n - Variable `p1_Tumor_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Tumor']` are not defined. No namespaces are defined for the target.\n - Data generator `IL2_1_task2` is invalid.\n - Variable `p1_IL2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IL2']` are not defined. No namespaces are defined for the target.\n - Data generator `Immune_cells_2_task2` is invalid.\n - Variable `p1_Immune_cells_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Immune_cells']` are not defined. No namespaces are defined for the target.\n - Data generator `Tumor_3_task2` is invalid.\n - Variable `p1_Tumor_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Tumor']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000733](https://www.ebi.ac.uk/biomodels/BIOMD0000000733)<br/><sup>Moore_2004_Mathematical model </sup></summary><sup>Moore_2004_Mathematical model for CML and T cell interaction</sup></details>|<details><summary>pass</summary>[Moore_2004_fig6.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000733#Files)</details>|FAIL|<details><summary>pass</summary>[Moore_2004_fig6.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000733#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Moore_2004_fig6.sedml33089.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Moore_2004_fig6.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (10541). The following is the first warning at line 708, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Naive_Tcells_production') are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = -1).\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Moore_2004_fig6.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Moore_2004_fig6.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (10541). The following is the first warning at line 708, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Naive_Tcells_production') are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = -1).\x1b[0m\r\n\r\n`/root/in/Moore_2004_fig6.sedml33089.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Moore_2004_fig6.sedml` is invalid.\n - Data generator `naive_Tcells_1_task1` is invalid.\n - Variable `p1_naive_Tcells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='naive_Tcells']` are not defined. No namespaces are defined for the target.\n - Data generator `eff_Tcells_2_task1` is invalid.\n - Variable `p1_eff_Tcells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='eff_Tcells']` are not defined. No namespaces are defined for the target.\n - Data generator `tumor_cells_3_task1` is invalid.\n - Variable `p1_tumor_cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='tumor_cells']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Moore_2004_fig6.sedml461599.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Moore_2004_fig6.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (10541). The following is the first warning at line 708, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Naive_Tcells_production') are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = -1).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Moore_2004_fig6.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Moore_2004_fig6.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (10541). The following is the first warning at line 708, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Naive_Tcells_production') are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = -1).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Moore_2004_fig6.sedml461599.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Moore_2004_fig6.sedml` is invalid.\n - Data generator `naive_Tcells_1_task1` is invalid.\n - Variable `p1_naive_Tcells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='naive_Tcells']` are not defined. No namespaces are defined for the target.\n - Data generator `eff_Tcells_2_task1` is invalid.\n - Variable `p1_eff_Tcells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='eff_Tcells']` are not defined. No namespaces are defined for the target.\n - Data generator `tumor_cells_3_task1` is invalid.\n - Variable `p1_tumor_cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='tumor_cells']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000734](https://www.ebi.ac.uk/biomodels/BIOMD0000000734)<br/><sup>Mouse Iron Distribution - Rich</sup></summary><sup>Mouse Iron Distribution - Rich and Deficient iron diets (tracer)</sup></details>|<details><summary>pass</summary>[BIOMD0000000734.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000734#Files)</details>|pass|<details><summary>pass</summary>[Parmar2017_Deficient_Rich_tracer.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000734#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Parmar2017_Deficient_Rich_tracer.sedml749091.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000734.xml` has warnings.\r\n - 69 warnings of type SBML unit consistency (99505). The following is the first warning at line 1787, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '100 * FeSpleen * quantity_to_number_factor * Spleen / Injected' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 43 warnings of type Modeling practice (80701). The following is the first warning at line 1194, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Total_Fe___in_body' does not have a 'units' attribute.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Parmar2017_Deficient_Rich_tracer.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000734.xml` has warnings.\r\n - 69 warnings of type SBML unit consistency (99505). The following is the first warning at line 1787, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '100 * FeSpleen * quantity_to_number_factor * Spleen / Injected' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 43 warnings of type Modeling practice (80701). The following is the first warning at line 1194, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Total_Fe___in_body' does not have a 'units' attribute.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n`/root/in/Parmar2017_Deficient_Rich_tracer.sedml749091.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Parmar2017_Deficient_Rich_tracer.sedml` is invalid.\n - Output `plot_1_task1` is invalid.\n - Plot must have at least one curve.\n - Output `plot_2_task1` is invalid.\n - Plot must have at least one curve.\n - Output `plot_3_task1` is invalid.\n - Plot must have at least one curve.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Parmar2017_Deficient_Rich_tracer.sedml959525.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000734.xml` has warnings.\r\n - 69 warnings of type SBML unit consistency (99505). The following is the first warning at line 1787, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '100 * FeSpleen * quantity_to_number_factor * Spleen / Injected' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 43 warnings of type Modeling practice (80701). The following is the first warning at line 1194, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Total_Fe___in_body' does not have a 'units' attribute.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Parmar2017_Deficient_Rich_tracer.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `BIOMD0000000734.xml` has warnings.\r\n - 69 warnings of type SBML unit consistency (99505). The following is the first warning at line 1787, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '100 * FeSpleen * quantity_to_number_factor * Spleen / Injected' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 43 warnings of type Modeling practice (80701). The following is the first warning at line 1194, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Total_Fe___in_body' does not have a 'units' attribute.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Parmar2017_Deficient_Rich_tracer.sedml959525.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Parmar2017_Deficient_Rich_tracer.sedml` is invalid.\n - Output `plot_1_task1` is invalid.\n - Plot must have at least one curve.\n - Output `plot_2_task1` is invalid.\n - Plot must have at least one curve.\n - Output `plot_3_task1` is invalid.\n - Plot must have at least one curve.\n"```</details>|
|<details><summary>[BIOMD0000000735](https://www.ebi.ac.uk/biomodels/BIOMD0000000735)<br/><sup>Mouse Iron Distribution - Adeq</sup></summary><sup>Mouse Iron Distribution - Adequate iron diet (tracer)</sup></details>|<details><summary>pass</summary>[Parmar2017_Adequate_tracer.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000735#Files)</details>|pass|<details><summary>pass</summary>[Parmar2017_Adequate_tracer.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000735#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Parmar2017_Adequate_tracer.sedml571800.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Parmar2017_Adequate_tracer.xml` has warnings.\r\n - 68 warnings of type SBML unit consistency (99505). The following is the first warning at line 1756, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'FeDuo * Avogadro * Duodenum + FeLiver * Avogadro * Liver + FeSpleen * Avogadro * Spleen + FeRBC * Avogadro * RBC + FeRest * Avogadro * RestOfBody + Fe2Tf * Avogadro * Plasma + NTBI * Avogadro * Plasma + 2 * Fe2Tf_ * Avogadro * Plasma + Fe1Tf * Avogadro * Plasma + FeBM_0 * Avogadro * BoneMarrow' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 42 warnings of type Modeling practice (80701). The following is the first warning at line 1156, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Total_Fe___in_body' does not have a 'units' attribute.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Parmar2017_Adequate_tracer.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Parmar2017_Adequate_tracer.xml` has warnings.\r\n - 68 warnings of type SBML unit consistency (99505). The following is the first warning at line 1756, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'FeDuo * Avogadro * Duodenum + FeLiver * Avogadro * Liver + FeSpleen * Avogadro * Spleen + FeRBC * Avogadro * RBC + FeRest * Avogadro * RestOfBody + Fe2Tf * Avogadro * Plasma + NTBI * Avogadro * Plasma + 2 * Fe2Tf_ * Avogadro * Plasma + Fe1Tf * Avogadro * Plasma + FeBM_0 * Avogadro * BoneMarrow' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 42 warnings of type Modeling practice (80701). The following is the first warning at line 1156, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Total_Fe___in_body' does not have a 'units' attribute.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n`/root/in/Parmar2017_Adequate_tracer.sedml571800.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Parmar2017_Adequate_tracer.sedml` is invalid.\n - Output `plot_1_task1` is invalid.\n - Plot must have at least one curve.\n - Output `plot_2_task1` is invalid.\n - Plot must have at least one curve.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Parmar2017_Adequate_tracer.sedml141872.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Parmar2017_Adequate_tracer.xml` has warnings.\r\n - 68 warnings of type SBML unit consistency (99505). The following is the first warning at line 1756, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'FeDuo * Avogadro * Duodenum + FeLiver * Avogadro * Liver + FeSpleen * Avogadro * Spleen + FeRBC * Avogadro * RBC + FeRest * Avogadro * RestOfBody + Fe2Tf * Avogadro * Plasma + NTBI * Avogadro * Plasma + 2 * Fe2Tf_ * Avogadro * Plasma + Fe1Tf * Avogadro * Plasma + FeBM_0 * Avogadro * BoneMarrow' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 42 warnings of type Modeling practice (80701). The following is the first warning at line 1156, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Total_Fe___in_body' does not have a 'units' attribute.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Parmar2017_Adequate_tracer.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Parmar2017_Adequate_tracer.xml` has warnings.\r\n - 68 warnings of type SBML unit consistency (99505). The following is the first warning at line 1756, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'FeDuo * Avogadro * Duodenum + FeLiver * Avogadro * Liver + FeSpleen * Avogadro * Spleen + FeRBC * Avogadro * RBC + FeRest * Avogadro * RestOfBody + Fe2Tf * Avogadro * Plasma + NTBI * Avogadro * Plasma + 2 * Fe2Tf_ * Avogadro * Plasma + Fe1Tf * Avogadro * Plasma + FeBM_0 * Avogadro * BoneMarrow' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 42 warnings of type Modeling practice (80701). The following is the first warning at line 1156, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Total_Fe___in_body' does not have a 'units' attribute.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Parmar2017_Adequate_tracer.sedml141872.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Parmar2017_Adequate_tracer.sedml` is invalid.\n - Output `plot_1_task1` is invalid.\n - Plot must have at least one curve.\n - Output `plot_2_task1` is invalid.\n - Plot must have at least one curve.\n"```</details>|
|<details><summary>[BIOMD0000000736](https://www.ebi.ac.uk/biomodels/BIOMD0000000736)<br/><sup>Mouse Iron Distribution - Adeq</sup></summary><sup>Mouse Iron Distribution - Adequate iron diet (No Tracer)</sup></details>|<details><summary>pass</summary>[Parmar2017_Adequate_diet_Anemia_NoTracer.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000736#Files)</details>|pass|<details><summary>pass</summary>[Parmar2017_Adequate_diet_Anemia_NoTracer.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000736#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Parmar2017_Adequate_diet_Anemia_NoTracer.sedml597162.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Parmar2017_Adequate_diet_Anemia_NoTracer.xml` has warnings.\r\n - 34 warnings of type SBML unit consistency (99505). The following is the first warning at line 1273, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Avogadro' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 32 warnings of type Modeling practice (80701). The following is the first warning at line 859, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kNTBI_Fe1Tf' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Parmar2017_Adequate_diet_Anemia_NoTracer.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Parmar2017_Adequate_diet_Anemia_NoTracer.xml` has warnings.\r\n - 34 warnings of type SBML unit consistency (99505). The following is the first warning at line 1273, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Avogadro' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 32 warnings of type Modeling practice (80701). The following is the first warning at line 859, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kNTBI_Fe1Tf' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Parmar2017_Adequate_diet_Anemia_NoTracer.sedml597162.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Parmar2017_Adequate_diet_Anemia_NoTracer.sedml` is invalid.\n - Data generator `FePlasma__particle_1_task1` is invalid.\n - Variable `p1_FePlasma__particle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='FePlasma__particle']` are not defined. No namespaces are defined for the target.\n - Data generator `FePlasma_conc_2_task1` is invalid.\n - Variable `p1_FePlasma_conc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='FePlasma_conc']` are not defined. No namespaces are defined for the target.\n - Data generator `TfSaturation_3_task1` is invalid.\n - Variable `p1_TfSaturation_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='TfSaturation']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Fe__conc_4_task1` is invalid.\n - Variable `p1_Total_Fe__conc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Fe__conc']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Fe__g_5_task1` is invalid.\n - Variable `p1_Total_Fe__g_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Fe__g']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Fe__particle_6_task1` is invalid.\n - Variable `p1_Total_Fe__particle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Fe__particle']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Tf__conc_7_task1` is invalid.\n - Variable `p1_Total_Tf__conc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Tf__conc']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Tf__mg_ml_8_task1` is invalid.\n - Variable `p1_Total_Tf__mg_ml_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Tf__mg_ml']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Tf__particle_9_task1` is invalid.\n - Variable `p1_Total_Tf__particle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Tf__particle']` are not defined. No namespaces are defined for the target.\n - Data generator `Fe1Tf_10_task1` is invalid.\n - Variable `p1_Fe1Tf_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Fe1Tf']` are not defined. No namespaces are defined for the target.\n - Data generator `Fe2Tf_11_task1` is invalid.\n - Variable `p1_Fe2Tf_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Fe2Tf']` are not defined. No namespaces are defined for the target.\n - Data generator `FeBM_12_task1` is invalid.\n - Variable `p1_FeBM_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeBM']` are not defined. No namespaces are defined for the target.\n - Data generator `FeDuo_13_task1` is invalid.\n - Variable `p1_FeDuo_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeDuo']` are not defined. No namespaces are defined for the target.\n - Data generator `FeLiver_14_task1` is invalid.\n - Variable `p1_FeLiver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `FeRBC_15_task1` is invalid.\n - Variable `p1_FeRBC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeRBC']` are not defined. No namespaces are defined for the target.\n - Data generator `FeRest_16_task1` is invalid.\n - Variable `p1_FeRest_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeRest']` are not defined. No namespaces are defined for the target.\n - Data generator `FeSpleen_17_task1` is invalid.\n - Variable `p1_FeSpleen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeSpleen']` are not defined. No namespaces are defined for the target.\n - Data generator `Hepcidin_18_task1` is invalid.\n - Variable `p1_Hepcidin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Hepcidin']` are not defined. No namespaces are defined for the target.\n - Data generator `NTBI_19_task1` is invalid.\n - Variable `p1_NTBI_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='NTBI']` are not defined. No namespaces are defined for the target.\n - Data generator `Tf_20_task1` is invalid.\n - Variable `p1_Tf_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Tf']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Parmar2017_Adequate_diet_Anemia_NoTracer.sedml264657.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Parmar2017_Adequate_diet_Anemia_NoTracer.xml` has warnings.\r\n - 34 warnings of type SBML unit consistency (99505). The following is the first warning at line 1273, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Avogadro' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 32 warnings of type Modeling practice (80701). The following is the first warning at line 859, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kNTBI_Fe1Tf' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Parmar2017_Adequate_diet_Anemia_NoTracer.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Parmar2017_Adequate_diet_Anemia_NoTracer.xml` has warnings.\r\n - 34 warnings of type SBML unit consistency (99505). The following is the first warning at line 1273, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Avogadro' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 32 warnings of type Modeling practice (80701). The following is the first warning at line 859, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kNTBI_Fe1Tf' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Parmar2017_Adequate_diet_Anemia_NoTracer.sedml264657.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Parmar2017_Adequate_diet_Anemia_NoTracer.sedml` is invalid.\n - Data generator `FePlasma__particle_1_task1` is invalid.\n - Variable `p1_FePlasma__particle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='FePlasma__particle']` are not defined. No namespaces are defined for the target.\n - Data generator `FePlasma_conc_2_task1` is invalid.\n - Variable `p1_FePlasma_conc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='FePlasma_conc']` are not defined. No namespaces are defined for the target.\n - Data generator `TfSaturation_3_task1` is invalid.\n - Variable `p1_TfSaturation_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='TfSaturation']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Fe__conc_4_task1` is invalid.\n - Variable `p1_Total_Fe__conc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Fe__conc']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Fe__g_5_task1` is invalid.\n - Variable `p1_Total_Fe__g_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Fe__g']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Fe__particle_6_task1` is invalid.\n - Variable `p1_Total_Fe__particle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Fe__particle']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Tf__conc_7_task1` is invalid.\n - Variable `p1_Total_Tf__conc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Tf__conc']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Tf__mg_ml_8_task1` is invalid.\n - Variable `p1_Total_Tf__mg_ml_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Tf__mg_ml']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Tf__particle_9_task1` is invalid.\n - Variable `p1_Total_Tf__particle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Tf__particle']` are not defined. No namespaces are defined for the target.\n - Data generator `Fe1Tf_10_task1` is invalid.\n - Variable `p1_Fe1Tf_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Fe1Tf']` are not defined. No namespaces are defined for the target.\n - Data generator `Fe2Tf_11_task1` is invalid.\n - Variable `p1_Fe2Tf_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Fe2Tf']` are not defined. No namespaces are defined for the target.\n - Data generator `FeBM_12_task1` is invalid.\n - Variable `p1_FeBM_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeBM']` are not defined. No namespaces are defined for the target.\n - Data generator `FeDuo_13_task1` is invalid.\n - Variable `p1_FeDuo_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeDuo']` are not defined. No namespaces are defined for the target.\n - Data generator `FeLiver_14_task1` is invalid.\n - Variable `p1_FeLiver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `FeRBC_15_task1` is invalid.\n - Variable `p1_FeRBC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeRBC']` are not defined. No namespaces are defined for the target.\n - Data generator `FeRest_16_task1` is invalid.\n - Variable `p1_FeRest_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeRest']` are not defined. No namespaces are defined for the target.\n - Data generator `FeSpleen_17_task1` is invalid.\n - Variable `p1_FeSpleen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeSpleen']` are not defined. No namespaces are defined for the target.\n - Data generator `Hepcidin_18_task1` is invalid.\n - Variable `p1_Hepcidin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Hepcidin']` are not defined. No namespaces are defined for the target.\n - Data generator `NTBI_19_task1` is invalid.\n - Variable `p1_NTBI_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='NTBI']` are not defined. No namespaces are defined for the target.\n - Data generator `Tf_20_task1` is invalid.\n - Variable `p1_Tf_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Tf']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000737](https://www.ebi.ac.uk/biomodels/BIOMD0000000737)<br/><sup>Mouse Iron Distribution - Defi</sup></summary><sup>Mouse Iron Distribution - Deficient iron diet (No Tracer)</sup></details>|<details><summary>pass</summary>[Parmar2017_Deficient_NoTracer.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000737#Files)</details>|pass|<details><summary>pass</summary>[Parmar2017_Deficient_NoTracer.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000737#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Parmar2017_Deficient_NoTracer.sedml886191.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Parmar2017_Deficient_NoTracer.xml` has warnings.\r\n - 32 warnings of type SBML unit consistency (99505). The following is the first warning at line 1261, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Avogadro' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 32 warnings of type Modeling practice (80701). The following is the first warning at line 847, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kNTBI_Fe1Tf' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Parmar2017_Deficient_NoTracer.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Parmar2017_Deficient_NoTracer.xml` has warnings.\r\n - 32 warnings of type SBML unit consistency (99505). The following is the first warning at line 1261, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Avogadro' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 32 warnings of type Modeling practice (80701). The following is the first warning at line 847, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kNTBI_Fe1Tf' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Parmar2017_Deficient_NoTracer.sedml886191.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Parmar2017_Deficient_NoTracer.sedml` is invalid.\n - Data generator `FePlasma__particle_1_task1` is invalid.\n - Variable `p1_FePlasma__particle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='FePlasma__particle']` are not defined. No namespaces are defined for the target.\n - Data generator `FePlasma_conc_2_task1` is invalid.\n - Variable `p1_FePlasma_conc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='FePlasma_conc']` are not defined. No namespaces are defined for the target.\n - Data generator `TfSaturation_3_task1` is invalid.\n - Variable `p1_TfSaturation_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='TfSaturation']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Fe__conc_4_task1` is invalid.\n - Variable `p1_Total_Fe__conc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Fe__conc']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Fe__g_5_task1` is invalid.\n - Variable `p1_Total_Fe__g_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Fe__g']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Fe__particle_6_task1` is invalid.\n - Variable `p1_Total_Fe__particle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Fe__particle']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Tf__conc_7_task1` is invalid.\n - Variable `p1_Total_Tf__conc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Tf__conc']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Tf__mg_ml_8_task1` is invalid.\n - Variable `p1_Total_Tf__mg_ml_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Tf__mg_ml']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Tf__particle_9_task1` is invalid.\n - Variable `p1_Total_Tf__particle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Tf__particle']` are not defined. No namespaces are defined for the target.\n - Data generator `Fe1Tf_10_task1` is invalid.\n - Variable `p1_Fe1Tf_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Fe1Tf']` are not defined. No namespaces are defined for the target.\n - Data generator `Fe2Tf_11_task1` is invalid.\n - Variable `p1_Fe2Tf_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Fe2Tf']` are not defined. No namespaces are defined for the target.\n - Data generator `FeBM_12_task1` is invalid.\n - Variable `p1_FeBM_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeBM']` are not defined. No namespaces are defined for the target.\n - Data generator `FeDuo_13_task1` is invalid.\n - Variable `p1_FeDuo_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeDuo']` are not defined. No namespaces are defined for the target.\n - Data generator `FeLiver_14_task1` is invalid.\n - Variable `p1_FeLiver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `FeRBC_15_task1` is invalid.\n - Variable `p1_FeRBC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeRBC']` are not defined. No namespaces are defined for the target.\n - Data generator `FeRest_16_task1` is invalid.\n - Variable `p1_FeRest_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeRest']` are not defined. No namespaces are defined for the target.\n - Data generator `FeSpleen_17_task1` is invalid.\n - Variable `p1_FeSpleen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeSpleen']` are not defined. No namespaces are defined for the target.\n - Data generator `Hepcidin_18_task1` is invalid.\n - Variable `p1_Hepcidin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Hepcidin']` are not defined. No namespaces are defined for the target.\n - Data generator `NTBI_19_task1` is invalid.\n - Variable `p1_NTBI_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='NTBI']` are not defined. No namespaces are defined for the target.\n - Data generator `Tf_20_task1` is invalid.\n - Variable `p1_Tf_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Tf']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Fe__g_1_task1` is invalid.\n - Variable `p2_Total_Fe__g_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Fe__g']` are not defined. No namespaces are defined for the target.\n - Data generator `TfSaturation_1_task1` is invalid.\n - Variable `p3_TfSaturation_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='TfSaturation']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Parmar2017_Deficient_NoTracer.sedml659257.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Parmar2017_Deficient_NoTracer.xml` has warnings.\r\n - 32 warnings of type SBML unit consistency (99505). The following is the first warning at line 1261, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Avogadro' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 32 warnings of type Modeling practice (80701). The following is the first warning at line 847, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kNTBI_Fe1Tf' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Parmar2017_Deficient_NoTracer.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Parmar2017_Deficient_NoTracer.xml` has warnings.\r\n - 32 warnings of type SBML unit consistency (99505). The following is the first warning at line 1261, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Avogadro' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 32 warnings of type Modeling practice (80701). The following is the first warning at line 847, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kNTBI_Fe1Tf' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Parmar2017_Deficient_NoTracer.sedml659257.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Parmar2017_Deficient_NoTracer.sedml` is invalid.\n - Data generator `FePlasma__particle_1_task1` is invalid.\n - Variable `p1_FePlasma__particle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='FePlasma__particle']` are not defined. No namespaces are defined for the target.\n - Data generator `FePlasma_conc_2_task1` is invalid.\n - Variable `p1_FePlasma_conc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='FePlasma_conc']` are not defined. No namespaces are defined for the target.\n - Data generator `TfSaturation_3_task1` is invalid.\n - Variable `p1_TfSaturation_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='TfSaturation']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Fe__conc_4_task1` is invalid.\n - Variable `p1_Total_Fe__conc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Fe__conc']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Fe__g_5_task1` is invalid.\n - Variable `p1_Total_Fe__g_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Fe__g']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Fe__particle_6_task1` is invalid.\n - Variable `p1_Total_Fe__particle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Fe__particle']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Tf__conc_7_task1` is invalid.\n - Variable `p1_Total_Tf__conc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Tf__conc']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Tf__mg_ml_8_task1` is invalid.\n - Variable `p1_Total_Tf__mg_ml_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Tf__mg_ml']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Tf__particle_9_task1` is invalid.\n - Variable `p1_Total_Tf__particle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Tf__particle']` are not defined. No namespaces are defined for the target.\n - Data generator `Fe1Tf_10_task1` is invalid.\n - Variable `p1_Fe1Tf_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Fe1Tf']` are not defined. No namespaces are defined for the target.\n - Data generator `Fe2Tf_11_task1` is invalid.\n - Variable `p1_Fe2Tf_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Fe2Tf']` are not defined. No namespaces are defined for the target.\n - Data generator `FeBM_12_task1` is invalid.\n - Variable `p1_FeBM_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeBM']` are not defined. No namespaces are defined for the target.\n - Data generator `FeDuo_13_task1` is invalid.\n - Variable `p1_FeDuo_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeDuo']` are not defined. No namespaces are defined for the target.\n - Data generator `FeLiver_14_task1` is invalid.\n - Variable `p1_FeLiver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `FeRBC_15_task1` is invalid.\n - Variable `p1_FeRBC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeRBC']` are not defined. No namespaces are defined for the target.\n - Data generator `FeRest_16_task1` is invalid.\n - Variable `p1_FeRest_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeRest']` are not defined. No namespaces are defined for the target.\n - Data generator `FeSpleen_17_task1` is invalid.\n - Variable `p1_FeSpleen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeSpleen']` are not defined. No namespaces are defined for the target.\n - Data generator `Hepcidin_18_task1` is invalid.\n - Variable `p1_Hepcidin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Hepcidin']` are not defined. No namespaces are defined for the target.\n - Data generator `NTBI_19_task1` is invalid.\n - Variable `p1_NTBI_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='NTBI']` are not defined. No namespaces are defined for the target.\n - Data generator `Tf_20_task1` is invalid.\n - Variable `p1_Tf_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Tf']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Fe__g_1_task1` is invalid.\n - Variable `p2_Total_Fe__g_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Fe__g']` are not defined. No namespaces are defined for the target.\n - Data generator `TfSaturation_1_task1` is invalid.\n - Variable `p3_TfSaturation_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='TfSaturation']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000738](https://www.ebi.ac.uk/biomodels/BIOMD0000000738)<br/><sup>Mouse Iron Distribution - Rich</sup></summary><sup>Mouse Iron Distribution - Rich iron diet (No Tracer)</sup></details>|<details><summary>pass</summary>[Parmar2017_Rich_NoTracer.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000738#Files)</details>|pass|<details><summary>pass</summary>[Parmar2017_Rich_NoTracer.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000738#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Parmar2017_Rich_NoTracer.sedml793356.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Parmar2017_Rich_NoTracer.xml` has warnings.\r\n - 32 warnings of type SBML unit consistency (99505). The following is the first warning at line 1261, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Avogadro' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 32 warnings of type Modeling practice (80701). The following is the first warning at line 847, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kNTBI_Fe1Tf' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Parmar2017_Rich_NoTracer.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Parmar2017_Rich_NoTracer.xml` has warnings.\r\n - 32 warnings of type SBML unit consistency (99505). The following is the first warning at line 1261, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Avogadro' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 32 warnings of type Modeling practice (80701). The following is the first warning at line 847, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kNTBI_Fe1Tf' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Parmar2017_Rich_NoTracer.sedml793356.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Parmar2017_Rich_NoTracer.sedml` is invalid.\n - Data generator `FeDuo_1_task1` is invalid.\n - Variable `p1_FeDuo_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeDuo']` are not defined. No namespaces are defined for the target.\n - Data generator `FeRBC_2_task1` is invalid.\n - Variable `p1_FeRBC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeRBC']` are not defined. No namespaces are defined for the target.\n - Data generator `FeSpleen_3_task1` is invalid.\n - Variable `p1_FeSpleen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeSpleen']` are not defined. No namespaces are defined for the target.\n - Data generator `FeLiver_4_task1` is invalid.\n - Variable `p1_FeLiver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `Tf_5_task1` is invalid.\n - Variable `p1_Tf_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Tf']` are not defined. No namespaces are defined for the target.\n - Data generator `Hepcidin_6_task1` is invalid.\n - Variable `p1_Hepcidin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Hepcidin']` are not defined. No namespaces are defined for the target.\n - Data generator `Fe2Tf_7_task1` is invalid.\n - Variable `p1_Fe2Tf_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Fe2Tf']` are not defined. No namespaces are defined for the target.\n - Data generator `Fe1Tf_8_task1` is invalid.\n - Variable `p1_Fe1Tf_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Fe1Tf']` are not defined. No namespaces are defined for the target.\n - Data generator `NTBI_9_task1` is invalid.\n - Variable `p1_NTBI_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='NTBI']` are not defined. No namespaces are defined for the target.\n - Data generator `FeRest_10_task1` is invalid.\n - Variable `p1_FeRest_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeRest']` are not defined. No namespaces are defined for the target.\n - Data generator `FeBM_11_task1` is invalid.\n - Variable `p1_FeBM_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeBM']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Fe__particle_12_task1` is invalid.\n - Variable `p1_Total_Fe__particle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Fe__particle']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Fe__conc_13_task1` is invalid.\n - Variable `p1_Total_Fe__conc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Fe__conc']` are not defined. No namespaces are defined for the target.\n - Data generator `FePlasma__particle_14_task1` is invalid.\n - Variable `p1_FePlasma__particle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='FePlasma__particle']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Fe__g_15_task1` is invalid.\n - Variable `p1_Total_Fe__g_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Fe__g']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Tf__particle_16_task1` is invalid.\n - Variable `p1_Total_Tf__particle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Tf__particle']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Tf__conc_17_task1` is invalid.\n - Variable `p1_Total_Tf__conc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Tf__conc']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Tf__mg_ml_18_task1` is invalid.\n - Variable `p1_Total_Tf__mg_ml_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Tf__mg_ml']` are not defined. No namespaces are defined for the target.\n - Data generator `FePlasma_conc_19_task1` is invalid.\n - Variable `p1_FePlasma_conc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='FePlasma_conc']` are not defined. No namespaces are defined for the target.\n - Data generator `TfSaturation_20_task1` is invalid.\n - Variable `p1_TfSaturation_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='TfSaturation']` are not defined. No namespaces are defined for the target.\n - Data generator `Hepcidin_1_task1` is invalid.\n - Variable `p2_Hepcidin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Hepcidin']` are not defined. No namespaces are defined for the target.\n - Data generator `TfSaturation_1_task1` is invalid.\n - Variable `p3_TfSaturation_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='TfSaturation']` are not defined. No namespaces are defined for the target.\n - Data generator `NTBI_1_task1` is invalid.\n - Variable `p4_NTBI_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='NTBI']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Fe__g_1_task1` is invalid.\n - Variable `p5_Total_Fe__g_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Fe__g']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Parmar2017_Rich_NoTracer.sedml274563.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Parmar2017_Rich_NoTracer.xml` has warnings.\r\n - 32 warnings of type SBML unit consistency (99505). The following is the first warning at line 1261, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Avogadro' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 32 warnings of type Modeling practice (80701). The following is the first warning at line 847, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kNTBI_Fe1Tf' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Parmar2017_Rich_NoTracer.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Parmar2017_Rich_NoTracer.xml` has warnings.\r\n - 32 warnings of type SBML unit consistency (99505). The following is the first warning at line 1261, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Avogadro' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 32 warnings of type Modeling practice (80701). The following is the first warning at line 847, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kNTBI_Fe1Tf' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Parmar2017_Rich_NoTracer.sedml274563.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Parmar2017_Rich_NoTracer.sedml` is invalid.\n - Data generator `FeDuo_1_task1` is invalid.\n - Variable `p1_FeDuo_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeDuo']` are not defined. No namespaces are defined for the target.\n - Data generator `FeRBC_2_task1` is invalid.\n - Variable `p1_FeRBC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeRBC']` are not defined. No namespaces are defined for the target.\n - Data generator `FeSpleen_3_task1` is invalid.\n - Variable `p1_FeSpleen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeSpleen']` are not defined. No namespaces are defined for the target.\n - Data generator `FeLiver_4_task1` is invalid.\n - Variable `p1_FeLiver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `Tf_5_task1` is invalid.\n - Variable `p1_Tf_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Tf']` are not defined. No namespaces are defined for the target.\n - Data generator `Hepcidin_6_task1` is invalid.\n - Variable `p1_Hepcidin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Hepcidin']` are not defined. No namespaces are defined for the target.\n - Data generator `Fe2Tf_7_task1` is invalid.\n - Variable `p1_Fe2Tf_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Fe2Tf']` are not defined. No namespaces are defined for the target.\n - Data generator `Fe1Tf_8_task1` is invalid.\n - Variable `p1_Fe1Tf_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Fe1Tf']` are not defined. No namespaces are defined for the target.\n - Data generator `NTBI_9_task1` is invalid.\n - Variable `p1_NTBI_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='NTBI']` are not defined. No namespaces are defined for the target.\n - Data generator `FeRest_10_task1` is invalid.\n - Variable `p1_FeRest_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeRest']` are not defined. No namespaces are defined for the target.\n - Data generator `FeBM_11_task1` is invalid.\n - Variable `p1_FeBM_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FeBM']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Fe__particle_12_task1` is invalid.\n - Variable `p1_Total_Fe__particle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Fe__particle']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Fe__conc_13_task1` is invalid.\n - Variable `p1_Total_Fe__conc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Fe__conc']` are not defined. No namespaces are defined for the target.\n - Data generator `FePlasma__particle_14_task1` is invalid.\n - Variable `p1_FePlasma__particle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='FePlasma__particle']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Fe__g_15_task1` is invalid.\n - Variable `p1_Total_Fe__g_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Fe__g']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Tf__particle_16_task1` is invalid.\n - Variable `p1_Total_Tf__particle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Tf__particle']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Tf__conc_17_task1` is invalid.\n - Variable `p1_Total_Tf__conc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Tf__conc']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Tf__mg_ml_18_task1` is invalid.\n - Variable `p1_Total_Tf__mg_ml_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Tf__mg_ml']` are not defined. No namespaces are defined for the target.\n - Data generator `FePlasma_conc_19_task1` is invalid.\n - Variable `p1_FePlasma_conc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='FePlasma_conc']` are not defined. No namespaces are defined for the target.\n - Data generator `TfSaturation_20_task1` is invalid.\n - Variable `p1_TfSaturation_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='TfSaturation']` are not defined. No namespaces are defined for the target.\n - Data generator `Hepcidin_1_task1` is invalid.\n - Variable `p2_Hepcidin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Hepcidin']` are not defined. No namespaces are defined for the target.\n - Data generator `TfSaturation_1_task1` is invalid.\n - Variable `p3_TfSaturation_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='TfSaturation']` are not defined. No namespaces are defined for the target.\n - Data generator `NTBI_1_task1` is invalid.\n - Variable `p4_NTBI_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='NTBI']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Fe__g_1_task1` is invalid.\n - Variable `p5_Total_Fe__g_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Fe__g']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000739](https://www.ebi.ac.uk/biomodels/BIOMD0000000739)<br/><sup>Bravo2012 - Modelling blood co</sup></summary><sup>Bravo2012 - Modelling blood coagulation factor Va inactivation by APC</sup></details>|<details><summary>pass</summary>[Bravo2012.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000739#Files)</details>|pass|<details><summary>pass</summary>[Bravo2012.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000739#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Bravo2012.sedml698670.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/Bravo2012.sedml698670.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Bravo2012.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - XDataReference from Curve `p1_curve_1_task1` cannot be resolved to a data generator with id `time_minutes_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_1_task1` cannot be resolved to a data generator with id `Curve_Va_1_task1` because multiple data generators have this id.\n - XDataReference from Curve `p2_curve_1_task1` cannot be resolved to a data generator with id `time_minutes_1_task1` because multiple data generators have this id.\n - XDataReference from Curve `p3_curve_1_task1` cannot be resolved to a data generator with id `time_minutes_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p3_curve_1_task1` cannot be resolved to a data generator with id `Va_307_506_1_task1` because multiple data generators have this id.\n - XDataReference from Curve `p5_curve_1_task1` cannot be resolved to a data generator with id `time_minutes_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p5_curve_1_task1` cannot be resolved to a data generator with id `Curve_Va_1_task1` because multiple data generators have this id.\n - XDataReference from Curve `p5_curve_2_task1` cannot be resolved to a data generator with id `time_minutes_2_task1` because multiple data generators have this id.\n - XDataReference from Curve `p6_curve_1_task1` cannot be resolved to a data generator with id `time_minutes_1_task1` because multiple data generators have this id.\n - XDataReference from Curve `p6_curve_2_task1` cannot be resolved to a data generator with id `time_minutes_2_task1` because multiple data generators have this id.\n - XDataReference from Curve `p7_curve_1_task1` cannot be resolved to a data generator with id `time_minutes_1_task1` because multiple data generators have this id.\n - XDataReference from Curve `p8_curve_1_task1` cannot be resolved to a data generator with id `time_minutes_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p8_curve_1_task1` cannot be resolved to a data generator with id `Va_307_506_1_task1` because multiple data generators have this id.\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Bravo2012.sedml328619.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Bravo2012.sedml328619.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Bravo2012.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - XDataReference from Curve `p1_curve_1_task1` cannot be resolved to a data generator with id `time_minutes_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_1_task1` cannot be resolved to a data generator with id `Curve_Va_1_task1` because multiple data generators have this id.\n - XDataReference from Curve `p2_curve_1_task1` cannot be resolved to a data generator with id `time_minutes_1_task1` because multiple data generators have this id.\n - XDataReference from Curve `p3_curve_1_task1` cannot be resolved to a data generator with id `time_minutes_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p3_curve_1_task1` cannot be resolved to a data generator with id `Va_307_506_1_task1` because multiple data generators have this id.\n - XDataReference from Curve `p5_curve_1_task1` cannot be resolved to a data generator with id `time_minutes_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p5_curve_1_task1` cannot be resolved to a data generator with id `Curve_Va_1_task1` because multiple data generators have this id.\n - XDataReference from Curve `p5_curve_2_task1` cannot be resolved to a data generator with id `time_minutes_2_task1` because multiple data generators have this id.\n - XDataReference from Curve `p6_curve_1_task1` cannot be resolved to a data generator with id `time_minutes_1_task1` because multiple data generators have this id.\n - XDataReference from Curve `p6_curve_2_task1` cannot be resolved to a data generator with id `time_minutes_2_task1` because multiple data generators have this id.\n - XDataReference from Curve `p7_curve_1_task1` cannot be resolved to a data generator with id `time_minutes_1_task1` because multiple data generators have this id.\n - XDataReference from Curve `p8_curve_1_task1` cannot be resolved to a data generator with id `time_minutes_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p8_curve_1_task1` cannot be resolved to a data generator with id `Va_307_506_1_task1` because multiple data generators have this id.\n'```</details>|
|<details><summary>[BIOMD0000000740](https://www.ebi.ac.uk/biomodels/BIOMD0000000740)<br/><sup>Panteleev2010 - Blood Coagulat</sup></summary><sup>Panteleev2010 - Blood Coagulation: Full Model</sup></details>|<details><summary>pass</summary>[Panteleev2010_full_model.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000740#Files)</details>|pass|<details><summary>pass</summary>[Panteleev2010_full_model.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000740#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Panteleev2010_full_model.sedml674420.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/Panteleev2010_full_model.sedml674420.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Panteleev2010_full_model.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - YDataReference from Curve `p1_curve_1_task1` cannot be resolved to a data generator with id `fibrin_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p2_curve_1_task1` cannot be resolved to a data generator with id `fibrin_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_1_task2` cannot be resolved to a data generator with id `fibrin_1_task2` because multiple data generators have this id.\n - YDataReference from Curve `p2_curve_1_task2` cannot be resolved to a data generator with id `fibrin_1_task2` because multiple data generators have this id.\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Panteleev2010_full_model.sedml668955.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Panteleev2010_full_model.sedml668955.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Panteleev2010_full_model.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - YDataReference from Curve `p1_curve_1_task1` cannot be resolved to a data generator with id `fibrin_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p2_curve_1_task1` cannot be resolved to a data generator with id `fibrin_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_1_task2` cannot be resolved to a data generator with id `fibrin_1_task2` because multiple data generators have this id.\n - YDataReference from Curve `p2_curve_1_task2` cannot be resolved to a data generator with id `fibrin_1_task2` because multiple data generators have this id.\n'```</details>|
|<details><summary>[BIOMD0000000741](https://www.ebi.ac.uk/biomodels/BIOMD0000000741)<br/><sup>Eftimie2018 - Cancer and Immun</sup></summary><sup>Eftimie2018 - Cancer and Immune biomarkers</sup></details>|<details><summary>pass</summary>[Eftimie2018.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000741#Files)</details>|FAIL|<details><summary>FAIL</summary>[Eftimie2018.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000741#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Eftimie2018.sedml104404.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Eftimie2018.sedml` has warnings.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n`/root/in/Eftimie2018.sedml104404.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Eftimie2018.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmpx8096iew/./model1.xml` is not a file.\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Eftimie2018.sedml707618.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Eftimie2018.sedml` has warnings.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Eftimie2018.sedml707618.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Eftimie2018.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmp0s5o1oa7/./model1.xml` is not a file.\n'```</details>|
|<details><summary>[BIOMD0000000742](https://www.ebi.ac.uk/biomodels/BIOMD0000000742)<br/><sup>Garcia2018basic - cancer and i</sup></summary><sup>Garcia2018basic - cancer and immune cell count basic model</sup></details>|<details><summary>pass</summary>[Garcia2018basic.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000742#Files)</details>|FAIL|<details><summary>FAIL</summary>[Garcia2018basic.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000742#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Garcia2018basic.sedml61039.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Garcia2018basic.sedml` has warnings.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n`/root/in/Garcia2018basic.sedml61039.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Garcia2018basic.sedml` is invalid.\n - Model `model2` is invalid.\n - The model file `model2.xml` is invalid.\n - `/tmp/tmp8sht5kvh/./model2.xml` is not a file.\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Garcia2018basic.sedml188636.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Garcia2018basic.sedml` has warnings.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Garcia2018basic.sedml188636.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Garcia2018basic.sedml` is invalid.\n - Model `model2` is invalid.\n - The model file `model2.xml` is invalid.\n - `/tmp/tmpufuoc9hw/./model2.xml` is not a file.\n'```</details>|
|<details><summary>[BIOMD0000000743](https://www.ebi.ac.uk/biomodels/BIOMD0000000743)<br/><sup>Gallaher2018 - TumorImmune dy</sup></summary><sup>Gallaher2018 - TumorImmune dynamics in multiple myeloma</sup></details>|<details><summary>pass</summary>[Gallaher2018.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000743#Files)</details>|FAIL|<details><summary>pass</summary>[Gallaher2018.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000743#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Gallaher2018.sedml257756.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Gallaher2018.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (10501). The following is the first warning at line 2023, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'anm * N / (bnm + N) + acm * Tc / (bcm + Tc) + acnm * N * Tc / ((bnm + N) * (bcm + Tc))' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 2 warnings of type SBML unit consistency (10503). The following is the first warning at line 2098, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'compartment * M_source_0(sm)' in the <kineticLaw> element of the <reaction> with id 'M_source' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 2023, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * M_breakup_0(M, anm, N, bnm, acm, Tc, bcm, acnm, amm, bmm, arm, Tr, brm, dm)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type SBML unit consistency (10541). The following is the first warning at line 2084, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'M_decay') are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = -1).\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Gallaher2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Gallaher2018.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (10501). The following is the first warning at line 2023, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'anm * N / (bnm + N) + acm * Tc / (bcm + Tc) + acnm * N * Tc / ((bnm + N) * (bcm + Tc))' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 2 warnings of type SBML unit consistency (10503). The following is the first warning at line 2098, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'compartment * M_source_0(sm)' in the <kineticLaw> element of the <reaction> with id 'M_source' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 2023, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * M_breakup_0(M, anm, N, bnm, acm, Tc, bcm, acnm, amm, bmm, arm, Tr, brm, dm)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type SBML unit consistency (10541). The following is the first warning at line 2084, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'M_decay') are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = -1).\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n`/root/in/Gallaher2018.sedml257756.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Gallaher2018.sedml` is invalid.\n - Output `plot_1_task1` is invalid.\n - Plot must have at least one curve.\n - Output `plot_2_task1` is invalid.\n - Plot must have at least one curve.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Gallaher2018.sedml541504.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Gallaher2018.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (10501). The following is the first warning at line 2023, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'anm * N / (bnm + N) + acm * Tc / (bcm + Tc) + acnm * N * Tc / ((bnm + N) * (bcm + Tc))' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 2 warnings of type SBML unit consistency (10503). The following is the first warning at line 2098, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'compartment * M_source_0(sm)' in the <kineticLaw> element of the <reaction> with id 'M_source' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 2023, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * M_breakup_0(M, anm, N, bnm, acm, Tc, bcm, acnm, amm, bmm, arm, Tr, brm, dm)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type SBML unit consistency (10541). The following is the first warning at line 2084, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'M_decay') are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = -1).\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Gallaher2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Gallaher2018.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (10501). The following is the first warning at line 2023, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'anm * N / (bnm + N) + acm * Tc / (bcm + Tc) + acnm * N * Tc / ((bnm + N) * (bcm + Tc))' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 2 warnings of type SBML unit consistency (10503). The following is the first warning at line 2098, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'compartment * M_source_0(sm)' in the <kineticLaw> element of the <reaction> with id 'M_source' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 2023, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * M_breakup_0(M, anm, N, bnm, acm, Tc, bcm, acnm, amm, bmm, arm, Tr, brm, dm)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type SBML unit consistency (10541). The following is the first warning at line 2084, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'M_decay') are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = -1).\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Gallaher2018.sedml541504.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Gallaher2018.sedml` is invalid.\n - Output `plot_1_task1` is invalid.\n - Plot must have at least one curve.\n - Output `plot_2_task1` is invalid.\n - Plot must have at least one curve.\n"```</details>|
|[BIOMD0000000744](https://www.ebi.ac.uk/biomodels/BIOMD0000000744)<br/><sup>Hu2019 - Pancreatic cancer dynamics</sup>|<details><summary>pass</summary>[Hu2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000744#Files)</details>|FAIL|<details><summary>pass</summary>[Hu2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000744#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Hu2019.sedml886323.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Hu2019.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (10501). The following is the first warning at line 1869, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'm1 + w' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (10503). The following is the first warning at line 1991, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'Pancreatic_tumor * PSC_death_0(u2, y)' in the <kineticLaw> element of the <reaction> with id 'PSC_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 1761, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Pancreatic_tumor * PCC_growth_0(r1, x, b1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type SBML unit consistency (10541). The following is the first warning at line 1869, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'PCC_death') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Hu2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Hu2019.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (10501). The following is the first warning at line 1869, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'm1 + w' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (10503). The following is the first warning at line 1991, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'Pancreatic_tumor * PSC_death_0(u2, y)' in the <kineticLaw> element of the <reaction> with id 'PSC_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 1761, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Pancreatic_tumor * PCC_growth_0(r1, x, b1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type SBML unit consistency (10541). The following is the first warning at line 1869, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'PCC_death') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n`/root/in/Hu2019.sedml886323.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Hu2019.sedml` is invalid.\n - Output `plot_1_task1` is invalid.\n - Plot must have at least one curve.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Hu2019.sedml829754.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Hu2019.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (10501). The following is the first warning at line 1869, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'm1 + w' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (10503). The following is the first warning at line 1991, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'Pancreatic_tumor * PSC_death_0(u2, y)' in the <kineticLaw> element of the <reaction> with id 'PSC_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 1761, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Pancreatic_tumor * PCC_growth_0(r1, x, b1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type SBML unit consistency (10541). The following is the first warning at line 1869, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'PCC_death') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Hu2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Hu2019.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (10501). The following is the first warning at line 1869, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'm1 + w' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (10503). The following is the first warning at line 1991, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'Pancreatic_tumor * PSC_death_0(u2, y)' in the <kineticLaw> element of the <reaction> with id 'PSC_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 1761, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Pancreatic_tumor * PCC_growth_0(r1, x, b1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type SBML unit consistency (10541). The following is the first warning at line 1869, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'PCC_death') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Hu2019.sedml829754.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Hu2019.sedml` is invalid.\n - Output `plot_1_task1` is invalid.\n - Plot must have at least one curve.\n"```</details>|
|<details><summary>[BIOMD0000000745](https://www.ebi.ac.uk/biomodels/BIOMD0000000745)<br/><sup>Jarrett2018 - trastuzumab-indu</sup></summary><sup>Jarrett2018 - trastuzumab-induced immune response in murine HER2+ breast cancer model</sup></details>|<details><summary>pass</summary>[Jarrett2018.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000745#Files)</details>|FAIL|<details><summary>pass</summary>[Jarrett2018.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000745#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Jarrett2018.sedml332568.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Jarrett2018.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 1703, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'beta + beta * V * N' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (10503). The following is the first warning at line 1434, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor * i__1(N, V, an, av)' in the <kineticLaw> element of the <reaction> with id 'immune_stimulation' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 9 warnings of type SBML unit consistency (10541). The following is the first warning at line 1355, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_death') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 1, scale = -3), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - 1 warning of type SBML unit consistency (99505). The following is the first warning at line 1418, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tumor * t_g_1(H, T, g, rho)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Jarrett2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Jarrett2018.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 1703, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'beta + beta * V * N' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (10503). The following is the first warning at line 1434, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor * i__1(N, V, an, av)' in the <kineticLaw> element of the <reaction> with id 'immune_stimulation' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 9 warnings of type SBML unit consistency (10541). The following is the first warning at line 1355, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_death') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 1, scale = -3), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - 1 warning of type SBML unit consistency (99505). The following is the first warning at line 1418, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tumor * t_g_1(H, T, g, rho)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n`/root/in/Jarrett2018.sedml332568.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Jarrett2018.sedml` is invalid.\n - Output `plot_1_task1` is invalid.\n - Plot must have at least one curve.\n - Output `plot_2_task1` is invalid.\n - Plot must have at least one curve.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Jarrett2018.sedml560216.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Jarrett2018.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 1703, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'beta + beta * V * N' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (10503). The following is the first warning at line 1434, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor * i__1(N, V, an, av)' in the <kineticLaw> element of the <reaction> with id 'immune_stimulation' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 9 warnings of type SBML unit consistency (10541). The following is the first warning at line 1355, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_death') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 1, scale = -3), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - 1 warning of type SBML unit consistency (99505). The following is the first warning at line 1418, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tumor * t_g_1(H, T, g, rho)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Jarrett2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Jarrett2018.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 1703, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'beta + beta * V * N' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (10503). The following is the first warning at line 1434, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor * i__1(N, V, an, av)' in the <kineticLaw> element of the <reaction> with id 'immune_stimulation' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 9 warnings of type SBML unit consistency (10541). The following is the first warning at line 1355, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_death') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 1, scale = -3), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - 1 warning of type SBML unit consistency (99505). The following is the first warning at line 1418, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tumor * t_g_1(H, T, g, rho)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Jarrett2018.sedml560216.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Jarrett2018.sedml` is invalid.\n - Output `plot_1_task1` is invalid.\n - Plot must have at least one curve.\n - Output `plot_2_task1` is invalid.\n - Plot must have at least one curve.\n"```</details>|
|<details><summary>[BIOMD0000000746](https://www.ebi.ac.uk/biomodels/BIOMD0000000746)<br/><sup>Saad2017 - immune checkpoint a</sup></summary><sup>Saad2017 - immune checkpoint and BCG in superficial bladder cancer</sup></details>|<details><summary>pass</summary>[Saad2017.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000746#Files)</details>|FAIL|<details><summary>pass</summary>[Saad2017.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000746#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Saad2017.sedml80381.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Saad2017.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 1053, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'P + k' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 7 warnings of type SBML unit consistency (10503). The following is the first warning at line 1007, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'bladder_cancer_tme * cancer_killing_0(alpha1, E, C, P, k)' in the <kineticLaw> element of the <reaction> with id 'cancer_killing' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 11 warnings of type SBML unit consistency (10541). The following is the first warning at line 993, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'cancer_growth') are second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 1, multiplier = 1, scale = -3).\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Saad2017.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Saad2017.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 1053, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'P + k' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 7 warnings of type SBML unit consistency (10503). The following is the first warning at line 1007, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'bladder_cancer_tme * cancer_killing_0(alpha1, E, C, P, k)' in the <kineticLaw> element of the <reaction> with id 'cancer_killing' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 11 warnings of type SBML unit consistency (10541). The following is the first warning at line 993, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'cancer_growth') are second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 1, multiplier = 1, scale = -3).\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n`/root/in/Saad2017.sedml80381.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Saad2017.sedml` is invalid.\n - Output `plot_1_task1` is invalid.\n - Plot must have at least one curve.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Saad2017.sedml191.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Saad2017.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 1053, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'P + k' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 7 warnings of type SBML unit consistency (10503). The following is the first warning at line 1007, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'bladder_cancer_tme * cancer_killing_0(alpha1, E, C, P, k)' in the <kineticLaw> element of the <reaction> with id 'cancer_killing' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 11 warnings of type SBML unit consistency (10541). The following is the first warning at line 993, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'cancer_growth') are second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 1, multiplier = 1, scale = -3).\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Saad2017.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Saad2017.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 1053, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'P + k' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 7 warnings of type SBML unit consistency (10503). The following is the first warning at line 1007, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'bladder_cancer_tme * cancer_killing_0(alpha1, E, C, P, k)' in the <kineticLaw> element of the <reaction> with id 'cancer_killing' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 11 warnings of type SBML unit consistency (10541). The following is the first warning at line 993, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'cancer_growth') are second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 1, multiplier = 1, scale = -3).\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Saad2017.sedml191.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Saad2017.sedml` is invalid.\n - Output `plot_1_task1` is invalid.\n - Plot must have at least one curve.\n"```</details>|
|<details><summary>[BIOMD0000000747](https://www.ebi.ac.uk/biomodels/BIOMD0000000747)<br/><sup>Nagashima2002 - Simulating blo</sup></summary><sup>Nagashima2002 - Simulating blood coagulation inhibitory effects</sup></details>|<details><summary>pass</summary>[Nagashima2002.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000747#Files)</details>|pass|<details><summary>pass</summary>[Nagashima2002.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000747#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Nagashima2002.sedml327433.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Nagashima2002.xml` has warnings.\r\n - 31 warnings of type SBML unit consistency (99505). The following is the first warning at line 1796, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '3' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 44 warnings of type Modeling practice (80701). The following is the first warning at line 1129, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k01' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Nagashima2002.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Nagashima2002.xml` has warnings.\r\n - 31 warnings of type SBML unit consistency (99505). The following is the first warning at line 1796, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '3' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 44 warnings of type Modeling practice (80701). The following is the first warning at line 1129, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k01' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Nagashima2002.sedml327433.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Nagashima2002.sedml` is invalid.\n - Data generator `IIa_1_task1` is invalid.\n - Variable `p1_IIa_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IIa']` are not defined. No namespaces are defined for the target.\n - Data generator `Fibrin_1_task1` is invalid.\n - Variable `p2_Fibrin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Fibrin']` are not defined. No namespaces are defined for the target.\n - Data generator `IIa_Inhibitor_1_task1` is invalid.\n - Variable `p3_IIa_Inhibitor_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IIa_Inhibitor']` are not defined. No namespaces are defined for the target.\n - Data generator `Xa_Inhibitor_2_task1` is invalid.\n - Variable `p3_Xa_Inhibitor_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Xa_Inhibitor']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Nagashima2002.sedml51612.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Nagashima2002.xml` has warnings.\r\n - 31 warnings of type SBML unit consistency (99505). The following is the first warning at line 1796, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '3' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 44 warnings of type Modeling practice (80701). The following is the first warning at line 1129, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k01' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Nagashima2002.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Nagashima2002.xml` has warnings.\r\n - 31 warnings of type SBML unit consistency (99505). The following is the first warning at line 1796, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '3' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 44 warnings of type Modeling practice (80701). The following is the first warning at line 1129, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k01' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Nagashima2002.sedml51612.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Nagashima2002.sedml` is invalid.\n - Data generator `IIa_1_task1` is invalid.\n - Variable `p1_IIa_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IIa']` are not defined. No namespaces are defined for the target.\n - Data generator `Fibrin_1_task1` is invalid.\n - Variable `p2_Fibrin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Fibrin']` are not defined. No namespaces are defined for the target.\n - Data generator `IIa_Inhibitor_1_task1` is invalid.\n - Variable `p3_IIa_Inhibitor_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IIa_Inhibitor']` are not defined. No namespaces are defined for the target.\n - Data generator `Xa_Inhibitor_2_task1` is invalid.\n - Variable `p3_Xa_Inhibitor_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Xa_Inhibitor']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000748](https://www.ebi.ac.uk/biomodels/BIOMD0000000748)<br/><sup>Phan2017 - innate immune in on</sup></summary><sup>Phan2017 - innate immune in oncolytic virotherapy</sup></details>|<details><summary>pass</summary>[Phan2017.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000748#Files)</details>|FAIL|<details><summary>pass</summary>[Phan2017.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000748#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Phan2017.sedml147353.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Phan2017.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (10503). The following is the first warning at line 977, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * t_d(r, x)' in the <kineticLaw> element of the <reaction> with id 'tumor_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 11 warnings of type SBML unit consistency (10541). The following is the first warning at line 963, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_growth') are mole (exponent = 1, multiplier = 0.001, scale = 0).\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Phan2017.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Phan2017.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (10503). The following is the first warning at line 977, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * t_d(r, x)' in the <kineticLaw> element of the <reaction> with id 'tumor_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 11 warnings of type SBML unit consistency (10541). The following is the first warning at line 963, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_growth') are mole (exponent = 1, multiplier = 0.001, scale = 0).\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n`/root/in/Phan2017.sedml147353.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Phan2017.sedml` is invalid.\n - Output `plot_1_task1` is invalid.\n - Plot must have at least one curve.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Phan2017.sedml499187.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Phan2017.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (10503). The following is the first warning at line 977, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * t_d(r, x)' in the <kineticLaw> element of the <reaction> with id 'tumor_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 11 warnings of type SBML unit consistency (10541). The following is the first warning at line 963, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_growth') are mole (exponent = 1, multiplier = 0.001, scale = 0).\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Phan2017.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Phan2017.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (10503). The following is the first warning at line 977, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * t_d(r, x)' in the <kineticLaw> element of the <reaction> with id 'tumor_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 11 warnings of type SBML unit consistency (10541). The following is the first warning at line 963, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_growth') are mole (exponent = 1, multiplier = 0.001, scale = 0).\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Phan2017.sedml499187.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Phan2017.sedml` is invalid.\n - Output `plot_1_task1` is invalid.\n - Plot must have at least one curve.\n"```</details>|
|<details><summary>[BIOMD0000000749](https://www.ebi.ac.uk/biomodels/BIOMD0000000749)<br/><sup>Reppas2015 - tumor control via</sup></summary><sup>Reppas2015 - tumor control via alternating immunostimulating and immunosuppressive phases</sup></details>|<details><summary>pass</summary>[Reppas2015.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000749#Files)</details>|FAIL|<details><summary>pass</summary>[Reppas2015.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000749#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Reppas2015.sedml674998.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Reppas2015.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (10501). The following is the first warning at line 1239, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'k + R * R * R' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 4 warnings of type SBML unit consistency (10503). The following is the first warning at line 1200, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * stimulation(r, R, E, k)' in the <kineticLaw> element of the <reaction> with id 'effector_stimulation' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 951, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'E / 100000' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10512). The following is the first warning at line 960, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <species>, the units of the rule's right-hand side are expected to be consistent with the units of the species' quantity.\r\n Reference: L3V1 Section 4.9.3\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3) but the units returned by the <math> expression of the <assignmentRule> with variable 'f' are dimensionless (exponent = 1, multiplier = 1, scale = 0).\r\n - 5 warnings of type SBML unit consistency (10541). The following is the first warning at line 1132, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_killing') are litre (exponent = -2, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 3, multiplier = 0.001, scale = 0).\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Reppas2015.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Reppas2015.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (10501). The following is the first warning at line 1239, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'k + R * R * R' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 4 warnings of type SBML unit consistency (10503). The following is the first warning at line 1200, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * stimulation(r, R, E, k)' in the <kineticLaw> element of the <reaction> with id 'effector_stimulation' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 951, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'E / 100000' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10512). The following is the first warning at line 960, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <species>, the units of the rule's right-hand side are expected to be consistent with the units of the species' quantity.\r\n Reference: L3V1 Section 4.9.3\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3) but the units returned by the <math> expression of the <assignmentRule> with variable 'f' are dimensionless (exponent = 1, multiplier = 1, scale = 0).\r\n - 5 warnings of type SBML unit consistency (10541). The following is the first warning at line 1132, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_killing') are litre (exponent = -2, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 3, multiplier = 0.001, scale = 0).\x1b[0m\r\n\r\n`/root/in/Reppas2015.sedml674998.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Reppas2015.sedml` is invalid.\n - Data generator `E_0_1_task1` is invalid.\n - Variable `p1_E_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='E_0']` are not defined. No namespaces are defined for the target.\n - Data generator `R_2_task1` is invalid.\n - Variable `p1_R_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='R']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Reppas2015.sedml690945.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Reppas2015.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (10501). The following is the first warning at line 1239, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'k + R * R * R' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 4 warnings of type SBML unit consistency (10503). The following is the first warning at line 1200, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * stimulation(r, R, E, k)' in the <kineticLaw> element of the <reaction> with id 'effector_stimulation' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 951, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'E / 100000' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10512). The following is the first warning at line 960, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <species>, the units of the rule's right-hand side are expected to be consistent with the units of the species' quantity.\r\n Reference: L3V1 Section 4.9.3\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3) but the units returned by the <math> expression of the <assignmentRule> with variable 'f' are dimensionless (exponent = 1, multiplier = 1, scale = 0).\r\n - 5 warnings of type SBML unit consistency (10541). The following is the first warning at line 1132, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_killing') are litre (exponent = -2, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 3, multiplier = 0.001, scale = 0).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Reppas2015.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Reppas2015.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (10501). The following is the first warning at line 1239, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'k + R * R * R' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 4 warnings of type SBML unit consistency (10503). The following is the first warning at line 1200, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * stimulation(r, R, E, k)' in the <kineticLaw> element of the <reaction> with id 'effector_stimulation' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 951, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'E / 100000' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10512). The following is the first warning at line 960, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <species>, the units of the rule's right-hand side are expected to be consistent with the units of the species' quantity.\r\n Reference: L3V1 Section 4.9.3\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3) but the units returned by the <math> expression of the <assignmentRule> with variable 'f' are dimensionless (exponent = 1, multiplier = 1, scale = 0).\r\n - 5 warnings of type SBML unit consistency (10541). The following is the first warning at line 1132, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_killing') are litre (exponent = -2, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 3, multiplier = 0.001, scale = 0).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Reppas2015.sedml690945.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Reppas2015.sedml` is invalid.\n - Data generator `E_0_1_task1` is invalid.\n - Variable `p1_E_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='E_0']` are not defined. No namespaces are defined for the target.\n - Data generator `R_2_task1` is invalid.\n - Variable `p1_R_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='R']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000750](https://www.ebi.ac.uk/biomodels/BIOMD0000000750)<br/><sup>Lolas2016 - tumour-induced neo</sup></summary><sup>Lolas2016 - tumour-induced neoneurogenesis and perineural tumour growth</sup></details>|<details><summary>pass</summary>[Lolas2016.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000750#Files)</details>|FAIL|<details><summary>pass</summary>[Lolas2016.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000750#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Lolas2016.sedml854732.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Lolas2016.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (10501). The following is the first warning at line 2297, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'rtp + G / (t1 + t2 * G)' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 20 warnings of type SBML unit consistency (10503). The following is the first warning at line 2432, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * _1nd_growth(rtm, Tm)' in the <kineticLaw> element of the <reaction> with id 'migrating_tumor_cell_growth' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 2239, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'theta1 / (1 + theta2 * Nn)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10513). The following is the first warning at line 2239, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are metre (exponent = -3, multiplier = 1, scale = -3) but the units returned by the <math> expression of the <assignmentRule> with variable 'ONn' are metre (exponent = -6, multiplier = 0.001, scale = 0), mole (exponent = -1, multiplier = 1, scale = -3), litre (exponent = 1, multiplier = 1, scale = -3).\r\n - 21 warnings of type SBML unit consistency (10541). The following is the first warning at line 2414, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_migration') are mole (exponent = 1, multiplier = 1, scale = -3).\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Lolas2016.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Lolas2016.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (10501). The following is the first warning at line 2297, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'rtp + G / (t1 + t2 * G)' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 20 warnings of type SBML unit consistency (10503). The following is the first warning at line 2432, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * _1nd_growth(rtm, Tm)' in the <kineticLaw> element of the <reaction> with id 'migrating_tumor_cell_growth' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 2239, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'theta1 / (1 + theta2 * Nn)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10513). The following is the first warning at line 2239, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are metre (exponent = -3, multiplier = 1, scale = -3) but the units returned by the <math> expression of the <assignmentRule> with variable 'ONn' are metre (exponent = -6, multiplier = 0.001, scale = 0), mole (exponent = -1, multiplier = 1, scale = -3), litre (exponent = 1, multiplier = 1, scale = -3).\r\n - 21 warnings of type SBML unit consistency (10541). The following is the first warning at line 2414, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_migration') are mole (exponent = 1, multiplier = 1, scale = -3).\x1b[0m\r\n\r\n`/root/in/Lolas2016.sedml854732.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Lolas2016.sedml` is invalid.\n - Data generator `A_1_task1` is invalid.\n - Variable `p1_A_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='A']` are not defined. No namespaces are defined for the target.\n - Data generator `Na_2_task1` is invalid.\n - Variable `p1_Na_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Na']` are not defined. No namespaces are defined for the target.\n - Data generator `G_3_task1` is invalid.\n - Variable `p1_G_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='G']` are not defined. No namespaces are defined for the target.\n - Data generator `Nn_4_task1` is invalid.\n - Variable `p1_Nn_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Nn']` are not defined. No namespaces are defined for the target.\n - Data generator `P_5_task1` is invalid.\n - Variable `p1_P_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='P']` are not defined. No namespaces are defined for the target.\n - Data generator `S_6_task1` is invalid.\n - Variable `p1_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S']` are not defined. No namespaces are defined for the target.\n - Data generator `Tm_7_task1` is invalid.\n - Variable `p1_Tm_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Tm']` are not defined. No namespaces are defined for the target.\n - Data generator `Tp_8_task1` is invalid.\n - Variable `p1_Tp_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Tp']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Lolas2016.sedml139447.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Lolas2016.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (10501). The following is the first warning at line 2297, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'rtp + G / (t1 + t2 * G)' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 20 warnings of type SBML unit consistency (10503). The following is the first warning at line 2432, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * _1nd_growth(rtm, Tm)' in the <kineticLaw> element of the <reaction> with id 'migrating_tumor_cell_growth' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 2239, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'theta1 / (1 + theta2 * Nn)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10513). The following is the first warning at line 2239, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are metre (exponent = -3, multiplier = 1, scale = -3) but the units returned by the <math> expression of the <assignmentRule> with variable 'ONn' are metre (exponent = -6, multiplier = 0.001, scale = 0), mole (exponent = -1, multiplier = 1, scale = -3), litre (exponent = 1, multiplier = 1, scale = -3).\r\n - 21 warnings of type SBML unit consistency (10541). The following is the first warning at line 2414, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_migration') are mole (exponent = 1, multiplier = 1, scale = -3).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Lolas2016.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Lolas2016.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (10501). The following is the first warning at line 2297, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'rtp + G / (t1 + t2 * G)' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 20 warnings of type SBML unit consistency (10503). The following is the first warning at line 2432, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * _1nd_growth(rtm, Tm)' in the <kineticLaw> element of the <reaction> with id 'migrating_tumor_cell_growth' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 2239, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'theta1 / (1 + theta2 * Nn)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10513). The following is the first warning at line 2239, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are metre (exponent = -3, multiplier = 1, scale = -3) but the units returned by the <math> expression of the <assignmentRule> with variable 'ONn' are metre (exponent = -6, multiplier = 0.001, scale = 0), mole (exponent = -1, multiplier = 1, scale = -3), litre (exponent = 1, multiplier = 1, scale = -3).\r\n - 21 warnings of type SBML unit consistency (10541). The following is the first warning at line 2414, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_migration') are mole (exponent = 1, multiplier = 1, scale = -3).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Lolas2016.sedml139447.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Lolas2016.sedml` is invalid.\n - Data generator `A_1_task1` is invalid.\n - Variable `p1_A_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='A']` are not defined. No namespaces are defined for the target.\n - Data generator `Na_2_task1` is invalid.\n - Variable `p1_Na_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Na']` are not defined. No namespaces are defined for the target.\n - Data generator `G_3_task1` is invalid.\n - Variable `p1_G_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='G']` are not defined. No namespaces are defined for the target.\n - Data generator `Nn_4_task1` is invalid.\n - Variable `p1_Nn_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Nn']` are not defined. No namespaces are defined for the target.\n - Data generator `P_5_task1` is invalid.\n - Variable `p1_P_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='P']` are not defined. No namespaces are defined for the target.\n - Data generator `S_6_task1` is invalid.\n - Variable `p1_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S']` are not defined. No namespaces are defined for the target.\n - Data generator `Tm_7_task1` is invalid.\n - Variable `p1_Tm_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Tm']` are not defined. No namespaces are defined for the target.\n - Data generator `Tp_8_task1` is invalid.\n - Variable `p1_Tp_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Tp']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000751](https://www.ebi.ac.uk/biomodels/BIOMD0000000751)<br/><sup>Wilkie2013b - immune-induced c</sup></summary><sup>Wilkie2013b - immune-induced cancer dormancy and immune evasion-basic</sup></details>|<details><summary>pass</summary>[Wilkie2013b.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000751#Files)</details>|FAIL|<details><summary>pass</summary>[Wilkie2013b.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000751#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Wilkie2013b.sedml835573.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Wilkie2013b.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 613, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'Ie + u * C * I' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 560, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '0.000105 * exp(-time / 168.1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 720, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'cancer_killing') are litre (exponent = -1, multiplier = 0.001, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Wilkie2013b.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Wilkie2013b.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 613, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'Ie + u * C * I' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 560, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '0.000105 * exp(-time / 168.1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 720, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'cancer_killing') are litre (exponent = -1, multiplier = 0.001, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\x1b[0m\r\n\r\n`/root/in/Wilkie2013b.sedml835573.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Wilkie2013b.sedml` is invalid.\n - Data generator `C_1_task1` is invalid.\n - Variable `p1_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C']` are not defined. No namespaces are defined for the target.\n - Data generator `I_2_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Wilkie2013b.sedml694921.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Wilkie2013b.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 613, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'Ie + u * C * I' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 560, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '0.000105 * exp(-time / 168.1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 720, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'cancer_killing') are litre (exponent = -1, multiplier = 0.001, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Wilkie2013b.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Wilkie2013b.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 613, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'Ie + u * C * I' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 560, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '0.000105 * exp(-time / 168.1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 720, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'cancer_killing') are litre (exponent = -1, multiplier = 0.001, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Wilkie2013b.sedml694921.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Wilkie2013b.sedml` is invalid.\n - Data generator `C_1_task1` is invalid.\n - Variable `p1_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C']` are not defined. No namespaces are defined for the target.\n - Data generator `I_2_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000752](https://www.ebi.ac.uk/biomodels/BIOMD0000000752)<br/><sup>Wilkie2013r - immune-induced c</sup></summary><sup>Wilkie2013r - immune-induced cancer dormancy and immune evasion-resistance</sup></details>|<details><summary>pass</summary>[Wilkie2013r.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000752#Files)</details>|FAIL|<details><summary>pass</summary>[Wilkie2013r.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000752#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Wilkie2013r.sedml548120.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Wilkie2013r.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 752, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tumor_microenvironment * growth_immune(y, I, Ie, u, C, R)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 864, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'cancer_killing') are litre (exponent = -1, multiplier = 0.001, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Wilkie2013r.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Wilkie2013r.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 752, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tumor_microenvironment * growth_immune(y, I, Ie, u, C, R)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 864, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'cancer_killing') are litre (exponent = -1, multiplier = 0.001, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\x1b[0m\r\n\r\n`/root/in/Wilkie2013r.sedml548120.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Wilkie2013r.sedml` is invalid.\n - Data generator `I_1_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `R_2_task1` is invalid.\n - Variable `p1_R_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='R']` are not defined. No namespaces are defined for the target.\n - Data generator `C_3_task1` is invalid.\n - Variable `p1_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Wilkie2013r.sedml366963.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Wilkie2013r.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 752, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tumor_microenvironment * growth_immune(y, I, Ie, u, C, R)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 864, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'cancer_killing') are litre (exponent = -1, multiplier = 0.001, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Wilkie2013r.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Wilkie2013r.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 752, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tumor_microenvironment * growth_immune(y, I, Ie, u, C, R)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 864, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'cancer_killing') are litre (exponent = -1, multiplier = 0.001, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Wilkie2013r.sedml366963.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Wilkie2013r.sedml` is invalid.\n - Data generator `I_1_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `R_2_task1` is invalid.\n - Variable `p1_R_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='R']` are not defined. No namespaces are defined for the target.\n - Data generator `C_3_task1` is invalid.\n - Variable `p1_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000753](https://www.ebi.ac.uk/biomodels/BIOMD0000000753)<br/><sup>Figueredo2013/1 - immunointera</sup></summary><sup>Figueredo2013/1 - immunointeraction base model</sup></details>|<details><summary>pass</summary>[Figueredo2013:1.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000753#Files)</details>|FAIL|<details><summary>pass</summary>[Figueredo2013:1.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000753#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Figueredo2013:1.sedml724177.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Figueredo2013:1.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 849, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'g + T' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (10503). The following is the first warning at line 705, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * t_d(a, b, T)' in the <kineticLaw> element of the <reaction> with id 'tumor_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 7 warnings of type SBML unit consistency (10541). The following is the first warning at line 691, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_growth') are mole (exponent = 1, multiplier = 0.001, scale = 0).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 100.3333333233\r\n - Number of steps: 301\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Figueredo2013:1.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Figueredo2013:1.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 849, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'g + T' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (10503). The following is the first warning at line 705, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * t_d(a, b, T)' in the <kineticLaw> element of the <reaction> with id 'tumor_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 7 warnings of type SBML unit consistency (10541). The following is the first warning at line 691, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_growth') are mole (exponent = 1, multiplier = 0.001, scale = 0).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 100.3333333233\r\n - Number of steps: 301\x1b[0m\r\n\r\n`/root/in/Figueredo2013:1.sedml724177.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Figueredo2013:1.sedml` is invalid.\n - Data generator `E_1_task1` is invalid.\n - Variable `p1_E_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='E']` are not defined. No namespaces are defined for the target.\n - Data generator `T_2_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Figueredo2013:1.sedml548675.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Figueredo2013:1.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 849, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'g + T' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (10503). The following is the first warning at line 705, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * t_d(a, b, T)' in the <kineticLaw> element of the <reaction> with id 'tumor_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 7 warnings of type SBML unit consistency (10541). The following is the first warning at line 691, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_growth') are mole (exponent = 1, multiplier = 0.001, scale = 0).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 100.3333333233\r\n - Number of steps: 301\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Figueredo2013:1.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Figueredo2013:1.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 849, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'g + T' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (10503). The following is the first warning at line 705, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * t_d(a, b, T)' in the <kineticLaw> element of the <reaction> with id 'tumor_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 7 warnings of type SBML unit consistency (10541). The following is the first warning at line 691, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_growth') are mole (exponent = 1, multiplier = 0.001, scale = 0).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 100.3333333233\r\n - Number of steps: 301\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Figueredo2013:1.sedml548675.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Figueredo2013:1.sedml` is invalid.\n - Data generator `E_1_task1` is invalid.\n - Variable `p1_E_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='E']` are not defined. No namespaces are defined for the target.\n - Data generator `T_2_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000754](https://www.ebi.ac.uk/biomodels/BIOMD0000000754)<br/><sup>Figueredo2013/2 - immunointera</sup></summary><sup>Figueredo2013/2 - immunointeraction model with IL2</sup></details>|<details><summary>pass</summary>[Figueredo2013:2.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000754#Files)</details>|FAIL|<details><summary>pass</summary>[Figueredo2013:2.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000754#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Figueredo2013:2.sedml549070.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Figueredo2013:2.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 1078, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'g2 + T' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 6 warnings of type SBML unit consistency (10503). The following is the first warning at line 988, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * t_d(a, b, T)' in the <kineticLaw> element of the <reaction> with id 'tumor_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 10 warnings of type SBML unit consistency (10541). The following is the first warning at line 974, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_growth') are mole (exponent = 1, multiplier = 0.001, scale = 0).\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Figueredo2013:2.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Figueredo2013:2.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 1078, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'g2 + T' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 6 warnings of type SBML unit consistency (10503). The following is the first warning at line 988, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * t_d(a, b, T)' in the <kineticLaw> element of the <reaction> with id 'tumor_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 10 warnings of type SBML unit consistency (10541). The following is the first warning at line 974, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_growth') are mole (exponent = 1, multiplier = 0.001, scale = 0).\x1b[0m\r\n\r\n`/root/in/Figueredo2013:2.sedml549070.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Figueredo2013:2.sedml` is invalid.\n - Data generator `E_1_task1` is invalid.\n - Variable `p1_E_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='E']` are not defined. No namespaces are defined for the target.\n - Data generator `I_2_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `T_3_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Figueredo2013:2.sedml418918.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Figueredo2013:2.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 1078, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'g2 + T' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 6 warnings of type SBML unit consistency (10503). The following is the first warning at line 988, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * t_d(a, b, T)' in the <kineticLaw> element of the <reaction> with id 'tumor_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 10 warnings of type SBML unit consistency (10541). The following is the first warning at line 974, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_growth') are mole (exponent = 1, multiplier = 0.001, scale = 0).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Figueredo2013:2.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Figueredo2013:2.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 1078, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'g2 + T' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 6 warnings of type SBML unit consistency (10503). The following is the first warning at line 988, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * t_d(a, b, T)' in the <kineticLaw> element of the <reaction> with id 'tumor_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 10 warnings of type SBML unit consistency (10541). The following is the first warning at line 974, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_growth') are mole (exponent = 1, multiplier = 0.001, scale = 0).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Figueredo2013:2.sedml418918.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Figueredo2013:2.sedml` is invalid.\n - Data generator `E_1_task1` is invalid.\n - Variable `p1_E_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='E']` are not defined. No namespaces are defined for the target.\n - Data generator `I_2_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `T_3_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000755](https://www.ebi.ac.uk/biomodels/BIOMD0000000755)<br/><sup>Hansen2019 - Nine species redu</sup></summary><sup>Hansen2019 - Nine species reduced model of blood coagulation</sup></details>|<details><summary>pass</summary>[Hansen2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000755#Files)</details>|pass|<details><summary>FAIL</summary>[Hansen2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000755#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Hansen2019.sedml301887.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Hansen2019.sedml` has warnings.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/Hansen2019.sedml301887.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Hansen2019.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmpe_jugpqo/./model1.xml` is not a file.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='TF']` are not defined. No namespaces are defined for the target.\n - Data generator `IIa_1_task1` is invalid.\n - Variable `p2_IIa_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IIa']` are not defined. No namespaces are defined for the target.\n - Data generator `IIa_1_task2` is invalid.\n - Variable `p2_IIa_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IIa']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Hansen2019.sedml218162.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Hansen2019.sedml` has warnings.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Hansen2019.sedml218162.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Hansen2019.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmpbfpasl00/./model1.xml` is not a file.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='TF']` are not defined. No namespaces are defined for the target.\n - Data generator `IIa_1_task1` is invalid.\n - Variable `p2_IIa_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IIa']` are not defined. No namespaces are defined for the target.\n - Data generator `IIa_1_task2` is invalid.\n - Variable `p2_IIa_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IIa']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000756](https://www.ebi.ac.uk/biomodels/BIOMD0000000756)<br/><sup>Figueredo2013/3 - immunointera</sup></summary><sup>Figueredo2013/3 - immunointeraction full model</sup></details>|<details><summary>pass</summary>[Figueredo2013:3.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000756#Files)</details>|FAIL|<details><summary>pass</summary>[Figueredo2013:3.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000756#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Figueredo2013:3.sedml289555.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Figueredo2013:3.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (10501). The following is the first warning at line 1479, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'g2 + T' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (10503). The following is the first warning at line 1389, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * td(a, T, k)' in the <kineticLaw> element of the <reaction> with id 'tumor_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 9 warnings of type SBML unit consistency (10541). The following is the first warning at line 1375, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_growth') are mole (exponent = 1, multiplier = 0.001, scale = 0).\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 1582, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tumor_microenvironment * ist(c, T, gamma, S)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Figueredo2013:3.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Figueredo2013:3.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (10501). The following is the first warning at line 1479, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'g2 + T' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (10503). The following is the first warning at line 1389, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * td(a, T, k)' in the <kineticLaw> element of the <reaction> with id 'tumor_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 9 warnings of type SBML unit consistency (10541). The following is the first warning at line 1375, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_growth') are mole (exponent = 1, multiplier = 0.001, scale = 0).\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 1582, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tumor_microenvironment * ist(c, T, gamma, S)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\x1b[0m\r\n\r\n`/root/in/Figueredo2013:3.sedml289555.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Figueredo2013:3.sedml` is invalid.\n - Data generator `E_1_task1` is invalid.\n - Variable `p1_E_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='E']` are not defined. No namespaces are defined for the target.\n - Data generator `I_2_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `T_3_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Figueredo2013:3.sedml143263.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Figueredo2013:3.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (10501). The following is the first warning at line 1479, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'g2 + T' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (10503). The following is the first warning at line 1389, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * td(a, T, k)' in the <kineticLaw> element of the <reaction> with id 'tumor_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 9 warnings of type SBML unit consistency (10541). The following is the first warning at line 1375, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_growth') are mole (exponent = 1, multiplier = 0.001, scale = 0).\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 1582, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tumor_microenvironment * ist(c, T, gamma, S)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Figueredo2013:3.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Figueredo2013:3.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (10501). The following is the first warning at line 1479, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'g2 + T' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (10503). The following is the first warning at line 1389, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * td(a, T, k)' in the <kineticLaw> element of the <reaction> with id 'tumor_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 9 warnings of type SBML unit consistency (10541). The following is the first warning at line 1375, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_growth') are mole (exponent = 1, multiplier = 0.001, scale = 0).\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 1582, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tumor_microenvironment * ist(c, T, gamma, S)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Figueredo2013:3.sedml143263.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Figueredo2013:3.sedml` is invalid.\n - Data generator `E_1_task1` is invalid.\n - Variable `p1_E_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='E']` are not defined. No namespaces are defined for the target.\n - Data generator `I_2_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `T_3_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000757](https://www.ebi.ac.uk/biomodels/BIOMD0000000757)<br/><sup>Abernathy2016 - glioblastoma t</sup></summary><sup>Abernathy2016 - glioblastoma treatment</sup></details>|<details><summary>pass</summary>[Abernathy2016.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000757#Files)</details>|FAIL|<details><summary>pass</summary>[Abernathy2016.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000757#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Abernathy2016.sedml482214.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Abernathy2016.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (10501). The following is the first warning at line 2035, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'k1 - Tumor' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 12 warnings of type SBML unit consistency (10503). The following is the first warning at line 2225, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * uc * CytotoxicTcell' in the <kineticLaw> element of the <reaction> with id 'ctl_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 1889, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'Tumor / 100000000' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type SBML unit consistency (10541). The following is the first warning at line 2035, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 3600, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_transformation') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 3600, scale = 2), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 3502.854812346\r\n - Number of steps: 1227\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Abernathy2016.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Abernathy2016.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (10501). The following is the first warning at line 2035, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'k1 - Tumor' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 12 warnings of type SBML unit consistency (10503). The following is the first warning at line 2225, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * uc * CytotoxicTcell' in the <kineticLaw> element of the <reaction> with id 'ctl_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 1889, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'Tumor / 100000000' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type SBML unit consistency (10541). The following is the first warning at line 2035, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 3600, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_transformation') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 3600, scale = 2), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 3502.854812346\r\n - Number of steps: 1227\x1b[0m\r\n\r\n`/root/in/Abernathy2016.sedml482214.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Abernathy2016.sedml` is invalid.\n - Data generator `CSC_Plot_1_task1` is invalid.\n - Variable `p1_CSC_Plot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CSC_Plot']` are not defined. No namespaces are defined for the target.\n - Data generator `CTL_Plot_2_task1` is invalid.\n - Variable `p1_CTL_Plot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CTL_Plot']` are not defined. No namespaces are defined for the target.\n - Data generator `Tumor_Plot_3_task1` is invalid.\n - Variable `p1_Tumor_Plot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Tumor_Plot']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Abernathy2016.sedml913153.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Abernathy2016.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (10501). The following is the first warning at line 2035, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'k1 - Tumor' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 12 warnings of type SBML unit consistency (10503). The following is the first warning at line 2225, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * uc * CytotoxicTcell' in the <kineticLaw> element of the <reaction> with id 'ctl_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 1889, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'Tumor / 100000000' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type SBML unit consistency (10541). The following is the first warning at line 2035, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 3600, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_transformation') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 3600, scale = 2), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 3502.854812346\r\n - Number of steps: 1227\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Abernathy2016.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Abernathy2016.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (10501). The following is the first warning at line 2035, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'k1 - Tumor' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 12 warnings of type SBML unit consistency (10503). The following is the first warning at line 2225, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * uc * CytotoxicTcell' in the <kineticLaw> element of the <reaction> with id 'ctl_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 1889, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'Tumor / 100000000' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type SBML unit consistency (10541). The following is the first warning at line 2035, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 3600, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_transformation') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 3600, scale = 2), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 3502.854812346\r\n - Number of steps: 1227\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Abernathy2016.sedml913153.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Abernathy2016.sedml` is invalid.\n - Data generator `CSC_Plot_1_task1` is invalid.\n - Variable `p1_CSC_Plot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CSC_Plot']` are not defined. No namespaces are defined for the target.\n - Data generator `CTL_Plot_2_task1` is invalid.\n - Variable `p1_CTL_Plot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CTL_Plot']` are not defined. No namespaces are defined for the target.\n - Data generator `Tumor_Plot_3_task1` is invalid.\n - Variable `p1_Tumor_Plot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Tumor_Plot']` are not defined. No namespaces are defined for the target.\n"```</details>|
|[BIOMD0000000758](https://www.ebi.ac.uk/biomodels/BIOMD0000000758)<br/><sup>Babbs2012 - immunotherapy</sup>|<details><summary>pass</summary>[Babbs2012.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000758#Files)</details>|FAIL|<details><summary>pass</summary>[Babbs2012.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000758#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Babbs2012.sedml72519.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Babbs2012.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10503). The following is the first warning at line 590, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * immune_stim(l, T, I)' in the <kineticLaw> element of the <reaction> with id 'immune_stimulation' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 4 warnings of type SBML unit consistency (10541). The following is the first warning at line 517, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_growth') are second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 1, multiplier = 1, scale = -3).\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Babbs2012.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Babbs2012.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10503). The following is the first warning at line 590, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * immune_stim(l, T, I)' in the <kineticLaw> element of the <reaction> with id 'immune_stimulation' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 4 warnings of type SBML unit consistency (10541). The following is the first warning at line 517, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_growth') are second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 1, multiplier = 1, scale = -3).\x1b[0m\r\n\r\n`/root/in/Babbs2012.sedml72519.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Babbs2012.sedml` is invalid.\n - Data generator `I_1_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Babbs2012.sedml139185.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Babbs2012.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10503). The following is the first warning at line 590, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * immune_stim(l, T, I)' in the <kineticLaw> element of the <reaction> with id 'immune_stimulation' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 4 warnings of type SBML unit consistency (10541). The following is the first warning at line 517, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_growth') are second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 1, multiplier = 1, scale = -3).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Babbs2012.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Babbs2012.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10503). The following is the first warning at line 590, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tumor_microenvironment * immune_stim(l, T, I)' in the <kineticLaw> element of the <reaction> with id 'immune_stimulation' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 4 warnings of type SBML unit consistency (10541). The following is the first warning at line 517, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_growth') are second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 1, multiplier = 1, scale = -3).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Babbs2012.sedml139185.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Babbs2012.sedml` is invalid.\n - Data generator `I_1_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000759](https://www.ebi.ac.uk/biomodels/BIOMD0000000759)<br/><sup>den Breems2015 - macrophage in</sup></summary><sup>den Breems2015 - macrophage in cancer</sup></details>|<details><summary>pass</summary>[denBreems2015.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000759#Files)</details>|FAIL|<details><summary>pass</summary>[denBreems2015.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000759#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/denBreems2015.sedml85434.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `denBreems2015.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10513). The following is the first warning at line 1756, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L2V5 Section 4.11.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'Tumor' are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3).\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 1810, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tumor_microenvironment * tumor_growth_1(Tn, Ts, bt, r)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type SBML unit consistency (10541). The following is the first warning at line 1912, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V5 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Tn_killing') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./denBreems2015.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `denBreems2015.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10513). The following is the first warning at line 1756, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L2V5 Section 4.11.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'Tumor' are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3).\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 1810, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tumor_microenvironment * tumor_growth_1(Tn, Ts, bt, r)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type SBML unit consistency (10541). The following is the first warning at line 1912, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V5 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Tn_killing') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\x1b[0m\r\n\r\n`/root/in/denBreems2015.sedml85434.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./denBreems2015.sedml` is invalid.\n - Data generator `Tn_1_task1` is invalid.\n - Variable `p1_Tn_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Tn']` are not defined. No namespaces are defined for the target.\n - Data generator `Ts_2_task1` is invalid.\n - Variable `p1_Ts_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Ts']` are not defined. No namespaces are defined for the target.\n - Data generator `Tumor_3_task1` is invalid.\n - Variable `p1_Tumor_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Tumor']` are not defined. No namespaces are defined for the target.\n - Data generator `M1_1_task1` is invalid.\n - Variable `p2_M1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M1']` are not defined. No namespaces are defined for the target.\n - Data generator `M2_2_task1` is invalid.\n - Variable `p2_M2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M2']` are not defined. No namespaces are defined for the target.\n - Data generator `Th1_1_task1` is invalid.\n - Variable `p3_Th1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Th1']` are not defined. No namespaces are defined for the target.\n - Data generator `Th2_1_task1` is invalid.\n - Variable `p4_Th2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Th2']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/denBreems2015.sedml424133.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `denBreems2015.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10513). The following is the first warning at line 1756, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L2V5 Section 4.11.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'Tumor' are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3).\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 1810, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tumor_microenvironment * tumor_growth_1(Tn, Ts, bt, r)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type SBML unit consistency (10541). The following is the first warning at line 1912, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V5 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Tn_killing') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./denBreems2015.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `denBreems2015.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10513). The following is the first warning at line 1756, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L2V5 Section 4.11.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'Tumor' are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3).\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 1810, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tumor_microenvironment * tumor_growth_1(Tn, Ts, bt, r)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type SBML unit consistency (10541). The following is the first warning at line 1912, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V5 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Tn_killing') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/denBreems2015.sedml424133.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./denBreems2015.sedml` is invalid.\n - Data generator `Tn_1_task1` is invalid.\n - Variable `p1_Tn_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Tn']` are not defined. No namespaces are defined for the target.\n - Data generator `Ts_2_task1` is invalid.\n - Variable `p1_Ts_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Ts']` are not defined. No namespaces are defined for the target.\n - Data generator `Tumor_3_task1` is invalid.\n - Variable `p1_Tumor_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Tumor']` are not defined. No namespaces are defined for the target.\n - Data generator `M1_1_task1` is invalid.\n - Variable `p2_M1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M1']` are not defined. No namespaces are defined for the target.\n - Data generator `M2_2_task1` is invalid.\n - Variable `p2_M2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M2']` are not defined. No namespaces are defined for the target.\n - Data generator `Th1_1_task1` is invalid.\n - Variable `p3_Th1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Th1']` are not defined. No namespaces are defined for the target.\n - Data generator `Th2_1_task1` is invalid.\n - Variable `p4_Th2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Th2']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000760](https://www.ebi.ac.uk/biomodels/BIOMD0000000760)<br/><sup>Feizabadi2011/1 - immunodefici</sup></summary><sup>Feizabadi2011/1 - immunodeficiency in cancer core model</sup></details>|<details><summary>pass</summary>[Feizabadi2011_1.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000760#Files)</details>|FAIL|<details><summary>pass</summary>[Feizabadi2011_1.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000760#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Feizabadi2011_1.sedml526491.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Feizabadi2011_1.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 789, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'r1 + N' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 682, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tumor_microenvironment * Function1(rt, T, kt)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Feizabadi2011_1.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Feizabadi2011_1.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 789, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'r1 + N' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 682, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tumor_microenvironment * Function1(rt, T, kt)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\x1b[0m\r\n\r\n`/root/in/Feizabadi2011_1.sedml526491.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Feizabadi2011_1.sedml` is invalid.\n - Data generator `N_1_task1` is invalid.\n - Variable `p1_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n - Data generator `T_2_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `t_3_task1` is invalid.\n - Variable `p1_t_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='t']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Feizabadi2011_1.sedml807078.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Feizabadi2011_1.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 789, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'r1 + N' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 682, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tumor_microenvironment * Function1(rt, T, kt)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Feizabadi2011_1.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Feizabadi2011_1.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 789, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'r1 + N' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 682, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tumor_microenvironment * Function1(rt, T, kt)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Feizabadi2011_1.sedml807078.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Feizabadi2011_1.sedml` is invalid.\n - Data generator `N_1_task1` is invalid.\n - Variable `p1_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n - Data generator `T_2_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `t_3_task1` is invalid.\n - Variable `p1_t_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='t']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000761](https://www.ebi.ac.uk/biomodels/BIOMD0000000761)<br/><sup>Cappuccio2006 - Cancer immunot</sup></summary><sup>Cappuccio2006 - Cancer immunotherapy by interleukin-21</sup></details>|<details><summary>pass</summary>[Cappuccio2006 v2.0.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000761#Files)</details>|FAIL|<details><summary>pass</summary>[Cappuccio2006 v9.0.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000761#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Cappuccio2006 v9.0.sedml421395.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 2: b'usage: biosimulators-tellurium [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-tellurium: error: unrecognized arguments: v9.0.sedml421395.omex\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Cappuccio2006 v9.0.sedml892046.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 2: b'usage: biosimulators-copasi [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-copasi: error: unrecognized arguments: v9.0.sedml892046.omex\n'```</details>|
|<details><summary>[BIOMD0000000762](https://www.ebi.ac.uk/biomodels/BIOMD0000000762)<br/><sup>Kuznetsov1994 - Nonlinear dyna</sup></summary><sup>Kuznetsov1994 - Nonlinear dynamics of immunogenic tumors</sup></details>|<details><summary>pass</summary>[Kuznetsov1994.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000762#Files)</details>|pass|<details><summary>pass</summary>[Kuznetsov1994.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000762#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Kuznetsov1994.sedml748453.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Kuznetsov1994.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 534, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Constant_flux__irreversible(s)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 384, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'p' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Kuznetsov1994.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Kuznetsov1994.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 534, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Constant_flux__irreversible(s)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 384, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'p' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Kuznetsov1994.sedml748453.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Kuznetsov1994.sedml` is invalid.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Kuznetsov1994.sedml570092.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Kuznetsov1994.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 534, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Constant_flux__irreversible(s)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 384, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'p' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Kuznetsov1994.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Kuznetsov1994.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 534, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Constant_flux__irreversible(s)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 384, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'p' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Kuznetsov1994.sedml570092.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Kuznetsov1994.sedml` is invalid.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000763](https://www.ebi.ac.uk/biomodels/BIOMD0000000763)<br/><sup>Dritschel2018 - A mathematical</sup></summary><sup>Dritschel2018 - A mathematical model of cytotoxic and helper T cell interactions in a tumour microenvironment</sup></details>|<details><summary>pass</summary>[Dritschel2018.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000763#Files)</details>|FAIL|<details><summary>pass</summary>[Dritschel2018.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000763#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Dritschel2018.sedml847202.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Dritschel2018.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 571, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth(gamma, N_Tumour)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 757, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'T_Cytotoxic_Infiltration_Active') are litre (exponent = -1, multiplier = 0.001, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 473, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'gamma' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Dritschel2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Dritschel2018.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 571, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth(gamma, N_Tumour)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 757, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'T_Cytotoxic_Infiltration_Active') are litre (exponent = -1, multiplier = 0.001, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 473, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'gamma' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Dritschel2018.sedml847202.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Dritschel2018.sedml` is invalid.\n - Data generator `N_Tumour_1_task1` is invalid.\n - Variable `p1_N_Tumour_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N_Tumour']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Dritschel2018.sedml709885.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Dritschel2018.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 571, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth(gamma, N_Tumour)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 757, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'T_Cytotoxic_Infiltration_Active') are litre (exponent = -1, multiplier = 0.001, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 473, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'gamma' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Dritschel2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Dritschel2018.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 571, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth(gamma, N_Tumour)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 757, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'T_Cytotoxic_Infiltration_Active') are litre (exponent = -1, multiplier = 0.001, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 473, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'gamma' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Dritschel2018.sedml709885.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Dritschel2018.sedml` is invalid.\n - Data generator `N_Tumour_1_task1` is invalid.\n - Variable `p1_N_Tumour_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N_Tumour']` are not defined. No namespaces are defined for the target.\n"```</details>|
|[BIOMD0000000764](https://www.ebi.ac.uk/biomodels/BIOMD0000000764)<br/><sup>Malinzi2019 - chemovirotherapy</sup>|<details><summary>pass</summary>[Malinzi2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000764#Files)</details>|FAIL|<details><summary>pass</summary>[Malinzi2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000764#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Malinzi2019.sedml715525.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Malinzi2019.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 1067, column 6:\r\n - The formula 'exp(-a * time)' in the math element of the <assignmentRule> uses a function which can only act on dimensionless variables.\r\n - 4 warnings of type SBML unit consistency (10503). The following is the first warning at line 1348, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * burst(b, I)' in the <kineticLaw> element of the <reaction> with id 'virus_burst' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 1 warning of type SBML unit consistency (99508). The following is the first warning at line 942, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'Metabolite_3' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (20702). The following is the first warning at line 942, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'Metabolite_3' does not have a 'units' attribute.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 956, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'Metabolite_3' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type SBML unit consistency (10541). The following is the first warning at line 1181, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_killing') are litre (exponent = -1, multiplier = 0.001, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - 2 warnings of type Modeling practice (80701). The following is the first warning at line 942, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Metabolite_3' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Malinzi2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Malinzi2019.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 1067, column 6:\r\n - The formula 'exp(-a * time)' in the math element of the <assignmentRule> uses a function which can only act on dimensionless variables.\r\n - 4 warnings of type SBML unit consistency (10503). The following is the first warning at line 1348, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * burst(b, I)' in the <kineticLaw> element of the <reaction> with id 'virus_burst' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 1 warning of type SBML unit consistency (99508). The following is the first warning at line 942, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'Metabolite_3' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (20702). The following is the first warning at line 942, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'Metabolite_3' does not have a 'units' attribute.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 956, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'Metabolite_3' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type SBML unit consistency (10541). The following is the first warning at line 1181, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_killing') are litre (exponent = -1, multiplier = 0.001, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - 2 warnings of type Modeling practice (80701). The following is the first warning at line 942, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Metabolite_3' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Malinzi2019.sedml715525.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Malinzi2019.sedml` is invalid.\n - Data generator `I_1_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `U_2_task1` is invalid.\n - Variable `p1_U_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='U']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Malinzi2019.sedml834708.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Malinzi2019.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 1067, column 6:\r\n - The formula 'exp(-a * time)' in the math element of the <assignmentRule> uses a function which can only act on dimensionless variables.\r\n - 4 warnings of type SBML unit consistency (10503). The following is the first warning at line 1348, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * burst(b, I)' in the <kineticLaw> element of the <reaction> with id 'virus_burst' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 1 warning of type SBML unit consistency (99508). The following is the first warning at line 942, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'Metabolite_3' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (20702). The following is the first warning at line 942, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'Metabolite_3' does not have a 'units' attribute.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 956, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'Metabolite_3' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type SBML unit consistency (10541). The following is the first warning at line 1181, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_killing') are litre (exponent = -1, multiplier = 0.001, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - 2 warnings of type Modeling practice (80701). The following is the first warning at line 942, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Metabolite_3' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Malinzi2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Malinzi2019.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 1067, column 6:\r\n - The formula 'exp(-a * time)' in the math element of the <assignmentRule> uses a function which can only act on dimensionless variables.\r\n - 4 warnings of type SBML unit consistency (10503). The following is the first warning at line 1348, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * burst(b, I)' in the <kineticLaw> element of the <reaction> with id 'virus_burst' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 1 warning of type SBML unit consistency (99508). The following is the first warning at line 942, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'Metabolite_3' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (20702). The following is the first warning at line 942, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'Metabolite_3' does not have a 'units' attribute.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 956, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'Metabolite_3' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type SBML unit consistency (10541). The following is the first warning at line 1181, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_killing') are litre (exponent = -1, multiplier = 0.001, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - 2 warnings of type Modeling practice (80701). The following is the first warning at line 942, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Metabolite_3' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Malinzi2019.sedml834708.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Malinzi2019.sedml` is invalid.\n - Data generator `I_1_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `U_2_task1` is invalid.\n - Variable `p1_U_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='U']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000765](https://www.ebi.ac.uk/biomodels/BIOMD0000000765)<br/><sup>Mager2005 - Quasi-equilibrium </sup></summary><sup>Mager2005 - Quasi-equilibrium pharmacokinetic model for drugs exhibiting target-mediated drug disposition</sup></details>|<details><summary>pass</summary>[Mager2005.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000765#Files)</details>|pass|<details><summary>pass</summary>[Mager2005.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000765#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Mager2005.sedml958277.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Mager2005.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 640, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'V_c' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 481, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k_el' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Mager2005.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Mager2005.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 640, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'V_c' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 481, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k_el' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Mager2005.sedml958277.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Mager2005.sedml` is invalid.\n - Data generator `C_tot_1_task1` is invalid.\n - Variable `p1_C_tot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='C_tot']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Mager2005.sedml254224.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Mager2005.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 640, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'V_c' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 481, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k_el' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Mager2005.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Mager2005.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 640, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'V_c' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 481, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k_el' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Mager2005.sedml254224.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Mager2005.sedml` is invalid.\n - Data generator `C_tot_1_task1` is invalid.\n - Variable `p1_C_tot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='C_tot']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000766](https://www.ebi.ac.uk/biomodels/BIOMD0000000766)<br/><sup>Macnamara2015/1 - virotherapy </sup></summary><sup>Macnamara2015/1 - virotherapy full model</sup></details>|<details><summary>pass</summary>[Macnamara2015:1.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000766#Files)</details>|FAIL|<details><summary>pass</summary>[Macnamara2015:1.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000766#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Macnamara2015:1.sedml943615.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Macnamara2015:1.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (10501). The following is the first warning at line 1220, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'hu + U' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 4 warnings of type SBML unit consistency (10503). The following is the first warning at line 1291, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * delta * I' in the <kineticLaw> element of the <reaction> with id 'infected_tumor_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 1181, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tme * tg(r, U, I, K)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type SBML unit consistency (10541). The following is the first warning at line 1220, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_infection') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 2, multiplier = 0.001, scale = 0).\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Macnamara2015:1.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Macnamara2015:1.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (10501). The following is the first warning at line 1220, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'hu + U' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 4 warnings of type SBML unit consistency (10503). The following is the first warning at line 1291, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * delta * I' in the <kineticLaw> element of the <reaction> with id 'infected_tumor_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 1181, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tme * tg(r, U, I, K)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type SBML unit consistency (10541). The following is the first warning at line 1220, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_infection') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 2, multiplier = 0.001, scale = 0).\x1b[0m\r\n\r\n`/root/in/Macnamara2015:1.sedml943615.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Macnamara2015:1.sedml` is invalid.\n - Data generator `E_1_task1` is invalid.\n - Variable `p1_E_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='E']` are not defined. No namespaces are defined for the target.\n - Data generator `U_2_task1` is invalid.\n - Variable `p1_U_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='U']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Macnamara2015:1.sedml926726.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Macnamara2015:1.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (10501). The following is the first warning at line 1220, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'hu + U' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 4 warnings of type SBML unit consistency (10503). The following is the first warning at line 1291, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * delta * I' in the <kineticLaw> element of the <reaction> with id 'infected_tumor_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 1181, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tme * tg(r, U, I, K)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type SBML unit consistency (10541). The following is the first warning at line 1220, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_infection') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 2, multiplier = 0.001, scale = 0).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Macnamara2015:1.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Macnamara2015:1.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (10501). The following is the first warning at line 1220, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'hu + U' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 4 warnings of type SBML unit consistency (10503). The following is the first warning at line 1291, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * delta * I' in the <kineticLaw> element of the <reaction> with id 'infected_tumor_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 1181, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tme * tg(r, U, I, K)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type SBML unit consistency (10541). The following is the first warning at line 1220, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_infection') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 2, multiplier = 0.001, scale = 0).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Macnamara2015:1.sedml926726.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Macnamara2015:1.sedml` is invalid.\n - Data generator `E_1_task1` is invalid.\n - Variable `p1_E_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='E']` are not defined. No namespaces are defined for the target.\n - Data generator `U_2_task1` is invalid.\n - Variable `p1_U_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='U']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000767](https://www.ebi.ac.uk/biomodels/BIOMD0000000767)<br/><sup>Macnamara2015/2 - virotherapy </sup></summary><sup>Macnamara2015/2 - virotherapy virus-free submodel</sup></details>|<details><summary>pass</summary>[Macnamara2015:2.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000767#Files)</details>|FAIL|<details><summary>pass</summary>[Macnamara2015:2.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000767#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Macnamara2015:2.sedml379974.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Macnamara2015:2.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 809, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'he + E' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 1 warning of type SBML unit consistency (10503). The following is the first warning at line 881, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * de * E' in the <kineticLaw> element of the <reaction> with id 'effector_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 1 warning of type SBML unit consistency (99505). The following is the first warning at line 755, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tme * tg(r, U, K)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 4 warnings of type SBML unit consistency (10541). The following is the first warning at line 809, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_killing') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 2, multiplier = 0.001, scale = 0).\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Macnamara2015:2.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Macnamara2015:2.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 809, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'he + E' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 1 warning of type SBML unit consistency (10503). The following is the first warning at line 881, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * de * E' in the <kineticLaw> element of the <reaction> with id 'effector_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 1 warning of type SBML unit consistency (99505). The following is the first warning at line 755, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tme * tg(r, U, K)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 4 warnings of type SBML unit consistency (10541). The following is the first warning at line 809, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_killing') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 2, multiplier = 0.001, scale = 0).\x1b[0m\r\n\r\n`/root/in/Macnamara2015:2.sedml379974.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Macnamara2015:2.sedml` is invalid.\n - Data generator `E_1_task1` is invalid.\n - Variable `p1_E_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='E']` are not defined. No namespaces are defined for the target.\n - Data generator `U_2_task1` is invalid.\n - Variable `p1_U_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='U']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Macnamara2015:2.sedml138729.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Macnamara2015:2.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 809, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'he + E' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 1 warning of type SBML unit consistency (10503). The following is the first warning at line 881, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * de * E' in the <kineticLaw> element of the <reaction> with id 'effector_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 1 warning of type SBML unit consistency (99505). The following is the first warning at line 755, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tme * tg(r, U, K)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 4 warnings of type SBML unit consistency (10541). The following is the first warning at line 809, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_killing') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 2, multiplier = 0.001, scale = 0).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Macnamara2015:2.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Macnamara2015:2.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 809, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'he + E' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 1 warning of type SBML unit consistency (10503). The following is the first warning at line 881, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * de * E' in the <kineticLaw> element of the <reaction> with id 'effector_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 1 warning of type SBML unit consistency (99505). The following is the first warning at line 755, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tme * tg(r, U, K)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 4 warnings of type SBML unit consistency (10541). The following is the first warning at line 809, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_killing') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 2, multiplier = 0.001, scale = 0).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Macnamara2015:2.sedml138729.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Macnamara2015:2.sedml` is invalid.\n - Data generator `E_1_task1` is invalid.\n - Variable `p1_E_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='E']` are not defined. No namespaces are defined for the target.\n - Data generator `U_2_task1` is invalid.\n - Variable `p1_U_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='U']` are not defined. No namespaces are defined for the target.\n"```</details>|
|[BIOMD0000000768](https://www.ebi.ac.uk/biomodels/BIOMD0000000768)<br/><sup>Eftimie2010 - immunity to melanoma</sup>|<details><summary>pass</summary>[Eftimie2010.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000768#Files)</details>|FAIL|<details><summary>pass</summary>[Eftimie2010.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000768#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Eftimie2010.sedml786630.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Eftimie2010.xml` has warnings.\r\n - 9 warnings of type SBML unit consistency (10501). The following is the first warning at line 1604, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'h2 + T' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 7 warnings of type SBML unit consistency (10503). The following is the first warning at line 1728, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * the(dth, T, Th)' in the <kineticLaw> element of the <reaction> with id 'Th_exhaustion' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 1 warning of type SBML unit consistency (10513). The following is the first warning at line 1556, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'f' are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3).\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 1604, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tme * ths(ath, C, T, h2, kp, Cp)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 8 warnings of type SBML unit consistency (10541). The following is the first warning at line 1714, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Th_death') are litre (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 8640, scale = 0).\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Eftimie2010.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Eftimie2010.xml` has warnings.\r\n - 9 warnings of type SBML unit consistency (10501). The following is the first warning at line 1604, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'h2 + T' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 7 warnings of type SBML unit consistency (10503). The following is the first warning at line 1728, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * the(dth, T, Th)' in the <kineticLaw> element of the <reaction> with id 'Th_exhaustion' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 1 warning of type SBML unit consistency (10513). The following is the first warning at line 1556, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'f' are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3).\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 1604, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tme * ths(ath, C, T, h2, kp, Cp)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 8 warnings of type SBML unit consistency (10541). The following is the first warning at line 1714, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Th_death') are litre (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 8640, scale = 0).\x1b[0m\r\n\r\n`/root/in/Eftimie2010.sedml786630.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Eftimie2010.sedml` is invalid.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Eftimie2010.sedml603786.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Eftimie2010.xml` has warnings.\r\n - 9 warnings of type SBML unit consistency (10501). The following is the first warning at line 1604, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'h2 + T' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 7 warnings of type SBML unit consistency (10503). The following is the first warning at line 1728, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * the(dth, T, Th)' in the <kineticLaw> element of the <reaction> with id 'Th_exhaustion' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 1 warning of type SBML unit consistency (10513). The following is the first warning at line 1556, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'f' are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3).\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 1604, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tme * ths(ath, C, T, h2, kp, Cp)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 8 warnings of type SBML unit consistency (10541). The following is the first warning at line 1714, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Th_death') are litre (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 8640, scale = 0).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Eftimie2010.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Eftimie2010.xml` has warnings.\r\n - 9 warnings of type SBML unit consistency (10501). The following is the first warning at line 1604, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'h2 + T' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 7 warnings of type SBML unit consistency (10503). The following is the first warning at line 1728, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * the(dth, T, Th)' in the <kineticLaw> element of the <reaction> with id 'Th_exhaustion' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 1 warning of type SBML unit consistency (10513). The following is the first warning at line 1556, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'f' are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3).\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 1604, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tme * ths(ath, C, T, h2, kp, Cp)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 8 warnings of type SBML unit consistency (10541). The following is the first warning at line 1714, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Th_death') are litre (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 8640, scale = 0).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Eftimie2010.sedml603786.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Eftimie2010.sedml` is invalid.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000769](https://www.ebi.ac.uk/biomodels/BIOMD0000000769)<br/><sup>Eftimie2017/2 - interaction of</sup></summary><sup>Eftimie2017/2 - interaction of Th and macrophage in melanoma</sup></details>|<details><summary>pass</summary>[Eftimie2017:2.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000769#Files)</details>|FAIL|<details><summary>pass</summary>[Eftimie2017:2.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000769#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Eftimie2017:2.sedml73795.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Eftimie2017:2.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (10503). The following is the first warning at line 1557, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * _1_factor(gh1, T, H1)' in the <kineticLaw> element of the <reaction> with id 'tumor_H1' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 1495, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tme * tg(a, T, b)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 19 warnings of type SBML unit consistency (10541). The following is the first warning at line 1546, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_death') are second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 1, multiplier = 1, scale = -3).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.1\r\n - Output end time: 30.0\r\n - Number of steps: 9967\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Eftimie2017:2.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Eftimie2017:2.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (10503). The following is the first warning at line 1557, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * _1_factor(gh1, T, H1)' in the <kineticLaw> element of the <reaction> with id 'tumor_H1' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 1495, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tme * tg(a, T, b)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 19 warnings of type SBML unit consistency (10541). The following is the first warning at line 1546, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_death') are second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 1, multiplier = 1, scale = -3).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.1\r\n - Output end time: 30.0\r\n - Number of steps: 9967\x1b[0m\r\n\r\n`/root/in/Eftimie2017:2.sedml73795.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Eftimie2017:2.sedml` is invalid.\n - Data generator `H1_1_task1` is invalid.\n - Variable `p1_H1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H1']` are not defined. No namespaces are defined for the target.\n - Data generator `H2_2_task1` is invalid.\n - Variable `p1_H2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H2']` are not defined. No namespaces are defined for the target.\n - Data generator `M1_3_task1` is invalid.\n - Variable `p1_M1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M1']` are not defined. No namespaces are defined for the target.\n - Data generator `M2_4_task1` is invalid.\n - Variable `p1_M2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M2']` are not defined. No namespaces are defined for the target.\n - Data generator `T_5_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Eftimie2017:2.sedml560021.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Eftimie2017:2.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (10503). The following is the first warning at line 1557, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * _1_factor(gh1, T, H1)' in the <kineticLaw> element of the <reaction> with id 'tumor_H1' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 1495, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tme * tg(a, T, b)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 19 warnings of type SBML unit consistency (10541). The following is the first warning at line 1546, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_death') are second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 1, multiplier = 1, scale = -3).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.1\r\n - Output end time: 30.0\r\n - Number of steps: 9967\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Eftimie2017:2.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Eftimie2017:2.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (10503). The following is the first warning at line 1557, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * _1_factor(gh1, T, H1)' in the <kineticLaw> element of the <reaction> with id 'tumor_H1' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 1495, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tme * tg(a, T, b)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 19 warnings of type SBML unit consistency (10541). The following is the first warning at line 1546, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_death') are second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 1, multiplier = 1, scale = -3).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.1\r\n - Output end time: 30.0\r\n - Number of steps: 9967\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Eftimie2017:2.sedml560021.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Eftimie2017:2.sedml` is invalid.\n - Data generator `H1_1_task1` is invalid.\n - Variable `p1_H1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H1']` are not defined. No namespaces are defined for the target.\n - Data generator `H2_2_task1` is invalid.\n - Variable `p1_H2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H2']` are not defined. No namespaces are defined for the target.\n - Data generator `M1_3_task1` is invalid.\n - Variable `p1_M1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M1']` are not defined. No namespaces are defined for the target.\n - Data generator `M2_4_task1` is invalid.\n - Variable `p1_M2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M2']` are not defined. No namespaces are defined for the target.\n - Data generator `T_5_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000770](https://www.ebi.ac.uk/biomodels/BIOMD0000000770)<br/><sup>Eftimie2017/1 - interaction of</sup></summary><sup>Eftimie2017/1 - interaction of Th and macrophage</sup></details>|<details><summary>pass</summary>[Eftimie2017:1.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000770#Files)</details>|FAIL|<details><summary>pass</summary>[Eftimie2017:1.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000770#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Eftimie2017:1.sedml295689.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Eftimie2017:1.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (10541). The following is the first warning at line 1019, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'H1_M1') are second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 1, multiplier = 1, scale = -3).\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 1073, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tme * growth(ph1, H1, M1, H2, m1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Eftimie2017:1.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Eftimie2017:1.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (10541). The following is the first warning at line 1019, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'H1_M1') are second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 1, multiplier = 1, scale = -3).\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 1073, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tme * growth(ph1, H1, M1, H2, m1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\x1b[0m\r\n\r\n`/root/in/Eftimie2017:1.sedml295689.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Eftimie2017:1.sedml` is invalid.\n - Data generator `H1_1_task1` is invalid.\n - Variable `p1_H1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H1']` are not defined. No namespaces are defined for the target.\n - Data generator `H2_2_task1` is invalid.\n - Variable `p1_H2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H2']` are not defined. No namespaces are defined for the target.\n - Data generator `M1_3_task1` is invalid.\n - Variable `p1_M1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M1']` are not defined. No namespaces are defined for the target.\n - Data generator `M2_4_task1` is invalid.\n - Variable `p1_M2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M2']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Eftimie2017:1.sedml856840.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Eftimie2017:1.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (10541). The following is the first warning at line 1019, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'H1_M1') are second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 1, multiplier = 1, scale = -3).\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 1073, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tme * growth(ph1, H1, M1, H2, m1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Eftimie2017:1.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Eftimie2017:1.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (10541). The following is the first warning at line 1019, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'H1_M1') are second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 1, multiplier = 1, scale = -3).\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 1073, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tme * growth(ph1, H1, M1, H2, m1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Eftimie2017:1.sedml856840.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Eftimie2017:1.sedml` is invalid.\n - Data generator `H1_1_task1` is invalid.\n - Variable `p1_H1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H1']` are not defined. No namespaces are defined for the target.\n - Data generator `H2_2_task1` is invalid.\n - Variable `p1_H2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H2']` are not defined. No namespaces are defined for the target.\n - Data generator `M1_3_task1` is invalid.\n - Variable `p1_M1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M1']` are not defined. No namespaces are defined for the target.\n - Data generator `M2_4_task1` is invalid.\n - Variable `p1_M2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M2']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000771](https://www.ebi.ac.uk/biomodels/BIOMD0000000771)<br/><sup>Bajzer2008 - Modeling of cance</sup></summary><sup>Bajzer2008 - Modeling of cancer virotherapy with recombinant measles viruses</sup></details>|<details><summary>pass</summary>[Bazjer2008.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000771#Files)</details>|FAIL|<details><summary>pass</summary>[Bazjer2008.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000771#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Bazjer2008.sedml387912.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Bazjer2008.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 497, column 8:\r\n - The formula 'pow(y_Tumor_Cell + x_Infected_Cell, epsilon)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 497, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth(r, y_Tumor_Cell, x_Infected_Cell, epsilon, K)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 343, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Bazjer2008.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Bazjer2008.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 497, column 8:\r\n - The formula 'pow(y_Tumor_Cell + x_Infected_Cell, epsilon)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 497, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth(r, y_Tumor_Cell, x_Infected_Cell, epsilon, K)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 343, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Bazjer2008.sedml387912.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Bazjer2008.sedml` is invalid.\n - Data generator `y_Tumor_Cell_1_task1` is invalid.\n - Variable `p1_y_Tumor_Cell_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='y_Tumor_Cell']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Bazjer2008.sedml800560.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Bazjer2008.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 497, column 8:\r\n - The formula 'pow(y_Tumor_Cell + x_Infected_Cell, epsilon)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 497, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth(r, y_Tumor_Cell, x_Infected_Cell, epsilon, K)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 343, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Bazjer2008.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Bazjer2008.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 497, column 8:\r\n - The formula 'pow(y_Tumor_Cell + x_Infected_Cell, epsilon)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 497, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth(r, y_Tumor_Cell, x_Infected_Cell, epsilon, K)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 343, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Bazjer2008.sedml800560.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Bazjer2008.sedml` is invalid.\n - Data generator `y_Tumor_Cell_1_task1` is invalid.\n - Variable `p1_y_Tumor_Cell_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='y_Tumor_Cell']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000772](https://www.ebi.ac.uk/biomodels/BIOMD0000000772)<br/><sup>Wang2019 - A mathematical mode</sup></summary><sup>Wang2019 - A mathematical model of oncolytic virotherapy with time delay</sup></details>|<details><summary>pass</summary>[Wang2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000772#Files)</details>|pass|<details><summary>pass</summary>[Wang2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000772#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Wang2019.sedml330282.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Wang2019.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 517, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Uninfected_Cells_Proliferation(r, x, y, k)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type Modeling practice (80701). The following is the first warning at line 418, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Wang2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Wang2019.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 517, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Uninfected_Cells_Proliferation(r, x, y, k)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type Modeling practice (80701). The following is the first warning at line 418, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Wang2019.sedml330282.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Wang2019.sedml` is invalid.\n - Data generator `x_1_task1` is invalid.\n - Variable `p1_x_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x']` are not defined. No namespaces are defined for the target.\n - Data generator `y_2_task1` is invalid.\n - Variable `p1_y_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='y']` are not defined. No namespaces are defined for the target.\n - Data generator `z_3_task1` is invalid.\n - Variable `p1_z_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='z']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Wang2019.sedml475447.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Wang2019.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 517, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Uninfected_Cells_Proliferation(r, x, y, k)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type Modeling practice (80701). The following is the first warning at line 418, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Wang2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Wang2019.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 517, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Uninfected_Cells_Proliferation(r, x, y, k)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type Modeling practice (80701). The following is the first warning at line 418, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Wang2019.sedml475447.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Wang2019.sedml` is invalid.\n - Data generator `x_1_task1` is invalid.\n - Variable `p1_x_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x']` are not defined. No namespaces are defined for the target.\n - Data generator `y_2_task1` is invalid.\n - Variable `p1_y_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='y']` are not defined. No namespaces are defined for the target.\n - Data generator `z_3_task1` is invalid.\n - Variable `p1_z_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='z']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000773](https://www.ebi.ac.uk/biomodels/BIOMD0000000773)<br/><sup>Wodarz2018/2 - model with tran</sup></summary><sup>Wodarz2018/2 - model with transit amplifying cells</sup></details>|<details><summary>pass</summary>[Wodarz2018:2.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000773#Files)</details>|FAIL|<details><summary>pass</summary>[Wodarz2018:2.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000773#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Wodarz2018:2.sedml826921.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Wodarz2018:2.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 1088, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'r2 / (1 + h3 * pow(D, k3))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 5 warnings of type SBML unit consistency (10513). The following is the first warning at line 1088, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'r2_0' are mole (exponent = -1, multiplier = 0.001, scale = 0), litre (exponent = 1, multiplier = 0.001, scale = 0).\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 1490, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'differentiation_cell_death') are mole (exponent = 1, multiplier = 1, scale = -3).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 1.0\r\n - Output end time: 3000.0\r\n - Number of steps: 3749\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Wodarz2018:2.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Wodarz2018:2.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 1088, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'r2 / (1 + h3 * pow(D, k3))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 5 warnings of type SBML unit consistency (10513). The following is the first warning at line 1088, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'r2_0' are mole (exponent = -1, multiplier = 0.001, scale = 0), litre (exponent = 1, multiplier = 0.001, scale = 0).\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 1490, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'differentiation_cell_death') are mole (exponent = 1, multiplier = 1, scale = -3).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 1.0\r\n - Output end time: 3000.0\r\n - Number of steps: 3749\x1b[0m\r\n\r\n`/root/in/Wodarz2018:2.sedml826921.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Wodarz2018:2.sedml` is invalid.\n - Data generator `D_1_task1` is invalid.\n - Variable `p1_D_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='D']` are not defined. No namespaces are defined for the target.\n - Data generator `S_2_task1` is invalid.\n - Variable `p1_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S']` are not defined. No namespaces are defined for the target.\n - Data generator `T_3_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Wodarz2018:2.sedml516485.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Wodarz2018:2.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 1088, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'r2 / (1 + h3 * pow(D, k3))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 5 warnings of type SBML unit consistency (10513). The following is the first warning at line 1088, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'r2_0' are mole (exponent = -1, multiplier = 0.001, scale = 0), litre (exponent = 1, multiplier = 0.001, scale = 0).\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 1490, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'differentiation_cell_death') are mole (exponent = 1, multiplier = 1, scale = -3).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 1.0\r\n - Output end time: 3000.0\r\n - Number of steps: 3749\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Wodarz2018:2.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Wodarz2018:2.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 1088, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'r2 / (1 + h3 * pow(D, k3))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 5 warnings of type SBML unit consistency (10513). The following is the first warning at line 1088, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'r2_0' are mole (exponent = -1, multiplier = 0.001, scale = 0), litre (exponent = 1, multiplier = 0.001, scale = 0).\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 1490, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'differentiation_cell_death') are mole (exponent = 1, multiplier = 1, scale = -3).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 1.0\r\n - Output end time: 3000.0\r\n - Number of steps: 3749\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Wodarz2018:2.sedml516485.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Wodarz2018:2.sedml` is invalid.\n - Data generator `D_1_task1` is invalid.\n - Variable `p1_D_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='D']` are not defined. No namespaces are defined for the target.\n - Data generator `S_2_task1` is invalid.\n - Variable `p1_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S']` are not defined. No namespaces are defined for the target.\n - Data generator `T_3_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|
|[BIOMD0000000774](https://www.ebi.ac.uk/biomodels/BIOMD0000000774)<br/><sup>Wodarz2018/1 - simple model</sup>|<details><summary>pass</summary>[Wodarz2018:1.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000774#Files)</details>|FAIL|<details><summary>pass</summary>[Wodarz2018:1.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000774#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Wodarz2018:1.sedml673684.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Wodarz2018:1.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 698, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'p_0 / (1 + h2 * pow(D, k2))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 3 warnings of type SBML unit consistency (10513). The following is the first warning at line 698, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'p' are mole (exponent = -1, multiplier = 0.001, scale = 0), litre (exponent = 1, multiplier = 0.001, scale = 0).\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 846, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'de_differentiation') are mole (exponent = 1, multiplier = 1, scale = -3).\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Wodarz2018:1.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Wodarz2018:1.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 698, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'p_0 / (1 + h2 * pow(D, k2))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 3 warnings of type SBML unit consistency (10513). The following is the first warning at line 698, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'p' are mole (exponent = -1, multiplier = 0.001, scale = 0), litre (exponent = 1, multiplier = 0.001, scale = 0).\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 846, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'de_differentiation') are mole (exponent = 1, multiplier = 1, scale = -3).\x1b[0m\r\n\r\n`/root/in/Wodarz2018:1.sedml673684.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Wodarz2018:1.sedml` is invalid.\n - Data generator `D_1_task1` is invalid.\n - Variable `p1_D_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='D']` are not defined. No namespaces are defined for the target.\n - Data generator `S_2_task1` is invalid.\n - Variable `p1_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Wodarz2018:1.sedml772394.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Wodarz2018:1.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 698, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'p_0 / (1 + h2 * pow(D, k2))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 3 warnings of type SBML unit consistency (10513). The following is the first warning at line 698, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'p' are mole (exponent = -1, multiplier = 0.001, scale = 0), litre (exponent = 1, multiplier = 0.001, scale = 0).\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 846, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'de_differentiation') are mole (exponent = 1, multiplier = 1, scale = -3).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Wodarz2018:1.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Wodarz2018:1.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 698, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'p_0 / (1 + h2 * pow(D, k2))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 3 warnings of type SBML unit consistency (10513). The following is the first warning at line 698, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'p' are mole (exponent = -1, multiplier = 0.001, scale = 0), litre (exponent = 1, multiplier = 0.001, scale = 0).\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 846, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'de_differentiation') are mole (exponent = 1, multiplier = 1, scale = -3).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Wodarz2018:1.sedml772394.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Wodarz2018:1.sedml` is invalid.\n - Data generator `D_1_task1` is invalid.\n - Variable `p1_D_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='D']` are not defined. No namespaces are defined for the target.\n - Data generator `S_2_task1` is invalid.\n - Variable `p1_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S']` are not defined. No namespaces are defined for the target.\n"```</details>|
|[BIOMD0000000775](https://www.ebi.ac.uk/biomodels/BIOMD0000000775)<br/><sup>Iarosz2015 - brain tumor</sup>|<details><summary>pass</summary>[Iarosz2015.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000775#Files)</details>|FAIL|<details><summary>pass</summary>[Iarosz2015.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000775#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Iarosz2015.sedml499259.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Iarosz2015.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 1181, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'a1 + G' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 3 warnings of type SBML unit consistency (10503). The following is the first warning at line 1356, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * n(a, gg, H, N)' in the <kineticLaw> element of the <reaction> with id 'n_increase' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 987, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'rateOf(G)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 8 warnings of type SBML unit consistency (10541). The following is the first warning at line 1127, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'gc_death') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 2, multiplier = 0.001, scale = 0).\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Iarosz2015.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Iarosz2015.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 1181, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'a1 + G' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 3 warnings of type SBML unit consistency (10503). The following is the first warning at line 1356, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * n(a, gg, H, N)' in the <kineticLaw> element of the <reaction> with id 'n_increase' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 987, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'rateOf(G)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 8 warnings of type SBML unit consistency (10541). The following is the first warning at line 1127, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'gc_death') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 2, multiplier = 0.001, scale = 0).\x1b[0m\r\n\r\n`/root/in/Iarosz2015.sedml499259.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Iarosz2015.sedml` is invalid.\n - Data generator `G_1_task1` is invalid.\n - Variable `p1_G_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='G']` are not defined. No namespaces are defined for the target.\n - Data generator `N_1_task1` is invalid.\n - Variable `p2_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n - Data generator `C_1_task1` is invalid.\n - Variable `p3_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C']` are not defined. No namespaces are defined for the target.\n - Data generator `Q_1_task1` is invalid.\n - Variable `p4_Q_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Q']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Iarosz2015.sedml214962.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Iarosz2015.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 1181, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'a1 + G' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 3 warnings of type SBML unit consistency (10503). The following is the first warning at line 1356, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * n(a, gg, H, N)' in the <kineticLaw> element of the <reaction> with id 'n_increase' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 987, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'rateOf(G)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 8 warnings of type SBML unit consistency (10541). The following is the first warning at line 1127, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'gc_death') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 2, multiplier = 0.001, scale = 0).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Iarosz2015.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Iarosz2015.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 1181, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'a1 + G' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 3 warnings of type SBML unit consistency (10503). The following is the first warning at line 1356, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * n(a, gg, H, N)' in the <kineticLaw> element of the <reaction> with id 'n_increase' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 987, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'rateOf(G)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 8 warnings of type SBML unit consistency (10541). The following is the first warning at line 1127, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'gc_death') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 2, multiplier = 0.001, scale = 0).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Iarosz2015.sedml214962.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Iarosz2015.sedml` is invalid.\n - Data generator `G_1_task1` is invalid.\n - Variable `p1_G_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='G']` are not defined. No namespaces are defined for the target.\n - Data generator `N_1_task1` is invalid.\n - Variable `p2_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n - Data generator `C_1_task1` is invalid.\n - Variable `p3_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C']` are not defined. No namespaces are defined for the target.\n - Data generator `Q_1_task1` is invalid.\n - Variable `p4_Q_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Q']` are not defined. No namespaces are defined for the target.\n"```</details>|
|[BIOMD0000000776](https://www.ebi.ac.uk/biomodels/BIOMD0000000776)<br/><sup>Monro2008 - chemotherapy resistance</sup>|<details><summary>pass</summary>[Monro2008.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000776#Files)</details>|FAIL|<details><summary>pass</summary>[Monro2008.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000776#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Monro2008.sedml948696.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Monro2008.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10513). The following is the first warning at line 627, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'N' are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3).\r\n - 4 warnings of type SBML unit consistency (10541). The following is the first warning at line 674, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'sensitive_cell_growth') are second (exponent = -1, multiplier = 8640, scale = 0), mole (exponent = 1, multiplier = 1, scale = -3).\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Monro2008.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Monro2008.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10513). The following is the first warning at line 627, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'N' are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3).\r\n - 4 warnings of type SBML unit consistency (10541). The following is the first warning at line 674, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'sensitive_cell_growth') are second (exponent = -1, multiplier = 8640, scale = 0), mole (exponent = 1, multiplier = 1, scale = -3).\x1b[0m\r\n\r\n`/root/in/Monro2008.sedml948696.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Monro2008.sedml` is invalid.\n - Data generator `Ncrit_1_task1` is invalid.\n - Variable `p1_Ncrit_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Ncrit']` are not defined. No namespaces are defined for the target.\n - Data generator `R_2_task1` is invalid.\n - Variable `p1_R_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='R']` are not defined. No namespaces are defined for the target.\n - Data generator `S_3_task1` is invalid.\n - Variable `p1_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S']` are not defined. No namespaces are defined for the target.\n - Data generator `N_4_task1` is invalid.\n - Variable `p1_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='N']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Monro2008.sedml526865.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Monro2008.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10513). The following is the first warning at line 627, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'N' are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3).\r\n - 4 warnings of type SBML unit consistency (10541). The following is the first warning at line 674, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'sensitive_cell_growth') are second (exponent = -1, multiplier = 8640, scale = 0), mole (exponent = 1, multiplier = 1, scale = -3).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Monro2008.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Monro2008.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10513). The following is the first warning at line 627, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'N' are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3).\r\n - 4 warnings of type SBML unit consistency (10541). The following is the first warning at line 674, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'sensitive_cell_growth') are second (exponent = -1, multiplier = 8640, scale = 0), mole (exponent = 1, multiplier = 1, scale = -3).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Monro2008.sedml526865.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Monro2008.sedml` is invalid.\n - Data generator `Ncrit_1_task1` is invalid.\n - Variable `p1_Ncrit_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Ncrit']` are not defined. No namespaces are defined for the target.\n - Data generator `R_2_task1` is invalid.\n - Variable `p1_R_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='R']` are not defined. No namespaces are defined for the target.\n - Data generator `S_3_task1` is invalid.\n - Variable `p1_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S']` are not defined. No namespaces are defined for the target.\n - Data generator `N_4_task1` is invalid.\n - Variable `p1_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='N']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000777](https://www.ebi.ac.uk/biomodels/BIOMD0000000777)<br/><sup>Chakrabarty2010 - A control th</sup></summary><sup>Chakrabarty2010 - A control theory approach to cancer remission aided by an optimal therapy</sup></details>|<details><summary>pass</summary>[Chakrabarty2010.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000777#Files)</details>|pass|<details><summary>FAIL</summary>[Chakrabarty2010.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000777#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Chakrabarty2010.sedml539429.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/Chakrabarty2010.sedml539429.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Chakrabarty2010.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmpwdikibis/./model1.xml` is not a file.\n - Data generator `M_Tumor_Cells_1_task1` is invalid.\n - Variable `p1_M_Tumor_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M_Tumor_Cells']` are not defined. No namespaces are defined for the target.\n - Data generator `N_CTL_2_task1` is invalid.\n - Variable `p1_N_CTL_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N_CTL']` are not defined. No namespaces are defined for the target.\n - Data generator `Z_THL_3_task1` is invalid.\n - Variable `p1_Z_THL_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Z_THL']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Chakrabarty2010.sedml807347.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Chakrabarty2010.sedml807347.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Chakrabarty2010.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmpip4kviya/./model1.xml` is not a file.\n - Data generator `M_Tumor_Cells_1_task1` is invalid.\n - Variable `p1_M_Tumor_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M_Tumor_Cells']` are not defined. No namespaces are defined for the target.\n - Data generator `N_CTL_2_task1` is invalid.\n - Variable `p1_N_CTL_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N_CTL']` are not defined. No namespaces are defined for the target.\n - Data generator `Z_THL_3_task1` is invalid.\n - Variable `p1_Z_THL_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Z_THL']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000778](https://www.ebi.ac.uk/biomodels/BIOMD0000000778)<br/><sup>Wei2017 - tumor, T cell and cy</sup></summary><sup>Wei2017 - tumor, T cell and cytokine interaction</sup></details>|<details><summary>pass</summary>[Wei2017.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000778#Files)</details>|FAIL|<details><summary>pass</summary>[Wei2017.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000778#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Wei2017.sedml155291.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Wei2017.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 1043, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'm + T' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 4 warnings of type SBML unit consistency (10503). The following is the first warning at line 1114, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * a * I' in the <kineticLaw> element of the <reaction> with id 'immune_cell_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 927, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'I / 100000' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type SBML unit consistency (10541). The following is the first warning at line 1043, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_killing') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 2, multiplier = 0.001, scale = 0).\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Wei2017.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Wei2017.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 1043, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'm + T' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 4 warnings of type SBML unit consistency (10503). The following is the first warning at line 1114, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * a * I' in the <kineticLaw> element of the <reaction> with id 'immune_cell_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 927, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'I / 100000' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type SBML unit consistency (10541). The following is the first warning at line 1043, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_killing') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 2, multiplier = 0.001, scale = 0).\x1b[0m\r\n\r\n`/root/in/Wei2017.sedml155291.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Wei2017.sedml` is invalid.\n - Data generator `I_plot_1_task1` is invalid.\n - Variable `p1_I_plot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='I_plot']` are not defined. No namespaces are defined for the target.\n - Data generator `T_plot_2_task1` is invalid.\n - Variable `p1_T_plot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='T_plot']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Wei2017.sedml332122.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Wei2017.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 1043, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'm + T' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 4 warnings of type SBML unit consistency (10503). The following is the first warning at line 1114, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * a * I' in the <kineticLaw> element of the <reaction> with id 'immune_cell_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 927, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'I / 100000' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type SBML unit consistency (10541). The following is the first warning at line 1043, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_killing') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 2, multiplier = 0.001, scale = 0).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Wei2017.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Wei2017.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 1043, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'm + T' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 4 warnings of type SBML unit consistency (10503). The following is the first warning at line 1114, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * a * I' in the <kineticLaw> element of the <reaction> with id 'immune_cell_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 927, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'I / 100000' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type SBML unit consistency (10541). The following is the first warning at line 1043, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_killing') are litre (exponent = -1, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 2, multiplier = 0.001, scale = 0).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Wei2017.sedml332122.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Wei2017.sedml` is invalid.\n - Data generator `I_plot_1_task1` is invalid.\n - Variable `p1_I_plot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='I_plot']` are not defined. No namespaces are defined for the target.\n - Data generator `T_plot_2_task1` is invalid.\n - Variable `p1_T_plot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='T_plot']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000779](https://www.ebi.ac.uk/biomodels/BIOMD0000000779)<br/><sup>dePillis2009 - Mathematical mo</sup></summary><sup>dePillis2009 - Mathematical model creation for cancer chemo-immunotherapy</sup></details>|<details><summary>pass</summary>[dePillis2009.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000779#Files)</details>|pass|<details><summary>pass</summary>[dePillis2009.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000779#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/dePillis2009.sedml381533.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `dePillis2009.xml` has warnings.\r\n - 29 warnings of type SBML unit consistency (99505). The following is the first warning at line 1267, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'd' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 44 warnings of type Modeling practice (80701). The following is the first warning at line 1083, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./dePillis2009.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `dePillis2009.xml` has warnings.\r\n - 29 warnings of type SBML unit consistency (99505). The following is the first warning at line 1267, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'd' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 44 warnings of type Modeling practice (80701). The following is the first warning at line 1083, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/dePillis2009.sedml381533.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./dePillis2009.sedml` is invalid.\n - Data generator `C_Lymphocytes_1_task1` is invalid.\n - Variable `p1_C_Lymphocytes_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C_Lymphocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `I_IL_2_2_task1` is invalid.\n - Variable `p1_I_IL_2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I_IL_2']` are not defined. No namespaces are defined for the target.\n - Data generator `L_CD8_T_Cells_3_task1` is invalid.\n - Variable `p1_L_CD8_T_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L_CD8_T_Cells']` are not defined. No namespaces are defined for the target.\n - Data generator `N_Natural_Killer_Cells_4_task1` is invalid.\n - Variable `p1_N_Natural_Killer_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N_Natural_Killer_Cells']` are not defined. No namespaces are defined for the target.\n - Data generator `T_Tumour_Cells_5_task1` is invalid.\n - Variable `p1_T_Tumour_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T_Tumour_Cells']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/dePillis2009.sedml49953.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `dePillis2009.xml` has warnings.\r\n - 29 warnings of type SBML unit consistency (99505). The following is the first warning at line 1267, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'd' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 44 warnings of type Modeling practice (80701). The following is the first warning at line 1083, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./dePillis2009.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `dePillis2009.xml` has warnings.\r\n - 29 warnings of type SBML unit consistency (99505). The following is the first warning at line 1267, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'd' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 44 warnings of type Modeling practice (80701). The following is the first warning at line 1083, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/dePillis2009.sedml49953.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./dePillis2009.sedml` is invalid.\n - Data generator `C_Lymphocytes_1_task1` is invalid.\n - Variable `p1_C_Lymphocytes_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C_Lymphocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `I_IL_2_2_task1` is invalid.\n - Variable `p1_I_IL_2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I_IL_2']` are not defined. No namespaces are defined for the target.\n - Data generator `L_CD8_T_Cells_3_task1` is invalid.\n - Variable `p1_L_CD8_T_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L_CD8_T_Cells']` are not defined. No namespaces are defined for the target.\n - Data generator `N_Natural_Killer_Cells_4_task1` is invalid.\n - Variable `p1_N_Natural_Killer_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N_Natural_Killer_Cells']` are not defined. No namespaces are defined for the target.\n - Data generator `T_Tumour_Cells_5_task1` is invalid.\n - Variable `p1_T_Tumour_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T_Tumour_Cells']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000780](https://www.ebi.ac.uk/biomodels/BIOMD0000000780)<br/><sup>Wang2016/1 - oncolytic efficac</sup></summary><sup>Wang2016/1 - oncolytic efficacy of M1 virus-SNTM model</sup></details>|<details><summary>pass</summary>[Wang2016:1.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000780#Files)</details>|FAIL|<details><summary>pass</summary>[Wang2016:1.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000780#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Wang2016:1.sedml55404.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Wang2016:1.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (10503). The following is the first warning at line 1040, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * d * S' in the <kineticLaw> element of the <reaction> with id 'nutrient_decay' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 12 warnings of type SBML unit consistency (10541). The following is the first warning at line 1027, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'nutrient_supply') are litre (exponent = 1, multiplier = 0.001, scale = 0).\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Wang2016:1.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Wang2016:1.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (10503). The following is the first warning at line 1040, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * d * S' in the <kineticLaw> element of the <reaction> with id 'nutrient_decay' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 12 warnings of type SBML unit consistency (10541). The following is the first warning at line 1027, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'nutrient_supply') are litre (exponent = 1, multiplier = 0.001, scale = 0).\x1b[0m\r\n\r\n`/root/in/Wang2016:1.sedml55404.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Wang2016:1.sedml` is invalid.\n - Data generator `M_1_task1` is invalid.\n - Variable `p1_M_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M']` are not defined. No namespaces are defined for the target.\n - Data generator `N_2_task1` is invalid.\n - Variable `p1_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n - Data generator `S_3_task1` is invalid.\n - Variable `p1_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S']` are not defined. No namespaces are defined for the target.\n - Data generator `T_4_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Wang2016:1.sedml783545.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Wang2016:1.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (10503). The following is the first warning at line 1040, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * d * S' in the <kineticLaw> element of the <reaction> with id 'nutrient_decay' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 12 warnings of type SBML unit consistency (10541). The following is the first warning at line 1027, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'nutrient_supply') are litre (exponent = 1, multiplier = 0.001, scale = 0).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Wang2016:1.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Wang2016:1.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (10503). The following is the first warning at line 1040, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * d * S' in the <kineticLaw> element of the <reaction> with id 'nutrient_decay' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 12 warnings of type SBML unit consistency (10541). The following is the first warning at line 1027, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'nutrient_supply') are litre (exponent = 1, multiplier = 0.001, scale = 0).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Wang2016:1.sedml783545.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Wang2016:1.sedml` is invalid.\n - Data generator `M_1_task1` is invalid.\n - Variable `p1_M_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M']` are not defined. No namespaces are defined for the target.\n - Data generator `N_2_task1` is invalid.\n - Variable `p1_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n - Data generator `S_3_task1` is invalid.\n - Variable `p1_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S']` are not defined. No namespaces are defined for the target.\n - Data generator `T_4_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000781](https://www.ebi.ac.uk/biomodels/BIOMD0000000781)<br/><sup>Wang2016/2 - oncolytic efficac</sup></summary><sup>Wang2016/2 - oncolytic efficacy of M1 virus-SNT model</sup></details>|<details><summary>pass</summary>[Wang2016:2.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000781#Files)</details>|FAIL|<details><summary>pass</summary>[Wang2016:2.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000781#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Wang2016:2.sedml101909.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Wang2016:2.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (10503). The following is the first warning at line 825, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * d * S' in the <kineticLaw> element of the <reaction> with id 'nutrient_decay' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 8 warnings of type SBML unit consistency (10541). The following is the first warning at line 812, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'nutrient_supply') are litre (exponent = 1, multiplier = 0.001, scale = 0).\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Wang2016:2.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Wang2016:2.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (10503). The following is the first warning at line 825, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * d * S' in the <kineticLaw> element of the <reaction> with id 'nutrient_decay' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 8 warnings of type SBML unit consistency (10541). The following is the first warning at line 812, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'nutrient_supply') are litre (exponent = 1, multiplier = 0.001, scale = 0).\x1b[0m\r\n\r\n`/root/in/Wang2016:2.sedml101909.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Wang2016:2.sedml` is invalid.\n - Data generator `N_1_task1` is invalid.\n - Variable `p1_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n - Data generator `S_2_task1` is invalid.\n - Variable `p1_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S']` are not defined. No namespaces are defined for the target.\n - Data generator `T_3_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Wang2016:2.sedml180506.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Wang2016:2.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (10503). The following is the first warning at line 825, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * d * S' in the <kineticLaw> element of the <reaction> with id 'nutrient_decay' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 8 warnings of type SBML unit consistency (10541). The following is the first warning at line 812, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'nutrient_supply') are litre (exponent = 1, multiplier = 0.001, scale = 0).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Wang2016:2.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Wang2016:2.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (10503). The following is the first warning at line 825, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * d * S' in the <kineticLaw> element of the <reaction> with id 'nutrient_decay' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 8 warnings of type SBML unit consistency (10541). The following is the first warning at line 812, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'nutrient_supply') are litre (exponent = 1, multiplier = 0.001, scale = 0).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Wang2016:2.sedml180506.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Wang2016:2.sedml` is invalid.\n - Data generator `N_1_task1` is invalid.\n - Variable `p1_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n - Data generator `S_2_task1` is invalid.\n - Variable `p1_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S']` are not defined. No namespaces are defined for the target.\n - Data generator `T_3_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000782](https://www.ebi.ac.uk/biomodels/BIOMD0000000782)<br/><sup>Wang2016/3 - oncolytic efficac</sup></summary><sup>Wang2016/3 - oncolytic efficacy of M1 virus-SN model</sup></details>|<details><summary>pass</summary>[Wang2016:3.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000782#Files)</details>|FAIL|<details><summary>pass</summary>[Wang2016:3.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000782#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Wang2016:3.sedml457821.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Wang2016:3.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (10503). The following is the first warning at line 661, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * d * S' in the <kineticLaw> element of the <reaction> with id 'nutrient_decay' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 5 warnings of type SBML unit consistency (10541). The following is the first warning at line 648, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'nutrient_supply') are litre (exponent = 1, multiplier = 0.001, scale = 0).\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Wang2016:3.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Wang2016:3.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (10503). The following is the first warning at line 661, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * d * S' in the <kineticLaw> element of the <reaction> with id 'nutrient_decay' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 5 warnings of type SBML unit consistency (10541). The following is the first warning at line 648, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'nutrient_supply') are litre (exponent = 1, multiplier = 0.001, scale = 0).\x1b[0m\r\n\r\n`/root/in/Wang2016:3.sedml457821.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Wang2016:3.sedml` is invalid.\n - Data generator `N_1_task1` is invalid.\n - Variable `p1_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n - Data generator `S_2_task1` is invalid.\n - Variable `p1_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Wang2016:3.sedml524251.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Wang2016:3.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (10503). The following is the first warning at line 661, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * d * S' in the <kineticLaw> element of the <reaction> with id 'nutrient_decay' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 5 warnings of type SBML unit consistency (10541). The following is the first warning at line 648, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'nutrient_supply') are litre (exponent = 1, multiplier = 0.001, scale = 0).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Wang2016:3.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Wang2016:3.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (10503). The following is the first warning at line 661, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * d * S' in the <kineticLaw> element of the <reaction> with id 'nutrient_decay' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 5 warnings of type SBML unit consistency (10541). The following is the first warning at line 648, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'nutrient_supply') are litre (exponent = 1, multiplier = 0.001, scale = 0).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Wang2016:3.sedml524251.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Wang2016:3.sedml` is invalid.\n - Data generator `N_1_task1` is invalid.\n - Variable `p1_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n - Data generator `S_2_task1` is invalid.\n - Variable `p1_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000783](https://www.ebi.ac.uk/biomodels/BIOMD0000000783)<br/><sup>Dong2014 - Mathematical modeli</sup></summary><sup>Dong2014 - Mathematical modeling on helper t cells in a tumor immune system</sup></details>|<details><summary>pass</summary>[Dong2014.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000783#Files)</details>|FAIL|<details><summary>pass</summary>[Dong2014.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000783#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Dong2014.sedml104304.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Dong2014.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 459, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth(alpha, x_Tumor_Cells, beta)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 505, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Tumor_Killing_ECs') are litre (exponent = -1, multiplier = 0.001, scale = 0), item (exponent = 2, multiplier = 1, scale = 0).\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 421, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Dong2014.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Dong2014.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 459, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth(alpha, x_Tumor_Cells, beta)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 505, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Tumor_Killing_ECs') are litre (exponent = -1, multiplier = 0.001, scale = 0), item (exponent = 2, multiplier = 1, scale = 0).\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 421, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Dong2014.sedml104304.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Dong2014.sedml` is invalid.\n - Data generator `x_Tumor_Cells_1_task1` is invalid.\n - Variable `p1_x_Tumor_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x_Tumor_Cells']` are not defined. No namespaces are defined for the target.\n - Data generator `y_Effector_Cells_2_task1` is invalid.\n - Variable `p1_y_Effector_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='y_Effector_Cells']` are not defined. No namespaces are defined for the target.\n - Data generator `z_Helper_T_Cells_3_task1` is invalid.\n - Variable `p1_z_Helper_T_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='z_Helper_T_Cells']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Dong2014.sedml178892.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Dong2014.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 459, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth(alpha, x_Tumor_Cells, beta)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 505, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Tumor_Killing_ECs') are litre (exponent = -1, multiplier = 0.001, scale = 0), item (exponent = 2, multiplier = 1, scale = 0).\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 421, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Dong2014.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Dong2014.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 459, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth(alpha, x_Tumor_Cells, beta)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 505, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Tumor_Killing_ECs') are litre (exponent = -1, multiplier = 0.001, scale = 0), item (exponent = 2, multiplier = 1, scale = 0).\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 421, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Dong2014.sedml178892.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Dong2014.sedml` is invalid.\n - Data generator `x_Tumor_Cells_1_task1` is invalid.\n - Variable `p1_x_Tumor_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x_Tumor_Cells']` are not defined. No namespaces are defined for the target.\n - Data generator `y_Effector_Cells_2_task1` is invalid.\n - Variable `p1_y_Effector_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='y_Effector_Cells']` are not defined. No namespaces are defined for the target.\n - Data generator `z_Helper_T_Cells_3_task1` is invalid.\n - Variable `p1_z_Helper_T_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='z_Helper_T_Cells']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000784](https://www.ebi.ac.uk/biomodels/BIOMD0000000784)<br/><sup>Lopez2014 - A Validated Mathem</sup></summary><sup>Lopez2014 - A Validated Mathematical Model of Tumor Growth Including Tumor-Host Interaction and Cell-Mediated Immune Response</sup></details>|<details><summary>pass</summary>[Lopez2014.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000784#Files)</details>|FAIL|<details><summary>pass</summary>[Lopez2014.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000784#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Lopez2014.sedml179439.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Lopez2014.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 547, column 6:\r\n - The formula 'pow(x_Tumor_Cells, ModelValue_2)' in the math element of the <assignmentRule> contains a power that is not an integer and thus may produce invalid units.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 530, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'd' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 16 warnings of type Modeling practice (80701). The following is the first warning at line 473, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a_12' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Lopez2014.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Lopez2014.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 547, column 6:\r\n - The formula 'pow(x_Tumor_Cells, ModelValue_2)' in the math element of the <assignmentRule> contains a power that is not an integer and thus may produce invalid units.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 530, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'd' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 16 warnings of type Modeling practice (80701). The following is the first warning at line 473, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a_12' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Lopez2014.sedml179439.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Lopez2014.sedml` is invalid.\n - Data generator `x_Tumor_Cells_1_task1` is invalid.\n - Variable `p1_x_Tumor_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x_Tumor_Cells']` are not defined. No namespaces are defined for the target.\n - Data generator `y_Healthy_Cells_2_task1` is invalid.\n - Variable `p1_y_Healthy_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='y_Healthy_Cells']` are not defined. No namespaces are defined for the target.\n - Data generator `z_Effector_Cells_3_task1` is invalid.\n - Variable `p1_z_Effector_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='z_Effector_Cells']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Lopez2014.sedml832558.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Lopez2014.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 547, column 6:\r\n - The formula 'pow(x_Tumor_Cells, ModelValue_2)' in the math element of the <assignmentRule> contains a power that is not an integer and thus may produce invalid units.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 530, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'd' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 16 warnings of type Modeling practice (80701). The following is the first warning at line 473, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a_12' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Lopez2014.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Lopez2014.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 547, column 6:\r\n - The formula 'pow(x_Tumor_Cells, ModelValue_2)' in the math element of the <assignmentRule> contains a power that is not an integer and thus may produce invalid units.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 530, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'd' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 16 warnings of type Modeling practice (80701). The following is the first warning at line 473, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a_12' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Lopez2014.sedml832558.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Lopez2014.sedml` is invalid.\n - Data generator `x_Tumor_Cells_1_task1` is invalid.\n - Variable `p1_x_Tumor_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x_Tumor_Cells']` are not defined. No namespaces are defined for the target.\n - Data generator `y_Healthy_Cells_2_task1` is invalid.\n - Variable `p1_y_Healthy_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='y_Healthy_Cells']` are not defined. No namespaces are defined for the target.\n - Data generator `z_Effector_Cells_3_task1` is invalid.\n - Variable `p1_z_Effector_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='z_Effector_Cells']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000785](https://www.ebi.ac.uk/biomodels/BIOMD0000000785)<br/><sup>Sotolongo-Costa2003 - Behavior</sup></summary><sup>Sotolongo-Costa2003 - Behavior of tumors under nonstationary therapy</sup></details>|<details><summary>pass</summary>[Sotolongo-Costa2003.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000785#Files)</details>|FAIL|<details><summary>pass</summary>[Sotolongo-Costa2003.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000785#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Sotolongo-Costa2003.sedml363814.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Sotolongo-Costa2003.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 393, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Malignant_Cells_Growth(alpha, x_Malignant_Cells)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 438, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Malignant_Cells_Death_Lymphocytes') are litre (exponent = -1, multiplier = 0.001, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - 3 warnings of type Modeling practice (80701). The following is the first warning at line 318, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Sotolongo-Costa2003.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Sotolongo-Costa2003.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 393, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Malignant_Cells_Growth(alpha, x_Malignant_Cells)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 438, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Malignant_Cells_Death_Lymphocytes') are litre (exponent = -1, multiplier = 0.001, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - 3 warnings of type Modeling practice (80701). The following is the first warning at line 318, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Sotolongo-Costa2003.sedml363814.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Sotolongo-Costa2003.sedml` is invalid.\n - Data generator `x_Malignant_Cells_1_task1` is invalid.\n - Variable `p1_x_Malignant_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x_Malignant_Cells']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Sotolongo-Costa2003.sedml287022.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Sotolongo-Costa2003.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 393, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Malignant_Cells_Growth(alpha, x_Malignant_Cells)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 438, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Malignant_Cells_Death_Lymphocytes') are litre (exponent = -1, multiplier = 0.001, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - 3 warnings of type Modeling practice (80701). The following is the first warning at line 318, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Sotolongo-Costa2003.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Sotolongo-Costa2003.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 393, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Malignant_Cells_Growth(alpha, x_Malignant_Cells)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 438, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Malignant_Cells_Death_Lymphocytes') are litre (exponent = -1, multiplier = 0.001, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - 3 warnings of type Modeling practice (80701). The following is the first warning at line 318, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Sotolongo-Costa2003.sedml287022.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Sotolongo-Costa2003.sedml` is invalid.\n - Data generator `x_Malignant_Cells_1_task1` is invalid.\n - Variable `p1_x_Malignant_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x_Malignant_Cells']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000786](https://www.ebi.ac.uk/biomodels/BIOMD0000000786)<br/><sup>Lipniacki2004 - Mathematical m</sup></summary><sup>Lipniacki2004 - Mathematical model of NFKB regulatory module</sup></details>|<details><summary>FAIL</summary>[Lipniacki2004.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000786#Files)</details>|FAIL|<details><summary>pass</summary>[Lipniacki2004.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000786#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Lipniacki2004.sedml546496.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/Lipniacki2004.sedml546496.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Lipniacki2004.sedml` is invalid.\n - Model `model` is invalid.\n - The model file `Lipniacki2004.xml` is invalid.\n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout' cannot be used in this document.\n \n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout_L2' cannot be used in this document.\n \n - General SBML conformance (10102) at line 3, column 2: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L2V4 Section 4.1\n Element 'listOfLayouts' is not part of the definition of SBML Level 2 Version 4.\n \n - Data generator `NFKB_nuc_1_task1` is invalid.\n - Variable `p1_NFKB_nuc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='NFKB_nuc']` are not defined. No namespaces are defined for the target.\n - Data generator `IKK_active_1_task1` is invalid.\n - Variable `p2_IKK_active_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IKK_active']` are not defined. No namespaces are defined for the target.\n - Data generator `IkB_mRNA_amp_1_task1` is invalid.\n - Variable `p3_IkB_mRNA_amp_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='IkB_mRNA_amp']` are not defined. No namespaces are defined for the target.\n - Data generator `A20_mRNA_amp_1_task1` is invalid.\n - Variable `p4_A20_mRNA_amp_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='A20_mRNA_amp']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Lipniacki2004.sedml589038.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Lipniacki2004.sedml589038.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Lipniacki2004.sedml` is invalid.\n - Model `model` is invalid.\n - The model file `Lipniacki2004.xml` is invalid.\n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout' cannot be used in this document.\n \n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout_L2' cannot be used in this document.\n \n - General SBML conformance (10102) at line 3, column 2: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L2V4 Section 4.1\n Element 'listOfLayouts' is not part of the definition of SBML Level 2 Version 4.\n \n - Data generator `NFKB_nuc_1_task1` is invalid.\n - Variable `p1_NFKB_nuc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='NFKB_nuc']` are not defined. No namespaces are defined for the target.\n - Data generator `IKK_active_1_task1` is invalid.\n - Variable `p2_IKK_active_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IKK_active']` are not defined. No namespaces are defined for the target.\n - Data generator `IkB_mRNA_amp_1_task1` is invalid.\n - Variable `p3_IkB_mRNA_amp_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='IkB_mRNA_amp']` are not defined. No namespaces are defined for the target.\n - Data generator `A20_mRNA_amp_1_task1` is invalid.\n - Variable `p4_A20_mRNA_amp_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='A20_mRNA_amp']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000787](https://www.ebi.ac.uk/biomodels/BIOMD0000000787)<br/><sup>Frascoli2014 - A dynamical mod</sup></summary><sup>Frascoli2014 - A dynamical model of tumour immunotherapy</sup></details>|<details><summary>pass</summary>[Frascoli2014.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000787#Files)</details>|FAIL|<details><summary>pass</summary>[Frascoli2014.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000787#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Frascoli2014.sedml506988.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Frascoli2014.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 474, column 8:\r\n - The formula 'pow(V_Tumor_Volume, 2 / 3)' in the math element of the <kineticLaw> contains an expression for the exponent of the power function and thus cannot be checked for unit validity.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 410, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(1, geq(C_Cytotoxic_T_Lymphocytes_Coverage, 1), C_Cytotoxic_T_Lymphocytes_Coverage)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 346, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r_t' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Frascoli2014.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Frascoli2014.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 474, column 8:\r\n - The formula 'pow(V_Tumor_Volume, 2 / 3)' in the math element of the <kineticLaw> contains an expression for the exponent of the power function and thus cannot be checked for unit validity.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 410, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(1, geq(C_Cytotoxic_T_Lymphocytes_Coverage, 1), C_Cytotoxic_T_Lymphocytes_Coverage)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 346, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r_t' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Frascoli2014.sedml506988.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Frascoli2014.sedml` is invalid.\n - Data generator `C_Cytotoxic_T_Lymphocytes_Coverage_1_task1` is invalid.\n - Variable `p1_C_Cytotoxic_T_Lymphocytes_Coverage_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C_Cytotoxic_T_Lymphocytes_Coverage']` are not defined. No namespaces are defined for the target.\n - Data generator `V_Tumor_Volume_2_task1` is invalid.\n - Variable `p1_V_Tumor_Volume_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V_Tumor_Volume']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Frascoli2014.sedml400299.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Frascoli2014.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 474, column 8:\r\n - The formula 'pow(V_Tumor_Volume, 2 / 3)' in the math element of the <kineticLaw> contains an expression for the exponent of the power function and thus cannot be checked for unit validity.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 410, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(1, geq(C_Cytotoxic_T_Lymphocytes_Coverage, 1), C_Cytotoxic_T_Lymphocytes_Coverage)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 346, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r_t' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Frascoli2014.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Frascoli2014.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 474, column 8:\r\n - The formula 'pow(V_Tumor_Volume, 2 / 3)' in the math element of the <kineticLaw> contains an expression for the exponent of the power function and thus cannot be checked for unit validity.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 410, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(1, geq(C_Cytotoxic_T_Lymphocytes_Coverage, 1), C_Cytotoxic_T_Lymphocytes_Coverage)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 346, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r_t' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Frascoli2014.sedml400299.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Frascoli2014.sedml` is invalid.\n - Data generator `C_Cytotoxic_T_Lymphocytes_Coverage_1_task1` is invalid.\n - Variable `p1_C_Cytotoxic_T_Lymphocytes_Coverage_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C_Cytotoxic_T_Lymphocytes_Coverage']` are not defined. No namespaces are defined for the target.\n - Data generator `V_Tumor_Volume_2_task1` is invalid.\n - Variable `p1_V_Tumor_Volume_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V_Tumor_Volume']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000788](https://www.ebi.ac.uk/biomodels/BIOMD0000000788)<br/><sup>Schropp2019 - Target-Mediated </sup></summary><sup>Schropp2019 - Target-Mediated Drug Disposition Model for Bispecific Antibodies</sup></details>|<details><summary>pass</summary>[Schropp2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000788#Files)</details>|FAIL|<details><summary>pass</summary>[Schropp2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000788#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Schropp2019.sedml504664.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Schropp2019.xml` has warnings.\r\n - 18 warnings of type SBML unit consistency (10541). The following is the first warning at line 1306, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -9), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Disassociation_RC_A') are second (exponent = -1, multiplier = 1, scale = -3), mole (exponent = 1, multiplier = 1, scale = -9).\r\n - 1 warning of type Modeling practice (80701). The following is the first warning at line 811, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'F' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Schropp2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Schropp2019.xml` has warnings.\r\n - 18 warnings of type SBML unit consistency (10541). The following is the first warning at line 1306, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -9), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Disassociation_RC_A') are second (exponent = -1, multiplier = 1, scale = -3), mole (exponent = 1, multiplier = 1, scale = -9).\r\n - 1 warning of type Modeling practice (80701). The following is the first warning at line 811, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'F' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Schropp2019.sedml504664.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Schropp2019.sedml` is invalid.\n - Data generator `RC_AB_1_task1` is invalid.\n - Variable `p1_RC_AB_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='RC_AB']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Schropp2019.sedml844080.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Schropp2019.xml` has warnings.\r\n - 18 warnings of type SBML unit consistency (10541). The following is the first warning at line 1306, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -9), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Disassociation_RC_A') are second (exponent = -1, multiplier = 1, scale = -3), mole (exponent = 1, multiplier = 1, scale = -9).\r\n - 1 warning of type Modeling practice (80701). The following is the first warning at line 811, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'F' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Schropp2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Schropp2019.xml` has warnings.\r\n - 18 warnings of type SBML unit consistency (10541). The following is the first warning at line 1306, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -9), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Disassociation_RC_A') are second (exponent = -1, multiplier = 1, scale = -3), mole (exponent = 1, multiplier = 1, scale = -9).\r\n - 1 warning of type Modeling practice (80701). The following is the first warning at line 811, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'F' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Schropp2019.sedml844080.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Schropp2019.sedml` is invalid.\n - Data generator `RC_AB_1_task1` is invalid.\n - Variable `p1_RC_AB_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='RC_AB']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000789](https://www.ebi.ac.uk/biomodels/BIOMD0000000789)<br/><sup>Jenner2018 - treatment of onco</sup></summary><sup>Jenner2018 - treatment of oncolytic virus</sup></details>|<details><summary>pass</summary>[Jenner2018.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000789#Files)</details>|FAIL|<details><summary>pass</summary>[Jenner2018.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000789#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Jenner2018.sedml78957.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Jenner2018.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 839, column 8:\r\n - The formula 'log(L / S)' in the math element of the <kineticLaw> uses a function which can only act on dimensionless variables.\r\n - 2 warnings of type SBML unit consistency (10522). The following is the first warning at line 690, column 6:\r\n - When the 'variable' in an <initialAssignment> refers to a <species>, the units of the <initialAssignment>'s <math> expression are expected to be consistent with the units of that species' quantity.\r\n Reference: L3V1 Section 4.8\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), dimensionless (exponent = -0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <initialAssignment> with symbol 'V' are dimensionless (exponent = 0, multiplier = 1, scale = 0).\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 971, column 14:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <event> <trigger> expression 'geq(time, 2)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 2 warnings of type SBML unit consistency (10562). The following is the first warning at line 981, column 10:\r\n - When the 'variable' in an <eventAssignment> refers to a <species>, the units of the <eventAssignment>'s <math> expression are expected to be consistent with the units of the species' quantity.\r\n Reference: L3V1 Section 4.12.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), dimensionless (exponent = -0, multiplier = 1, scale = 0) but the units returned by the <math> expression from the <eventAssignment> with variable 'V' are dimensionless (exponent = 0, multiplier = 1, scale = 0).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 64.009846152846\r\n - Number of steps: 6501\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Jenner2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Jenner2018.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 839, column 8:\r\n - The formula 'log(L / S)' in the math element of the <kineticLaw> uses a function which can only act on dimensionless variables.\r\n - 2 warnings of type SBML unit consistency (10522). The following is the first warning at line 690, column 6:\r\n - When the 'variable' in an <initialAssignment> refers to a <species>, the units of the <initialAssignment>'s <math> expression are expected to be consistent with the units of that species' quantity.\r\n Reference: L3V1 Section 4.8\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), dimensionless (exponent = -0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <initialAssignment> with symbol 'V' are dimensionless (exponent = 0, multiplier = 1, scale = 0).\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 971, column 14:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <event> <trigger> expression 'geq(time, 2)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 2 warnings of type SBML unit consistency (10562). The following is the first warning at line 981, column 10:\r\n - When the 'variable' in an <eventAssignment> refers to a <species>, the units of the <eventAssignment>'s <math> expression are expected to be consistent with the units of the species' quantity.\r\n Reference: L3V1 Section 4.12.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), dimensionless (exponent = -0, multiplier = 1, scale = 0) but the units returned by the <math> expression from the <eventAssignment> with variable 'V' are dimensionless (exponent = 0, multiplier = 1, scale = 0).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 64.009846152846\r\n - Number of steps: 6501\x1b[0m\r\n\r\n`/root/in/Jenner2018.sedml78957.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Jenner2018.sedml` is invalid.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Jenner2018.sedml944438.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Jenner2018.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 839, column 8:\r\n - The formula 'log(L / S)' in the math element of the <kineticLaw> uses a function which can only act on dimensionless variables.\r\n - 2 warnings of type SBML unit consistency (10522). The following is the first warning at line 690, column 6:\r\n - When the 'variable' in an <initialAssignment> refers to a <species>, the units of the <initialAssignment>'s <math> expression are expected to be consistent with the units of that species' quantity.\r\n Reference: L3V1 Section 4.8\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), dimensionless (exponent = -0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <initialAssignment> with symbol 'V' are dimensionless (exponent = 0, multiplier = 1, scale = 0).\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 971, column 14:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <event> <trigger> expression 'geq(time, 2)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 2 warnings of type SBML unit consistency (10562). The following is the first warning at line 981, column 10:\r\n - When the 'variable' in an <eventAssignment> refers to a <species>, the units of the <eventAssignment>'s <math> expression are expected to be consistent with the units of the species' quantity.\r\n Reference: L3V1 Section 4.12.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), dimensionless (exponent = -0, multiplier = 1, scale = 0) but the units returned by the <math> expression from the <eventAssignment> with variable 'V' are dimensionless (exponent = 0, multiplier = 1, scale = 0).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 64.009846152846\r\n - Number of steps: 6501\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Jenner2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Jenner2018.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 839, column 8:\r\n - The formula 'log(L / S)' in the math element of the <kineticLaw> uses a function which can only act on dimensionless variables.\r\n - 2 warnings of type SBML unit consistency (10522). The following is the first warning at line 690, column 6:\r\n - When the 'variable' in an <initialAssignment> refers to a <species>, the units of the <initialAssignment>'s <math> expression are expected to be consistent with the units of that species' quantity.\r\n Reference: L3V1 Section 4.8\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), dimensionless (exponent = -0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <initialAssignment> with symbol 'V' are dimensionless (exponent = 0, multiplier = 1, scale = 0).\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 971, column 14:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <event> <trigger> expression 'geq(time, 2)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 2 warnings of type SBML unit consistency (10562). The following is the first warning at line 981, column 10:\r\n - When the 'variable' in an <eventAssignment> refers to a <species>, the units of the <eventAssignment>'s <math> expression are expected to be consistent with the units of the species' quantity.\r\n Reference: L3V1 Section 4.12.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), dimensionless (exponent = -0, multiplier = 1, scale = 0) but the units returned by the <math> expression from the <eventAssignment> with variable 'V' are dimensionless (exponent = 0, multiplier = 1, scale = 0).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 64.009846152846\r\n - Number of steps: 6501\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Jenner2018.sedml944438.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Jenner2018.sedml` is invalid.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000790](https://www.ebi.ac.uk/biomodels/BIOMD0000000790)<br/><sup>Alvarez2019 - A nonlinear math</sup></summary><sup>Alvarez2019 - A nonlinear mathematical model of cell-mediated immune response for tumor phenotypic heterogeneity</sup></details>|<details><summary>pass</summary>[Alvarez2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000790#Files)</details>|pass|<details><summary>pass</summary>[Alvarez2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000790#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Alvarez2019.sedml980346.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Alvarez2019.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 904, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth_1(a, T_1, b)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 18 warnings of type Modeling practice (80701). The following is the first warning at line 705, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Alvarez2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Alvarez2019.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 904, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth_1(a, T_1, b)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 18 warnings of type Modeling practice (80701). The following is the first warning at line 705, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Alvarez2019.sedml980346.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Alvarez2019.sedml` is invalid.\n - Data generator `T_Total_1_task1` is invalid.\n - Variable `p1_T_Total_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='T_Total']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Alvarez2019.sedml186924.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Alvarez2019.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 904, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth_1(a, T_1, b)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 18 warnings of type Modeling practice (80701). The following is the first warning at line 705, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Alvarez2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Alvarez2019.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 904, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth_1(a, T_1, b)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 18 warnings of type Modeling practice (80701). The following is the first warning at line 705, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Alvarez2019.sedml186924.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Alvarez2019.sedml` is invalid.\n - Data generator `T_Total_1_task1` is invalid.\n - Variable `p1_T_Total_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='T_Total']` are not defined. No namespaces are defined for the target.\n"```</details>|
|[BIOMD0000000791](https://www.ebi.ac.uk/biomodels/BIOMD0000000791)<br/><sup>Wilson2012 - tumor vaccine efficacy</sup>|<details><summary>pass</summary>[Wilson2012.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000791#Files)</details>|FAIL|<details><summary>pass</summary>[Wilson2012.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000791#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Wilson2012.sedml86715.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Wilson2012.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 1248, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'c2 + pow(T, 2)' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (10503). The following is the first warning at line 1263, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * d * B' in the <kineticLaw> element of the <reaction> with id 'tgfb_decay' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 1032, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '1 / c0inv' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 3 warnings of type SBML unit consistency (10541). The following is the first warning at line 1193, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_immune_induced') are second (exponent = -1, multiplier = 86400, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - 1 warning of type SBML unit consistency (10562). The following is the first warning at line 1638, column 10:\r\n - When the 'variable' in an <eventAssignment> refers to a <species>, the units of the <eventAssignment>'s <math> expression are expected to be consistent with the units of the species' quantity.\r\n Reference: L3V1 Section 4.12.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), dimensionless (exponent = -0, multiplier = 1, scale = 0) but the units returned by the <math> expression from the <eventAssignment> with variable 'V' are dimensionless (exponent = 0, multiplier = 1, scale = 0).\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Wilson2012.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Wilson2012.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 1248, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'c2 + pow(T, 2)' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (10503). The following is the first warning at line 1263, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * d * B' in the <kineticLaw> element of the <reaction> with id 'tgfb_decay' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 1032, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '1 / c0inv' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 3 warnings of type SBML unit consistency (10541). The following is the first warning at line 1193, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_immune_induced') are second (exponent = -1, multiplier = 86400, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - 1 warning of type SBML unit consistency (10562). The following is the first warning at line 1638, column 10:\r\n - When the 'variable' in an <eventAssignment> refers to a <species>, the units of the <eventAssignment>'s <math> expression are expected to be consistent with the units of the species' quantity.\r\n Reference: L3V1 Section 4.12.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), dimensionless (exponent = -0, multiplier = 1, scale = 0) but the units returned by the <math> expression from the <eventAssignment> with variable 'V' are dimensionless (exponent = 0, multiplier = 1, scale = 0).\x1b[0m\r\n\r\n`/root/in/Wilson2012.sedml86715.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Wilson2012.sedml` is invalid.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `B_1_task1` is invalid.\n - Variable `p2_B_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='B']` are not defined. No namespaces are defined for the target.\n - Data generator `E_1_task1` is invalid.\n - Variable `p3_E_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='E']` are not defined. No namespaces are defined for the target.\n - Data generator `R_1_task1` is invalid.\n - Variable `p4_R_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='R']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Wilson2012.sedml850502.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Wilson2012.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 1248, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'c2 + pow(T, 2)' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (10503). The following is the first warning at line 1263, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * d * B' in the <kineticLaw> element of the <reaction> with id 'tgfb_decay' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 1032, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '1 / c0inv' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 3 warnings of type SBML unit consistency (10541). The following is the first warning at line 1193, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_immune_induced') are second (exponent = -1, multiplier = 86400, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - 1 warning of type SBML unit consistency (10562). The following is the first warning at line 1638, column 10:\r\n - When the 'variable' in an <eventAssignment> refers to a <species>, the units of the <eventAssignment>'s <math> expression are expected to be consistent with the units of the species' quantity.\r\n Reference: L3V1 Section 4.12.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), dimensionless (exponent = -0, multiplier = 1, scale = 0) but the units returned by the <math> expression from the <eventAssignment> with variable 'V' are dimensionless (exponent = 0, multiplier = 1, scale = 0).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Wilson2012.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Wilson2012.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 1248, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'c2 + pow(T, 2)' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (10503). The following is the first warning at line 1263, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * d * B' in the <kineticLaw> element of the <reaction> with id 'tgfb_decay' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 1032, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '1 / c0inv' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 3 warnings of type SBML unit consistency (10541). The following is the first warning at line 1193, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_immune_induced') are second (exponent = -1, multiplier = 86400, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - 1 warning of type SBML unit consistency (10562). The following is the first warning at line 1638, column 10:\r\n - When the 'variable' in an <eventAssignment> refers to a <species>, the units of the <eventAssignment>'s <math> expression are expected to be consistent with the units of the species' quantity.\r\n Reference: L3V1 Section 4.12.4\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), dimensionless (exponent = -0, multiplier = 1, scale = 0) but the units returned by the <math> expression from the <eventAssignment> with variable 'V' are dimensionless (exponent = 0, multiplier = 1, scale = 0).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Wilson2012.sedml850502.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Wilson2012.sedml` is invalid.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `B_1_task1` is invalid.\n - Variable `p2_B_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='B']` are not defined. No namespaces are defined for the target.\n - Data generator `E_1_task1` is invalid.\n - Variable `p3_E_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='E']` are not defined. No namespaces are defined for the target.\n - Data generator `R_1_task1` is invalid.\n - Variable `p4_R_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='R']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000792](https://www.ebi.ac.uk/biomodels/BIOMD0000000792)<br/><sup>Hu2019 - Modeling Pancreatic C</sup></summary><sup>Hu2019 - Modeling Pancreatic Cancer Dynamics with Immunotherapy</sup></details>|<details><summary>pass</summary>[Hu2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000792#Files)</details>|pass|<details><summary>pass</summary>[Hu2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000792#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Hu2019.sedml147915.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Hu2019.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 717, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_PCC_Tumor_Growth(r_1, beta_1, y_PSC, x_PCC, b_1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 28 warnings of type Modeling practice (80701). The following is the first warning at line 657, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r_1' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Hu2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Hu2019.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 717, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_PCC_Tumor_Growth(r_1, beta_1, y_PSC, x_PCC, b_1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 28 warnings of type Modeling practice (80701). The following is the first warning at line 657, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r_1' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Hu2019.sedml147915.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Hu2019.sedml` is invalid.\n - Data generator `x_PCC_1_task1` is invalid.\n - Variable `p1_x_PCC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x_PCC']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Hu2019.sedml898594.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Hu2019.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 717, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_PCC_Tumor_Growth(r_1, beta_1, y_PSC, x_PCC, b_1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 28 warnings of type Modeling practice (80701). The following is the first warning at line 657, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r_1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Hu2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Hu2019.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 717, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_PCC_Tumor_Growth(r_1, beta_1, y_PSC, x_PCC, b_1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 28 warnings of type Modeling practice (80701). The following is the first warning at line 657, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r_1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Hu2019.sedml898594.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Hu2019.sedml` is invalid.\n - Data generator `x_PCC_1_task1` is invalid.\n - Variable `p1_x_PCC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x_PCC']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000793](https://www.ebi.ac.uk/biomodels/BIOMD0000000793)<br/><sup>Chen2011/1 - bone marrow invas</sup></summary><sup>Chen2011/1 - bone marrow invasion absolute model</sup></details>|<details><summary>pass</summary>[Chen2011:1.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000793#Files)</details>|FAIL|<details><summary>MultipleSEDMLs</summary>['Chen2011:1swapped.sedml', 'Chen2011:1.sedml']</details>|NA|NA|NA|
|<details><summary>[BIOMD0000000794](https://www.ebi.ac.uk/biomodels/BIOMD0000000794)<br/><sup>Benary2019 - Controlling NFKB </sup></summary><sup>Benary2019 - Controlling NFKB dynamics by B-TrCP</sup></details>|<details><summary>FAIL</summary>[Benary2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000794#Files)</details>|FAIL|<details><summary>pass</summary>[Benary2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000794#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Benary2019.sedml489356.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/Benary2019.sedml489356.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Benary2019.sedml` is invalid.\n - Model `model` is invalid.\n - The model file `Benary2019.xml` is invalid.\n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout' cannot be used in this document.\n \n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout_L2' cannot be used in this document.\n \n - General SBML conformance (10102) at line 3, column 2: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L2V4 Section 4.1\n Element 'listOfLayouts' is not part of the definition of SBML Level 2 Version 4.\n \n - Data generator `NFKB_nuc_1_task1` is invalid.\n - Variable `p1_NFKB_nuc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='NFKB_nuc']` are not defined. No namespaces are defined for the target.\n - Data generator `NFKB_nuc_actual_1_task1` is invalid.\n - Variable `p2_NFKB_nuc_actual_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='NFKB_nuc_actual']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Benary2019.sedml487764.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Benary2019.sedml487764.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Benary2019.sedml` is invalid.\n - Model `model` is invalid.\n - The model file `Benary2019.xml` is invalid.\n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout' cannot be used in this document.\n \n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout_L2' cannot be used in this document.\n \n - General SBML conformance (10102) at line 3, column 2: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L2V4 Section 4.1\n Element 'listOfLayouts' is not part of the definition of SBML Level 2 Version 4.\n \n - Data generator `NFKB_nuc_1_task1` is invalid.\n - Variable `p1_NFKB_nuc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='NFKB_nuc']` are not defined. No namespaces are defined for the target.\n - Data generator `NFKB_nuc_actual_1_task1` is invalid.\n - Variable `p2_NFKB_nuc_actual_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='NFKB_nuc_actual']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000795](https://www.ebi.ac.uk/biomodels/BIOMD0000000795)<br/><sup>Chen2011/2 - bone marrow invas</sup></summary><sup>Chen2011/2 - bone marrow invasion relative model</sup></details>|<details><summary>pass</summary>[Chen2011:2.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000795#Files)</details>|FAIL|<details><summary>pass</summary>[Chen2011:2.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000795#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Chen2011:2.sedml558254.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Chen2011:2.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10503). The following is the first warning at line 747, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'bone_marrow * ti(b2, T, H, v)' in the <kineticLaw> element of the <reaction> with id 'Tumor_inhibition' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 582, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'bone_marrow * hg(b2, H, v, T)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 4 warnings of type SBML unit consistency (10541). The following is the first warning at line 634, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'HSC_death') are mole (exponent = 1, multiplier = 1, scale = -3).\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Chen2011:2.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Chen2011:2.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10503). The following is the first warning at line 747, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'bone_marrow * ti(b2, T, H, v)' in the <kineticLaw> element of the <reaction> with id 'Tumor_inhibition' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 582, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'bone_marrow * hg(b2, H, v, T)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 4 warnings of type SBML unit consistency (10541). The following is the first warning at line 634, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'HSC_death') are mole (exponent = 1, multiplier = 1, scale = -3).\x1b[0m\r\n\r\n`/root/in/Chen2011:2.sedml558254.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Chen2011:2.sedml` is invalid.\n - Data generator `H_1_task1` is invalid.\n - Variable `p1_H_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H']` are not defined. No namespaces are defined for the target.\n - Data generator `T_2_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Chen2011:2.sedml725235.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Chen2011:2.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10503). The following is the first warning at line 747, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'bone_marrow * ti(b2, T, H, v)' in the <kineticLaw> element of the <reaction> with id 'Tumor_inhibition' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 582, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'bone_marrow * hg(b2, H, v, T)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 4 warnings of type SBML unit consistency (10541). The following is the first warning at line 634, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'HSC_death') are mole (exponent = 1, multiplier = 1, scale = -3).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Chen2011:2.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Chen2011:2.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10503). The following is the first warning at line 747, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'bone_marrow * ti(b2, T, H, v)' in the <kineticLaw> element of the <reaction> with id 'Tumor_inhibition' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 582, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'bone_marrow * hg(b2, H, v, T)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 4 warnings of type SBML unit consistency (10541). The following is the first warning at line 634, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'HSC_death') are mole (exponent = 1, multiplier = 1, scale = -3).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Chen2011:2.sedml725235.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Chen2011:2.sedml` is invalid.\n - Data generator `H_1_task1` is invalid.\n - Variable `p1_H_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H']` are not defined. No namespaces are defined for the target.\n - Data generator `T_2_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000796](https://www.ebi.ac.uk/biomodels/BIOMD0000000796)<br/><sup>Yang2012 - cancer growth with </sup></summary><sup>Yang2012 - cancer growth with angiogenesis</sup></details>|<details><summary>pass</summary>[Yang2012.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000796#Files)</details>|FAIL|<details><summary>pass</summary>[Yang2012.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000796#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Yang2012.sedml431505.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Yang2012.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10503). The following is the first warning at line 1475, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * deplete(b1, T, C)' in the <kineticLaw> element of the <reaction> with id 'normal_cell_depletion' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 5 warnings of type SBML unit consistency (10513). The following is the first warning at line 1338, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'P_plot' are mole (exponent = 1, multiplier = 0.001, scale = 0), litre (exponent = -1, multiplier = 0.001, scale = 0).\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 1413, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tme * logistic(a1, C, k1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type SBML unit consistency (10541). The following is the first warning at line 1464, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'normal_cell_death') are second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 1, multiplier = 1, scale = -3).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 630.000000168\r\n - Number of steps: 583\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Yang2012.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Yang2012.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10503). The following is the first warning at line 1475, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * deplete(b1, T, C)' in the <kineticLaw> element of the <reaction> with id 'normal_cell_depletion' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 5 warnings of type SBML unit consistency (10513). The following is the first warning at line 1338, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'P_plot' are mole (exponent = 1, multiplier = 0.001, scale = 0), litre (exponent = -1, multiplier = 0.001, scale = 0).\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 1413, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tme * logistic(a1, C, k1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type SBML unit consistency (10541). The following is the first warning at line 1464, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'normal_cell_death') are second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 1, multiplier = 1, scale = -3).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 630.000000168\r\n - Number of steps: 583\x1b[0m\r\n\r\n`/root/in/Yang2012.sedml431505.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Yang2012.sedml` is invalid.\n - Data generator `A_plot_1_task1` is invalid.\n - Variable `p1_A_plot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='A_plot']` are not defined. No namespaces are defined for the target.\n - Data generator `C_plot_2_task1` is invalid.\n - Variable `p1_C_plot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='C_plot']` are not defined. No namespaces are defined for the target.\n - Data generator `E_plot_3_task1` is invalid.\n - Variable `p1_E_plot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='E_plot']` are not defined. No namespaces are defined for the target.\n - Data generator `P_plot_4_task1` is invalid.\n - Variable `p1_P_plot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='P_plot']` are not defined. No namespaces are defined for the target.\n - Data generator `T_plot_5_task1` is invalid.\n - Variable `p1_T_plot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='T_plot']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Yang2012.sedml657460.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Yang2012.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10503). The following is the first warning at line 1475, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * deplete(b1, T, C)' in the <kineticLaw> element of the <reaction> with id 'normal_cell_depletion' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 5 warnings of type SBML unit consistency (10513). The following is the first warning at line 1338, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'P_plot' are mole (exponent = 1, multiplier = 0.001, scale = 0), litre (exponent = -1, multiplier = 0.001, scale = 0).\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 1413, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tme * logistic(a1, C, k1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type SBML unit consistency (10541). The following is the first warning at line 1464, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'normal_cell_death') are second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 1, multiplier = 1, scale = -3).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 630.000000168\r\n - Number of steps: 583\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Yang2012.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Yang2012.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10503). The following is the first warning at line 1475, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * deplete(b1, T, C)' in the <kineticLaw> element of the <reaction> with id 'normal_cell_depletion' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 5 warnings of type SBML unit consistency (10513). The following is the first warning at line 1338, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'P_plot' are mole (exponent = 1, multiplier = 0.001, scale = 0), litre (exponent = -1, multiplier = 0.001, scale = 0).\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 1413, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tme * logistic(a1, C, k1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type SBML unit consistency (10541). The following is the first warning at line 1464, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'normal_cell_death') are second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 1, multiplier = 1, scale = -3).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 630.000000168\r\n - Number of steps: 583\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Yang2012.sedml657460.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Yang2012.sedml` is invalid.\n - Data generator `A_plot_1_task1` is invalid.\n - Variable `p1_A_plot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='A_plot']` are not defined. No namespaces are defined for the target.\n - Data generator `C_plot_2_task1` is invalid.\n - Variable `p1_C_plot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='C_plot']` are not defined. No namespaces are defined for the target.\n - Data generator `E_plot_3_task1` is invalid.\n - Variable `p1_E_plot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='E_plot']` are not defined. No namespaces are defined for the target.\n - Data generator `P_plot_4_task1` is invalid.\n - Variable `p1_P_plot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='P_plot']` are not defined. No namespaces are defined for the target.\n - Data generator `T_plot_5_task1` is invalid.\n - Variable `p1_T_plot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='T_plot']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000797](https://www.ebi.ac.uk/biomodels/BIOMD0000000797)<br/><sup>Hu2018 - Dynamics of tumor-CD4</sup></summary><sup>Hu2018 - Dynamics of tumor-CD4+-cytokine-host cells interactions with treatments</sup></details>|<details><summary>pass</summary>[Hu2018.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000797#Files)</details>|pass|<details><summary>pass</summary>[Hu2018.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000797#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Hu2018.sedml758377.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Hu2018.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 694, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'x_Tumor_Cells / 100000000' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 19 warnings of type Modeling practice (80701). The following is the first warning at line 589, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r_1' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Hu2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Hu2018.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 694, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'x_Tumor_Cells / 100000000' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 19 warnings of type Modeling practice (80701). The following is the first warning at line 589, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r_1' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Hu2018.sedml758377.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Hu2018.sedml` is invalid.\n - Data generator `Host_Cells_1e8_1_task1` is invalid.\n - Variable `p1_Host_Cells_1e8_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Host_Cells_1e8']` are not defined. No namespaces are defined for the target.\n - Data generator `Tumor_Cells_1e8_2_task1` is invalid.\n - Variable `p1_Tumor_Cells_1e8_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Tumor_Cells_1e8']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Hu2018.sedml179135.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Hu2018.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 694, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'x_Tumor_Cells / 100000000' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 19 warnings of type Modeling practice (80701). The following is the first warning at line 589, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r_1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Hu2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Hu2018.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 694, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'x_Tumor_Cells / 100000000' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 19 warnings of type Modeling practice (80701). The following is the first warning at line 589, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r_1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Hu2018.sedml179135.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Hu2018.sedml` is invalid.\n - Data generator `Host_Cells_1e8_1_task1` is invalid.\n - Variable `p1_Host_Cells_1e8_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Host_Cells_1e8']` are not defined. No namespaces are defined for the target.\n - Data generator `Tumor_Cells_1e8_2_task1` is invalid.\n - Variable `p1_Tumor_Cells_1e8_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Tumor_Cells_1e8']` are not defined. No namespaces are defined for the target.\n"```</details>|
|[BIOMD0000000798](https://www.ebi.ac.uk/biomodels/BIOMD0000000798)<br/><sup>Sharp2019 - AML</sup>|<details><summary>pass</summary>[Sharp2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000798#Files)</details>|FAIL|<details><summary>pass</summary>[Sharp2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000798#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Sharp2019.sedml858870.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Sharp2019.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 1229, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'y + L' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 2 warnings of type SBML unit consistency (10513). The following is the first warning at line 836, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'Z2' are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3).\r\n - 9 warnings of type SBML unit consistency (10541). The following is the first warning at line 888, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'HSC_growth') are mole (exponent = 1, multiplier = 0.001, scale = 0).\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Sharp2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Sharp2019.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 1229, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'y + L' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 2 warnings of type SBML unit consistency (10513). The following is the first warning at line 836, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'Z2' are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3).\r\n - 9 warnings of type SBML unit consistency (10541). The following is the first warning at line 888, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'HSC_growth') are mole (exponent = 1, multiplier = 0.001, scale = 0).\x1b[0m\r\n\r\n`/root/in/Sharp2019.sedml858870.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Sharp2019.sedml` is invalid.\n - Data generator `A_1_task1` is invalid.\n - Variable `p1_A_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='A']` are not defined. No namespaces are defined for the target.\n - Data generator `D_2_task1` is invalid.\n - Variable `p1_D_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='D']` are not defined. No namespaces are defined for the target.\n - Data generator `L_3_task1` is invalid.\n - Variable `p1_L_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L']` are not defined. No namespaces are defined for the target.\n - Data generator `S_4_task1` is invalid.\n - Variable `p1_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S']` are not defined. No namespaces are defined for the target.\n - Data generator `T_5_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Sharp2019.sedml155821.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Sharp2019.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 1229, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'y + L' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 2 warnings of type SBML unit consistency (10513). The following is the first warning at line 836, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'Z2' are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3).\r\n - 9 warnings of type SBML unit consistency (10541). The following is the first warning at line 888, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'HSC_growth') are mole (exponent = 1, multiplier = 0.001, scale = 0).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Sharp2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Sharp2019.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 1229, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'y + L' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 2 warnings of type SBML unit consistency (10513). The following is the first warning at line 836, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'Z2' are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3).\r\n - 9 warnings of type SBML unit consistency (10541). The following is the first warning at line 888, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'HSC_growth') are mole (exponent = 1, multiplier = 0.001, scale = 0).\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Sharp2019.sedml155821.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Sharp2019.sedml` is invalid.\n - Data generator `A_1_task1` is invalid.\n - Variable `p1_A_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='A']` are not defined. No namespaces are defined for the target.\n - Data generator `D_2_task1` is invalid.\n - Variable `p1_D_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='D']` are not defined. No namespaces are defined for the target.\n - Data generator `L_3_task1` is invalid.\n - Variable `p1_L_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L']` are not defined. No namespaces are defined for the target.\n - Data generator `S_4_task1` is invalid.\n - Variable `p1_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S']` are not defined. No namespaces are defined for the target.\n - Data generator `T_5_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000799](https://www.ebi.ac.uk/biomodels/BIOMD0000000799)<br/><sup>Cucuianu2010 - A hypothetical-</sup></summary><sup>Cucuianu2010 - A hypothetical-mathematical model of acute myeloid leukaemia pathogenesis</sup></details>|<details><summary>pass</summary>[Cucuianu2010.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000799#Files)</details>|pass|<details><summary>FAIL</summary>[Cucuianu2010.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000799#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Cucuianu2010.sedml461989.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/Cucuianu2010.sedml461989.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Cucuianu2010.sedml` is invalid.\n - Model `model2` is invalid.\n - The model file `model2.xml` is invalid.\n - `/tmp/tmp7zrlcl0j/./model2.xml` is not a file.\n - Data generator `x_Normal_Hematopoietic_Stem_Cell_1_task1` is invalid.\n - Variable `p1_x_Normal_Hematopoietic_Stem_Cell_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x_Normal_Hematopoietic_Stem_Cell']` are not defined. No namespaces are defined for the target.\n - Data generator `y_Leukemic_Cell_2_task1` is invalid.\n - Variable `p1_y_Leukemic_Cell_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='y_Leukemic_Cell']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Cucuianu2010.sedml137757.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Cucuianu2010.sedml137757.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Cucuianu2010.sedml` is invalid.\n - Model `model2` is invalid.\n - The model file `model2.xml` is invalid.\n - `/tmp/tmp44bo9yby/./model2.xml` is not a file.\n - Data generator `x_Normal_Hematopoietic_Stem_Cell_1_task1` is invalid.\n - Variable `p1_x_Normal_Hematopoietic_Stem_Cell_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x_Normal_Hematopoietic_Stem_Cell']` are not defined. No namespaces are defined for the target.\n - Data generator `y_Leukemic_Cell_2_task1` is invalid.\n - Variable `p1_y_Leukemic_Cell_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='y_Leukemic_Cell']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000800](https://www.ebi.ac.uk/biomodels/BIOMD0000000800)<br/><sup>Precup2012 - Mathematical mode</sup></summary><sup>Precup2012 - Mathematical modeling of cell dynamics after allogeneic bone marrow transplantation</sup></details>|<details><summary>pass</summary>[Precup2012.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000800#Files)</details>|pass|<details><summary>pass</summary>[Precup2012.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000800#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Precup2012.sedml273614.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Precup2012.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 551, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Normal_Cells_Growth(x, a, b, y, z, epsilon, g)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 507, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Precup2012.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Precup2012.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 551, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Normal_Cells_Growth(x, a, b, y, z, epsilon, g)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 507, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Precup2012.sedml273614.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Precup2012.sedml` is invalid.\n - Data generator `x_1_task1` is invalid.\n - Variable `p1_x_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x']` are not defined. No namespaces are defined for the target.\n - Data generator `y_2_task1` is invalid.\n - Variable `p1_y_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='y']` are not defined. No namespaces are defined for the target.\n - Data generator `z_3_task1` is invalid.\n - Variable `p1_z_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='z']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Precup2012.sedml148815.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Precup2012.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 551, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Normal_Cells_Growth(x, a, b, y, z, epsilon, g)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 507, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Precup2012.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Precup2012.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 551, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Normal_Cells_Growth(x, a, b, y, z, epsilon, g)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 507, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Precup2012.sedml148815.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Precup2012.sedml` is invalid.\n - Data generator `x_1_task1` is invalid.\n - Variable `p1_x_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x']` are not defined. No namespaces are defined for the target.\n - Data generator `y_2_task1` is invalid.\n - Variable `p1_y_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='y']` are not defined. No namespaces are defined for the target.\n - Data generator `z_3_task1` is invalid.\n - Variable `p1_z_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='z']` are not defined. No namespaces are defined for the target.\n"```</details>|
|[BIOMD0000000801](https://www.ebi.ac.uk/biomodels/BIOMD0000000801)<br/><sup>Sturrock2015 - glioma growth</sup>|<details><summary>pass</summary>[Sturrock2015.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000801#Files)</details>|FAIL|<details><summary>pass</summary>[Sturrock2015.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000801#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Sturrock2015.sedml646008.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Sturrock2015.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 1488, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'v + I' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (10503). The following is the first warning at line 1310, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * _1mod(dti, I, T)' in the <kineticLaw> element of the <reaction> with id 'tumor_immune' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 1093, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(piecewise(2, lt(time, 4745), 1), lt(1095, time), 1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type SBML unit consistency (10541). The following is the first warning at line 1299, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_death') are second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 1, multiplier = 1, scale = -3).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 3285.000000045\r\n - Number of steps: 333\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Sturrock2015.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Sturrock2015.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 1488, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'v + I' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (10503). The following is the first warning at line 1310, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * _1mod(dti, I, T)' in the <kineticLaw> element of the <reaction> with id 'tumor_immune' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 1093, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(piecewise(2, lt(time, 4745), 1), lt(1095, time), 1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type SBML unit consistency (10541). The following is the first warning at line 1299, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_death') are second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 1, multiplier = 1, scale = -3).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 3285.000000045\r\n - Number of steps: 333\x1b[0m\r\n\r\n`/root/in/Sturrock2015.sedml646008.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Sturrock2015.sedml` is invalid.\n - Data generator `B_1_task1` is invalid.\n - Variable `p1_B_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='B']` are not defined. No namespaces are defined for the target.\n - Data generator `I_2_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `S_3_task1` is invalid.\n - Variable `p1_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S']` are not defined. No namespaces are defined for the target.\n - Data generator `T_4_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Sturrock2015.sedml100116.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Sturrock2015.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 1488, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'v + I' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (10503). The following is the first warning at line 1310, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * _1mod(dti, I, T)' in the <kineticLaw> element of the <reaction> with id 'tumor_immune' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 1093, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(piecewise(2, lt(time, 4745), 1), lt(1095, time), 1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type SBML unit consistency (10541). The following is the first warning at line 1299, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_death') are second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 1, multiplier = 1, scale = -3).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 3285.000000045\r\n - Number of steps: 333\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Sturrock2015.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Sturrock2015.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 1488, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'v + I' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (10503). The following is the first warning at line 1310, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * _1mod(dti, I, T)' in the <kineticLaw> element of the <reaction> with id 'tumor_immune' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 1093, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(piecewise(2, lt(time, 4745), 1), lt(1095, time), 1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type SBML unit consistency (10541). The following is the first warning at line 1299, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'tumor_death') are second (exponent = -1, multiplier = 86400, scale = -1), mole (exponent = 1, multiplier = 1, scale = -3).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 3285.000000045\r\n - Number of steps: 333\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Sturrock2015.sedml100116.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Sturrock2015.sedml` is invalid.\n - Data generator `B_1_task1` is invalid.\n - Variable `p1_B_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='B']` are not defined. No namespaces are defined for the target.\n - Data generator `I_2_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `S_3_task1` is invalid.\n - Variable `p1_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S']` are not defined. No namespaces are defined for the target.\n - Data generator `T_4_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|
|[BIOMD0000000802](https://www.ebi.ac.uk/biomodels/BIOMD0000000802)<br/><sup>Hoffman2018- ADCC against cancer</sup>|<details><summary>pass</summary>[Hoffman2017.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000802#Files)</details>|FAIL|<details><summary>pass</summary>[Hoffman2017.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000802#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Hoffman2017.sedml600819.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Hoffman2017.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 861, column 6:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'R + rk' in the math element of the <assignmentRule> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (10503). The following is the first warning at line 1016, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * a2 * R' in the <kineticLaw> element of the <reaction> with id 'complex_dissociation' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 1 warning of type SBML unit consistency (10513). The following is the first warning at line 861, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'f' are mole (exponent = 1, multiplier = 0.001, scale = 0), litre (exponent = -1, multiplier = 0.001, scale = 0).\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 915, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tme * aa(a1, R, A, S)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type SBML unit consistency (10541). The following is the first warning at line 952, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'antibody_dissociation') are litre (exponent = -1, multiplier = 0.001, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 1e-06\r\n - Output end time: 10.0\r\n - Number of steps: 9999999\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Hoffman2017.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Hoffman2017.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 861, column 6:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'R + rk' in the math element of the <assignmentRule> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (10503). The following is the first warning at line 1016, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * a2 * R' in the <kineticLaw> element of the <reaction> with id 'complex_dissociation' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 1 warning of type SBML unit consistency (10513). The following is the first warning at line 861, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'f' are mole (exponent = 1, multiplier = 0.001, scale = 0), litre (exponent = -1, multiplier = 0.001, scale = 0).\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 915, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tme * aa(a1, R, A, S)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type SBML unit consistency (10541). The following is the first warning at line 952, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'antibody_dissociation') are litre (exponent = -1, multiplier = 0.001, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 1e-06\r\n - Output end time: 10.0\r\n - Number of steps: 9999999\x1b[0m\r\n\r\n`/root/in/Hoffman2017.sedml600819.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Hoffman2017.sedml` is invalid.\n - Data generator `C_1_task1` is invalid.\n - Variable `p1_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C']` are not defined. No namespaces are defined for the target.\n - Data generator `R_2_task1` is invalid.\n - Variable `p1_R_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='R']` are not defined. No namespaces are defined for the target.\n - Data generator `S_3_task1` is invalid.\n - Variable `p1_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Hoffman2017.sedml588261.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Hoffman2017.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 861, column 6:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'R + rk' in the math element of the <assignmentRule> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (10503). The following is the first warning at line 1016, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * a2 * R' in the <kineticLaw> element of the <reaction> with id 'complex_dissociation' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 1 warning of type SBML unit consistency (10513). The following is the first warning at line 861, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'f' are mole (exponent = 1, multiplier = 0.001, scale = 0), litre (exponent = -1, multiplier = 0.001, scale = 0).\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 915, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tme * aa(a1, R, A, S)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type SBML unit consistency (10541). The following is the first warning at line 952, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'antibody_dissociation') are litre (exponent = -1, multiplier = 0.001, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 1e-06\r\n - Output end time: 10.0\r\n - Number of steps: 9999999\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Hoffman2017.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Hoffman2017.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 861, column 6:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L3V1 Section 3.4\r\n The formula 'R + rk' in the math element of the <assignmentRule> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (10503). The following is the first warning at line 1016, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * a2 * R' in the <kineticLaw> element of the <reaction> with id 'complex_dissociation' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 1 warning of type SBML unit consistency (10513). The following is the first warning at line 861, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <parameter>, the units of the rule's right-hand side are expected to be consistent with the units declared for that parameter. \r\n Reference: L3V1 Section 4.9.3\r\n Expected units are dimensionless (exponent = 0, multiplier = 1, scale = 0) but the units returned by the <math> expression of the <assignmentRule> with variable 'f' are mole (exponent = 1, multiplier = 0.001, scale = 0), litre (exponent = -1, multiplier = 0.001, scale = 0).\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 915, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'tme * aa(a1, R, A, S)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type SBML unit consistency (10541). The following is the first warning at line 952, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'antibody_dissociation') are litre (exponent = -1, multiplier = 0.001, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 1e-06\r\n - Output end time: 10.0\r\n - Number of steps: 9999999\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Hoffman2017.sedml588261.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Hoffman2017.sedml` is invalid.\n - Data generator `C_1_task1` is invalid.\n - Variable `p1_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C']` are not defined. No namespaces are defined for the target.\n - Data generator `R_2_task1` is invalid.\n - Variable `p1_R_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='R']` are not defined. No namespaces are defined for the target.\n - Data generator `S_3_task1` is invalid.\n - Variable `p1_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000803](https://www.ebi.ac.uk/biomodels/BIOMD0000000803)<br/><sup>Park2019 - IL7 receptor signal</sup></summary><sup>Park2019 - IL7 receptor signaling in T cells</sup></details>|<details><summary>pass</summary>[Park2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000803#Files)</details>|pass|<details><summary>pass</summary>[Park2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000803#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Park2019.sedml802321.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Park2019.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 446, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * k_f_1 * IL7Ra * gamma_c' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 373, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'rho' does not have a 'units' attribute.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Park2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Park2019.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 446, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * k_f_1 * IL7Ra * gamma_c' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 373, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'rho' does not have a 'units' attribute.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n\r\n`/root/in/Park2019.sedml802321.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Park2019.sedml` is invalid.\n - Task `task3` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IL7']` are not defined. No namespaces are defined for the target.\n - Data generator `f_7_1_task1` is invalid.\n - Variable `p1_f_7_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='f_7']` are not defined. No namespaces are defined for the target.\n - Data generator `IL7_1_task1` is invalid.\n - Variable `p1_IL7_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IL7']` are not defined. No namespaces are defined for the target.\n - Data generator `f_7_1_task3` is invalid.\n - Variable `p1_f_7_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='f_7']` are not defined. No namespaces are defined for the target.\n - Data generator `IL7_1_task3` is invalid.\n - Variable `p1_IL7_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IL7']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Park2019.sedml804225.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Park2019.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 446, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * k_f_1 * IL7Ra * gamma_c' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 373, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'rho' does not have a 'units' attribute.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Park2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Park2019.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 446, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * k_f_1 * IL7Ra * gamma_c' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 373, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'rho' does not have a 'units' attribute.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Park2019.sedml804225.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Park2019.sedml` is invalid.\n - Task `task3` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IL7']` are not defined. No namespaces are defined for the target.\n - Data generator `f_7_1_task1` is invalid.\n - Variable `p1_f_7_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='f_7']` are not defined. No namespaces are defined for the target.\n - Data generator `IL7_1_task1` is invalid.\n - Variable `p1_IL7_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IL7']` are not defined. No namespaces are defined for the target.\n - Data generator `f_7_1_task3` is invalid.\n - Variable `p1_f_7_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='f_7']` are not defined. No namespaces are defined for the target.\n - Data generator `IL7_1_task3` is invalid.\n - Variable `p1_IL7_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IL7']` are not defined. No namespaces are defined for the target.\n"```</details>|
|[BIOMD0000000804](https://www.ebi.ac.uk/biomodels/BIOMD0000000804)<br/><sup>Koenders2015 - multiple myeloma</sup>|<details><summary>pass</summary>[Koenders2015.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000804#Files)</details>|FAIL|<details><summary>pass</summary>[Koenders2015.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000804#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Koenders2015.sedml330516.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Koenders2015.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10503). The following is the first warning at line 1279, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * bc * C' in the <kineticLaw> element of the <reaction> with id 'osteoclast_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 3 warnings of type SBML unit consistency (99508). The following is the first warning at line 989, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'Metabolite_1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 3 warnings of type SBML unit consistency (20702). The following is the first warning at line 989, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'Metabolite_1' does not have a 'units' attribute.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 1006, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression '1.1 * pow(3000, 2 / 3) / 200' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 4 warnings of type SBML unit consistency (10541). The following is the first warning at line 1262, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'myeloma_growth') are litre (exponent = 0.5, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 8640, scale = 0), mole (exponent = 0.5, multiplier = 1, scale = -3).\r\n - 3 warnings of type Modeling practice (80701). The following is the first warning at line 989, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Metabolite_1' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Koenders2015.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Koenders2015.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10503). The following is the first warning at line 1279, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * bc * C' in the <kineticLaw> element of the <reaction> with id 'osteoclast_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 3 warnings of type SBML unit consistency (99508). The following is the first warning at line 989, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'Metabolite_1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 3 warnings of type SBML unit consistency (20702). The following is the first warning at line 989, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'Metabolite_1' does not have a 'units' attribute.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 1006, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression '1.1 * pow(3000, 2 / 3) / 200' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 4 warnings of type SBML unit consistency (10541). The following is the first warning at line 1262, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'myeloma_growth') are litre (exponent = 0.5, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 8640, scale = 0), mole (exponent = 0.5, multiplier = 1, scale = -3).\r\n - 3 warnings of type Modeling practice (80701). The following is the first warning at line 989, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Metabolite_1' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Koenders2015.sedml330516.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Koenders2015.sedml` is invalid.\n - Data generator `B_plot_1_task1` is invalid.\n - Variable `p1_B_plot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='B_plot']` are not defined. No namespaces are defined for the target.\n - Data generator `C_plot_2_task1` is invalid.\n - Variable `p1_C_plot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='C_plot']` are not defined. No namespaces are defined for the target.\n - Data generator `T_plot_3_task1` is invalid.\n - Variable `p1_T_plot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='T_plot']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Koenders2015.sedml865919.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Koenders2015.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10503). The following is the first warning at line 1279, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * bc * C' in the <kineticLaw> element of the <reaction> with id 'osteoclast_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 3 warnings of type SBML unit consistency (99508). The following is the first warning at line 989, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'Metabolite_1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 3 warnings of type SBML unit consistency (20702). The following is the first warning at line 989, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'Metabolite_1' does not have a 'units' attribute.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 1006, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression '1.1 * pow(3000, 2 / 3) / 200' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 4 warnings of type SBML unit consistency (10541). The following is the first warning at line 1262, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'myeloma_growth') are litre (exponent = 0.5, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 8640, scale = 0), mole (exponent = 0.5, multiplier = 1, scale = -3).\r\n - 3 warnings of type Modeling practice (80701). The following is the first warning at line 989, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Metabolite_1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Koenders2015.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Koenders2015.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10503). The following is the first warning at line 1279, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * bc * C' in the <kineticLaw> element of the <reaction> with id 'osteoclast_death' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 3 warnings of type SBML unit consistency (99508). The following is the first warning at line 989, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'Metabolite_1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 3 warnings of type SBML unit consistency (20702). The following is the first warning at line 989, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'Metabolite_1' does not have a 'units' attribute.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 1006, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression '1.1 * pow(3000, 2 / 3) / 200' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 4 warnings of type SBML unit consistency (10541). The following is the first warning at line 1262, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'myeloma_growth') are litre (exponent = 0.5, multiplier = 0.001, scale = 0), second (exponent = -1, multiplier = 8640, scale = 0), mole (exponent = 0.5, multiplier = 1, scale = -3).\r\n - 3 warnings of type Modeling practice (80701). The following is the first warning at line 989, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Metabolite_1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Koenders2015.sedml865919.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Koenders2015.sedml` is invalid.\n - Data generator `B_plot_1_task1` is invalid.\n - Variable `p1_B_plot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='B_plot']` are not defined. No namespaces are defined for the target.\n - Data generator `C_plot_2_task1` is invalid.\n - Variable `p1_C_plot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='C_plot']` are not defined. No namespaces are defined for the target.\n - Data generator `T_plot_3_task1` is invalid.\n - Variable `p1_T_plot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='T_plot']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000805](https://www.ebi.ac.uk/biomodels/BIOMD0000000805)<br/><sup>Al-Husari2013 - pH and lactate</sup></summary><sup>Al-Husari2013 - pH and lactate in tumor</sup></details>|<details><summary>pass</summary>[Al-Husari2013.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000805#Files)</details>|FAIL|<details><summary>pass</summary>[Al-Husari2013.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000805#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Al-Husari2013.sedml100060.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Al-Husari2013.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10503). The following is the first warning at line 1138, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * mct(k3, p, Hi, Li, He, Le)' in the <kineticLaw> element of the <reaction> with id 'MCT' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 976, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'pow(10, -0.3)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 5 warnings of type SBML unit consistency (10541). The following is the first warning at line 1069, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'H_leak') are mole (exponent = 1, multiplier = 0.001, scale = 0).\r\n - 1 warning of type Modeling practice (81121). The following is the first warning at line 1370, column 12:\r\n - In SBML's simple symbol system, there is no separation of symbols by class of object; consequently, inside the kinetic law mathematical formula, the value of a local parameter having the same identifier as a species or compartment or other global model entity will override the global value. Modelers may wish to take precautions to avoid this happening accidentally.\r\n In this instance the local parameter with id 'v' will shadow the 'Parameter' with an identical id.\r\n - 2 warnings of type Modeling practice (80701). The following is the first warning at line 1370, column 12:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <localParameter> with the id 'v' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Al-Husari2013.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Al-Husari2013.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10503). The following is the first warning at line 1138, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * mct(k3, p, Hi, Li, He, Le)' in the <kineticLaw> element of the <reaction> with id 'MCT' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 976, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'pow(10, -0.3)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 5 warnings of type SBML unit consistency (10541). The following is the first warning at line 1069, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'H_leak') are mole (exponent = 1, multiplier = 0.001, scale = 0).\r\n - 1 warning of type Modeling practice (81121). The following is the first warning at line 1370, column 12:\r\n - In SBML's simple symbol system, there is no separation of symbols by class of object; consequently, inside the kinetic law mathematical formula, the value of a local parameter having the same identifier as a species or compartment or other global model entity will override the global value. Modelers may wish to take precautions to avoid this happening accidentally.\r\n In this instance the local parameter with id 'v' will shadow the 'Parameter' with an identical id.\r\n - 2 warnings of type Modeling practice (80701). The following is the first warning at line 1370, column 12:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <localParameter> with the id 'v' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Al-Husari2013.sedml100060.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Al-Husari2013.sedml` is invalid.\n - Data generator `pHe_1_task1` is invalid.\n - Variable `p1_pHe_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='pHe']` are not defined. No namespaces are defined for the target.\n - Data generator `pHi_2_task1` is invalid.\n - Variable `p1_pHi_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='pHi']` are not defined. No namespaces are defined for the target.\n - Data generator `Le_1_task1` is invalid.\n - Variable `p2_Le_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Le']` are not defined. No namespaces are defined for the target.\n - Data generator `Li_2_task1` is invalid.\n - Variable `p2_Li_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Li']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Al-Husari2013.sedml189834.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Al-Husari2013.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10503). The following is the first warning at line 1138, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * mct(k3, p, Hi, Li, He, Le)' in the <kineticLaw> element of the <reaction> with id 'MCT' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 976, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'pow(10, -0.3)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 5 warnings of type SBML unit consistency (10541). The following is the first warning at line 1069, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'H_leak') are mole (exponent = 1, multiplier = 0.001, scale = 0).\r\n - 1 warning of type Modeling practice (81121). The following is the first warning at line 1370, column 12:\r\n - In SBML's simple symbol system, there is no separation of symbols by class of object; consequently, inside the kinetic law mathematical formula, the value of a local parameter having the same identifier as a species or compartment or other global model entity will override the global value. Modelers may wish to take precautions to avoid this happening accidentally.\r\n In this instance the local parameter with id 'v' will shadow the 'Parameter' with an identical id.\r\n - 2 warnings of type Modeling practice (80701). The following is the first warning at line 1370, column 12:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <localParameter> with the id 'v' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Al-Husari2013.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Al-Husari2013.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10503). The following is the first warning at line 1138, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'tme * mct(k3, p, Hi, Li, He, Le)' in the <kineticLaw> element of the <reaction> with id 'MCT' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 976, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'pow(10, -0.3)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 5 warnings of type SBML unit consistency (10541). The following is the first warning at line 1069, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'H_leak') are mole (exponent = 1, multiplier = 0.001, scale = 0).\r\n - 1 warning of type Modeling practice (81121). The following is the first warning at line 1370, column 12:\r\n - In SBML's simple symbol system, there is no separation of symbols by class of object; consequently, inside the kinetic law mathematical formula, the value of a local parameter having the same identifier as a species or compartment or other global model entity will override the global value. Modelers may wish to take precautions to avoid this happening accidentally.\r\n In this instance the local parameter with id 'v' will shadow the 'Parameter' with an identical id.\r\n - 2 warnings of type Modeling practice (80701). The following is the first warning at line 1370, column 12:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <localParameter> with the id 'v' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Al-Husari2013.sedml189834.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Al-Husari2013.sedml` is invalid.\n - Data generator `pHe_1_task1` is invalid.\n - Variable `p1_pHe_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='pHe']` are not defined. No namespaces are defined for the target.\n - Data generator `pHi_2_task1` is invalid.\n - Variable `p1_pHi_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='pHi']` are not defined. No namespaces are defined for the target.\n - Data generator `Le_1_task1` is invalid.\n - Variable `p2_Le_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Le']` are not defined. No namespaces are defined for the target.\n - Data generator `Li_2_task1` is invalid.\n - Variable `p2_Li_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Li']` are not defined. No namespaces are defined for the target.\n"```</details>|
|[BIOMD0000000806](https://www.ebi.ac.uk/biomodels/BIOMD0000000806)<br/><sup>Eftimie2019-Macrophages Plasticity</sup>|<details><summary>pass</summary>[model.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000806#Files)</details>|FAIL|<details><summary>MultipleSEDMLs</summary>['Macrophages Plasticity a.sedml', 'Macrophages Plasticity b.sedml']</details>|NA|NA|NA|
|<details><summary>[BIOMD0000000807](https://www.ebi.ac.uk/biomodels/BIOMD0000000807)<br/><sup>Fassoni2019 - Oncogenesis enco</sup></summary><sup>Fassoni2019 - Oncogenesis encompassing mutations and genetic instability</sup></details>|<details><summary>pass</summary>[model.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000807#Files)</details>|FAIL|<details><summary>pass</summary>[Fassoni2019 - 5A.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000807#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Fassoni2019 - 5A.sedml688066.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 2: b'usage: biosimulators-tellurium [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-tellurium: error: unrecognized arguments: - 5A.sedml688066.omex\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Fassoni2019 - 5A.sedml687367.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 2: b'usage: biosimulators-copasi [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-copasi: error: unrecognized arguments: - 5A.sedml687367.omex\n'```</details>|
|<details><summary>[BIOMD0000000808](https://www.ebi.ac.uk/biomodels/BIOMD0000000808)<br/><sup>Kronik2008 - Improving allorea</sup></summary><sup>Kronik2008 - Improving alloreactive CTL immunotherapy for malignant gliomas using a simulation model of their interactive dynamics</sup></details>|<details><summary>pass</summary>[Kronik2008.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000808#Files)</details>|pass|<details><summary>pass</summary>[Kronik2008.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000808#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Kronik2008.sedml44423.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Kronik2008.xml` has warnings.\r\n - 25 warnings of type SBML unit consistency (99505). The following is the first warning at line 873, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'S_dose' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 40 warnings of type Modeling practice (80701). The following is the first warning at line 587, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Kronik2008.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Kronik2008.xml` has warnings.\r\n - 25 warnings of type SBML unit consistency (99505). The following is the first warning at line 873, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'S_dose' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 40 warnings of type Modeling practice (80701). The following is the first warning at line 587, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Kronik2008.sedml44423.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Kronik2008.sedml` is invalid.\n - Data generator `C_e7_1_task1` is invalid.\n - Variable `p1_C_e7_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='C_e7']` are not defined. No namespaces are defined for the target.\n - Data generator `T_e9_2_task1` is invalid.\n - Variable `p1_T_e9_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='T_e9']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Kronik2008.sedml255895.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Kronik2008.xml` has warnings.\r\n - 25 warnings of type SBML unit consistency (99505). The following is the first warning at line 873, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'S_dose' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 40 warnings of type Modeling practice (80701). The following is the first warning at line 587, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Kronik2008.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Kronik2008.xml` has warnings.\r\n - 25 warnings of type SBML unit consistency (99505). The following is the first warning at line 873, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'S_dose' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 40 warnings of type Modeling practice (80701). The following is the first warning at line 587, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Kronik2008.sedml255895.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Kronik2008.sedml` is invalid.\n - Data generator `C_e7_1_task1` is invalid.\n - Variable `p1_C_e7_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='C_e7']` are not defined. No namespaces are defined for the target.\n - Data generator `T_e9_2_task1` is invalid.\n - Variable `p1_T_e9_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='T_e9']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000809](https://www.ebi.ac.uk/biomodels/BIOMD0000000809)<br/><sup>Malinzi2018 - tumour-immune in</sup></summary><sup>Malinzi2018 - tumour-immune interaction model</sup></details>|<details><summary>pass</summary>[Malinzi2018 - tumour-immune interaction model.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000809#Files)</details>|pass|<details><summary>FAIL</summary>[Malinzi2018 - tumour-immune interaction model.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000809#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Malinzi2018 - tumour-immune interaction model.sedml219380.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 2: b'usage: biosimulators-tellurium [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-tellurium: error: unrecognized arguments: - tumour-immune interaction model.sedml219380.omex\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Malinzi2018 - tumour-immune interaction model.sedml235566.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 2: b'usage: biosimulators-copasi [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-copasi: error: unrecognized arguments: - tumour-immune interaction model.sedml235566.omex\n'```</details>|
|<details><summary>[BIOMD0000000810](https://www.ebi.ac.uk/biomodels/BIOMD0000000810)<br/><sup>Ganguli2018-immuno regulatory </sup></summary><sup>Ganguli2018-immuno regulatory mechanisms in tumor microenvironment</sup></details>|<details><summary>pass</summary>[Ganguli2018-immuno regulatory mechanisms in tumor microenvironment.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000810#Files)</details>|FAIL|<details><summary>FAIL</summary>[Ganguli2018-immuno regulatory mechanisms in tumor microenvironment.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000810#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Ganguli2018-immuno regulatory mechanisms in tumor microenvironment.sedml675703.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 2: b'usage: biosimulators-tellurium [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-tellurium: error: unrecognized arguments: regulatory mechanisms in tumor microenvironment.sedml675703.omex\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Ganguli2018-immuno regulatory mechanisms in tumor microenvironment.sedml700146.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 2: b'usage: biosimulators-copasi [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-copasi: error: unrecognized arguments: regulatory mechanisms in tumor microenvironment.sedml700146.omex\n'```</details>|
|<details><summary>[BIOMD0000000811](https://www.ebi.ac.uk/biomodels/BIOMD0000000811)<br/><sup>He2017 - A mathematical model </sup></summary><sup>He2017 - A mathematical model of pancreatic cancer with two kinds of treatments</sup></details>|<details><summary>pass</summary>[He2017.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000811#Files)</details>|pass|<details><summary>pass</summary>[He2017.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000811#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/He2017.sedml925993.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `He2017.xml` has warnings.\r\n - 34 warnings of type SBML unit consistency (99505). The following is the first warning at line 1448, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'a_c' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 52 warnings of type Modeling practice (80701). The following is the first warning at line 1298, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k_c' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./He2017.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `He2017.xml` has warnings.\r\n - 34 warnings of type SBML unit consistency (99505). The following is the first warning at line 1448, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'a_c' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 52 warnings of type Modeling practice (80701). The following is the first warning at line 1298, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k_c' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/He2017.sedml925993.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./He2017.sedml` is invalid.\n - Data generator `C_PCC_1_task1` is invalid.\n - Variable `p1_C_PCC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C_PCC']` are not defined. No namespaces are defined for the target.\n - Data generator `E_CD8_2_task1` is invalid.\n - Variable `p1_E_CD8_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='E_CD8']` are not defined. No namespaces are defined for the target.\n - Data generator `H_T_Helper_3_task1` is invalid.\n - Variable `p1_H_T_Helper_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H_T_Helper']` are not defined. No namespaces are defined for the target.\n - Data generator `N_Killer_4_task1` is invalid.\n - Variable `p1_N_Killer_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N_Killer']` are not defined. No namespaces are defined for the target.\n - Data generator `P_PSC_5_task1` is invalid.\n - Variable `p1_P_PSC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='P_PSC']` are not defined. No namespaces are defined for the target.\n - Data generator `R_T_Regulatory_6_task1` is invalid.\n - Variable `p1_R_T_Regulatory_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='R_T_Regulatory']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/He2017.sedml610230.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `He2017.xml` has warnings.\r\n - 34 warnings of type SBML unit consistency (99505). The following is the first warning at line 1448, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'a_c' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 52 warnings of type Modeling practice (80701). The following is the first warning at line 1298, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k_c' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./He2017.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `He2017.xml` has warnings.\r\n - 34 warnings of type SBML unit consistency (99505). The following is the first warning at line 1448, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'a_c' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 52 warnings of type Modeling practice (80701). The following is the first warning at line 1298, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k_c' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/He2017.sedml610230.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./He2017.sedml` is invalid.\n - Data generator `C_PCC_1_task1` is invalid.\n - Variable `p1_C_PCC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C_PCC']` are not defined. No namespaces are defined for the target.\n - Data generator `E_CD8_2_task1` is invalid.\n - Variable `p1_E_CD8_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='E_CD8']` are not defined. No namespaces are defined for the target.\n - Data generator `H_T_Helper_3_task1` is invalid.\n - Variable `p1_H_T_Helper_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H_T_Helper']` are not defined. No namespaces are defined for the target.\n - Data generator `N_Killer_4_task1` is invalid.\n - Variable `p1_N_Killer_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N_Killer']` are not defined. No namespaces are defined for the target.\n - Data generator `P_PSC_5_task1` is invalid.\n - Variable `p1_P_PSC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='P_PSC']` are not defined. No namespaces are defined for the target.\n - Data generator `R_T_Regulatory_6_task1` is invalid.\n - Variable `p1_R_T_Regulatory_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='R_T_Regulatory']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000812](https://www.ebi.ac.uk/biomodels/BIOMD0000000812)<br/><sup>Galante2012 - B7-H1 and a Math</sup></summary><sup>Galante2012 - B7-H1 and a Mathematical Model for Cytotoxic T Cell and Tumor Cell Interaction</sup></details>|<details><summary>pass</summary>[Galante2012.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000812#Files)</details>|pass|<details><summary>pass</summary>[Galante2012.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000812#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Galante2012.sedml243284.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Galante2012.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 621, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'E_init' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 16 warnings of type Modeling practice (80701). The following is the first warning at line 501, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Galante2012.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Galante2012.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 621, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'E_init' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 16 warnings of type Modeling practice (80701). The following is the first warning at line 501, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Galante2012.sedml243284.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Galante2012.sedml` is invalid.\n - Data generator `P_10_1_task1` is invalid.\n - Variable `p1_P_10_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='P_10']` are not defined. No namespaces are defined for the target.\n - Data generator `X_1000_2_task1` is invalid.\n - Variable `p1_X_1000_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='X_1000']` are not defined. No namespaces are defined for the target.\n - Data generator `T_Cancer_Cell_Total_3_task1` is invalid.\n - Variable `p1_T_Cancer_Cell_Total_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T_Cancer_Cell_Total']` are not defined. No namespaces are defined for the target.\n - Data generator `T_ast_4_task1` is invalid.\n - Variable `p1_T_ast_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T_ast']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Galante2012.sedml263161.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Galante2012.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 621, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'E_init' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 16 warnings of type Modeling practice (80701). The following is the first warning at line 501, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Galante2012.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Galante2012.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 621, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'E_init' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 16 warnings of type Modeling practice (80701). The following is the first warning at line 501, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Galante2012.sedml263161.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Galante2012.sedml` is invalid.\n - Data generator `P_10_1_task1` is invalid.\n - Variable `p1_P_10_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='P_10']` are not defined. No namespaces are defined for the target.\n - Data generator `X_1000_2_task1` is invalid.\n - Variable `p1_X_1000_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='X_1000']` are not defined. No namespaces are defined for the target.\n - Data generator `T_Cancer_Cell_Total_3_task1` is invalid.\n - Variable `p1_T_Cancer_Cell_Total_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T_Cancer_Cell_Total']` are not defined. No namespaces are defined for the target.\n - Data generator `T_ast_4_task1` is invalid.\n - Variable `p1_T_ast_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T_ast']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000813](https://www.ebi.ac.uk/biomodels/BIOMD0000000813)<br/><sup>Anderson2015 - Qualitative beh</sup></summary><sup>Anderson2015 - Qualitative behavior of systems of tumor-CD4+-cytokine interactions with treatments</sup></details>|<details><summary>pass</summary>[Anderson2015.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000813#Files)</details>|pass|<details><summary>pass</summary>[Anderson2015.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000813#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Anderson2015.sedml318916.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Anderson2015.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 467, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth(r, x_Tumor_Cells, K)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 425, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Anderson2015.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Anderson2015.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 467, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth(r, x_Tumor_Cells, K)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 425, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Anderson2015.sedml318916.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Anderson2015.sedml` is invalid.\n - Data generator `x_Tumor_Cells_1_task1` is invalid.\n - Variable `p1_x_Tumor_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x_Tumor_Cells']` are not defined. No namespaces are defined for the target.\n - Data generator `y_CD4_T_Cells_1_task1` is invalid.\n - Variable `p2_y_CD4_T_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='y_CD4_T_Cells']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Anderson2015.sedml5512.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Anderson2015.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 467, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth(r, x_Tumor_Cells, K)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 425, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Anderson2015.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Anderson2015.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 467, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth(r, x_Tumor_Cells, K)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 425, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Anderson2015.sedml5512.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Anderson2015.sedml` is invalid.\n - Data generator `x_Tumor_Cells_1_task1` is invalid.\n - Variable `p1_x_Tumor_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x_Tumor_Cells']` are not defined. No namespaces are defined for the target.\n - Data generator `y_CD4_T_Cells_1_task1` is invalid.\n - Variable `p2_y_CD4_T_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='y_CD4_T_Cells']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000814](https://www.ebi.ac.uk/biomodels/BIOMD0000000814)<br/><sup>Perez-Garcia19 - Computational</sup></summary><sup>Perez-Garcia19 - Computational design of improved standardized chemotherapy protocols for grade 2 oligodendrogliomas</sup></details>|<details><summary>pass</summary>[Perez-Garcia19 Computational design of improved standardized chemotherapy protocols for grade 2 oligodendrogliomas.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000814#Files)</details>|FAIL|<details><summary>FAIL</summary>[Perez-Garcia19 Computational design of improved standardized chemotherapy protocols for grade 2 oligodendrogliomas.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000814#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Perez-Garcia19 Computational design of improved standardized chemotherapy protocols for grade 2 oligodendrogliomas.sedml901061.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 2: b'usage: biosimulators-tellurium [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-tellurium: error: unrecognized arguments: Computational design of improved standardized chemotherapy protocols for grade 2 oligodendrogliomas.sedml901061.omex\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Perez-Garcia19 Computational design of improved standardized chemotherapy protocols for grade 2 oligodendrogliomas.sedml900544.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 2: b'usage: biosimulators-copasi [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-copasi: error: unrecognized arguments: Computational design of improved standardized chemotherapy protocols for grade 2 oligodendrogliomas.sedml900544.omex\n'```</details>|
|<details><summary>[BIOMD0000000815](https://www.ebi.ac.uk/biomodels/BIOMD0000000815)<br/><sup>Chrobak2011 - A mathematical m</sup></summary><sup>Chrobak2011 - A mathematical model of induced cancer-adaptive immune system competition</sup></details>|<details><summary>pass</summary>[Chrobak2011.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000815#Files)</details>|pass|<details><summary>pass</summary>[Chrobak2011.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000815#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Chrobak2011.sedml949512.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Chrobak2011.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 499, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '1 / 32' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 407, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Chrobak2011.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Chrobak2011.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 499, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '1 / 32' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 407, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Chrobak2011.sedml949512.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Chrobak2011.sedml` is invalid.\n - Data generator `x_Cancer_1_task1` is invalid.\n - Variable `p1_x_Cancer_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x_Cancer']` are not defined. No namespaces are defined for the target.\n - Data generator `y_Immune_System_2_task1` is invalid.\n - Variable `p1_y_Immune_System_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='y_Immune_System']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Chrobak2011.sedml215906.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Chrobak2011.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 499, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '1 / 32' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 407, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Chrobak2011.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Chrobak2011.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 499, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '1 / 32' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 407, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Chrobak2011.sedml215906.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Chrobak2011.sedml` is invalid.\n - Data generator `x_Cancer_1_task1` is invalid.\n - Variable `p1_x_Cancer_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x_Cancer']` are not defined. No namespaces are defined for the target.\n - Data generator `y_Immune_System_2_task1` is invalid.\n - Variable `p1_y_Immune_System_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='y_Immune_System']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000816](https://www.ebi.ac.uk/biomodels/BIOMD0000000816)<br/><sup>Gevertz2018 - Cancer Treatment</sup></summary><sup>Gevertz2018 - Cancer Treatment with Oncolytic Viruses and Dendritic Cell injections original model</sup></details>|<details><summary>pass</summary>[model.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000816#Files)</details>|pass|<details><summary>MultipleSEDMLs</summary>['OVDC.sedml', 'OV25.sedml', 'OV5.sedml']</details>|NA|NA|NA|
|<details><summary>[BIOMD0000000817](https://www.ebi.ac.uk/biomodels/BIOMD0000000817)<br/><sup>Gevertz2018 - cancer treatment</sup></summary><sup>Gevertz2018 - cancer treatment with oncolytic viruses and dendritic cell injections minimal model</sup></details>|<details><summary>pass</summary>[model.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000817#Files)</details>|pass|<details><summary>MultipleSEDMLs</summary>['OV5.sedml', 'OV25.sedml', 'OVDC.sedml']</details>|NA|NA|NA|
|<details><summary>[BIOMD0000000818](https://www.ebi.ac.uk/biomodels/BIOMD0000000818)<br/><sup>Lee2008 - ERK and PI3K signal </sup></summary><sup>Lee2008 - ERK and PI3K signal integration by Myc</sup></details>|<details><summary>pass</summary>[Lee2008.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000818#Files)</details>|pass|<details><summary>pass</summary>[Lee2008.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000818#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Lee2008.sedml532746.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Lee2008.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 1328, column 14:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <event> <trigger> expression 'geq(time, 1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Lee2008.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Lee2008.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 1328, column 14:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <event> <trigger> expression 'geq(time, 1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\x1b[0m\r\n\r\n`/root/in/Lee2008.sedml532746.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Lee2008.sedml` is invalid.\n - Data generator `Myc_1_task1` is invalid.\n - Variable `p1_Myc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Myc']` are not defined. No namespaces are defined for the target.\n - Data generator `Myc_ser62_2_task1` is invalid.\n - Variable `p1_Myc_ser62_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Myc_ser62']` are not defined. No namespaces are defined for the target.\n - Data generator `Myc_thr58_3_task1` is invalid.\n - Variable `p1_Myc_thr58_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Myc_thr58']` are not defined. No namespaces are defined for the target.\n - Data generator `Myc_total_4_task1` is invalid.\n - Variable `p1_Myc_total_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Myc_total']` are not defined. No namespaces are defined for the target.\n - Data generator `ERK_1_task1` is invalid.\n - Variable `p2_ERK_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ERK']` are not defined. No namespaces are defined for the target.\n - Data generator `PI3K_1_task1` is invalid.\n - Variable `p3_PI3K_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PI3K']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Lee2008.sedml503731.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Lee2008.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 1328, column 14:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <event> <trigger> expression 'geq(time, 1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Lee2008.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Lee2008.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 1328, column 14:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <event> <trigger> expression 'geq(time, 1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Lee2008.sedml503731.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Lee2008.sedml` is invalid.\n - Data generator `Myc_1_task1` is invalid.\n - Variable `p1_Myc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Myc']` are not defined. No namespaces are defined for the target.\n - Data generator `Myc_ser62_2_task1` is invalid.\n - Variable `p1_Myc_ser62_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Myc_ser62']` are not defined. No namespaces are defined for the target.\n - Data generator `Myc_thr58_3_task1` is invalid.\n - Variable `p1_Myc_thr58_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Myc_thr58']` are not defined. No namespaces are defined for the target.\n - Data generator `Myc_total_4_task1` is invalid.\n - Variable `p1_Myc_total_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Myc_total']` are not defined. No namespaces are defined for the target.\n - Data generator `ERK_1_task1` is invalid.\n - Variable `p2_ERK_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ERK']` are not defined. No namespaces are defined for the target.\n - Data generator `PI3K_1_task1` is invalid.\n - Variable `p3_PI3K_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PI3K']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000819](https://www.ebi.ac.uk/biomodels/BIOMD0000000819)<br/><sup>Nazari2018 - IL6 mediated stem</sup></summary><sup>Nazari2018 - IL6 mediated stem cell driven tumor growth and targeted treatment</sup></details>|<details><summary>pass</summary>[Nazari2018 - IL6 mediated stem cell driven tumor growth and targeted treatment.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000819#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000820](https://www.ebi.ac.uk/biomodels/BIOMD0000000820)<br/><sup>West2019 - Cellular interactio</sup></summary><sup>West2019 - Cellular interactions constrain tumor growth</sup></details>|<details><summary>pass</summary>[West2019 - Cellular interactions constrain tumor growth.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000820#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000821](https://www.ebi.ac.uk/biomodels/BIOMD0000000821)<br/><sup>Yazdjer2019 - reinforcement le</sup></summary><sup>Yazdjer2019 - reinforcement learning-based control of tumor growth under anti-angiogenic therapy</sup></details>|<details><summary>pass</summary>[Yazdjer2019 - reinforcement learning-based control of tumor growth under anti-angiogenic therapy.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000821#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000822](https://www.ebi.ac.uk/biomodels/BIOMD0000000822)<br/><sup>Dorvash2019 - Dynamic modeling</sup></summary><sup>Dorvash2019 - Dynamic modeling of signal transduction by mTOR complexes in cancer</sup></details>|<details><summary>pass</summary>[Dorvash2019 - Dynamic modeling of signal transduction by mTOR complexes in cancer.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000822#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000823](https://www.ebi.ac.uk/biomodels/BIOMD0000000823)<br/><sup>Varusai2018 - Dynamic modellin</sup></summary><sup>Varusai2018 - Dynamic modelling of the mTOR signalling network reveals complex emergent behaviours conferred by DEPTOR</sup></details>|<details><summary>pass</summary>[Varusai2018.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000823#Files)</details>|pass|<details><summary>pass</summary>[Varusai2018.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000823#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Varusai2018.sedml997300.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Varusai2018.xml` has warnings.\r\n - 18 warnings of type SBML unit consistency (99505). The following is the first warning at line 974, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Henri_Michaelis_Menten__irreversible(IR, Km1, V1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 38 warnings of type Modeling practice (80701). The following is the first warning at line 819, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'V1' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Varusai2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Varusai2018.xml` has warnings.\r\n - 18 warnings of type SBML unit consistency (99505). The following is the first warning at line 974, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Henri_Michaelis_Menten__irreversible(IR, Km1, V1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 38 warnings of type Modeling practice (80701). The following is the first warning at line 819, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'V1' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Varusai2018.sedml997300.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Varusai2018.sedml` is invalid.\n - Data generator `pmTORC1_1_task1` is invalid.\n - Variable `p1_pmTORC1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='pmTORC1']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Varusai2018.sedml832970.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Varusai2018.xml` has warnings.\r\n - 18 warnings of type SBML unit consistency (99505). The following is the first warning at line 974, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Henri_Michaelis_Menten__irreversible(IR, Km1, V1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 38 warnings of type Modeling practice (80701). The following is the first warning at line 819, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'V1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Varusai2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Varusai2018.xml` has warnings.\r\n - 18 warnings of type SBML unit consistency (99505). The following is the first warning at line 974, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Henri_Michaelis_Menten__irreversible(IR, Km1, V1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 38 warnings of type Modeling practice (80701). The following is the first warning at line 819, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'V1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Varusai2018.sedml832970.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Varusai2018.sedml` is invalid.\n - Data generator `pmTORC1_1_task1` is invalid.\n - Variable `p1_pmTORC1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='pmTORC1']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000824](https://www.ebi.ac.uk/biomodels/BIOMD0000000824)<br/><sup>Lewkiewics2019 - effects of ag</sup></summary><sup>Lewkiewics2019 - effects of aging on naive T cell populations and diversity</sup></details>|<details><summary>pass</summary>[Lewkiewics2019 - effects of aging on naive T cell populations and diversity.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000824#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000825](https://www.ebi.ac.uk/biomodels/BIOMD0000000825)<br/><sup>Greene2019 - Differentiate Spo</sup></summary><sup>Greene2019 - Differentiate Spontaneous and Induced Evolution to Drug Resistance During Cancer Treatment</sup></details>|<details><summary>pass</summary>[Greene2019 - Differentiate Spontaneous and Induced Evolution to Drug Resistance During Cancer Treatment.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000825#Files)</details>|pass|<details><summary>FAIL</summary>[Greene2019 - Differentiate Spontaneous and Induced Evolution to Drug Resistance During Cancer Treatment.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000825#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Greene2019 - Differentiate Spontaneous and Induced Evolution to Drug Resistance During Cancer Treatment.sedml347655.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 2: b'usage: biosimulators-tellurium [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-tellurium: error: unrecognized arguments: - Differentiate Spontaneous and Induced Evolution to Drug Resistance During Cancer Treatment.sedml347655.omex\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Greene2019 - Differentiate Spontaneous and Induced Evolution to Drug Resistance During Cancer Treatment.sedml627369.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 2: b'usage: biosimulators-copasi [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-copasi: error: unrecognized arguments: - Differentiate Spontaneous and Induced Evolution to Drug Resistance During Cancer Treatment.sedml627369.omex\n'```</details>|
|<details><summary>[BIOMD0000000826](https://www.ebi.ac.uk/biomodels/BIOMD0000000826)<br/><sup>Shin_2018_EGFR-PYK2-c-Met inte</sup></summary><sup>Shin_2018_EGFR-PYK2-c-Met interaction network_model</sup></details>|<details><summary>pass</summary>[Shin2018.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000826#Files)</details>|pass|<details><summary>pass</summary>[Shin2018.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000826#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Shin2018.sedml985539.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Shin2018.xml` has warnings.\r\n - 33 warnings of type SBML unit consistency (99505). The following is the first warning at line 2413, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'PYK2_max' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 89 warnings of type Modeling practice (80701). The following is the first warning at line 2124, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'EGF' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Shin2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Shin2018.xml` has warnings.\r\n - 33 warnings of type SBML unit consistency (99505). The following is the first warning at line 2413, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'PYK2_max' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 89 warnings of type Modeling practice (80701). The following is the first warning at line 2124, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'EGF' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Shin2018.sedml985539.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Shin2018.sedml` is invalid.\n - Data generator `pSTAT3_norm_1_task1` is invalid.\n - Variable `p1_pSTAT3_norm_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='pSTAT3_norm']` are not defined. No namespaces are defined for the target.\n - Data generator `pERK_norm_1_task1` is invalid.\n - Variable `p2_pERK_norm_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='pERK_norm']` are not defined. No namespaces are defined for the target.\n - Data generator `c_Met_norm_1_task1` is invalid.\n - Variable `p3_c_Met_norm_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='c_Met_norm']` are not defined. No namespaces are defined for the target.\n - Data generator `PYK2_norm_1_task1` is invalid.\n - Variable `p4_PYK2_norm_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='PYK2_norm']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Shin2018.sedml887014.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Shin2018.xml` has warnings.\r\n - 33 warnings of type SBML unit consistency (99505). The following is the first warning at line 2413, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'PYK2_max' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 89 warnings of type Modeling practice (80701). The following is the first warning at line 2124, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'EGF' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Shin2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Shin2018.xml` has warnings.\r\n - 33 warnings of type SBML unit consistency (99505). The following is the first warning at line 2413, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'PYK2_max' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 89 warnings of type Modeling practice (80701). The following is the first warning at line 2124, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'EGF' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Shin2018.sedml887014.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Shin2018.sedml` is invalid.\n - Data generator `pSTAT3_norm_1_task1` is invalid.\n - Variable `p1_pSTAT3_norm_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='pSTAT3_norm']` are not defined. No namespaces are defined for the target.\n - Data generator `pERK_norm_1_task1` is invalid.\n - Variable `p2_pERK_norm_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='pERK_norm']` are not defined. No namespaces are defined for the target.\n - Data generator `c_Met_norm_1_task1` is invalid.\n - Variable `p3_c_Met_norm_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='c_Met_norm']` are not defined. No namespaces are defined for the target.\n - Data generator `PYK2_norm_1_task1` is invalid.\n - Variable `p4_PYK2_norm_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='PYK2_norm']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000827](https://www.ebi.ac.uk/biomodels/BIOMD0000000827)<br/><sup>Ito2019 - gefitnib resistance </sup></summary><sup>Ito2019 - gefitnib resistance of lung adenocarcinoma caused by MET amplification</sup></details>|<details><summary>pass</summary>[Ito2019 - gefitnib resistance of lung adenocarcinoma caused by MET amplification.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000827#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000828](https://www.ebi.ac.uk/biomodels/BIOMD0000000828)<br/><sup>Jung2019 - Regulating glioblas</sup></summary><sup>Jung2019 - Regulating glioblastoma signaling pathways and anti-invasion therapy - core control model</sup></details>|<details><summary>pass</summary>[Jung2019 model.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000828#Files)</details>|FAIL|<details><summary>pass</summary>[Jung2019 simulation.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000828#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Jung2019 simulation.sedml342042.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 2: b'usage: biosimulators-tellurium [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-tellurium: error: unrecognized arguments: simulation.sedml342042.omex\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Jung2019 simulation.sedml633478.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 2: b'usage: biosimulators-copasi [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-copasi: error: unrecognized arguments: simulation.sedml633478.omex\n'```</details>|
|<details><summary>[BIOMD0000000829](https://www.ebi.ac.uk/biomodels/BIOMD0000000829)<br/><sup>Jung2019 - egulating glioblast</sup></summary><sup>Jung2019 - egulating glioblastoma signaling pathways and anti-invasion therapy cell cycle dynamics model</sup></details>|<details><summary>pass</summary>[Jung2019 - egulating glioblastoma signaling pathways and anti-invasion therapy cell cycle dynamics model.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000829#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000830](https://www.ebi.ac.uk/biomodels/BIOMD0000000830)<br/><sup>GiantsosAdams2013 - Growth of </sup></summary><sup>GiantsosAdams2013 - Growth of glycocalyx under static conditions</sup></details>|<details><summary>FAIL</summary>[Giantsos-Adams2013.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000830#Files)</details>|FAIL|<details><summary>pass</summary>[Giantsos-Adams2013.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000830#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Giantsos-Adams2013.sedml730459.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Giantsos-Adams2013.sedml` has warnings.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n`/root/in/Giantsos-Adams2013.sedml730459.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Giantsos-Adams2013.sedml` is invalid.\n - Model `model` is invalid.\n - The model file `Giantsos-Adams2013.xml` is invalid.\n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout' cannot be used in this document.\n \n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout_L2' cannot be used in this document.\n \n - General SBML conformance (10102) at line 3, column 2: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L2V4 Section 4.1\n Element 'listOfLayouts' is not part of the definition of SBML Level 2 Version 4.\n \n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Giantsos-Adams2013.sedml99546.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Giantsos-Adams2013.sedml` has warnings.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Giantsos-Adams2013.sedml99546.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Giantsos-Adams2013.sedml` is invalid.\n - Model `model` is invalid.\n - The model file `Giantsos-Adams2013.xml` is invalid.\n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout' cannot be used in this document.\n \n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout_L2' cannot be used in this document.\n \n - General SBML conformance (10102) at line 3, column 2: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L2V4 Section 4.1\n Element 'listOfLayouts' is not part of the definition of SBML Level 2 Version 4.\n \n"```</details>|
|[BIOMD0000000831](https://www.ebi.ac.uk/biomodels/BIOMD0000000831)<br/><sup>Smith1980 - Hypothalamic Regulation</sup>|<details><summary>pass</summary>[Smith1980_HypothalamicRegulation.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000831#Files)</details>|FAIL|<details><summary>FAIL</summary>[Smith1980_HypothalamicRegulation.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000831#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Smith1980_HypothalamicRegulation.sedml828547.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Smith1980_HypothalamicRegulation.sedml` has warnings.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n`/root/in/Smith1980_HypothalamicRegulation.sedml828547.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Smith1980_HypothalamicRegulation.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmp2e9fwrl_/./model1.xml` is not a file.\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Smith1980_HypothalamicRegulation.sedml311784.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Smith1980_HypothalamicRegulation.sedml` has warnings.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Smith1980_HypothalamicRegulation.sedml311784.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Smith1980_HypothalamicRegulation.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmpynu3io1w/./model1.xml` is not a file.\n'```</details>|
|<details><summary>[BIOMD0000000832](https://www.ebi.ac.uk/biomodels/BIOMD0000000832)<br/><sup>Shin2016 - Unveiling Hidden Dy</sup></summary><sup>Shin2016 - Unveiling Hidden Dynamics of Hippo Signalling</sup></details>|<details><summary>pass</summary>[Shin2016.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000832#Files)</details>|pass|<details><summary>pass</summary>[Shin2016.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000832#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Shin2016.sedml532856.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Shin2016.xml` has warnings.\r\n - 31 warnings of type SBML unit consistency (99505). The following is the first warning at line 2027, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'pAkt_max' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 61 warnings of type Modeling practice (80701). The following is the first warning at line 1784, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kc_121' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Shin2016.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Shin2016.xml` has warnings.\r\n - 31 warnings of type SBML unit consistency (99505). The following is the first warning at line 2027, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'pAkt_max' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 61 warnings of type Modeling practice (80701). The following is the first warning at line 1784, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kc_121' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Shin2016.sedml532856.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Shin2016.sedml` is invalid.\n - Data generator `aMST2_norm_1_task1` is invalid.\n - Variable `p1_aMST2_norm_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='aMST2_norm']` are not defined. No namespaces are defined for the target.\n - Data generator `pAkt_norm_2_task1` is invalid.\n - Variable `p1_pAkt_norm_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='pAkt_norm']` are not defined. No namespaces are defined for the target.\n - Data generator `ppERK_norm_3_task1` is invalid.\n - Variable `p1_ppERK_norm_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='ppERK_norm']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Shin2016.sedml660634.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Shin2016.xml` has warnings.\r\n - 31 warnings of type SBML unit consistency (99505). The following is the first warning at line 2027, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'pAkt_max' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 61 warnings of type Modeling practice (80701). The following is the first warning at line 1784, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kc_121' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Shin2016.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Shin2016.xml` has warnings.\r\n - 31 warnings of type SBML unit consistency (99505). The following is the first warning at line 2027, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'pAkt_max' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 61 warnings of type Modeling practice (80701). The following is the first warning at line 1784, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kc_121' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Shin2016.sedml660634.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Shin2016.sedml` is invalid.\n - Data generator `aMST2_norm_1_task1` is invalid.\n - Variable `p1_aMST2_norm_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='aMST2_norm']` are not defined. No namespaces are defined for the target.\n - Data generator `pAkt_norm_2_task1` is invalid.\n - Variable `p1_pAkt_norm_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='pAkt_norm']` are not defined. No namespaces are defined for the target.\n - Data generator `ppERK_norm_3_task1` is invalid.\n - Variable `p1_ppERK_norm_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='ppERK_norm']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000833](https://www.ebi.ac.uk/biomodels/BIOMD0000000833)<br/><sup>DiCamillo2016 - Insulin signal</sup></summary><sup>DiCamillo2016 - Insulin signalling pathway - Rule-based model</sup></details>|<details><summary>pass</summary>[DiCamillo2016.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000833#Files)</details>|pass|<details><summary>pass</summary>[DiCamillo2016.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000833#Files)</details>|FAIL|pass|pass|
|<details><summary>[BIOMD0000000834](https://www.ebi.ac.uk/biomodels/BIOMD0000000834)<br/><sup>Verma2016 - Ca(2+) Signal Prop</sup></summary><sup>Verma2016 - Ca(2+) Signal Propagation Along Hepatocyte Cords</sup></details>|<details><summary>pass</summary>[Verma2016.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000834#Files)</details>|pass|<details><summary>pass</summary>[Verma2016.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000834#Files)</details>|FAIL|pass|pass|
|<details><summary>[BIOMD0000000835](https://www.ebi.ac.uk/biomodels/BIOMD0000000835)<br/><sup>Rao2014 - Fatty acid beta-oxid</sup></summary><sup>Rao2014 - Fatty acid beta-oxidation (reduced model)</sup></details>|<details><summary>pass</summary>[Rao2014.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000835#Files)</details>|pass|<details><summary>pass</summary>[Rao2014.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000835#Files)</details>|FAIL|pass|pass|
|<details><summary>[BIOMD0000000836](https://www.ebi.ac.uk/biomodels/BIOMD0000000836)<br/><sup>Radosavljevic2009_Bioterrorist</sup></summary><sup>Radosavljevic2009_BioterroristAttack_PanicProtection_1</sup></details>|<details><summary>pass</summary>[Radosavljevic2009.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000836#Files)</details>|pass|<details><summary>pass</summary>[Radosavljevic2009.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000836#Files)</details>|FAIL|pass|pass|
|<details><summary>[BIOMD0000000837](https://www.ebi.ac.uk/biomodels/BIOMD0000000837)<br/><sup>Hanson2016 - Toxicity Manageme</sup></summary><sup>Hanson2016 - Toxicity Management in CAR T cell therapy for B-ALL</sup></details>|<details><summary>pass</summary>[Hanson2016.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000837#Files)</details>|FAIL|<details><summary>pass</summary>[Hanson2016.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000837#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Hanson2016.sedml861860.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Hanson2016.xml` has warnings.\r\n - 18 warnings of type SBML unit consistency (99505). The following is the first warning at line 1009, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'l' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 1275, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 3600, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Inflammation_Suppression_Drug') are item (exponent = 1, multiplier = 1, scale = 0).\r\n - 22 warnings of type Modeling practice (80701). The following is the first warning at line 897, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'n' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Hanson2016.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Hanson2016.xml` has warnings.\r\n - 18 warnings of type SBML unit consistency (99505). The following is the first warning at line 1009, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'l' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 1275, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 3600, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Inflammation_Suppression_Drug') are item (exponent = 1, multiplier = 1, scale = 0).\r\n - 22 warnings of type Modeling practice (80701). The following is the first warning at line 897, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'n' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Hanson2016.sedml861860.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Hanson2016.sedml` is invalid.\n - Data generator `B_1_task1` is invalid.\n - Variable `p1_B_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='B']` are not defined. No namespaces are defined for the target.\n - Data generator `C_e_2_task1` is invalid.\n - Variable `p1_C_e_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C_e']` are not defined. No namespaces are defined for the target.\n - Data generator `C_m_3_task1` is invalid.\n - Variable `p1_C_m_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C_m']` are not defined. No namespaces are defined for the target.\n - Data generator `H_e_4_task1` is invalid.\n - Variable `p1_H_e_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H_e']` are not defined. No namespaces are defined for the target.\n - Data generator `H_m_5_task1` is invalid.\n - Variable `p1_H_m_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H_m']` are not defined. No namespaces are defined for the target.\n - Data generator `Inflam_6_task1` is invalid.\n - Variable `p1_Inflam_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Inflam']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Hanson2016.sedml398035.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Hanson2016.xml` has warnings.\r\n - 18 warnings of type SBML unit consistency (99505). The following is the first warning at line 1009, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'l' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 1275, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 3600, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Inflammation_Suppression_Drug') are item (exponent = 1, multiplier = 1, scale = 0).\r\n - 22 warnings of type Modeling practice (80701). The following is the first warning at line 897, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'n' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Hanson2016.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Hanson2016.xml` has warnings.\r\n - 18 warnings of type SBML unit consistency (99505). The following is the first warning at line 1009, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'l' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 1275, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 3600, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Inflammation_Suppression_Drug') are item (exponent = 1, multiplier = 1, scale = 0).\r\n - 22 warnings of type Modeling practice (80701). The following is the first warning at line 897, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'n' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Hanson2016.sedml398035.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Hanson2016.sedml` is invalid.\n - Data generator `B_1_task1` is invalid.\n - Variable `p1_B_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='B']` are not defined. No namespaces are defined for the target.\n - Data generator `C_e_2_task1` is invalid.\n - Variable `p1_C_e_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C_e']` are not defined. No namespaces are defined for the target.\n - Data generator `C_m_3_task1` is invalid.\n - Variable `p1_C_m_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C_m']` are not defined. No namespaces are defined for the target.\n - Data generator `H_e_4_task1` is invalid.\n - Variable `p1_H_e_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H_e']` are not defined. No namespaces are defined for the target.\n - Data generator `H_m_5_task1` is invalid.\n - Variable `p1_H_m_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H_m']` are not defined. No namespaces are defined for the target.\n - Data generator `Inflam_6_task1` is invalid.\n - Variable `p1_Inflam_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Inflam']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000838](https://www.ebi.ac.uk/biomodels/BIOMD0000000838)<br/><sup>Tsur2019 - Response of patient</sup></summary><sup>Tsur2019 - Response of patients with melanoma to immune checkpoint blockade</sup></details>|<details><summary>pass</summary>[Tsur2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000838#Files)</details>|pass|<details><summary>pass</summary>[Tsur2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000838#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Tsur2019.sedml699155.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Tsur2019.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 489, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'alpha_A / mu_a / (M / (M + b))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 381, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha_A' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Tsur2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Tsur2019.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 489, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'alpha_A / mu_a / (M / (M + b))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 381, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha_A' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Tsur2019.sedml699155.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Tsur2019.sedml` is invalid.\n - Data generator `A_1_task1` is invalid.\n - Variable `p1_A_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='A']` are not defined. No namespaces are defined for the target.\n - Data generator `M_2_task1` is invalid.\n - Variable `p1_M_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M']` are not defined. No namespaces are defined for the target.\n - Data generator `T_3_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Tsur2019.sedml445786.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Tsur2019.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 489, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'alpha_A / mu_a / (M / (M + b))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 381, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha_A' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Tsur2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Tsur2019.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 489, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'alpha_A / mu_a / (M / (M + b))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 381, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha_A' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Tsur2019.sedml445786.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Tsur2019.sedml` is invalid.\n - Data generator `A_1_task1` is invalid.\n - Variable `p1_A_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='A']` are not defined. No namespaces are defined for the target.\n - Data generator `M_2_task1` is invalid.\n - Variable `p1_M_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M']` are not defined. No namespaces are defined for the target.\n - Data generator `T_3_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000839](https://www.ebi.ac.uk/biomodels/BIOMD0000000839)<br/><sup>Almeida2019 - Transcription-ba</sup></summary><sup>Almeida2019 - Transcription-based circadian mechanism controls the duration of molecular clock states in response to signaling inputs</sup></details>|<details><summary>pass</summary>[Almeida2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000839#Files)</details>|pass|<details><summary>pass</summary>[Almeida2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000839#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Almeida2019.sedml791476.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Almeida2019.xml` has warnings.\r\n - 31 warnings of type SBML unit consistency (99505). The following is the first warning at line 620, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'V_D' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 29 warnings of type Modeling practice (80701). The following is the first warning at line 487, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'V_R' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Almeida2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Almeida2019.xml` has warnings.\r\n - 31 warnings of type SBML unit consistency (99505). The following is the first warning at line 620, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'V_D' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 29 warnings of type Modeling practice (80701). The following is the first warning at line 487, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'V_R' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Almeida2019.sedml791476.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Almeida2019.sedml` is invalid.\n - Data generator `BMAL1_1_task1` is invalid.\n - Variable `p1_BMAL1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='BMAL1']` are not defined. No namespaces are defined for the target.\n - Data generator `CRY_2_task1` is invalid.\n - Variable `p1_CRY_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CRY']` are not defined. No namespaces are defined for the target.\n - Data generator `DBP_3_task1` is invalid.\n - Variable `p1_DBP_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='DBP']` are not defined. No namespaces are defined for the target.\n - Data generator `E4BP4_4_task1` is invalid.\n - Variable `p1_E4BP4_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='E4BP4']` are not defined. No namespaces are defined for the target.\n - Data generator `PERCRY_5_task1` is invalid.\n - Variable `p1_PERCRY_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PERCRY']` are not defined. No namespaces are defined for the target.\n - Data generator `PER_6_task1` is invalid.\n - Variable `p1_PER_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PER']` are not defined. No namespaces are defined for the target.\n - Data generator `REV_7_task1` is invalid.\n - Variable `p1_REV_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='REV']` are not defined. No namespaces are defined for the target.\n - Data generator `ROR_8_task1` is invalid.\n - Variable `p1_ROR_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ROR']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Almeida2019.sedml190744.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Almeida2019.xml` has warnings.\r\n - 31 warnings of type SBML unit consistency (99505). The following is the first warning at line 620, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'V_D' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 29 warnings of type Modeling practice (80701). The following is the first warning at line 487, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'V_R' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Almeida2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Almeida2019.xml` has warnings.\r\n - 31 warnings of type SBML unit consistency (99505). The following is the first warning at line 620, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'V_D' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 29 warnings of type Modeling practice (80701). The following is the first warning at line 487, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'V_R' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Almeida2019.sedml190744.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Almeida2019.sedml` is invalid.\n - Data generator `BMAL1_1_task1` is invalid.\n - Variable `p1_BMAL1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='BMAL1']` are not defined. No namespaces are defined for the target.\n - Data generator `CRY_2_task1` is invalid.\n - Variable `p1_CRY_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CRY']` are not defined. No namespaces are defined for the target.\n - Data generator `DBP_3_task1` is invalid.\n - Variable `p1_DBP_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='DBP']` are not defined. No namespaces are defined for the target.\n - Data generator `E4BP4_4_task1` is invalid.\n - Variable `p1_E4BP4_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='E4BP4']` are not defined. No namespaces are defined for the target.\n - Data generator `PERCRY_5_task1` is invalid.\n - Variable `p1_PERCRY_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PERCRY']` are not defined. No namespaces are defined for the target.\n - Data generator `PER_6_task1` is invalid.\n - Variable `p1_PER_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PER']` are not defined. No namespaces are defined for the target.\n - Data generator `REV_7_task1` is invalid.\n - Variable `p1_REV_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='REV']` are not defined. No namespaces are defined for the target.\n - Data generator `ROR_8_task1` is invalid.\n - Variable `p1_ROR_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ROR']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000840](https://www.ebi.ac.uk/biomodels/BIOMD0000000840)<br/><sup>Caldwell2019 - The Vicodin abu</sup></summary><sup>Caldwell2019 - The Vicodin abuse problem</sup></details>|<details><summary>pass</summary>[Caldwell2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000840#Files)</details>|pass|<details><summary>pass</summary>[Caldwell2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000840#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Caldwell2019.sedml364232.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Caldwell2019.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (99505). The following is the first warning at line 395, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_New_Vicodin_Prescriptions(lambda, rho, A)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 352, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lambda' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Caldwell2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Caldwell2019.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (99505). The following is the first warning at line 395, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_New_Vicodin_Prescriptions(lambda, rho, A)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 352, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lambda' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Caldwell2019.sedml364232.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Caldwell2019.sedml` is invalid.\n - Data generator `A_1_task1` is invalid.\n - Variable `p1_A_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='A']` are not defined. No namespaces are defined for the target.\n - Data generator `C1_2_task1` is invalid.\n - Variable `p1_C1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C1']` are not defined. No namespaces are defined for the target.\n - Data generator `C2_3_task1` is invalid.\n - Variable `p1_C2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C2']` are not defined. No namespaces are defined for the target.\n - Data generator `M_4_task1` is invalid.\n - Variable `p1_M_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M']` are not defined. No namespaces are defined for the target.\n - Data generator `T_5_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Caldwell2019.sedml730062.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Caldwell2019.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (99505). The following is the first warning at line 395, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_New_Vicodin_Prescriptions(lambda, rho, A)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 352, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lambda' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Caldwell2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Caldwell2019.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (99505). The following is the first warning at line 395, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_New_Vicodin_Prescriptions(lambda, rho, A)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 352, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lambda' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Caldwell2019.sedml730062.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Caldwell2019.sedml` is invalid.\n - Data generator `A_1_task1` is invalid.\n - Variable `p1_A_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='A']` are not defined. No namespaces are defined for the target.\n - Data generator `C1_2_task1` is invalid.\n - Variable `p1_C1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C1']` are not defined. No namespaces are defined for the target.\n - Data generator `C2_3_task1` is invalid.\n - Variable `p1_C2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C2']` are not defined. No namespaces are defined for the target.\n - Data generator `M_4_task1` is invalid.\n - Variable `p1_M_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M']` are not defined. No namespaces are defined for the target.\n - Data generator `T_5_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000841](https://www.ebi.ac.uk/biomodels/BIOMD0000000841)<br/><sup>Dhawan2019 - Endogenous miRNA </sup></summary><sup>Dhawan2019 - Endogenous miRNA sponges mediate the generation of oscillatory dynamics for a non-coding RNA network</sup></details>|<details><summary>pass</summary>[Dhawan2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000841#Files)</details>|pass|NoSEDML|NA|NA|NA|
|[BIOMD0000000842](https://www.ebi.ac.uk/biomodels/BIOMD0000000842)<br/><sup>Heitzler2012 - GPCR signalling</sup>|<details><summary>pass</summary>[MODEL1012080000.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000842#Files)</details>|pass|<details><summary>pass</summary>[MODEL1012080000.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000842#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1012080000.sedml298333.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1012080000.xml` has warnings.\r\n - 26 warnings of type SBML unit consistency (99505). The following is the first warning at line 2475, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartmentOne * Function_for_R_1(G, compartmentOne, k0)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 35 warnings of type Modeling practice (80701). The following is the first warning at line 1902, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k0' does not have a 'units' attribute.\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1012080000.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1012080000.xml` has warnings.\r\n - 26 warnings of type SBML unit consistency (99505). The following is the first warning at line 2475, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartmentOne * Function_for_R_1(G, compartmentOne, k0)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 35 warnings of type Modeling practice (80701). The following is the first warning at line 1902, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k0' does not have a 'units' attribute.\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/MODEL1012080000.sedml298333.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1012080000.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='G']` are not defined. No namespaces are defined for the target.\n - Data generator `HR_1_task1` is invalid.\n - Variable `p1_HR_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='HR']` are not defined. No namespaces are defined for the target.\n - Data generator `G_2_task1` is invalid.\n - Variable `p1_G_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='G']` are not defined. No namespaces are defined for the target.\n - Data generator `ERK_3_task1` is invalid.\n - Variable `p1_ERK_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ERK']` are not defined. No namespaces are defined for the target.\n - Data generator `PIP2_4_task1` is invalid.\n - Variable `p1_PIP2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PIP2']` are not defined. No namespaces are defined for the target.\n - Data generator `DAG_5_task1` is invalid.\n - Variable `p1_DAG_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='DAG']` are not defined. No namespaces are defined for the target.\n - Data generator `PKC_6_task1` is invalid.\n - Variable `p1_PKC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PKC']` are not defined. No namespaces are defined for the target.\n - Data generator `PKC_a_7_task1` is invalid.\n - Variable `p1_PKC_a_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PKC_a']` are not defined. No namespaces are defined for the target.\n - Data generator `GpERK_8_task1` is invalid.\n - Variable `p1_GpERK_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='GpERK']` are not defined. No namespaces are defined for the target.\n - Data generator `bpERK_9_task1` is invalid.\n - Variable `p1_bpERK_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='bpERK']` are not defined. No namespaces are defined for the target.\n - Data generator `barr1_10_task1` is invalid.\n - Variable `p1_barr1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='barr1']` are not defined. No namespaces are defined for the target.\n - Data generator `barr2_11_task1` is invalid.\n - Variable `p1_barr2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='barr2']` are not defined. No namespaces are defined for the target.\n - Data generator `G_a_12_task1` is invalid.\n - Variable `p1_G_a_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='G_a']` are not defined. No namespaces are defined for the target.\n - Data generator `HRP1_13_task1` is invalid.\n - Variable `p1_HRP1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='HRP1']` are not defined. No namespaces are defined for the target.\n - Data generator `Hbarr1RP1_14_task1` is invalid.\n - Variable `p1_Hbarr1RP1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Hbarr1RP1']` are not defined. No namespaces are defined for the target.\n - Data generator `Hbarr2RP1_15_task1` is invalid.\n - Variable `p1_Hbarr2RP1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Hbarr2RP1']` are not defined. No namespaces are defined for the target.\n - Data generator `HRP2_16_task1` is invalid.\n - Variable `p1_HRP2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='HRP2']` are not defined. No namespaces are defined for the target.\n - Data generator `HRbarr2_17_task1` is invalid.\n - Variable `p1_HRbarr2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='HRbarr2']` are not defined. No namespaces are defined for the target.\n - Data generator `Hbarr2RP2_18_task1` is invalid.\n - Variable `p1_Hbarr2RP2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Hbarr2RP2']` are not defined. No namespaces are defined for the target.\n - Data generator `GRK2_3_19_task1` is invalid.\n - Variable `p1_GRK2_3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='GRK2_3']` are not defined. No namespaces are defined for the target.\n - Data generator `GRK5_6_20_task1` is invalid.\n - Variable `p1_GRK5_6_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='GRK5_6']` are not defined. No namespaces are defined for the target.\n - Data generator `pERK_21_task1` is invalid.\n - Variable `p1_pERK_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='pERK']` are not defined. No namespaces are defined for the target.\n - Data generator `pERK_1_task1` is invalid.\n - Variable `p2_pERK_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='pERK']` are not defined. No namespaces are defined for the target.\n - Data generator `HR_1_task2` is invalid.\n - Variable `p1_HR_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='HR']` are not defined. No namespaces are defined for the target.\n - Data generator `G_2_task2` is invalid.\n - Variable `p1_G_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='G']` are not defined. No namespaces are defined for the target.\n - Data generator `ERK_3_task2` is invalid.\n - Variable `p1_ERK_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ERK']` are not defined. No namespaces are defined for the target.\n - Data generator `PIP2_4_task2` is invalid.\n - Variable `p1_PIP2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PIP2']` are not defined. No namespaces are defined for the target.\n - Data generator `DAG_5_task2` is invalid.\n - Variable `p1_DAG_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='DAG']` are not defined. No namespaces are defined for the target.\n - Data generator `PKC_6_task2` is invalid.\n - Variable `p1_PKC_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PKC']` are not defined. No namespaces are defined for the target.\n - Data generator `PKC_a_7_task2` is invalid.\n - Variable `p1_PKC_a_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PKC_a']` are not defined. No namespaces are defined for the target.\n - Data generator `GpERK_8_task2` is invalid.\n - Variable `p1_GpERK_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='GpERK']` are not defined. No namespaces are defined for the target.\n - Data generator `bpERK_9_task2` is invalid.\n - Variable `p1_bpERK_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='bpERK']` are not defined. No namespaces are defined for the target.\n - Data generator `barr1_10_task2` is invalid.\n - Variable `p1_barr1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='barr1']` are not defined. No namespaces are defined for the target.\n - Data generator `barr2_11_task2` is invalid.\n - Variable `p1_barr2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='barr2']` are not defined. No namespaces are defined for the target.\n - Data generator `G_a_12_task2` is invalid.\n - Variable `p1_G_a_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='G_a']` are not defined. No namespaces are defined for the target.\n - Data generator `HRP1_13_task2` is invalid.\n - Variable `p1_HRP1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='HRP1']` are not defined. No namespaces are defined for the target.\n - Data generator `Hbarr1RP1_14_task2` is invalid.\n - Variable `p1_Hbarr1RP1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Hbarr1RP1']` are not defined. No namespaces are defined for the target.\n - Data generator `Hbarr2RP1_15_task2` is invalid.\n - Variable `p1_Hbarr2RP1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Hbarr2RP1']` are not defined. No namespaces are defined for the target.\n - Data generator `HRP2_16_task2` is invalid.\n - Variable `p1_HRP2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='HRP2']` are not defined. No namespaces are defined for the target.\n - Data generator `HRbarr2_17_task2` is invalid.\n - Variable `p1_HRbarr2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='HRbarr2']` are not defined. No namespaces are defined for the target.\n - Data generator `Hbarr2RP2_18_task2` is invalid.\n - Variable `p1_Hbarr2RP2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Hbarr2RP2']` are not defined. No namespaces are defined for the target.\n - Data generator `GRK2_3_19_task2` is invalid.\n - Variable `p1_GRK2_3_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='GRK2_3']` are not defined. No namespaces are defined for the target.\n - Data generator `GRK5_6_20_task2` is invalid.\n - Variable `p1_GRK5_6_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='GRK5_6']` are not defined. No namespaces are defined for the target.\n - Data generator `pERK_21_task2` is invalid.\n - Variable `p1_pERK_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='pERK']` are not defined. No namespaces are defined for the target.\n - Data generator `pERK_1_task2` is invalid.\n - Variable `p2_pERK_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='pERK']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1012080000.sedml608826.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1012080000.xml` has warnings.\r\n - 26 warnings of type SBML unit consistency (99505). The following is the first warning at line 2475, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartmentOne * Function_for_R_1(G, compartmentOne, k0)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 35 warnings of type Modeling practice (80701). The following is the first warning at line 1902, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k0' does not have a 'units' attribute.\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1012080000.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1012080000.xml` has warnings.\r\n - 26 warnings of type SBML unit consistency (99505). The following is the first warning at line 2475, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartmentOne * Function_for_R_1(G, compartmentOne, k0)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 35 warnings of type Modeling practice (80701). The following is the first warning at line 1902, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k0' does not have a 'units' attribute.\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1012080000.sedml608826.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1012080000.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='G']` are not defined. No namespaces are defined for the target.\n - Data generator `HR_1_task1` is invalid.\n - Variable `p1_HR_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='HR']` are not defined. No namespaces are defined for the target.\n - Data generator `G_2_task1` is invalid.\n - Variable `p1_G_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='G']` are not defined. No namespaces are defined for the target.\n - Data generator `ERK_3_task1` is invalid.\n - Variable `p1_ERK_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ERK']` are not defined. No namespaces are defined for the target.\n - Data generator `PIP2_4_task1` is invalid.\n - Variable `p1_PIP2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PIP2']` are not defined. No namespaces are defined for the target.\n - Data generator `DAG_5_task1` is invalid.\n - Variable `p1_DAG_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='DAG']` are not defined. No namespaces are defined for the target.\n - Data generator `PKC_6_task1` is invalid.\n - Variable `p1_PKC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PKC']` are not defined. No namespaces are defined for the target.\n - Data generator `PKC_a_7_task1` is invalid.\n - Variable `p1_PKC_a_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PKC_a']` are not defined. No namespaces are defined for the target.\n - Data generator `GpERK_8_task1` is invalid.\n - Variable `p1_GpERK_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='GpERK']` are not defined. No namespaces are defined for the target.\n - Data generator `bpERK_9_task1` is invalid.\n - Variable `p1_bpERK_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='bpERK']` are not defined. No namespaces are defined for the target.\n - Data generator `barr1_10_task1` is invalid.\n - Variable `p1_barr1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='barr1']` are not defined. No namespaces are defined for the target.\n - Data generator `barr2_11_task1` is invalid.\n - Variable `p1_barr2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='barr2']` are not defined. No namespaces are defined for the target.\n - Data generator `G_a_12_task1` is invalid.\n - Variable `p1_G_a_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='G_a']` are not defined. No namespaces are defined for the target.\n - Data generator `HRP1_13_task1` is invalid.\n - Variable `p1_HRP1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='HRP1']` are not defined. No namespaces are defined for the target.\n - Data generator `Hbarr1RP1_14_task1` is invalid.\n - Variable `p1_Hbarr1RP1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Hbarr1RP1']` are not defined. No namespaces are defined for the target.\n - Data generator `Hbarr2RP1_15_task1` is invalid.\n - Variable `p1_Hbarr2RP1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Hbarr2RP1']` are not defined. No namespaces are defined for the target.\n - Data generator `HRP2_16_task1` is invalid.\n - Variable `p1_HRP2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='HRP2']` are not defined. No namespaces are defined for the target.\n - Data generator `HRbarr2_17_task1` is invalid.\n - Variable `p1_HRbarr2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='HRbarr2']` are not defined. No namespaces are defined for the target.\n - Data generator `Hbarr2RP2_18_task1` is invalid.\n - Variable `p1_Hbarr2RP2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Hbarr2RP2']` are not defined. No namespaces are defined for the target.\n - Data generator `GRK2_3_19_task1` is invalid.\n - Variable `p1_GRK2_3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='GRK2_3']` are not defined. No namespaces are defined for the target.\n - Data generator `GRK5_6_20_task1` is invalid.\n - Variable `p1_GRK5_6_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='GRK5_6']` are not defined. No namespaces are defined for the target.\n - Data generator `pERK_21_task1` is invalid.\n - Variable `p1_pERK_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='pERK']` are not defined. No namespaces are defined for the target.\n - Data generator `pERK_1_task1` is invalid.\n - Variable `p2_pERK_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='pERK']` are not defined. No namespaces are defined for the target.\n - Data generator `HR_1_task2` is invalid.\n - Variable `p1_HR_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='HR']` are not defined. No namespaces are defined for the target.\n - Data generator `G_2_task2` is invalid.\n - Variable `p1_G_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='G']` are not defined. No namespaces are defined for the target.\n - Data generator `ERK_3_task2` is invalid.\n - Variable `p1_ERK_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ERK']` are not defined. No namespaces are defined for the target.\n - Data generator `PIP2_4_task2` is invalid.\n - Variable `p1_PIP2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PIP2']` are not defined. No namespaces are defined for the target.\n - Data generator `DAG_5_task2` is invalid.\n - Variable `p1_DAG_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='DAG']` are not defined. No namespaces are defined for the target.\n - Data generator `PKC_6_task2` is invalid.\n - Variable `p1_PKC_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PKC']` are not defined. No namespaces are defined for the target.\n - Data generator `PKC_a_7_task2` is invalid.\n - Variable `p1_PKC_a_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PKC_a']` are not defined. No namespaces are defined for the target.\n - Data generator `GpERK_8_task2` is invalid.\n - Variable `p1_GpERK_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='GpERK']` are not defined. No namespaces are defined for the target.\n - Data generator `bpERK_9_task2` is invalid.\n - Variable `p1_bpERK_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='bpERK']` are not defined. No namespaces are defined for the target.\n - Data generator `barr1_10_task2` is invalid.\n - Variable `p1_barr1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='barr1']` are not defined. No namespaces are defined for the target.\n - Data generator `barr2_11_task2` is invalid.\n - Variable `p1_barr2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='barr2']` are not defined. No namespaces are defined for the target.\n - Data generator `G_a_12_task2` is invalid.\n - Variable `p1_G_a_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='G_a']` are not defined. No namespaces are defined for the target.\n - Data generator `HRP1_13_task2` is invalid.\n - Variable `p1_HRP1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='HRP1']` are not defined. No namespaces are defined for the target.\n - Data generator `Hbarr1RP1_14_task2` is invalid.\n - Variable `p1_Hbarr1RP1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Hbarr1RP1']` are not defined. No namespaces are defined for the target.\n - Data generator `Hbarr2RP1_15_task2` is invalid.\n - Variable `p1_Hbarr2RP1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Hbarr2RP1']` are not defined. No namespaces are defined for the target.\n - Data generator `HRP2_16_task2` is invalid.\n - Variable `p1_HRP2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='HRP2']` are not defined. No namespaces are defined for the target.\n - Data generator `HRbarr2_17_task2` is invalid.\n - Variable `p1_HRbarr2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='HRbarr2']` are not defined. No namespaces are defined for the target.\n - Data generator `Hbarr2RP2_18_task2` is invalid.\n - Variable `p1_Hbarr2RP2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Hbarr2RP2']` are not defined. No namespaces are defined for the target.\n - Data generator `GRK2_3_19_task2` is invalid.\n - Variable `p1_GRK2_3_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='GRK2_3']` are not defined. No namespaces are defined for the target.\n - Data generator `GRK5_6_20_task2` is invalid.\n - Variable `p1_GRK5_6_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='GRK5_6']` are not defined. No namespaces are defined for the target.\n - Data generator `pERK_21_task2` is invalid.\n - Variable `p1_pERK_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='pERK']` are not defined. No namespaces are defined for the target.\n - Data generator `pERK_1_task2` is invalid.\n - Variable `p2_pERK_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='pERK']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000843](https://www.ebi.ac.uk/biomodels/BIOMD0000000843)<br/><sup>Dudziuk2019 - Biologically sou</sup></summary><sup>Dudziuk2019 - Biologically sound formal model of Hsp70 heat induction</sup></details>|<details><summary>pass</summary>[Dudziuk2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000843#Files)</details>|pass|<details><summary>pass</summary>[Dudziuk2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000843#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Dudziuk2019.sedml460155.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Dudziuk2019.xml` has warnings.\r\n - 25 warnings of type SBML unit consistency (99505). The following is the first warning at line 963, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'I_7_42' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 39 warnings of type Modeling practice (80701). The following is the first warning at line 562, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_5_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_6_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Dudziuk2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Dudziuk2019.xml` has warnings.\r\n - 25 warnings of type SBML unit consistency (99505). The following is the first warning at line 963, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'I_7_42' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 39 warnings of type Modeling practice (80701). The following is the first warning at line 562, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_5_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_6_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/Dudziuk2019.sedml460155.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Dudziuk2019.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `V_den_T_1_task1` is invalid.\n - Variable `p1_V_den_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='V_den_T']` are not defined. No namespaces are defined for the target.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `HSF_3_tot_1_task1` is invalid.\n - Variable `p2_HSF_3_tot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='HSF_3_tot']` are not defined. No namespaces are defined for the target.\n - Data generator `HSE_HSF_3_1_task1` is invalid.\n - Variable `p3_HSE_HSF_3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='HSE_HSF_3']` are not defined. No namespaces are defined for the target.\n - Data generator `V_den_T_1_task2` is invalid.\n - Variable `p1_V_den_T_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='V_den_T']` are not defined. No namespaces are defined for the target.\n - Data generator `T_1_task2` is invalid.\n - Variable `p1_T_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `HSF_3_tot_1_task2` is invalid.\n - Variable `p2_HSF_3_tot_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='HSF_3_tot']` are not defined. No namespaces are defined for the target.\n - Data generator `HSE_HSF_3_1_task2` is invalid.\n - Variable `p3_HSE_HSF_3_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='HSE_HSF_3']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Dudziuk2019.sedml452625.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Dudziuk2019.xml` has warnings.\r\n - 25 warnings of type SBML unit consistency (99505). The following is the first warning at line 963, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'I_7_42' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 39 warnings of type Modeling practice (80701). The following is the first warning at line 562, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_5_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_6_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Dudziuk2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Dudziuk2019.xml` has warnings.\r\n - 25 warnings of type SBML unit consistency (99505). The following is the first warning at line 963, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'I_7_42' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 39 warnings of type Modeling practice (80701). The following is the first warning at line 562, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_5_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_6_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Dudziuk2019.sedml452625.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Dudziuk2019.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `V_den_T_1_task1` is invalid.\n - Variable `p1_V_den_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='V_den_T']` are not defined. No namespaces are defined for the target.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `HSF_3_tot_1_task1` is invalid.\n - Variable `p2_HSF_3_tot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='HSF_3_tot']` are not defined. No namespaces are defined for the target.\n - Data generator `HSE_HSF_3_1_task1` is invalid.\n - Variable `p3_HSE_HSF_3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='HSE_HSF_3']` are not defined. No namespaces are defined for the target.\n - Data generator `V_den_T_1_task2` is invalid.\n - Variable `p1_V_den_T_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='V_den_T']` are not defined. No namespaces are defined for the target.\n - Data generator `T_1_task2` is invalid.\n - Variable `p1_T_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `HSF_3_tot_1_task2` is invalid.\n - Variable `p2_HSF_3_tot_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='HSF_3_tot']` are not defined. No namespaces are defined for the target.\n - Data generator `HSE_HSF_3_1_task2` is invalid.\n - Variable `p3_HSE_HSF_3_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='HSE_HSF_3']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000844](https://www.ebi.ac.uk/biomodels/BIOMD0000000844)<br/><sup>Viertel2019 - A Computational </sup></summary><sup>Viertel2019 - A Computational model of the mammalian external tufted cell</sup></details>|<details><summary>pass</summary>[Viertel2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000844#Files)</details>|pass|<details><summary>pass</summary>[Viertel2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000844#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Viertel2019.sedml264498.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Viertel2019.xml` has warnings.\r\n - 1 warning of type SBO term consistency (10713). The following is the first warning at line 355, column 6:\r\n - The value of the \'sboTerm\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, "participant physical type"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, "material entity").\r\n Reference: L2V4 Section 5\r\n SBO term \'SBO:0000259\' on the <species> is not in the appropriate branch.\r\n - 41 warnings of type SBML unit consistency (99505). The following is the first warning at line 1865, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression \'1 / (1 + exp((V - theta_hNaP) / beta_hNaP))\' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 82 warnings of type Modeling practice (80701). The following is the first warning at line 633, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id \'vL\' does not have a \'units\' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Viertel2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Viertel2019.xml` has warnings.\r\n - 1 warning of type SBO term consistency (10713). The following is the first warning at line 355, column 6:\r\n - The value of the \'sboTerm\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, "participant physical type"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, "material entity").\r\n Reference: L2V4 Section 5\r\n SBO term \'SBO:0000259\' on the <species> is not in the appropriate branch.\r\n - 41 warnings of type SBML unit consistency (99505). The following is the first warning at line 1865, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression \'1 / (1 + exp((V - theta_hNaP) / beta_hNaP))\' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 82 warnings of type Modeling practice (80701). The following is the first warning at line 633, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id \'vL\' does not have a \'units\' attribute.\x1b[0m\r\n\r\n`/root/in/Viertel2019.sedml264498.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Viertel2019.sedml` is invalid.\n - Data generator `IBK_1_task1` is invalid.\n - Variable `p1_IBK_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id=\'IBK\']` are not defined. No namespaces are defined for the target.\n - Data generator `IH_2_task1` is invalid.\n - Variable `p1_IH_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id=\'IH\']` are not defined. No namespaces are defined for the target.\n - Data generator `ILVA_3_task1` is invalid.\n - Variable `p1_ILVA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id=\'ILVA\']` are not defined. No namespaces are defined for the target.\n - Data generator `INaP_4_task1` is invalid.\n - Variable `p1_INaP_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id=\'INaP\']` are not defined. No namespaces are defined for the target.\n - Data generator `Ca_5_task1` is invalid.\n - Variable `p1_Ca_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'Ca\']` are not defined. No namespaces are defined for the target.\n - Data generator `V_6_task1` is invalid.\n - Variable `p1_V_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'V\']` are not defined. No namespaces are defined for the target.\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Viertel2019.sedml483151.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Viertel2019.xml` has warnings.\r\n - 1 warning of type SBO term consistency (10713). The following is the first warning at line 355, column 6:\r\n - The value of the \'sboTerm\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, "participant physical type"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, "material entity").\r\n Reference: L2V4 Section 5\r\n SBO term \'SBO:0000259\' on the <species> is not in the appropriate branch.\r\n - 41 warnings of type SBML unit consistency (99505). The following is the first warning at line 1865, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression \'1 / (1 + exp((V - theta_hNaP) / beta_hNaP))\' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 82 warnings of type Modeling practice (80701). The following is the first warning at line 633, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id \'vL\' does not have a \'units\' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Viertel2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Viertel2019.xml` has warnings.\r\n - 1 warning of type SBO term consistency (10713). The following is the first warning at line 355, column 6:\r\n - The value of the \'sboTerm\' attribute on a <species> is expected to be an SBO identifier (http://www.biomodels.net/SBO/). In SBML Level 2 prior to Version 4 it is expected to refer to a participant physical type (i.e., terms derived from SBO:0000236, "participant physical type"); in Versions 4 and above it is expected to refer to a material entity (i.e., terms derived from SBO:0000240, "material entity").\r\n Reference: L2V4 Section 5\r\n SBO term \'SBO:0000259\' on the <species> is not in the appropriate branch.\r\n - 41 warnings of type SBML unit consistency (99505). The following is the first warning at line 1865, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression \'1 / (1 + exp((V - theta_hNaP) / beta_hNaP))\' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 82 warnings of type Modeling practice (80701). The following is the first warning at line 633, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id \'vL\' does not have a \'units\' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Viertel2019.sedml483151.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Viertel2019.sedml` is invalid.\n - Data generator `IBK_1_task1` is invalid.\n - Variable `p1_IBK_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id=\'IBK\']` are not defined. No namespaces are defined for the target.\n - Data generator `IH_2_task1` is invalid.\n - Variable `p1_IH_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id=\'IH\']` are not defined. No namespaces are defined for the target.\n - Data generator `ILVA_3_task1` is invalid.\n - Variable `p1_ILVA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id=\'ILVA\']` are not defined. No namespaces are defined for the target.\n - Data generator `INaP_4_task1` is invalid.\n - Variable `p1_INaP_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id=\'INaP\']` are not defined. No namespaces are defined for the target.\n - Data generator `Ca_5_task1` is invalid.\n - Variable `p1_Ca_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'Ca\']` are not defined. No namespaces are defined for the target.\n - Data generator `V_6_task1` is invalid.\n - Variable `p1_V_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'V\']` are not defined. No namespaces are defined for the target.\n'```</details>|
|<details><summary>[BIOMD0000000845](https://www.ebi.ac.uk/biomodels/BIOMD0000000845)<br/><sup>Gulbudak2019.1 - Heterogeneous</sup></summary><sup>Gulbudak2019.1 - Heterogeneous viral strategies promote coexistence in virus-microbe systems (Lytic)</sup></details>|<details><summary>pass</summary>[Gulbudak2019.1.Lytic.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000845#Files)</details>|pass|<details><summary>pass</summary>[Gulbudak2019.1.Lytic.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000845#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Gulbudak2019.1.Lytic.sedml252407.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Gulbudak2019.1.Lytic.xml` has warnings.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 378, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '1 / 24' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 308, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Gulbudak2019.1.Lytic.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Gulbudak2019.1.Lytic.xml` has warnings.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 378, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '1 / 24' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 308, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Gulbudak2019.1.Lytic.sedml252407.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Gulbudak2019.1.Lytic.sedml` is invalid.\n - Data generator `I_1_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `S_2_task1` is invalid.\n - Variable `p1_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S']` are not defined. No namespaces are defined for the target.\n - Data generator `V_L_3_task1` is invalid.\n - Variable `p1_V_L_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V_L']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Gulbudak2019.1.Lytic.sedml174308.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Gulbudak2019.1.Lytic.xml` has warnings.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 378, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '1 / 24' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 308, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Gulbudak2019.1.Lytic.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Gulbudak2019.1.Lytic.xml` has warnings.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 378, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '1 / 24' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 308, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Gulbudak2019.1.Lytic.sedml174308.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Gulbudak2019.1.Lytic.sedml` is invalid.\n - Data generator `I_1_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `S_2_task1` is invalid.\n - Variable `p1_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S']` are not defined. No namespaces are defined for the target.\n - Data generator `V_L_3_task1` is invalid.\n - Variable `p1_V_L_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V_L']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000846](https://www.ebi.ac.uk/biomodels/BIOMD0000000846)<br/><sup>Gulbudak2019.2 - Heterogeneous</sup></summary><sup>Gulbudak2019.2 - Heterogeneous viral strategies promote coexistence in virus-microbe systems (Chronic)</sup></details>|<details><summary>pass</summary>[Gulbudak2019.2.Chronic.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000846#Files)</details>|pass|<details><summary>FAIL</summary>[Gulbudak2019.2.Chronic.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000846#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Gulbudak2019.2.Chronic.sedml364541.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/Gulbudak2019.2.Chronic.sedml364541.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Gulbudak2019.2.Chronic.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmpq4xeq1h8/./model1.xml` is not a file.\n - Data generator `C_1_task1` is invalid.\n - Variable `p1_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C']` are not defined. No namespaces are defined for the target.\n - Data generator `S_2_task1` is invalid.\n - Variable `p1_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S']` are not defined. No namespaces are defined for the target.\n - Data generator `V_C_3_task1` is invalid.\n - Variable `p1_V_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V_C']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Gulbudak2019.2.Chronic.sedml991708.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Gulbudak2019.2.Chronic.sedml991708.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Gulbudak2019.2.Chronic.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmp0ymih7va/./model1.xml` is not a file.\n - Data generator `C_1_task1` is invalid.\n - Variable `p1_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C']` are not defined. No namespaces are defined for the target.\n - Data generator `S_2_task1` is invalid.\n - Variable `p1_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S']` are not defined. No namespaces are defined for the target.\n - Data generator `V_C_3_task1` is invalid.\n - Variable `p1_V_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V_C']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000847](https://www.ebi.ac.uk/biomodels/BIOMD0000000847)<br/><sup>Adams2019 - The regulatory rol</sup></summary><sup>Adams2019 - The regulatory role of shikimate in plant phenylalanine metabolism</sup></details>|<details><summary>pass</summary>[Adams2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000847#Files)</details>|pass|<details><summary>pass</summary>[Adams2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000847#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Adams2019.sedml229026.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Adams2019.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 660, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Constant_flux__irreversible(a_0)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 17 warnings of type Modeling practice (80701). The following is the first warning at line 528, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a_1' does not have a 'units' attribute.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Adams2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Adams2019.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 660, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Constant_flux__irreversible(a_0)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 17 warnings of type Modeling practice (80701). The following is the first warning at line 528, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a_1' does not have a 'units' attribute.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n\r\n`/root/in/Adams2019.sedml229026.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Adams2019.sedml` is invalid.\n - Each identified SED object must have a unique id. Multiple objects have the following ids:\n - p1_a_0_task1\n - p1_a_0_task3\n - Task `task3` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='a_0']` are not defined. No namespaces are defined for the target.\n - Data generator `V_1_Phenylalanine_Biosynthesis_Shikimate_1_task1` is invalid.\n - Variable `p1_V_1_Phenylalanine_Biosynthesis_Shikimate_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='V_1_Phenylalanine_Biosynthesis_Shikimate']` are not defined. No namespaces are defined for the target.\n - Data generator `a_0_1_task1` is invalid.\n - Variable `p1_a_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='a_0']` are not defined. No namespaces are defined for the target.\n - Data generator `V_2_Shikimate_Transport_2_task1` is invalid.\n - Variable `p1_V_2_Shikimate_Transport_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='V_2_Shikimate_Transport']` are not defined. No namespaces are defined for the target.\n - Data generator `a_0_2_task1` is invalid.\n - Variable `p1_a_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='a_0']` are not defined. No namespaces are defined for the target.\n - Data generator `V_3_Caffeoyl_Shikimate_Biosynthesis_3_task1` is invalid.\n - Variable `p1_V_3_Caffeoyl_Shikimate_Biosynthesis_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='V_3_Caffeoyl_Shikimate_Biosynthesis']` are not defined. No namespaces are defined for the target.\n - Data generator `a_0_3_task1` is invalid.\n - Variable `p1_a_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='a_0']` are not defined. No namespaces are defined for the target.\n - Data generator `V_4_Mass_Release_Monolignol_Pathway_4_task1` is invalid.\n - Variable `p1_V_4_Mass_Release_Monolignol_Pathway_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='V_4_Mass_Release_Monolignol_Pathway']` are not defined. No namespaces are defined for the target.\n - Data generator `a_0_4_task1` is invalid.\n - Variable `p1_a_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='a_0']` are not defined. No namespaces are defined for the target.\n - Data generator `V_5_Phenylalanine_To_Protein_5_task1` is invalid.\n - Variable `p1_V_5_Phenylalanine_To_Protein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='V_5_Phenylalanine_To_Protein']` are not defined. No namespaces are defined for the target.\n - Data generator `a_0_5_task1` is invalid.\n - Variable `p1_a_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='a_0']` are not defined. No namespaces are defined for the target.\n - Data generator `V_1_Phenylalanine_Biosynthesis_Shikimate_1_task3` is invalid.\n - Variable `p1_V_1_Phenylalanine_Biosynthesis_Shikimate_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='V_1_Phenylalanine_Biosynthesis_Shikimate']` are not defined. No namespaces are defined for the target.\n - Data generator `a_0_1_task3` is invalid.\n - Variable `p1_a_0_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='a_0']` are not defined. No namespaces are defined for the target.\n - Data generator `V_2_Shikimate_Transport_2_task3` is invalid.\n - Variable `p1_V_2_Shikimate_Transport_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='V_2_Shikimate_Transport']` are not defined. No namespaces are defined for the target.\n - Data generator `a_0_2_task3` is invalid.\n - Variable `p1_a_0_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='a_0']` are not defined. No namespaces are defined for the target.\n - Data generator `V_3_Caffeoyl_Shikimate_Biosynthesis_3_task3` is invalid.\n - Variable `p1_V_3_Caffeoyl_Shikimate_Biosynthesis_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='V_3_Caffeoyl_Shikimate_Biosynthesis']` are not defined. No namespaces are defined for the target.\n - Data generator `a_0_3_task3` is invalid.\n - Variable `p1_a_0_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='a_0']` are not defined. No namespaces are defined for the target.\n - Data generator `V_4_Mass_Release_Monolignol_Pathway_4_task3` is invalid.\n - Variable `p1_V_4_Mass_Release_Monolignol_Pathway_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='V_4_Mass_Release_Monolignol_Pathway']` are not defined. No namespaces are defined for the target.\n - Data generator `a_0_4_task3` is invalid.\n - Variable `p1_a_0_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='a_0']` are not defined. No namespaces are defined for the target.\n - Data generator `V_5_Phenylalanine_To_Protein_5_task3` is invalid.\n - Variable `p1_V_5_Phenylalanine_To_Protein_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='V_5_Phenylalanine_To_Protein']` are not defined. No namespaces are defined for the target.\n - Data generator `a_0_5_task3` is invalid.\n - Variable `p1_a_0_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='a_0']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Adams2019.sedml977909.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Adams2019.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 660, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Constant_flux__irreversible(a_0)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 17 warnings of type Modeling practice (80701). The following is the first warning at line 528, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a_1' does not have a 'units' attribute.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Adams2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Adams2019.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 660, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Constant_flux__irreversible(a_0)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 17 warnings of type Modeling practice (80701). The following is the first warning at line 528, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a_1' does not have a 'units' attribute.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Adams2019.sedml977909.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Adams2019.sedml` is invalid.\n - Each identified SED object must have a unique id. Multiple objects have the following ids:\n - p1_a_0_task1\n - p1_a_0_task3\n - Task `task3` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='a_0']` are not defined. No namespaces are defined for the target.\n - Data generator `V_1_Phenylalanine_Biosynthesis_Shikimate_1_task1` is invalid.\n - Variable `p1_V_1_Phenylalanine_Biosynthesis_Shikimate_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='V_1_Phenylalanine_Biosynthesis_Shikimate']` are not defined. No namespaces are defined for the target.\n - Data generator `a_0_1_task1` is invalid.\n - Variable `p1_a_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='a_0']` are not defined. No namespaces are defined for the target.\n - Data generator `V_2_Shikimate_Transport_2_task1` is invalid.\n - Variable `p1_V_2_Shikimate_Transport_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='V_2_Shikimate_Transport']` are not defined. No namespaces are defined for the target.\n - Data generator `a_0_2_task1` is invalid.\n - Variable `p1_a_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='a_0']` are not defined. No namespaces are defined for the target.\n - Data generator `V_3_Caffeoyl_Shikimate_Biosynthesis_3_task1` is invalid.\n - Variable `p1_V_3_Caffeoyl_Shikimate_Biosynthesis_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='V_3_Caffeoyl_Shikimate_Biosynthesis']` are not defined. No namespaces are defined for the target.\n - Data generator `a_0_3_task1` is invalid.\n - Variable `p1_a_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='a_0']` are not defined. No namespaces are defined for the target.\n - Data generator `V_4_Mass_Release_Monolignol_Pathway_4_task1` is invalid.\n - Variable `p1_V_4_Mass_Release_Monolignol_Pathway_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='V_4_Mass_Release_Monolignol_Pathway']` are not defined. No namespaces are defined for the target.\n - Data generator `a_0_4_task1` is invalid.\n - Variable `p1_a_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='a_0']` are not defined. No namespaces are defined for the target.\n - Data generator `V_5_Phenylalanine_To_Protein_5_task1` is invalid.\n - Variable `p1_V_5_Phenylalanine_To_Protein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='V_5_Phenylalanine_To_Protein']` are not defined. No namespaces are defined for the target.\n - Data generator `a_0_5_task1` is invalid.\n - Variable `p1_a_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='a_0']` are not defined. No namespaces are defined for the target.\n - Data generator `V_1_Phenylalanine_Biosynthesis_Shikimate_1_task3` is invalid.\n - Variable `p1_V_1_Phenylalanine_Biosynthesis_Shikimate_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='V_1_Phenylalanine_Biosynthesis_Shikimate']` are not defined. No namespaces are defined for the target.\n - Data generator `a_0_1_task3` is invalid.\n - Variable `p1_a_0_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='a_0']` are not defined. No namespaces are defined for the target.\n - Data generator `V_2_Shikimate_Transport_2_task3` is invalid.\n - Variable `p1_V_2_Shikimate_Transport_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='V_2_Shikimate_Transport']` are not defined. No namespaces are defined for the target.\n - Data generator `a_0_2_task3` is invalid.\n - Variable `p1_a_0_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='a_0']` are not defined. No namespaces are defined for the target.\n - Data generator `V_3_Caffeoyl_Shikimate_Biosynthesis_3_task3` is invalid.\n - Variable `p1_V_3_Caffeoyl_Shikimate_Biosynthesis_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='V_3_Caffeoyl_Shikimate_Biosynthesis']` are not defined. No namespaces are defined for the target.\n - Data generator `a_0_3_task3` is invalid.\n - Variable `p1_a_0_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='a_0']` are not defined. No namespaces are defined for the target.\n - Data generator `V_4_Mass_Release_Monolignol_Pathway_4_task3` is invalid.\n - Variable `p1_V_4_Mass_Release_Monolignol_Pathway_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='V_4_Mass_Release_Monolignol_Pathway']` are not defined. No namespaces are defined for the target.\n - Data generator `a_0_4_task3` is invalid.\n - Variable `p1_a_0_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='a_0']` are not defined. No namespaces are defined for the target.\n - Data generator `V_5_Phenylalanine_To_Protein_5_task3` is invalid.\n - Variable `p1_V_5_Phenylalanine_To_Protein_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='V_5_Phenylalanine_To_Protein']` are not defined. No namespaces are defined for the target.\n - Data generator `a_0_5_task3` is invalid.\n - Variable `p1_a_0_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='a_0']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000848](https://www.ebi.ac.uk/biomodels/BIOMD0000000848)<br/><sup>FatehiChenar2018 - Mathematica</sup></summary><sup>FatehiChenar2018 - Mathematical model of immune response to hepatitis B</sup></details>|<details><summary>pass</summary>[FatehiChenar2018.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000848#Files)</details>|pass|<details><summary>pass</summary>[FatehiChenar2018.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000848#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/FatehiChenar2018.sedml507003.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `FatehiChenar2018.xml` has warnings.\r\n - 21 warnings of type SBML unit consistency (99505). The following is the first warning at line 1067, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Uninfected_Death(d, T)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 28 warnings of type Modeling practice (80701). The following is the first warning at line 1009, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'd' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./FatehiChenar2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `FatehiChenar2018.xml` has warnings.\r\n - 21 warnings of type SBML unit consistency (99505). The following is the first warning at line 1067, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Uninfected_Death(d, T)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 28 warnings of type Modeling practice (80701). The following is the first warning at line 1009, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'd' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/FatehiChenar2018.sedml507003.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./FatehiChenar2018.sedml` is invalid.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `I_1_task1` is invalid.\n - Variable `p2_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `F_1_1_task1` is invalid.\n - Variable `p3_F_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='F_1']` are not defined. No namespaces are defined for the target.\n - Data generator `F_2_1_task1` is invalid.\n - Variable `p4_F_2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='F_2']` are not defined. No namespaces are defined for the target.\n - Data generator `R_1_task1` is invalid.\n - Variable `p5_R_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='R']` are not defined. No namespaces are defined for the target.\n - Data generator `A_1_task1` is invalid.\n - Variable `p6_A_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='A']` are not defined. No namespaces are defined for the target.\n - Data generator `N_1_task1` is invalid.\n - Variable `p7_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n - Data generator `E_1_task1` is invalid.\n - Variable `p8_E_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='E']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/FatehiChenar2018.sedml480141.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `FatehiChenar2018.xml` has warnings.\r\n - 21 warnings of type SBML unit consistency (99505). The following is the first warning at line 1067, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Uninfected_Death(d, T)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 28 warnings of type Modeling practice (80701). The following is the first warning at line 1009, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'd' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./FatehiChenar2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `FatehiChenar2018.xml` has warnings.\r\n - 21 warnings of type SBML unit consistency (99505). The following is the first warning at line 1067, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Uninfected_Death(d, T)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 28 warnings of type Modeling practice (80701). The following is the first warning at line 1009, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'd' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/FatehiChenar2018.sedml480141.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./FatehiChenar2018.sedml` is invalid.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `I_1_task1` is invalid.\n - Variable `p2_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `F_1_1_task1` is invalid.\n - Variable `p3_F_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='F_1']` are not defined. No namespaces are defined for the target.\n - Data generator `F_2_1_task1` is invalid.\n - Variable `p4_F_2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='F_2']` are not defined. No namespaces are defined for the target.\n - Data generator `R_1_task1` is invalid.\n - Variable `p5_R_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='R']` are not defined. No namespaces are defined for the target.\n - Data generator `A_1_task1` is invalid.\n - Variable `p6_A_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='A']` are not defined. No namespaces are defined for the target.\n - Data generator `N_1_task1` is invalid.\n - Variable `p7_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n - Data generator `E_1_task1` is invalid.\n - Variable `p8_E_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='E']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000849](https://www.ebi.ac.uk/biomodels/BIOMD0000000849)<br/><sup>Potassium balance in lactating</sup></summary><sup>Potassium balance in lactating and non-lactating dairy cows</sup></details>|<details><summary>FAIL</summary>[Berg2017.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000849#Files)</details>|FAIL|<details><summary>pass</summary>[Berg2017.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000849#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Berg2017.sedml330846.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Berg2017.sedml` has warnings.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n`/root/in/Berg2017.sedml330846.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Berg2017.sedml` is invalid.\n - Model `model` is invalid.\n - The model file `Berg2017.xml` is invalid.\n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout' cannot be used in this document.\n \n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout_L2' cannot be used in this document.\n \n - General SBML conformance (10102) at line 3, column 2: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L2V4 Section 4.1\n Element 'listOfLayouts' is not part of the definition of SBML Level 2 Version 4.\n \n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Berg2017.sedml251249.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Berg2017.sedml` has warnings.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Berg2017.sedml251249.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Berg2017.sedml` is invalid.\n - Model `model` is invalid.\n - The model file `Berg2017.xml` is invalid.\n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout' cannot be used in this document.\n \n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout_L2' cannot be used in this document.\n \n - General SBML conformance (10102) at line 3, column 2: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L2V4 Section 4.1\n Element 'listOfLayouts' is not part of the definition of SBML Level 2 Version 4.\n \n"```</details>|
|<details><summary>[BIOMD0000000850](https://www.ebi.ac.uk/biomodels/BIOMD0000000850)<br/><sup>Jenner2019 - Oncolytic virothe</sup></summary><sup>Jenner2019 - Oncolytic virotherapy for tumours following a Gompertz growth law</sup></details>|<details><summary>pass</summary>[Jenner2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000850#Files)</details>|FAIL|<details><summary>pass</summary>[Jenner2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000850#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Jenner2019.sedml949178.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Jenner2019.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 324, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth(m, K, U)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 373, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Conversion_Uninfected_to_Infected') are item (exponent = 1, multiplier = 1, scale = 0).\r\n - 4 warnings of type Modeling practice (80701). The following is the first warning at line 290, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'm' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Jenner2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Jenner2019.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 324, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth(m, K, U)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 373, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Conversion_Uninfected_to_Infected') are item (exponent = 1, multiplier = 1, scale = 0).\r\n - 4 warnings of type Modeling practice (80701). The following is the first warning at line 290, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'm' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Jenner2019.sedml949178.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Jenner2019.sedml` is invalid.\n - Data generator `I_1_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `U_2_task1` is invalid.\n - Variable `p1_U_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='U']` are not defined. No namespaces are defined for the target.\n - Data generator `V_3_task1` is invalid.\n - Variable `p1_V_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Jenner2019.sedml968030.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Jenner2019.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 324, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth(m, K, U)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 373, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Conversion_Uninfected_to_Infected') are item (exponent = 1, multiplier = 1, scale = 0).\r\n - 4 warnings of type Modeling practice (80701). The following is the first warning at line 290, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'm' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Jenner2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Jenner2019.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 324, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth(m, K, U)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 373, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Conversion_Uninfected_to_Infected') are item (exponent = 1, multiplier = 1, scale = 0).\r\n - 4 warnings of type Modeling practice (80701). The following is the first warning at line 290, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'm' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Jenner2019.sedml968030.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Jenner2019.sedml` is invalid.\n - Data generator `I_1_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `U_2_task1` is invalid.\n - Variable `p1_U_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='U']` are not defined. No namespaces are defined for the target.\n - Data generator `V_3_task1` is invalid.\n - Variable `p1_V_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000851](https://www.ebi.ac.uk/biomodels/BIOMD0000000851)<br/><sup>Ho2019 - Mathematical models o</sup></summary><sup>Ho2019 - Mathematical models of transmission dynamics and vaccine strategies in Hong Kong during the 2017-2018 winter influenza season (Simple)</sup></details>|<details><summary>pass</summary>[Ho2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000851#Files)</details>|pass|<details><summary>pass</summary>[Ho2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000851#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Ho2019.sedml393143.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Ho2019.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (99505). The following is the first warning at line 514, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'beta' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 17 warnings of type Modeling practice (80701). The following is the first warning at line 403, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 11.0\r\n - Number of steps: 11\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Ho2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Ho2019.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (99505). The following is the first warning at line 514, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'beta' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 17 warnings of type Modeling practice (80701). The following is the first warning at line 403, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 11.0\r\n - Number of steps: 11\x1b[0m\r\n\r\n`/root/in/Ho2019.sedml393143.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Ho2019.sedml` is invalid.\n - Data generator `I_1_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `R_2_task1` is invalid.\n - Variable `p1_R_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='R']` are not defined. No namespaces are defined for the target.\n - Data generator `S_3_task1` is invalid.\n - Variable `p1_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S']` are not defined. No namespaces are defined for the target.\n - Data generator `V_4_task1` is invalid.\n - Variable `p1_V_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V']` are not defined. No namespaces are defined for the target.\n - Data generator `V_e_5_task1` is invalid.\n - Variable `p1_V_e_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V_e']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Ho2019.sedml252175.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Ho2019.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (99505). The following is the first warning at line 514, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'beta' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 17 warnings of type Modeling practice (80701). The following is the first warning at line 403, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 11.0\r\n - Number of steps: 11\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Ho2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Ho2019.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (99505). The following is the first warning at line 514, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'beta' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 17 warnings of type Modeling practice (80701). The following is the first warning at line 403, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 11.0\r\n - Number of steps: 11\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Ho2019.sedml252175.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Ho2019.sedml` is invalid.\n - Data generator `I_1_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `R_2_task1` is invalid.\n - Variable `p1_R_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='R']` are not defined. No namespaces are defined for the target.\n - Data generator `S_3_task1` is invalid.\n - Variable `p1_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S']` are not defined. No namespaces are defined for the target.\n - Data generator `V_4_task1` is invalid.\n - Variable `p1_V_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V']` are not defined. No namespaces are defined for the target.\n - Data generator `V_e_5_task1` is invalid.\n - Variable `p1_V_e_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V_e']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000852](https://www.ebi.ac.uk/biomodels/BIOMD0000000852)<br/><sup>Andersen2017 - Mathematical mo</sup></summary><sup>Andersen2017 - Mathematical modelling as a proof of concept for MPNs as a human inflammation model for cancer development</sup></details>|<details><summary>pass</summary>[Andersen2017.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000852#Files)</details>|pass|<details><summary>pass</summary>[Andersen2017.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000852#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Andersen2017.sedml937845.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Andersen2017.xml` has warnings.\r\n - 38 warnings of type SBML unit consistency (99505). The following is the first warning at line 924, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Ax' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 35 warnings of type Modeling practice (80701). The following is the first warning at line 589, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'rx' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Andersen2017.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Andersen2017.xml` has warnings.\r\n - 38 warnings of type SBML unit consistency (99505). The following is the first warning at line 924, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Ax' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 35 warnings of type Modeling practice (80701). The following is the first warning at line 589, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'rx' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Andersen2017.sedml937845.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Andersen2017.sedml` is invalid.\n - Data generator `x0_e4_1_task1` is invalid.\n - Variable `p1_x0_e4_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='x0_e4']` are not defined. No namespaces are defined for the target.\n - Data generator `x0_y0_2_task1` is invalid.\n - Variable `p1_x0_y0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='x0_y0']` are not defined. No namespaces are defined for the target.\n - Data generator `y0_e4_3_task1` is invalid.\n - Variable `p1_y0_e4_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='y0_e4']` are not defined. No namespaces are defined for the target.\n - Data generator `x1_e10_1_task1` is invalid.\n - Variable `p2_x1_e10_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='x1_e10']` are not defined. No namespaces are defined for the target.\n - Data generator `x1_y1_2_task1` is invalid.\n - Variable `p2_x1_y1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='x1_y1']` are not defined. No namespaces are defined for the target.\n - Data generator `y1_e10_3_task1` is invalid.\n - Variable `p2_y1_e10_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='y1_e10']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Andersen2017.sedml78686.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Andersen2017.xml` has warnings.\r\n - 38 warnings of type SBML unit consistency (99505). The following is the first warning at line 924, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Ax' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 35 warnings of type Modeling practice (80701). The following is the first warning at line 589, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'rx' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Andersen2017.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Andersen2017.xml` has warnings.\r\n - 38 warnings of type SBML unit consistency (99505). The following is the first warning at line 924, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Ax' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 35 warnings of type Modeling practice (80701). The following is the first warning at line 589, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'rx' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Andersen2017.sedml78686.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Andersen2017.sedml` is invalid.\n - Data generator `x0_e4_1_task1` is invalid.\n - Variable `p1_x0_e4_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='x0_e4']` are not defined. No namespaces are defined for the target.\n - Data generator `x0_y0_2_task1` is invalid.\n - Variable `p1_x0_y0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='x0_y0']` are not defined. No namespaces are defined for the target.\n - Data generator `y0_e4_3_task1` is invalid.\n - Variable `p1_y0_e4_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='y0_e4']` are not defined. No namespaces are defined for the target.\n - Data generator `x1_e10_1_task1` is invalid.\n - Variable `p2_x1_e10_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='x1_e10']` are not defined. No namespaces are defined for the target.\n - Data generator `x1_y1_2_task1` is invalid.\n - Variable `p2_x1_y1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='x1_y1']` are not defined. No namespaces are defined for the target.\n - Data generator `y1_e10_3_task1` is invalid.\n - Variable `p2_y1_e10_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='y1_e10']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000853](https://www.ebi.ac.uk/biomodels/BIOMD0000000853)<br/><sup>Smolen2018 - Paradoxical LTP m</sup></summary><sup>Smolen2018 - Paradoxical LTP maintenance with inhibition of protein synthesis and the proteasome</sup></details>|<details><summary>pass</summary>[Smolen2018.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000853#Files)</details>|pass|<details><summary>pass</summary>[Smolen2018.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000853#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Smolen2018.sedml983592.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Smolen2018.xml` has warnings.\r\n - 26 warnings of type SBML unit consistency (99505). The following is the first warning at line 1094, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'kf1bas_value' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 31 warnings of type Modeling practice (80701). The following is the first warning at line 913, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'ksyn1' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Smolen2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Smolen2018.xml` has warnings.\r\n - 26 warnings of type SBML unit consistency (99505). The following is the first warning at line 1094, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'kf1bas_value' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 31 warnings of type Modeling practice (80701). The following is the first warning at line 913, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'ksyn1' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Smolen2018.sedml983592.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Smolen2018.sedml` is invalid.\n - Data generator `STIM_1_task1` is invalid.\n - Variable `p1_STIM_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='STIM']` are not defined. No namespaces are defined for the target.\n - Data generator `BAS_2_task1` is invalid.\n - Variable `p1_BAS_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='BAS']` are not defined. No namespaces are defined for the target.\n - Data generator `EP1_3_task1` is invalid.\n - Variable `p1_EP1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='EP1']` are not defined. No namespaces are defined for the target.\n - Data generator `EP2_4_task1` is invalid.\n - Variable `p1_EP2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='EP2']` are not defined. No namespaces are defined for the target.\n - Data generator `NP_1_task1` is invalid.\n - Variable `p2_NP_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='NP']` are not defined. No namespaces are defined for the target.\n - Data generator `PP_2_task1` is invalid.\n - Variable `p2_PP_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PP']` are not defined. No namespaces are defined for the target.\n - Data generator `STAB_3_task1` is invalid.\n - Variable `p2_STAB_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAB']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Smolen2018.sedml834749.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Smolen2018.xml` has warnings.\r\n - 26 warnings of type SBML unit consistency (99505). The following is the first warning at line 1094, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'kf1bas_value' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 31 warnings of type Modeling practice (80701). The following is the first warning at line 913, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'ksyn1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Smolen2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Smolen2018.xml` has warnings.\r\n - 26 warnings of type SBML unit consistency (99505). The following is the first warning at line 1094, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'kf1bas_value' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 31 warnings of type Modeling practice (80701). The following is the first warning at line 913, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'ksyn1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Smolen2018.sedml834749.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Smolen2018.sedml` is invalid.\n - Data generator `STIM_1_task1` is invalid.\n - Variable `p1_STIM_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='STIM']` are not defined. No namespaces are defined for the target.\n - Data generator `BAS_2_task1` is invalid.\n - Variable `p1_BAS_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='BAS']` are not defined. No namespaces are defined for the target.\n - Data generator `EP1_3_task1` is invalid.\n - Variable `p1_EP1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='EP1']` are not defined. No namespaces are defined for the target.\n - Data generator `EP2_4_task1` is invalid.\n - Variable `p1_EP2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='EP2']` are not defined. No namespaces are defined for the target.\n - Data generator `NP_1_task1` is invalid.\n - Variable `p2_NP_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='NP']` are not defined. No namespaces are defined for the target.\n - Data generator `PP_2_task1` is invalid.\n - Variable `p2_PP_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PP']` are not defined. No namespaces are defined for the target.\n - Data generator `STAB_3_task1` is invalid.\n - Variable `p2_STAB_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAB']` are not defined. No namespaces are defined for the target.\n"```</details>|
|[BIOMD0000000854](https://www.ebi.ac.uk/biomodels/BIOMD0000000854)<br/><sup>Gray2016 - The Akt switch model</sup>|<details><summary>pass</summary>[Gray2016.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000854#Files)</details>|pass|<details><summary>pass</summary>[Gray2016.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000854#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Gray2016.sedml709956.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Gray2016.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 505, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '100 * (Pc / Akt_Total)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type Modeling practice (80701). The following is the first warning at line 399, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kin' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Gray2016.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Gray2016.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 505, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '100 * (Pc / Akt_Total)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type Modeling practice (80701). The following is the first warning at line 399, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kin' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Gray2016.sedml709956.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Gray2016.sedml` is invalid.\n - Data generator `Pc_Perc_1_task1` is invalid.\n - Variable `p1_Pc_Perc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Pc_Perc']` are not defined. No namespaces are defined for the target.\n - Data generator `Pp_Perc_2_task1` is invalid.\n - Variable `p1_Pp_Perc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Pp_Perc']` are not defined. No namespaces are defined for the target.\n - Data generator `P_Perc_3_task1` is invalid.\n - Variable `p1_P_Perc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='P_Perc']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Gray2016.sedml764071.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Gray2016.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 505, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '100 * (Pc / Akt_Total)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type Modeling practice (80701). The following is the first warning at line 399, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kin' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Gray2016.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Gray2016.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 505, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '100 * (Pc / Akt_Total)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type Modeling practice (80701). The following is the first warning at line 399, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kin' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Gray2016.sedml764071.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Gray2016.sedml` is invalid.\n - Data generator `Pc_Perc_1_task1` is invalid.\n - Variable `p1_Pc_Perc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Pc_Perc']` are not defined. No namespaces are defined for the target.\n - Data generator `Pp_Perc_2_task1` is invalid.\n - Variable `p1_Pp_Perc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Pp_Perc']` are not defined. No namespaces are defined for the target.\n - Data generator `P_Perc_3_task1` is invalid.\n - Variable `p1_P_Perc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='P_Perc']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000855](https://www.ebi.ac.uk/biomodels/BIOMD0000000855)<br/><sup>Cooper2015 - Modeling the effe</sup></summary><sup>Cooper2015 - Modeling the effects of systemic mediators on the inflammatory phase of wound healing</sup></details>|<details><summary>pass</summary>[Cooper2015.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000855#Files)</details>|pass|<details><summary>pass</summary>[Cooper2015.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000855#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Cooper2015.sedml13976.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Cooper2015.xml` has warnings.\r\n - 17 warnings of type SBML unit consistency (99505). The following is the first warning at line 1256, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'kmm / (1 + pow(E / EMinf, 2))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 39 warnings of type Modeling practice (80701). The following is the first warning at line 669, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kpn' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Cooper2015.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Cooper2015.xml` has warnings.\r\n - 17 warnings of type SBML unit consistency (99505). The following is the first warning at line 1256, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'kmm / (1 + pow(E / EMinf, 2))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 39 warnings of type Modeling practice (80701). The following is the first warning at line 669, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kpn' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Cooper2015.sedml13976.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Cooper2015.sedml` is invalid.\n - Data generator `Pt_1_task1` is invalid.\n - Variable `p1_Pt_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Pt']` are not defined. No namespaces are defined for the target.\n - Data generator `M_2_task1` is invalid.\n - Variable `p1_M_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M']` are not defined. No namespaces are defined for the target.\n - Data generator `N_3_task1` is invalid.\n - Variable `p1_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Cooper2015.sedml400557.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Cooper2015.xml` has warnings.\r\n - 17 warnings of type SBML unit consistency (99505). The following is the first warning at line 1256, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'kmm / (1 + pow(E / EMinf, 2))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 39 warnings of type Modeling practice (80701). The following is the first warning at line 669, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kpn' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Cooper2015.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Cooper2015.xml` has warnings.\r\n - 17 warnings of type SBML unit consistency (99505). The following is the first warning at line 1256, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'kmm / (1 + pow(E / EMinf, 2))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 39 warnings of type Modeling practice (80701). The following is the first warning at line 669, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kpn' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Cooper2015.sedml400557.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Cooper2015.sedml` is invalid.\n - Data generator `Pt_1_task1` is invalid.\n - Variable `p1_Pt_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Pt']` are not defined. No namespaces are defined for the target.\n - Data generator `M_2_task1` is invalid.\n - Variable `p1_M_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M']` are not defined. No namespaces are defined for the target.\n - Data generator `N_3_task1` is invalid.\n - Variable `p1_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000856](https://www.ebi.ac.uk/biomodels/BIOMD0000000856)<br/><sup>Budding yeast size control by </sup></summary><sup>Budding yeast size control by titration of nuclear sites</sup></details>|<details><summary>pass</summary>[Heldt2018.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000856#Files)</details>|FAIL|<details><summary>pass</summary>[Heldt2018.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000856#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Heldt2018.sedml798880.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Heldt2018.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (10501). The following is the first warning at line 2732, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'kAsGiTm * (TM / tV) * (GI / tV) - kDsGiTm * (GITM / tV)' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 47 warnings of type SBML unit consistency (99505). The following is the first warning at line 2335, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression '0.66 * NSt' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 25 warnings of type SBML unit consistency (10541). The following is the first warning at line 2691, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 60, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Synthesis_of_transcription_machninery') are second (exponent = -1, multiplier = 0.06, scale = 0), item (exponent = 1, multiplier = 1, scale = 0).\r\n - 3 warnings of type Modeling practice (80701). The following is the first warning at line 1493, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'GRm' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Heldt2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Heldt2018.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (10501). The following is the first warning at line 2732, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'kAsGiTm * (TM / tV) * (GI / tV) - kDsGiTm * (GITM / tV)' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 47 warnings of type SBML unit consistency (99505). The following is the first warning at line 2335, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression '0.66 * NSt' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 25 warnings of type SBML unit consistency (10541). The following is the first warning at line 2691, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 60, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Synthesis_of_transcription_machninery') are second (exponent = -1, multiplier = 0.06, scale = 0), item (exponent = 1, multiplier = 1, scale = 0).\r\n - 3 warnings of type Modeling practice (80701). The following is the first warning at line 1493, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'GRm' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Heldt2018.sedml798880.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Heldt2018.sedml` is invalid.\n - Data generator `CLN3t_1_task1` is invalid.\n - Variable `p1_CLN3t_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CLN3t']` are not defined. No namespaces are defined for the target.\n - Data generator `WHIt_2_task1` is invalid.\n - Variable `p1_WHIt_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='WHIt']` are not defined. No namespaces are defined for the target.\n - Data generator `active_SBF_3_task1` is invalid.\n - Variable `p1_active_SBF_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='active_SBF']` are not defined. No namespaces are defined for the target.\n - Data generator `CLB_1_task1` is invalid.\n - Variable `p2_CLB_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CLB']` are not defined. No namespaces are defined for the target.\n - Data generator `CLN_2_task1` is invalid.\n - Variable `p2_CLN_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CLN']` are not defined. No namespaces are defined for the target.\n - Data generator `CDHa_3_task1` is invalid.\n - Variable `p2_CDHa_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CDHa']` are not defined. No namespaces are defined for the target.\n - Data generator `dV_1_task1` is invalid.\n - Variable `p3_dV_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfCompartments/sbml:compartment[@id='dV']` are not defined. No namespaces are defined for the target.\n - Data generator `mV_2_task1` is invalid.\n - Variable `p3_mV_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfCompartments/sbml:compartment[@id='mV']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Heldt2018.sedml916209.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Heldt2018.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (10501). The following is the first warning at line 2732, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'kAsGiTm * (TM / tV) * (GI / tV) - kDsGiTm * (GITM / tV)' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 47 warnings of type SBML unit consistency (99505). The following is the first warning at line 2335, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression '0.66 * NSt' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 25 warnings of type SBML unit consistency (10541). The following is the first warning at line 2691, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 60, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Synthesis_of_transcription_machninery') are second (exponent = -1, multiplier = 0.06, scale = 0), item (exponent = 1, multiplier = 1, scale = 0).\r\n - 3 warnings of type Modeling practice (80701). The following is the first warning at line 1493, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'GRm' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Heldt2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Heldt2018.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (10501). The following is the first warning at line 2732, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'kAsGiTm * (TM / tV) * (GI / tV) - kDsGiTm * (GITM / tV)' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 47 warnings of type SBML unit consistency (99505). The following is the first warning at line 2335, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression '0.66 * NSt' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 25 warnings of type SBML unit consistency (10541). The following is the first warning at line 2691, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 60, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Synthesis_of_transcription_machninery') are second (exponent = -1, multiplier = 0.06, scale = 0), item (exponent = 1, multiplier = 1, scale = 0).\r\n - 3 warnings of type Modeling practice (80701). The following is the first warning at line 1493, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'GRm' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Heldt2018.sedml916209.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Heldt2018.sedml` is invalid.\n - Data generator `CLN3t_1_task1` is invalid.\n - Variable `p1_CLN3t_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CLN3t']` are not defined. No namespaces are defined for the target.\n - Data generator `WHIt_2_task1` is invalid.\n - Variable `p1_WHIt_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='WHIt']` are not defined. No namespaces are defined for the target.\n - Data generator `active_SBF_3_task1` is invalid.\n - Variable `p1_active_SBF_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='active_SBF']` are not defined. No namespaces are defined for the target.\n - Data generator `CLB_1_task1` is invalid.\n - Variable `p2_CLB_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CLB']` are not defined. No namespaces are defined for the target.\n - Data generator `CLN_2_task1` is invalid.\n - Variable `p2_CLN_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CLN']` are not defined. No namespaces are defined for the target.\n - Data generator `CDHa_3_task1` is invalid.\n - Variable `p2_CDHa_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CDHa']` are not defined. No namespaces are defined for the target.\n - Data generator `dV_1_task1` is invalid.\n - Variable `p3_dV_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfCompartments/sbml:compartment[@id='dV']` are not defined. No namespaces are defined for the target.\n - Data generator `mV_2_task1` is invalid.\n - Variable `p3_mV_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfCompartments/sbml:compartment[@id='mV']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000857](https://www.ebi.ac.uk/biomodels/BIOMD0000000857)<br/><sup>Larbat2016.1 - Modeling the di</sup></summary><sup>Larbat2016.1 - Modeling the diversion of primary carbon flux into secondary metabolism under variable nitrate and light or dark conditions (Base Model)</sup></details>|<details><summary>pass</summary>[Larbat2016.1.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000857#Files)</details>|pass|<details><summary>pass</summary>[Larbat2016.1.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000857#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Larbat2016.1.sedml448984.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Larbat2016.1.xml` has warnings.\r\n - 24 warnings of type SBML unit consistency (99505). The following is the first warning at line 1140, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'N / (k13 + N)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 34 warnings of type Modeling practice (80701). The following is the first warning at line 992, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Larbat2016.1.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Larbat2016.1.xml` has warnings.\r\n - 24 warnings of type SBML unit consistency (99505). The following is the first warning at line 1140, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'N / (k13 + N)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 34 warnings of type Modeling practice (80701). The following is the first warning at line 992, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Larbat2016.1.sedml448984.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Larbat2016.1.sedml` is invalid.\n - Data generator `sucr_1_task1` is invalid.\n - Variable `p1_sucr_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='sucr']` are not defined. No namespaces are defined for the target.\n - Data generator `trioseP_2_task1` is invalid.\n - Variable `p1_trioseP_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='trioseP']` are not defined. No namespaces are defined for the target.\n - Data generator `N_1_task1` is invalid.\n - Variable `p2_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n - Data generator `Next_2_task1` is invalid.\n - Variable `p2_Next_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Next']` are not defined. No namespaces are defined for the target.\n - Data generator `starch_3_task1` is invalid.\n - Variable `p2_starch_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='starch']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Larbat2016.1.sedml110337.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Larbat2016.1.xml` has warnings.\r\n - 24 warnings of type SBML unit consistency (99505). The following is the first warning at line 1140, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'N / (k13 + N)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 34 warnings of type Modeling practice (80701). The following is the first warning at line 992, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Larbat2016.1.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Larbat2016.1.xml` has warnings.\r\n - 24 warnings of type SBML unit consistency (99505). The following is the first warning at line 1140, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'N / (k13 + N)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 34 warnings of type Modeling practice (80701). The following is the first warning at line 992, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Larbat2016.1.sedml110337.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Larbat2016.1.sedml` is invalid.\n - Data generator `sucr_1_task1` is invalid.\n - Variable `p1_sucr_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='sucr']` are not defined. No namespaces are defined for the target.\n - Data generator `trioseP_2_task1` is invalid.\n - Variable `p1_trioseP_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='trioseP']` are not defined. No namespaces are defined for the target.\n - Data generator `N_1_task1` is invalid.\n - Variable `p2_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n - Data generator `Next_2_task1` is invalid.\n - Variable `p2_Next_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Next']` are not defined. No namespaces are defined for the target.\n - Data generator `starch_3_task1` is invalid.\n - Variable `p2_starch_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='starch']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000858](https://www.ebi.ac.uk/biomodels/BIOMD0000000858)<br/><sup>Larbat2016.2 - Modeling the di</sup></summary><sup>Larbat2016.2 - Modeling the diversion of primary carbon flux into secondary metabolism under variable nitrate and light or dark conditions (Light Dark Cycles)</sup></details>|<details><summary>pass</summary>[Larbat2016.2.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000858#Files)</details>|pass|<details><summary>FAIL</summary>[Larbat2016.2.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000858#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Larbat2016.2.sedml423125.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/Larbat2016.2.sedml423125.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Larbat2016.2.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmpviws58s1/./model1.xml` is not a file.\n - Data generator `starch_1_task1` is invalid.\n - Variable `p1_starch_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='starch']` are not defined. No namespaces are defined for the target.\n - Data generator `sucr_1_task1` is invalid.\n - Variable `p2_sucr_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='sucr']` are not defined. No namespaces are defined for the target.\n - Data generator `trioseP_2_task1` is invalid.\n - Variable `p2_trioseP_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='trioseP']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Larbat2016.2.sedml832644.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Larbat2016.2.sedml832644.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Larbat2016.2.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmps7gs4bwb/./model1.xml` is not a file.\n - Data generator `starch_1_task1` is invalid.\n - Variable `p1_starch_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='starch']` are not defined. No namespaces are defined for the target.\n - Data generator `sucr_1_task1` is invalid.\n - Variable `p2_sucr_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='sucr']` are not defined. No namespaces are defined for the target.\n - Data generator `trioseP_2_task1` is invalid.\n - Variable `p2_trioseP_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='trioseP']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000859](https://www.ebi.ac.uk/biomodels/BIOMD0000000859)<br/><sup>Larbat2016.3 - Modeling the di</sup></summary><sup>Larbat2016.3 - Modeling the diversion of primary carbon flux into secondary metabolism under variable nitrate and light or dark conditions (Light Dark Cycles with Minimum Starch Adaption)</sup></details>|<details><summary>pass</summary>[Larbat2016.3.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000859#Files)</details>|pass|<details><summary>FAIL</summary>[Larbat2016.3.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000859#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Larbat2016.3.sedml694398.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Larbat2016.3.sedml` has warnings.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/Larbat2016.3.sedml694398.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Larbat2016.3.sedml` is invalid.\n - Model `model2` is invalid.\n - The model file `model2.xml` is invalid.\n - `/tmp/tmp583zp_is/./model2.xml` is not a file.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='k56']` are not defined. No namespaces are defined for the target.\n - Data generator `starch_1_task1` is invalid.\n - Variable `p1_starch_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='starch']` are not defined. No namespaces are defined for the target.\n - Data generator `k29_2_task1` is invalid.\n - Variable `p1_k29_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='k29']` are not defined. No namespaces are defined for the target.\n - Data generator `starch_1_task2` is invalid.\n - Variable `p1_starch_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='starch']` are not defined. No namespaces are defined for the target.\n - Data generator `k29_2_task2` is invalid.\n - Variable `p1_k29_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='k29']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Larbat2016.3.sedml780487.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Larbat2016.3.sedml` has warnings.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Larbat2016.3.sedml780487.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Larbat2016.3.sedml` is invalid.\n - Model `model2` is invalid.\n - The model file `model2.xml` is invalid.\n - `/tmp/tmpuot01bjy/./model2.xml` is not a file.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='k56']` are not defined. No namespaces are defined for the target.\n - Data generator `starch_1_task1` is invalid.\n - Variable `p1_starch_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='starch']` are not defined. No namespaces are defined for the target.\n - Data generator `k29_2_task1` is invalid.\n - Variable `p1_k29_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='k29']` are not defined. No namespaces are defined for the target.\n - Data generator `starch_1_task2` is invalid.\n - Variable `p1_starch_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='starch']` are not defined. No namespaces are defined for the target.\n - Data generator `k29_2_task2` is invalid.\n - Variable `p1_k29_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='k29']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000860](https://www.ebi.ac.uk/biomodels/BIOMD0000000860)<br/><sup>Proctor2017- Role of microRNAs</sup></summary><sup>Proctor2017- Role of microRNAs in osteoarthritis (Positive Feedforward Incoherent By MicroRNA)_1</sup></details>|<details><summary>pass</summary>[proctor2017_model4.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000860#Files)</details>|FAIL|<details><summary>pass</summary>[proctor2017_model4.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000860#Files)</details>|FAIL|pass|pass|
|<details><summary>[BIOMD0000000861](https://www.ebi.ac.uk/biomodels/BIOMD0000000861)<br/><sup>Bachmann2011 - Division of lab</sup></summary><sup>Bachmann2011 - Division of labor by dual feedback regulators controls JAK2/STAT5 signaling over broad ligand range</sup></details>|<details><summary>pass</summary>[Bachmann2011.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000861#Files)</details>|pass|<details><summary>pass</summary>[Bachmann2011.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000861#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Bachmann2011.sedml727167.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Bachmann2011.xml` has warnings.\r\n - 36 warnings of type SBML unit consistency (99505). The following is the first warning at line 2047, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'cyt * Function_for_reaction_1(Epo, EpoRJAK2, JAK2ActEpo, SOCS3, SOCS3Eqc, SOCS3Inh)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 32 warnings of type Modeling practice (80701). The following is the first warning at line 1964, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'ActD' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Bachmann2011.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Bachmann2011.xml` has warnings.\r\n - 36 warnings of type SBML unit consistency (99505). The following is the first warning at line 2047, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'cyt * Function_for_reaction_1(Epo, EpoRJAK2, JAK2ActEpo, SOCS3, SOCS3Eqc, SOCS3Inh)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 32 warnings of type Modeling practice (80701). The following is the first warning at line 1964, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'ActD' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Bachmann2011.sedml727167.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Bachmann2011.sedml` is invalid.\n - Data generator `npSTAT5_1_task1` is invalid.\n - Variable `p1_npSTAT5_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='npSTAT5']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Bachmann2011.sedml969519.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Bachmann2011.xml` has warnings.\r\n - 36 warnings of type SBML unit consistency (99505). The following is the first warning at line 2047, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'cyt * Function_for_reaction_1(Epo, EpoRJAK2, JAK2ActEpo, SOCS3, SOCS3Eqc, SOCS3Inh)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 32 warnings of type Modeling practice (80701). The following is the first warning at line 1964, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'ActD' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Bachmann2011.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Bachmann2011.xml` has warnings.\r\n - 36 warnings of type SBML unit consistency (99505). The following is the first warning at line 2047, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'cyt * Function_for_reaction_1(Epo, EpoRJAK2, JAK2ActEpo, SOCS3, SOCS3Eqc, SOCS3Inh)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 32 warnings of type Modeling practice (80701). The following is the first warning at line 1964, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'ActD' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Bachmann2011.sedml969519.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Bachmann2011.sedml` is invalid.\n - Data generator `npSTAT5_1_task1` is invalid.\n - Variable `p1_npSTAT5_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='npSTAT5']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000862](https://www.ebi.ac.uk/biomodels/BIOMD0000000862)<br/><sup>Proctor2017- Role of microRNAs</sup></summary><sup>Proctor2017- Role of microRNAs in osteoarthritis (Positive Feedback By Micro RNA)</sup></details>|<details><summary>pass</summary>[Proctor2017_model0.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000862#Files)</details>|pass|<details><summary>pass</summary>[Proctor2017_model0.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000862#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Proctor2017_model0.sedml408159.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Proctor2017_model0.xml` has warnings.\r\n - 17 warnings of type SBML unit consistency (99505). The following is the first warning at line 497, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'cell * Function_for_miR_gene_TF1_binding(TF1, cell, kbinTF1miRgene, miR_gene)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 476, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kbinTF1miRgene' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Proctor2017_model0.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Proctor2017_model0.xml` has warnings.\r\n - 17 warnings of type SBML unit consistency (99505). The following is the first warning at line 497, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'cell * Function_for_miR_gene_TF1_binding(TF1, cell, kbinTF1miRgene, miR_gene)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 476, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kbinTF1miRgene' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Proctor2017_model0.sedml408159.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Proctor2017_model0.sedml` is invalid.\n - Data generator `miR_1_task1` is invalid.\n - Variable `p1_miR_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='miR']` are not defined. No namespaces are defined for the target.\n - Data generator `miR_gene_2_task1` is invalid.\n - Variable `p1_miR_gene_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='miR_gene']` are not defined. No namespaces are defined for the target.\n - Data generator `miR_gene_TF1_3_task1` is invalid.\n - Variable `p1_miR_gene_TF1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='miR_gene_TF1']` are not defined. No namespaces are defined for the target.\n - Data generator `miR_gene_TF2_4_task1` is invalid.\n - Variable `p1_miR_gene_TF2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='miR_gene_TF2']` are not defined. No namespaces are defined for the target.\n - Data generator `Signal_5_task1` is invalid.\n - Variable `p1_Signal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Signal']` are not defined. No namespaces are defined for the target.\n - Data generator `TF1_6_task1` is invalid.\n - Variable `p1_TF1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='TF1']` are not defined. No namespaces are defined for the target.\n - Data generator `TF1_mRNA_7_task1` is invalid.\n - Variable `p1_TF1_mRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='TF1_mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `TF2_8_task1` is invalid.\n - Variable `p1_TF2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='TF2']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Proctor2017_model0.sedml70496.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Proctor2017_model0.xml` has warnings.\r\n - 17 warnings of type SBML unit consistency (99505). The following is the first warning at line 497, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'cell * Function_for_miR_gene_TF1_binding(TF1, cell, kbinTF1miRgene, miR_gene)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 476, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kbinTF1miRgene' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Proctor2017_model0.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Proctor2017_model0.xml` has warnings.\r\n - 17 warnings of type SBML unit consistency (99505). The following is the first warning at line 497, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'cell * Function_for_miR_gene_TF1_binding(TF1, cell, kbinTF1miRgene, miR_gene)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 476, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kbinTF1miRgene' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Proctor2017_model0.sedml70496.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Proctor2017_model0.sedml` is invalid.\n - Data generator `miR_1_task1` is invalid.\n - Variable `p1_miR_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='miR']` are not defined. No namespaces are defined for the target.\n - Data generator `miR_gene_2_task1` is invalid.\n - Variable `p1_miR_gene_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='miR_gene']` are not defined. No namespaces are defined for the target.\n - Data generator `miR_gene_TF1_3_task1` is invalid.\n - Variable `p1_miR_gene_TF1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='miR_gene_TF1']` are not defined. No namespaces are defined for the target.\n - Data generator `miR_gene_TF2_4_task1` is invalid.\n - Variable `p1_miR_gene_TF2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='miR_gene_TF2']` are not defined. No namespaces are defined for the target.\n - Data generator `Signal_5_task1` is invalid.\n - Variable `p1_Signal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Signal']` are not defined. No namespaces are defined for the target.\n - Data generator `TF1_6_task1` is invalid.\n - Variable `p1_TF1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='TF1']` are not defined. No namespaces are defined for the target.\n - Data generator `TF1_mRNA_7_task1` is invalid.\n - Variable `p1_TF1_mRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='TF1_mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `TF2_8_task1` is invalid.\n - Variable `p1_TF2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='TF2']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000863](https://www.ebi.ac.uk/biomodels/BIOMD0000000863)<br/><sup>Kosinsky2018 - Radiation and P</sup></summary><sup>Kosinsky2018 - Radiation and PD-(L)1 treatment combinations</sup></details>|<details><summary>pass</summary>[Kosinsky2018.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000863#Files)</details>|FAIL|<details><summary>pass</summary>[Kosinsky2018.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000863#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Kosinsky2018.sedml651361.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Kosinsky2018.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (10501). The following is the first warning at line 1808, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'n_e * dTeff + d0' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 1629, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '0' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 1808, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -9), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Tumor_Elimination') are mole (exponent = 1, multiplier = 1, scale = -9).\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 1282, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'S_L' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Kosinsky2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Kosinsky2018.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (10501). The following is the first warning at line 1808, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'n_e * dTeff + d0' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 1629, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '0' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 1808, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -9), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Tumor_Elimination') are mole (exponent = 1, multiplier = 1, scale = -9).\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 1282, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'S_L' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/Kosinsky2018.sedml651361.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Kosinsky2018.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mAb']` are not defined. No namespaces are defined for the target.\n - Data generator `TV_1_task1` is invalid.\n - Variable `p1_TV_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='TV']` are not defined. No namespaces are defined for the target.\n - Data generator `TV_1_task2` is invalid.\n - Variable `p1_TV_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='TV']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Kosinsky2018.sedml832753.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Kosinsky2018.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (10501). The following is the first warning at line 1808, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'n_e * dTeff + d0' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 1629, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '0' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 1808, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -9), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Tumor_Elimination') are mole (exponent = 1, multiplier = 1, scale = -9).\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 1282, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'S_L' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Kosinsky2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Kosinsky2018.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (10501). The following is the first warning at line 1808, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'n_e * dTeff + d0' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 1629, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '0' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 1808, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -9), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Tumor_Elimination') are mole (exponent = 1, multiplier = 1, scale = -9).\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 1282, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'S_L' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Kosinsky2018.sedml832753.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Kosinsky2018.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mAb']` are not defined. No namespaces are defined for the target.\n - Data generator `TV_1_task1` is invalid.\n - Variable `p1_TV_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='TV']` are not defined. No namespaces are defined for the target.\n - Data generator `TV_1_task2` is invalid.\n - Variable `p1_TV_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='TV']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000864](https://www.ebi.ac.uk/biomodels/BIOMD0000000864)<br/><sup>Proctor2017- Role of microRNAs</sup></summary><sup>Proctor2017- Role of microRNAs in osteoarthritis (Negative Feedback By MicroRNA)</sup></details>|<details><summary>pass</summary>[Proctor2017_model1.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000864#Files)</details>|pass|<details><summary>pass</summary>[Proctor2017_model1.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000864#Files)</details>|FAIL|pass|pass|
|<details><summary>[BIOMD0000000865](https://www.ebi.ac.uk/biomodels/BIOMD0000000865)<br/><sup>Nikolaev2019 - Immunobiochemic</sup></summary><sup>Nikolaev2019 - Immunobiochemical reconstruction of influenza lung infection-melanoma skin cancer interactions</sup></details>|<details><summary>pass</summary>[Nikolaev2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000865#Files)</details>|pass|<details><summary>pass</summary>[Nikolaev2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000865#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Nikolaev2019.sedml647736.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Nikolaev2019.xml` has warnings.\r\n - 16 warnings of type SBML unit consistency (99505). The following is the first warning at line 1055, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'sigma_p + k_T * pow(T, n_T) / (pow(K_T, n_T) + pow(T, n_T))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 58 warnings of type Modeling practice (80701). The following is the first warning at line 743, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'sigma_i' does not have a 'units' attribute.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Nikolaev2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Nikolaev2019.xml` has warnings.\r\n - 16 warnings of type SBML unit consistency (99505). The following is the first warning at line 1055, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'sigma_p + k_T * pow(T, n_T) / (pow(K_T, n_T) + pow(T, n_T))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 58 warnings of type Modeling practice (80701). The following is the first warning at line 743, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'sigma_i' does not have a 'units' attribute.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n\r\n`/root/in/Nikolaev2019.sedml647736.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Nikolaev2019.sedml` is invalid.\n - Each identified SED object must have a unique id. Multiple objects have the following ids:\n - p1_alpha_task1\n - p1_alpha_task3\n - Task `task3` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n - Data generator `C_1_task1` is invalid.\n - Variable `p1_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C']` are not defined. No namespaces are defined for the target.\n - Data generator `alpha_1_task1` is invalid.\n - Variable `p1_alpha_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n - Data generator `B_2_task1` is invalid.\n - Variable `p1_B_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='B']` are not defined. No namespaces are defined for the target.\n - Data generator `alpha_2_task1` is invalid.\n - Variable `p1_alpha_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n - Data generator `I_3_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `alpha_3_task1` is invalid.\n - Variable `p1_alpha_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n - Data generator `P_4_task1` is invalid.\n - Variable `p1_P_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='P']` are not defined. No namespaces are defined for the target.\n - Data generator `alpha_4_task1` is invalid.\n - Variable `p1_alpha_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n - Data generator `C_1_task3` is invalid.\n - Variable `p1_C_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C']` are not defined. No namespaces are defined for the target.\n - Data generator `alpha_1_task3` is invalid.\n - Variable `p1_alpha_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n - Data generator `B_2_task3` is invalid.\n - Variable `p1_B_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='B']` are not defined. No namespaces are defined for the target.\n - Data generator `alpha_2_task3` is invalid.\n - Variable `p1_alpha_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n - Data generator `I_3_task3` is invalid.\n - Variable `p1_I_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `alpha_3_task3` is invalid.\n - Variable `p1_alpha_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n - Data generator `P_4_task3` is invalid.\n - Variable `p1_P_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='P']` are not defined. No namespaces are defined for the target.\n - Data generator `alpha_4_task3` is invalid.\n - Variable `p1_alpha_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Nikolaev2019.sedml523243.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Nikolaev2019.xml` has warnings.\r\n - 16 warnings of type SBML unit consistency (99505). The following is the first warning at line 1055, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'sigma_p + k_T * pow(T, n_T) / (pow(K_T, n_T) + pow(T, n_T))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 58 warnings of type Modeling practice (80701). The following is the first warning at line 743, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'sigma_i' does not have a 'units' attribute.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Nikolaev2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Nikolaev2019.xml` has warnings.\r\n - 16 warnings of type SBML unit consistency (99505). The following is the first warning at line 1055, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'sigma_p + k_T * pow(T, n_T) / (pow(K_T, n_T) + pow(T, n_T))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 58 warnings of type Modeling practice (80701). The following is the first warning at line 743, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'sigma_i' does not have a 'units' attribute.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Nikolaev2019.sedml523243.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Nikolaev2019.sedml` is invalid.\n - Each identified SED object must have a unique id. Multiple objects have the following ids:\n - p1_alpha_task1\n - p1_alpha_task3\n - Task `task3` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n - Data generator `C_1_task1` is invalid.\n - Variable `p1_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C']` are not defined. No namespaces are defined for the target.\n - Data generator `alpha_1_task1` is invalid.\n - Variable `p1_alpha_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n - Data generator `B_2_task1` is invalid.\n - Variable `p1_B_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='B']` are not defined. No namespaces are defined for the target.\n - Data generator `alpha_2_task1` is invalid.\n - Variable `p1_alpha_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n - Data generator `I_3_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `alpha_3_task1` is invalid.\n - Variable `p1_alpha_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n - Data generator `P_4_task1` is invalid.\n - Variable `p1_P_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='P']` are not defined. No namespaces are defined for the target.\n - Data generator `alpha_4_task1` is invalid.\n - Variable `p1_alpha_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n - Data generator `C_1_task3` is invalid.\n - Variable `p1_C_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C']` are not defined. No namespaces are defined for the target.\n - Data generator `alpha_1_task3` is invalid.\n - Variable `p1_alpha_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n - Data generator `B_2_task3` is invalid.\n - Variable `p1_B_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='B']` are not defined. No namespaces are defined for the target.\n - Data generator `alpha_2_task3` is invalid.\n - Variable `p1_alpha_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n - Data generator `I_3_task3` is invalid.\n - Variable `p1_I_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `alpha_3_task3` is invalid.\n - Variable `p1_alpha_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n - Data generator `P_4_task3` is invalid.\n - Variable `p1_P_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='P']` are not defined. No namespaces are defined for the target.\n - Data generator `alpha_4_task3` is invalid.\n - Variable `p1_alpha_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000866](https://www.ebi.ac.uk/biomodels/BIOMD0000000866)<br/><sup>Simon2019 - NIK-dependent p100</sup></summary><sup>Simon2019 - NIK-dependent p100 processing into p52, Michaelis-Menten, SBML 2v4</sup></details>|<details><summary>pass</summary>[Simon2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000866#Files)</details>|pass|<details><summary>pass</summary>[Simon2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000866#Files)</details>|FAIL|pass|pass|
|<details><summary>[BIOMD0000000867](https://www.ebi.ac.uk/biomodels/BIOMD0000000867)<br/><sup>Coulibaly2019 - Interleukin-15</sup></summary><sup>Coulibaly2019 - Interleukin-15 Signaling in HIF-1a Regulation in Natural Killer Cells</sup></details>|<details><summary>pass</summary>[Coulibaly2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000867#Files)</details>|pass|<details><summary>pass</summary>[Coulibaly2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000867#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Coulibaly2019.sedml825475.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/Coulibaly2019.sedml825475.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Coulibaly2019.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - YDataReference from Curve `p2_curve_1_task1` cannot be resolved to a data generator with id `HIF_1a_Total_Fold_Change_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p3_curve_1_task1` cannot be resolved to a data generator with id `HIF_1a_Total_Fold_Change_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p2_curve_1_task2` cannot be resolved to a data generator with id `HIF_1a_Total_Fold_Change_1_task2` because multiple data generators have this id.\n - YDataReference from Curve `p3_curve_1_task2` cannot be resolved to a data generator with id `HIF_1a_Total_Fold_Change_1_task2` because multiple data generators have this id.\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Coulibaly2019.sedml891747.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Coulibaly2019.sedml891747.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Coulibaly2019.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - YDataReference from Curve `p2_curve_1_task1` cannot be resolved to a data generator with id `HIF_1a_Total_Fold_Change_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p3_curve_1_task1` cannot be resolved to a data generator with id `HIF_1a_Total_Fold_Change_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p2_curve_1_task2` cannot be resolved to a data generator with id `HIF_1a_Total_Fold_Change_1_task2` because multiple data generators have this id.\n - YDataReference from Curve `p3_curve_1_task2` cannot be resolved to a data generator with id `HIF_1a_Total_Fold_Change_1_task2` because multiple data generators have this id.\n'```</details>|
|<details><summary>[BIOMD0000000868](https://www.ebi.ac.uk/biomodels/BIOMD0000000868)<br/><sup>Simon2019 - NIK-dependent p100</sup></summary><sup>Simon2019 - NIK-dependent p100 processing into p52, Mass Action, SBML 2v4</sup></details>|<details><summary>pass</summary>[simon2019_model2.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000868#Files)</details>|pass|<details><summary>pass</summary>[simon2019_model2.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000868#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/simon2019_model2.sedml190809.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `simon2019_model2.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 362, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'p52 / 1000' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type Modeling practice (80701). The following is the first warning at line 345, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'p52_uM' does not have a 'units' attribute.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./simon2019_model2.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `simon2019_model2.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 362, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'p52 / 1000' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type Modeling practice (80701). The following is the first warning at line 345, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'p52_uM' does not have a 'units' attribute.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n\r\n`/root/in/simon2019_model2.sedml190809.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./simon2019_model2.sedml` is invalid.\n - Task `task3` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='p100t']` are not defined. No namespaces are defined for the target.\n - Data generator `p52_uM_1_task1` is invalid.\n - Variable `p1_p52_uM_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='p52_uM']` are not defined. No namespaces are defined for the target.\n - Data generator `p100_1_task1` is invalid.\n - Variable `p1_p100_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='p100']` are not defined. No namespaces are defined for the target.\n - Data generator `p52_uM_1_task3` is invalid.\n - Variable `p1_p52_uM_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='p52_uM']` are not defined. No namespaces are defined for the target.\n - Data generator `p100_1_task3` is invalid.\n - Variable `p1_p100_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='p100']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/simon2019_model2.sedml763589.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `simon2019_model2.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 362, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'p52 / 1000' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type Modeling practice (80701). The following is the first warning at line 345, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'p52_uM' does not have a 'units' attribute.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./simon2019_model2.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `simon2019_model2.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 362, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'p52 / 1000' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type Modeling practice (80701). The following is the first warning at line 345, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'p52_uM' does not have a 'units' attribute.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/simon2019_model2.sedml763589.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./simon2019_model2.sedml` is invalid.\n - Task `task3` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='p100t']` are not defined. No namespaces are defined for the target.\n - Data generator `p52_uM_1_task1` is invalid.\n - Variable `p1_p52_uM_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='p52_uM']` are not defined. No namespaces are defined for the target.\n - Data generator `p100_1_task1` is invalid.\n - Variable `p1_p100_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='p100']` are not defined. No namespaces are defined for the target.\n - Data generator `p52_uM_1_task3` is invalid.\n - Variable `p1_p52_uM_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='p52_uM']` are not defined. No namespaces are defined for the target.\n - Data generator `p100_1_task3` is invalid.\n - Variable `p1_p100_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='p100']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000869](https://www.ebi.ac.uk/biomodels/BIOMD0000000869)<br/><sup>Simon2019 - NIK-dependent p100</sup></summary><sup>Simon2019 - NIK-dependent p100 processing into p52 and IkBd degradation, Michaelis-Menten, SBML 2v4</sup></details>|<details><summary>pass</summary>[Simon2019_model3.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000869#Files)</details>|pass|<details><summary>pass</summary>[Simon2019_model3.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000869#Files)</details>|FAIL|pass|pass|
|<details><summary>[BIOMD0000000870](https://www.ebi.ac.uk/biomodels/BIOMD0000000870)<br/><sup>Simon2019 - NIK-dependent p100</sup></summary><sup>Simon2019 - NIK-dependent p100 processing into p52 and IkBd degradation, mass action, SBML 2v4</sup></details>|<details><summary>pass</summary>[Simon2019_model4.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000870#Files)</details>|pass|<details><summary>FAIL</summary>[Simon2019_model4.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000870#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Simon2019_model4.sedml589324.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Simon2019_model4.sedml` has warnings.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n\r\n`/root/in/Simon2019_model4.sedml589324.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Simon2019_model4.sedml` is invalid.\n - Each identified SED object must have a unique id. Multiple objects have the following ids:\n - p1_p100t_task1\n - p1_p100t_task3\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmpdje39aq3/./model1.xml` is not a file.\n - Task `task3` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='p100t']` are not defined. No namespaces are defined for the target.\n - Data generator `IkBd_uM_1_task1` is invalid.\n - Variable `p1_IkBd_uM_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='IkBd_uM']` are not defined. No namespaces are defined for the target.\n - Data generator `p100t_1_task1` is invalid.\n - Variable `p1_p100t_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='p100t']` are not defined. No namespaces are defined for the target.\n - Data generator `p52_uM_2_task1` is invalid.\n - Variable `p1_p52_uM_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='p52_uM']` are not defined. No namespaces are defined for the target.\n - Data generator `p100t_2_task1` is invalid.\n - Variable `p1_p100t_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='p100t']` are not defined. No namespaces are defined for the target.\n - Data generator `IkBd_uM_1_task3` is invalid.\n - Variable `p1_IkBd_uM_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='IkBd_uM']` are not defined. No namespaces are defined for the target.\n - Data generator `p100t_1_task3` is invalid.\n - Variable `p1_p100t_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='p100t']` are not defined. No namespaces are defined for the target.\n - Data generator `p52_uM_2_task3` is invalid.\n - Variable `p1_p52_uM_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='p52_uM']` are not defined. No namespaces are defined for the target.\n - Data generator `p100t_2_task3` is invalid.\n - Variable `p1_p100t_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='p100t']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Simon2019_model4.sedml584785.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Simon2019_model4.sedml` has warnings.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Simon2019_model4.sedml584785.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Simon2019_model4.sedml` is invalid.\n - Each identified SED object must have a unique id. Multiple objects have the following ids:\n - p1_p100t_task1\n - p1_p100t_task3\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmp03c38x5s/./model1.xml` is not a file.\n - Task `task3` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='p100t']` are not defined. No namespaces are defined for the target.\n - Data generator `IkBd_uM_1_task1` is invalid.\n - Variable `p1_IkBd_uM_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='IkBd_uM']` are not defined. No namespaces are defined for the target.\n - Data generator `p100t_1_task1` is invalid.\n - Variable `p1_p100t_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='p100t']` are not defined. No namespaces are defined for the target.\n - Data generator `p52_uM_2_task1` is invalid.\n - Variable `p1_p52_uM_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='p52_uM']` are not defined. No namespaces are defined for the target.\n - Data generator `p100t_2_task1` is invalid.\n - Variable `p1_p100t_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='p100t']` are not defined. No namespaces are defined for the target.\n - Data generator `IkBd_uM_1_task3` is invalid.\n - Variable `p1_IkBd_uM_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='IkBd_uM']` are not defined. No namespaces are defined for the target.\n - Data generator `p100t_1_task3` is invalid.\n - Variable `p1_p100t_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='p100t']` are not defined. No namespaces are defined for the target.\n - Data generator `p52_uM_2_task3` is invalid.\n - Variable `p1_p52_uM_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='p52_uM']` are not defined. No namespaces are defined for the target.\n - Data generator `p100t_2_task3` is invalid.\n - Variable `p1_p100t_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='p100t']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000871](https://www.ebi.ac.uk/biomodels/BIOMD0000000871)<br/><sup>NIK-dependent p100 processing </sup></summary><sup>NIK-dependent p100 processing into p52 with RelB binding and IkBd degradation, mass action, SBML 2v4</sup></details>|<details><summary>pass</summary>[Simon2019_Figure3b.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000871#Files)</details>|pass|<details><summary>pass</summary>[Simon2019_Figure3b.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000871#Files)</details>|FAIL|pass|pass|
|<details><summary>[BIOMD0000000872](https://www.ebi.ac.uk/biomodels/BIOMD0000000872)<br/><sup>Verma2016 - HIV and HPV co-inf</sup></summary><sup>Verma2016 - HIV and HPV co-infection, T-cell response</sup></details>|<details><summary>FAIL</summary>[verma2017.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000872#Files)</details>|FAIL|<details><summary>pass</summary>[verma2017.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000872#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/verma2017.sedml881239.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./verma2017.sedml` has warnings.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n`/root/in/verma2017.sedml881239.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./verma2017.sedml` is invalid.\n - Model `model` is invalid.\n - The model file `verma2017.xml` is invalid.\n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout' cannot be used in this document.\n \n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout_L2' cannot be used in this document.\n \n - General SBML conformance (10102) at line 3, column 2: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L2V4 Section 4.1\n Element 'listOfLayouts' is not part of the definition of SBML Level 2 Version 4.\n \n"```</details>|<details><summary>other</summary>```Command '-i /root/in/verma2017.sedml546570.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./verma2017.sedml` has warnings.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/verma2017.sedml546570.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./verma2017.sedml` is invalid.\n - Model `model` is invalid.\n - The model file `verma2017.xml` is invalid.\n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout' cannot be used in this document.\n \n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout_L2' cannot be used in this document.\n \n - General SBML conformance (10102) at line 3, column 2: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L2V4 Section 4.1\n Element 'listOfLayouts' is not part of the definition of SBML Level 2 Version 4.\n \n"```</details>|
|<details><summary>[BIOMD0000000873](https://www.ebi.ac.uk/biomodels/BIOMD0000000873)<br/><sup>Soni2018 - IL6 induced M2 Phen</sup></summary><sup>Soni2018 - IL6 induced M2 Phenotype in Leishmania major infected macrophage</sup></details>|<details><summary>pass</summary>[Soni2018.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000873#Files)</details>|FAIL|<details><summary>pass</summary>[Soni2018.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000873#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Soni2018.sedml734644.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Soni2018.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (10501). The following is the first warning at line 2788, column 8:\r\n - The formula 'pow(mwbe46ba92_97de_4cc4_970d_2dec54671573 * mwdfcbcdb1_3058_4a8b_9166_5b5e144c52c9, mwf9b630f8_c8a5_4fd0_bef1_92819b1257d2)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 38 warnings of type SBML unit consistency (99505). The following is the first warning at line 2420, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'mwfef402b9_4b7e_4fbd_bba2_ff8998ab0b22 * Function_for_reaction_3(mw0930c7cf_99a6_4a20_a33d_02fd90e424bb, mwc42127ea_2b78_4381_b230_30e95cd5a9d6, mwfef402b9_4b7e_4fbd_bba2_ff8998ab0b22)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 57 warnings of type Modeling practice (80701). The following is the first warning at line 2434, column 12:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'mw0930c7cf_99a6_4a20_a33d_02fd90e424bb' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Soni2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Soni2018.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (10501). The following is the first warning at line 2788, column 8:\r\n - The formula 'pow(mwbe46ba92_97de_4cc4_970d_2dec54671573 * mwdfcbcdb1_3058_4a8b_9166_5b5e144c52c9, mwf9b630f8_c8a5_4fd0_bef1_92819b1257d2)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 38 warnings of type SBML unit consistency (99505). The following is the first warning at line 2420, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'mwfef402b9_4b7e_4fbd_bba2_ff8998ab0b22 * Function_for_reaction_3(mw0930c7cf_99a6_4a20_a33d_02fd90e424bb, mwc42127ea_2b78_4381_b230_30e95cd5a9d6, mwfef402b9_4b7e_4fbd_bba2_ff8998ab0b22)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 57 warnings of type Modeling practice (80701). The following is the first warning at line 2434, column 12:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'mw0930c7cf_99a6_4a20_a33d_02fd90e424bb' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Soni2018.sedml734644.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Soni2018.sedml` is invalid.\n - Data generator `mwf219928e_abba_4c09_9597_5d6910f7e4d9_1_task1` is invalid.\n - Variable `p1_mwf219928e_abba_4c09_9597_5d6910f7e4d9_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mwf219928e_abba_4c09_9597_5d6910f7e4d9']` are not defined. No namespaces are defined for the target.\n - Data generator `mwd7270399_0429_4c85_920a_de2e0ae74440_2_task1` is invalid.\n - Variable `p1_mwd7270399_0429_4c85_920a_de2e0ae74440_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mwd7270399_0429_4c85_920a_de2e0ae74440']` are not defined. No namespaces are defined for the target.\n - Data generator `mw6339814d_af4c_4eee_9455_7e20795f6aeb_3_task1` is invalid.\n - Variable `p1_mw6339814d_af4c_4eee_9455_7e20795f6aeb_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mw6339814d_af4c_4eee_9455_7e20795f6aeb']` are not defined. No namespaces are defined for the target.\n - Data generator `mwc42127ea_2b78_4381_b230_30e95cd5a9d6_4_task1` is invalid.\n - Variable `p1_mwc42127ea_2b78_4381_b230_30e95cd5a9d6_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mwc42127ea_2b78_4381_b230_30e95cd5a9d6']` are not defined. No namespaces are defined for the target.\n - Data generator `mw4fd8b902_e2a2_4910_899c_2a7d3425e0e0_5_task1` is invalid.\n - Variable `p1_mw4fd8b902_e2a2_4910_899c_2a7d3425e0e0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mw4fd8b902_e2a2_4910_899c_2a7d3425e0e0']` are not defined. No namespaces are defined for the target.\n - Data generator `mwf06c0537_577b_4f13_a9a3_521f9d2217fb_6_task1` is invalid.\n - Variable `p1_mwf06c0537_577b_4f13_a9a3_521f9d2217fb_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mwf06c0537_577b_4f13_a9a3_521f9d2217fb']` are not defined. No namespaces are defined for the target.\n - Data generator `mwbba20281_3d8b_48c3_8e13_dee78e87dfb8_7_task1` is invalid.\n - Variable `p1_mwbba20281_3d8b_48c3_8e13_dee78e87dfb8_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mwbba20281_3d8b_48c3_8e13_dee78e87dfb8']` are not defined. No namespaces are defined for the target.\n - Data generator `mw78198c86_4b16_4117_8592_6a95c3953126_8_task1` is invalid.\n - Variable `p1_mw78198c86_4b16_4117_8592_6a95c3953126_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mw78198c86_4b16_4117_8592_6a95c3953126']` are not defined. No namespaces are defined for the target.\n - Data generator `mw7938fab7_d0c6_497b_8fb6_75922fcc19d5_9_task1` is invalid.\n - Variable `p1_mw7938fab7_d0c6_497b_8fb6_75922fcc19d5_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mw7938fab7_d0c6_497b_8fb6_75922fcc19d5']` are not defined. No namespaces are defined for the target.\n - Data generator `mw8675a533_92fb_4fbf_b747_7bca05c5841c_10_task1` is invalid.\n - Variable `p1_mw8675a533_92fb_4fbf_b747_7bca05c5841c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mw8675a533_92fb_4fbf_b747_7bca05c5841c']` are not defined. No namespaces are defined for the target.\n - Data generator `mw869055b5_5d27_4f4a_a390_b3fa48d6780e_11_task1` is invalid.\n - Variable `p1_mw869055b5_5d27_4f4a_a390_b3fa48d6780e_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mw869055b5_5d27_4f4a_a390_b3fa48d6780e']` are not defined. No namespaces are defined for the target.\n - Data generator `mwd5f166e6_df0a_45bf_b662_e87b91b79a27_12_task1` is invalid.\n - Variable `p1_mwd5f166e6_df0a_45bf_b662_e87b91b79a27_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mwd5f166e6_df0a_45bf_b662_e87b91b79a27']` are not defined. No namespaces are defined for the target.\n - Data generator `mw9275f30d_c42b_459c_91c5_67b7e08b6486_13_task1` is invalid.\n - Variable `p1_mw9275f30d_c42b_459c_91c5_67b7e08b6486_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mw9275f30d_c42b_459c_91c5_67b7e08b6486']` are not defined. No namespaces are defined for the target.\n - Data generator `mwc84af692_e3fc_4ede_99b6_b0cce3729bf7_14_task1` is invalid.\n - Variable `p1_mwc84af692_e3fc_4ede_99b6_b0cce3729bf7_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mwc84af692_e3fc_4ede_99b6_b0cce3729bf7']` are not defined. No namespaces are defined for the target.\n - Data generator `mwacd53f34_4935_4b6a_8267_024f0d966c8c_15_task1` is invalid.\n - Variable `p1_mwacd53f34_4935_4b6a_8267_024f0d966c8c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mwacd53f34_4935_4b6a_8267_024f0d966c8c']` are not defined. No namespaces are defined for the target.\n - Data generator `mwf26f605f_29e9_4454_834c_7b3edab4bbc2_16_task1` is invalid.\n - Variable `p1_mwf26f605f_29e9_4454_834c_7b3edab4bbc2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mwf26f605f_29e9_4454_834c_7b3edab4bbc2']` are not defined. No namespaces are defined for the target.\n - Data generator `mw10b15557_55f5_4525_b19d_161b056f5791_17_task1` is invalid.\n - Variable `p1_mw10b15557_55f5_4525_b19d_161b056f5791_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mw10b15557_55f5_4525_b19d_161b056f5791']` are not defined. No namespaces are defined for the target.\n - Data generator `mwb9a40fab_a7c9_4984_805f_045fefc4ff32_18_task1` is invalid.\n - Variable `p1_mwb9a40fab_a7c9_4984_805f_045fefc4ff32_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mwb9a40fab_a7c9_4984_805f_045fefc4ff32']` are not defined. No namespaces are defined for the target.\n - Data generator `mw58370246_a992_4253_8029_12fbb07a417d_19_task1` is invalid.\n - Variable `p1_mw58370246_a992_4253_8029_12fbb07a417d_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mw58370246_a992_4253_8029_12fbb07a417d']` are not defined. No namespaces are defined for the target.\n - Data generator `mwba110304_bd9a_4fd0_9b4c_b8bfc975e30b_20_task1` is invalid.\n - Variable `p1_mwba110304_bd9a_4fd0_9b4c_b8bfc975e30b_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mwba110304_bd9a_4fd0_9b4c_b8bfc975e30b']` are not defined. No namespaces are defined for the target.\n - Data generator `mw5607cee0_ee75_4065_9368_d07c3abbe18b_21_task1` is invalid.\n - Variable `p1_mw5607cee0_ee75_4065_9368_d07c3abbe18b_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mw5607cee0_ee75_4065_9368_d07c3abbe18b']` are not defined. No namespaces are defined for the target.\n - Data generator `mwbe46ba92_97de_4cc4_970d_2dec54671573_22_task1` is invalid.\n - Variable `p1_mwbe46ba92_97de_4cc4_970d_2dec54671573_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mwbe46ba92_97de_4cc4_970d_2dec54671573']` are not defined. No namespaces are defined for the target.\n - Data generator `mw55b6f083_2b28_4a1a_ab90_82a751525d72_23_task1` is invalid.\n - Variable `p1_mw55b6f083_2b28_4a1a_ab90_82a751525d72_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mw55b6f083_2b28_4a1a_ab90_82a751525d72']` are not defined. No namespaces are defined for the target.\n - Data generator `mwe3af2471_5087_4796_982a_56fad9a9a972_24_task1` is invalid.\n - Variable `p1_mwe3af2471_5087_4796_982a_56fad9a9a972_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mwe3af2471_5087_4796_982a_56fad9a9a972']` are not defined. No namespaces are defined for the target.\n - Data generator `mw364ca1a4_fc56_4138_8031_16341ac865de_25_task1` is invalid.\n - Variable `p1_mw364ca1a4_fc56_4138_8031_16341ac865de_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mw364ca1a4_fc56_4138_8031_16341ac865de']` are not defined. No namespaces are defined for the target.\n - Data generator `mw4fc13b75_10cc_41fb_b9f8_1ce95fccae73_26_task1` is invalid.\n - Variable `p1_mw4fc13b75_10cc_41fb_b9f8_1ce95fccae73_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mw4fc13b75_10cc_41fb_b9f8_1ce95fccae73']` are not defined. No namespaces are defined for the target.\n - Data generator `mw3d58864c_79c7_4ff2_98fe_1a85b2ccc43d_27_task1` is invalid.\n - Variable `p1_mw3d58864c_79c7_4ff2_98fe_1a85b2ccc43d_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mw3d58864c_79c7_4ff2_98fe_1a85b2ccc43d']` are not defined. No namespaces are defined for the target.\n - Data generator `mw6949b379_107d_4822_b13e_680a491d2425_28_task1` is invalid.\n - Variable `p1_mw6949b379_107d_4822_b13e_680a491d2425_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mw6949b379_107d_4822_b13e_680a491d2425']` are not defined. No namespaces are defined for the target.\n - Data generator `mwcbadb505_1cfb_4903_9975_0e53de2ba877_29_task1` is invalid.\n - Variable `p1_mwcbadb505_1cfb_4903_9975_0e53de2ba877_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mwcbadb505_1cfb_4903_9975_0e53de2ba877']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Soni2018.sedml982340.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Soni2018.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (10501). The following is the first warning at line 2788, column 8:\r\n - The formula 'pow(mwbe46ba92_97de_4cc4_970d_2dec54671573 * mwdfcbcdb1_3058_4a8b_9166_5b5e144c52c9, mwf9b630f8_c8a5_4fd0_bef1_92819b1257d2)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 38 warnings of type SBML unit consistency (99505). The following is the first warning at line 2420, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'mwfef402b9_4b7e_4fbd_bba2_ff8998ab0b22 * Function_for_reaction_3(mw0930c7cf_99a6_4a20_a33d_02fd90e424bb, mwc42127ea_2b78_4381_b230_30e95cd5a9d6, mwfef402b9_4b7e_4fbd_bba2_ff8998ab0b22)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 57 warnings of type Modeling practice (80701). The following is the first warning at line 2434, column 12:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'mw0930c7cf_99a6_4a20_a33d_02fd90e424bb' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Soni2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Soni2018.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (10501). The following is the first warning at line 2788, column 8:\r\n - The formula 'pow(mwbe46ba92_97de_4cc4_970d_2dec54671573 * mwdfcbcdb1_3058_4a8b_9166_5b5e144c52c9, mwf9b630f8_c8a5_4fd0_bef1_92819b1257d2)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 38 warnings of type SBML unit consistency (99505). The following is the first warning at line 2420, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'mwfef402b9_4b7e_4fbd_bba2_ff8998ab0b22 * Function_for_reaction_3(mw0930c7cf_99a6_4a20_a33d_02fd90e424bb, mwc42127ea_2b78_4381_b230_30e95cd5a9d6, mwfef402b9_4b7e_4fbd_bba2_ff8998ab0b22)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 57 warnings of type Modeling practice (80701). The following is the first warning at line 2434, column 12:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'mw0930c7cf_99a6_4a20_a33d_02fd90e424bb' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Soni2018.sedml982340.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Soni2018.sedml` is invalid.\n - Data generator `mwf219928e_abba_4c09_9597_5d6910f7e4d9_1_task1` is invalid.\n - Variable `p1_mwf219928e_abba_4c09_9597_5d6910f7e4d9_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mwf219928e_abba_4c09_9597_5d6910f7e4d9']` are not defined. No namespaces are defined for the target.\n - Data generator `mwd7270399_0429_4c85_920a_de2e0ae74440_2_task1` is invalid.\n - Variable `p1_mwd7270399_0429_4c85_920a_de2e0ae74440_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mwd7270399_0429_4c85_920a_de2e0ae74440']` are not defined. No namespaces are defined for the target.\n - Data generator `mw6339814d_af4c_4eee_9455_7e20795f6aeb_3_task1` is invalid.\n - Variable `p1_mw6339814d_af4c_4eee_9455_7e20795f6aeb_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mw6339814d_af4c_4eee_9455_7e20795f6aeb']` are not defined. No namespaces are defined for the target.\n - Data generator `mwc42127ea_2b78_4381_b230_30e95cd5a9d6_4_task1` is invalid.\n - Variable `p1_mwc42127ea_2b78_4381_b230_30e95cd5a9d6_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mwc42127ea_2b78_4381_b230_30e95cd5a9d6']` are not defined. No namespaces are defined for the target.\n - Data generator `mw4fd8b902_e2a2_4910_899c_2a7d3425e0e0_5_task1` is invalid.\n - Variable `p1_mw4fd8b902_e2a2_4910_899c_2a7d3425e0e0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mw4fd8b902_e2a2_4910_899c_2a7d3425e0e0']` are not defined. No namespaces are defined for the target.\n - Data generator `mwf06c0537_577b_4f13_a9a3_521f9d2217fb_6_task1` is invalid.\n - Variable `p1_mwf06c0537_577b_4f13_a9a3_521f9d2217fb_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mwf06c0537_577b_4f13_a9a3_521f9d2217fb']` are not defined. No namespaces are defined for the target.\n - Data generator `mwbba20281_3d8b_48c3_8e13_dee78e87dfb8_7_task1` is invalid.\n - Variable `p1_mwbba20281_3d8b_48c3_8e13_dee78e87dfb8_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mwbba20281_3d8b_48c3_8e13_dee78e87dfb8']` are not defined. No namespaces are defined for the target.\n - Data generator `mw78198c86_4b16_4117_8592_6a95c3953126_8_task1` is invalid.\n - Variable `p1_mw78198c86_4b16_4117_8592_6a95c3953126_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mw78198c86_4b16_4117_8592_6a95c3953126']` are not defined. No namespaces are defined for the target.\n - Data generator `mw7938fab7_d0c6_497b_8fb6_75922fcc19d5_9_task1` is invalid.\n - Variable `p1_mw7938fab7_d0c6_497b_8fb6_75922fcc19d5_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mw7938fab7_d0c6_497b_8fb6_75922fcc19d5']` are not defined. No namespaces are defined for the target.\n - Data generator `mw8675a533_92fb_4fbf_b747_7bca05c5841c_10_task1` is invalid.\n - Variable `p1_mw8675a533_92fb_4fbf_b747_7bca05c5841c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mw8675a533_92fb_4fbf_b747_7bca05c5841c']` are not defined. No namespaces are defined for the target.\n - Data generator `mw869055b5_5d27_4f4a_a390_b3fa48d6780e_11_task1` is invalid.\n - Variable `p1_mw869055b5_5d27_4f4a_a390_b3fa48d6780e_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mw869055b5_5d27_4f4a_a390_b3fa48d6780e']` are not defined. No namespaces are defined for the target.\n - Data generator `mwd5f166e6_df0a_45bf_b662_e87b91b79a27_12_task1` is invalid.\n - Variable `p1_mwd5f166e6_df0a_45bf_b662_e87b91b79a27_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mwd5f166e6_df0a_45bf_b662_e87b91b79a27']` are not defined. No namespaces are defined for the target.\n - Data generator `mw9275f30d_c42b_459c_91c5_67b7e08b6486_13_task1` is invalid.\n - Variable `p1_mw9275f30d_c42b_459c_91c5_67b7e08b6486_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mw9275f30d_c42b_459c_91c5_67b7e08b6486']` are not defined. No namespaces are defined for the target.\n - Data generator `mwc84af692_e3fc_4ede_99b6_b0cce3729bf7_14_task1` is invalid.\n - Variable `p1_mwc84af692_e3fc_4ede_99b6_b0cce3729bf7_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mwc84af692_e3fc_4ede_99b6_b0cce3729bf7']` are not defined. No namespaces are defined for the target.\n - Data generator `mwacd53f34_4935_4b6a_8267_024f0d966c8c_15_task1` is invalid.\n - Variable `p1_mwacd53f34_4935_4b6a_8267_024f0d966c8c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mwacd53f34_4935_4b6a_8267_024f0d966c8c']` are not defined. No namespaces are defined for the target.\n - Data generator `mwf26f605f_29e9_4454_834c_7b3edab4bbc2_16_task1` is invalid.\n - Variable `p1_mwf26f605f_29e9_4454_834c_7b3edab4bbc2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mwf26f605f_29e9_4454_834c_7b3edab4bbc2']` are not defined. No namespaces are defined for the target.\n - Data generator `mw10b15557_55f5_4525_b19d_161b056f5791_17_task1` is invalid.\n - Variable `p1_mw10b15557_55f5_4525_b19d_161b056f5791_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mw10b15557_55f5_4525_b19d_161b056f5791']` are not defined. No namespaces are defined for the target.\n - Data generator `mwb9a40fab_a7c9_4984_805f_045fefc4ff32_18_task1` is invalid.\n - Variable `p1_mwb9a40fab_a7c9_4984_805f_045fefc4ff32_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mwb9a40fab_a7c9_4984_805f_045fefc4ff32']` are not defined. No namespaces are defined for the target.\n - Data generator `mw58370246_a992_4253_8029_12fbb07a417d_19_task1` is invalid.\n - Variable `p1_mw58370246_a992_4253_8029_12fbb07a417d_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mw58370246_a992_4253_8029_12fbb07a417d']` are not defined. No namespaces are defined for the target.\n - Data generator `mwba110304_bd9a_4fd0_9b4c_b8bfc975e30b_20_task1` is invalid.\n - Variable `p1_mwba110304_bd9a_4fd0_9b4c_b8bfc975e30b_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mwba110304_bd9a_4fd0_9b4c_b8bfc975e30b']` are not defined. No namespaces are defined for the target.\n - Data generator `mw5607cee0_ee75_4065_9368_d07c3abbe18b_21_task1` is invalid.\n - Variable `p1_mw5607cee0_ee75_4065_9368_d07c3abbe18b_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mw5607cee0_ee75_4065_9368_d07c3abbe18b']` are not defined. No namespaces are defined for the target.\n - Data generator `mwbe46ba92_97de_4cc4_970d_2dec54671573_22_task1` is invalid.\n - Variable `p1_mwbe46ba92_97de_4cc4_970d_2dec54671573_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mwbe46ba92_97de_4cc4_970d_2dec54671573']` are not defined. No namespaces are defined for the target.\n - Data generator `mw55b6f083_2b28_4a1a_ab90_82a751525d72_23_task1` is invalid.\n - Variable `p1_mw55b6f083_2b28_4a1a_ab90_82a751525d72_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mw55b6f083_2b28_4a1a_ab90_82a751525d72']` are not defined. No namespaces are defined for the target.\n - Data generator `mwe3af2471_5087_4796_982a_56fad9a9a972_24_task1` is invalid.\n - Variable `p1_mwe3af2471_5087_4796_982a_56fad9a9a972_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mwe3af2471_5087_4796_982a_56fad9a9a972']` are not defined. No namespaces are defined for the target.\n - Data generator `mw364ca1a4_fc56_4138_8031_16341ac865de_25_task1` is invalid.\n - Variable `p1_mw364ca1a4_fc56_4138_8031_16341ac865de_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mw364ca1a4_fc56_4138_8031_16341ac865de']` are not defined. No namespaces are defined for the target.\n - Data generator `mw4fc13b75_10cc_41fb_b9f8_1ce95fccae73_26_task1` is invalid.\n - Variable `p1_mw4fc13b75_10cc_41fb_b9f8_1ce95fccae73_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mw4fc13b75_10cc_41fb_b9f8_1ce95fccae73']` are not defined. No namespaces are defined for the target.\n - Data generator `mw3d58864c_79c7_4ff2_98fe_1a85b2ccc43d_27_task1` is invalid.\n - Variable `p1_mw3d58864c_79c7_4ff2_98fe_1a85b2ccc43d_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mw3d58864c_79c7_4ff2_98fe_1a85b2ccc43d']` are not defined. No namespaces are defined for the target.\n - Data generator `mw6949b379_107d_4822_b13e_680a491d2425_28_task1` is invalid.\n - Variable `p1_mw6949b379_107d_4822_b13e_680a491d2425_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mw6949b379_107d_4822_b13e_680a491d2425']` are not defined. No namespaces are defined for the target.\n - Data generator `mwcbadb505_1cfb_4903_9975_0e53de2ba877_29_task1` is invalid.\n - Variable `p1_mwcbadb505_1cfb_4903_9975_0e53de2ba877_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mwcbadb505_1cfb_4903_9975_0e53de2ba877']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000874](https://www.ebi.ac.uk/biomodels/BIOMD0000000874)<br/><sup>Perelson1993 - HIVinfection_CD</sup></summary><sup>Perelson1993 - HIVinfection_CD4Tcells_ModelA</sup></details>|<details><summary>pass</summary>[Perelson1993.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000874#Files)</details>|pass|<details><summary>pass</summary>[Perelson1993.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000874#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Perelson1993.sedml35729.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Perelson1993.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 804, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'COMpartment * Function_for_production_of_CD4__T_cell(s, r, T)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 721, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 's' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 3675.88652444\r\n - Number of steps: 142\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Perelson1993.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Perelson1993.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 804, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'COMpartment * Function_for_production_of_CD4__T_cell(s, r, T)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 721, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 's' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 3675.88652444\r\n - Number of steps: 142\x1b[0m\r\n\r\n`/root/in/Perelson1993.sedml35729.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Perelson1993.sedml` is invalid.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `T_1_2_task1` is invalid.\n - Variable `p1_T_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T_1']` are not defined. No namespaces are defined for the target.\n - Data generator `T_2_3_task1` is invalid.\n - Variable `p1_T_2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T_2']` are not defined. No namespaces are defined for the target.\n - Data generator `V_4_task1` is invalid.\n - Variable `p1_V_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V']` are not defined. No namespaces are defined for the target.\n - Data generator `T_tot_5_task1` is invalid.\n - Variable `p1_T_tot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='T_tot']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Perelson1993.sedml302621.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Perelson1993.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 804, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'COMpartment * Function_for_production_of_CD4__T_cell(s, r, T)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 721, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 's' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 3675.88652444\r\n - Number of steps: 142\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Perelson1993.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Perelson1993.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 804, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'COMpartment * Function_for_production_of_CD4__T_cell(s, r, T)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 721, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 's' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 3675.88652444\r\n - Number of steps: 142\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Perelson1993.sedml302621.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Perelson1993.sedml` is invalid.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `T_1_2_task1` is invalid.\n - Variable `p1_T_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T_1']` are not defined. No namespaces are defined for the target.\n - Data generator `T_2_3_task1` is invalid.\n - Variable `p1_T_2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T_2']` are not defined. No namespaces are defined for the target.\n - Data generator `V_4_task1` is invalid.\n - Variable `p1_V_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V']` are not defined. No namespaces are defined for the target.\n - Data generator `T_tot_5_task1` is invalid.\n - Variable `p1_T_tot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='T_tot']` are not defined. No namespaces are defined for the target.\n"```</details>|
|[BIOMD0000000875](https://www.ebi.ac.uk/biomodels/BIOMD0000000875)<br/><sup>Nelson2000- HIV-1 general model 1</sup>|<details><summary>pass</summary>[Nelson2000.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000875#Files)</details>|FAIL|<details><summary>pass</summary>[Nelson2000.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000875#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Nelson2000.sedml11341.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Nelson2000.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (10541). The following is the first warning at line 467, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'T_cell_source') are second (exponent = -1, multiplier = 1000, scale = 0), item (exponent = 1, multiplier = 1, scale = 0).\r\n - 1 warning of type SBML unit consistency (99505). The following is the first warning at line 555, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'plasma * Function_for_release_of_infectious_virions_on_lysis_1(N, T_i, delta, np)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Nelson2000.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Nelson2000.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (10541). The following is the first warning at line 467, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'T_cell_source') are second (exponent = -1, multiplier = 1000, scale = 0), item (exponent = 1, multiplier = 1, scale = 0).\r\n - 1 warning of type SBML unit consistency (99505). The following is the first warning at line 555, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'plasma * Function_for_release_of_infectious_virions_on_lysis_1(N, T_i, delta, np)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\x1b[0m\r\n\r\n`/root/in/Nelson2000.sedml11341.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Nelson2000.sedml` is invalid.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `T_i_2_task1` is invalid.\n - Variable `p1_T_i_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T_i']` are not defined. No namespaces are defined for the target.\n - Data generator `V_I_3_task1` is invalid.\n - Variable `p1_V_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V_I']` are not defined. No namespaces are defined for the target.\n - Data generator `V_NI_4_task1` is invalid.\n - Variable `p1_V_NI_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V_NI']` are not defined. No namespaces are defined for the target.\n - Data generator `HIV_5_task1` is invalid.\n - Variable `p1_HIV_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='HIV']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Nelson2000.sedml394918.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Nelson2000.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (10541). The following is the first warning at line 467, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'T_cell_source') are second (exponent = -1, multiplier = 1000, scale = 0), item (exponent = 1, multiplier = 1, scale = 0).\r\n - 1 warning of type SBML unit consistency (99505). The following is the first warning at line 555, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'plasma * Function_for_release_of_infectious_virions_on_lysis_1(N, T_i, delta, np)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Nelson2000.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Nelson2000.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (10541). The following is the first warning at line 467, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'T_cell_source') are second (exponent = -1, multiplier = 1000, scale = 0), item (exponent = 1, multiplier = 1, scale = 0).\r\n - 1 warning of type SBML unit consistency (99505). The following is the first warning at line 555, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'plasma * Function_for_release_of_infectious_virions_on_lysis_1(N, T_i, delta, np)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Nelson2000.sedml394918.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Nelson2000.sedml` is invalid.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `T_i_2_task1` is invalid.\n - Variable `p1_T_i_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T_i']` are not defined. No namespaces are defined for the target.\n - Data generator `V_I_3_task1` is invalid.\n - Variable `p1_V_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V_I']` are not defined. No namespaces are defined for the target.\n - Data generator `V_NI_4_task1` is invalid.\n - Variable `p1_V_NI_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V_NI']` are not defined. No namespaces are defined for the target.\n - Data generator `HIV_5_task1` is invalid.\n - Variable `p1_HIV_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='HIV']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000876](https://www.ebi.ac.uk/biomodels/BIOMD0000000876)<br/><sup>Aavani2019 - The role of CD4 T</sup></summary><sup>Aavani2019 - The role of CD4 T cells in immune system activation and viral reproduction in a simple model for HIV infection</sup></details>|<details><summary>pass</summary>[Aavani2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000876#Files)</details>|pass|<details><summary>pass</summary>[Aavani2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000876#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Aavani2019.sedml595720.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Aavani2019.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 518, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'C_ast' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 463, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'C_ast' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Aavani2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Aavani2019.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 518, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'C_ast' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 463, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'C_ast' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Aavani2019.sedml595720.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Aavani2019.sedml` is invalid.\n - Data generator `C_Uninfected_CD4_1_task1` is invalid.\n - Variable `p1_C_Uninfected_CD4_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C_Uninfected_CD4']` are not defined. No namespaces are defined for the target.\n - Data generator `F_CTL_2_task1` is invalid.\n - Variable `p1_F_CTL_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='F_CTL']` are not defined. No namespaces are defined for the target.\n - Data generator `I_Infected_CD4_3_task1` is invalid.\n - Variable `p1_I_Infected_CD4_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I_Infected_CD4']` are not defined. No namespaces are defined for the target.\n - Data generator `V_Virus_4_task1` is invalid.\n - Variable `p1_V_Virus_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V_Virus']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Aavani2019.sedml215847.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Aavani2019.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 518, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'C_ast' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 463, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'C_ast' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Aavani2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Aavani2019.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 518, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'C_ast' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 463, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'C_ast' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Aavani2019.sedml215847.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Aavani2019.sedml` is invalid.\n - Data generator `C_Uninfected_CD4_1_task1` is invalid.\n - Variable `p1_C_Uninfected_CD4_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C_Uninfected_CD4']` are not defined. No namespaces are defined for the target.\n - Data generator `F_CTL_2_task1` is invalid.\n - Variable `p1_F_CTL_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='F_CTL']` are not defined. No namespaces are defined for the target.\n - Data generator `I_Infected_CD4_3_task1` is invalid.\n - Variable `p1_I_Infected_CD4_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I_Infected_CD4']` are not defined. No namespaces are defined for the target.\n - Data generator `V_Virus_4_task1` is invalid.\n - Variable `p1_V_Virus_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V_Virus']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000877](https://www.ebi.ac.uk/biomodels/BIOMD0000000877)<br/><sup>Ontah2019 - Dynamic analysis o</sup></summary><sup>Ontah2019 - Dynamic analysis of a tumor treatment model using oncolytic virus and chemotherapy with saturated infection rate</sup></details>|<details><summary>pass</summary>[Ontah2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000877#Files)</details>|pass|<details><summary>pass</summary>[Ontah2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000877#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Ontah2019.sedml423014.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Ontah2019.xml` has warnings.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 554, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth(r, U, I, K)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 509, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Ontah2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Ontah2019.xml` has warnings.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 554, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth(r, U, I, K)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 509, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Ontah2019.sedml423014.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Ontah2019.sedml` is invalid.\n - Data generator `C_1_task1` is invalid.\n - Variable `p1_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C']` are not defined. No namespaces are defined for the target.\n - Data generator `I_2_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `U_3_task1` is invalid.\n - Variable `p1_U_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='U']` are not defined. No namespaces are defined for the target.\n - Data generator `V_4_task1` is invalid.\n - Variable `p1_V_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Ontah2019.sedml665786.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Ontah2019.xml` has warnings.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 554, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth(r, U, I, K)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 509, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Ontah2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Ontah2019.xml` has warnings.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 554, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth(r, U, I, K)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 509, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Ontah2019.sedml665786.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Ontah2019.sedml` is invalid.\n - Data generator `C_1_task1` is invalid.\n - Variable `p1_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C']` are not defined. No namespaces are defined for the target.\n - Data generator `I_2_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `U_3_task1` is invalid.\n - Variable `p1_U_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='U']` are not defined. No namespaces are defined for the target.\n - Data generator `V_4_task1` is invalid.\n - Variable `p1_V_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V']` are not defined. No namespaces are defined for the target.\n"```</details>|
|[BIOMD0000000878](https://www.ebi.ac.uk/biomodels/BIOMD0000000878)<br/><sup>Lenbury2001_InsulinKineticsModel_A</sup>|<details><summary>pass</summary>[Lenbury2001.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000878#Files)</details>|pass|<details><summary>pass</summary>[Lenbury2001.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000878#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Lenbury2001.sedml480228.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Lenbury2001.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 729, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'COMpartment * function_for_formation_of_insulin(r_1, y, c_1, z)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 500, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'time_environment' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Lenbury2001.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Lenbury2001.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 729, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'COMpartment * function_for_formation_of_insulin(r_1, y, c_1, z)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 500, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'time_environment' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Lenbury2001.sedml480228.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Lenbury2001.sedml` is invalid.\n - Data generator `x_1_task1` is invalid.\n - Variable `p1_x_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x']` are not defined. No namespaces are defined for the target.\n - Data generator `y_2_task1` is invalid.\n - Variable `p1_y_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='y']` are not defined. No namespaces are defined for the target.\n - Data generator `z_3_task1` is invalid.\n - Variable `p1_z_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='z']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Lenbury2001.sedml412904.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Lenbury2001.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 729, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'COMpartment * function_for_formation_of_insulin(r_1, y, c_1, z)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 500, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'time_environment' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Lenbury2001.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Lenbury2001.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 729, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'COMpartment * function_for_formation_of_insulin(r_1, y, c_1, z)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 500, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'time_environment' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Lenbury2001.sedml412904.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Lenbury2001.sedml` is invalid.\n - Data generator `x_1_task1` is invalid.\n - Variable `p1_x_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x']` are not defined. No namespaces are defined for the target.\n - Data generator `y_2_task1` is invalid.\n - Variable `p1_y_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='y']` are not defined. No namespaces are defined for the target.\n - Data generator `z_3_task1` is invalid.\n - Variable `p1_z_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='z']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000879](https://www.ebi.ac.uk/biomodels/BIOMD0000000879)<br/><sup>Rodrigues2019 - A mathematical</sup></summary><sup>Rodrigues2019 - A mathematical model for chemoimmunotherapy of chronic lymphocytic leukemia</sup></details>|<details><summary>pass</summary>[Rodrigues2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000879#Files)</details>|pass|<details><summary>pass</summary>[Rodrigues2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000879#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Rodrigues2019.sedml176029.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Rodrigues2019.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 656, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '1 / 8' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 18 warnings of type Modeling practice (80701). The following is the first warning at line 510, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Rodrigues2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Rodrigues2019.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 656, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '1 / 8' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 18 warnings of type Modeling practice (80701). The following is the first warning at line 510, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Rodrigues2019.sedml176029.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Rodrigues2019.sedml` is invalid.\n - Data generator `I_1_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `N_2_task1` is invalid.\n - Variable `p1_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Rodrigues2019.sedml166429.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Rodrigues2019.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 656, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '1 / 8' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 18 warnings of type Modeling practice (80701). The following is the first warning at line 510, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Rodrigues2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Rodrigues2019.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 656, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '1 / 8' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 18 warnings of type Modeling practice (80701). The following is the first warning at line 510, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Rodrigues2019.sedml166429.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Rodrigues2019.sedml` is invalid.\n - Data generator `I_1_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `N_2_task1` is invalid.\n - Variable `p1_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000880](https://www.ebi.ac.uk/biomodels/BIOMD0000000880)<br/><sup>Trisilowati2018 - Optimal cont</sup></summary><sup>Trisilowati2018 - Optimal control of tumor-immune system interaction with treatment</sup></details>|<details><summary>pass</summary>[Trisilowati2018.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000880#Files)</details>|pass|<details><summary>pass</summary>[Trisilowati2018.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000880#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Trisilowati2018.sedml567270.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Trisilowati2018.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (99505). The following is the first warning at line 678, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth(a_1, T_Tumor_Cells, b_1__1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 14 warnings of type Modeling practice (80701). The following is the first warning at line 578, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a_1' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Trisilowati2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Trisilowati2018.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (99505). The following is the first warning at line 678, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth(a_1, T_Tumor_Cells, b_1__1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 14 warnings of type Modeling practice (80701). The following is the first warning at line 578, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a_1' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Trisilowati2018.sedml567270.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Trisilowati2018.sedml` is invalid.\n - Data generator `D_Dendritic_Cells_1_task1` is invalid.\n - Variable `p1_D_Dendritic_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='D_Dendritic_Cells']` are not defined. No namespaces are defined for the target.\n - Data generator `H_CD4_T_Cells_2_task1` is invalid.\n - Variable `p1_H_CD4_T_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H_CD4_T_Cells']` are not defined. No namespaces are defined for the target.\n - Data generator `L_CD8_T_Cells_3_task1` is invalid.\n - Variable `p1_L_CD8_T_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L_CD8_T_Cells']` are not defined. No namespaces are defined for the target.\n - Data generator `T_Tumor_Cells_4_task1` is invalid.\n - Variable `p1_T_Tumor_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T_Tumor_Cells']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Trisilowati2018.sedml497524.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Trisilowati2018.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (99505). The following is the first warning at line 678, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth(a_1, T_Tumor_Cells, b_1__1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 14 warnings of type Modeling practice (80701). The following is the first warning at line 578, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a_1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Trisilowati2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Trisilowati2018.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (99505). The following is the first warning at line 678, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Tumor_Growth(a_1, T_Tumor_Cells, b_1__1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 14 warnings of type Modeling practice (80701). The following is the first warning at line 578, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a_1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Trisilowati2018.sedml497524.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Trisilowati2018.sedml` is invalid.\n - Data generator `D_Dendritic_Cells_1_task1` is invalid.\n - Variable `p1_D_Dendritic_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='D_Dendritic_Cells']` are not defined. No namespaces are defined for the target.\n - Data generator `H_CD4_T_Cells_2_task1` is invalid.\n - Variable `p1_H_CD4_T_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H_CD4_T_Cells']` are not defined. No namespaces are defined for the target.\n - Data generator `L_CD8_T_Cells_3_task1` is invalid.\n - Variable `p1_L_CD8_T_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L_CD8_T_Cells']` are not defined. No namespaces are defined for the target.\n - Data generator `T_Tumor_Cells_4_task1` is invalid.\n - Variable `p1_T_Tumor_Cells_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T_Tumor_Cells']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000881](https://www.ebi.ac.uk/biomodels/BIOMD0000000881)<br/><sup>Kogan2013 - A mathematical mod</sup></summary><sup>Kogan2013 - A mathematical model for the immunotherapeutic control of the TH1 TH2 imbalance in melanoma</sup></details>|<details><summary>pass</summary>[Kogan2013.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000881#Files)</details>|pass|<details><summary>pass</summary>[Kogan2013.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000881#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Kogan2013.sedml139485.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Kogan2013.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 752, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'T_1 / 100000' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 27 warnings of type Modeling practice (80701). The following is the first warning at line 625, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'mu_T' does not have a 'units' attribute.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Kogan2013.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Kogan2013.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 752, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'T_1 / 100000' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 27 warnings of type Modeling practice (80701). The following is the first warning at line 625, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'mu_T' does not have a 'units' attribute.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n\r\n`/root/in/Kogan2013.sedml139485.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Kogan2013.sedml` is invalid.\n - Each identified SED object must have a unique id. Multiple objects have the following ids:\n - p1_p_I_task1\n - p1_p_I_task3\n - Task `task3` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='p_I']` are not defined. No namespaces are defined for the target.\n - Data generator `T_1_e5_1_task1` is invalid.\n - Variable `p1_T_1_e5_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='T_1_e5']` are not defined. No namespaces are defined for the target.\n - Data generator `p_I_1_task1` is invalid.\n - Variable `p1_p_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='p_I']` are not defined. No namespaces are defined for the target.\n - Data generator `T_2_e5_2_task1` is invalid.\n - Variable `p1_T_2_e5_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='T_2_e5']` are not defined. No namespaces are defined for the target.\n - Data generator `p_I_2_task1` is invalid.\n - Variable `p1_p_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='p_I']` are not defined. No namespaces are defined for the target.\n - Data generator `T_1_e5_1_task3` is invalid.\n - Variable `p1_T_1_e5_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='T_1_e5']` are not defined. No namespaces are defined for the target.\n - Data generator `p_I_1_task3` is invalid.\n - Variable `p1_p_I_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='p_I']` are not defined. No namespaces are defined for the target.\n - Data generator `T_2_e5_2_task3` is invalid.\n - Variable `p1_T_2_e5_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='T_2_e5']` are not defined. No namespaces are defined for the target.\n - Data generator `p_I_2_task3` is invalid.\n - Variable `p1_p_I_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='p_I']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Kogan2013.sedml440053.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Kogan2013.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 752, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'T_1 / 100000' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 27 warnings of type Modeling practice (80701). The following is the first warning at line 625, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'mu_T' does not have a 'units' attribute.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Kogan2013.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Kogan2013.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 752, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'T_1 / 100000' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 27 warnings of type Modeling practice (80701). The following is the first warning at line 625, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'mu_T' does not have a 'units' attribute.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Kogan2013.sedml440053.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Kogan2013.sedml` is invalid.\n - Each identified SED object must have a unique id. Multiple objects have the following ids:\n - p1_p_I_task1\n - p1_p_I_task3\n - Task `task3` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='p_I']` are not defined. No namespaces are defined for the target.\n - Data generator `T_1_e5_1_task1` is invalid.\n - Variable `p1_T_1_e5_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='T_1_e5']` are not defined. No namespaces are defined for the target.\n - Data generator `p_I_1_task1` is invalid.\n - Variable `p1_p_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='p_I']` are not defined. No namespaces are defined for the target.\n - Data generator `T_2_e5_2_task1` is invalid.\n - Variable `p1_T_2_e5_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='T_2_e5']` are not defined. No namespaces are defined for the target.\n - Data generator `p_I_2_task1` is invalid.\n - Variable `p1_p_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='p_I']` are not defined. No namespaces are defined for the target.\n - Data generator `T_1_e5_1_task3` is invalid.\n - Variable `p1_T_1_e5_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='T_1_e5']` are not defined. No namespaces are defined for the target.\n - Data generator `p_I_1_task3` is invalid.\n - Variable `p1_p_I_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='p_I']` are not defined. No namespaces are defined for the target.\n - Data generator `T_2_e5_2_task3` is invalid.\n - Variable `p1_T_2_e5_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='T_2_e5']` are not defined. No namespaces are defined for the target.\n - Data generator `p_I_2_task3` is invalid.\n - Variable `p1_p_I_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='p_I']` are not defined. No namespaces are defined for the target.\n"```</details>|
|[BIOMD0000000882](https://www.ebi.ac.uk/biomodels/BIOMD0000000882)<br/><sup>Munz2009 - Zombie SIZRC</sup>|<details><summary>pass</summary>[munz2000.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000882#Files)</details>|pass|<details><summary>pass</summary>[munz2000.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000882#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/munz2000.sedml743343.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `munz2000.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 558, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Birth(p)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 5 warnings of type Modeling practice (80701). The following is the first warning at line 451, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./munz2000.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `munz2000.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 558, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Birth(p)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 5 warnings of type Modeling practice (80701). The following is the first warning at line 451, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/munz2000.sedml743343.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./munz2000.sedml` is invalid.\n - Data generator `Susceptible_1_task1` is invalid.\n - Variable `p1_Susceptible_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Susceptible']` are not defined. No namespaces are defined for the target.\n - Data generator `Zombie_2_task1` is invalid.\n - Variable `p1_Zombie_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Zombie']` are not defined. No namespaces are defined for the target.\n - Data generator `Removal_3_task1` is invalid.\n - Variable `p1_Removal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Removal']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/munz2000.sedml492465.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `munz2000.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 558, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Birth(p)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 5 warnings of type Modeling practice (80701). The following is the first warning at line 451, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./munz2000.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `munz2000.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 558, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Birth(p)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 5 warnings of type Modeling practice (80701). The following is the first warning at line 451, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/munz2000.sedml492465.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./munz2000.sedml` is invalid.\n - Data generator `Susceptible_1_task1` is invalid.\n - Variable `p1_Susceptible_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Susceptible']` are not defined. No namespaces are defined for the target.\n - Data generator `Zombie_2_task1` is invalid.\n - Variable `p1_Zombie_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Zombie']` are not defined. No namespaces are defined for the target.\n - Data generator `Removal_3_task1` is invalid.\n - Variable `p1_Removal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Removal']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000883](https://www.ebi.ac.uk/biomodels/BIOMD0000000883)<br/><sup>Giani2019 - Computational mode</sup></summary><sup>Giani2019 - Computational modeling to predict MAP3K8 effects as mediator of resistance to vemurafenib in thyroid cancer stem cells</sup></details>|<details><summary>pass</summary>[Giani2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000883#Files)</details>|pass|<details><summary>pass</summary>[Giani2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000883#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Giani2019.sedml636396.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/Giani2019.sedml636396.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Giani2019.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - DataReference from DataSet `ds_2_task2` cannot be resolved to a data generator with id `species_10_1_task2` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_1_task2` cannot be resolved to a data generator with id `species_10_1_task2` because multiple data generators have this id.\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Giani2019.sedml480607.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Giani2019.sedml480607.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Giani2019.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - DataReference from DataSet `ds_2_task2` cannot be resolved to a data generator with id `species_10_1_task2` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_1_task2` cannot be resolved to a data generator with id `species_10_1_task2` because multiple data generators have this id.\n'```</details>|
|<details><summary>[BIOMD0000000884](https://www.ebi.ac.uk/biomodels/BIOMD0000000884)<br/><sup>Cortes2019 - Optimality of the</sup></summary><sup>Cortes2019 - Optimality of the spontaneous prophage induction rate.</sup></details>|<details><summary>pass</summary>[Cortes2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000884#Files)</details>|pass|<details><summary>FAIL</summary>[Cortes2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000884#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Cortes2019.sedml296816.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/Cortes2019.sedml296816.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Cortes2019.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmpojptin48/./model1.xml` is not a file.\n - Data generator `L_1_task1` is invalid.\n - Variable `p1_L_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L']` are not defined. No namespaces are defined for the target.\n - Data generator `U_2_task1` is invalid.\n - Variable `p1_U_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='U']` are not defined. No namespaces are defined for the target.\n - Data generator `V_3_task1` is invalid.\n - Variable `p1_V_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V']` are not defined. No namespaces are defined for the target.\n - Data generator `phi_4_task1` is invalid.\n - Variable `p1_phi_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='phi']` are not defined. No namespaces are defined for the target.\n - Data generator `K_5_task1` is invalid.\n - Variable `p1_K_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='K']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Cortes2019.sedml972215.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Cortes2019.sedml972215.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Cortes2019.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmpp2nklo57/./model1.xml` is not a file.\n - Data generator `L_1_task1` is invalid.\n - Variable `p1_L_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L']` are not defined. No namespaces are defined for the target.\n - Data generator `U_2_task1` is invalid.\n - Variable `p1_U_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='U']` are not defined. No namespaces are defined for the target.\n - Data generator `V_3_task1` is invalid.\n - Variable `p1_V_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V']` are not defined. No namespaces are defined for the target.\n - Data generator `phi_4_task1` is invalid.\n - Variable `p1_phi_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='phi']` are not defined. No namespaces are defined for the target.\n - Data generator `K_5_task1` is invalid.\n - Variable `p1_K_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='K']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000885](https://www.ebi.ac.uk/biomodels/BIOMD0000000885)<br/><sup>Sumana2018 - Mathematical mode</sup></summary><sup>Sumana2018 - Mathematical modeling of cancer-immune system, considering the role of antibodies.</sup></details>|<details><summary>pass</summary>[Sumana2018.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000885#Files)</details>|pass|<details><summary>pass</summary>[Sumana2018.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000885#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Sumana2018.sedml385289.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Sumana2018.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 572, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_formation_of_B_cell(a, u, B, k_1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 531, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Sumana2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Sumana2018.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 572, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_formation_of_B_cell(a, u, B, k_1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 531, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Sumana2018.sedml385289.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Sumana2018.sedml` is invalid.\n - Data generator `B_1_task1` is invalid.\n - Variable `p1_B_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='B']` are not defined. No namespaces are defined for the target.\n - Data generator `P_2_task1` is invalid.\n - Variable `p1_P_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='P']` are not defined. No namespaces are defined for the target.\n - Data generator `A_3_task1` is invalid.\n - Variable `p1_A_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='A']` are not defined. No namespaces are defined for the target.\n - Data generator `T_4_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Sumana2018.sedml790310.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Sumana2018.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 572, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_formation_of_B_cell(a, u, B, k_1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 531, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Sumana2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Sumana2018.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 572, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_formation_of_B_cell(a, u, B, k_1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 531, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Sumana2018.sedml790310.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Sumana2018.sedml` is invalid.\n - Data generator `B_1_task1` is invalid.\n - Variable `p1_B_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='B']` are not defined. No namespaces are defined for the target.\n - Data generator `P_2_task1` is invalid.\n - Variable `p1_P_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='P']` are not defined. No namespaces are defined for the target.\n - Data generator `A_3_task1` is invalid.\n - Variable `p1_A_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='A']` are not defined. No namespaces are defined for the target.\n - Data generator `T_4_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000886](https://www.ebi.ac.uk/biomodels/BIOMD0000000886)<br/><sup>Dubey2008 - Modeling the inter</sup></summary><sup>Dubey2008 - Modeling the interaction between avascular cancerous cells and acquired immune response</sup></details>|<details><summary>pass</summary>[Dubey2008.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000886#Files)</details>|pass|<details><summary>pass</summary>[Dubey2008.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000886#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Dubey2008.sedml836715.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Dubey2008.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 752, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Formation_of_tumor_cell(alpha, T)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 18 warnings of type Modeling practice (80701). The following is the first warning at line 661, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Dubey2008.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Dubey2008.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 752, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Formation_of_tumor_cell(alpha, T)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 18 warnings of type Modeling practice (80701). The following is the first warning at line 661, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Dubey2008.sedml836715.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Dubey2008.sedml` is invalid.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `Th_2_task1` is invalid.\n - Variable `p1_Th_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Th']` are not defined. No namespaces are defined for the target.\n - Data generator `Tc_3_task1` is invalid.\n - Variable `p1_Tc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Tc']` are not defined. No namespaces are defined for the target.\n - Data generator `B_4_task1` is invalid.\n - Variable `p1_B_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='B']` are not defined. No namespaces are defined for the target.\n - Data generator `A_5_task1` is invalid.\n - Variable `p1_A_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='A']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Dubey2008.sedml218695.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Dubey2008.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 752, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Formation_of_tumor_cell(alpha, T)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 18 warnings of type Modeling practice (80701). The following is the first warning at line 661, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Dubey2008.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Dubey2008.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 752, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Formation_of_tumor_cell(alpha, T)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 18 warnings of type Modeling practice (80701). The following is the first warning at line 661, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Dubey2008.sedml218695.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Dubey2008.sedml` is invalid.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `Th_2_task1` is invalid.\n - Variable `p1_Th_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Th']` are not defined. No namespaces are defined for the target.\n - Data generator `Tc_3_task1` is invalid.\n - Variable `p1_Tc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Tc']` are not defined. No namespaces are defined for the target.\n - Data generator `B_4_task1` is invalid.\n - Variable `p1_B_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='B']` are not defined. No namespaces are defined for the target.\n - Data generator `A_5_task1` is invalid.\n - Variable `p1_A_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='A']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000887](https://www.ebi.ac.uk/biomodels/BIOMD0000000887)<br/><sup>Lim2014 - HTLV-I infection A d</sup></summary><sup>Lim2014 - HTLV-I infection A dynamic struggle between viral persistence and host immunity</sup></details>|<details><summary>pass</summary>[Lim2014.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000887#Files)</details>|pass|<details><summary>pass</summary>[Lim2014.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000887#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Lim2014.sedml575814.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Lim2014.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 462, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Constant_flux__irreversible(lambda)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 422, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lambda' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Lim2014.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Lim2014.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 462, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Constant_flux__irreversible(lambda)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 422, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lambda' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Lim2014.sedml575814.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Lim2014.sedml` is invalid.\n - Data generator `y_1_task1` is invalid.\n - Variable `p1_y_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='y']` are not defined. No namespaces are defined for the target.\n - Data generator `z_2_task1` is invalid.\n - Variable `p1_z_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='z']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Lim2014.sedml112632.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Lim2014.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 462, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Constant_flux__irreversible(lambda)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 422, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lambda' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Lim2014.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Lim2014.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 462, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Constant_flux__irreversible(lambda)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 422, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lambda' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Lim2014.sedml112632.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Lim2014.sedml` is invalid.\n - Data generator `y_1_task1` is invalid.\n - Variable `p1_y_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='y']` are not defined. No namespaces are defined for the target.\n - Data generator `z_2_task1` is invalid.\n - Variable `p1_z_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='z']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000888](https://www.ebi.ac.uk/biomodels/BIOMD0000000888)<br/><sup>Unni2019 - Mathematical Modeli</sup></summary><sup>Unni2019 - Mathematical Modeling, Analysis, and Simulation of Tumor Dynamics with Drug Interventions</sup></details>|<details><summary>pass</summary>[unni2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000888#Files)</details>|pass|<details><summary>pass</summary>[unni2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000888#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/unni2019.sedml27234.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `unni2019.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 1028, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_growth_of_tumor_cell(a, T, b)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 21 warnings of type Modeling practice (80701). The following is the first warning at line 682, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./unni2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `unni2019.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 1028, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_growth_of_tumor_cell(a, T, b)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 21 warnings of type Modeling practice (80701). The following is the first warning at line 682, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/unni2019.sedml27234.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./unni2019.sedml` is invalid.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `N_2_task1` is invalid.\n - Variable `p1_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n - Data generator `D_3_task1` is invalid.\n - Variable `p1_D_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='D']` are not defined. No namespaces are defined for the target.\n - Data generator `L_4_task1` is invalid.\n - Variable `p1_L_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/unni2019.sedml241155.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `unni2019.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 1028, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_growth_of_tumor_cell(a, T, b)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 21 warnings of type Modeling practice (80701). The following is the first warning at line 682, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./unni2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `unni2019.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 1028, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_growth_of_tumor_cell(a, T, b)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 21 warnings of type Modeling practice (80701). The following is the first warning at line 682, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/unni2019.sedml241155.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./unni2019.sedml` is invalid.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `N_2_task1` is invalid.\n - Variable `p1_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n - Data generator `D_3_task1` is invalid.\n - Variable `p1_D_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='D']` are not defined. No namespaces are defined for the target.\n - Data generator `L_4_task1` is invalid.\n - Variable `p1_L_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000889](https://www.ebi.ac.uk/biomodels/BIOMD0000000889)<br/><sup>Fribourg2014 - Model of influe</sup></summary><sup>Fribourg2014 - Model of influenza A virus infection dynamics of viral antagonism and innate immune response.</sup></details>|<details><summary>pass</summary>[Fribourg2014.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000889#Files)</details>|pass|<details><summary>pass</summary>[Fribourg2014.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000889#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Fribourg2014.sedml703984.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Fribourg2014.xml` has warnings.\r\n - 30 warnings of type SBML unit consistency (99505). The following is the first warning at line 1675, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '1000000000 / 6.023e23' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 50 warnings of type Modeling practice (80701). The following is the first warning at line 1329, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r_0' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Fribourg2014.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Fribourg2014.xml` has warnings.\r\n - 30 warnings of type SBML unit consistency (99505). The following is the first warning at line 1675, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '1000000000 / 6.023e23' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 50 warnings of type Modeling practice (80701). The following is the first warning at line 1329, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r_0' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Fribourg2014.sedml703984.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Fribourg2014.sedml` is invalid.\n - Data generator `IC1_1_task1` is invalid.\n - Variable `p1_IC1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='IC1']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Fribourg2014.sedml86740.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Fribourg2014.xml` has warnings.\r\n - 30 warnings of type SBML unit consistency (99505). The following is the first warning at line 1675, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '1000000000 / 6.023e23' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 50 warnings of type Modeling practice (80701). The following is the first warning at line 1329, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r_0' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Fribourg2014.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Fribourg2014.xml` has warnings.\r\n - 30 warnings of type SBML unit consistency (99505). The following is the first warning at line 1675, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '1000000000 / 6.023e23' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 50 warnings of type Modeling practice (80701). The following is the first warning at line 1329, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r_0' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Fribourg2014.sedml86740.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Fribourg2014.sedml` is invalid.\n - Data generator `IC1_1_task1` is invalid.\n - Variable `p1_IC1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='IC1']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000890](https://www.ebi.ac.uk/biomodels/BIOMD0000000890)<br/><sup>Bhattacharya2014 - A mathemati</sup></summary><sup>Bhattacharya2014 - A mathematical model of the sterol regulatory element binding protein 2 cholesterol biosynthesis pathway</sup></details>|<details><summary>pass</summary>[Bhattacharya2014.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000890#Files)</details>|FAIL|<details><summary>pass</summary>[Bhattacharya2014.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000890#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Bhattacharya2014.sedml121620.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Bhattacharya2014.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 429, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'h / 0.001' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 568, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'HMGCR_Translation') are item (exponent = 1, multiplier = 1, scale = 0).\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 346, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'mu_c' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Bhattacharya2014.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Bhattacharya2014.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 429, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'h / 0.001' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 568, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'HMGCR_Translation') are item (exponent = 1, multiplier = 1, scale = 0).\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 346, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'mu_c' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Bhattacharya2014.sedml121620.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Bhattacharya2014.sedml` is invalid.\n - Data generator `c_1_task1` is invalid.\n - Variable `p1_c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='c']` are not defined. No namespaces are defined for the target.\n - Data generator `HMGCR_Rescaled_2_task1` is invalid.\n - Variable `p1_HMGCR_Rescaled_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='HMGCR_Rescaled']` are not defined. No namespaces are defined for the target.\n - Data generator `mRNA_Rescaled_3_task1` is invalid.\n - Variable `p1_mRNA_Rescaled_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mRNA_Rescaled']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Bhattacharya2014.sedml824456.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Bhattacharya2014.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 429, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'h / 0.001' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 568, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'HMGCR_Translation') are item (exponent = 1, multiplier = 1, scale = 0).\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 346, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'mu_c' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Bhattacharya2014.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Bhattacharya2014.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 429, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'h / 0.001' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 568, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'HMGCR_Translation') are item (exponent = 1, multiplier = 1, scale = 0).\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 346, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'mu_c' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Bhattacharya2014.sedml824456.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Bhattacharya2014.sedml` is invalid.\n - Data generator `c_1_task1` is invalid.\n - Variable `p1_c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='c']` are not defined. No namespaces are defined for the target.\n - Data generator `HMGCR_Rescaled_2_task1` is invalid.\n - Variable `p1_HMGCR_Rescaled_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='HMGCR_Rescaled']` are not defined. No namespaces are defined for the target.\n - Data generator `mRNA_Rescaled_3_task1` is invalid.\n - Variable `p1_mRNA_Rescaled_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mRNA_Rescaled']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000891](https://www.ebi.ac.uk/biomodels/BIOMD0000000891)<br/><sup>Khajanchi2019 - Stability Anal</sup></summary><sup>Khajanchi2019 - Stability Analysis of a Mathematical Model forGlioma-Immune Interaction under OptimalTherapy</sup></details>|<details><summary>pass</summary>[Khajanchi2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000891#Files)</details>|pass|<details><summary>pass</summary>[Khajanchi2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000891#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Khajanchi2019.sedml854281.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Khajanchi2019.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 535, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_increase_in_density_of_glioma_cell(r_1, u)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 492, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r_1' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Khajanchi2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Khajanchi2019.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 535, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_increase_in_density_of_glioma_cell(r_1, u)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 492, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r_1' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Khajanchi2019.sedml854281.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Khajanchi2019.sedml` is invalid.\n - Data generator `u_1_task1` is invalid.\n - Variable `p1_u_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='u']` are not defined. No namespaces are defined for the target.\n - Data generator `v_2_task1` is invalid.\n - Variable `p1_v_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='v']` are not defined. No namespaces are defined for the target.\n - Data generator `w_3_task1` is invalid.\n - Variable `p1_w_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='w']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Khajanchi2019.sedml741882.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Khajanchi2019.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 535, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_increase_in_density_of_glioma_cell(r_1, u)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 492, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r_1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Khajanchi2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Khajanchi2019.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 535, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_increase_in_density_of_glioma_cell(r_1, u)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 492, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r_1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Khajanchi2019.sedml741882.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Khajanchi2019.sedml` is invalid.\n - Data generator `u_1_task1` is invalid.\n - Variable `p1_u_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='u']` are not defined. No namespaces are defined for the target.\n - Data generator `v_2_task1` is invalid.\n - Variable `p1_v_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='v']` are not defined. No namespaces are defined for the target.\n - Data generator `w_3_task1` is invalid.\n - Variable `p1_w_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='w']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000892](https://www.ebi.ac.uk/biomodels/BIOMD0000000892)<br/><sup>Sandip2013 - Modeling the dyna</sup></summary><sup>Sandip2013 - Modeling the dynamics of hepatitis C virus with combined antiviral drug therapy: interferon and ribavirin.</sup></details>|<details><summary>pass</summary>[Sandip2013.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000892#Files)</details>|pass|<details><summary>pass</summary>[Sandip2013.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000892#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Sandip2013.sedml402929.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Sandip2013.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 624, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_formation_of_target_cells(s, r, T, k)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 555, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 's' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 35.0000000008\r\n - Number of steps: 152\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Sandip2013.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Sandip2013.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 624, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_formation_of_target_cells(s, r, T, k)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 555, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 's' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 35.0000000008\r\n - Number of steps: 152\x1b[0m\r\n\r\n`/root/in/Sandip2013.sedml402929.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Sandip2013.sedml` is invalid.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `I_2_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `VI_3_task1` is invalid.\n - Variable `p1_VI_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='VI']` are not defined. No namespaces are defined for the target.\n - Data generator `VNI_4_task1` is invalid.\n - Variable `p1_VNI_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='VNI']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Sandip2013.sedml484152.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Sandip2013.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 624, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_formation_of_target_cells(s, r, T, k)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 555, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 's' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 35.0000000008\r\n - Number of steps: 152\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Sandip2013.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Sandip2013.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 624, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_formation_of_target_cells(s, r, T, k)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 555, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 's' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 35.0000000008\r\n - Number of steps: 152\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Sandip2013.sedml484152.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Sandip2013.sedml` is invalid.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `I_2_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `VI_3_task1` is invalid.\n - Variable `p1_VI_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='VI']` are not defined. No namespaces are defined for the target.\n - Data generator `VNI_4_task1` is invalid.\n - Variable `p1_VNI_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='VNI']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000893](https://www.ebi.ac.uk/biomodels/BIOMD0000000893)<br/><sup>GonzalezMiranda2013 - The effe</sup></summary><sup>GonzalezMiranda2013 - The effect of circadian oscillations on biochemical cell signaling by NF-B</sup></details>|<details><summary>pass</summary>[GonzalezMiranda2013.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000893#Files)</details>|FAIL|<details><summary>pass</summary>[GonzalezMiranda2013.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000893#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/GonzalezMiranda2013.sedml779696.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `GonzalezMiranda2013.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 456, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_x_Formation(A, x, epsilon, z)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 550, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'y_Formation') are litre (exponent = -1, multiplier = 1, scale = 0), item (exponent = 2, multiplier = 1, scale = 0).\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 418, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'A' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./GonzalezMiranda2013.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `GonzalezMiranda2013.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 456, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_x_Formation(A, x, epsilon, z)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 550, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'y_Formation') are litre (exponent = -1, multiplier = 1, scale = 0), item (exponent = 2, multiplier = 1, scale = 0).\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 418, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'A' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/GonzalezMiranda2013.sedml779696.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./GonzalezMiranda2013.sedml` is invalid.\n - Data generator `x_1_task1` is invalid.\n - Variable `p1_x_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/GonzalezMiranda2013.sedml939293.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `GonzalezMiranda2013.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 456, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_x_Formation(A, x, epsilon, z)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 550, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'y_Formation') are litre (exponent = -1, multiplier = 1, scale = 0), item (exponent = 2, multiplier = 1, scale = 0).\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 418, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'A' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./GonzalezMiranda2013.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `GonzalezMiranda2013.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 456, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_x_Formation(A, x, epsilon, z)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 550, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'y_Formation') are litre (exponent = -1, multiplier = 1, scale = 0), item (exponent = 2, multiplier = 1, scale = 0).\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 418, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'A' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/GonzalezMiranda2013.sedml939293.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./GonzalezMiranda2013.sedml` is invalid.\n - Data generator `x_1_task1` is invalid.\n - Variable `p1_x_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000894](https://www.ebi.ac.uk/biomodels/BIOMD0000000894)<br/><sup>Bose2011 - Noise-assisted inte</sup></summary><sup>Bose2011 - Noise-assisted interactions of tumor and immune cells</sup></details>|<details><summary>pass</summary>[Bose2011.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000894#Files)</details>|FAIL|<details><summary>pass</summary>[Bose2011.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000894#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Bose2011.sedml459591.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Bose2011.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 604, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_formation_of_tumor_system(R, Dx, tau, Dz, x, z)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 653, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Removal_of_tumor_system') are litre (exponent = -1, multiplier = 0.001, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 535, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 't' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Bose2011.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Bose2011.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 604, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_formation_of_tumor_system(R, Dx, tau, Dz, x, z)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 653, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Removal_of_tumor_system') are litre (exponent = -1, multiplier = 0.001, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 535, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 't' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Bose2011.sedml459591.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Bose2011.sedml` is invalid.\n - Data generator `x_1_task1` is invalid.\n - Variable `p1_x_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x']` are not defined. No namespaces are defined for the target.\n - Data generator `y_2_task1` is invalid.\n - Variable `p1_y_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='y']` are not defined. No namespaces are defined for the target.\n - Data generator `z_3_task1` is invalid.\n - Variable `p1_z_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='z']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Bose2011.sedml603141.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Bose2011.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 604, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_formation_of_tumor_system(R, Dx, tau, Dz, x, z)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 653, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Removal_of_tumor_system') are litre (exponent = -1, multiplier = 0.001, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 535, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 't' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Bose2011.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Bose2011.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 604, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_formation_of_tumor_system(R, Dx, tau, Dz, x, z)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 653, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Removal_of_tumor_system') are litre (exponent = -1, multiplier = 0.001, scale = 0), mole (exponent = 2, multiplier = 0.001, scale = 0).\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 535, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 't' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Bose2011.sedml603141.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Bose2011.sedml` is invalid.\n - Data generator `x_1_task1` is invalid.\n - Variable `p1_x_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x']` are not defined. No namespaces are defined for the target.\n - Data generator `y_2_task1` is invalid.\n - Variable `p1_y_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='y']` are not defined. No namespaces are defined for the target.\n - Data generator `z_3_task1` is invalid.\n - Variable `p1_z_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='z']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000895](https://www.ebi.ac.uk/biomodels/BIOMD0000000895)<br/><sup>Schokker2013 - A mathematical </sup></summary><sup>Schokker2013 - A mathematical model representing cellular immune development and response to Salmonella of chicken intestinal tissue</sup></details>|<details><summary>pass</summary>[Schokker2013.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000895#Files)</details>|FAIL|<details><summary>pass</summary>[Schokker2013.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000895#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Schokker2013.sedml245841.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Schokker2013.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (10501). The following is the first warning at line 2395, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'CD4 + CD8 / Mi' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 29 warnings of type SBML unit consistency (99505). The following is the first warning at line 1459, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Constant_flux__irreversible(sMr)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 49 warnings of type Modeling practice (80701). The following is the first warning at line 1324, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'sMr' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Schokker2013.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Schokker2013.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (10501). The following is the first warning at line 2395, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'CD4 + CD8 / Mi' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 29 warnings of type SBML unit consistency (99505). The following is the first warning at line 1459, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Constant_flux__irreversible(sMr)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 49 warnings of type Modeling practice (80701). The following is the first warning at line 1324, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'sMr' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Schokker2013.sedml245841.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Schokker2013.sedml` is invalid.\n - Data generator `CD4_1_task1` is invalid.\n - Variable `p1_CD4_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CD4']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Schokker2013.sedml887148.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Schokker2013.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (10501). The following is the first warning at line 2395, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'CD4 + CD8 / Mi' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 29 warnings of type SBML unit consistency (99505). The following is the first warning at line 1459, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Constant_flux__irreversible(sMr)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 49 warnings of type Modeling practice (80701). The following is the first warning at line 1324, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'sMr' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Schokker2013.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Schokker2013.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (10501). The following is the first warning at line 2395, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'CD4 + CD8 / Mi' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 29 warnings of type SBML unit consistency (99505). The following is the first warning at line 1459, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Constant_flux__irreversible(sMr)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 49 warnings of type Modeling practice (80701). The following is the first warning at line 1324, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'sMr' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Schokker2013.sedml887148.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Schokker2013.sedml` is invalid.\n - Data generator `CD4_1_task1` is invalid.\n - Variable `p1_CD4_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CD4']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000896](https://www.ebi.ac.uk/biomodels/BIOMD0000000896)<br/><sup>Szymanska2009 - Mathematical m</sup></summary><sup>Szymanska2009 - Mathematical modeling of heat shock protein synthesis in response to temperature change</sup></details>|<details><summary>pass</summary>[Szymanska2009.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000896#Files)</details>|pass|<details><summary>pass</summary>[Szymanska2009.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000896#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Szymanska2009.sedml246397.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Szymanska2009.xml` has warnings.\r\n - 18 warnings of type SBML unit consistency (99505). The following is the first warning at line 762, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(37, leq(time, 10), 42)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 18 warnings of type Modeling practice (80701). The following is the first warning at line 490, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k_1' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Szymanska2009.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Szymanska2009.xml` has warnings.\r\n - 18 warnings of type SBML unit consistency (99505). The following is the first warning at line 762, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(37, leq(time, 10), 42)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 18 warnings of type Modeling practice (80701). The following is the first warning at line 490, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k_1' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Szymanska2009.sedml246397.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Szymanska2009.sedml` is invalid.\n - Data generator `mRNA_1_task1` is invalid.\n - Variable `p1_mRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mRNA']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Szymanska2009.sedml160721.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Szymanska2009.xml` has warnings.\r\n - 18 warnings of type SBML unit consistency (99505). The following is the first warning at line 762, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(37, leq(time, 10), 42)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 18 warnings of type Modeling practice (80701). The following is the first warning at line 490, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k_1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Szymanska2009.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Szymanska2009.xml` has warnings.\r\n - 18 warnings of type SBML unit consistency (99505). The following is the first warning at line 762, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(37, leq(time, 10), 42)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 18 warnings of type Modeling practice (80701). The following is the first warning at line 490, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k_1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Szymanska2009.sedml160721.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Szymanska2009.sedml` is invalid.\n - Data generator `mRNA_1_task1` is invalid.\n - Variable `p1_mRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mRNA']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000897](https://www.ebi.ac.uk/biomodels/BIOMD0000000897)<br/><sup>Khajanchi2015 - The combined e</sup></summary><sup>Khajanchi2015 - The combined effects of optimal control in cancer remission</sup></details>|<details><summary>pass</summary>[Khajanchi2015.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000897#Files)</details>|pass|<details><summary>FAIL</summary>[Khajanchi2015.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000897#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Khajanchi2015.sedml807691.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/Khajanchi2015.sedml807691.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Khajanchi2015.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmpaeuvjuc2/./model1.xml` is not a file.\n - Data generator `E_1_task1` is invalid.\n - Variable `p1_E_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='E']` are not defined. No namespaces are defined for the target.\n - Data generator `T_2_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Khajanchi2015.sedml613035.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Khajanchi2015.sedml613035.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Khajanchi2015.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmp7oyg4iuh/./model1.xml` is not a file.\n - Data generator `E_1_task1` is invalid.\n - Variable `p1_E_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='E']` are not defined. No namespaces are defined for the target.\n - Data generator `T_2_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000898](https://www.ebi.ac.uk/biomodels/BIOMD0000000898)<br/><sup>Jiao2018 - Feedback regulation</sup></summary><sup>Jiao2018 - Feedback regulation in a stem cell model with acute myeloid leukaemia</sup></details>|<details><summary>pass</summary>[Jiao2018.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000898#Files)</details>|pass|<details><summary>pass</summary>[Jiao2018.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000898#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Jiao2018.sedml223150.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Jiao2018.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 804, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'v_20 / (1 + h_2 * pow(D_TDSC, n))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 22 warnings of type Modeling practice (80701). The following is the first warning at line 542, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'p_10' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Jiao2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Jiao2018.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 804, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'v_20 / (1 + h_2 * pow(D_TDSC, n))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 22 warnings of type Modeling practice (80701). The following is the first warning at line 542, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'p_10' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Jiao2018.sedml223150.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Jiao2018.sedml` is invalid.\n - Data generator `A_PC_1_task1` is invalid.\n - Variable `p1_A_PC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='A_PC']` are not defined. No namespaces are defined for the target.\n - Data generator `D_TDSC_2_task1` is invalid.\n - Variable `p1_D_TDSC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='D_TDSC']` are not defined. No namespaces are defined for the target.\n - Data generator `L_LSC_3_task1` is invalid.\n - Variable `p1_L_LSC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L_LSC']` are not defined. No namespaces are defined for the target.\n - Data generator `S_HSC_4_task1` is invalid.\n - Variable `p1_S_HSC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S_HSC']` are not defined. No namespaces are defined for the target.\n - Data generator `T_TDLC_5_task1` is invalid.\n - Variable `p1_T_TDLC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T_TDLC']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Jiao2018.sedml342751.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Jiao2018.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 804, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'v_20 / (1 + h_2 * pow(D_TDSC, n))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 22 warnings of type Modeling practice (80701). The following is the first warning at line 542, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'p_10' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Jiao2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Jiao2018.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 804, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'v_20 / (1 + h_2 * pow(D_TDSC, n))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 22 warnings of type Modeling practice (80701). The following is the first warning at line 542, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'p_10' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Jiao2018.sedml342751.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Jiao2018.sedml` is invalid.\n - Data generator `A_PC_1_task1` is invalid.\n - Variable `p1_A_PC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='A_PC']` are not defined. No namespaces are defined for the target.\n - Data generator `D_TDSC_2_task1` is invalid.\n - Variable `p1_D_TDSC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='D_TDSC']` are not defined. No namespaces are defined for the target.\n - Data generator `L_LSC_3_task1` is invalid.\n - Variable `p1_L_LSC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L_LSC']` are not defined. No namespaces are defined for the target.\n - Data generator `S_HSC_4_task1` is invalid.\n - Variable `p1_S_HSC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S_HSC']` are not defined. No namespaces are defined for the target.\n - Data generator `T_TDLC_5_task1` is invalid.\n - Variable `p1_T_TDLC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T_TDLC']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000899](https://www.ebi.ac.uk/biomodels/BIOMD0000000899)<br/><sup>Ota2015 - Positive regulation </sup></summary><sup>Ota2015 - Positive regulation of Rho GTPase activity by RhoGDIs as a result of their direct interaction with GAPs (GDI integrated)</sup></details>|<details><summary>FAIL</summary>[Ota2015_GDI-integrated.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000899#Files)</details>|FAIL|<details><summary>pass</summary>[Ota2015_GDI-integrated.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000899#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Ota2015_GDI-integrated.sedml561691.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Ota2015_GDI-integrated.sedml` has warnings.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/Ota2015_GDI-integrated.sedml561691.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Ota2015_GDI-integrated.sedml` is invalid.\n - Model `model` is invalid.\n - The model file `Ota2015_GDI-integrated.xml` is invalid.\n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout' cannot be used in this document.\n \n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout_L2' cannot be used in this document.\n \n - General SBML conformance (10102) at line 3, column 2: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L2V4 Section 4.1\n Element 'listOfLayouts' is not part of the definition of SBML Level 2 Version 4.\n \n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s7']` are not defined. No namespaces are defined for the target.\n - Data generator `s16_1_task1` is invalid.\n - Variable `p1_s16_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s16']` are not defined. No namespaces are defined for the target.\n - Data generator `s16_1_task2` is invalid.\n - Variable `p1_s16_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s16']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Ota2015_GDI-integrated.sedml976273.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Ota2015_GDI-integrated.sedml` has warnings.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Ota2015_GDI-integrated.sedml976273.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Ota2015_GDI-integrated.sedml` is invalid.\n - Model `model` is invalid.\n - The model file `Ota2015_GDI-integrated.xml` is invalid.\n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout' cannot be used in this document.\n \n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout_L2' cannot be used in this document.\n \n - General SBML conformance (10102) at line 3, column 2: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L2V4 Section 4.1\n Element 'listOfLayouts' is not part of the definition of SBML Level 2 Version 4.\n \n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s7']` are not defined. No namespaces are defined for the target.\n - Data generator `s16_1_task1` is invalid.\n - Variable `p1_s16_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s16']` are not defined. No namespaces are defined for the target.\n - Data generator `s16_1_task2` is invalid.\n - Variable `p1_s16_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s16']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000900](https://www.ebi.ac.uk/biomodels/BIOMD0000000900)<br/><sup>Bianca2013 - Persistence analy</sup></summary><sup>Bianca2013 - Persistence analysis in a Kolmogorov-type model for cancer-immune system competition</sup></details>|<details><summary>pass</summary>[Bianca2013.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000900#Files)</details>|pass|<details><summary>pass</summary>[Bianca2013.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000900#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Bianca2013.sedml190099.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Bianca2013.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 570, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'log(2) / 4' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 516, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha_1' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Bianca2013.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Bianca2013.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 570, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'log(2) / 4' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 516, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha_1' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Bianca2013.sedml190099.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Bianca2013.sedml` is invalid.\n - Data generator `C_1_task1` is invalid.\n - Variable `p1_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C']` are not defined. No namespaces are defined for the target.\n - Data generator `B_2_task1` is invalid.\n - Variable `p1_B_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='B']` are not defined. No namespaces are defined for the target.\n - Data generator `T_3_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `delta_1_4_task1` is invalid.\n - Variable `p1_delta_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='delta_1']` are not defined. No namespaces are defined for the target.\n - Data generator `delta_2_5_task1` is invalid.\n - Variable `p1_delta_2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='delta_2']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Bianca2013.sedml874963.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Bianca2013.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 570, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'log(2) / 4' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 516, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha_1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Bianca2013.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Bianca2013.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 570, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'log(2) / 4' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 516, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha_1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Bianca2013.sedml874963.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Bianca2013.sedml` is invalid.\n - Data generator `C_1_task1` is invalid.\n - Variable `p1_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C']` are not defined. No namespaces are defined for the target.\n - Data generator `B_2_task1` is invalid.\n - Variable `p1_B_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='B']` are not defined. No namespaces are defined for the target.\n - Data generator `T_3_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `delta_1_4_task1` is invalid.\n - Variable `p1_delta_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='delta_1']` are not defined. No namespaces are defined for the target.\n - Data generator `delta_2_5_task1` is invalid.\n - Variable `p1_delta_2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='delta_2']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000901](https://www.ebi.ac.uk/biomodels/BIOMD0000000901)<br/><sup>ChowHall2008 Dynamics of Human</sup></summary><sup>ChowHall2008 Dynamics of Human Weight Change_ODE_1</sup></details>|<details><summary>pass</summary>[ChowHall2008 - Dynamics of Human Weight Change.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000901#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000902](https://www.ebi.ac.uk/biomodels/BIOMD0000000902)<br/><sup>Wang2019 - A mathematical mode</sup></summary><sup>Wang2019 - A mathematical model of oncolytic virotherapy with time delay</sup></details>|<details><summary>pass</summary>[Wang2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000902#Files)</details>|pass|<details><summary>pass</summary>[Wang2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000902#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Wang2019.sedml264747.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Wang2019.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 517, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Uninfected_Cells_Proliferation(r, x, y, k)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type Modeling practice (80701). The following is the first warning at line 418, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Wang2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Wang2019.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 517, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Uninfected_Cells_Proliferation(r, x, y, k)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type Modeling practice (80701). The following is the first warning at line 418, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Wang2019.sedml264747.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Wang2019.sedml` is invalid.\n - Data generator `x_1_task1` is invalid.\n - Variable `p1_x_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x']` are not defined. No namespaces are defined for the target.\n - Data generator `y_2_task1` is invalid.\n - Variable `p1_y_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='y']` are not defined. No namespaces are defined for the target.\n - Data generator `z_3_task1` is invalid.\n - Variable `p1_z_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='z']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Wang2019.sedml99546.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Wang2019.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 517, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Uninfected_Cells_Proliferation(r, x, y, k)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type Modeling practice (80701). The following is the first warning at line 418, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Wang2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Wang2019.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 517, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_Uninfected_Cells_Proliferation(r, x, y, k)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type Modeling practice (80701). The following is the first warning at line 418, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Wang2019.sedml99546.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Wang2019.sedml` is invalid.\n - Data generator `x_1_task1` is invalid.\n - Variable `p1_x_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x']` are not defined. No namespaces are defined for the target.\n - Data generator `y_2_task1` is invalid.\n - Variable `p1_y_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='y']` are not defined. No namespaces are defined for the target.\n - Data generator `z_3_task1` is invalid.\n - Variable `p1_z_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='z']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000903](https://www.ebi.ac.uk/biomodels/BIOMD0000000903)<br/><sup>Solis-perez2019 - A fractional</sup></summary><sup>Solis-perez2019 - A fractional mathematical model of breast cancer competition model</sup></details>|<details><summary>pass</summary>[Solis-perez2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000903#Files)</details>|pass|<details><summary>pass</summary>[Solis-perez2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000903#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Solis-perez2019.sedml714202.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Solis-perez2019.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 898, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_formation_of_Stem_cell(k1, C, M1, p1, E, a1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 28 warnings of type Modeling practice (80701). The following is the first warning at line 837, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Solis-perez2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Solis-perez2019.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 898, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_formation_of_Stem_cell(k1, C, M1, p1, E, a1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 28 warnings of type Modeling practice (80701). The following is the first warning at line 837, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Solis-perez2019.sedml714202.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Solis-perez2019.sedml` is invalid.\n - Data generator `C_1_task1` is invalid.\n - Variable `p1_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C']` are not defined. No namespaces are defined for the target.\n - Data generator `T_2_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `H_3_task1` is invalid.\n - Variable `p1_H_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H']` are not defined. No namespaces are defined for the target.\n - Data generator `I_4_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `E_5_task1` is invalid.\n - Variable `p1_E_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='E']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Solis-perez2019.sedml735307.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Solis-perez2019.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 898, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_formation_of_Stem_cell(k1, C, M1, p1, E, a1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 28 warnings of type Modeling practice (80701). The following is the first warning at line 837, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Solis-perez2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Solis-perez2019.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 898, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_formation_of_Stem_cell(k1, C, M1, p1, E, a1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 28 warnings of type Modeling practice (80701). The following is the first warning at line 837, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Solis-perez2019.sedml735307.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Solis-perez2019.sedml` is invalid.\n - Data generator `C_1_task1` is invalid.\n - Variable `p1_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C']` are not defined. No namespaces are defined for the target.\n - Data generator `T_2_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `H_3_task1` is invalid.\n - Variable `p1_H_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H']` are not defined. No namespaces are defined for the target.\n - Data generator `I_4_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `E_5_task1` is invalid.\n - Variable `p1_E_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='E']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000904](https://www.ebi.ac.uk/biomodels/BIOMD0000000904)<br/><sup>Admon2017 - Modelling tumor gr</sup></summary><sup>Admon2017 - Modelling tumor growth with immune response and drug using ordinary differential equations</sup></details>|<details><summary>pass</summary>[Admon2017.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000904#Files)</details>|pass|<details><summary>pass</summary>[Admon2017.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000904#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Admon2017.sedml672735.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Admon2017.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 692, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '2 * a1 * a4 / (a1 + d2)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 19 warnings of type Modeling practice (80701). The following is the first warning at line 601, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a1' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 60.000299997\r\n - Number of steps: 200002\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Admon2017.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Admon2017.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 692, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '2 * a1 * a4 / (a1 + d2)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 19 warnings of type Modeling practice (80701). The following is the first warning at line 601, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a1' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 60.000299997\r\n - Number of steps: 200002\x1b[0m\r\n\r\n`/root/in/Admon2017.sedml672735.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Admon2017.sedml` is invalid.\n - Data generator `Ti_1_task1` is invalid.\n - Variable `p1_Ti_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Ti']` are not defined. No namespaces are defined for the target.\n - Data generator `Tm_2_task1` is invalid.\n - Variable `p1_Tm_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Tm']` are not defined. No namespaces are defined for the target.\n - Data generator `I_3_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `u_4_task1` is invalid.\n - Variable `p1_u_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='u']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Admon2017.sedml131146.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Admon2017.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 692, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '2 * a1 * a4 / (a1 + d2)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 19 warnings of type Modeling practice (80701). The following is the first warning at line 601, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a1' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 60.000299997\r\n - Number of steps: 200002\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Admon2017.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Admon2017.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 692, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '2 * a1 * a4 / (a1 + d2)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 19 warnings of type Modeling practice (80701). The following is the first warning at line 601, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a1' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 60.000299997\r\n - Number of steps: 200002\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Admon2017.sedml131146.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Admon2017.sedml` is invalid.\n - Data generator `Ti_1_task1` is invalid.\n - Variable `p1_Ti_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Ti']` are not defined. No namespaces are defined for the target.\n - Data generator `Tm_2_task1` is invalid.\n - Variable `p1_Tm_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Tm']` are not defined. No namespaces are defined for the target.\n - Data generator `I_3_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `u_4_task1` is invalid.\n - Variable `p1_u_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='u']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000905](https://www.ebi.ac.uk/biomodels/BIOMD0000000905)<br/><sup>Dubey2007 - A mathematical mod</sup></summary><sup>Dubey2007 - A mathematical model for the effect of toxicant on the immune system (with toxicant effect) Model2</sup></details>|<details><summary>pass</summary>[Dubey2007_model2.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000905#Files)</details>|pass|<details><summary>pass</summary>[Dubey2007_model2.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000905#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Dubey2007_model2.sedml255402.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Dubey2007_model2.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 653, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_increase_in_the_amount_of_pathogenic_micro_organism(beta, P)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 15 warnings of type Modeling practice (80701). The following is the first warning at line 594, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Dubey2007_model2.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Dubey2007_model2.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 653, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_increase_in_the_amount_of_pathogenic_micro_organism(beta, P)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 15 warnings of type Modeling practice (80701). The following is the first warning at line 594, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Dubey2007_model2.sedml255402.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Dubey2007_model2.sedml` is invalid.\n - Data generator `P_1_task1` is invalid.\n - Variable `p1_P_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='P']` are not defined. No namespaces are defined for the target.\n - Data generator `I_2_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `M_3_task1` is invalid.\n - Variable `p1_M_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Dubey2007_model2.sedml366471.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Dubey2007_model2.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 653, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_increase_in_the_amount_of_pathogenic_micro_organism(beta, P)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 15 warnings of type Modeling practice (80701). The following is the first warning at line 594, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Dubey2007_model2.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Dubey2007_model2.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 653, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_increase_in_the_amount_of_pathogenic_micro_organism(beta, P)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 15 warnings of type Modeling practice (80701). The following is the first warning at line 594, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Dubey2007_model2.sedml366471.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Dubey2007_model2.sedml` is invalid.\n - Data generator `P_1_task1` is invalid.\n - Variable `p1_P_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='P']` are not defined. No namespaces are defined for the target.\n - Data generator `I_2_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `M_3_task1` is invalid.\n - Variable `p1_M_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000906](https://www.ebi.ac.uk/biomodels/BIOMD0000000906)<br/><sup>Dubey2007 - A mathematical mod</sup></summary><sup>Dubey2007 - A mathematical model for the effect of toxicant on the immune system (without toxicant effect) Model1</sup></details>|<details><summary>pass</summary>[Dubey2007_model1.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000906#Files)</details>|pass|<details><summary>pass</summary>[Dubey2007_model1.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000906#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Dubey2007_model1.sedml197997.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Dubey2007_model1.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 476, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_increase_in_the_amount_of_pathogenic_micro_organism(beta, P)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type Modeling practice (80701). The following is the first warning at line 423, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Dubey2007_model1.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Dubey2007_model1.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 476, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_increase_in_the_amount_of_pathogenic_micro_organism(beta, P)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type Modeling practice (80701). The following is the first warning at line 423, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Dubey2007_model1.sedml197997.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Dubey2007_model1.sedml` is invalid.\n - Data generator `P_1_task1` is invalid.\n - Variable `p1_P_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='P']` are not defined. No namespaces are defined for the target.\n - Data generator `I_2_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `M_3_task1` is invalid.\n - Variable `p1_M_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Dubey2007_model1.sedml955720.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Dubey2007_model1.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 476, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_increase_in_the_amount_of_pathogenic_micro_organism(beta, P)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type Modeling practice (80701). The following is the first warning at line 423, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Dubey2007_model1.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Dubey2007_model1.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 476, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_increase_in_the_amount_of_pathogenic_micro_organism(beta, P)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type Modeling practice (80701). The following is the first warning at line 423, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Dubey2007_model1.sedml955720.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Dubey2007_model1.sedml` is invalid.\n - Data generator `P_1_task1` is invalid.\n - Variable `p1_P_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='P']` are not defined. No namespaces are defined for the target.\n - Data generator `I_2_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `M_3_task1` is invalid.\n - Variable `p1_M_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000907](https://www.ebi.ac.uk/biomodels/BIOMD0000000907)<br/><sup>HeberleRazquinNavas2019 - The </sup></summary><sup>HeberleRazquinNavas2019 - The PI3K and MAPK/p38 pathways control stress granuleassembly in a hierarchical manner model 3</sup></details>|<details><summary>pass</summary>[Navas2019_model3.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000907#Files)</details>|FAIL|<details><summary>pass</summary>[Navas2019_model3.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000907#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Navas2019_model3.sedml50807.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Navas2019_model3.xml` has warnings.\r\n - 58 warnings of type SBML unit consistency (99505). The following is the first warning at line 2676, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'init_X2_0_MCF7_Wm' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 3032, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'reaction4') are litre (exponent = -2, multiplier = 1, scale = 0), mole (exponent = 2, multiplier = 1, scale = 0).\r\n - 68 warnings of type Modeling practice (80701). The following is the first warning at line 2444, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a1_X10_0' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Navas2019_model3.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Navas2019_model3.xml` has warnings.\r\n - 58 warnings of type SBML unit consistency (99505). The following is the first warning at line 2676, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'init_X2_0_MCF7_Wm' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 3032, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'reaction4') are litre (exponent = -2, multiplier = 1, scale = 0), mole (exponent = 2, multiplier = 1, scale = 0).\r\n - 68 warnings of type Modeling practice (80701). The following is the first warning at line 2444, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a1_X10_0' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Navas2019_model3.sedml50807.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Navas2019_model3.sedml` is invalid.\n - Data generator `X1_0_1_task1` is invalid.\n - Variable `p1_X1_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X1_0']` are not defined. No namespaces are defined for the target.\n - Data generator `X1_1_2_task1` is invalid.\n - Variable `p1_X1_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X1_1']` are not defined. No namespaces are defined for the target.\n - Data generator `X2_0_3_task1` is invalid.\n - Variable `p1_X2_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X2_0']` are not defined. No namespaces are defined for the target.\n - Data generator `X2_1_4_task1` is invalid.\n - Variable `p1_X2_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X2_1']` are not defined. No namespaces are defined for the target.\n - Data generator `X2_2_5_task1` is invalid.\n - Variable `p1_X2_2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X2_2']` are not defined. No namespaces are defined for the target.\n - Data generator `X4_0_6_task1` is invalid.\n - Variable `p1_X4_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X4_0']` are not defined. No namespaces are defined for the target.\n - Data generator `X4_1_7_task1` is invalid.\n - Variable `p1_X4_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X4_1']` are not defined. No namespaces are defined for the target.\n - Data generator `X5_0_8_task1` is invalid.\n - Variable `p1_X5_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X5_0']` are not defined. No namespaces are defined for the target.\n - Data generator `X5_1_9_task1` is invalid.\n - Variable `p1_X5_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X5_1']` are not defined. No namespaces are defined for the target.\n - Data generator `X8_0_10_task1` is invalid.\n - Variable `p1_X8_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X8_0']` are not defined. No namespaces are defined for the target.\n - Data generator `X8_1_11_task1` is invalid.\n - Variable `p1_X8_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X8_1']` are not defined. No namespaces are defined for the target.\n - Data generator `X8_2_12_task1` is invalid.\n - Variable `p1_X8_2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X8_2']` are not defined. No namespaces are defined for the target.\n - Data generator `X8_3_13_task1` is invalid.\n - Variable `p1_X8_3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X8_3']` are not defined. No namespaces are defined for the target.\n - Data generator `X9_0_14_task1` is invalid.\n - Variable `p1_X9_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X9_0']` are not defined. No namespaces are defined for the target.\n - Data generator `X9_2_15_task1` is invalid.\n - Variable `p1_X9_2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X9_2']` are not defined. No namespaces are defined for the target.\n - Data generator `X10_0_16_task1` is invalid.\n - Variable `p1_X10_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X10_0']` are not defined. No namespaces are defined for the target.\n - Data generator `X10_1_17_task1` is invalid.\n - Variable `p1_X10_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X10_1']` are not defined. No namespaces are defined for the target.\n - Data generator `X10_2_18_task1` is invalid.\n - Variable `p1_X10_2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X10_2']` are not defined. No namespaces are defined for the target.\n - Data generator `X10_3_19_task1` is invalid.\n - Variable `p1_X10_3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X10_3']` are not defined. No namespaces are defined for the target.\n - Data generator `X11_0_20_task1` is invalid.\n - Variable `p1_X11_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X11_0']` are not defined. No namespaces are defined for the target.\n - Data generator `X11_1_21_task1` is invalid.\n - Variable `p1_X11_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X11_1']` are not defined. No namespaces are defined for the target.\n - Data generator `X11_2_22_task1` is invalid.\n - Variable `p1_X11_2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X11_2']` are not defined. No namespaces are defined for the target.\n - Data generator `X11_3_23_task1` is invalid.\n - Variable `p1_X11_3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X11_3']` are not defined. No namespaces are defined for the target.\n - Data generator `X12_0_24_task1` is invalid.\n - Variable `p1_X12_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X12_0']` are not defined. No namespaces are defined for the target.\n - Data generator `X12_1_25_task1` is invalid.\n - Variable `p1_X12_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X12_1']` are not defined. No namespaces are defined for the target.\n - Data generator `Akt_pT308_obs_26_task1` is invalid.\n - Variable `p1_Akt_pT308_obs_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Akt_pT308_obs']` are not defined. No namespaces are defined for the target.\n - Data generator `Akt_pS473_obs_27_task1` is invalid.\n - Variable `p1_Akt_pS473_obs_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Akt_pS473_obs']` are not defined. No namespaces are defined for the target.\n - Data generator `TSC1_TSC2_pT1462_obs_28_task1` is invalid.\n - Variable `p1_TSC1_TSC2_pT1462_obs_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='TSC1_TSC2_pT1462_obs']` are not defined. No namespaces are defined for the target.\n - Data generator `PRAS40_pT246_obs_29_task1` is invalid.\n - Variable `p1_PRAS40_pT246_obs_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PRAS40_pT246_obs']` are not defined. No namespaces are defined for the target.\n - Data generator `PRAS40_pS183_obs_30_task1` is invalid.\n - Variable `p1_PRAS40_pS183_obs_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PRAS40_pS183_obs']` are not defined. No namespaces are defined for the target.\n - Data generator `fourEBP1_pT37_46_obs_31_task1` is invalid.\n - Variable `p1_fourEBP1_pT37_46_obs_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='fourEBP1_pT37_46_obs']` are not defined. No namespaces are defined for the target.\n - Data generator `p70_S6K_pT389_obs_32_task1` is invalid.\n - Variable `p1_p70_S6K_pT389_obs_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='p70_S6K_pT389_obs']` are not defined. No namespaces are defined for the target.\n - Data generator `p70_S6K_pT229_obs_33_task1` is invalid.\n - Variable `p1_p70_S6K_pT229_obs_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='p70_S6K_pT229_obs']` are not defined. No namespaces are defined for the target.\n - Data generator `IRS1_pS636_639_obs_34_task1` is invalid.\n - Variable `p1_IRS1_pS636_639_obs_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IRS1_pS636_639_obs']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Navas2019_model3.sedml541670.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Navas2019_model3.xml` has warnings.\r\n - 58 warnings of type SBML unit consistency (99505). The following is the first warning at line 2676, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'init_X2_0_MCF7_Wm' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 3032, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'reaction4') are litre (exponent = -2, multiplier = 1, scale = 0), mole (exponent = 2, multiplier = 1, scale = 0).\r\n - 68 warnings of type Modeling practice (80701). The following is the first warning at line 2444, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a1_X10_0' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Navas2019_model3.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Navas2019_model3.xml` has warnings.\r\n - 58 warnings of type SBML unit consistency (99505). The following is the first warning at line 2676, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'init_X2_0_MCF7_Wm' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 3032, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 1, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'reaction4') are litre (exponent = -2, multiplier = 1, scale = 0), mole (exponent = 2, multiplier = 1, scale = 0).\r\n - 68 warnings of type Modeling practice (80701). The following is the first warning at line 2444, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a1_X10_0' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Navas2019_model3.sedml541670.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Navas2019_model3.sedml` is invalid.\n - Data generator `X1_0_1_task1` is invalid.\n - Variable `p1_X1_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X1_0']` are not defined. No namespaces are defined for the target.\n - Data generator `X1_1_2_task1` is invalid.\n - Variable `p1_X1_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X1_1']` are not defined. No namespaces are defined for the target.\n - Data generator `X2_0_3_task1` is invalid.\n - Variable `p1_X2_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X2_0']` are not defined. No namespaces are defined for the target.\n - Data generator `X2_1_4_task1` is invalid.\n - Variable `p1_X2_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X2_1']` are not defined. No namespaces are defined for the target.\n - Data generator `X2_2_5_task1` is invalid.\n - Variable `p1_X2_2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X2_2']` are not defined. No namespaces are defined for the target.\n - Data generator `X4_0_6_task1` is invalid.\n - Variable `p1_X4_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X4_0']` are not defined. No namespaces are defined for the target.\n - Data generator `X4_1_7_task1` is invalid.\n - Variable `p1_X4_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X4_1']` are not defined. No namespaces are defined for the target.\n - Data generator `X5_0_8_task1` is invalid.\n - Variable `p1_X5_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X5_0']` are not defined. No namespaces are defined for the target.\n - Data generator `X5_1_9_task1` is invalid.\n - Variable `p1_X5_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X5_1']` are not defined. No namespaces are defined for the target.\n - Data generator `X8_0_10_task1` is invalid.\n - Variable `p1_X8_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X8_0']` are not defined. No namespaces are defined for the target.\n - Data generator `X8_1_11_task1` is invalid.\n - Variable `p1_X8_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X8_1']` are not defined. No namespaces are defined for the target.\n - Data generator `X8_2_12_task1` is invalid.\n - Variable `p1_X8_2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X8_2']` are not defined. No namespaces are defined for the target.\n - Data generator `X8_3_13_task1` is invalid.\n - Variable `p1_X8_3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X8_3']` are not defined. No namespaces are defined for the target.\n - Data generator `X9_0_14_task1` is invalid.\n - Variable `p1_X9_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X9_0']` are not defined. No namespaces are defined for the target.\n - Data generator `X9_2_15_task1` is invalid.\n - Variable `p1_X9_2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X9_2']` are not defined. No namespaces are defined for the target.\n - Data generator `X10_0_16_task1` is invalid.\n - Variable `p1_X10_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X10_0']` are not defined. No namespaces are defined for the target.\n - Data generator `X10_1_17_task1` is invalid.\n - Variable `p1_X10_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X10_1']` are not defined. No namespaces are defined for the target.\n - Data generator `X10_2_18_task1` is invalid.\n - Variable `p1_X10_2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X10_2']` are not defined. No namespaces are defined for the target.\n - Data generator `X10_3_19_task1` is invalid.\n - Variable `p1_X10_3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X10_3']` are not defined. No namespaces are defined for the target.\n - Data generator `X11_0_20_task1` is invalid.\n - Variable `p1_X11_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X11_0']` are not defined. No namespaces are defined for the target.\n - Data generator `X11_1_21_task1` is invalid.\n - Variable `p1_X11_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X11_1']` are not defined. No namespaces are defined for the target.\n - Data generator `X11_2_22_task1` is invalid.\n - Variable `p1_X11_2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X11_2']` are not defined. No namespaces are defined for the target.\n - Data generator `X11_3_23_task1` is invalid.\n - Variable `p1_X11_3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X11_3']` are not defined. No namespaces are defined for the target.\n - Data generator `X12_0_24_task1` is invalid.\n - Variable `p1_X12_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X12_0']` are not defined. No namespaces are defined for the target.\n - Data generator `X12_1_25_task1` is invalid.\n - Variable `p1_X12_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X12_1']` are not defined. No namespaces are defined for the target.\n - Data generator `Akt_pT308_obs_26_task1` is invalid.\n - Variable `p1_Akt_pT308_obs_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Akt_pT308_obs']` are not defined. No namespaces are defined for the target.\n - Data generator `Akt_pS473_obs_27_task1` is invalid.\n - Variable `p1_Akt_pS473_obs_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Akt_pS473_obs']` are not defined. No namespaces are defined for the target.\n - Data generator `TSC1_TSC2_pT1462_obs_28_task1` is invalid.\n - Variable `p1_TSC1_TSC2_pT1462_obs_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='TSC1_TSC2_pT1462_obs']` are not defined. No namespaces are defined for the target.\n - Data generator `PRAS40_pT246_obs_29_task1` is invalid.\n - Variable `p1_PRAS40_pT246_obs_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PRAS40_pT246_obs']` are not defined. No namespaces are defined for the target.\n - Data generator `PRAS40_pS183_obs_30_task1` is invalid.\n - Variable `p1_PRAS40_pS183_obs_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='PRAS40_pS183_obs']` are not defined. No namespaces are defined for the target.\n - Data generator `fourEBP1_pT37_46_obs_31_task1` is invalid.\n - Variable `p1_fourEBP1_pT37_46_obs_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='fourEBP1_pT37_46_obs']` are not defined. No namespaces are defined for the target.\n - Data generator `p70_S6K_pT389_obs_32_task1` is invalid.\n - Variable `p1_p70_S6K_pT389_obs_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='p70_S6K_pT389_obs']` are not defined. No namespaces are defined for the target.\n - Data generator `p70_S6K_pT229_obs_33_task1` is invalid.\n - Variable `p1_p70_S6K_pT229_obs_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='p70_S6K_pT229_obs']` are not defined. No namespaces are defined for the target.\n - Data generator `IRS1_pS636_639_obs_34_task1` is invalid.\n - Variable `p1_IRS1_pS636_639_obs_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IRS1_pS636_639_obs']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000908](https://www.ebi.ac.uk/biomodels/BIOMD0000000908)<br/><sup>dePillis2013 - Mathematical mo</sup></summary><sup>dePillis2013 - Mathematical modeling of regulatory T cell effects on renal cell carcinoma treatment</sup></details>|<details><summary>pass</summary>[dePillis2013.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000908#Files)</details>|pass|<details><summary>pass</summary>[dePillis2013.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000908#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/dePillis2013.sedml798179.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `dePillis2013.xml` has warnings.\r\n - 17 warnings of type SBML unit consistency (99505). The following is the first warning at line 1110, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'd' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 43 warnings of type Modeling practice (80701). The following is the first warning at line 1007, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'D' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./dePillis2013.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `dePillis2013.xml` has warnings.\r\n - 17 warnings of type SBML unit consistency (99505). The following is the first warning at line 1110, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'd' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 43 warnings of type Modeling practice (80701). The following is the first warning at line 1007, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'D' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/dePillis2013.sedml798179.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./dePillis2013.sedml` is invalid.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `N_2_task1` is invalid.\n - Variable `p1_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n - Data generator `L_3_task1` is invalid.\n - Variable `p1_L_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L']` are not defined. No namespaces are defined for the target.\n - Data generator `R_4_task1` is invalid.\n - Variable `p1_R_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='R']` are not defined. No namespaces are defined for the target.\n - Data generator `C_5_task1` is invalid.\n - Variable `p1_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C']` are not defined. No namespaces are defined for the target.\n - Data generator `I_6_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `S_7_task1` is invalid.\n - Variable `p1_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S']` are not defined. No namespaces are defined for the target.\n - Data generator `D_8_task1` is invalid.\n - Variable `p1_D_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='D']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/dePillis2013.sedml772722.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `dePillis2013.xml` has warnings.\r\n - 17 warnings of type SBML unit consistency (99505). The following is the first warning at line 1110, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'd' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 43 warnings of type Modeling practice (80701). The following is the first warning at line 1007, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'D' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./dePillis2013.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `dePillis2013.xml` has warnings.\r\n - 17 warnings of type SBML unit consistency (99505). The following is the first warning at line 1110, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'd' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 43 warnings of type Modeling practice (80701). The following is the first warning at line 1007, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'D' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/dePillis2013.sedml772722.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./dePillis2013.sedml` is invalid.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `N_2_task1` is invalid.\n - Variable `p1_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n - Data generator `L_3_task1` is invalid.\n - Variable `p1_L_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L']` are not defined. No namespaces are defined for the target.\n - Data generator `R_4_task1` is invalid.\n - Variable `p1_R_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='R']` are not defined. No namespaces are defined for the target.\n - Data generator `C_5_task1` is invalid.\n - Variable `p1_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C']` are not defined. No namespaces are defined for the target.\n - Data generator `I_6_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `S_7_task1` is invalid.\n - Variable `p1_S_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S']` are not defined. No namespaces are defined for the target.\n - Data generator `D_8_task1` is invalid.\n - Variable `p1_D_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='D']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000909](https://www.ebi.ac.uk/biomodels/BIOMD0000000909)<br/><sup>dePillis2003 - The dynamics of</sup></summary><sup>dePillis2003 - The dynamics of an optimally controlled tumor model: A case study</sup></details>|<details><summary>pass</summary>[dePillis2003.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000909#Files)</details>|FAIL|<details><summary>pass</summary>[dePillis2003.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000909#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/dePillis2003.sedml890022.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `dePillis2003.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 722, column 8:\r\n - The formula 'exp(-u)' in the math element of the <kineticLaw> uses a function which can only act on dimensionless variables.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 673, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_presence_of_normal_cell(r2, N, b2)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 17 warnings of type Modeling practice (80701). The following is the first warning at line 624, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a1' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./dePillis2003.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `dePillis2003.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 722, column 8:\r\n - The formula 'exp(-u)' in the math element of the <kineticLaw> uses a function which can only act on dimensionless variables.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 673, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_presence_of_normal_cell(r2, N, b2)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 17 warnings of type Modeling practice (80701). The following is the first warning at line 624, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a1' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/dePillis2003.sedml890022.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./dePillis2003.sedml` is invalid.\n - Data generator `N_1_task1` is invalid.\n - Variable `p1_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n - Data generator `T_2_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `I_3_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `u_4_task1` is invalid.\n - Variable `p1_u_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='u']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/dePillis2003.sedml751915.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `dePillis2003.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 722, column 8:\r\n - The formula 'exp(-u)' in the math element of the <kineticLaw> uses a function which can only act on dimensionless variables.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 673, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_presence_of_normal_cell(r2, N, b2)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 17 warnings of type Modeling practice (80701). The following is the first warning at line 624, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./dePillis2003.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `dePillis2003.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (10501). The following is the first warning at line 722, column 8:\r\n - The formula 'exp(-u)' in the math element of the <kineticLaw> uses a function which can only act on dimensionless variables.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 673, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_presence_of_normal_cell(r2, N, b2)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 17 warnings of type Modeling practice (80701). The following is the first warning at line 624, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/dePillis2003.sedml751915.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./dePillis2003.sedml` is invalid.\n - Data generator `N_1_task1` is invalid.\n - Variable `p1_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n - Data generator `T_2_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `I_3_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `u_4_task1` is invalid.\n - Variable `p1_u_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='u']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000910](https://www.ebi.ac.uk/biomodels/BIOMD0000000910)<br/><sup>Isaeva2008 - Modelling of Anti</sup></summary><sup>Isaeva2008 - Modelling of Anti-Tumour Immune Response Immunocorrective Effect of Weak Centimetre Electromagnetic Waves</sup></details>|<details><summary>pass</summary>[Isaeva2008.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000910#Files)</details>|pass|<details><summary>pass</summary>[Isaeva2008.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000910#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Isaeva2008.sedml753163.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Isaeva2008.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 486, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_activation_and_transfer_of_cytotoxic_T_lymphocyte_at_the_site_of_tumor(VL, alpha_L, L, I2)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 442, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha_T' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Isaeva2008.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Isaeva2008.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 486, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_activation_and_transfer_of_cytotoxic_T_lymphocyte_at_the_site_of_tumor(VL, alpha_L, L, I2)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 442, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha_T' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Isaeva2008.sedml753163.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Isaeva2008.sedml` is invalid.\n - Data generator `L_1_task1` is invalid.\n - Variable `p1_L_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L']` are not defined. No namespaces are defined for the target.\n - Data generator `T_2_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `I2_3_task1` is invalid.\n - Variable `p1_I2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I2']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Isaeva2008.sedml319037.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Isaeva2008.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 486, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_activation_and_transfer_of_cytotoxic_T_lymphocyte_at_the_site_of_tumor(VL, alpha_L, L, I2)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 442, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha_T' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Isaeva2008.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Isaeva2008.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 486, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_activation_and_transfer_of_cytotoxic_T_lymphocyte_at_the_site_of_tumor(VL, alpha_L, L, I2)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 442, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha_T' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Isaeva2008.sedml319037.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Isaeva2008.sedml` is invalid.\n - Data generator `L_1_task1` is invalid.\n - Variable `p1_L_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L']` are not defined. No namespaces are defined for the target.\n - Data generator `T_2_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `I2_3_task1` is invalid.\n - Variable `p1_I2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I2']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000911](https://www.ebi.ac.uk/biomodels/BIOMD0000000911)<br/><sup>Merola2008 - An insight into t</sup></summary><sup>Merola2008 - An insight into tumor dormancy equilibrium via the analysis of its domain of attraction</sup></details>|<details><summary>pass</summary>[Merola2008.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000911#Files)</details>|pass|<details><summary>pass</summary>[Merola2008.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000911#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Merola2008.sedml8828.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Merola2008.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 615, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_increase_in_the_density_of_tumor_cell(q, r, M, k1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type Modeling practice (80701). The following is the first warning at line 449, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'q' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Merola2008.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Merola2008.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 615, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_increase_in_the_density_of_tumor_cell(q, r, M, k1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type Modeling practice (80701). The following is the first warning at line 449, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'q' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Merola2008.sedml8828.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Merola2008.sedml` is invalid.\n - Data generator `M_1_task1` is invalid.\n - Variable `p1_M_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M']` are not defined. No namespaces are defined for the target.\n - Data generator `N_2_task1` is invalid.\n - Variable `p1_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n - Data generator `Z_3_task1` is invalid.\n - Variable `p1_Z_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Z']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Merola2008.sedml11530.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Merola2008.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 615, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_increase_in_the_density_of_tumor_cell(q, r, M, k1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type Modeling practice (80701). The following is the first warning at line 449, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'q' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Merola2008.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Merola2008.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 615, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_increase_in_the_density_of_tumor_cell(q, r, M, k1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type Modeling practice (80701). The following is the first warning at line 449, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'q' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Merola2008.sedml11530.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Merola2008.sedml` is invalid.\n - Data generator `M_1_task1` is invalid.\n - Variable `p1_M_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M']` are not defined. No namespaces are defined for the target.\n - Data generator `N_2_task1` is invalid.\n - Variable `p1_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n - Data generator `Z_3_task1` is invalid.\n - Variable `p1_Z_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Z']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000912](https://www.ebi.ac.uk/biomodels/BIOMD0000000912)<br/><sup>Caravagna2010 - Tumour suppres</sup></summary><sup>Caravagna2010 - Tumour suppression by immune system</sup></details>|<details><summary>pass</summary>[Caravagna2010.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000912#Files)</details>|pass|<details><summary>FAIL</summary>[Caravagna2010.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000912#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Caravagna2010.sedml815906.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/Caravagna2010.sedml815906.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Caravagna2010.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmpl5_vzzvl/./model1.xml` is not a file.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `E_2_task1` is invalid.\n - Variable `p1_E_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='E']` are not defined. No namespaces are defined for the target.\n - Data generator `I_3_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Caravagna2010.sedml81756.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Caravagna2010.sedml81756.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Caravagna2010.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmp14qzhyan/./model1.xml` is not a file.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `E_2_task1` is invalid.\n - Variable `p1_E_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='E']` are not defined. No namespaces are defined for the target.\n - Data generator `I_3_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000913](https://www.ebi.ac.uk/biomodels/BIOMD0000000913)<br/><sup>dePillis2008 - Optimal control</sup></summary><sup>dePillis2008 - Optimal control of mixed immunotherapy and chemotherapy of tumors</sup></details>|<details><summary>pass</summary>[dePillis2008.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000913#Files)</details>|pass|<details><summary>pass</summary>[dePillis2008.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000913#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/dePillis2008.sedml701631.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `dePillis2008.xml` has warnings.\r\n - 16 warnings of type SBML unit consistency (99505). The following is the first warning at line 943, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'd' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 37 warnings of type Modeling practice (80701). The following is the first warning at line 881, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'd' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./dePillis2008.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `dePillis2008.xml` has warnings.\r\n - 16 warnings of type SBML unit consistency (99505). The following is the first warning at line 943, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'd' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 37 warnings of type Modeling practice (80701). The following is the first warning at line 881, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'd' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/dePillis2008.sedml701631.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./dePillis2008.sedml` is invalid.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `L_2_task1` is invalid.\n - Variable `p1_L_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L']` are not defined. No namespaces are defined for the target.\n - Data generator `N_3_task1` is invalid.\n - Variable `p1_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n - Data generator `C_4_task1` is invalid.\n - Variable `p1_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C']` are not defined. No namespaces are defined for the target.\n - Data generator `I_5_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `M_6_task1` is invalid.\n - Variable `p1_M_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M']` are not defined. No namespaces are defined for the target.\n - Data generator `D_7_task1` is invalid.\n - Variable `p1_D_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='D']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/dePillis2008.sedml904313.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `dePillis2008.xml` has warnings.\r\n - 16 warnings of type SBML unit consistency (99505). The following is the first warning at line 943, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'd' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 37 warnings of type Modeling practice (80701). The following is the first warning at line 881, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'd' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./dePillis2008.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `dePillis2008.xml` has warnings.\r\n - 16 warnings of type SBML unit consistency (99505). The following is the first warning at line 943, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'd' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 37 warnings of type Modeling practice (80701). The following is the first warning at line 881, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'd' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/dePillis2008.sedml904313.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./dePillis2008.sedml` is invalid.\n - Data generator `T_1_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n - Data generator `L_2_task1` is invalid.\n - Variable `p1_L_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L']` are not defined. No namespaces are defined for the target.\n - Data generator `N_3_task1` is invalid.\n - Variable `p1_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n - Data generator `C_4_task1` is invalid.\n - Variable `p1_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C']` are not defined. No namespaces are defined for the target.\n - Data generator `I_5_task1` is invalid.\n - Variable `p1_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` are not defined. No namespaces are defined for the target.\n - Data generator `M_6_task1` is invalid.\n - Variable `p1_M_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M']` are not defined. No namespaces are defined for the target.\n - Data generator `D_7_task1` is invalid.\n - Variable `p1_D_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='D']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000914](https://www.ebi.ac.uk/biomodels/BIOMD0000000914)<br/><sup>Parra_Guillen2013 -  Mathemati</sup></summary><sup>Parra_Guillen2013 -  Mathematical model approach to describe tumour response in mice after vaccine administration_model1</sup></details>|<details><summary>pass</summary>[Parra_Guillen2013.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000914#Files)</details>|FAIL|<details><summary>pass</summary>[Parra_Guillen2013.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000914#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Parra_Guillen2013.sedml694501.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Parra_Guillen2013.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 794, column 8:\r\n - The formula 'pow(REG, gamma)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 575, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_removal_of_CyaA_E7_vaccine(k1, VAC)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 546, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lamda' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Parra_Guillen2013.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Parra_Guillen2013.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 794, column 8:\r\n - The formula 'pow(REG, gamma)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 575, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_removal_of_CyaA_E7_vaccine(k1, VAC)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 546, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lamda' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Parra_Guillen2013.sedml694501.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Parra_Guillen2013.sedml` is invalid.\n - Data generator `VAC_1_task1` is invalid.\n - Variable `p1_VAC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='VAC']` are not defined. No namespaces are defined for the target.\n - Data generator `TRAN_2_task1` is invalid.\n - Variable `p1_TRAN_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='TRAN']` are not defined. No namespaces are defined for the target.\n - Data generator `SVAC_3_task1` is invalid.\n - Variable `p1_SVAC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SVAC']` are not defined. No namespaces are defined for the target.\n - Data generator `Ts_4_task1` is invalid.\n - Variable `p1_Ts_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Ts']` are not defined. No namespaces are defined for the target.\n - Data generator `REG_5_task1` is invalid.\n - Variable `p1_REG_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='REG']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Parra_Guillen2013.sedml50153.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Parra_Guillen2013.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 794, column 8:\r\n - The formula 'pow(REG, gamma)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 575, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_removal_of_CyaA_E7_vaccine(k1, VAC)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 546, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lamda' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Parra_Guillen2013.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Parra_Guillen2013.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 794, column 8:\r\n - The formula 'pow(REG, gamma)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 575, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_removal_of_CyaA_E7_vaccine(k1, VAC)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 546, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lamda' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Parra_Guillen2013.sedml50153.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Parra_Guillen2013.sedml` is invalid.\n - Data generator `VAC_1_task1` is invalid.\n - Variable `p1_VAC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='VAC']` are not defined. No namespaces are defined for the target.\n - Data generator `TRAN_2_task1` is invalid.\n - Variable `p1_TRAN_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='TRAN']` are not defined. No namespaces are defined for the target.\n - Data generator `SVAC_3_task1` is invalid.\n - Variable `p1_SVAC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SVAC']` are not defined. No namespaces are defined for the target.\n - Data generator `Ts_4_task1` is invalid.\n - Variable `p1_Ts_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Ts']` are not defined. No namespaces are defined for the target.\n - Data generator `REG_5_task1` is invalid.\n - Variable `p1_REG_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='REG']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000915](https://www.ebi.ac.uk/biomodels/BIOMD0000000915)<br/><sup>Sun2018 - Instantaneous mutati</sup></summary><sup>Sun2018 - Instantaneous mutation rate in cancer initiation and progression</sup></details>|<details><summary>pass</summary>[Sun2018.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000915#Files)</details>|pass|<details><summary>pass</summary>[Sun2018.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000915#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Sun2018.sedml877858.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Sun2018.xml` has warnings.\r\n - 19 warnings of type SBML unit consistency (99505). The following is the first warning at line 1136, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'a_0 * exp(b_0 * time)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 30 warnings of type Modeling practice (80701). The following is the first warning at line 555, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'myu' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 2016.0\r\n - Number of steps: 56\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Sun2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Sun2018.xml` has warnings.\r\n - 19 warnings of type SBML unit consistency (99505). The following is the first warning at line 1136, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'a_0 * exp(b_0 * time)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 30 warnings of type Modeling practice (80701). The following is the first warning at line 555, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'myu' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 2016.0\r\n - Number of steps: 56\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/Sun2018.sedml877858.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Sun2018.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='b_0']` are not defined. No namespaces are defined for the target.\n - Data generator `DT1_1_task1` is invalid.\n - Variable `p2_DT1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='DT1']` are not defined. No namespaces are defined for the target.\n - Data generator `DT2_2_task1` is invalid.\n - Variable `p2_DT2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='DT2']` are not defined. No namespaces are defined for the target.\n - Data generator `DT3_3_task1` is invalid.\n - Variable `p2_DT3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='DT3']` are not defined. No namespaces are defined for the target.\n - Data generator `DT4_4_task1` is invalid.\n - Variable `p2_DT4_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='DT4']` are not defined. No namespaces are defined for the target.\n - Data generator `DT5_5_task1` is invalid.\n - Variable `p2_DT5_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='DT5']` are not defined. No namespaces are defined for the target.\n - Data generator `DT6_6_task1` is invalid.\n - Variable `p2_DT6_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='DT6']` are not defined. No namespaces are defined for the target.\n - Data generator `DT7_7_task1` is invalid.\n - Variable `p2_DT7_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='DT7']` are not defined. No namespaces are defined for the target.\n - Data generator `c_8_task1` is invalid.\n - Variable `p2_c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='c']` are not defined. No namespaces are defined for the target.\n - Data generator `DT1_1_task2` is invalid.\n - Variable `p2_DT1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='DT1']` are not defined. No namespaces are defined for the target.\n - Data generator `DT2_2_task2` is invalid.\n - Variable `p2_DT2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='DT2']` are not defined. No namespaces are defined for the target.\n - Data generator `DT3_3_task2` is invalid.\n - Variable `p2_DT3_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='DT3']` are not defined. No namespaces are defined for the target.\n - Data generator `DT4_4_task2` is invalid.\n - Variable `p2_DT4_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='DT4']` are not defined. No namespaces are defined for the target.\n - Data generator `DT5_5_task2` is invalid.\n - Variable `p2_DT5_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='DT5']` are not defined. No namespaces are defined for the target.\n - Data generator `DT6_6_task2` is invalid.\n - Variable `p2_DT6_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='DT6']` are not defined. No namespaces are defined for the target.\n - Data generator `DT7_7_task2` is invalid.\n - Variable `p2_DT7_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='DT7']` are not defined. No namespaces are defined for the target.\n - Data generator `c_8_task2` is invalid.\n - Variable `p2_c_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='c']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Sun2018.sedml603163.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Sun2018.xml` has warnings.\r\n - 19 warnings of type SBML unit consistency (99505). The following is the first warning at line 1136, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'a_0 * exp(b_0 * time)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 30 warnings of type Modeling practice (80701). The following is the first warning at line 555, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'myu' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 2016.0\r\n - Number of steps: 56\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Sun2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Sun2018.xml` has warnings.\r\n - 19 warnings of type SBML unit consistency (99505). The following is the first warning at line 1136, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'a_0 * exp(b_0 * time)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 30 warnings of type Modeling practice (80701). The following is the first warning at line 555, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'myu' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 2016.0\r\n - Number of steps: 56\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Sun2018.sedml603163.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Sun2018.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='b_0']` are not defined. No namespaces are defined for the target.\n - Data generator `DT1_1_task1` is invalid.\n - Variable `p2_DT1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='DT1']` are not defined. No namespaces are defined for the target.\n - Data generator `DT2_2_task1` is invalid.\n - Variable `p2_DT2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='DT2']` are not defined. No namespaces are defined for the target.\n - Data generator `DT3_3_task1` is invalid.\n - Variable `p2_DT3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='DT3']` are not defined. No namespaces are defined for the target.\n - Data generator `DT4_4_task1` is invalid.\n - Variable `p2_DT4_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='DT4']` are not defined. No namespaces are defined for the target.\n - Data generator `DT5_5_task1` is invalid.\n - Variable `p2_DT5_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='DT5']` are not defined. No namespaces are defined for the target.\n - Data generator `DT6_6_task1` is invalid.\n - Variable `p2_DT6_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='DT6']` are not defined. No namespaces are defined for the target.\n - Data generator `DT7_7_task1` is invalid.\n - Variable `p2_DT7_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='DT7']` are not defined. No namespaces are defined for the target.\n - Data generator `c_8_task1` is invalid.\n - Variable `p2_c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='c']` are not defined. No namespaces are defined for the target.\n - Data generator `DT1_1_task2` is invalid.\n - Variable `p2_DT1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='DT1']` are not defined. No namespaces are defined for the target.\n - Data generator `DT2_2_task2` is invalid.\n - Variable `p2_DT2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='DT2']` are not defined. No namespaces are defined for the target.\n - Data generator `DT3_3_task2` is invalid.\n - Variable `p2_DT3_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='DT3']` are not defined. No namespaces are defined for the target.\n - Data generator `DT4_4_task2` is invalid.\n - Variable `p2_DT4_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='DT4']` are not defined. No namespaces are defined for the target.\n - Data generator `DT5_5_task2` is invalid.\n - Variable `p2_DT5_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='DT5']` are not defined. No namespaces are defined for the target.\n - Data generator `DT6_6_task2` is invalid.\n - Variable `p2_DT6_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='DT6']` are not defined. No namespaces are defined for the target.\n - Data generator `DT7_7_task2` is invalid.\n - Variable `p2_DT7_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='DT7']` are not defined. No namespaces are defined for the target.\n - Data generator `c_8_task2` is invalid.\n - Variable `p2_c_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='c']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000916](https://www.ebi.ac.uk/biomodels/BIOMD0000000916)<br/><sup>Kraan199_Kinetics of Cortisol </sup></summary><sup>Kraan199_Kinetics of Cortisol Metabolism and Excretion.</sup></details>|<details><summary>pass</summary>[Kraan1992.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000916#Files)</details>|pass|<details><summary>pass</summary>[Kraan1992.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000916#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Kraan1992.sedml666880.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Kraan1992.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 631, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Rate_Law_for_R1(K3, The_FOD_in_the_circulation__X1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 5 warnings of type Modeling practice (80701). The following is the first warning at line 457, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K1' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Kraan1992.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Kraan1992.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 631, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Rate_Law_for_R1(K3, The_FOD_in_the_circulation__X1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 5 warnings of type Modeling practice (80701). The following is the first warning at line 457, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K1' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Kraan1992.sedml666880.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Kraan1992.sedml` is invalid.\n - Data generator `The_FOD_in_the_circulation__X1_1_task1` is invalid.\n - Variable `p1_The_FOD_in_the_circulation__X1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='The_FOD_in_the_circulation__X1']` are not defined. No namespaces are defined for the target.\n - Data generator `The_FOD_in_the_metabolizing_tissues__X4_2_task1` is invalid.\n - Variable `p1_The_FOD_in_the_metabolizing_tissues__X4_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='The_FOD_in_the_metabolizing_tissues__X4']` are not defined. No namespaces are defined for the target.\n - Data generator `The_cumulative_FOD_excreted_in_the_non_urinary_pool__X3_3_task1` is invalid.\n - Variable `p1_The_cumulative_FOD_excreted_in_the_non_urinary_pool__X3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='The_cumulative_FOD_excreted_in_the_non_urinary_pool__X3']` are not defined. No namespaces are defined for the target.\n - Data generator `The_FOD_in_the_gallbladder___intestinal_lumen__X5_4_task1` is invalid.\n - Variable `p1_The_FOD_in_the_gallbladder___intestinal_lumen__X5_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='The_FOD_in_the_gallbladder___intestinal_lumen__X5']` are not defined. No namespaces are defined for the target.\n - Data generator `The_cumulative_FOD_excreted_in_the_urine__X2_5_task1` is invalid.\n - Variable `p1_The_cumulative_FOD_excreted_in_the_urine__X2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='The_cumulative_FOD_excreted_in_the_urine__X2']` are not defined. No namespaces are defined for the target.\n - Data generator `The_cumulative_FOD_excreted_in_the_urine__X2_1_task1` is invalid.\n - Variable `p2_The_cumulative_FOD_excreted_in_the_urine__X2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='The_cumulative_FOD_excreted_in_the_urine__X2']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Kraan1992.sedml168018.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Kraan1992.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 631, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Rate_Law_for_R1(K3, The_FOD_in_the_circulation__X1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 5 warnings of type Modeling practice (80701). The following is the first warning at line 457, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Kraan1992.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Kraan1992.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 631, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Rate_Law_for_R1(K3, The_FOD_in_the_circulation__X1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 5 warnings of type Modeling practice (80701). The following is the first warning at line 457, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Kraan1992.sedml168018.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Kraan1992.sedml` is invalid.\n - Data generator `The_FOD_in_the_circulation__X1_1_task1` is invalid.\n - Variable `p1_The_FOD_in_the_circulation__X1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='The_FOD_in_the_circulation__X1']` are not defined. No namespaces are defined for the target.\n - Data generator `The_FOD_in_the_metabolizing_tissues__X4_2_task1` is invalid.\n - Variable `p1_The_FOD_in_the_metabolizing_tissues__X4_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='The_FOD_in_the_metabolizing_tissues__X4']` are not defined. No namespaces are defined for the target.\n - Data generator `The_cumulative_FOD_excreted_in_the_non_urinary_pool__X3_3_task1` is invalid.\n - Variable `p1_The_cumulative_FOD_excreted_in_the_non_urinary_pool__X3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='The_cumulative_FOD_excreted_in_the_non_urinary_pool__X3']` are not defined. No namespaces are defined for the target.\n - Data generator `The_FOD_in_the_gallbladder___intestinal_lumen__X5_4_task1` is invalid.\n - Variable `p1_The_FOD_in_the_gallbladder___intestinal_lumen__X5_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='The_FOD_in_the_gallbladder___intestinal_lumen__X5']` are not defined. No namespaces are defined for the target.\n - Data generator `The_cumulative_FOD_excreted_in_the_urine__X2_5_task1` is invalid.\n - Variable `p1_The_cumulative_FOD_excreted_in_the_urine__X2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='The_cumulative_FOD_excreted_in_the_urine__X2']` are not defined. No namespaces are defined for the target.\n - Data generator `The_cumulative_FOD_excreted_in_the_urine__X2_1_task1` is invalid.\n - Variable `p2_The_cumulative_FOD_excreted_in_the_urine__X2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='The_cumulative_FOD_excreted_in_the_urine__X2']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000917](https://www.ebi.ac.uk/biomodels/BIOMD0000000917)<br/><sup>Phillips2007_AscendingArousalS</sup></summary><sup>Phillips2007_AscendingArousalSystem_SleepWakeDynamics</sup></details>|<details><summary>pass</summary>[Phillips2007_Curated.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000917#Files)</details>|pass|<details><summary>FAIL</summary>[Phillips2007.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000917#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Phillips2007.sedml534069.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 72.00000000576\r\n - Number of steps: 1296\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_5_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_6_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Phillips2007.sedml` has warnings.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 72.00000000576\r\n - Number of steps: 1296\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_5_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_6_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/Phillips2007.sedml534069.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Phillips2007.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmps_dp8syj/./model1.xml` is not a file.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='chi']` are not defined. No namespaces are defined for the target.\n - Data generator `Monoaminergic__MA__voltage_1_task1` is invalid.\n - Variable `p4_Monoaminergic__MA__voltage_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Monoaminergic__MA__voltage']` are not defined. No namespaces are defined for the target.\n - Data generator `Ventrolateral_preopticarea__VLPO__voltage_1_task1` is invalid.\n - Variable `p5_Ventrolateral_preopticarea__VLPO__voltage_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Ventrolateral_preopticarea__VLPO__voltage']` are not defined. No namespaces are defined for the target.\n - Data generator `Somnogen_level_H_1_task1` is invalid.\n - Variable `p6_Somnogen_level_H_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Somnogen_level_H']` are not defined. No namespaces are defined for the target.\n - Data generator `Monoaminergic__MA__voltage_1_task2` is invalid.\n - Variable `p4_Monoaminergic__MA__voltage_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Monoaminergic__MA__voltage']` are not defined. No namespaces are defined for the target.\n - Data generator `Ventrolateral_preopticarea__VLPO__voltage_1_task2` is invalid.\n - Variable `p5_Ventrolateral_preopticarea__VLPO__voltage_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Ventrolateral_preopticarea__VLPO__voltage']` are not defined. No namespaces are defined for the target.\n - Data generator `Somnogen_level_H_1_task2` is invalid.\n - Variable `p6_Somnogen_level_H_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Somnogen_level_H']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Phillips2007.sedml452671.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 72.00000000576\r\n - Number of steps: 1296\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_5_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_6_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Phillips2007.sedml` has warnings.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 72.00000000576\r\n - Number of steps: 1296\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_5_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_6_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Phillips2007.sedml452671.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Phillips2007.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmpxpeja7ru/./model1.xml` is not a file.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='chi']` are not defined. No namespaces are defined for the target.\n - Data generator `Monoaminergic__MA__voltage_1_task1` is invalid.\n - Variable `p4_Monoaminergic__MA__voltage_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Monoaminergic__MA__voltage']` are not defined. No namespaces are defined for the target.\n - Data generator `Ventrolateral_preopticarea__VLPO__voltage_1_task1` is invalid.\n - Variable `p5_Ventrolateral_preopticarea__VLPO__voltage_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Ventrolateral_preopticarea__VLPO__voltage']` are not defined. No namespaces are defined for the target.\n - Data generator `Somnogen_level_H_1_task1` is invalid.\n - Variable `p6_Somnogen_level_H_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Somnogen_level_H']` are not defined. No namespaces are defined for the target.\n - Data generator `Monoaminergic__MA__voltage_1_task2` is invalid.\n - Variable `p4_Monoaminergic__MA__voltage_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Monoaminergic__MA__voltage']` are not defined. No namespaces are defined for the target.\n - Data generator `Ventrolateral_preopticarea__VLPO__voltage_1_task2` is invalid.\n - Variable `p5_Ventrolateral_preopticarea__VLPO__voltage_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Ventrolateral_preopticarea__VLPO__voltage']` are not defined. No namespaces are defined for the target.\n - Data generator `Somnogen_level_H_1_task2` is invalid.\n - Variable `p6_Somnogen_level_H_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Somnogen_level_H']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000918](https://www.ebi.ac.uk/biomodels/BIOMD0000000918)<br/><sup>Schwarz2018-Cdk Activity Thres</sup></summary><sup>Schwarz2018-Cdk Activity Threshold Determines Passage through the Restriction Point</sup></details>|<details><summary>pass</summary>[Schwarz2018.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000918#Files)</details>|pass|<details><summary>FAIL</summary>[Schwarz2018.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000918#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Schwarz2018.sedml605333.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/Schwarz2018.sedml605333.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Schwarz2018.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmpdk41w1zh/./model1.xml` is not a file.\n - Data generator `CycE_1_task1` is invalid.\n - Variable `p1_CycE_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CycE']` are not defined. No namespaces are defined for the target.\n - Data generator `serum_2_task1` is invalid.\n - Variable `p1_serum_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='serum']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Schwarz2018.sedml951716.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Schwarz2018.sedml951716.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Schwarz2018.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmp2jycmchr/./model1.xml` is not a file.\n - Data generator `CycE_1_task1` is invalid.\n - Variable `p1_CycE_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CycE']` are not defined. No namespaces are defined for the target.\n - Data generator `serum_2_task1` is invalid.\n - Variable `p1_serum_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='serum']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000919](https://www.ebi.ac.uk/biomodels/BIOMD0000000919)<br/><sup>Ledzewicz2013 - On optimal che</sup></summary><sup>Ledzewicz2013 - On optimal chemotherapy with a strongly targeted agent for a model of tumor immune system interactions with generalized logistic growth</sup></details>|<details><summary>pass</summary>[Ledzewicz2013.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000919#Files)</details>|pass|<details><summary>FAIL</summary>[Ledzewicz2013.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000919#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Ledzewicz2013.sedml551515.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Ledzewicz2013.sedml` has warnings.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n\r\n`/root/in/Ledzewicz2013.sedml551515.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Ledzewicz2013.sedml` is invalid.\n - Each identified SED object must have a unique id. Multiple objects have the following ids:\n - p1_v_task1\n - p1_v_task3\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmpvmn6ri_h/./model1.xml` is not a file.\n - Task `task3` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='v']` are not defined. No namespaces are defined for the target.\n - Data generator `x_1_task1` is invalid.\n - Variable `p1_x_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x']` are not defined. No namespaces are defined for the target.\n - Data generator `v_1_task1` is invalid.\n - Variable `p1_v_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='v']` are not defined. No namespaces are defined for the target.\n - Data generator `y_2_task1` is invalid.\n - Variable `p1_y_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='y']` are not defined. No namespaces are defined for the target.\n - Data generator `v_2_task1` is invalid.\n - Variable `p1_v_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='v']` are not defined. No namespaces are defined for the target.\n - Data generator `x_1_task3` is invalid.\n - Variable `p1_x_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x']` are not defined. No namespaces are defined for the target.\n - Data generator `v_1_task3` is invalid.\n - Variable `p1_v_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='v']` are not defined. No namespaces are defined for the target.\n - Data generator `y_2_task3` is invalid.\n - Variable `p1_y_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='y']` are not defined. No namespaces are defined for the target.\n - Data generator `v_2_task3` is invalid.\n - Variable `p1_v_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='v']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Ledzewicz2013.sedml718024.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Ledzewicz2013.sedml` has warnings.\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Ledzewicz2013.sedml718024.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Ledzewicz2013.sedml` is invalid.\n - Each identified SED object must have a unique id. Multiple objects have the following ids:\n - p1_v_task1\n - p1_v_task3\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmpt_hsndmj/./model1.xml` is not a file.\n - Task `task3` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='v']` are not defined. No namespaces are defined for the target.\n - Data generator `x_1_task1` is invalid.\n - Variable `p1_x_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x']` are not defined. No namespaces are defined for the target.\n - Data generator `v_1_task1` is invalid.\n - Variable `p1_v_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='v']` are not defined. No namespaces are defined for the target.\n - Data generator `y_2_task1` is invalid.\n - Variable `p1_y_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='y']` are not defined. No namespaces are defined for the target.\n - Data generator `v_2_task1` is invalid.\n - Variable `p1_v_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='v']` are not defined. No namespaces are defined for the target.\n - Data generator `x_1_task3` is invalid.\n - Variable `p1_x_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x']` are not defined. No namespaces are defined for the target.\n - Data generator `v_1_task3` is invalid.\n - Variable `p1_v_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='v']` are not defined. No namespaces are defined for the target.\n - Data generator `y_2_task3` is invalid.\n - Variable `p1_y_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='y']` are not defined. No namespaces are defined for the target.\n - Data generator `v_2_task3` is invalid.\n - Variable `p1_v_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='v']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000920](https://www.ebi.ac.uk/biomodels/BIOMD0000000920)<br/><sup>Jarrett2015 - Modelling the in</sup></summary><sup>Jarrett2015 - Modelling the interaction between immune response, bacterial dynamics and inflammatory damage</sup></details>|<details><summary>pass</summary>[Jarrett2015.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000920#Files)</details>|pass|<details><summary>pass</summary>[Jarrett2015.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000920#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Jarrett2015.sedml182370.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Jarrett2015.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 1051, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'BALB_c_Mouse * Rate_Law_for_reaction_for_A(alpha_2, pro_inflammatory__P, beta_2, inflammation__I, mu_2, bacterial_infection__B, anti_inflammatory__A, K_B)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 16 warnings of type Modeling practice (80701). The following is the first warning at line 691, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha_1' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Jarrett2015.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Jarrett2015.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 1051, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'BALB_c_Mouse * Rate_Law_for_reaction_for_A(alpha_2, pro_inflammatory__P, beta_2, inflammation__I, mu_2, bacterial_infection__B, anti_inflammatory__A, K_B)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 16 warnings of type Modeling practice (80701). The following is the first warning at line 691, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha_1' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Jarrett2015.sedml182370.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Jarrett2015.sedml` is invalid.\n - Data generator `anti_inflammatory__A_1_task1` is invalid.\n - Variable `p1_anti_inflammatory__A_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='anti_inflammatory__A']` are not defined. No namespaces are defined for the target.\n - Data generator `bacterial_infection__B_2_task1` is invalid.\n - Variable `p1_bacterial_infection__B_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='bacterial_infection__B']` are not defined. No namespaces are defined for the target.\n - Data generator `inflammation__I_3_task1` is invalid.\n - Variable `p1_inflammation__I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='inflammation__I']` are not defined. No namespaces are defined for the target.\n - Data generator `pro_inflammatory__P_4_task1` is invalid.\n - Variable `p1_pro_inflammatory__P_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='pro_inflammatory__P']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Jarrett2015.sedml627724.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Jarrett2015.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 1051, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'BALB_c_Mouse * Rate_Law_for_reaction_for_A(alpha_2, pro_inflammatory__P, beta_2, inflammation__I, mu_2, bacterial_infection__B, anti_inflammatory__A, K_B)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 16 warnings of type Modeling practice (80701). The following is the first warning at line 691, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha_1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Jarrett2015.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Jarrett2015.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 1051, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'BALB_c_Mouse * Rate_Law_for_reaction_for_A(alpha_2, pro_inflammatory__P, beta_2, inflammation__I, mu_2, bacterial_infection__B, anti_inflammatory__A, K_B)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 16 warnings of type Modeling practice (80701). The following is the first warning at line 691, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha_1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Jarrett2015.sedml627724.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Jarrett2015.sedml` is invalid.\n - Data generator `anti_inflammatory__A_1_task1` is invalid.\n - Variable `p1_anti_inflammatory__A_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='anti_inflammatory__A']` are not defined. No namespaces are defined for the target.\n - Data generator `bacterial_infection__B_2_task1` is invalid.\n - Variable `p1_bacterial_infection__B_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='bacterial_infection__B']` are not defined. No namespaces are defined for the target.\n - Data generator `inflammation__I_3_task1` is invalid.\n - Variable `p1_inflammation__I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='inflammation__I']` are not defined. No namespaces are defined for the target.\n - Data generator `pro_inflammatory__P_4_task1` is invalid.\n - Variable `p1_pro_inflammatory__P_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='pro_inflammatory__P']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000921](https://www.ebi.ac.uk/biomodels/BIOMD0000000921)<br/><sup>Khajanchi2017 - Uniform Persis</sup></summary><sup>Khajanchi2017 - Uniform Persistence and Global Stability for a Brain Tumor and Immune System Interaction</sup></details>|<details><summary>pass</summary>[khajanchi2017.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000921#Files)</details>|pass|<details><summary>pass</summary>[khajanchi2017.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000921#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/khajanchi2017.sedml411016.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `khajanchi2017.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (99505). The following is the first warning at line 1199, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_The_rate_of_change_of_glioma_whose_growth_follows_the_logistic_fashion_without_any_immune_intervention(r1, G, G_max)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 24 warnings of type Modeling practice (80701). The following is the first warning at line 806, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r1' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./khajanchi2017.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `khajanchi2017.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (99505). The following is the first warning at line 1199, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_The_rate_of_change_of_glioma_whose_growth_follows_the_logistic_fashion_without_any_immune_intervention(r1, G, G_max)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 24 warnings of type Modeling practice (80701). The following is the first warning at line 806, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r1' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/khajanchi2017.sedml411016.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./khajanchi2017.sedml` is invalid.\n - Data generator `G_1_task1` is invalid.\n - Variable `p1_G_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='G']` are not defined. No namespaces are defined for the target.\n - Data generator `M_2_task1` is invalid.\n - Variable `p1_M_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M']` are not defined. No namespaces are defined for the target.\n - Data generator `C_T_3_task1` is invalid.\n - Variable `p1_C_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C_T']` are not defined. No namespaces are defined for the target.\n - Data generator `T_beta_4_task1` is invalid.\n - Variable `p1_T_beta_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T_beta']` are not defined. No namespaces are defined for the target.\n - Data generator `I_gamma_5_task1` is invalid.\n - Variable `p1_I_gamma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I_gamma']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/khajanchi2017.sedml508300.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `khajanchi2017.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (99505). The following is the first warning at line 1199, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_The_rate_of_change_of_glioma_whose_growth_follows_the_logistic_fashion_without_any_immune_intervention(r1, G, G_max)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 24 warnings of type Modeling practice (80701). The following is the first warning at line 806, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./khajanchi2017.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `khajanchi2017.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (99505). The following is the first warning at line 1199, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Function_for_The_rate_of_change_of_glioma_whose_growth_follows_the_logistic_fashion_without_any_immune_intervention(r1, G, G_max)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 24 warnings of type Modeling practice (80701). The following is the first warning at line 806, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/khajanchi2017.sedml508300.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./khajanchi2017.sedml` is invalid.\n - Data generator `G_1_task1` is invalid.\n - Variable `p1_G_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='G']` are not defined. No namespaces are defined for the target.\n - Data generator `M_2_task1` is invalid.\n - Variable `p1_M_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M']` are not defined. No namespaces are defined for the target.\n - Data generator `C_T_3_task1` is invalid.\n - Variable `p1_C_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C_T']` are not defined. No namespaces are defined for the target.\n - Data generator `T_beta_4_task1` is invalid.\n - Variable `p1_T_beta_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T_beta']` are not defined. No namespaces are defined for the target.\n - Data generator `I_gamma_5_task1` is invalid.\n - Variable `p1_I_gamma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I_gamma']` are not defined. No namespaces are defined for the target.\n"```</details>|
|[BIOMD0000000922](https://www.ebi.ac.uk/biomodels/BIOMD0000000922)<br/><sup>Turner2015-Human/Mosquito ELP Model</sup>|<details><summary>pass</summary>[Turner2015_ELP.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000922#Files)</details>|pass|<details><summary>FAIL</summary>[Turner2015_ELP.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000922#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Turner2015_ELP.sedml492724.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Turner2015_ELP.sedml` has warnings.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n`/root/in/Turner2015_ELP.sedml492724.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Turner2015_ELP.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmpa5dq0bcv/./model1.xml` is not a file.\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Turner2015_ELP.sedml891980.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Turner2015_ELP.sedml` has warnings.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Turner2015_ELP.sedml891980.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Turner2015_ELP.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmp87hdlik9/./model1.xml` is not a file.\n'```</details>|
|<details><summary>[BIOMD0000000923](https://www.ebi.ac.uk/biomodels/BIOMD0000000923)<br/><sup>Li2012_Modelling osteomyeliti</sup></summary><sup>Li2012_Modelling osteomyelitis_Control Model</sup></details>|<details><summary>pass</summary>[Li2012_Modelling osteomyelitis_Control Model.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000923#Files)</details>|FAIL|<details><summary>pass</summary>[Li2012_Modelling osteomyelitis_Control Model.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000923#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Li2012_Modelling osteomyelitis_Control Model.sedml448352.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 2: b'usage: biosimulators-tellurium [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-tellurium: error: unrecognized arguments: osteomyelitis_Control Model.sedml448352.omex\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Li2012_Modelling osteomyelitis_Control Model.sedml729082.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 2: b'usage: biosimulators-copasi [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-copasi: error: unrecognized arguments: osteomyelitis_Control Model.sedml729082.omex\n'```</details>|
|<details><summary>[BIOMD0000000924](https://www.ebi.ac.uk/biomodels/BIOMD0000000924)<br/><sup>Smith2011 - Three Stage Innate</sup></summary><sup>Smith2011 - Three Stage Innate Immune Response to a Pneumococcal Lung Infection</sup></details>|<details><summary>pass</summary>[Smith2011_V1.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000924#Files)</details>|pass|<details><summary>MultipleSEDMLs</summary>['Smith2011_V1.sedml', 'Smith2011_V1_Fig5.sedml']</details>|NA|NA|NA|
|<details><summary>[BIOMD0000000925](https://www.ebi.ac.uk/biomodels/BIOMD0000000925)<br/><sup>Dunster2016 - Nondimensional C</sup></summary><sup>Dunster2016 - Nondimensional Coagulation Model</sup></details>|<details><summary>FAIL</summary>[Dunster2016_Nondimensional_Model_Curated.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000925#Files)</details>|FAIL|<details><summary>pass</summary>[Dunster2016_Nondimensional_Model1.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000925#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Dunster2016_Nondimensional_Model1.sedml607888.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0001\r\n - Output end time: 1000.0\r\n - Number of steps: 9999999\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Dunster2016_Nondimensional_Model1.sedml` has warnings.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0001\r\n - Output end time: 1000.0\r\n - Number of steps: 9999999\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n`/root/in/Dunster2016_Nondimensional_Model1.sedml607888.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Dunster2016_Nondimensional_Model1.sedml` is invalid.\n - Each identified SED object must have a unique id. Multiple objects have the following ids:\n - p1_log_time_task1\n - Model `model` is invalid.\n - The model file `Dunster2016_Nondimensional_Model_Curated.xml` is invalid.\n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout' cannot be used in this document.\n \n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout_L2' cannot be used in this document.\n \n - General SBML conformance (10102) at line 3, column 2: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L2V4 Section 4.1\n Element 'listOfLayouts' is not part of the definition of SBML Level 2 Version 4.\n \n - Data generator `APC_1_task1` is invalid.\n - Variable `p1_APC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='APC']` are not defined. No namespaces are defined for the target.\n - Data generator `log_time_1_task1` is invalid.\n - Variable `p1_log_time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_time']` are not defined. No namespaces are defined for the target.\n - Data generator `Factor_V_2_task1` is invalid.\n - Variable `p1_Factor_V_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Factor_V']` are not defined. No namespaces are defined for the target.\n - Data generator `log_time_2_task1` is invalid.\n - Variable `p1_log_time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_time']` are not defined. No namespaces are defined for the target.\n - Data generator `Factor_Va_3_task1` is invalid.\n - Variable `p1_Factor_Va_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Factor_Va']` are not defined. No namespaces are defined for the target.\n - Data generator `log_time_3_task1` is invalid.\n - Variable `p1_log_time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_time']` are not defined. No namespaces are defined for the target.\n - Data generator `Factor_Xi_4_task1` is invalid.\n - Variable `p1_Factor_Xi_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Factor_Xi']` are not defined. No namespaces are defined for the target.\n - Data generator `log_time_4_task1` is invalid.\n - Variable `p1_log_time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_time']` are not defined. No namespaces are defined for the target.\n - Data generator `FactorXa_5_task1` is invalid.\n - Variable `p1_FactorXa_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FactorXa']` are not defined. No namespaces are defined for the target.\n - Data generator `log_time_5_task1` is invalid.\n - Variable `p1_log_time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_time']` are not defined. No namespaces are defined for the target.\n - Data generator `Fibrin_6_task1` is invalid.\n - Variable `p1_Fibrin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Fibrin']` are not defined. No namespaces are defined for the target.\n - Data generator `log_time_6_task1` is invalid.\n - Variable `p1_log_time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_time']` are not defined. No namespaces are defined for the target.\n - Data generator `Fibrinogen_7_task1` is invalid.\n - Variable `p1_Fibrinogen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Fibrinogen']` are not defined. No namespaces are defined for the target.\n - Data generator `log_time_7_task1` is invalid.\n - Variable `p1_log_time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_time']` are not defined. No namespaces are defined for the target.\n - Data generator `Inactive_Factor_V_8_task1` is invalid.\n - Variable `p1_Inactive_Factor_V_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Inactive_Factor_V']` are not defined. No namespaces are defined for the target.\n - Data generator `log_time_8_task1` is invalid.\n - Variable `p1_log_time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_time']` are not defined. No namespaces are defined for the target.\n - Data generator `Inactive_Thrombin_9_task1` is invalid.\n - Variable `p1_Inactive_Thrombin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Inactive_Thrombin']` are not defined. No namespaces are defined for the target.\n - Data generator `log_time_9_task1` is invalid.\n - Variable `p1_log_time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_time']` are not defined. No namespaces are defined for the target.\n - Data generator `Inactive_protein_C_10_task1` is invalid.\n - Variable `p1_Inactive_protein_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Inactive_protein_C']` are not defined. No namespaces are defined for the target.\n - Data generator `log_time_10_task1` is invalid.\n - Variable `p1_log_time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_time']` are not defined. No namespaces are defined for the target.\n - Data generator `Protein_C_11_task1` is invalid.\n - Variable `p1_Protein_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Protein_C']` are not defined. No namespaces are defined for the target.\n - Data generator `log_time_11_task1` is invalid.\n - Variable `p1_log_time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_time']` are not defined. No namespaces are defined for the target.\n - Data generator `Prothrombin_12_task1` is invalid.\n - Variable `p1_Prothrombin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Prothrombin']` are not defined. No namespaces are defined for the target.\n - Data generator `log_time_12_task1` is invalid.\n - Variable `p1_log_time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_time']` are not defined. No namespaces are defined for the target.\n - Data generator `Prothrombinase_13_task1` is invalid.\n - Variable `p1_Prothrombinase_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Prothrombinase']` are not defined. No namespaces are defined for the target.\n - Data generator `log_time_13_task1` is invalid.\n - Variable `p1_log_time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_time']` are not defined. No namespaces are defined for the target.\n - Data generator `Thrombin_14_task1` is invalid.\n - Variable `p1_Thrombin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Thrombin']` are not defined. No namespaces are defined for the target.\n - Data generator `log_time_14_task1` is invalid.\n - Variable `p1_log_time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_time']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Dunster2016_Nondimensional_Model1.sedml854281.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0001\r\n - Output end time: 1000.0\r\n - Number of steps: 9999999\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Dunster2016_Nondimensional_Model1.sedml` has warnings.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0001\r\n - Output end time: 1000.0\r\n - Number of steps: 9999999\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Dunster2016_Nondimensional_Model1.sedml854281.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Dunster2016_Nondimensional_Model1.sedml` is invalid.\n - Each identified SED object must have a unique id. Multiple objects have the following ids:\n - p1_log_time_task1\n - Model `model` is invalid.\n - The model file `Dunster2016_Nondimensional_Model_Curated.xml` is invalid.\n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout' cannot be used in this document.\n \n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout_L2' cannot be used in this document.\n \n - General SBML conformance (10102) at line 3, column 2: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L2V4 Section 4.1\n Element 'listOfLayouts' is not part of the definition of SBML Level 2 Version 4.\n \n - Data generator `APC_1_task1` is invalid.\n - Variable `p1_APC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='APC']` are not defined. No namespaces are defined for the target.\n - Data generator `log_time_1_task1` is invalid.\n - Variable `p1_log_time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_time']` are not defined. No namespaces are defined for the target.\n - Data generator `Factor_V_2_task1` is invalid.\n - Variable `p1_Factor_V_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Factor_V']` are not defined. No namespaces are defined for the target.\n - Data generator `log_time_2_task1` is invalid.\n - Variable `p1_log_time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_time']` are not defined. No namespaces are defined for the target.\n - Data generator `Factor_Va_3_task1` is invalid.\n - Variable `p1_Factor_Va_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Factor_Va']` are not defined. No namespaces are defined for the target.\n - Data generator `log_time_3_task1` is invalid.\n - Variable `p1_log_time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_time']` are not defined. No namespaces are defined for the target.\n - Data generator `Factor_Xi_4_task1` is invalid.\n - Variable `p1_Factor_Xi_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Factor_Xi']` are not defined. No namespaces are defined for the target.\n - Data generator `log_time_4_task1` is invalid.\n - Variable `p1_log_time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_time']` are not defined. No namespaces are defined for the target.\n - Data generator `FactorXa_5_task1` is invalid.\n - Variable `p1_FactorXa_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FactorXa']` are not defined. No namespaces are defined for the target.\n - Data generator `log_time_5_task1` is invalid.\n - Variable `p1_log_time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_time']` are not defined. No namespaces are defined for the target.\n - Data generator `Fibrin_6_task1` is invalid.\n - Variable `p1_Fibrin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Fibrin']` are not defined. No namespaces are defined for the target.\n - Data generator `log_time_6_task1` is invalid.\n - Variable `p1_log_time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_time']` are not defined. No namespaces are defined for the target.\n - Data generator `Fibrinogen_7_task1` is invalid.\n - Variable `p1_Fibrinogen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Fibrinogen']` are not defined. No namespaces are defined for the target.\n - Data generator `log_time_7_task1` is invalid.\n - Variable `p1_log_time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_time']` are not defined. No namespaces are defined for the target.\n - Data generator `Inactive_Factor_V_8_task1` is invalid.\n - Variable `p1_Inactive_Factor_V_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Inactive_Factor_V']` are not defined. No namespaces are defined for the target.\n - Data generator `log_time_8_task1` is invalid.\n - Variable `p1_log_time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_time']` are not defined. No namespaces are defined for the target.\n - Data generator `Inactive_Thrombin_9_task1` is invalid.\n - Variable `p1_Inactive_Thrombin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Inactive_Thrombin']` are not defined. No namespaces are defined for the target.\n - Data generator `log_time_9_task1` is invalid.\n - Variable `p1_log_time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_time']` are not defined. No namespaces are defined for the target.\n - Data generator `Inactive_protein_C_10_task1` is invalid.\n - Variable `p1_Inactive_protein_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Inactive_protein_C']` are not defined. No namespaces are defined for the target.\n - Data generator `log_time_10_task1` is invalid.\n - Variable `p1_log_time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_time']` are not defined. No namespaces are defined for the target.\n - Data generator `Protein_C_11_task1` is invalid.\n - Variable `p1_Protein_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Protein_C']` are not defined. No namespaces are defined for the target.\n - Data generator `log_time_11_task1` is invalid.\n - Variable `p1_log_time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_time']` are not defined. No namespaces are defined for the target.\n - Data generator `Prothrombin_12_task1` is invalid.\n - Variable `p1_Prothrombin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Prothrombin']` are not defined. No namespaces are defined for the target.\n - Data generator `log_time_12_task1` is invalid.\n - Variable `p1_log_time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_time']` are not defined. No namespaces are defined for the target.\n - Data generator `Prothrombinase_13_task1` is invalid.\n - Variable `p1_Prothrombinase_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Prothrombinase']` are not defined. No namespaces are defined for the target.\n - Data generator `log_time_13_task1` is invalid.\n - Variable `p1_log_time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_time']` are not defined. No namespaces are defined for the target.\n - Data generator `Thrombin_14_task1` is invalid.\n - Variable `p1_Thrombin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Thrombin']` are not defined. No namespaces are defined for the target.\n - Data generator `log_time_14_task1` is invalid.\n - Variable `p1_log_time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='log_time']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000926](https://www.ebi.ac.uk/biomodels/BIOMD0000000926)<br/><sup>Rhodes2019 - Immune-Mediated t</sup></summary><sup>Rhodes2019 - Immune-Mediated theory of Metastasis</sup></details>|<details><summary>pass</summary>[Rhodes2019-Immune-Mediated theory of Metastasis_V1.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000926#Files)</details>|FAIL|<details><summary>pass</summary>[Rhodes2019-Immune-Mediated theory of Metastasis_V1.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000926#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Rhodes2019-Immune-Mediated theory of Metastasis_V1.sedml281779.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 2: b'usage: biosimulators-tellurium [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-tellurium: error: unrecognized arguments: theory of Metastasis_V1.sedml281779.omex\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Rhodes2019-Immune-Mediated theory of Metastasis_V1.sedml670046.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 2: b'usage: biosimulators-copasi [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-copasi: error: unrecognized arguments: theory of Metastasis_V1.sedml670046.omex\n'```</details>|
|<details><summary>[BIOMD0000000927](https://www.ebi.ac.uk/biomodels/BIOMD0000000927)<br/><sup>Grigolon2018-Responses to auxi</sup></summary><sup>Grigolon2018-Responses to auxin signals</sup></details>|<details><summary>pass</summary>[Grigolon2018_fig1b.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000927#Files)</details>|pass|<details><summary>pass</summary>[Grigolon2018_fig1b.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000927#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Grigolon2018_fig1b.sedml305918.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Grigolon2018_fig1b.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 594, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Rate_Law_for_reaction_auxin_induction(Sauxin, auxin, Tauxin)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type Modeling practice (80701). The following is the first warning at line 428, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Tauxin' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 1000.0\r\n - Number of steps: 1\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Grigolon2018_fig1b.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Grigolon2018_fig1b.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 594, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Rate_Law_for_reaction_auxin_induction(Sauxin, auxin, Tauxin)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type Modeling practice (80701). The following is the first warning at line 428, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Tauxin' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 1000.0\r\n - Number of steps: 1\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n\r\n`/root/in/Grigolon2018_fig1b.sedml305918.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Grigolon2018_fig1b.sedml` is invalid.\n - Each identified SED object must have a unique id. Multiple objects have the following ids:\n - p1_Sauxin_task1\n - p1_Sauxin_task3\n - Task `task3` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Sauxin']` are not defined. No namespaces are defined for the target.\n - Data generator `ARF_1_task1` is invalid.\n - Variable `p1_ARF_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ARF']` are not defined. No namespaces are defined for the target.\n - Data generator `Sauxin_1_task1` is invalid.\n - Variable `p1_Sauxin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Sauxin']` are not defined. No namespaces are defined for the target.\n - Data generator `IAA_2_task1` is invalid.\n - Variable `p1_IAA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IAA']` are not defined. No namespaces are defined for the target.\n - Data generator `Sauxin_2_task1` is invalid.\n - Variable `p1_Sauxin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Sauxin']` are not defined. No namespaces are defined for the target.\n - Data generator `auxin_3_task1` is invalid.\n - Variable `p1_auxin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='auxin']` are not defined. No namespaces are defined for the target.\n - Data generator `Sauxin_3_task1` is invalid.\n - Variable `p1_Sauxin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Sauxin']` are not defined. No namespaces are defined for the target.\n - Data generator `ARF_1_task3` is invalid.\n - Variable `p1_ARF_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ARF']` are not defined. No namespaces are defined for the target.\n - Data generator `Sauxin_1_task3` is invalid.\n - Variable `p1_Sauxin_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Sauxin']` are not defined. No namespaces are defined for the target.\n - Data generator `IAA_2_task3` is invalid.\n - Variable `p1_IAA_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IAA']` are not defined. No namespaces are defined for the target.\n - Data generator `Sauxin_2_task3` is invalid.\n - Variable `p1_Sauxin_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Sauxin']` are not defined. No namespaces are defined for the target.\n - Data generator `auxin_3_task3` is invalid.\n - Variable `p1_auxin_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='auxin']` are not defined. No namespaces are defined for the target.\n - Data generator `Sauxin_3_task3` is invalid.\n - Variable `p1_Sauxin_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Sauxin']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Grigolon2018_fig1b.sedml6680.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Grigolon2018_fig1b.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 594, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Rate_Law_for_reaction_auxin_induction(Sauxin, auxin, Tauxin)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type Modeling practice (80701). The following is the first warning at line 428, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Tauxin' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 1000.0\r\n - Number of steps: 1\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Grigolon2018_fig1b.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Grigolon2018_fig1b.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 594, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Rate_Law_for_reaction_auxin_induction(Sauxin, auxin, Tauxin)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type Modeling practice (80701). The following is the first warning at line 428, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Tauxin' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 1000.0\r\n - Number of steps: 1\r\n - Output `plot_2_task3` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task3`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Grigolon2018_fig1b.sedml6680.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Grigolon2018_fig1b.sedml` is invalid.\n - Each identified SED object must have a unique id. Multiple objects have the following ids:\n - p1_Sauxin_task1\n - p1_Sauxin_task3\n - Task `task3` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Sauxin']` are not defined. No namespaces are defined for the target.\n - Data generator `ARF_1_task1` is invalid.\n - Variable `p1_ARF_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ARF']` are not defined. No namespaces are defined for the target.\n - Data generator `Sauxin_1_task1` is invalid.\n - Variable `p1_Sauxin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Sauxin']` are not defined. No namespaces are defined for the target.\n - Data generator `IAA_2_task1` is invalid.\n - Variable `p1_IAA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IAA']` are not defined. No namespaces are defined for the target.\n - Data generator `Sauxin_2_task1` is invalid.\n - Variable `p1_Sauxin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Sauxin']` are not defined. No namespaces are defined for the target.\n - Data generator `auxin_3_task1` is invalid.\n - Variable `p1_auxin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='auxin']` are not defined. No namespaces are defined for the target.\n - Data generator `Sauxin_3_task1` is invalid.\n - Variable `p1_Sauxin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Sauxin']` are not defined. No namespaces are defined for the target.\n - Data generator `ARF_1_task3` is invalid.\n - Variable `p1_ARF_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ARF']` are not defined. No namespaces are defined for the target.\n - Data generator `Sauxin_1_task3` is invalid.\n - Variable `p1_Sauxin_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Sauxin']` are not defined. No namespaces are defined for the target.\n - Data generator `IAA_2_task3` is invalid.\n - Variable `p1_IAA_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IAA']` are not defined. No namespaces are defined for the target.\n - Data generator `Sauxin_2_task3` is invalid.\n - Variable `p1_Sauxin_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Sauxin']` are not defined. No namespaces are defined for the target.\n - Data generator `auxin_3_task3` is invalid.\n - Variable `p1_auxin_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='auxin']` are not defined. No namespaces are defined for the target.\n - Data generator `Sauxin_3_task3` is invalid.\n - Variable `p1_Sauxin_task3` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Sauxin']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000928](https://www.ebi.ac.uk/biomodels/BIOMD0000000928)<br/><sup>Baker2017 - The role of cytoki</sup></summary><sup>Baker2017 - The role of cytokines, MMPs and fibronectin fragments osteoarthritis</sup></details>|<details><summary>pass</summary>[Baker2017_Fig14..xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000928#Files)</details>|FAIL|<details><summary>MultipleSEDMLs</summary>['Baker2017_Fig14.b_1.sedml', 'Baker2017_Fig14.a_3.sedml', 'Baker2017_Fig14.b_2.sedml', 'Baker2017_Fig14.a_2.sedml', 'Baker2017_Fig14.a_1.sedml']</details>|NA|NA|NA|
|<details><summary>[BIOMD0000000929](https://www.ebi.ac.uk/biomodels/BIOMD0000000929)<br/><sup>Li2016 - Model for pancreatic </sup></summary><sup>Li2016 - Model for pancreatic cancer patients receiving immunotherapy</sup></details>|<details><summary>pass</summary>[Li2016.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000929#Files)</details>|pass|<details><summary>FAIL</summary>[Li2016.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000929#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Li2016.sedml712685.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/Li2016.sedml712685.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Li2016.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - YDataReference from Curve `p1_curve_5_task1` cannot be resolved to a data generator with id `helper_T_cells__H_5_task1` because multiple data generators have this id.\n - YDataReference from Curve `p2_curve_5_task1` cannot be resolved to a data generator with id `helper_T_cells__H_5_task1` because multiple data generators have this id.\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Li2016.sedml705604.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Li2016.sedml705604.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Li2016.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - YDataReference from Curve `p1_curve_5_task1` cannot be resolved to a data generator with id `helper_T_cells__H_5_task1` because multiple data generators have this id.\n - YDataReference from Curve `p2_curve_5_task1` cannot be resolved to a data generator with id `helper_T_cells__H_5_task1` because multiple data generators have this id.\n'```</details>|
|<details><summary>[BIOMD0000000930](https://www.ebi.ac.uk/biomodels/BIOMD0000000930)<br/><sup>Liu2017 - chemotherapy targete</sup></summary><sup>Liu2017 - chemotherapy targeted model of tumor immune system</sup></details>|<details><summary>pass</summary>[Liu2017.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000930#Files)</details>|pass|<details><summary>pass</summary>[Liu2017.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000930#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Liu2017.sedml927249.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Liu2017.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 897, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Tumor_Microenvironment * Function_for_reaction_1(a, Tumor_Cell_Population__T, b)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 17 warnings of type Modeling practice (80701). The following is the first warning at line 623, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Liu2017.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Liu2017.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 897, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Tumor_Microenvironment * Function_for_reaction_1(a, Tumor_Cell_Population__T, b)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 17 warnings of type Modeling practice (80701). The following is the first warning at line 623, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Liu2017.sedml927249.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Liu2017.sedml` is invalid.\n - Data generator `Circulating_lymphocyte_population__C_1_task1` is invalid.\n - Variable `p1_Circulating_lymphocyte_population__C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Circulating_lymphocyte_population__C']` are not defined. No namespaces are defined for the target.\n - Data generator `Chemotherapeutic_drug_concentration__M_2_task1` is invalid.\n - Variable `p1_Chemotherapeutic_drug_concentration__M_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Chemotherapeutic_drug_concentration__M']` are not defined. No namespaces are defined for the target.\n - Data generator `Effector_immune_cell_population__N_3_task1` is invalid.\n - Variable `p1_Effector_immune_cell_population__N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Effector_immune_cell_population__N']` are not defined. No namespaces are defined for the target.\n - Data generator `Tumor_Cell_Population__T_4_task1` is invalid.\n - Variable `p1_Tumor_Cell_Population__T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Tumor_Cell_Population__T']` are not defined. No namespaces are defined for the target.\n - Data generator `Tumor_Cell_Population__T_1_task1` is invalid.\n - Variable `p2_Tumor_Cell_Population__T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Tumor_Cell_Population__T']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Liu2017.sedml724819.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Liu2017.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 897, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Tumor_Microenvironment * Function_for_reaction_1(a, Tumor_Cell_Population__T, b)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 17 warnings of type Modeling practice (80701). The following is the first warning at line 623, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Liu2017.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Liu2017.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 897, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Tumor_Microenvironment * Function_for_reaction_1(a, Tumor_Cell_Population__T, b)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 17 warnings of type Modeling practice (80701). The following is the first warning at line 623, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Liu2017.sedml724819.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Liu2017.sedml` is invalid.\n - Data generator `Circulating_lymphocyte_population__C_1_task1` is invalid.\n - Variable `p1_Circulating_lymphocyte_population__C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Circulating_lymphocyte_population__C']` are not defined. No namespaces are defined for the target.\n - Data generator `Chemotherapeutic_drug_concentration__M_2_task1` is invalid.\n - Variable `p1_Chemotherapeutic_drug_concentration__M_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Chemotherapeutic_drug_concentration__M']` are not defined. No namespaces are defined for the target.\n - Data generator `Effector_immune_cell_population__N_3_task1` is invalid.\n - Variable `p1_Effector_immune_cell_population__N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Effector_immune_cell_population__N']` are not defined. No namespaces are defined for the target.\n - Data generator `Tumor_Cell_Population__T_4_task1` is invalid.\n - Variable `p1_Tumor_Cell_Population__T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Tumor_Cell_Population__T']` are not defined. No namespaces are defined for the target.\n - Data generator `Tumor_Cell_Population__T_1_task1` is invalid.\n - Variable `p2_Tumor_Cell_Population__T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Tumor_Cell_Population__T']` are not defined. No namespaces are defined for the target.\n"```</details>|
|[BIOMD0000000931](https://www.ebi.ac.uk/biomodels/BIOMD0000000931)<br/><sup>Voliotis2019-GnRH Pulse Generation</sup>|<details><summary>pass</summary>[Voliotis2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000931#Files)</details>|pass|<details><summary>pass</summary>[Voliotis2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000931#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Voliotis2019.sedml51910.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Voliotis2019.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 548, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'I_0 + p_v * v * (pow(N, n_4) / (pow(N, n_4) + pow(K_N, n_4)))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 22 warnings of type Modeling practice (80701). The following is the first warning at line 216, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'd_D' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Voliotis2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Voliotis2019.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 548, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'I_0 + p_v * v * (pow(N, n_4) / (pow(N, n_4) + pow(K_N, n_4)))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 22 warnings of type Modeling practice (80701). The following is the first warning at line 216, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'd_D' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Voliotis2019.sedml51910.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Voliotis2019.sedml` is invalid.\n - Data generator `Firing_Rate__HZ_1_task1` is invalid.\n - Variable `p1_Firing_Rate__HZ_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Firing_Rate__HZ']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Voliotis2019.sedml101794.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Voliotis2019.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 548, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'I_0 + p_v * v * (pow(N, n_4) / (pow(N, n_4) + pow(K_N, n_4)))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 22 warnings of type Modeling practice (80701). The following is the first warning at line 216, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'd_D' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Voliotis2019.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Voliotis2019.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 548, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'I_0 + p_v * v * (pow(N, n_4) / (pow(N, n_4) + pow(K_N, n_4)))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 22 warnings of type Modeling practice (80701). The following is the first warning at line 216, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'd_D' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Voliotis2019.sedml101794.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Voliotis2019.sedml` is invalid.\n - Data generator `Firing_Rate__HZ_1_task1` is invalid.\n - Variable `p1_Firing_Rate__HZ_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Firing_Rate__HZ']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000932](https://www.ebi.ac.uk/biomodels/BIOMD0000000932)<br/><sup>Garde2020-Minimal model descri</sup></summary><sup>Garde2020-Minimal model describing metabolic oscillations in Bacillus subtilis biofilms</sup></details>|<details><summary>pass</summary>[Garde2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000932#Files)</details>|pass|<details><summary>MultipleSEDMLs</summary>['Garde2020-Fig2.sedml', 'Garde2020-Fig4.sedml']</details>|NA|NA|NA|
|<details><summary>[BIOMD0000000933](https://www.ebi.ac.uk/biomodels/BIOMD0000000933)<br/><sup>Kosiuk2015-Geometric analysis </sup></summary><sup>Kosiuk2015-Geometric analysis of the Goldbeter minimal model for the embryonic cell cycle</sup></details>|<details><summary>pass</summary>[Kosiuk2015.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000933#Files)</details>|pass|<details><summary>pass</summary>[Kosiuk2015.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000933#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Kosiuk2015.sedml298509.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Kosiuk2015.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 638, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'VM1 * C / (Kc + C)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 14 warnings of type Modeling practice (80701). The following is the first warning at line 426, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kd' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Kosiuk2015.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Kosiuk2015.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 638, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'VM1 * C / (Kc + C)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 14 warnings of type Modeling practice (80701). The following is the first warning at line 426, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kd' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Kosiuk2015.sedml298509.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Kosiuk2015.sedml` is invalid.\n - Data generator `C_1_task1` is invalid.\n - Variable `p1_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C']` are not defined. No namespaces are defined for the target.\n - Data generator `M_2_task1` is invalid.\n - Variable `p1_M_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M']` are not defined. No namespaces are defined for the target.\n - Data generator `X_3_task1` is invalid.\n - Variable `p1_X_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Kosiuk2015.sedml632307.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Kosiuk2015.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 638, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'VM1 * C / (Kc + C)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 14 warnings of type Modeling practice (80701). The following is the first warning at line 426, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kd' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Kosiuk2015.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Kosiuk2015.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 638, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'VM1 * C / (Kc + C)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 14 warnings of type Modeling practice (80701). The following is the first warning at line 426, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kd' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Kosiuk2015.sedml632307.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Kosiuk2015.sedml` is invalid.\n - Data generator `C_1_task1` is invalid.\n - Variable `p1_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C']` are not defined. No namespaces are defined for the target.\n - Data generator `M_2_task1` is invalid.\n - Variable `p1_M_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M']` are not defined. No namespaces are defined for the target.\n - Data generator `X_3_task1` is invalid.\n - Variable `p1_X_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='X']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000934](https://www.ebi.ac.uk/biomodels/BIOMD0000000934)<br/><sup>Linke2017 - Synchronization of</sup></summary><sup>Linke2017 - Synchronization of Cyclins' expression by the Fkh2 transcription factor in the budding yeast cell cycle</sup></details>|<details><summary>pass</summary>[Linke2017.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000934#Files)</details>|pass|<details><summary>MultipleSEDMLs</summary>['Linke2017_figure1e.sedml', 'Linke2017_figure1d.sedml', 'Linke2017_figure1c.sedml', 'Linke2017f1b.sedml']</details>|NA|NA|NA|
|<details><summary>[BIOMD0000000935](https://www.ebi.ac.uk/biomodels/BIOMD0000000935)<br/><sup>Ferrel2011 - Cdk1 and APC regu</sup></summary><sup>Ferrel2011 - Cdk1 and APC regulation in cell cycle in Xenopus laevis</sup></details>|<details><summary>pass</summary>[Ferrel2011.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000935#Files)</details>|pass|<details><summary>pass</summary>[Ferrel2011.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000935#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Ferrel2011.sedml366385.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Ferrel2011.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 565, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'nuclear * Constant_flux__irreversible(a1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 415, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a1' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Ferrel2011.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Ferrel2011.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 565, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'nuclear * Constant_flux__irreversible(a1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 415, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a1' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Ferrel2011.sedml366385.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Ferrel2011.sedml` is invalid.\n - Data generator `APC_active_1_task1` is invalid.\n - Variable `p1_APC_active_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='APC_active']` are not defined. No namespaces are defined for the target.\n - Data generator `CDK1_active_2_task1` is invalid.\n - Variable `p1_CDK1_active_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CDK1_active']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Ferrel2011.sedml94961.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Ferrel2011.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 565, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'nuclear * Constant_flux__irreversible(a1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 415, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Ferrel2011.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Ferrel2011.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 565, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'nuclear * Constant_flux__irreversible(a1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 415, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Ferrel2011.sedml94961.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Ferrel2011.sedml` is invalid.\n - Data generator `APC_active_1_task1` is invalid.\n - Variable `p1_APC_active_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='APC_active']` are not defined. No namespaces are defined for the target.\n - Data generator `CDK1_active_2_task1` is invalid.\n - Variable `p1_CDK1_active_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CDK1_active']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000936](https://www.ebi.ac.uk/biomodels/BIOMD0000000936)<br/><sup>ferrel2011 - autonomous bioche</sup></summary><sup>ferrel2011 - autonomous biochemical oscillator in cell cycle in Xenopus laevis v2</sup></details>|<details><summary>pass</summary>[Ferrel2011_V2.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000936#Files)</details>|FAIL|<details><summary>pass</summary>[Ferrel2011_V2.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000936#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Ferrel2011_V2.sedml377849.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Ferrel2011_V2.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 424, column 8:\r\n - The formula 'pow(CDK1_active, n1 + 1)' in the math element of the <kineticLaw> contains an expression for the exponent of the power function and thus cannot be checked for unit validity.\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 394, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Constant_flux__irreversible(a1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 4 warnings of type Modeling practice (80701). The following is the first warning at line 304, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a1' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Ferrel2011_V2.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Ferrel2011_V2.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 424, column 8:\r\n - The formula 'pow(CDK1_active, n1 + 1)' in the math element of the <kineticLaw> contains an expression for the exponent of the power function and thus cannot be checked for unit validity.\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 394, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Constant_flux__irreversible(a1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 4 warnings of type Modeling practice (80701). The following is the first warning at line 304, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a1' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Ferrel2011_V2.sedml377849.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Ferrel2011_V2.sedml` is invalid.\n - Data generator `CDK1_active_1_task1` is invalid.\n - Variable `p1_CDK1_active_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CDK1_active']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Ferrel2011_V2.sedml990229.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Ferrel2011_V2.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 424, column 8:\r\n - The formula 'pow(CDK1_active, n1 + 1)' in the math element of the <kineticLaw> contains an expression for the exponent of the power function and thus cannot be checked for unit validity.\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 394, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Constant_flux__irreversible(a1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 4 warnings of type Modeling practice (80701). The following is the first warning at line 304, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Ferrel2011_V2.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Ferrel2011_V2.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 424, column 8:\r\n - The formula 'pow(CDK1_active, n1 + 1)' in the math element of the <kineticLaw> contains an expression for the exponent of the power function and thus cannot be checked for unit validity.\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 394, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Constant_flux__irreversible(a1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 4 warnings of type Modeling practice (80701). The following is the first warning at line 304, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Ferrel2011_V2.sedml990229.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Ferrel2011_V2.sedml` is invalid.\n - Data generator `CDK1_active_1_task1` is invalid.\n - Variable `p1_CDK1_active_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CDK1_active']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000937](https://www.ebi.ac.uk/biomodels/BIOMD0000000937)<br/><sup>Ferrel2011 - Autonomous bioche</sup></summary><sup>Ferrel2011 - Autonomous biochemical oscillator in regulation of CDK1, Plk1, and APC in Xenopus Laevis cell cycle</sup></details>|<details><summary>pass</summary>[Ferrel2011_V3.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000937#Files)</details>|pass|<details><summary>pass</summary>[Ferrel2011_V3.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000937#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Ferrel2011_V3.sedml475364.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Ferrel2011_V3.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 635, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Constant_flux__irreversible(a1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 424, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a1' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Ferrel2011_V3.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Ferrel2011_V3.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 635, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Constant_flux__irreversible(a1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 424, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a1' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Ferrel2011_V3.sedml475364.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Ferrel2011_V3.sedml` is invalid.\n - Data generator `APC_active_1_task1` is invalid.\n - Variable `p1_APC_active_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='APC_active']` are not defined. No namespaces are defined for the target.\n - Data generator `CDK1_active_2_task1` is invalid.\n - Variable `p1_CDK1_active_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CDK1_active']` are not defined. No namespaces are defined for the target.\n - Data generator `Plk1_active_3_task1` is invalid.\n - Variable `p1_Plk1_active_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Plk1_active']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Ferrel2011_V3.sedml138904.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Ferrel2011_V3.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 635, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Constant_flux__irreversible(a1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 424, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Ferrel2011_V3.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Ferrel2011_V3.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 635, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'compartment * Constant_flux__irreversible(a1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 424, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Ferrel2011_V3.sedml138904.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Ferrel2011_V3.sedml` is invalid.\n - Data generator `APC_active_1_task1` is invalid.\n - Variable `p1_APC_active_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='APC_active']` are not defined. No namespaces are defined for the target.\n - Data generator `CDK1_active_2_task1` is invalid.\n - Variable `p1_CDK1_active_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CDK1_active']` are not defined. No namespaces are defined for the target.\n - Data generator `Plk1_active_3_task1` is invalid.\n - Variable `p1_Plk1_active_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Plk1_active']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000938](https://www.ebi.ac.uk/biomodels/BIOMD0000000938)<br/><sup>Gerard2013 - Model 3 - Embryon</sup></summary><sup>Gerard2013 - Model 3 - Embryonic-type eukaryotic Cell Cycle regulation based on negative feedback between Cdk/cyclin and APC and competitive inhibition between Cdk/cyclin and securin for polyubiquitylation_1</sup></details>|<details><summary>pass</summary>[Gerard2013.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000938#Files)</details>|pass|<details><summary>pass</summary>[Gerard2013.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000938#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Gerard2013.sedml417073.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Gerard2013.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 810, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'nuclear * Constant_flux__irreversible(V_scdk)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 589, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'V_scdk' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Gerard2013.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Gerard2013.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 810, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'nuclear * Constant_flux__irreversible(V_scdk)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 589, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'V_scdk' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Gerard2013.sedml417073.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Gerard2013.sedml` is invalid.\n - Data generator `Cdk_1_task1` is invalid.\n - Variable `p1_Cdk_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cdk']` are not defined. No namespaces are defined for the target.\n - Data generator `Anaphase_promoting_complex_Phosphorylated_2_task1` is invalid.\n - Variable `p1_Anaphase_promoting_complex_Phosphorylated_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Anaphase_promoting_complex_Phosphorylated']` are not defined. No namespaces are defined for the target.\n - Data generator `Securin_3_task1` is invalid.\n - Variable `p1_Securin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Securin']` are not defined. No namespaces are defined for the target.\n - Data generator `Anaphase_promoting_complex_4_task1` is invalid.\n - Variable `p1_Anaphase_promoting_complex_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Anaphase_promoting_complex']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Gerard2013.sedml256681.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Gerard2013.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 810, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'nuclear * Constant_flux__irreversible(V_scdk)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 589, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'V_scdk' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Gerard2013.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Gerard2013.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 810, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'nuclear * Constant_flux__irreversible(V_scdk)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 589, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'V_scdk' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Gerard2013.sedml256681.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Gerard2013.sedml` is invalid.\n - Data generator `Cdk_1_task1` is invalid.\n - Variable `p1_Cdk_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cdk']` are not defined. No namespaces are defined for the target.\n - Data generator `Anaphase_promoting_complex_Phosphorylated_2_task1` is invalid.\n - Variable `p1_Anaphase_promoting_complex_Phosphorylated_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Anaphase_promoting_complex_Phosphorylated']` are not defined. No namespaces are defined for the target.\n - Data generator `Securin_3_task1` is invalid.\n - Variable `p1_Securin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Securin']` are not defined. No namespaces are defined for the target.\n - Data generator `Anaphase_promoting_complex_4_task1` is invalid.\n - Variable `p1_Anaphase_promoting_complex_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Anaphase_promoting_complex']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000939](https://www.ebi.ac.uk/biomodels/BIOMD0000000939)<br/><sup>Iwamoto2010 - Cell cycle repon</sup></summary><sup>Iwamoto2010 - Cell cycle reponse to DNA damage</sup></details>|<details><summary>pass</summary>[Iwamoto2010.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000939#Files)</details>|FAIL|<details><summary>MultipleSEDMLs</summary>['iwamoto2010_Fig6.sedml', 'iwamoto2010_Fig4.sedml', 'iwamoto2010_Fig3.sedml']</details>|NA|NA|NA|
|<details><summary>[BIOMD0000000940](https://www.ebi.ac.uk/biomodels/BIOMD0000000940)<br/><sup>Tang2019 - Pharmacology modell</sup></summary><sup>Tang2019 - Pharmacology modelling of AURKB and ZAK interaction in TNBC</sup></details>|<details><summary>pass</summary>[Tang2019.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000940#Files)</details>|pass|<details><summary>FAIL</summary>[Tang2019.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000940#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Tang2019.sedml81952.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Tang2019.sedml` has warnings.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n`/root/in/Tang2019.sedml81952.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Tang2019.sedml` is invalid.\n - Model `model2` is invalid.\n - The model file `model2.xml` is invalid.\n - `/tmp/tmp17252lxr/./model2.xml` is not a file.\n - Output `plot_1_task1` is invalid.\n - Plot must have at least one curve.\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Tang2019.sedml524739.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Tang2019.sedml` has warnings.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Tang2019.sedml524739.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Tang2019.sedml` is invalid.\n - Model `model2` is invalid.\n - The model file `model2.xml` is invalid.\n - `/tmp/tmp1mkshvvg/./model2.xml` is not a file.\n - Output `plot_1_task1` is invalid.\n - Plot must have at least one curve.\n'```</details>|
|<details><summary>[BIOMD0000000941](https://www.ebi.ac.uk/biomodels/BIOMD0000000941)<br/><sup>Gerard2010 - Progression of ma</sup></summary><sup>Gerard2010 - Progression of mammalian cell cycle by successive activation of various cyclin cdk complexes</sup></details>|<details><summary>pass</summary>[gerard2010.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000941#Files)</details>|pass|<details><summary>pass</summary>[gerard2010.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000941#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/gerard2010.sedml342638.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `gerard2010.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 1204, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'nuclear * Function_for_R1(vsd, GF, Kgf)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 22 warnings of type Modeling practice (80701). The following is the first warning at line 843, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'GF' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./gerard2010.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `gerard2010.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 1204, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'nuclear * Function_for_R1(vsd, GF, Kgf)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 22 warnings of type Modeling practice (80701). The following is the first warning at line 843, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'GF' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/gerard2010.sedml342638.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./gerard2010.sedml` is invalid.\n - Data generator `cyclin_A_Cdk2_1_task1` is invalid.\n - Variable `p1_cyclin_A_Cdk2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='cyclin_A_Cdk2']` are not defined. No namespaces are defined for the target.\n - Data generator `cyclin_B_Cdk1_2_task1` is invalid.\n - Variable `p1_cyclin_B_Cdk1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='cyclin_B_Cdk1']` are not defined. No namespaces are defined for the target.\n - Data generator `cyclin_E_Cdk2_3_task1` is invalid.\n - Variable `p1_cyclin_E_Cdk2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='cyclin_E_Cdk2']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/gerard2010.sedml133134.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `gerard2010.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 1204, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'nuclear * Function_for_R1(vsd, GF, Kgf)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 22 warnings of type Modeling practice (80701). The following is the first warning at line 843, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'GF' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./gerard2010.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `gerard2010.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 1204, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'nuclear * Function_for_R1(vsd, GF, Kgf)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 22 warnings of type Modeling practice (80701). The following is the first warning at line 843, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'GF' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/gerard2010.sedml133134.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./gerard2010.sedml` is invalid.\n - Data generator `cyclin_A_Cdk2_1_task1` is invalid.\n - Variable `p1_cyclin_A_Cdk2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='cyclin_A_Cdk2']` are not defined. No namespaces are defined for the target.\n - Data generator `cyclin_B_Cdk1_2_task1` is invalid.\n - Variable `p1_cyclin_B_Cdk1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='cyclin_B_Cdk1']` are not defined. No namespaces are defined for the target.\n - Data generator `cyclin_E_Cdk2_3_task1` is invalid.\n - Variable `p1_cyclin_E_Cdk2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='cyclin_E_Cdk2']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000942](https://www.ebi.ac.uk/biomodels/BIOMD0000000942)<br/><sup>Sible2007 - Mitotic cell cycle</sup></summary><sup>Sible2007 - Mitotic cell cycle mecanism in Xenopus Laevis</sup></details>|<details><summary>pass</summary>[Sible2007.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000942#Files)</details>|pass|<details><summary>pass</summary>[Sible2007.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000942#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Sible2007.sedml543265.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Sible2007.xml` has warnings.\r\n - 18 warnings of type SBML unit consistency (99505). The following is the first warning at line 1283, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'vwee * Wee1_phosphorylated + vwee_ * (Wee1_total - Wee1_phosphorylated)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 27 warnings of type Modeling practice (80701). The following is the first warning at line 834, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Sible2007.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Sible2007.xml` has warnings.\r\n - 18 warnings of type SBML unit consistency (99505). The following is the first warning at line 1283, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'vwee * Wee1_phosphorylated + vwee_ * (Wee1_total - Wee1_phosphorylated)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 27 warnings of type Modeling practice (80701). The following is the first warning at line 834, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Sible2007.sedml543265.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Sible2007.sedml` is invalid.\n - Data generator `Cyclin_Cdk1_MPF_1_task1` is invalid.\n - Variable `p1_Cyclin_Cdk1_MPF_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cyclin_Cdk1_MPF']` are not defined. No namespaces are defined for the target.\n - Data generator `Cyclin_Cdk1_preMPF_2_task1` is invalid.\n - Variable `p1_Cyclin_Cdk1_preMPF_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cyclin_Cdk1_preMPF']` are not defined. No namespaces are defined for the target.\n - Data generator `Cyclin_total_3_task1` is invalid.\n - Variable `p1_Cyclin_total_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cyclin_total']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Sible2007.sedml977287.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Sible2007.xml` has warnings.\r\n - 18 warnings of type SBML unit consistency (99505). The following is the first warning at line 1283, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'vwee * Wee1_phosphorylated + vwee_ * (Wee1_total - Wee1_phosphorylated)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 27 warnings of type Modeling practice (80701). The following is the first warning at line 834, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Sible2007.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Sible2007.xml` has warnings.\r\n - 18 warnings of type SBML unit consistency (99505). The following is the first warning at line 1283, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'vwee * Wee1_phosphorylated + vwee_ * (Wee1_total - Wee1_phosphorylated)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 27 warnings of type Modeling practice (80701). The following is the first warning at line 834, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Sible2007.sedml977287.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Sible2007.sedml` is invalid.\n - Data generator `Cyclin_Cdk1_MPF_1_task1` is invalid.\n - Variable `p1_Cyclin_Cdk1_MPF_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cyclin_Cdk1_MPF']` are not defined. No namespaces are defined for the target.\n - Data generator `Cyclin_Cdk1_preMPF_2_task1` is invalid.\n - Variable `p1_Cyclin_Cdk1_preMPF_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cyclin_Cdk1_preMPF']` are not defined. No namespaces are defined for the target.\n - Data generator `Cyclin_total_3_task1` is invalid.\n - Variable `p1_Cyclin_total_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cyclin_total']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000943](https://www.ebi.ac.uk/biomodels/BIOMD0000000943)<br/><sup>Hat2016 - Reponse of p53 Syste</sup></summary><sup>Hat2016 - Reponse of p53 System to irradiation in cell fate decision making</sup></details>|<details><summary>pass</summary>[Hat2016.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000943#Files)</details>|pass|<details><summary>pass</summary>[Hat2016.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000943#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Hat2016.sedml401309.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/Hat2016.sedml401309.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Hat2016.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - XDataReference from Curve `p1_curve_1_task1` cannot be resolved to a data generator with id `time_days_1_task1` because multiple data generators have this id.\n - XDataReference from Curve `p2_curve_1_task1` cannot be resolved to a data generator with id `time_days_1_task1` because multiple data generators have this id.\n - XDataReference from Curve `p3_curve_1_task1` cannot be resolved to a data generator with id `time_days_1_task1` because multiple data generators have this id.\n - XDataReference from Curve `p4_curve_1_task1` cannot be resolved to a data generator with id `time_days_1_task1` because multiple data generators have this id.\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Hat2016.sedml149353.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Hat2016.sedml149353.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Hat2016.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - XDataReference from Curve `p1_curve_1_task1` cannot be resolved to a data generator with id `time_days_1_task1` because multiple data generators have this id.\n - XDataReference from Curve `p2_curve_1_task1` cannot be resolved to a data generator with id `time_days_1_task1` because multiple data generators have this id.\n - XDataReference from Curve `p3_curve_1_task1` cannot be resolved to a data generator with id `time_days_1_task1` because multiple data generators have this id.\n - XDataReference from Curve `p4_curve_1_task1` cannot be resolved to a data generator with id `time_days_1_task1` because multiple data generators have this id.\n'```</details>|
|<details><summary>[BIOMD0000000944](https://www.ebi.ac.uk/biomodels/BIOMD0000000944)<br/><sup>Goldbeter2013-Oscillatory acti</sup></summary><sup>Goldbeter2013-Oscillatory activity of cyclin-dependent kinases in the cell cycle</sup></details>|<details><summary>pass</summary>[Goldbeter2013.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000944#Files)</details>|pass|<details><summary>pass</summary>[Goldbeter2013.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000944#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Goldbeter2013.sedml682699.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Goldbeter2013.xml` has warnings.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 714, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'K1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 15 warnings of type Modeling practice (80701). The following is the first warning at line 487, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K1' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Goldbeter2013.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Goldbeter2013.xml` has warnings.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 714, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'K1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 15 warnings of type Modeling practice (80701). The following is the first warning at line 487, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K1' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/Goldbeter2013.sedml682699.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Goldbeter2013.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='K1']` are not defined. No namespaces are defined for the target.\n - Data generator `Active_Cdc2_kinase_1_task1` is invalid.\n - Variable `p1_Active_Cdc2_kinase_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Active_Cdc2_kinase']` are not defined. No namespaces are defined for the target.\n - Data generator `Active_Cdc2_kinase_1_task2` is invalid.\n - Variable `p1_Active_Cdc2_kinase_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Active_Cdc2_kinase']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Goldbeter2013.sedml114032.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Goldbeter2013.xml` has warnings.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 714, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'K1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 15 warnings of type Modeling practice (80701). The following is the first warning at line 487, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K1' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Goldbeter2013.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Goldbeter2013.xml` has warnings.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 714, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'K1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 15 warnings of type Modeling practice (80701). The following is the first warning at line 487, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K1' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Goldbeter2013.sedml114032.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Goldbeter2013.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='K1']` are not defined. No namespaces are defined for the target.\n - Data generator `Active_Cdc2_kinase_1_task1` is invalid.\n - Variable `p1_Active_Cdc2_kinase_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Active_Cdc2_kinase']` are not defined. No namespaces are defined for the target.\n - Data generator `Active_Cdc2_kinase_1_task2` is invalid.\n - Variable `p1_Active_Cdc2_kinase_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Active_Cdc2_kinase']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000945](https://www.ebi.ac.uk/biomodels/BIOMD0000000945)<br/><sup>Evans2004 - Cell based mathema</sup></summary><sup>Evans2004 - Cell based mathematical model of topotecan</sup></details>|<details><summary>pass</summary>[Evans2004.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000945#Files)</details>|pass|<details><summary>pass</summary>[Evans2004.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000945#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Evans2004.sedml383683.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Evans2004.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 669, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'D' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 15 warnings of type Modeling practice (80701). The following is the first warning at line 420, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k_0_m' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Evans2004.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Evans2004.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 669, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'D' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 15 warnings of type Modeling practice (80701). The following is the first warning at line 420, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k_0_m' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Evans2004.sedml383683.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Evans2004.sedml` is invalid.\n - Data generator `H_m_1_task1` is invalid.\n - Variable `p1_H_m_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H_m']` are not defined. No namespaces are defined for the target.\n - Data generator `L_m_2_task1` is invalid.\n - Variable `p1_L_m_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L_m']` are not defined. No namespaces are defined for the target.\n - Data generator `H_c_1_task1` is invalid.\n - Variable `p2_H_c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H_c']` are not defined. No namespaces are defined for the target.\n - Data generator `L_c_2_task1` is invalid.\n - Variable `p2_L_c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L_c']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Evans2004.sedml728284.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Evans2004.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 669, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'D' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 15 warnings of type Modeling practice (80701). The following is the first warning at line 420, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k_0_m' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Evans2004.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Evans2004.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 669, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'D' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 15 warnings of type Modeling practice (80701). The following is the first warning at line 420, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k_0_m' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Evans2004.sedml728284.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Evans2004.sedml` is invalid.\n - Data generator `H_m_1_task1` is invalid.\n - Variable `p1_H_m_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H_m']` are not defined. No namespaces are defined for the target.\n - Data generator `L_m_2_task1` is invalid.\n - Variable `p1_L_m_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L_m']` are not defined. No namespaces are defined for the target.\n - Data generator `H_c_1_task1` is invalid.\n - Variable `p2_H_c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H_c']` are not defined. No namespaces are defined for the target.\n - Data generator `L_c_2_task1` is invalid.\n - Variable `p2_L_c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L_c']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000946](https://www.ebi.ac.uk/biomodels/BIOMD0000000946)<br/><sup>Evans2005 - Compartmental mode</sup></summary><sup>Evans2005 - Compartmental model for antineoplastic drug topotecan in breast cancer cells</sup></details>|<details><summary>pass</summary>[Evans2005.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000946#Files)</details>|pass|<details><summary>pass</summary>[Evans2005.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000946#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Evans2005.sedml537657.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Evans2005.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 970, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression '(1 + v0) * D' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 21 warnings of type Modeling practice (80701). The following is the first warning at line 504, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k_mi' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Evans2005.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Evans2005.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 970, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression '(1 + v0) * D' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 21 warnings of type Modeling practice (80701). The following is the first warning at line 504, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k_mi' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Evans2005.sedml537657.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Evans2005.sedml` is invalid.\n - Data generator `L_c_1_task1` is invalid.\n - Variable `p1_L_c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L_c']` are not defined. No namespaces are defined for the target.\n - Data generator `L_e_2_task1` is invalid.\n - Variable `p1_L_e_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L_e']` are not defined. No namespaces are defined for the target.\n - Data generator `L_n_3_task1` is invalid.\n - Variable `p1_L_n_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L_n']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Evans2005.sedml535464.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Evans2005.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 970, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression '(1 + v0) * D' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 21 warnings of type Modeling practice (80701). The following is the first warning at line 504, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k_mi' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Evans2005.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Evans2005.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 970, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression '(1 + v0) * D' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 21 warnings of type Modeling practice (80701). The following is the first warning at line 504, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k_mi' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Evans2005.sedml535464.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Evans2005.sedml` is invalid.\n - Data generator `L_c_1_task1` is invalid.\n - Variable `p1_L_c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L_c']` are not defined. No namespaces are defined for the target.\n - Data generator `L_e_2_task1` is invalid.\n - Variable `p1_L_e_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L_e']` are not defined. No namespaces are defined for the target.\n - Data generator `L_n_3_task1` is invalid.\n - Variable `p1_L_n_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L_n']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000947](https://www.ebi.ac.uk/biomodels/BIOMD0000000947)<br/><sup>Lee2017 - Paracetamol first-pa</sup></summary><sup>Lee2017 - Paracetamol first-pass metabolism PK model</sup></details>|<details><summary>pass</summary>[Lee2017_Paracetamol_Metabolism.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000947#Files)</details>|pass|<details><summary>pass</summary>[Evans2005.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000947#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Evans2005.sedml384172.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Lee2017_Paracetamol_Metabolism.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 830, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(500, leq(time, time_threshold), 400)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 504, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'P_para' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Evans2005.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Lee2017_Paracetamol_Metabolism.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 830, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(500, leq(time, time_threshold), 400)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 504, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'P_para' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Evans2005.sedml384172.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Evans2005.sedml` is invalid.\n - Data generator `L_c_1_task1` is invalid.\n - Variable `p1_L_c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L_c']` are not defined. No namespaces are defined for the target.\n - Data generator `L_e_2_task1` is invalid.\n - Variable `p1_L_e_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L_e']` are not defined. No namespaces are defined for the target.\n - Data generator `L_n_3_task1` is invalid.\n - Variable `p1_L_n_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L_n']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Evans2005.sedml677997.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Lee2017_Paracetamol_Metabolism.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 830, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(500, leq(time, time_threshold), 400)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 504, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'P_para' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Evans2005.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Lee2017_Paracetamol_Metabolism.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 830, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(500, leq(time, time_threshold), 400)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 504, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'P_para' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Evans2005.sedml677997.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Evans2005.sedml` is invalid.\n - Data generator `L_c_1_task1` is invalid.\n - Variable `p1_L_c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L_c']` are not defined. No namespaces are defined for the target.\n - Data generator `L_e_2_task1` is invalid.\n - Variable `p1_L_e_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L_e']` are not defined. No namespaces are defined for the target.\n - Data generator `L_n_3_task1` is invalid.\n - Variable `p1_L_n_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L_n']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000948](https://www.ebi.ac.uk/biomodels/BIOMD0000000948)<br/><sup>Landberg2009 - Alkylresorcinol</sup></summary><sup>Landberg2009 - Alkylresorcinol Dose Response</sup></details>|<details><summary>pass</summary>[Landberg2009.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000948#Files)</details>|FAIL|<details><summary>pass</summary>[Landberg2009.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000948#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Landberg2009.sedml871898.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Landberg2009.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 462, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(0, lt(time, Lag_time_1), piecewise(0.1, and(geq(time, Lag_time_1), lt(time, Lag_time_2)), 0.052))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 541, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -9), second (exponent = -1, multiplier = 3600, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'reaction') are mole (exponent = 2, multiplier = 1e-09, scale = 0), litre (exponent = -2, multiplier = 0.001, scale = 0).\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 370, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Lag_time_1' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Landberg2009.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Landberg2009.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 462, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(0, lt(time, Lag_time_1), piecewise(0.1, and(geq(time, Lag_time_1), lt(time, Lag_time_2)), 0.052))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 541, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -9), second (exponent = -1, multiplier = 3600, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'reaction') are mole (exponent = 2, multiplier = 1e-09, scale = 0), litre (exponent = -2, multiplier = 0.001, scale = 0).\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 370, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Lag_time_1' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Landberg2009.sedml871898.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Landberg2009.sedml` is invalid.\n - Data generator `AR_Central_1_task1` is invalid.\n - Variable `p1_AR_Central_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='AR_Central']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Landberg2009.sedml525326.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Landberg2009.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 462, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(0, lt(time, Lag_time_1), piecewise(0.1, and(geq(time, Lag_time_1), lt(time, Lag_time_2)), 0.052))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 541, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -9), second (exponent = -1, multiplier = 3600, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'reaction') are mole (exponent = 2, multiplier = 1e-09, scale = 0), litre (exponent = -2, multiplier = 0.001, scale = 0).\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 370, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Lag_time_1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Landberg2009.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Landberg2009.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 462, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'piecewise(0, lt(time, Lag_time_1), piecewise(0.1, and(geq(time, Lag_time_1), lt(time, Lag_time_2)), 0.052))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 541, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -9), second (exponent = -1, multiplier = 3600, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'reaction') are mole (exponent = 2, multiplier = 1e-09, scale = 0), litre (exponent = -2, multiplier = 0.001, scale = 0).\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 370, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Lag_time_1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Landberg2009.sedml525326.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Landberg2009.sedml` is invalid.\n - Data generator `AR_Central_1_task1` is invalid.\n - Variable `p1_AR_Central_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='AR_Central']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000949](https://www.ebi.ac.uk/biomodels/BIOMD0000000949)<br/><sup>Chitnis2008 - Mathematical mod</sup></summary><sup>Chitnis2008 - Mathematical model of malaria transmission</sup></details>|<details><summary>pass</summary>[Chitnis2008.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000949#Files)</details>|pass|<details><summary>pass</summary>[Chitnis2008.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000949#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Chitnis2008.sedml345346.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/Chitnis2008.sedml345346.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Chitnis2008.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - YDataReference from Curve `p1_curve_1_task1` cannot be resolved to a data generator with id `Exposed_Human_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_2_task1` cannot be resolved to a data generator with id `Infected_Human_2_task1` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_3_task1` cannot be resolved to a data generator with id `Recovered_3_task1` because multiple data generators have this id.\n - YDataReference from Curve `p3_curve_1_task1` cannot be resolved to a data generator with id `Exposed_Mosquito_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p3_curve_2_task1` cannot be resolved to a data generator with id `Infected_Mosquito_2_task1` because multiple data generators have this id.\n - YDataReference from Curve `p5_curve_1_task1` cannot be resolved to a data generator with id `Exposed_Human_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p5_curve_2_task1` cannot be resolved to a data generator with id `Infected_Human_2_task1` because multiple data generators have this id.\n - YDataReference from Curve `p5_curve_3_task1` cannot be resolved to a data generator with id `Recovered_3_task1` because multiple data generators have this id.\n - YDataReference from Curve `p6_curve_1_task1` cannot be resolved to a data generator with id `Exposed_Mosquito_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p6_curve_2_task1` cannot be resolved to a data generator with id `Infected_Mosquito_2_task1` because multiple data generators have this id.\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Chitnis2008.sedml337467.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Chitnis2008.sedml337467.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Chitnis2008.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - YDataReference from Curve `p1_curve_1_task1` cannot be resolved to a data generator with id `Exposed_Human_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_2_task1` cannot be resolved to a data generator with id `Infected_Human_2_task1` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_3_task1` cannot be resolved to a data generator with id `Recovered_3_task1` because multiple data generators have this id.\n - YDataReference from Curve `p3_curve_1_task1` cannot be resolved to a data generator with id `Exposed_Mosquito_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p3_curve_2_task1` cannot be resolved to a data generator with id `Infected_Mosquito_2_task1` because multiple data generators have this id.\n - YDataReference from Curve `p5_curve_1_task1` cannot be resolved to a data generator with id `Exposed_Human_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p5_curve_2_task1` cannot be resolved to a data generator with id `Infected_Human_2_task1` because multiple data generators have this id.\n - YDataReference from Curve `p5_curve_3_task1` cannot be resolved to a data generator with id `Recovered_3_task1` because multiple data generators have this id.\n - YDataReference from Curve `p6_curve_1_task1` cannot be resolved to a data generator with id `Exposed_Mosquito_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p6_curve_2_task1` cannot be resolved to a data generator with id `Infected_Mosquito_2_task1` because multiple data generators have this id.\n'```</details>|
|<details><summary>[BIOMD0000000950](https://www.ebi.ac.uk/biomodels/BIOMD0000000950)<br/><sup>Chitnis2012 - Model Rift Valle</sup></summary><sup>Chitnis2012 - Model Rift Valley Fever transmission between cattle and mosquitoes (Model 1)</sup></details>|<details><summary>pass</summary>[Chitnis2012_model_1.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000950#Files)</details>|pass|<details><summary>pass</summary>[Chitnis2012_model_1.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000950#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Chitnis2012_model_1.sedml345256.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Chitnis2012_model_1.xml` has warnings.\r\n - 27 warnings of type SBML unit consistency (99505). The following is the first warning at line 761, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'piecewise(1000, eq(Season_Dummy_Variable, 0), 1000)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 22 warnings of type Modeling practice (80701). The following is the first warning at line 429, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'u_h' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.1\r\n - Output end time: 3650.0\r\n - Number of steps: 36499\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Chitnis2012_model_1.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Chitnis2012_model_1.xml` has warnings.\r\n - 27 warnings of type SBML unit consistency (99505). The following is the first warning at line 761, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'piecewise(1000, eq(Season_Dummy_Variable, 0), 1000)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 22 warnings of type Modeling practice (80701). The following is the first warning at line 429, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'u_h' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.1\r\n - Output end time: 3650.0\r\n - Number of steps: 36499\x1b[0m\r\n\r\n`/root/in/Chitnis2012_model_1.sedml345256.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Chitnis2012_model_1.sedml` is invalid.\n - Data generator `Total_Infected_Cattle_1_task1` is invalid.\n - Variable `p1_Total_Infected_Cattle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Infected_Cattle']` are not defined. No namespaces are defined for the target.\n - Data generator `A_h_2_task1` is invalid.\n - Variable `p1_A_h_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='A_h']` are not defined. No namespaces are defined for the target.\n - Data generator `I_h_3_task1` is invalid.\n - Variable `p1_I_h_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I_h']` are not defined. No namespaces are defined for the target.\n - Data generator `E_v_1_task1` is invalid.\n - Variable `p2_E_v_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='E_v']` are not defined. No namespaces are defined for the target.\n - Data generator `I_v_2_task1` is invalid.\n - Variable `p2_I_v_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I_v']` are not defined. No namespaces are defined for the target.\n - Data generator `N_h_1_task1` is invalid.\n - Variable `p3_N_h_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='N_h']` are not defined. No namespaces are defined for the target.\n - Data generator `R_h_2_task1` is invalid.\n - Variable `p3_R_h_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='R_h']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Chitnis2012_model_1.sedml337334.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Chitnis2012_model_1.xml` has warnings.\r\n - 27 warnings of type SBML unit consistency (99505). The following is the first warning at line 761, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'piecewise(1000, eq(Season_Dummy_Variable, 0), 1000)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 22 warnings of type Modeling practice (80701). The following is the first warning at line 429, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'u_h' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.1\r\n - Output end time: 3650.0\r\n - Number of steps: 36499\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Chitnis2012_model_1.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Chitnis2012_model_1.xml` has warnings.\r\n - 27 warnings of type SBML unit consistency (99505). The following is the first warning at line 761, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'piecewise(1000, eq(Season_Dummy_Variable, 0), 1000)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 22 warnings of type Modeling practice (80701). The following is the first warning at line 429, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'u_h' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.1\r\n - Output end time: 3650.0\r\n - Number of steps: 36499\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Chitnis2012_model_1.sedml337334.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Chitnis2012_model_1.sedml` is invalid.\n - Data generator `Total_Infected_Cattle_1_task1` is invalid.\n - Variable `p1_Total_Infected_Cattle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Infected_Cattle']` are not defined. No namespaces are defined for the target.\n - Data generator `A_h_2_task1` is invalid.\n - Variable `p1_A_h_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='A_h']` are not defined. No namespaces are defined for the target.\n - Data generator `I_h_3_task1` is invalid.\n - Variable `p1_I_h_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I_h']` are not defined. No namespaces are defined for the target.\n - Data generator `E_v_1_task1` is invalid.\n - Variable `p2_E_v_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='E_v']` are not defined. No namespaces are defined for the target.\n - Data generator `I_v_2_task1` is invalid.\n - Variable `p2_I_v_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I_v']` are not defined. No namespaces are defined for the target.\n - Data generator `N_h_1_task1` is invalid.\n - Variable `p3_N_h_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='N_h']` are not defined. No namespaces are defined for the target.\n - Data generator `R_h_2_task1` is invalid.\n - Variable `p3_R_h_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='R_h']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000951](https://www.ebi.ac.uk/biomodels/BIOMD0000000951)<br/><sup>Mitrophanov2015 - Simulating e</sup></summary><sup>Mitrophanov2015 - Simulating extended Hockin Blood Coagulation Model under varied pH</sup></details>|<details><summary>pass</summary>[Mitrophanov2015.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000951#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000000952](https://www.ebi.ac.uk/biomodels/BIOMD0000000952)<br/><sup>Rodenfels2019 - Heat Oscillati</sup></summary><sup>Rodenfels2019 - Heat Oscillations Driven by the Embryonic Cell Cycle Reveal the Energetic Costs of Signaling</sup></details>|<details><summary>pass</summary>[Rodenfels2019_V1.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000952#Files)</details>|pass|<details><summary>FAIL</summary>[Rodenfels2019_V1.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000952#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Rodenfels2019_V1.sedml699793.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/Rodenfels2019_V1.sedml699793.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Rodenfels2019_V1.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - DataReference from DataSet `ds_2_task1` cannot be resolved to a data generator with id `Cyclin_B1_Cdk1_complex_phosphorylated_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_1_task1` cannot be resolved to a data generator with id `Cyclin_B1_Cdk1_complex_phosphorylated_1_task1` because multiple data generators have this id.\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Rodenfels2019_V1.sedml240111.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Rodenfels2019_V1.sedml240111.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Rodenfels2019_V1.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - DataReference from DataSet `ds_2_task1` cannot be resolved to a data generator with id `Cyclin_B1_Cdk1_complex_phosphorylated_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_1_task1` cannot be resolved to a data generator with id `Cyclin_B1_Cdk1_complex_phosphorylated_1_task1` because multiple data generators have this id.\n'```</details>|
|<details><summary>[BIOMD0000000953](https://www.ebi.ac.uk/biomodels/BIOMD0000000953)<br/><sup>Queralt2006 - Initiation of mi</sup></summary><sup>Queralt2006 - Initiation of mitotic exit by downregulation of PP2A in budding yeast</sup></details>|<details><summary>pass</summary>[queralt2006_final.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000953#Files)</details>|FAIL|<details><summary>pass</summary>[queralt2006_final.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000953#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/queralt2006_final.sedml110421.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `queralt2006_final.xml` has warnings.\r\n - 32 warnings of type SBML unit consistency (99505). The following is the first warning at line 2434, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'clb2 / (1 + Inh)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10512). The following is the first warning at line 2434, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <species>, the units of the rule's right-hand side are expected to be consistent with the units of the species' quantity.\r\n Reference: L2V4 Section 4.11.3\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3) but the units returned by the <math> expression of the <assignmentRule> with variable 'cdk' are dimensionless (exponent = 1, multiplier = 1, scale = 0).\r\n - 46 warnings of type Modeling practice (80701). The following is the first warning at line 1720, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'ks_clb2' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./queralt2006_final.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `queralt2006_final.xml` has warnings.\r\n - 32 warnings of type SBML unit consistency (99505). The following is the first warning at line 2434, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'clb2 / (1 + Inh)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10512). The following is the first warning at line 2434, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <species>, the units of the rule's right-hand side are expected to be consistent with the units of the species' quantity.\r\n Reference: L2V4 Section 4.11.3\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3) but the units returned by the <math> expression of the <assignmentRule> with variable 'cdk' are dimensionless (exponent = 1, multiplier = 1, scale = 0).\r\n - 46 warnings of type Modeling practice (80701). The following is the first warning at line 1720, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'ks_clb2' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/queralt2006_final.sedml110421.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./queralt2006_final.sedml` is invalid.\n - Data generator `Cdc14x2_1_task1` is invalid.\n - Variable `p1_Cdc14x2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cdc14x2']` are not defined. No namespaces are defined for the target.\n - Data generator `cdc20_2_task1` is invalid.\n - Variable `p1_cdc20_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='cdc20']` are not defined. No namespaces are defined for the target.\n - Data generator `cdh1_3_task1` is invalid.\n - Variable `p1_cdh1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='cdh1']` are not defined. No namespaces are defined for the target.\n - Data generator `clb2_4_task1` is invalid.\n - Variable `p1_clb2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='clb2']` are not defined. No namespaces are defined for the target.\n - Data generator `men_5_task1` is invalid.\n - Variable `p1_men_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='men']` are not defined. No namespaces are defined for the target.\n - Data generator `net1p_1_task1` is invalid.\n - Variable `p2_net1p_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='net1p']` are not defined. No namespaces are defined for the target.\n - Data generator `pp2a_2_task1` is invalid.\n - Variable `p2_pp2a_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='pp2a']` are not defined. No namespaces are defined for the target.\n - Data generator `securin_total_3_task1` is invalid.\n - Variable `p2_securin_total_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='securin_total']` are not defined. No namespaces are defined for the target.\n - Data generator `separase_4_task1` is invalid.\n - Variable `p2_separase_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='separase']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/queralt2006_final.sedml218422.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `queralt2006_final.xml` has warnings.\r\n - 32 warnings of type SBML unit consistency (99505). The following is the first warning at line 2434, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'clb2 / (1 + Inh)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10512). The following is the first warning at line 2434, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <species>, the units of the rule's right-hand side are expected to be consistent with the units of the species' quantity.\r\n Reference: L2V4 Section 4.11.3\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3) but the units returned by the <math> expression of the <assignmentRule> with variable 'cdk' are dimensionless (exponent = 1, multiplier = 1, scale = 0).\r\n - 46 warnings of type Modeling practice (80701). The following is the first warning at line 1720, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'ks_clb2' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./queralt2006_final.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `queralt2006_final.xml` has warnings.\r\n - 32 warnings of type SBML unit consistency (99505). The following is the first warning at line 2434, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'clb2 / (1 + Inh)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10512). The following is the first warning at line 2434, column 6:\r\n - When the 'variable' in an <assignmentRule> refers to a <species>, the units of the rule's right-hand side are expected to be consistent with the units of the species' quantity.\r\n Reference: L2V4 Section 4.11.3\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -3), litre (exponent = -1, multiplier = 1, scale = -3) but the units returned by the <math> expression of the <assignmentRule> with variable 'cdk' are dimensionless (exponent = 1, multiplier = 1, scale = 0).\r\n - 46 warnings of type Modeling practice (80701). The following is the first warning at line 1720, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'ks_clb2' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/queralt2006_final.sedml218422.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./queralt2006_final.sedml` is invalid.\n - Data generator `Cdc14x2_1_task1` is invalid.\n - Variable `p1_Cdc14x2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cdc14x2']` are not defined. No namespaces are defined for the target.\n - Data generator `cdc20_2_task1` is invalid.\n - Variable `p1_cdc20_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='cdc20']` are not defined. No namespaces are defined for the target.\n - Data generator `cdh1_3_task1` is invalid.\n - Variable `p1_cdh1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='cdh1']` are not defined. No namespaces are defined for the target.\n - Data generator `clb2_4_task1` is invalid.\n - Variable `p1_clb2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='clb2']` are not defined. No namespaces are defined for the target.\n - Data generator `men_5_task1` is invalid.\n - Variable `p1_men_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='men']` are not defined. No namespaces are defined for the target.\n - Data generator `net1p_1_task1` is invalid.\n - Variable `p2_net1p_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='net1p']` are not defined. No namespaces are defined for the target.\n - Data generator `pp2a_2_task1` is invalid.\n - Variable `p2_pp2a_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='pp2a']` are not defined. No namespaces are defined for the target.\n - Data generator `securin_total_3_task1` is invalid.\n - Variable `p2_securin_total_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='securin_total']` are not defined. No namespaces are defined for the target.\n - Data generator `separase_4_task1` is invalid.\n - Variable `p2_separase_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='separase']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000954](https://www.ebi.ac.uk/biomodels/BIOMD0000000954)<br/><sup>Pandey2018-reversible transiti</sup></summary><sup>Pandey2018-reversible transition between quiescence and proliferation</sup></details>|<details><summary>pass</summary>[Pandey2018_wildtype.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000954#Files)</details>|pass|<details><summary>MultipleSEDMLs</summary>['Pandey2018_Figure2D.sedml', 'Pandey2018_Figure2C.sedml', 'Pandey2018_Figure2B.sedml', 'Pandey2018_Figure2A.sedml']</details>|NA|NA|NA|
|<details><summary>[BIOMD0000000955](https://www.ebi.ac.uk/biomodels/BIOMD0000000955)<br/><sup>Giordano2020 - SIDARTHE model </sup></summary><sup>Giordano2020 - SIDARTHE model of COVID-19 spread in Italy</sup></details>|<details><summary>pass</summary>[Giordano2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000955#Files)</details>|FAIL|<details><summary>pass</summary>[Giordano2020.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000955#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Giordano2020.sedml550465.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Giordano2020.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (10503). The following is the first warning at line 1140, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'Italy * epsilon * Infected' in the <kineticLaw> element of the <reaction> with id 'Infected_to_Diagnosed' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 6 warnings of type SBML unit consistency (99508). The following is the first warning at line 1007, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'ModelValue_16' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type SBML unit consistency (20702). The following is the first warning at line 1007, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'ModelValue_16' does not have a 'units' attribute.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 1119, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Susceptible_to_Infected') are litre (exponent = -1, multiplier = 0.001, scale = 0), item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0).\r\n - 41 warnings of type SBML unit consistency (99505). The following is the first warning at line 1761, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <event> <delay> expression '0' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 1007, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'ModelValue_16' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Giordano2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Giordano2020.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (10503). The following is the first warning at line 1140, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'Italy * epsilon * Infected' in the <kineticLaw> element of the <reaction> with id 'Infected_to_Diagnosed' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 6 warnings of type SBML unit consistency (99508). The following is the first warning at line 1007, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'ModelValue_16' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type SBML unit consistency (20702). The following is the first warning at line 1007, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'ModelValue_16' does not have a 'units' attribute.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 1119, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Susceptible_to_Infected') are litre (exponent = -1, multiplier = 0.001, scale = 0), item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0).\r\n - 41 warnings of type SBML unit consistency (99505). The following is the first warning at line 1761, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <event> <delay> expression '0' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 1007, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'ModelValue_16' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Giordano2020.sedml550465.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Giordano2020.sedml` is invalid.\n - Data generator `Infected_1_task1` is invalid.\n - Variable `p1_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Diagnosed_2_task1` is invalid.\n - Variable `p1_Diagnosed_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Diagnosed']` are not defined. No namespaces are defined for the target.\n - Data generator `Ailing_3_task1` is invalid.\n - Variable `p1_Ailing_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Ailing']` are not defined. No namespaces are defined for the target.\n - Data generator `Recognized_4_task1` is invalid.\n - Variable `p1_Recognized_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Recognized']` are not defined. No namespaces are defined for the target.\n - Data generator `Threatened_5_task1` is invalid.\n - Variable `p1_Threatened_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Threatened']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Giordano2020.sedml739486.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Giordano2020.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (10503). The following is the first warning at line 1140, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'Italy * epsilon * Infected' in the <kineticLaw> element of the <reaction> with id 'Infected_to_Diagnosed' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 6 warnings of type SBML unit consistency (99508). The following is the first warning at line 1007, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'ModelValue_16' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type SBML unit consistency (20702). The following is the first warning at line 1007, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'ModelValue_16' does not have a 'units' attribute.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 1119, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Susceptible_to_Infected') are litre (exponent = -1, multiplier = 0.001, scale = 0), item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0).\r\n - 41 warnings of type SBML unit consistency (99505). The following is the first warning at line 1761, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <event> <delay> expression '0' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 1007, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'ModelValue_16' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Giordano2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Giordano2020.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (10503). The following is the first warning at line 1140, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'Italy * epsilon * Infected' in the <kineticLaw> element of the <reaction> with id 'Infected_to_Diagnosed' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 6 warnings of type SBML unit consistency (99508). The following is the first warning at line 1007, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'ModelValue_16' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type SBML unit consistency (20702). The following is the first warning at line 1007, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'ModelValue_16' does not have a 'units' attribute.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 1119, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Susceptible_to_Infected') are litre (exponent = -1, multiplier = 0.001, scale = 0), item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0).\r\n - 41 warnings of type SBML unit consistency (99505). The following is the first warning at line 1761, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <event> <delay> expression '0' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 1007, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'ModelValue_16' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Giordano2020.sedml739486.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Giordano2020.sedml` is invalid.\n - Data generator `Infected_1_task1` is invalid.\n - Variable `p1_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Diagnosed_2_task1` is invalid.\n - Variable `p1_Diagnosed_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Diagnosed']` are not defined. No namespaces are defined for the target.\n - Data generator `Ailing_3_task1` is invalid.\n - Variable `p1_Ailing_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Ailing']` are not defined. No namespaces are defined for the target.\n - Data generator `Recognized_4_task1` is invalid.\n - Variable `p1_Recognized_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Recognized']` are not defined. No namespaces are defined for the target.\n - Data generator `Threatened_5_task1` is invalid.\n - Variable `p1_Threatened_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Threatened']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000956](https://www.ebi.ac.uk/biomodels/BIOMD0000000956)<br/><sup>Bertozzi2020 - SIR model of sc</sup></summary><sup>Bertozzi2020 - SIR model of scenarios of COVID-19 spread in CA and NY</sup></details>|<details><summary>pass</summary>[Bertozzi2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000956#Files)</details>|pass|<details><summary>pass</summary>[Bertozzi2020.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000956#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Bertozzi2020.sedml827625.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Bertozzi2020.xml` has warnings.\r\n - 34 warnings of type SBML unit consistency (99505). The following is the first warning at line 747, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Io / Pop' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 36 warnings of type Modeling practice (80701). The following is the first warning at line 355, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Pop_CA' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 215.0\r\n - Number of steps: 282\r\n - Task `task2` has warnings.\r\n - Child range `range1` is longer than its main range (4 > 1). The tail elements of the range will be ignored.\r\n - Child range `range1` is longer than its main range (4 > 1). The tail elements of the range will be ignored.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task2`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Bertozzi2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Bertozzi2020.xml` has warnings.\r\n - 34 warnings of type SBML unit consistency (99505). The following is the first warning at line 747, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Io / Pop' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 36 warnings of type Modeling practice (80701). The following is the first warning at line 355, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Pop_CA' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 215.0\r\n - Number of steps: 282\r\n - Task `task2` has warnings.\r\n - Child range `range1` is longer than its main range (4 > 1). The tail elements of the range will be ignored.\r\n - Child range `range1` is longer than its main range (4 > 1). The tail elements of the range will be ignored.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task2`\x1b[0m\r\n\r\n`/root/in/Bertozzi2020.sedml827625.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Bertozzi2020.sedml` is invalid.\n - Each identified SED object must have a unique id. Multiple objects have the following ids:\n - p3_Time_task1\n - p3_Time_task2\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Ro_CA']` are not defined. No namespaces are defined for the target.\n - Change 2 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Io_CA']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_1_task1` is invalid.\n - Variable `p3_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Time_1_task1` is invalid.\n - Variable `p3_Time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Time']` are not defined. No namespaces are defined for the target.\n - Data generator `Recovered_2_task1` is invalid.\n - Variable `p3_Recovered_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Recovered']` are not defined. No namespaces are defined for the target.\n - Data generator `Time_2_task1` is invalid.\n - Variable `p3_Time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Time']` are not defined. No namespaces are defined for the target.\n - Data generator `Susceptible_3_task1` is invalid.\n - Variable `p3_Susceptible_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Susceptible']` are not defined. No namespaces are defined for the target.\n - Data generator `Time_3_task1` is invalid.\n - Variable `p3_Time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Time']` are not defined. No namespaces are defined for the target.\n - Data generator `Peak_Time_4_task1` is invalid.\n - Variable `p3_Peak_Time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Peak_Time']` are not defined. No namespaces are defined for the target.\n - Data generator `Time_4_task1` is invalid.\n - Variable `p3_Time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Time']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_1_task2` is invalid.\n - Variable `p3_Infected_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Time_1_task2` is invalid.\n - Variable `p3_Time_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Time']` are not defined. No namespaces are defined for the target.\n - Data generator `Recovered_2_task2` is invalid.\n - Variable `p3_Recovered_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Recovered']` are not defined. No namespaces are defined for the target.\n - Data generator `Time_2_task2` is invalid.\n - Variable `p3_Time_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Time']` are not defined. No namespaces are defined for the target.\n - Data generator `Susceptible_3_task2` is invalid.\n - Variable `p3_Susceptible_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Susceptible']` are not defined. No namespaces are defined for the target.\n - Data generator `Time_3_task2` is invalid.\n - Variable `p3_Time_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Time']` are not defined. No namespaces are defined for the target.\n - Data generator `Peak_Time_4_task2` is invalid.\n - Variable `p3_Peak_Time_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Peak_Time']` are not defined. No namespaces are defined for the target.\n - Data generator `Time_4_task2` is invalid.\n - Variable `p3_Time_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Time']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Bertozzi2020.sedml288030.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Bertozzi2020.xml` has warnings.\r\n - 34 warnings of type SBML unit consistency (99505). The following is the first warning at line 747, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Io / Pop' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 36 warnings of type Modeling practice (80701). The following is the first warning at line 355, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Pop_CA' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 215.0\r\n - Number of steps: 282\r\n - Task `task2` has warnings.\r\n - Child range `range1` is longer than its main range (4 > 1). The tail elements of the range will be ignored.\r\n - Child range `range1` is longer than its main range (4 > 1). The tail elements of the range will be ignored.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task2`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Bertozzi2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Bertozzi2020.xml` has warnings.\r\n - 34 warnings of type SBML unit consistency (99505). The following is the first warning at line 747, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Io / Pop' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 36 warnings of type Modeling practice (80701). The following is the first warning at line 355, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Pop_CA' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 215.0\r\n - Number of steps: 282\r\n - Task `task2` has warnings.\r\n - Child range `range1` is longer than its main range (4 > 1). The tail elements of the range will be ignored.\r\n - Child range `range1` is longer than its main range (4 > 1). The tail elements of the range will be ignored.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task2`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Bertozzi2020.sedml288030.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Bertozzi2020.sedml` is invalid.\n - Each identified SED object must have a unique id. Multiple objects have the following ids:\n - p3_Time_task1\n - p3_Time_task2\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Ro_CA']` are not defined. No namespaces are defined for the target.\n - Change 2 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Io_CA']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_1_task1` is invalid.\n - Variable `p3_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Time_1_task1` is invalid.\n - Variable `p3_Time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Time']` are not defined. No namespaces are defined for the target.\n - Data generator `Recovered_2_task1` is invalid.\n - Variable `p3_Recovered_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Recovered']` are not defined. No namespaces are defined for the target.\n - Data generator `Time_2_task1` is invalid.\n - Variable `p3_Time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Time']` are not defined. No namespaces are defined for the target.\n - Data generator `Susceptible_3_task1` is invalid.\n - Variable `p3_Susceptible_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Susceptible']` are not defined. No namespaces are defined for the target.\n - Data generator `Time_3_task1` is invalid.\n - Variable `p3_Time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Time']` are not defined. No namespaces are defined for the target.\n - Data generator `Peak_Time_4_task1` is invalid.\n - Variable `p3_Peak_Time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Peak_Time']` are not defined. No namespaces are defined for the target.\n - Data generator `Time_4_task1` is invalid.\n - Variable `p3_Time_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Time']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_1_task2` is invalid.\n - Variable `p3_Infected_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Time_1_task2` is invalid.\n - Variable `p3_Time_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Time']` are not defined. No namespaces are defined for the target.\n - Data generator `Recovered_2_task2` is invalid.\n - Variable `p3_Recovered_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Recovered']` are not defined. No namespaces are defined for the target.\n - Data generator `Time_2_task2` is invalid.\n - Variable `p3_Time_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Time']` are not defined. No namespaces are defined for the target.\n - Data generator `Susceptible_3_task2` is invalid.\n - Variable `p3_Susceptible_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Susceptible']` are not defined. No namespaces are defined for the target.\n - Data generator `Time_3_task2` is invalid.\n - Variable `p3_Time_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Time']` are not defined. No namespaces are defined for the target.\n - Data generator `Peak_Time_4_task2` is invalid.\n - Variable `p3_Peak_Time_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Peak_Time']` are not defined. No namespaces are defined for the target.\n - Data generator `Time_4_task2` is invalid.\n - Variable `p3_Time_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Time']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000957](https://www.ebi.ac.uk/biomodels/BIOMD0000000957)<br/><sup>Roda2020 - SIR model of COVID-</sup></summary><sup>Roda2020 - SIR model of COVID-19 spread in Wuhan</sup></details>|<details><summary>pass</summary>[Roda2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000957#Files)</details>|pass|<details><summary>pass</summary>[Roda2020.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000957#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Roda2020.sedml990308.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Roda2020.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 471, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Wuhan * Rate_Law_for_Susceptible_to_Infected(beta, Infected, Susceptible)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 3 warnings of type Modeling practice (80701). The following is the first warning at line 369, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Roda2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Roda2020.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 471, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Wuhan * Rate_Law_for_Susceptible_to_Infected(beta, Infected, Susceptible)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 3 warnings of type Modeling practice (80701). The following is the first warning at line 369, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/Roda2020.sedml990308.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Roda2020.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='beta']` are not defined. No namespaces are defined for the target.\n - Data generator `Susceptible_1_task1` is invalid.\n - Variable `p1_Susceptible_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Susceptible']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_2_task1` is invalid.\n - Variable `p1_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Recovered_3_task1` is invalid.\n - Variable `p1_Recovered_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Recovered']` are not defined. No namespaces are defined for the target.\n - Data generator `Confirmed_4_task1` is invalid.\n - Variable `p1_Confirmed_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Confirmed']` are not defined. No namespaces are defined for the target.\n - Data generator `Susceptible_1_task2` is invalid.\n - Variable `p1_Susceptible_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Susceptible']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_2_task2` is invalid.\n - Variable `p1_Infected_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Recovered_3_task2` is invalid.\n - Variable `p1_Recovered_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Recovered']` are not defined. No namespaces are defined for the target.\n - Data generator `Confirmed_4_task2` is invalid.\n - Variable `p1_Confirmed_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Confirmed']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Roda2020.sedml620554.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Roda2020.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 471, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Wuhan * Rate_Law_for_Susceptible_to_Infected(beta, Infected, Susceptible)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 3 warnings of type Modeling practice (80701). The following is the first warning at line 369, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Roda2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Roda2020.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 471, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Wuhan * Rate_Law_for_Susceptible_to_Infected(beta, Infected, Susceptible)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 3 warnings of type Modeling practice (80701). The following is the first warning at line 369, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Roda2020.sedml620554.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Roda2020.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='beta']` are not defined. No namespaces are defined for the target.\n - Data generator `Susceptible_1_task1` is invalid.\n - Variable `p1_Susceptible_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Susceptible']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_2_task1` is invalid.\n - Variable `p1_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Recovered_3_task1` is invalid.\n - Variable `p1_Recovered_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Recovered']` are not defined. No namespaces are defined for the target.\n - Data generator `Confirmed_4_task1` is invalid.\n - Variable `p1_Confirmed_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Confirmed']` are not defined. No namespaces are defined for the target.\n - Data generator `Susceptible_1_task2` is invalid.\n - Variable `p1_Susceptible_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Susceptible']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_2_task2` is invalid.\n - Variable `p1_Infected_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Recovered_3_task2` is invalid.\n - Variable `p1_Recovered_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Recovered']` are not defined. No namespaces are defined for the target.\n - Data generator `Confirmed_4_task2` is invalid.\n - Variable `p1_Confirmed_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Confirmed']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000958](https://www.ebi.ac.uk/biomodels/BIOMD0000000958)<br/><sup>Ndairou2020 - early-stage tran</sup></summary><sup>Ndairou2020 - early-stage transmission dynamics of COVID-19 in Wuhan</sup></details>|<details><summary>pass</summary>[Ndairou2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000958#Files)</details>|FAIL|<details><summary>FAIL</summary>[Ndairou2020.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000958#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Ndairou2020.sedml294943.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/Ndairou2020.sedml294943.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Ndairou2020.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmp7qms5uql/./model1.xml` is not a file.\n - Data generator `Confimed_cases_1_task1` is invalid.\n - Variable `p1_Confimed_cases_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Confimed_cases']` are not defined. No namespaces are defined for the target.\n - Output `plot_2_task1` is invalid.\n - Plot must have at least one curve.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Ndairou2020.sedml506103.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Ndairou2020.sedml506103.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Ndairou2020.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmpxq05pf21/./model1.xml` is not a file.\n - Data generator `Confimed_cases_1_task1` is invalid.\n - Variable `p1_Confimed_cases_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Confimed_cases']` are not defined. No namespaces are defined for the target.\n - Output `plot_2_task1` is invalid.\n - Plot must have at least one curve.\n"```</details>|
|<details><summary>[BIOMD0000000959](https://www.ebi.ac.uk/biomodels/BIOMD0000000959)<br/><sup>Kok2020 - IFNalpha-induced sig</sup></summary><sup>Kok2020 - IFNalpha-induced signaling in Huh7.5 cells</sup></details>|<details><summary>pass</summary>[Kok2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000959#Files)</details>|pass|<details><summary>pass</summary>[Kok2020.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000959#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Kok2020.sedml282961.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Kok2020.xml` has warnings.\r\n - 82 warnings of type SBML unit consistency (99505). The following is the first warning at line 2609, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'log10(IRF2mRNA)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 87 warnings of type Modeling practice (80701). The following is the first warning at line 2502, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'IRF2_mRNA_log10' does not have a 'units' attribute.\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Kok2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Kok2020.xml` has warnings.\r\n - 82 warnings of type SBML unit consistency (99505). The following is the first warning at line 2609, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'log10(IRF2mRNA)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 87 warnings of type Modeling practice (80701). The following is the first warning at line 2502, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'IRF2_mRNA_log10' does not have a 'units' attribute.\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/Kok2020.sedml282961.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Kok2020.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IFN']` are not defined. No namespaces are defined for the target.\n - Data generator `Rec_1_task1` is invalid.\n - Variable `p1_Rec_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Rec']` are not defined. No namespaces are defined for the target.\n - Data generator `SOCS1_2_task1` is invalid.\n - Variable `p1_SOCS1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SOCS1']` are not defined. No namespaces are defined for the target.\n - Data generator `IFN_3_task1` is invalid.\n - Variable `p1_IFN_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IFN']` are not defined. No namespaces are defined for the target.\n - Data generator `aRecIFN_4_task1` is invalid.\n - Variable `p1_aRecIFN_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='aRecIFN']` are not defined. No namespaces are defined for the target.\n - Data generator `USP18_5_task1` is invalid.\n - Variable `p1_USP18_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='USP18']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT1c_6_task1` is invalid.\n - Variable `p1_STAT1c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT1c']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2c_7_task1` is invalid.\n - Variable `p1_STAT2c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2c']` are not defined. No namespaces are defined for the target.\n - Data generator `pSTAT1pSTAT2c_8_task1` is invalid.\n - Variable `p1_pSTAT1pSTAT2c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='pSTAT1pSTAT2c']` are not defined. No namespaces are defined for the target.\n - Data generator `pSTAT1pSTAT2n_9_task1` is invalid.\n - Variable `p1_pSTAT1pSTAT2n_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='pSTAT1pSTAT2n']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT1n_10_task1` is invalid.\n - Variable `p1_STAT1n_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT1n']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2n_11_task1` is invalid.\n - Variable `p1_STAT2n_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2n']` are not defined. No namespaces are defined for the target.\n - Data generator `ISGF3c_12_task1` is invalid.\n - Variable `p1_ISGF3c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ISGF3c']` are not defined. No namespaces are defined for the target.\n - Data generator `ISGF3n_13_task1` is invalid.\n - Variable `p1_ISGF3n_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ISGF3n']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF9n_14_task1` is invalid.\n - Variable `p1_IRF9n_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IRF9n']` are not defined. No namespaces are defined for the target.\n - Data generator `pSTAT1dimc_15_task1` is invalid.\n - Variable `p1_pSTAT1dimc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='pSTAT1dimc']` are not defined. No namespaces are defined for the target.\n - Data generator `pSTAT1dimn_16_task1` is invalid.\n - Variable `p1_pSTAT1dimn_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='pSTAT1dimn']` are not defined. No namespaces are defined for the target.\n - Data generator `OccGAS_ISREbs_17_task1` is invalid.\n - Variable `p1_OccGAS_ISREbs_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='OccGAS_ISREbs']` are not defined. No namespaces are defined for the target.\n - Data generator `OccGASbs_18_task1` is invalid.\n - Variable `p1_OccGASbs_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='OccGASbs']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF9c_19_task1` is invalid.\n - Variable `p1_IRF9c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IRF9c']` are not defined. No namespaces are defined for the target.\n - Data generator `SOCS3_20_task1` is invalid.\n - Variable `p1_SOCS3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SOCS3']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT1mRNA_21_task1` is invalid.\n - Variable `p1_STAT1mRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT1mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2mRNA_22_task1` is invalid.\n - Variable `p1_STAT2mRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF9mRNA_23_task1` is invalid.\n - Variable `p1_IRF9mRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IRF9mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF2mRNA_24_task1` is invalid.\n - Variable `p1_IRF2mRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IRF2mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF2_25_task1` is invalid.\n - Variable `p1_IRF2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IRF2']` are not defined. No namespaces are defined for the target.\n - Data generator `USP18mRNA_26_task1` is invalid.\n - Variable `p1_USP18mRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='USP18mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `SOCS1mRNA_27_task1` is invalid.\n - Variable `p1_SOCS1mRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SOCS1mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `SOCS3mRNA_28_task1` is invalid.\n - Variable `p1_SOCS3mRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SOCS3mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT1_LC_1_29_task1` is invalid.\n - Variable `p1_STAT1_LC_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT1_LC_1']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT1_LC_2_30_task1` is invalid.\n - Variable `p1_STAT1_LC_2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT1_LC_2']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT1_LC_3_31_task1` is invalid.\n - Variable `p1_STAT1_LC_3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT1_LC_3']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2_LC_1_32_task1` is invalid.\n - Variable `p1_STAT2_LC_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2_LC_1']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2_LC_2_33_task1` is invalid.\n - Variable `p1_STAT2_LC_2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2_LC_2']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2_LC_3_34_task1` is invalid.\n - Variable `p1_STAT2_LC_3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2_LC_3']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2_LC_4_35_task1` is invalid.\n - Variable `p1_STAT2_LC_4_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2_LC_4']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2_LC_5_36_task1` is invalid.\n - Variable `p1_STAT2_LC_5_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2_LC_5']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF9_LC_1_37_task1` is invalid.\n - Variable `p1_IRF9_LC_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IRF9_LC_1']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF9_LC_2_38_task1` is invalid.\n - Variable `p1_IRF9_LC_2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IRF9_LC_2']` are not defined. No namespaces are defined for the target.\n - Data generator `USP18_LC_1_39_task1` is invalid.\n - Variable `p1_USP18_LC_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='USP18_LC_1']` are not defined. No namespaces are defined for the target.\n - Data generator `USP18_LC_2_40_task1` is invalid.\n - Variable `p1_USP18_LC_2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='USP18_LC_2']` are not defined. No namespaces are defined for the target.\n - Data generator `SOCS1_LC_1_41_task1` is invalid.\n - Variable `p1_SOCS1_LC_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SOCS1_LC_1']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF2_mRNA_log10_42_task1` is invalid.\n - Variable `p1_IRF2_mRNA_log10_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='IRF2_mRNA_log10']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF2_mRNA_log10_1_task1` is invalid.\n - Variable `p2_IRF2_mRNA_log10_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='IRF2_mRNA_log10']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF9_mRNA_log10_2_task1` is invalid.\n - Variable `p2_IRF9_mRNA_log10_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='IRF9_mRNA_log10']` are not defined. No namespaces are defined for the target.\n - Data generator `SOCS1mRNA_log10_3_task1` is invalid.\n - Variable `p2_SOCS1mRNA_log10_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='SOCS1mRNA_log10']` are not defined. No namespaces are defined for the target.\n - Data generator `SOCS3mRNA_log10_4_task1` is invalid.\n - Variable `p2_SOCS3mRNA_log10_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='SOCS3mRNA_log10']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT1mRNA_log10_5_task1` is invalid.\n - Variable `p2_STAT1mRNA_log10_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='STAT1mRNA_log10']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2mRNA_log10_6_task1` is invalid.\n - Variable `p2_STAT2mRNA_log10_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='STAT2mRNA_log10']` are not defined. No namespaces are defined for the target.\n - Data generator `USP18mRNA_log10_7_task1` is invalid.\n - Variable `p2_USP18mRNA_log10_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='USP18mRNA_log10']` are not defined. No namespaces are defined for the target.\n - Data generator `Rec_1_task2` is invalid.\n - Variable `p1_Rec_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Rec']` are not defined. No namespaces are defined for the target.\n - Data generator `SOCS1_2_task2` is invalid.\n - Variable `p1_SOCS1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SOCS1']` are not defined. No namespaces are defined for the target.\n - Data generator `IFN_3_task2` is invalid.\n - Variable `p1_IFN_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IFN']` are not defined. No namespaces are defined for the target.\n - Data generator `aRecIFN_4_task2` is invalid.\n - Variable `p1_aRecIFN_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='aRecIFN']` are not defined. No namespaces are defined for the target.\n - Data generator `USP18_5_task2` is invalid.\n - Variable `p1_USP18_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='USP18']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT1c_6_task2` is invalid.\n - Variable `p1_STAT1c_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT1c']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2c_7_task2` is invalid.\n - Variable `p1_STAT2c_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2c']` are not defined. No namespaces are defined for the target.\n - Data generator `pSTAT1pSTAT2c_8_task2` is invalid.\n - Variable `p1_pSTAT1pSTAT2c_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='pSTAT1pSTAT2c']` are not defined. No namespaces are defined for the target.\n - Data generator `pSTAT1pSTAT2n_9_task2` is invalid.\n - Variable `p1_pSTAT1pSTAT2n_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='pSTAT1pSTAT2n']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT1n_10_task2` is invalid.\n - Variable `p1_STAT1n_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT1n']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2n_11_task2` is invalid.\n - Variable `p1_STAT2n_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2n']` are not defined. No namespaces are defined for the target.\n - Data generator `ISGF3c_12_task2` is invalid.\n - Variable `p1_ISGF3c_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ISGF3c']` are not defined. No namespaces are defined for the target.\n - Data generator `ISGF3n_13_task2` is invalid.\n - Variable `p1_ISGF3n_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ISGF3n']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF9n_14_task2` is invalid.\n - Variable `p1_IRF9n_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IRF9n']` are not defined. No namespaces are defined for the target.\n - Data generator `pSTAT1dimc_15_task2` is invalid.\n - Variable `p1_pSTAT1dimc_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='pSTAT1dimc']` are not defined. No namespaces are defined for the target.\n - Data generator `pSTAT1dimn_16_task2` is invalid.\n - Variable `p1_pSTAT1dimn_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='pSTAT1dimn']` are not defined. No namespaces are defined for the target.\n - Data generator `OccGAS_ISREbs_17_task2` is invalid.\n - Variable `p1_OccGAS_ISREbs_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='OccGAS_ISREbs']` are not defined. No namespaces are defined for the target.\n - Data generator `OccGASbs_18_task2` is invalid.\n - Variable `p1_OccGASbs_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='OccGASbs']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF9c_19_task2` is invalid.\n - Variable `p1_IRF9c_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IRF9c']` are not defined. No namespaces are defined for the target.\n - Data generator `SOCS3_20_task2` is invalid.\n - Variable `p1_SOCS3_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SOCS3']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT1mRNA_21_task2` is invalid.\n - Variable `p1_STAT1mRNA_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT1mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2mRNA_22_task2` is invalid.\n - Variable `p1_STAT2mRNA_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF9mRNA_23_task2` is invalid.\n - Variable `p1_IRF9mRNA_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IRF9mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF2mRNA_24_task2` is invalid.\n - Variable `p1_IRF2mRNA_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IRF2mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF2_25_task2` is invalid.\n - Variable `p1_IRF2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IRF2']` are not defined. No namespaces are defined for the target.\n - Data generator `USP18mRNA_26_task2` is invalid.\n - Variable `p1_USP18mRNA_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='USP18mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `SOCS1mRNA_27_task2` is invalid.\n - Variable `p1_SOCS1mRNA_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SOCS1mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `SOCS3mRNA_28_task2` is invalid.\n - Variable `p1_SOCS3mRNA_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SOCS3mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT1_LC_1_29_task2` is invalid.\n - Variable `p1_STAT1_LC_1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT1_LC_1']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT1_LC_2_30_task2` is invalid.\n - Variable `p1_STAT1_LC_2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT1_LC_2']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT1_LC_3_31_task2` is invalid.\n - Variable `p1_STAT1_LC_3_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT1_LC_3']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2_LC_1_32_task2` is invalid.\n - Variable `p1_STAT2_LC_1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2_LC_1']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2_LC_2_33_task2` is invalid.\n - Variable `p1_STAT2_LC_2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2_LC_2']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2_LC_3_34_task2` is invalid.\n - Variable `p1_STAT2_LC_3_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2_LC_3']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2_LC_4_35_task2` is invalid.\n - Variable `p1_STAT2_LC_4_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2_LC_4']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2_LC_5_36_task2` is invalid.\n - Variable `p1_STAT2_LC_5_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2_LC_5']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF9_LC_1_37_task2` is invalid.\n - Variable `p1_IRF9_LC_1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IRF9_LC_1']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF9_LC_2_38_task2` is invalid.\n - Variable `p1_IRF9_LC_2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IRF9_LC_2']` are not defined. No namespaces are defined for the target.\n - Data generator `USP18_LC_1_39_task2` is invalid.\n - Variable `p1_USP18_LC_1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='USP18_LC_1']` are not defined. No namespaces are defined for the target.\n - Data generator `USP18_LC_2_40_task2` is invalid.\n - Variable `p1_USP18_LC_2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='USP18_LC_2']` are not defined. No namespaces are defined for the target.\n - Data generator `SOCS1_LC_1_41_task2` is invalid.\n - Variable `p1_SOCS1_LC_1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SOCS1_LC_1']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF2_mRNA_log10_42_task2` is invalid.\n - Variable `p1_IRF2_mRNA_log10_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='IRF2_mRNA_log10']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF2_mRNA_log10_1_task2` is invalid.\n - Variable `p2_IRF2_mRNA_log10_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='IRF2_mRNA_log10']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF9_mRNA_log10_2_task2` is invalid.\n - Variable `p2_IRF9_mRNA_log10_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='IRF9_mRNA_log10']` are not defined. No namespaces are defined for the target.\n - Data generator `SOCS1mRNA_log10_3_task2` is invalid.\n - Variable `p2_SOCS1mRNA_log10_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='SOCS1mRNA_log10']` are not defined. No namespaces are defined for the target.\n - Data generator `SOCS3mRNA_log10_4_task2` is invalid.\n - Variable `p2_SOCS3mRNA_log10_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='SOCS3mRNA_log10']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT1mRNA_log10_5_task2` is invalid.\n - Variable `p2_STAT1mRNA_log10_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='STAT1mRNA_log10']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2mRNA_log10_6_task2` is invalid.\n - Variable `p2_STAT2mRNA_log10_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='STAT2mRNA_log10']` are not defined. No namespaces are defined for the target.\n - Data generator `USP18mRNA_log10_7_task2` is invalid.\n - Variable `p2_USP18mRNA_log10_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='USP18mRNA_log10']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Kok2020.sedml583236.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Kok2020.xml` has warnings.\r\n - 82 warnings of type SBML unit consistency (99505). The following is the first warning at line 2609, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'log10(IRF2mRNA)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 87 warnings of type Modeling practice (80701). The following is the first warning at line 2502, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'IRF2_mRNA_log10' does not have a 'units' attribute.\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Kok2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Kok2020.xml` has warnings.\r\n - 82 warnings of type SBML unit consistency (99505). The following is the first warning at line 2609, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'log10(IRF2mRNA)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 87 warnings of type Modeling practice (80701). The following is the first warning at line 2502, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'IRF2_mRNA_log10' does not have a 'units' attribute.\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Kok2020.sedml583236.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Kok2020.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IFN']` are not defined. No namespaces are defined for the target.\n - Data generator `Rec_1_task1` is invalid.\n - Variable `p1_Rec_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Rec']` are not defined. No namespaces are defined for the target.\n - Data generator `SOCS1_2_task1` is invalid.\n - Variable `p1_SOCS1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SOCS1']` are not defined. No namespaces are defined for the target.\n - Data generator `IFN_3_task1` is invalid.\n - Variable `p1_IFN_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IFN']` are not defined. No namespaces are defined for the target.\n - Data generator `aRecIFN_4_task1` is invalid.\n - Variable `p1_aRecIFN_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='aRecIFN']` are not defined. No namespaces are defined for the target.\n - Data generator `USP18_5_task1` is invalid.\n - Variable `p1_USP18_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='USP18']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT1c_6_task1` is invalid.\n - Variable `p1_STAT1c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT1c']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2c_7_task1` is invalid.\n - Variable `p1_STAT2c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2c']` are not defined. No namespaces are defined for the target.\n - Data generator `pSTAT1pSTAT2c_8_task1` is invalid.\n - Variable `p1_pSTAT1pSTAT2c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='pSTAT1pSTAT2c']` are not defined. No namespaces are defined for the target.\n - Data generator `pSTAT1pSTAT2n_9_task1` is invalid.\n - Variable `p1_pSTAT1pSTAT2n_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='pSTAT1pSTAT2n']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT1n_10_task1` is invalid.\n - Variable `p1_STAT1n_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT1n']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2n_11_task1` is invalid.\n - Variable `p1_STAT2n_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2n']` are not defined. No namespaces are defined for the target.\n - Data generator `ISGF3c_12_task1` is invalid.\n - Variable `p1_ISGF3c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ISGF3c']` are not defined. No namespaces are defined for the target.\n - Data generator `ISGF3n_13_task1` is invalid.\n - Variable `p1_ISGF3n_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ISGF3n']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF9n_14_task1` is invalid.\n - Variable `p1_IRF9n_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IRF9n']` are not defined. No namespaces are defined for the target.\n - Data generator `pSTAT1dimc_15_task1` is invalid.\n - Variable `p1_pSTAT1dimc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='pSTAT1dimc']` are not defined. No namespaces are defined for the target.\n - Data generator `pSTAT1dimn_16_task1` is invalid.\n - Variable `p1_pSTAT1dimn_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='pSTAT1dimn']` are not defined. No namespaces are defined for the target.\n - Data generator `OccGAS_ISREbs_17_task1` is invalid.\n - Variable `p1_OccGAS_ISREbs_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='OccGAS_ISREbs']` are not defined. No namespaces are defined for the target.\n - Data generator `OccGASbs_18_task1` is invalid.\n - Variable `p1_OccGASbs_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='OccGASbs']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF9c_19_task1` is invalid.\n - Variable `p1_IRF9c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IRF9c']` are not defined. No namespaces are defined for the target.\n - Data generator `SOCS3_20_task1` is invalid.\n - Variable `p1_SOCS3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SOCS3']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT1mRNA_21_task1` is invalid.\n - Variable `p1_STAT1mRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT1mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2mRNA_22_task1` is invalid.\n - Variable `p1_STAT2mRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF9mRNA_23_task1` is invalid.\n - Variable `p1_IRF9mRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IRF9mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF2mRNA_24_task1` is invalid.\n - Variable `p1_IRF2mRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IRF2mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF2_25_task1` is invalid.\n - Variable `p1_IRF2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IRF2']` are not defined. No namespaces are defined for the target.\n - Data generator `USP18mRNA_26_task1` is invalid.\n - Variable `p1_USP18mRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='USP18mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `SOCS1mRNA_27_task1` is invalid.\n - Variable `p1_SOCS1mRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SOCS1mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `SOCS3mRNA_28_task1` is invalid.\n - Variable `p1_SOCS3mRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SOCS3mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT1_LC_1_29_task1` is invalid.\n - Variable `p1_STAT1_LC_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT1_LC_1']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT1_LC_2_30_task1` is invalid.\n - Variable `p1_STAT1_LC_2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT1_LC_2']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT1_LC_3_31_task1` is invalid.\n - Variable `p1_STAT1_LC_3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT1_LC_3']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2_LC_1_32_task1` is invalid.\n - Variable `p1_STAT2_LC_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2_LC_1']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2_LC_2_33_task1` is invalid.\n - Variable `p1_STAT2_LC_2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2_LC_2']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2_LC_3_34_task1` is invalid.\n - Variable `p1_STAT2_LC_3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2_LC_3']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2_LC_4_35_task1` is invalid.\n - Variable `p1_STAT2_LC_4_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2_LC_4']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2_LC_5_36_task1` is invalid.\n - Variable `p1_STAT2_LC_5_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2_LC_5']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF9_LC_1_37_task1` is invalid.\n - Variable `p1_IRF9_LC_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IRF9_LC_1']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF9_LC_2_38_task1` is invalid.\n - Variable `p1_IRF9_LC_2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IRF9_LC_2']` are not defined. No namespaces are defined for the target.\n - Data generator `USP18_LC_1_39_task1` is invalid.\n - Variable `p1_USP18_LC_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='USP18_LC_1']` are not defined. No namespaces are defined for the target.\n - Data generator `USP18_LC_2_40_task1` is invalid.\n - Variable `p1_USP18_LC_2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='USP18_LC_2']` are not defined. No namespaces are defined for the target.\n - Data generator `SOCS1_LC_1_41_task1` is invalid.\n - Variable `p1_SOCS1_LC_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SOCS1_LC_1']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF2_mRNA_log10_42_task1` is invalid.\n - Variable `p1_IRF2_mRNA_log10_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='IRF2_mRNA_log10']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF2_mRNA_log10_1_task1` is invalid.\n - Variable `p2_IRF2_mRNA_log10_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='IRF2_mRNA_log10']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF9_mRNA_log10_2_task1` is invalid.\n - Variable `p2_IRF9_mRNA_log10_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='IRF9_mRNA_log10']` are not defined. No namespaces are defined for the target.\n - Data generator `SOCS1mRNA_log10_3_task1` is invalid.\n - Variable `p2_SOCS1mRNA_log10_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='SOCS1mRNA_log10']` are not defined. No namespaces are defined for the target.\n - Data generator `SOCS3mRNA_log10_4_task1` is invalid.\n - Variable `p2_SOCS3mRNA_log10_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='SOCS3mRNA_log10']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT1mRNA_log10_5_task1` is invalid.\n - Variable `p2_STAT1mRNA_log10_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='STAT1mRNA_log10']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2mRNA_log10_6_task1` is invalid.\n - Variable `p2_STAT2mRNA_log10_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='STAT2mRNA_log10']` are not defined. No namespaces are defined for the target.\n - Data generator `USP18mRNA_log10_7_task1` is invalid.\n - Variable `p2_USP18mRNA_log10_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='USP18mRNA_log10']` are not defined. No namespaces are defined for the target.\n - Data generator `Rec_1_task2` is invalid.\n - Variable `p1_Rec_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Rec']` are not defined. No namespaces are defined for the target.\n - Data generator `SOCS1_2_task2` is invalid.\n - Variable `p1_SOCS1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SOCS1']` are not defined. No namespaces are defined for the target.\n - Data generator `IFN_3_task2` is invalid.\n - Variable `p1_IFN_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IFN']` are not defined. No namespaces are defined for the target.\n - Data generator `aRecIFN_4_task2` is invalid.\n - Variable `p1_aRecIFN_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='aRecIFN']` are not defined. No namespaces are defined for the target.\n - Data generator `USP18_5_task2` is invalid.\n - Variable `p1_USP18_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='USP18']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT1c_6_task2` is invalid.\n - Variable `p1_STAT1c_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT1c']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2c_7_task2` is invalid.\n - Variable `p1_STAT2c_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2c']` are not defined. No namespaces are defined for the target.\n - Data generator `pSTAT1pSTAT2c_8_task2` is invalid.\n - Variable `p1_pSTAT1pSTAT2c_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='pSTAT1pSTAT2c']` are not defined. No namespaces are defined for the target.\n - Data generator `pSTAT1pSTAT2n_9_task2` is invalid.\n - Variable `p1_pSTAT1pSTAT2n_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='pSTAT1pSTAT2n']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT1n_10_task2` is invalid.\n - Variable `p1_STAT1n_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT1n']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2n_11_task2` is invalid.\n - Variable `p1_STAT2n_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2n']` are not defined. No namespaces are defined for the target.\n - Data generator `ISGF3c_12_task2` is invalid.\n - Variable `p1_ISGF3c_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ISGF3c']` are not defined. No namespaces are defined for the target.\n - Data generator `ISGF3n_13_task2` is invalid.\n - Variable `p1_ISGF3n_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='ISGF3n']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF9n_14_task2` is invalid.\n - Variable `p1_IRF9n_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IRF9n']` are not defined. No namespaces are defined for the target.\n - Data generator `pSTAT1dimc_15_task2` is invalid.\n - Variable `p1_pSTAT1dimc_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='pSTAT1dimc']` are not defined. No namespaces are defined for the target.\n - Data generator `pSTAT1dimn_16_task2` is invalid.\n - Variable `p1_pSTAT1dimn_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='pSTAT1dimn']` are not defined. No namespaces are defined for the target.\n - Data generator `OccGAS_ISREbs_17_task2` is invalid.\n - Variable `p1_OccGAS_ISREbs_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='OccGAS_ISREbs']` are not defined. No namespaces are defined for the target.\n - Data generator `OccGASbs_18_task2` is invalid.\n - Variable `p1_OccGASbs_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='OccGASbs']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF9c_19_task2` is invalid.\n - Variable `p1_IRF9c_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IRF9c']` are not defined. No namespaces are defined for the target.\n - Data generator `SOCS3_20_task2` is invalid.\n - Variable `p1_SOCS3_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SOCS3']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT1mRNA_21_task2` is invalid.\n - Variable `p1_STAT1mRNA_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT1mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2mRNA_22_task2` is invalid.\n - Variable `p1_STAT2mRNA_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF9mRNA_23_task2` is invalid.\n - Variable `p1_IRF9mRNA_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IRF9mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF2mRNA_24_task2` is invalid.\n - Variable `p1_IRF2mRNA_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IRF2mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF2_25_task2` is invalid.\n - Variable `p1_IRF2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IRF2']` are not defined. No namespaces are defined for the target.\n - Data generator `USP18mRNA_26_task2` is invalid.\n - Variable `p1_USP18mRNA_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='USP18mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `SOCS1mRNA_27_task2` is invalid.\n - Variable `p1_SOCS1mRNA_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SOCS1mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `SOCS3mRNA_28_task2` is invalid.\n - Variable `p1_SOCS3mRNA_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SOCS3mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT1_LC_1_29_task2` is invalid.\n - Variable `p1_STAT1_LC_1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT1_LC_1']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT1_LC_2_30_task2` is invalid.\n - Variable `p1_STAT1_LC_2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT1_LC_2']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT1_LC_3_31_task2` is invalid.\n - Variable `p1_STAT1_LC_3_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT1_LC_3']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2_LC_1_32_task2` is invalid.\n - Variable `p1_STAT2_LC_1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2_LC_1']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2_LC_2_33_task2` is invalid.\n - Variable `p1_STAT2_LC_2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2_LC_2']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2_LC_3_34_task2` is invalid.\n - Variable `p1_STAT2_LC_3_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2_LC_3']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2_LC_4_35_task2` is invalid.\n - Variable `p1_STAT2_LC_4_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2_LC_4']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2_LC_5_36_task2` is invalid.\n - Variable `p1_STAT2_LC_5_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT2_LC_5']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF9_LC_1_37_task2` is invalid.\n - Variable `p1_IRF9_LC_1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IRF9_LC_1']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF9_LC_2_38_task2` is invalid.\n - Variable `p1_IRF9_LC_2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IRF9_LC_2']` are not defined. No namespaces are defined for the target.\n - Data generator `USP18_LC_1_39_task2` is invalid.\n - Variable `p1_USP18_LC_1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='USP18_LC_1']` are not defined. No namespaces are defined for the target.\n - Data generator `USP18_LC_2_40_task2` is invalid.\n - Variable `p1_USP18_LC_2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='USP18_LC_2']` are not defined. No namespaces are defined for the target.\n - Data generator `SOCS1_LC_1_41_task2` is invalid.\n - Variable `p1_SOCS1_LC_1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='SOCS1_LC_1']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF2_mRNA_log10_42_task2` is invalid.\n - Variable `p1_IRF2_mRNA_log10_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='IRF2_mRNA_log10']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF2_mRNA_log10_1_task2` is invalid.\n - Variable `p2_IRF2_mRNA_log10_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='IRF2_mRNA_log10']` are not defined. No namespaces are defined for the target.\n - Data generator `IRF9_mRNA_log10_2_task2` is invalid.\n - Variable `p2_IRF9_mRNA_log10_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='IRF9_mRNA_log10']` are not defined. No namespaces are defined for the target.\n - Data generator `SOCS1mRNA_log10_3_task2` is invalid.\n - Variable `p2_SOCS1mRNA_log10_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='SOCS1mRNA_log10']` are not defined. No namespaces are defined for the target.\n - Data generator `SOCS3mRNA_log10_4_task2` is invalid.\n - Variable `p2_SOCS3mRNA_log10_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='SOCS3mRNA_log10']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT1mRNA_log10_5_task2` is invalid.\n - Variable `p2_STAT1mRNA_log10_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='STAT1mRNA_log10']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT2mRNA_log10_6_task2` is invalid.\n - Variable `p2_STAT2mRNA_log10_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='STAT2mRNA_log10']` are not defined. No namespaces are defined for the target.\n - Data generator `USP18mRNA_log10_7_task2` is invalid.\n - Variable `p2_USP18mRNA_log10_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='USP18mRNA_log10']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000960](https://www.ebi.ac.uk/biomodels/BIOMD0000000960)<br/><sup>Paiva2020 - SEIAHRD model of t</sup></summary><sup>Paiva2020 - SEIAHRD model of transmission dynamics of COVID-19</sup></details>|<details><summary>pass</summary>[Paiva2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000960#Files)</details>|pass|<details><summary>pass</summary>[Paiva2020.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000960#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Paiva2020.sedml431783.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Paiva2020.xml` has warnings.\r\n - 132 warnings of type SBML unit consistency (99505). The following is the first warning at line 3083, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression '0.9 * Total_pop' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 210 warnings of type Modeling practice (80701). The following is the first warning at line 666, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id '_1_Trigger_China' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Paiva2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Paiva2020.xml` has warnings.\r\n - 132 warnings of type SBML unit consistency (99505). The following is the first warning at line 3083, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression '0.9 * Total_pop' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 210 warnings of type Modeling practice (80701). The following is the first warning at line 666, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id '_1_Trigger_China' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Paiva2020.sedml431783.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Paiva2020.sedml` is invalid.\n - Data generator `Deceased_1_task1` is invalid.\n - Variable `p1_Deceased_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Deceased']` are not defined. No namespaces are defined for the target.\n - Data generator `Cumulative_Cases_1_task1` is invalid.\n - Variable `p2_Cumulative_Cases_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cumulative_Cases']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Paiva2020.sedml570520.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Paiva2020.xml` has warnings.\r\n - 132 warnings of type SBML unit consistency (99505). The following is the first warning at line 3083, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression '0.9 * Total_pop' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 210 warnings of type Modeling practice (80701). The following is the first warning at line 666, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id '_1_Trigger_China' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Paiva2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Paiva2020.xml` has warnings.\r\n - 132 warnings of type SBML unit consistency (99505). The following is the first warning at line 3083, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression '0.9 * Total_pop' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 210 warnings of type Modeling practice (80701). The following is the first warning at line 666, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id '_1_Trigger_China' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Paiva2020.sedml570520.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Paiva2020.sedml` is invalid.\n - Data generator `Deceased_1_task1` is invalid.\n - Variable `p1_Deceased_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Deceased']` are not defined. No namespaces are defined for the target.\n - Data generator `Cumulative_Cases_1_task1` is invalid.\n - Variable `p2_Cumulative_Cases_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cumulative_Cases']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000961](https://www.ebi.ac.uk/biomodels/BIOMD0000000961)<br/><sup>McDougal2017 - Metabolism in i</sup></summary><sup>McDougal2017 - Metabolism in ischemic cardiomyocytes</sup></details>|<details><summary>FAIL</summary>[McDougal2017.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000961#Files)</details>|FAIL|<details><summary>pass</summary>[McDougal2017.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000961#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/McDougal2017.sedml846639.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 5050.0000001061\r\n - Number of steps: 8081\r\n - Task `task2` has warnings.\r\n - Child range `range1` is longer than its main range (3 > 1). The tail elements of the range will be ignored.\r\n - Child range `range1` is longer than its main range (3 > 1). The tail elements of the range will be ignored.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./McDougal2017.sedml` has warnings.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 5050.0000001061\r\n - Number of steps: 8081\r\n - Task `task2` has warnings.\r\n - Child range `range1` is longer than its main range (3 > 1). The tail elements of the range will be ignored.\r\n - Child range `range1` is longer than its main range (3 > 1). The tail elements of the range will be ignored.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/McDougal2017.sedml846639.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./McDougal2017.sedml` is invalid.\n - Model `model` is invalid.\n - The model file `McDougal2017.xml` is invalid.\n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout' cannot be used in this document.\n \n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout_L2' cannot be used in this document.\n \n - General SBML conformance (10102) at line 3, column 2: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L2V4 Section 4.1\n Element 'listOfLayouts' is not part of the definition of SBML Level 2 Version 4.\n \n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='level']` are not defined. No namespaces are defined for the target.\n - Change 2 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='lowO2']` are not defined. No namespaces are defined for the target.\n - Data generator `s31_1_task1` is invalid.\n - Variable `p1_s31_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s31']` are not defined. No namespaces are defined for the target.\n - Data generator `s35_2_task1` is invalid.\n - Variable `p1_s35_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s35']` are not defined. No namespaces are defined for the target.\n - Data generator `s30_3_task1` is invalid.\n - Variable `p1_s30_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s30']` are not defined. No namespaces are defined for the target.\n - Data generator `s33_4_task1` is invalid.\n - Variable `p1_s33_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s33']` are not defined. No namespaces are defined for the target.\n - Data generator `s36_5_task1` is invalid.\n - Variable `p1_s36_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s36']` are not defined. No namespaces are defined for the target.\n - Data generator `s31_1_task2` is invalid.\n - Variable `p1_s31_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s31']` are not defined. No namespaces are defined for the target.\n - Data generator `s35_2_task2` is invalid.\n - Variable `p1_s35_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s35']` are not defined. No namespaces are defined for the target.\n - Data generator `s30_3_task2` is invalid.\n - Variable `p1_s30_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s30']` are not defined. No namespaces are defined for the target.\n - Data generator `s33_4_task2` is invalid.\n - Variable `p1_s33_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s33']` are not defined. No namespaces are defined for the target.\n - Data generator `s36_5_task2` is invalid.\n - Variable `p1_s36_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s36']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/McDougal2017.sedml319459.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 5050.0000001061\r\n - Number of steps: 8081\r\n - Task `task2` has warnings.\r\n - Child range `range1` is longer than its main range (3 > 1). The tail elements of the range will be ignored.\r\n - Child range `range1` is longer than its main range (3 > 1). The tail elements of the range will be ignored.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./McDougal2017.sedml` has warnings.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 5050.0000001061\r\n - Number of steps: 8081\r\n - Task `task2` has warnings.\r\n - Child range `range1` is longer than its main range (3 > 1). The tail elements of the range will be ignored.\r\n - Child range `range1` is longer than its main range (3 > 1). The tail elements of the range will be ignored.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/McDougal2017.sedml319459.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./McDougal2017.sedml` is invalid.\n - Model `model` is invalid.\n - The model file `McDougal2017.xml` is invalid.\n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout' cannot be used in this document.\n \n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout_L2' cannot be used in this document.\n \n - General SBML conformance (10102) at line 3, column 2: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L2V4 Section 4.1\n Element 'listOfLayouts' is not part of the definition of SBML Level 2 Version 4.\n \n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='level']` are not defined. No namespaces are defined for the target.\n - Change 2 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='lowO2']` are not defined. No namespaces are defined for the target.\n - Data generator `s31_1_task1` is invalid.\n - Variable `p1_s31_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s31']` are not defined. No namespaces are defined for the target.\n - Data generator `s35_2_task1` is invalid.\n - Variable `p1_s35_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s35']` are not defined. No namespaces are defined for the target.\n - Data generator `s30_3_task1` is invalid.\n - Variable `p1_s30_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s30']` are not defined. No namespaces are defined for the target.\n - Data generator `s33_4_task1` is invalid.\n - Variable `p1_s33_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s33']` are not defined. No namespaces are defined for the target.\n - Data generator `s36_5_task1` is invalid.\n - Variable `p1_s36_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s36']` are not defined. No namespaces are defined for the target.\n - Data generator `s31_1_task2` is invalid.\n - Variable `p1_s31_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s31']` are not defined. No namespaces are defined for the target.\n - Data generator `s35_2_task2` is invalid.\n - Variable `p1_s35_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s35']` are not defined. No namespaces are defined for the target.\n - Data generator `s30_3_task2` is invalid.\n - Variable `p1_s30_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s30']` are not defined. No namespaces are defined for the target.\n - Data generator `s33_4_task2` is invalid.\n - Variable `p1_s33_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s33']` are not defined. No namespaces are defined for the target.\n - Data generator `s36_5_task2` is invalid.\n - Variable `p1_s36_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='s36']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000962](https://www.ebi.ac.uk/biomodels/BIOMD0000000962)<br/><sup>Zhao2020 - SUQC model of COVID</sup></summary><sup>Zhao2020 - SUQC model of COVID-19 transmission dynamics in Wuhan, Hubei, and China</sup></details>|<details><summary>pass</summary>[Zhao2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000962#Files)</details>|pass|<details><summary>FAIL</summary>[Zhao2020.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000962#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Zhao2020.sedml929422.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/Zhao2020.sedml929422.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Zhao2020.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmpe0dgo24l/./model1.xml` is not a file.\n - Data generator `Unquarantined_Infected_1_task1` is invalid.\n - Variable `p1_Unquarantined_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Unquarantined_Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Quarantined_Infected_2_task1` is invalid.\n - Variable `p1_Quarantined_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Quarantined_Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Confirmed_Infected_3_task1` is invalid.\n - Variable `p1_Confirmed_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Confirmed_Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Cumulative_Infected_4_task1` is invalid.\n - Variable `p1_Cumulative_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cumulative_Infected']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Zhao2020.sedml280050.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Zhao2020.sedml280050.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Zhao2020.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmp79zpb1dj/./model1.xml` is not a file.\n - Data generator `Unquarantined_Infected_1_task1` is invalid.\n - Variable `p1_Unquarantined_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Unquarantined_Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Quarantined_Infected_2_task1` is invalid.\n - Variable `p1_Quarantined_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Quarantined_Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Confirmed_Infected_3_task1` is invalid.\n - Variable `p1_Confirmed_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Confirmed_Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Cumulative_Infected_4_task1` is invalid.\n - Variable `p1_Cumulative_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cumulative_Infected']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000963](https://www.ebi.ac.uk/biomodels/BIOMD0000000963)<br/><sup>Weitz2020 - SIR model of COVID</sup></summary><sup>Weitz2020 - SIR model of COVID-19 transmission with shielding</sup></details>|<details><summary>pass</summary>[Weitz2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000963#Files)</details>|pass|<details><summary>pass</summary>[Weitz2020.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000963#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Weitz2020.sedml861299.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Weitz2020.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 465, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Country * Rate_Law_for_reaction(beta, Susceptible, Infected, alpha, Recovered)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 342, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha' does not have a 'units' attribute.\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Weitz2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Weitz2020.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 465, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Country * Rate_Law_for_reaction(beta, Susceptible, Infected, alpha, Recovered)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 342, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha' does not have a 'units' attribute.\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/Weitz2020.sedml861299.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Weitz2020.sedml` is invalid.\n - Each identified SED object must have a unique id. Multiple objects have the following ids:\n - p2_alpha_task1\n - p2_alpha_task2\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_1_task1` is invalid.\n - Variable `p1_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Uninfected_1_task1` is invalid.\n - Variable `p2_Uninfected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Uninfected']` are not defined. No namespaces are defined for the target.\n - Data generator `alpha_1_task1` is invalid.\n - Variable `p2_alpha_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n - Data generator `Epidemic_size_2_task1` is invalid.\n - Variable `p2_Epidemic_size_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Epidemic_size']` are not defined. No namespaces are defined for the target.\n - Data generator `alpha_2_task1` is invalid.\n - Variable `p2_alpha_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_1_task2` is invalid.\n - Variable `p1_Infected_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Uninfected_1_task2` is invalid.\n - Variable `p2_Uninfected_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Uninfected']` are not defined. No namespaces are defined for the target.\n - Data generator `alpha_1_task2` is invalid.\n - Variable `p2_alpha_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n - Data generator `Epidemic_size_2_task2` is invalid.\n - Variable `p2_Epidemic_size_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Epidemic_size']` are not defined. No namespaces are defined for the target.\n - Data generator `alpha_2_task2` is invalid.\n - Variable `p2_alpha_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Weitz2020.sedml575337.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Weitz2020.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 465, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Country * Rate_Law_for_reaction(beta, Susceptible, Infected, alpha, Recovered)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 342, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha' does not have a 'units' attribute.\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Weitz2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Weitz2020.xml` has warnings.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 465, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Country * Rate_Law_for_reaction(beta, Susceptible, Infected, alpha, Recovered)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 342, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha' does not have a 'units' attribute.\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Weitz2020.sedml575337.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Weitz2020.sedml` is invalid.\n - Each identified SED object must have a unique id. Multiple objects have the following ids:\n - p2_alpha_task1\n - p2_alpha_task2\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_1_task1` is invalid.\n - Variable `p1_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Uninfected_1_task1` is invalid.\n - Variable `p2_Uninfected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Uninfected']` are not defined. No namespaces are defined for the target.\n - Data generator `alpha_1_task1` is invalid.\n - Variable `p2_alpha_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n - Data generator `Epidemic_size_2_task1` is invalid.\n - Variable `p2_Epidemic_size_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Epidemic_size']` are not defined. No namespaces are defined for the target.\n - Data generator `alpha_2_task1` is invalid.\n - Variable `p2_alpha_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_1_task2` is invalid.\n - Variable `p1_Infected_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Uninfected_1_task2` is invalid.\n - Variable `p2_Uninfected_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Uninfected']` are not defined. No namespaces are defined for the target.\n - Data generator `alpha_1_task2` is invalid.\n - Variable `p2_alpha_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n - Data generator `Epidemic_size_2_task2` is invalid.\n - Variable `p2_Epidemic_size_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Epidemic_size']` are not defined. No namespaces are defined for the target.\n - Data generator `alpha_2_task2` is invalid.\n - Variable `p2_alpha_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000964](https://www.ebi.ac.uk/biomodels/BIOMD0000000964)<br/><sup>Mwalili2020 - SEIR model of CO</sup></summary><sup>Mwalili2020 - SEIR model of COVID-19 transmission and environmental pathogen prevalence</sup></details>|<details><summary>pass</summary>[Mwalili2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000964#Files)</details>|FAIL|<details><summary>pass</summary>[Mwalili2020.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000964#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Mwalili2020.sedml163225.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Mwalili2020.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (10503). The following is the first warning at line 1019, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'Human * psi * Exposed' in the <kineticLaw> element of the <reaction> with id 'Exposed_to_Susceptible' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 1006, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Birth_Rate') are litre (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0), item (exponent = 1, multiplier = 1, scale = 0).\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 1125, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Rate_Law_for_reaction_2(beta_1, Susceptible, Pathogen_0, alpha_1, beta_2, Infected_Asymptomatic, Infected_Symptomatic, alpha_2)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Mwalili2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Mwalili2020.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (10503). The following is the first warning at line 1019, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'Human * psi * Exposed' in the <kineticLaw> element of the <reaction> with id 'Exposed_to_Susceptible' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 1006, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Birth_Rate') are litre (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0), item (exponent = 1, multiplier = 1, scale = 0).\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 1125, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Rate_Law_for_reaction_2(beta_1, Susceptible, Pathogen_0, alpha_1, beta_2, Infected_Asymptomatic, Infected_Symptomatic, alpha_2)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/Mwalili2020.sedml163225.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Mwalili2020.sedml` is invalid.\n - Data generator `Pathogen_0_1_task1` is invalid.\n - Variable `p1_Pathogen_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Pathogen_0']` are not defined. No namespaces are defined for the target.\n - Data generator `Susceptible_1_task1` is invalid.\n - Variable `p2_Susceptible_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Susceptible']` are not defined. No namespaces are defined for the target.\n - Data generator `Exposed_2_task1` is invalid.\n - Variable `p2_Exposed_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Exposed']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_Symptomatic_3_task1` is invalid.\n - Variable `p2_Infected_Symptomatic_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected_Symptomatic']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_Asymptomatic_4_task1` is invalid.\n - Variable `p2_Infected_Asymptomatic_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected_Asymptomatic']` are not defined. No namespaces are defined for the target.\n - Data generator `Recovered_5_task1` is invalid.\n - Variable `p2_Recovered_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Recovered']` are not defined. No namespaces are defined for the target.\n - Data generator `Pathogen_0_1_task2` is invalid.\n - Variable `p1_Pathogen_0_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Pathogen_0']` are not defined. No namespaces are defined for the target.\n - Data generator `Susceptible_1_task2` is invalid.\n - Variable `p2_Susceptible_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Susceptible']` are not defined. No namespaces are defined for the target.\n - Data generator `Exposed_2_task2` is invalid.\n - Variable `p2_Exposed_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Exposed']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_Symptomatic_3_task2` is invalid.\n - Variable `p2_Infected_Symptomatic_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected_Symptomatic']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_Asymptomatic_4_task2` is invalid.\n - Variable `p2_Infected_Asymptomatic_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected_Asymptomatic']` are not defined. No namespaces are defined for the target.\n - Data generator `Recovered_5_task2` is invalid.\n - Variable `p2_Recovered_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Recovered']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Mwalili2020.sedml488498.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Mwalili2020.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (10503). The following is the first warning at line 1019, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'Human * psi * Exposed' in the <kineticLaw> element of the <reaction> with id 'Exposed_to_Susceptible' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 1006, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Birth_Rate') are litre (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0), item (exponent = 1, multiplier = 1, scale = 0).\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 1125, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Rate_Law_for_reaction_2(beta_1, Susceptible, Pathogen_0, alpha_1, beta_2, Infected_Asymptomatic, Infected_Symptomatic, alpha_2)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Mwalili2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Mwalili2020.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (10503). The following is the first warning at line 1019, column 6:\r\n - The unit of measurement associated with the mathematical formula in the MathML math element of every KineticLaw object in a model should be identical to all KineticLaw objects in the model.\r\n Reference: L3V1 Section 3.4\r\n The formula 'Human * psi * Exposed' in the <kineticLaw> element of the <reaction> with id 'Exposed_to_Susceptible' produces units that are inconsistent with units of earlier KineticLaw elements.\r\n - 2 warnings of type SBML unit consistency (10541). The following is the first warning at line 1006, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L3V1 Sections 4.11.7, 4.2.4 AND 4.9.4\r\n In level 3 the expected units are extent_per_time. Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Birth_Rate') are litre (exponent = 1, multiplier = 1, scale = -3), second (exponent = -1, multiplier = 86400, scale = 0), item (exponent = 1, multiplier = 1, scale = 0).\r\n - 2 warnings of type SBML unit consistency (99505). The following is the first warning at line 1125, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Rate_Law_for_reaction_2(beta_1, Susceptible, Pathogen_0, alpha_1, beta_2, Infected_Asymptomatic, Infected_Symptomatic, alpha_2)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Mwalili2020.sedml488498.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Mwalili2020.sedml` is invalid.\n - Data generator `Pathogen_0_1_task1` is invalid.\n - Variable `p1_Pathogen_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Pathogen_0']` are not defined. No namespaces are defined for the target.\n - Data generator `Susceptible_1_task1` is invalid.\n - Variable `p2_Susceptible_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Susceptible']` are not defined. No namespaces are defined for the target.\n - Data generator `Exposed_2_task1` is invalid.\n - Variable `p2_Exposed_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Exposed']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_Symptomatic_3_task1` is invalid.\n - Variable `p2_Infected_Symptomatic_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected_Symptomatic']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_Asymptomatic_4_task1` is invalid.\n - Variable `p2_Infected_Asymptomatic_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected_Asymptomatic']` are not defined. No namespaces are defined for the target.\n - Data generator `Recovered_5_task1` is invalid.\n - Variable `p2_Recovered_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Recovered']` are not defined. No namespaces are defined for the target.\n - Data generator `Pathogen_0_1_task2` is invalid.\n - Variable `p1_Pathogen_0_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Pathogen_0']` are not defined. No namespaces are defined for the target.\n - Data generator `Susceptible_1_task2` is invalid.\n - Variable `p2_Susceptible_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Susceptible']` are not defined. No namespaces are defined for the target.\n - Data generator `Exposed_2_task2` is invalid.\n - Variable `p2_Exposed_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Exposed']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_Symptomatic_3_task2` is invalid.\n - Variable `p2_Infected_Symptomatic_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected_Symptomatic']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_Asymptomatic_4_task2` is invalid.\n - Variable `p2_Infected_Asymptomatic_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected_Asymptomatic']` are not defined. No namespaces are defined for the target.\n - Data generator `Recovered_5_task2` is invalid.\n - Variable `p2_Recovered_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Recovered']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000965](https://www.ebi.ac.uk/biomodels/BIOMD0000000965)<br/><sup>LeBeau1999 - IP3-dependent int</sup></summary><sup>LeBeau1999 - IP3-dependent intracellular calcium oscillations due to agonist stimulation from Cholecytokinin</sup></details>|<details><summary>FAIL</summary>[LeBeau1999_Phosphorylation_CCK.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000965#Files)</details>|FAIL|<details><summary>pass</summary>[LeBeau1999_Phosphorylation_CCK.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000965#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/LeBeau1999_Phosphorylation_CCK.sedml577813.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/LeBeau1999_Phosphorylation_CCK.sedml577813.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./LeBeau1999_Phosphorylation_CCK.sedml` is invalid.\n - Model `model` is invalid.\n - The model file `LeBeau1999_Phosphorylation_CCK.xml` is invalid.\n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout' cannot be used in this document.\n \n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout_L2' cannot be used in this document.\n \n - General SBML conformance (10102) at line 3, column 2: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L2V4 Section 4.1\n Element 'listOfLayouts' is not part of the definition of SBML Level 2 Version 4.\n \n - Data generator `c_1_task1` is invalid.\n - Variable `p1_c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='c']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/LeBeau1999_Phosphorylation_CCK.sedml308612.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/LeBeau1999_Phosphorylation_CCK.sedml308612.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./LeBeau1999_Phosphorylation_CCK.sedml` is invalid.\n - Model `model` is invalid.\n - The model file `LeBeau1999_Phosphorylation_CCK.xml` is invalid.\n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout' cannot be used in this document.\n \n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout_L2' cannot be used in this document.\n \n - General SBML conformance (10102) at line 3, column 2: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L2V4 Section 4.1\n Element 'listOfLayouts' is not part of the definition of SBML Level 2 Version 4.\n \n - Data generator `c_1_task1` is invalid.\n - Variable `p1_c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='c']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000966](https://www.ebi.ac.uk/biomodels/BIOMD0000000966)<br/><sup>Cui2008 - in vitro transcripti</sup></summary><sup>Cui2008 - in vitro transcriptional response of zinc homeostasis system in Escherichia coli</sup></details>|<details><summary>FAIL</summary>[Cui2008.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000966#Files)</details>|FAIL|<details><summary>pass</summary>[Cui2008.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000966#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Cui2008.sedml504182.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/Cui2008.sedml504182.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Cui2008.sedml` is invalid.\n - Model `model` is invalid.\n - The model file `Cui2008.xml` is invalid.\n - General SBML conformance (10102) at line 214, column 6: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L3V1 Section 4.1\n Element 'globalRenderInformation' is not part of the definition of 'listOfGlobalRenderInformation' in SBML Level 3 Version 1 Package render Version 1.\n \n - Data generator `Py_1_task1` is invalid.\n - Variable `p1_Py_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Py']` are not defined. No namespaces are defined for the target.\n - Data generator `Py1_2_task1` is invalid.\n - Variable `p1_Py1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Py1']` are not defined. No namespaces are defined for the target.\n - Data generator `Rw_3_task1` is invalid.\n - Variable `p1_Rw_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Rw']` are not defined. No namespaces are defined for the target.\n - Data generator `Dw_1_task1` is invalid.\n - Variable `p2_Dw_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Dw']` are not defined. No namespaces are defined for the target.\n - Data generator `Qw1_2_task1` is invalid.\n - Variable `p2_Qw1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Qw1']` are not defined. No namespaces are defined for the target.\n - Data generator `Qw2_3_task1` is invalid.\n - Variable `p2_Qw2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Qw2']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Cui2008.sedml403216.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Cui2008.sedml403216.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Cui2008.sedml` is invalid.\n - Model `model` is invalid.\n - The model file `Cui2008.xml` is invalid.\n - General SBML conformance (10102) at line 214, column 6: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L3V1 Section 4.1\n Element 'globalRenderInformation' is not part of the definition of 'listOfGlobalRenderInformation' in SBML Level 3 Version 1 Package render Version 1.\n \n - Data generator `Py_1_task1` is invalid.\n - Variable `p1_Py_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Py']` are not defined. No namespaces are defined for the target.\n - Data generator `Py1_2_task1` is invalid.\n - Variable `p1_Py1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Py1']` are not defined. No namespaces are defined for the target.\n - Data generator `Rw_3_task1` is invalid.\n - Variable `p1_Rw_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Rw']` are not defined. No namespaces are defined for the target.\n - Data generator `Dw_1_task1` is invalid.\n - Variable `p2_Dw_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Dw']` are not defined. No namespaces are defined for the target.\n - Data generator `Qw1_2_task1` is invalid.\n - Variable `p2_Qw1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Qw1']` are not defined. No namespaces are defined for the target.\n - Data generator `Qw2_3_task1` is invalid.\n - Variable `p2_Qw2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Qw2']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000967](https://www.ebi.ac.uk/biomodels/BIOMD0000000967)<br/><sup>McLean1991 - Behaviour of HIV </sup></summary><sup>McLean1991 - Behaviour of HIV in the presence of zidovudine</sup></details>|<details><summary>FAIL</summary>[McLean1991.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000967#Files)</details>|FAIL|<details><summary>pass</summary>[McLean1991.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000967#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/McLean1991.sedml519924.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 50.0000000031\r\n - Number of steps: 63\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./McLean1991.sedml` has warnings.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 50.0000000031\r\n - Number of steps: 63\x1b[0m\r\n\r\n`/root/in/McLean1991.sedml519924.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./McLean1991.sedml` is invalid.\n - Model `model` is invalid.\n - The model file `McLean1991.xml` is invalid.\n - General SBML conformance (10102) at line 211, column 6: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L3V1 Section 4.1\n Element 'globalRenderInformation' is not part of the definition of 'listOfGlobalRenderInformation' in SBML Level 3 Version 1 Package render Version 1.\n \n - Data generator `R_1_task1` is invalid.\n - Variable `p1_R_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='R']` are not defined. No namespaces are defined for the target.\n - Data generator `L_1_task1` is invalid.\n - Variable `p2_L_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L']` are not defined. No namespaces are defined for the target.\n - Data generator `V_2_task1` is invalid.\n - Variable `p2_V_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/McLean1991.sedml178589.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 50.0000000031\r\n - Number of steps: 63\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./McLean1991.sedml` has warnings.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 50.0000000031\r\n - Number of steps: 63\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/McLean1991.sedml178589.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./McLean1991.sedml` is invalid.\n - Model `model` is invalid.\n - The model file `McLean1991.xml` is invalid.\n - General SBML conformance (10102) at line 211, column 6: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L3V1 Section 4.1\n Element 'globalRenderInformation' is not part of the definition of 'listOfGlobalRenderInformation' in SBML Level 3 Version 1 Package render Version 1.\n \n - Data generator `R_1_task1` is invalid.\n - Variable `p1_R_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='R']` are not defined. No namespaces are defined for the target.\n - Data generator `L_1_task1` is invalid.\n - Variable `p2_L_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='L']` are not defined. No namespaces are defined for the target.\n - Data generator `V_2_task1` is invalid.\n - Variable `p2_V_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='V']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000968](https://www.ebi.ac.uk/biomodels/BIOMD0000000968)<br/><sup>Palmer2008 - Negative Feedback</sup></summary><sup>Palmer2008 - Negative Feedback in IL-7 mediated Jak-Stat signaling</sup></details>|<details><summary>pass</summary>[Palmer2008.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000968#Files)</details>|pass|<details><summary>pass</summary>[Palmer2008.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000968#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Palmer2008.sedml112198.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Palmer2008.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (99508). The following is the first warning at line 736, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'Kf1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type SBML unit consistency (20702). The following is the first warning at line 736, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'Kf1' does not have a 'units' attribute.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 955, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'cell * (Kf1 * IL7 * IL7RJAK1 - Kr1 * IL7IL7RJAK1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 736, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Kf1' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Palmer2008.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Palmer2008.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (99508). The following is the first warning at line 736, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'Kf1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type SBML unit consistency (20702). The following is the first warning at line 736, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'Kf1' does not have a 'units' attribute.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 955, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'cell * (Kf1 * IL7 * IL7RJAK1 - Kr1 * IL7IL7RJAK1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 736, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Kf1' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/Palmer2008.sedml112198.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Palmer2008.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Kf5']` are not defined. No namespaces are defined for the target.\n - Data generator `RatiopSTAT5_1_task1` is invalid.\n - Variable `p1_RatiopSTAT5_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='RatiopSTAT5']` are not defined. No namespaces are defined for the target.\n - Data generator `RatiopSTAT5_1_task2` is invalid.\n - Variable `p1_RatiopSTAT5_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='RatiopSTAT5']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Palmer2008.sedml525921.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Palmer2008.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (99508). The following is the first warning at line 736, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'Kf1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type SBML unit consistency (20702). The following is the first warning at line 736, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'Kf1' does not have a 'units' attribute.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 955, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'cell * (Kf1 * IL7 * IL7RJAK1 - Kr1 * IL7IL7RJAK1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 736, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Kf1' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Palmer2008.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Palmer2008.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (99508). The following is the first warning at line 736, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'Kf1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type SBML unit consistency (20702). The following is the first warning at line 736, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'Kf1' does not have a 'units' attribute.\r\n - 7 warnings of type SBML unit consistency (99505). The following is the first warning at line 955, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'cell * (Kf1 * IL7 * IL7RJAK1 - Kr1 * IL7IL7RJAK1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 736, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Kf1' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Palmer2008.sedml525921.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Palmer2008.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Kf5']` are not defined. No namespaces are defined for the target.\n - Data generator `RatiopSTAT5_1_task1` is invalid.\n - Variable `p1_RatiopSTAT5_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='RatiopSTAT5']` are not defined. No namespaces are defined for the target.\n - Data generator `RatiopSTAT5_1_task2` is invalid.\n - Variable `p1_RatiopSTAT5_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='RatiopSTAT5']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000969](https://www.ebi.ac.uk/biomodels/BIOMD0000000969)<br/><sup>Cuadros2020 - SIHRD spatiotemp</sup></summary><sup>Cuadros2020 - SIHRD spatiotemporal model of COVID-19 transmission in Ohio</sup></details>|<details><summary>pass</summary>[Cuadros2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000969#Files)</details>|pass|<details><summary>pass</summary>[Cuadros2020.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000969#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Cuadros2020.sedml729699.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Cuadros2020.xml` has warnings.\r\n - 64 warnings of type SBML unit consistency (99508). The following is the first warning at line 1551, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'lambda_Counties_with_airports' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 64 warnings of type SBML unit consistency (20702). The following is the first warning at line 1551, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'lambda_Counties_with_airports' does not have a 'units' attribute.\r\n - 40 warnings of type SBML unit consistency (99505). The following is the first warning at line 2375, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'Metabolite_3 + Metabolite_2 + Metabolite_1 + Metabolite_4 + Metabolite_5 + Metabolite_6 + Metabolite_0' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 64 warnings of type Modeling practice (80701). The following is the first warning at line 1551, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lambda_Counties_with_airports' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Cuadros2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Cuadros2020.xml` has warnings.\r\n - 64 warnings of type SBML unit consistency (99508). The following is the first warning at line 1551, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'lambda_Counties_with_airports' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 64 warnings of type SBML unit consistency (20702). The following is the first warning at line 1551, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'lambda_Counties_with_airports' does not have a 'units' attribute.\r\n - 40 warnings of type SBML unit consistency (99505). The following is the first warning at line 2375, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'Metabolite_3 + Metabolite_2 + Metabolite_1 + Metabolite_4 + Metabolite_5 + Metabolite_6 + Metabolite_0' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 64 warnings of type Modeling practice (80701). The following is the first warning at line 1551, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lambda_Counties_with_airports' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Cuadros2020.sedml729699.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Cuadros2020.sedml` is invalid.\n - Data generator `Total_cumulative_cases_1_task1` is invalid.\n - Variable `p1_Total_cumulative_cases_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_cumulative_cases']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_deaths_1_task1` is invalid.\n - Variable `p2_Total_deaths_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_deaths']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Cuadros2020.sedml712243.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Cuadros2020.xml` has warnings.\r\n - 64 warnings of type SBML unit consistency (99508). The following is the first warning at line 1551, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'lambda_Counties_with_airports' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 64 warnings of type SBML unit consistency (20702). The following is the first warning at line 1551, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'lambda_Counties_with_airports' does not have a 'units' attribute.\r\n - 40 warnings of type SBML unit consistency (99505). The following is the first warning at line 2375, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'Metabolite_3 + Metabolite_2 + Metabolite_1 + Metabolite_4 + Metabolite_5 + Metabolite_6 + Metabolite_0' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 64 warnings of type Modeling practice (80701). The following is the first warning at line 1551, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lambda_Counties_with_airports' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Cuadros2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Cuadros2020.xml` has warnings.\r\n - 64 warnings of type SBML unit consistency (99508). The following is the first warning at line 1551, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'lambda_Counties_with_airports' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 64 warnings of type SBML unit consistency (20702). The following is the first warning at line 1551, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'lambda_Counties_with_airports' does not have a 'units' attribute.\r\n - 40 warnings of type SBML unit consistency (99505). The following is the first warning at line 2375, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'Metabolite_3 + Metabolite_2 + Metabolite_1 + Metabolite_4 + Metabolite_5 + Metabolite_6 + Metabolite_0' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 64 warnings of type Modeling practice (80701). The following is the first warning at line 1551, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lambda_Counties_with_airports' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Cuadros2020.sedml712243.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Cuadros2020.sedml` is invalid.\n - Data generator `Total_cumulative_cases_1_task1` is invalid.\n - Variable `p1_Total_cumulative_cases_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_cumulative_cases']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_deaths_1_task1` is invalid.\n - Variable `p2_Total_deaths_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_deaths']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000970](https://www.ebi.ac.uk/biomodels/BIOMD0000000970)<br/><sup>Hou2020 - SEIR model of COVID-</sup></summary><sup>Hou2020 - SEIR model of COVID-19 transmission in Wuhan</sup></details>|<details><summary>pass</summary>[Hou2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000970#Files)</details>|pass|<details><summary>pass</summary>[Hou2020.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000970#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Hou2020.sedml138702.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Hou2020.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99508). The following is the first warning at line 399, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'r_1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type SBML unit consistency (20702). The following is the first warning at line 399, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'r_1' does not have a 'units' attribute.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 554, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Wuhan * Rate_Law_for_reaction(r_1, beta_1, Infected, Susceptible, r_2, beta_2, Exposed, Total_Population)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 399, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r_1' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 183.0\r\n - Number of steps: 183\r\n - Output `plot_5_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_6_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_7_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_8_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Hou2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Hou2020.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99508). The following is the first warning at line 399, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'r_1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type SBML unit consistency (20702). The following is the first warning at line 399, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'r_1' does not have a 'units' attribute.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 554, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Wuhan * Rate_Law_for_reaction(r_1, beta_1, Infected, Susceptible, r_2, beta_2, Exposed, Total_Population)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 399, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r_1' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 183.0\r\n - Number of steps: 183\r\n - Output `plot_5_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_6_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_7_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_8_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/Hou2020.sedml138702.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Hou2020.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='r_2']` are not defined. No namespaces are defined for the target.\n - Data generator `Susceptible_1_task1` is invalid.\n - Variable `p1_Susceptible_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Susceptible']` are not defined. No namespaces are defined for the target.\n - Data generator `Exposed_1_task1` is invalid.\n - Variable `p2_Exposed_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Exposed']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_1_task1` is invalid.\n - Variable `p3_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Recovered_1_task1` is invalid.\n - Variable `p4_Recovered_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Recovered']` are not defined. No namespaces are defined for the target.\n - Data generator `Susceptible_1_task2` is invalid.\n - Variable `p1_Susceptible_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Susceptible']` are not defined. No namespaces are defined for the target.\n - Data generator `Exposed_1_task2` is invalid.\n - Variable `p2_Exposed_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Exposed']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_1_task2` is invalid.\n - Variable `p3_Infected_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Recovered_1_task2` is invalid.\n - Variable `p4_Recovered_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Recovered']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Hou2020.sedml525794.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Hou2020.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99508). The following is the first warning at line 399, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'r_1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type SBML unit consistency (20702). The following is the first warning at line 399, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'r_1' does not have a 'units' attribute.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 554, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Wuhan * Rate_Law_for_reaction(r_1, beta_1, Infected, Susceptible, r_2, beta_2, Exposed, Total_Population)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 399, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r_1' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 183.0\r\n - Number of steps: 183\r\n - Output `plot_5_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_6_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_7_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_8_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Hou2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Hou2020.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99508). The following is the first warning at line 399, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'r_1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type SBML unit consistency (20702). The following is the first warning at line 399, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'r_1' does not have a 'units' attribute.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 554, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Wuhan * Rate_Law_for_reaction(r_1, beta_1, Infected, Susceptible, r_2, beta_2, Exposed, Total_Population)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 399, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r_1' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 183.0\r\n - Number of steps: 183\r\n - Output `plot_5_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_6_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_7_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_8_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Hou2020.sedml525794.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Hou2020.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='r_2']` are not defined. No namespaces are defined for the target.\n - Data generator `Susceptible_1_task1` is invalid.\n - Variable `p1_Susceptible_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Susceptible']` are not defined. No namespaces are defined for the target.\n - Data generator `Exposed_1_task1` is invalid.\n - Variable `p2_Exposed_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Exposed']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_1_task1` is invalid.\n - Variable `p3_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Recovered_1_task1` is invalid.\n - Variable `p4_Recovered_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Recovered']` are not defined. No namespaces are defined for the target.\n - Data generator `Susceptible_1_task2` is invalid.\n - Variable `p1_Susceptible_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Susceptible']` are not defined. No namespaces are defined for the target.\n - Data generator `Exposed_1_task2` is invalid.\n - Variable `p2_Exposed_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Exposed']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_1_task2` is invalid.\n - Variable `p3_Infected_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Recovered_1_task2` is invalid.\n - Variable `p4_Recovered_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Recovered']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000971](https://www.ebi.ac.uk/biomodels/BIOMD0000000971)<br/><sup>Tang2020 - Estimation of trans</sup></summary><sup>Tang2020 - Estimation of transmission risk of COVID-19 and impact of public health interventions</sup></details>|<details><summary>pass</summary>[Tang2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000971#Files)</details>|pass|<details><summary>pass</summary>[Tang2020.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000971#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Tang2020.sedml200197.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Tang2020.xml` has warnings.\r\n - 16 warnings of type SBML unit consistency (99508). The following is the first warning at line 729, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'c' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 16 warnings of type SBML unit consistency (20702). The following is the first warning at line 729, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'c' does not have a 'units' attribute.\r\n - 17 warnings of type SBML unit consistency (99505). The following is the first warning at line 1026, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Wuhan * Rate_Law_for_reaction(beta, c, Susceptible, Infected, theta, Asymptomatic, q)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 16 warnings of type Modeling practice (80701). The following is the first warning at line 729, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'c' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Tang2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Tang2020.xml` has warnings.\r\n - 16 warnings of type SBML unit consistency (99508). The following is the first warning at line 729, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'c' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 16 warnings of type SBML unit consistency (20702). The following is the first warning at line 729, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'c' does not have a 'units' attribute.\r\n - 17 warnings of type SBML unit consistency (99505). The following is the first warning at line 1026, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Wuhan * Rate_Law_for_reaction(beta, c, Susceptible, Infected, theta, Asymptomatic, q)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 16 warnings of type Modeling practice (80701). The following is the first warning at line 729, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'c' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/Tang2020.sedml200197.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Tang2020.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='q_lockdown']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_1_task1` is invalid.\n - Variable `p1_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_1_task2` is invalid.\n - Variable `p1_Infected_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Tang2020.sedml417997.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Tang2020.xml` has warnings.\r\n - 16 warnings of type SBML unit consistency (99508). The following is the first warning at line 729, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'c' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 16 warnings of type SBML unit consistency (20702). The following is the first warning at line 729, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'c' does not have a 'units' attribute.\r\n - 17 warnings of type SBML unit consistency (99505). The following is the first warning at line 1026, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Wuhan * Rate_Law_for_reaction(beta, c, Susceptible, Infected, theta, Asymptomatic, q)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 16 warnings of type Modeling practice (80701). The following is the first warning at line 729, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'c' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Tang2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Tang2020.xml` has warnings.\r\n - 16 warnings of type SBML unit consistency (99508). The following is the first warning at line 729, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'c' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 16 warnings of type SBML unit consistency (20702). The following is the first warning at line 729, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'c' does not have a 'units' attribute.\r\n - 17 warnings of type SBML unit consistency (99505). The following is the first warning at line 1026, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Wuhan * Rate_Law_for_reaction(beta, c, Susceptible, Infected, theta, Asymptomatic, q)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 16 warnings of type Modeling practice (80701). The following is the first warning at line 729, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'c' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Tang2020.sedml417997.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Tang2020.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='q_lockdown']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_1_task1` is invalid.\n - Variable `p1_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_1_task2` is invalid.\n - Variable `p1_Infected_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000972](https://www.ebi.ac.uk/biomodels/BIOMD0000000972)<br/><sup>Tang2020 - Estimation of trans</sup></summary><sup>Tang2020 - Estimation of transmission risk of COVID-19 and impact of public health interventions - update</sup></details>|<details><summary>pass</summary>[Tang2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000972#Files)</details>|pass|<details><summary>pass</summary>[Tang2020.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000972#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Tang2020.sedml763034.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Tang2020.xml` has warnings.\r\n - 22 warnings of type SBML unit consistency (99508). The following is the first warning at line 741, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'c' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 22 warnings of type SBML unit consistency (20702). The following is the first warning at line 741, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'c' does not have a 'units' attribute.\r\n - 15 warnings of type SBML unit consistency (99505). The following is the first warning at line 1073, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '(c_0 - c_b) * exp(-r_1 * time) + c_b' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 22 warnings of type Modeling practice (80701). The following is the first warning at line 741, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'c' does not have a 'units' attribute.\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Tang2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Tang2020.xml` has warnings.\r\n - 22 warnings of type SBML unit consistency (99508). The following is the first warning at line 741, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'c' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 22 warnings of type SBML unit consistency (20702). The following is the first warning at line 741, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'c' does not have a 'units' attribute.\r\n - 15 warnings of type SBML unit consistency (99505). The following is the first warning at line 1073, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '(c_0 - c_b) * exp(-r_1 * time) + c_b' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 22 warnings of type Modeling practice (80701). The following is the first warning at line 741, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'c' does not have a 'units' attribute.\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/Tang2020.sedml763034.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Tang2020.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='q_lockdown']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_1_task1` is invalid.\n - Variable `p1_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `c_1_task1` is invalid.\n - Variable `p2_c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='c']` are not defined. No namespaces are defined for the target.\n - Data generator `delta_I_2_task1` is invalid.\n - Variable `p2_delta_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='delta_I']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_1_task2` is invalid.\n - Variable `p1_Infected_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `c_1_task2` is invalid.\n - Variable `p2_c_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='c']` are not defined. No namespaces are defined for the target.\n - Data generator `delta_I_2_task2` is invalid.\n - Variable `p2_delta_I_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='delta_I']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Tang2020.sedml823379.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Tang2020.xml` has warnings.\r\n - 22 warnings of type SBML unit consistency (99508). The following is the first warning at line 741, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'c' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 22 warnings of type SBML unit consistency (20702). The following is the first warning at line 741, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'c' does not have a 'units' attribute.\r\n - 15 warnings of type SBML unit consistency (99505). The following is the first warning at line 1073, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '(c_0 - c_b) * exp(-r_1 * time) + c_b' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 22 warnings of type Modeling practice (80701). The following is the first warning at line 741, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'c' does not have a 'units' attribute.\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Tang2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Tang2020.xml` has warnings.\r\n - 22 warnings of type SBML unit consistency (99508). The following is the first warning at line 741, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'c' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 22 warnings of type SBML unit consistency (20702). The following is the first warning at line 741, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'c' does not have a 'units' attribute.\r\n - 15 warnings of type SBML unit consistency (99505). The following is the first warning at line 1073, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '(c_0 - c_b) * exp(-r_1 * time) + c_b' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 22 warnings of type Modeling practice (80701). The following is the first warning at line 741, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'c' does not have a 'units' attribute.\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Tang2020.sedml823379.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Tang2020.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='q_lockdown']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_1_task1` is invalid.\n - Variable `p1_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `c_1_task1` is invalid.\n - Variable `p2_c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='c']` are not defined. No namespaces are defined for the target.\n - Data generator `delta_I_2_task1` is invalid.\n - Variable `p2_delta_I_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='delta_I']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_1_task2` is invalid.\n - Variable `p1_Infected_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `c_1_task2` is invalid.\n - Variable `p2_c_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='c']` are not defined. No namespaces are defined for the target.\n - Data generator `delta_I_2_task2` is invalid.\n - Variable `p2_delta_I_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='delta_I']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000973](https://www.ebi.ac.uk/biomodels/BIOMD0000000973)<br/><sup>Dasgupta2020 - Reduced model o</sup></summary><sup>Dasgupta2020 - Reduced model of receptor clusturing and aggregation</sup></details>|<details><summary>pass</summary>[Dasgupta2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000973#Files)</details>|pass|<details><summary>pass</summary>[Dasgupta2020.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000973#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Dasgupta2020.sedml407396.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/Dasgupta2020.sedml407396.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Dasgupta2020.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - YDataReference from Curve `p1_curve_1_task1` cannot be resolved to a data generator with id `P_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_2_task1` cannot be resolved to a data generator with id `N_2_task1` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_3_task1` cannot be resolved to a data generator with id `s_3_task1` because multiple data generators have this id.\n - XDataReference from Curve `p2_curve_1_task1` cannot be resolved to a data generator with id `P_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p2_curve_1_task1` cannot be resolved to a data generator with id `P_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p2_curve_2_task1` cannot be resolved to a data generator with id `N_2_task1` because multiple data generators have this id.\n - YDataReference from Curve `p2_curve_3_task1` cannot be resolved to a data generator with id `s_3_task1` because multiple data generators have this id.\n - YDataReference from Curve `p3_curve_1_task1` cannot be resolved to a data generator with id `P_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_1_task2` cannot be resolved to a data generator with id `P_1_task2` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_2_task2` cannot be resolved to a data generator with id `N_2_task2` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_3_task2` cannot be resolved to a data generator with id `s_3_task2` because multiple data generators have this id.\n - XDataReference from Curve `p2_curve_1_task2` cannot be resolved to a data generator with id `P_1_task2` because multiple data generators have this id.\n - YDataReference from Curve `p2_curve_1_task2` cannot be resolved to a data generator with id `P_1_task2` because multiple data generators have this id.\n - YDataReference from Curve `p2_curve_2_task2` cannot be resolved to a data generator with id `N_2_task2` because multiple data generators have this id.\n - YDataReference from Curve `p2_curve_3_task2` cannot be resolved to a data generator with id `s_3_task2` because multiple data generators have this id.\n - YDataReference from Curve `p3_curve_1_task2` cannot be resolved to a data generator with id `P_1_task2` because multiple data generators have this id.\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Dasgupta2020.sedml593762.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Dasgupta2020.sedml593762.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Dasgupta2020.sedml` is invalid.\n - The elements of the document have references that could not be resolved.\n - YDataReference from Curve `p1_curve_1_task1` cannot be resolved to a data generator with id `P_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_2_task1` cannot be resolved to a data generator with id `N_2_task1` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_3_task1` cannot be resolved to a data generator with id `s_3_task1` because multiple data generators have this id.\n - XDataReference from Curve `p2_curve_1_task1` cannot be resolved to a data generator with id `P_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p2_curve_1_task1` cannot be resolved to a data generator with id `P_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p2_curve_2_task1` cannot be resolved to a data generator with id `N_2_task1` because multiple data generators have this id.\n - YDataReference from Curve `p2_curve_3_task1` cannot be resolved to a data generator with id `s_3_task1` because multiple data generators have this id.\n - YDataReference from Curve `p3_curve_1_task1` cannot be resolved to a data generator with id `P_1_task1` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_1_task2` cannot be resolved to a data generator with id `P_1_task2` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_2_task2` cannot be resolved to a data generator with id `N_2_task2` because multiple data generators have this id.\n - YDataReference from Curve `p1_curve_3_task2` cannot be resolved to a data generator with id `s_3_task2` because multiple data generators have this id.\n - XDataReference from Curve `p2_curve_1_task2` cannot be resolved to a data generator with id `P_1_task2` because multiple data generators have this id.\n - YDataReference from Curve `p2_curve_1_task2` cannot be resolved to a data generator with id `P_1_task2` because multiple data generators have this id.\n - YDataReference from Curve `p2_curve_2_task2` cannot be resolved to a data generator with id `N_2_task2` because multiple data generators have this id.\n - YDataReference from Curve `p2_curve_3_task2` cannot be resolved to a data generator with id `s_3_task2` because multiple data generators have this id.\n - YDataReference from Curve `p3_curve_1_task2` cannot be resolved to a data generator with id `P_1_task2` because multiple data generators have this id.\n'```</details>|
|<details><summary>[BIOMD0000000974](https://www.ebi.ac.uk/biomodels/BIOMD0000000974)<br/><sup>Carcione2020 - Deterministic S</sup></summary><sup>Carcione2020 - Deterministic SEIR simulation of a COVID-19 outbreak</sup></details>|<details><summary>pass</summary>[Carcione2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000974#Files)</details>|pass|<details><summary>pass</summary>[Carcione2020.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000974#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Carcione2020.sedml991876.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Carcione2020.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99508). The following is the first warning at line 398, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'lambda' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type SBML unit consistency (20702). The following is the first warning at line 398, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'lambda' does not have a 'units' attribute.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 505, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'mu * Total_population' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 398, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lambda' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Carcione2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Carcione2020.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99508). The following is the first warning at line 398, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'lambda' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type SBML unit consistency (20702). The following is the first warning at line 398, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'lambda' does not have a 'units' attribute.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 505, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'mu * Total_population' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 398, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lambda' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Carcione2020.sedml991876.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Carcione2020.sedml` is invalid.\n - Data generator `Susceptible_1_task1` is invalid.\n - Variable `p1_Susceptible_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Susceptible']` are not defined. No namespaces are defined for the target.\n - Data generator `Exposed_2_task1` is invalid.\n - Variable `p1_Exposed_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Exposed']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_3_task1` is invalid.\n - Variable `p1_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Recovered_4_task1` is invalid.\n - Variable `p1_Recovered_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Recovered']` are not defined. No namespaces are defined for the target.\n - Data generator `Deceased_1_task1` is invalid.\n - Variable `p2_Deceased_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Deceased']` are not defined. No namespaces are defined for the target.\n - Output `plot_3_task1` is invalid.\n - Plot must have at least one curve.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Carcione2020.sedml923661.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Carcione2020.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99508). The following is the first warning at line 398, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'lambda' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type SBML unit consistency (20702). The following is the first warning at line 398, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'lambda' does not have a 'units' attribute.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 505, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'mu * Total_population' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 398, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lambda' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Carcione2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Carcione2020.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99508). The following is the first warning at line 398, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'lambda' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type SBML unit consistency (20702). The following is the first warning at line 398, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'lambda' does not have a 'units' attribute.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 505, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'mu * Total_population' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 398, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lambda' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Carcione2020.sedml923661.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Carcione2020.sedml` is invalid.\n - Data generator `Susceptible_1_task1` is invalid.\n - Variable `p1_Susceptible_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Susceptible']` are not defined. No namespaces are defined for the target.\n - Data generator `Exposed_2_task1` is invalid.\n - Variable `p1_Exposed_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Exposed']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_3_task1` is invalid.\n - Variable `p1_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Recovered_4_task1` is invalid.\n - Variable `p1_Recovered_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Recovered']` are not defined. No namespaces are defined for the target.\n - Data generator `Deceased_1_task1` is invalid.\n - Variable `p2_Deceased_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Deceased']` are not defined. No namespaces are defined for the target.\n - Output `plot_3_task1` is invalid.\n - Plot must have at least one curve.\n"```</details>|
|<details><summary>[BIOMD0000000975](https://www.ebi.ac.uk/biomodels/BIOMD0000000975)<br/><sup>Leloup2004 - Mammalian Circadi</sup></summary><sup>Leloup2004 - Mammalian Circadian Rhythm models for 23.8 and 24.2 hours timeperiod</sup></details>|<details><summary>pass</summary>[Leloup2004_Set1.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000975#Files)</details>|FAIL|<details><summary>pass</summary>[Leloup2004_Set1.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000975#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Leloup2004_Set1.sedml537561.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Leloup2004_Set1.xml` has warnings.\r\n - 40 warnings of type SBML unit consistency (10501). The following is the first warning at line 4305, column 6:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'vmP * (MP / (KmP + MP)) + kdmp * MP' in the math element of the <rateRule> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 4214, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '1.1 * vstot' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 46 warnings of type SBML unit consistency (10541). The following is the first warning at line 5235, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -9), second (exponent = -1, multiplier = 3600, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Reversible_reaction__between_PER_CRY_complex_in_cytosol_and_nucleus') are second (exponent = -1, multiplier = 1, scale = -3), mole (exponent = 1, multiplier = 1, scale = -9).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 18.0\r\n - Output end time: 92.0\r\n - Number of steps: 74\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Leloup2004_Set1.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Leloup2004_Set1.xml` has warnings.\r\n - 40 warnings of type SBML unit consistency (10501). The following is the first warning at line 4305, column 6:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'vmP * (MP / (KmP + MP)) + kdmp * MP' in the math element of the <rateRule> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 4214, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '1.1 * vstot' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 46 warnings of type SBML unit consistency (10541). The following is the first warning at line 5235, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -9), second (exponent = -1, multiplier = 3600, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Reversible_reaction__between_PER_CRY_complex_in_cytosol_and_nucleus') are second (exponent = -1, multiplier = 1, scale = -3), mole (exponent = 1, multiplier = 1, scale = -9).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 18.0\r\n - Output end time: 92.0\r\n - Number of steps: 74\x1b[0m\r\n\r\n`/root/in/Leloup2004_Set1.sedml537561.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Leloup2004_Set1.sedml` is invalid.\n - Data generator `MC_0_1_task1` is invalid.\n - Variable `p1_MC_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='MC_0']` are not defined. No namespaces are defined for the target.\n - Data generator `MP_0_2_task1` is invalid.\n - Variable `p1_MP_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='MP_0']` are not defined. No namespaces are defined for the target.\n - Data generator `MB_0_1_task1` is invalid.\n - Variable `p2_MB_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='MB_0']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Leloup2004_Set1.sedml278615.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Leloup2004_Set1.xml` has warnings.\r\n - 40 warnings of type SBML unit consistency (10501). The following is the first warning at line 4305, column 6:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'vmP * (MP / (KmP + MP)) + kdmp * MP' in the math element of the <rateRule> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 4214, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '1.1 * vstot' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 46 warnings of type SBML unit consistency (10541). The following is the first warning at line 5235, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -9), second (exponent = -1, multiplier = 3600, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Reversible_reaction__between_PER_CRY_complex_in_cytosol_and_nucleus') are second (exponent = -1, multiplier = 1, scale = -3), mole (exponent = 1, multiplier = 1, scale = -9).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 18.0\r\n - Output end time: 92.0\r\n - Number of steps: 74\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Leloup2004_Set1.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Leloup2004_Set1.xml` has warnings.\r\n - 40 warnings of type SBML unit consistency (10501). The following is the first warning at line 4305, column 6:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'vmP * (MP / (KmP + MP)) + kdmp * MP' in the math element of the <rateRule> can only act on variables with the same units.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 4214, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '1.1 * vstot' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 46 warnings of type SBML unit consistency (10541). The following is the first warning at line 5235, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are mole (exponent = 1, multiplier = 1, scale = -9), second (exponent = -1, multiplier = 3600, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'Reversible_reaction__between_PER_CRY_complex_in_cytosol_and_nucleus') are second (exponent = -1, multiplier = 1, scale = -3), mole (exponent = 1, multiplier = 1, scale = -9).\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 18.0\r\n - Output end time: 92.0\r\n - Number of steps: 74\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Leloup2004_Set1.sedml278615.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Leloup2004_Set1.sedml` is invalid.\n - Data generator `MC_0_1_task1` is invalid.\n - Variable `p1_MC_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='MC_0']` are not defined. No namespaces are defined for the target.\n - Data generator `MP_0_2_task1` is invalid.\n - Variable `p1_MP_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='MP_0']` are not defined. No namespaces are defined for the target.\n - Data generator `MB_0_1_task1` is invalid.\n - Variable `p2_MB_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='MB_0']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000976](https://www.ebi.ac.uk/biomodels/BIOMD0000000976)<br/><sup>Ghanbari2020 - forecasting the</sup></summary><sup>Ghanbari2020 - forecasting the second wave of COVID-19 in Iran</sup></details>|<details><summary>pass</summary>[Ghanbari2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000976#Files)</details>|pass|<details><summary>pass</summary>[Ghanbari2020.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000976#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Ghanbari2020.sedml654641.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Ghanbari2020.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99508). The following is the first warning at line 407, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'alpha' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type SBML unit consistency (20702). The following is the first warning at line 407, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'alpha' does not have a 'units' attribute.\r\n - 11 warnings of type SBML unit consistency (99505). The following is the first warning at line 555, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Iran * Rate_Law_for_reaction(tau, Susceptible, alpha, Infected_strong_immune_system, Infected_weak_immune_system)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type Modeling practice (80701). The following is the first warning at line 407, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Ghanbari2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Ghanbari2020.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99508). The following is the first warning at line 407, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'alpha' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type SBML unit consistency (20702). The following is the first warning at line 407, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'alpha' does not have a 'units' attribute.\r\n - 11 warnings of type SBML unit consistency (99505). The following is the first warning at line 555, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Iran * Rate_Law_for_reaction(tau, Susceptible, alpha, Infected_strong_immune_system, Infected_weak_immune_system)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type Modeling practice (80701). The following is the first warning at line 407, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Ghanbari2020.sedml654641.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Ghanbari2020.sedml` is invalid.\n - Data generator `Infected_strong_immune_system_1_task1` is invalid.\n - Variable `p1_Infected_strong_immune_system_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected_strong_immune_system']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_weak_immune_system_1_task1` is invalid.\n - Variable `p2_Infected_weak_immune_system_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected_weak_immune_system']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Ghanbari2020.sedml263391.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Ghanbari2020.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99508). The following is the first warning at line 407, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'alpha' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type SBML unit consistency (20702). The following is the first warning at line 407, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'alpha' does not have a 'units' attribute.\r\n - 11 warnings of type SBML unit consistency (99505). The following is the first warning at line 555, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Iran * Rate_Law_for_reaction(tau, Susceptible, alpha, Infected_strong_immune_system, Infected_weak_immune_system)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type Modeling practice (80701). The following is the first warning at line 407, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Ghanbari2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Ghanbari2020.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99508). The following is the first warning at line 407, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'alpha' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type SBML unit consistency (20702). The following is the first warning at line 407, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'alpha' does not have a 'units' attribute.\r\n - 11 warnings of type SBML unit consistency (99505). The following is the first warning at line 555, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Iran * Rate_Law_for_reaction(tau, Susceptible, alpha, Infected_strong_immune_system, Infected_weak_immune_system)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type Modeling practice (80701). The following is the first warning at line 407, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'alpha' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Ghanbari2020.sedml263391.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Ghanbari2020.sedml` is invalid.\n - Data generator `Infected_strong_immune_system_1_task1` is invalid.\n - Variable `p1_Infected_strong_immune_system_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected_strong_immune_system']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_weak_immune_system_1_task1` is invalid.\n - Variable `p2_Infected_weak_immune_system_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected_weak_immune_system']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000977](https://www.ebi.ac.uk/biomodels/BIOMD0000000977)<br/><sup>Sarkar2020 - SAIR model of COV</sup></summary><sup>Sarkar2020 - SAIR model of COVID-19 transmission with quarantine measures in India</sup></details>|<details><summary>pass</summary>[Sarkar2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000977#Files)</details>|pass|<details><summary>pass</summary>[Sarkar2020.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000977#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Sarkar2020.sedml728076.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Sarkar2020.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (99508). The following is the first warning at line 537, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'lambda_s' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type SBML unit consistency (20702). The following is the first warning at line 537, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'lambda_s' does not have a 'units' attribute.\r\n - 15 warnings of type SBML unit consistency (99505). The following is the first warning at line 797, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Wuhan * Rate_Law_for_Susceptible_to_Susceptible_quarantined(beta_s, rho_s, epsilon_s, Susceptible, Infected, Total_population)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 537, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lambda_s' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 91.0\r\n - Number of steps: 91\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Sarkar2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Sarkar2020.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (99508). The following is the first warning at line 537, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'lambda_s' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type SBML unit consistency (20702). The following is the first warning at line 537, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'lambda_s' does not have a 'units' attribute.\r\n - 15 warnings of type SBML unit consistency (99505). The following is the first warning at line 797, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Wuhan * Rate_Law_for_Susceptible_to_Susceptible_quarantined(beta_s, rho_s, epsilon_s, Susceptible, Infected, Total_population)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 537, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lambda_s' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 91.0\r\n - Number of steps: 91\x1b[0m\r\n\r\n`/root/in/Sarkar2020.sedml728076.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Sarkar2020.sedml` is invalid.\n - Data generator `Infected_1_task1` is invalid.\n - Variable `p1_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Cumulative_cases_1_task1` is invalid.\n - Variable `p2_Cumulative_cases_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Cumulative_cases']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Sarkar2020.sedml686269.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Sarkar2020.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (99508). The following is the first warning at line 537, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'lambda_s' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type SBML unit consistency (20702). The following is the first warning at line 537, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'lambda_s' does not have a 'units' attribute.\r\n - 15 warnings of type SBML unit consistency (99505). The following is the first warning at line 797, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Wuhan * Rate_Law_for_Susceptible_to_Susceptible_quarantined(beta_s, rho_s, epsilon_s, Susceptible, Infected, Total_population)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 537, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lambda_s' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 91.0\r\n - Number of steps: 91\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Sarkar2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Sarkar2020.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (99508). The following is the first warning at line 537, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'lambda_s' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type SBML unit consistency (20702). The following is the first warning at line 537, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'lambda_s' does not have a 'units' attribute.\r\n - 15 warnings of type SBML unit consistency (99505). The following is the first warning at line 797, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Wuhan * Rate_Law_for_Susceptible_to_Susceptible_quarantined(beta_s, rho_s, epsilon_s, Susceptible, Infected, Total_population)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 537, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lambda_s' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 91.0\r\n - Number of steps: 91\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Sarkar2020.sedml686269.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Sarkar2020.sedml` is invalid.\n - Data generator `Infected_1_task1` is invalid.\n - Variable `p1_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Cumulative_cases_1_task1` is invalid.\n - Variable `p2_Cumulative_cases_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Cumulative_cases']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000978](https://www.ebi.ac.uk/biomodels/BIOMD0000000978)<br/><sup>Mukandavire2020 - SEIR model o</sup></summary><sup>Mukandavire2020 - SEIR model of early COVID-19 transmission in South Africa</sup></details>|<details><summary>pass</summary>[Mukandavire2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000978#Files)</details>|pass|<details><summary>pass</summary>[Mukandavire2020.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000978#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Mukandavire2020.sedml152518.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Mukandavire2020.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99508). The following is the first warning at line 342, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'beta' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type SBML unit consistency (20702). The following is the first warning at line 342, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'beta' does not have a 'units' attribute.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 449, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'N' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type Modeling practice (80701). The following is the first warning at line 342, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 23.0\r\n - Number of steps: 23\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Mukandavire2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Mukandavire2020.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99508). The following is the first warning at line 342, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'beta' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type SBML unit consistency (20702). The following is the first warning at line 342, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'beta' does not have a 'units' attribute.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 449, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'N' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type Modeling practice (80701). The following is the first warning at line 342, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 23.0\r\n - Number of steps: 23\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/Mukandavire2020.sedml152518.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Mukandavire2020.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Event_trigger']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_1_task1` is invalid.\n - Variable `p1_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_1_task2` is invalid.\n - Variable `p1_Infected_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Mukandavire2020.sedml463458.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Mukandavire2020.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99508). The following is the first warning at line 342, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'beta' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type SBML unit consistency (20702). The following is the first warning at line 342, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'beta' does not have a 'units' attribute.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 449, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'N' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type Modeling practice (80701). The following is the first warning at line 342, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 23.0\r\n - Number of steps: 23\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Mukandavire2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Mukandavire2020.xml` has warnings.\r\n - 7 warnings of type SBML unit consistency (99508). The following is the first warning at line 342, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'beta' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type SBML unit consistency (20702). The following is the first warning at line 342, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'beta' does not have a 'units' attribute.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 449, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'N' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type Modeling practice (80701). The following is the first warning at line 342, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 23.0\r\n - Number of steps: 23\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Mukandavire2020.sedml463458.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Mukandavire2020.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Event_trigger']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_1_task1` is invalid.\n - Variable `p1_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_1_task2` is invalid.\n - Variable `p1_Infected_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000979](https://www.ebi.ac.uk/biomodels/BIOMD0000000979)<br/><sup>Malkov2020 - SEIRS model of CO</sup></summary><sup>Malkov2020 - SEIRS model of COVID-19 transmission with reinfection</sup></details>|<details><summary>pass</summary>[Malkov2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000979#Files)</details>|pass|<details><summary>pass</summary>[Malkov2020.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000979#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Malkov2020.sedml120278.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Malkov2020.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99508). The following is the first warning at line 375, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'beta' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type SBML unit consistency (20702). The following is the first warning at line 375, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'beta' does not have a 'units' attribute.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 486, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'gamma * R_fixed' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 375, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Malkov2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Malkov2020.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99508). The following is the first warning at line 375, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'beta' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type SBML unit consistency (20702). The following is the first warning at line 375, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'beta' does not have a 'units' attribute.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 486, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'gamma * R_fixed' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 375, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/Malkov2020.sedml120278.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Malkov2020.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='R_fixed']` are not defined. No namespaces are defined for the target.\n - Change 2 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='omega']` are not defined. No namespaces are defined for the target.\n - The child ranges of repeated tasks must be at least as long as the main range of their parent repeated tasks. Range `range1` is shorter than its main range, 2 < 7.\n - The child ranges of repeated tasks must be at least as long as the main range of their parent repeated tasks. Range `range1` is shorter than its main range, 2 < 7.\n - Data generator `Fraction_infected_1_task1` is invalid.\n - Variable `p1_Fraction_infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Fraction_infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Fraction_infected_1_task2` is invalid.\n - Variable `p1_Fraction_infected_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Fraction_infected']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Malkov2020.sedml828457.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Malkov2020.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99508). The following is the first warning at line 375, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'beta' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type SBML unit consistency (20702). The following is the first warning at line 375, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'beta' does not have a 'units' attribute.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 486, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'gamma * R_fixed' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 375, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Malkov2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Malkov2020.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99508). The following is the first warning at line 375, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'beta' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type SBML unit consistency (20702). The following is the first warning at line 375, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'beta' does not have a 'units' attribute.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 486, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'gamma * R_fixed' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 6 warnings of type Modeling practice (80701). The following is the first warning at line 375, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Malkov2020.sedml828457.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Malkov2020.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='R_fixed']` are not defined. No namespaces are defined for the target.\n - Change 2 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='omega']` are not defined. No namespaces are defined for the target.\n - The child ranges of repeated tasks must be at least as long as the main range of their parent repeated tasks. Range `range1` is shorter than its main range, 2 < 7.\n - The child ranges of repeated tasks must be at least as long as the main range of their parent repeated tasks. Range `range1` is shorter than its main range, 2 < 7.\n - Data generator `Fraction_infected_1_task1` is invalid.\n - Variable `p1_Fraction_infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Fraction_infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Fraction_infected_1_task2` is invalid.\n - Variable `p1_Fraction_infected_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Fraction_infected']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000980](https://www.ebi.ac.uk/biomodels/BIOMD0000000980)<br/><sup>Malkov2020 - SEIRS model of CO</sup></summary><sup>Malkov2020 - SEIRS model of COVID-19 transmission with time-varying R values and reinfection</sup></details>|<details><summary>pass</summary>[Malkov2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000980#Files)</details>|pass|<details><summary>pass</summary>[Malkov2020.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000980#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Malkov2020.sedml519149.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Malkov2020.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (99508). The following is the first warning at line 356, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'beta' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type SBML unit consistency (20702). The following is the first warning at line 356, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'beta' does not have a 'units' attribute.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 581, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'exp(-eta * time) * R_1_0 + (1 - exp(-eta * time)) * R_1_f' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 356, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Malkov2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Malkov2020.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (99508). The following is the first warning at line 356, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'beta' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type SBML unit consistency (20702). The following is the first warning at line 356, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'beta' does not have a 'units' attribute.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 581, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'exp(-eta * time) * R_1_0 + (1 - exp(-eta * time)) * R_1_f' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 356, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/Malkov2020.sedml519149.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Malkov2020.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='eta']` are not defined. No namespaces are defined for the target.\n - Change 2 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='omega']` are not defined. No namespaces are defined for the target.\n - The child ranges of repeated tasks must be at least as long as the main range of their parent repeated tasks. Range `range1` is shorter than its main range, 2 < 5.\n - The child ranges of repeated tasks must be at least as long as the main range of their parent repeated tasks. Range `range1` is shorter than its main range, 2 < 5.\n - Data generator `Fraction_infected_1_task1` is invalid.\n - Variable `p1_Fraction_infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Fraction_infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Fraction_infected_1_task2` is invalid.\n - Variable `p1_Fraction_infected_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Fraction_infected']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Malkov2020.sedml649651.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Malkov2020.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (99508). The following is the first warning at line 356, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'beta' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type SBML unit consistency (20702). The following is the first warning at line 356, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'beta' does not have a 'units' attribute.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 581, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'exp(-eta * time) * R_1_0 + (1 - exp(-eta * time)) * R_1_f' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 356, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Malkov2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Malkov2020.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (99508). The following is the first warning at line 356, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'beta' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type SBML unit consistency (20702). The following is the first warning at line 356, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'beta' does not have a 'units' attribute.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 581, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'exp(-eta * time) * R_1_0 + (1 - exp(-eta * time)) * R_1_f' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 356, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Malkov2020.sedml649651.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Malkov2020.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='eta']` are not defined. No namespaces are defined for the target.\n - Change 2 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='omega']` are not defined. No namespaces are defined for the target.\n - The child ranges of repeated tasks must be at least as long as the main range of their parent repeated tasks. Range `range1` is shorter than its main range, 2 < 5.\n - The child ranges of repeated tasks must be at least as long as the main range of their parent repeated tasks. Range `range1` is shorter than its main range, 2 < 5.\n - Data generator `Fraction_infected_1_task1` is invalid.\n - Variable `p1_Fraction_infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Fraction_infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Fraction_infected_1_task2` is invalid.\n - Variable `p1_Fraction_infected_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Fraction_infected']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000981](https://www.ebi.ac.uk/biomodels/BIOMD0000000981)<br/><sup>Wan2020 - risk estimation and </sup></summary><sup>Wan2020 - risk estimation and prediction of the transmission of COVID-19 in maninland China excluding Hubei province</sup></details>|<details><summary>pass</summary>[Wan2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000981#Files)</details>|pass|<details><summary>pass</summary>[Wan2020.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000981#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Wan2020.sedml419577.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Wan2020.xml` has warnings.\r\n - 17 warnings of type SBML unit consistency (99508). The following is the first warning at line 798, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'beta' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 17 warnings of type SBML unit consistency (20702). The following is the first warning at line 798, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'beta' does not have a 'units' attribute.\r\n - 19 warnings of type SBML unit consistency (99505). The following is the first warning at line 1067, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'q1 * exp(-delta * Total_reported_cases)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 17 warnings of type Modeling practice (80701). The following is the first warning at line 798, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Wan2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Wan2020.xml` has warnings.\r\n - 17 warnings of type SBML unit consistency (99508). The following is the first warning at line 798, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'beta' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 17 warnings of type SBML unit consistency (20702). The following is the first warning at line 798, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'beta' does not have a 'units' attribute.\r\n - 19 warnings of type SBML unit consistency (99505). The following is the first warning at line 1067, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'q1 * exp(-delta * Total_reported_cases)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 17 warnings of type Modeling practice (80701). The following is the first warning at line 798, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Wan2020.sedml419577.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Wan2020.sedml` is invalid.\n - Data generator `Total_reported_cases_1_task1` is invalid.\n - Variable `p1_Total_reported_cases_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Total_reported_cases']` are not defined. No namespaces are defined for the target.\n - Data generator `Deceased_1_task1` is invalid.\n - Variable `p2_Deceased_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Deceased']` are not defined. No namespaces are defined for the target.\n - Data generator `Hospitalised_1_task1` is invalid.\n - Variable `p3_Hospitalised_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Hospitalised']` are not defined. No namespaces are defined for the target.\n - Data generator `Recovered_from_hospitals_1_task1` is invalid.\n - Variable `p4_Recovered_from_hospitals_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Recovered_from_hospitals']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Wan2020.sedml571785.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Wan2020.xml` has warnings.\r\n - 17 warnings of type SBML unit consistency (99508). The following is the first warning at line 798, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'beta' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 17 warnings of type SBML unit consistency (20702). The following is the first warning at line 798, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'beta' does not have a 'units' attribute.\r\n - 19 warnings of type SBML unit consistency (99505). The following is the first warning at line 1067, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'q1 * exp(-delta * Total_reported_cases)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 17 warnings of type Modeling practice (80701). The following is the first warning at line 798, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Wan2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Wan2020.xml` has warnings.\r\n - 17 warnings of type SBML unit consistency (99508). The following is the first warning at line 798, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'beta' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 17 warnings of type SBML unit consistency (20702). The following is the first warning at line 798, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'beta' does not have a 'units' attribute.\r\n - 19 warnings of type SBML unit consistency (99505). The following is the first warning at line 1067, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'q1 * exp(-delta * Total_reported_cases)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 17 warnings of type Modeling practice (80701). The following is the first warning at line 798, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Wan2020.sedml571785.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Wan2020.sedml` is invalid.\n - Data generator `Total_reported_cases_1_task1` is invalid.\n - Variable `p1_Total_reported_cases_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Total_reported_cases']` are not defined. No namespaces are defined for the target.\n - Data generator `Deceased_1_task1` is invalid.\n - Variable `p2_Deceased_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Deceased']` are not defined. No namespaces are defined for the target.\n - Data generator `Hospitalised_1_task1` is invalid.\n - Variable `p3_Hospitalised_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Hospitalised']` are not defined. No namespaces are defined for the target.\n - Data generator `Recovered_from_hospitals_1_task1` is invalid.\n - Variable `p4_Recovered_from_hospitals_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Recovered_from_hospitals']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000982](https://www.ebi.ac.uk/biomodels/BIOMD0000000982)<br/><sup>Law2020 - SIR model of COVID-1</sup></summary><sup>Law2020 - SIR model of COVID-19 transmission in Malyasia with time-varying parameters</sup></details>|<details><summary>pass</summary>[Law2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000982#Files)</details>|FAIL|<details><summary>pass</summary>[Law2020.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000982#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Law2020.sedml248765.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Law2020.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 540, column 6:\r\n - The formula 'pow(1 - p, time)' in the math element of the <assignmentRule> contains a power that is not dimensionless and thus may produce invalid units.\r\n - 10 warnings of type SBML unit consistency (99508). The following is the first warning at line 303, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'z' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type SBML unit consistency (20702). The following is the first warning at line 303, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'z' does not have a 'units' attribute.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 455, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'Trigger_Stage_1 * 0.025 + Trigger_Stage_2 * 0.042 + Trigger_Stage_3 * 0.05' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 303, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'z' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Law2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Law2020.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 540, column 6:\r\n - The formula 'pow(1 - p, time)' in the math element of the <assignmentRule> contains a power that is not dimensionless and thus may produce invalid units.\r\n - 10 warnings of type SBML unit consistency (99508). The following is the first warning at line 303, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'z' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type SBML unit consistency (20702). The following is the first warning at line 303, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'z' does not have a 'units' attribute.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 455, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'Trigger_Stage_1 * 0.025 + Trigger_Stage_2 * 0.042 + Trigger_Stage_3 * 0.05' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 303, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'z' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Law2020.sedml248765.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Law2020.sedml` is invalid.\n - Data generator `I_total_1_task1` is invalid.\n - Variable `p1_I_total_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='I_total']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_2_task1` is invalid.\n - Variable `p1_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Removed_3_task1` is invalid.\n - Variable `p1_Removed_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Removed']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Law2020.sedml177514.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Law2020.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 540, column 6:\r\n - The formula 'pow(1 - p, time)' in the math element of the <assignmentRule> contains a power that is not dimensionless and thus may produce invalid units.\r\n - 10 warnings of type SBML unit consistency (99508). The following is the first warning at line 303, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'z' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type SBML unit consistency (20702). The following is the first warning at line 303, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'z' does not have a 'units' attribute.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 455, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'Trigger_Stage_1 * 0.025 + Trigger_Stage_2 * 0.042 + Trigger_Stage_3 * 0.05' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 303, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'z' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Law2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Law2020.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 540, column 6:\r\n - The formula 'pow(1 - p, time)' in the math element of the <assignmentRule> contains a power that is not dimensionless and thus may produce invalid units.\r\n - 10 warnings of type SBML unit consistency (99508). The following is the first warning at line 303, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'z' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type SBML unit consistency (20702). The following is the first warning at line 303, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'z' does not have a 'units' attribute.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 455, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'Trigger_Stage_1 * 0.025 + Trigger_Stage_2 * 0.042 + Trigger_Stage_3 * 0.05' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 303, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'z' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Law2020.sedml177514.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Law2020.sedml` is invalid.\n - Data generator `I_total_1_task1` is invalid.\n - Variable `p1_I_total_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='I_total']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_2_task1` is invalid.\n - Variable `p1_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Removed_3_task1` is invalid.\n - Variable `p1_Removed_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Removed']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000983](https://www.ebi.ac.uk/biomodels/BIOMD0000000983)<br/><sup>Zongo2020 - model of COVID-19 </sup></summary><sup>Zongo2020 - model of COVID-19 transmission dynamics under containment measures in France</sup></details>|<details><summary>pass</summary>[Zongo2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000983#Files)</details>|pass|<details><summary>pass</summary>[Zongo2020.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000983#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Zongo2020.sedml418604.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Zongo2020.xml` has warnings.\r\n - 21 warnings of type SBML unit consistency (99508). The following is the first warning at line 706, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'p' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 21 warnings of type SBML unit consistency (20702). The following is the first warning at line 706, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'p' does not have a 'units' attribute.\r\n - 27 warnings of type SBML unit consistency (99505). The following is the first warning at line 1023, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '-p / (tau_f - tau_3)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 21 warnings of type Modeling practice (80701). The following is the first warning at line 706, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'p' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 126.0\r\n - Number of steps: 126\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Zongo2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Zongo2020.xml` has warnings.\r\n - 21 warnings of type SBML unit consistency (99508). The following is the first warning at line 706, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'p' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 21 warnings of type SBML unit consistency (20702). The following is the first warning at line 706, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'p' does not have a 'units' attribute.\r\n - 27 warnings of type SBML unit consistency (99505). The following is the first warning at line 1023, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '-p / (tau_f - tau_3)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 21 warnings of type Modeling practice (80701). The following is the first warning at line 706, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'p' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 126.0\r\n - Number of steps: 126\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/Zongo2020.sedml418604.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Zongo2020.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='tau_3']` are not defined. No namespaces are defined for the target.\n - Data generator `m_t_1_task1` is invalid.\n - Variable `p1_m_t_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='m_t']` are not defined. No namespaces are defined for the target.\n - Data generator `sigma_2_task1` is invalid.\n - Variable `p1_sigma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='sigma']` are not defined. No namespaces are defined for the target.\n - Data generator `S_c_1_task1` is invalid.\n - Variable `p2_S_c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S_c']` are not defined. No namespaces are defined for the target.\n - Data generator `S_u_2_task1` is invalid.\n - Variable `p2_S_u_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S_u']` are not defined. No namespaces are defined for the target.\n - Data generator `m_t_1_task2` is invalid.\n - Variable `p1_m_t_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='m_t']` are not defined. No namespaces are defined for the target.\n - Data generator `sigma_2_task2` is invalid.\n - Variable `p1_sigma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='sigma']` are not defined. No namespaces are defined for the target.\n - Data generator `S_c_1_task2` is invalid.\n - Variable `p2_S_c_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S_c']` are not defined. No namespaces are defined for the target.\n - Data generator `S_u_2_task2` is invalid.\n - Variable `p2_S_u_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S_u']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Zongo2020.sedml49178.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Zongo2020.xml` has warnings.\r\n - 21 warnings of type SBML unit consistency (99508). The following is the first warning at line 706, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'p' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 21 warnings of type SBML unit consistency (20702). The following is the first warning at line 706, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'p' does not have a 'units' attribute.\r\n - 27 warnings of type SBML unit consistency (99505). The following is the first warning at line 1023, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '-p / (tau_f - tau_3)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 21 warnings of type Modeling practice (80701). The following is the first warning at line 706, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'p' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 126.0\r\n - Number of steps: 126\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Zongo2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Zongo2020.xml` has warnings.\r\n - 21 warnings of type SBML unit consistency (99508). The following is the first warning at line 706, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'p' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 21 warnings of type SBML unit consistency (20702). The following is the first warning at line 706, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'p' does not have a 'units' attribute.\r\n - 27 warnings of type SBML unit consistency (99505). The following is the first warning at line 1023, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression '-p / (tau_f - tau_3)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 21 warnings of type Modeling practice (80701). The following is the first warning at line 706, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'p' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 126.0\r\n - Number of steps: 126\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Zongo2020.sedml49178.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Zongo2020.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='tau_3']` are not defined. No namespaces are defined for the target.\n - Data generator `m_t_1_task1` is invalid.\n - Variable `p1_m_t_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='m_t']` are not defined. No namespaces are defined for the target.\n - Data generator `sigma_2_task1` is invalid.\n - Variable `p1_sigma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='sigma']` are not defined. No namespaces are defined for the target.\n - Data generator `S_c_1_task1` is invalid.\n - Variable `p2_S_c_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S_c']` are not defined. No namespaces are defined for the target.\n - Data generator `S_u_2_task1` is invalid.\n - Variable `p2_S_u_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S_u']` are not defined. No namespaces are defined for the target.\n - Data generator `m_t_1_task2` is invalid.\n - Variable `p1_m_t_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='m_t']` are not defined. No namespaces are defined for the target.\n - Data generator `sigma_2_task2` is invalid.\n - Variable `p1_sigma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='sigma']` are not defined. No namespaces are defined for the target.\n - Data generator `S_c_1_task2` is invalid.\n - Variable `p2_S_c_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S_c']` are not defined. No namespaces are defined for the target.\n - Data generator `S_u_2_task2` is invalid.\n - Variable `p2_S_u_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='S_u']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000984](https://www.ebi.ac.uk/biomodels/BIOMD0000000984)<br/><sup>Fang2020 - SEIR model of COVID</sup></summary><sup>Fang2020 - SEIR model of COVID-19 transmission considering government interventions in Wuhan</sup></details>|<details><summary>pass</summary>[Fang2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000984#Files)</details>|pass|<details><summary>pass</summary>[Fang2020.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000984#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Fang2020.sedml232774.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Fang2020.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (99508). The following is the first warning at line 343, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'beta_o' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 5 warnings of type SBML unit consistency (20702). The following is the first warning at line 343, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'beta_o' does not have a 'units' attribute.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 439, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'k * beta_o' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 5 warnings of type Modeling practice (80701). The following is the first warning at line 343, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta_o' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Fang2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Fang2020.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (99508). The following is the first warning at line 343, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'beta_o' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 5 warnings of type SBML unit consistency (20702). The following is the first warning at line 343, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'beta_o' does not have a 'units' attribute.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 439, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'k * beta_o' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 5 warnings of type Modeling practice (80701). The following is the first warning at line 343, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta_o' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Fang2020.sedml232774.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Fang2020.sedml` is invalid.\n - Data generator `Susceptible_1_task1` is invalid.\n - Variable `p1_Susceptible_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Susceptible']` are not defined. No namespaces are defined for the target.\n - Data generator `Exposed_2_task1` is invalid.\n - Variable `p1_Exposed_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Exposed']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_3_task1` is invalid.\n - Variable `p1_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Recovered_4_task1` is invalid.\n - Variable `p1_Recovered_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Recovered']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Fang2020.sedml243942.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Fang2020.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (99508). The following is the first warning at line 343, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'beta_o' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 5 warnings of type SBML unit consistency (20702). The following is the first warning at line 343, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'beta_o' does not have a 'units' attribute.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 439, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'k * beta_o' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 5 warnings of type Modeling practice (80701). The following is the first warning at line 343, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta_o' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Fang2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Fang2020.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (99508). The following is the first warning at line 343, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'beta_o' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 5 warnings of type SBML unit consistency (20702). The following is the first warning at line 343, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'beta_o' does not have a 'units' attribute.\r\n - 4 warnings of type SBML unit consistency (99505). The following is the first warning at line 439, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'k * beta_o' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 5 warnings of type Modeling practice (80701). The following is the first warning at line 343, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'beta_o' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Fang2020.sedml243942.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Fang2020.sedml` is invalid.\n - Data generator `Susceptible_1_task1` is invalid.\n - Variable `p1_Susceptible_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Susceptible']` are not defined. No namespaces are defined for the target.\n - Data generator `Exposed_2_task1` is invalid.\n - Variable `p1_Exposed_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Exposed']` are not defined. No namespaces are defined for the target.\n - Data generator `Infected_3_task1` is invalid.\n - Variable `p1_Infected_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Infected']` are not defined. No namespaces are defined for the target.\n - Data generator `Recovered_4_task1` is invalid.\n - Variable `p1_Recovered_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Recovered']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000985](https://www.ebi.ac.uk/biomodels/BIOMD0000000985)<br/><sup>Gex-Fabry1984 - model of recep</sup></summary><sup>Gex-Fabry1984 - model of receptor-mediated endocytosis of EGF in BALB/c 3T3 cells</sup></details>|<details><summary>pass</summary>[Gex-Fabry1984.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000985#Files)</details>|pass|<details><summary>pass</summary>[Gex-Fabry1984.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000985#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Gex-Fabry1984.sedml295602.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Gex-Fabry1984.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (99508). The following is the first warning at line 485, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'k1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type SBML unit consistency (20702). The following is the first warning at line 485, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'k1' does not have a 'units' attribute.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 682, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'ke * (epsilon + (1 - epsilon) * (1 - pow(exp(-(time / delTe)), 3)))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 485, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Gex-Fabry1984.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Gex-Fabry1984.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (99508). The following is the first warning at line 485, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'k1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type SBML unit consistency (20702). The following is the first warning at line 485, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'k1' does not have a 'units' attribute.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 682, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'ke * (epsilon + (1 - epsilon) * (1 - pow(exp(-(time / delTe)), 3)))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 485, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/Gex-Fabry1984.sedml295602.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Gex-Fabry1984.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='ke']` are not defined. No namespaces are defined for the target.\n - Data generator `Surface_receptor_ligand_complex_1000_1_task1` is invalid.\n - Variable `p1_Surface_receptor_ligand_complex_1000_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Surface_receptor_ligand_complex_1000']` are not defined. No namespaces are defined for the target.\n - Data generator `Internalised_ligand_1000_2_task1` is invalid.\n - Variable `p1_Internalised_ligand_1000_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Internalised_ligand_1000']` are not defined. No namespaces are defined for the target.\n - Data generator `Surface_receptor_ligand_complex_1000_1_task2` is invalid.\n - Variable `p1_Surface_receptor_ligand_complex_1000_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Surface_receptor_ligand_complex_1000']` are not defined. No namespaces are defined for the target.\n - Data generator `Internalised_ligand_1000_2_task2` is invalid.\n - Variable `p1_Internalised_ligand_1000_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Internalised_ligand_1000']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Gex-Fabry1984.sedml943021.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Gex-Fabry1984.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (99508). The following is the first warning at line 485, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'k1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type SBML unit consistency (20702). The following is the first warning at line 485, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'k1' does not have a 'units' attribute.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 682, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'ke * (epsilon + (1 - epsilon) * (1 - pow(exp(-(time / delTe)), 3)))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 485, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Gex-Fabry1984.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Gex-Fabry1984.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (99508). The following is the first warning at line 485, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'k1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type SBML unit consistency (20702). The following is the first warning at line 485, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'k1' does not have a 'units' attribute.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 682, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'ke * (epsilon + (1 - epsilon) * (1 - pow(exp(-(time / delTe)), 3)))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 13 warnings of type Modeling practice (80701). The following is the first warning at line 485, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Gex-Fabry1984.sedml943021.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Gex-Fabry1984.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='ke']` are not defined. No namespaces are defined for the target.\n - Data generator `Surface_receptor_ligand_complex_1000_1_task1` is invalid.\n - Variable `p1_Surface_receptor_ligand_complex_1000_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Surface_receptor_ligand_complex_1000']` are not defined. No namespaces are defined for the target.\n - Data generator `Internalised_ligand_1000_2_task1` is invalid.\n - Variable `p1_Internalised_ligand_1000_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Internalised_ligand_1000']` are not defined. No namespaces are defined for the target.\n - Data generator `Surface_receptor_ligand_complex_1000_1_task2` is invalid.\n - Variable `p1_Surface_receptor_ligand_complex_1000_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Surface_receptor_ligand_complex_1000']` are not defined. No namespaces are defined for the target.\n - Data generator `Internalised_ligand_1000_2_task2` is invalid.\n - Variable `p1_Internalised_ligand_1000_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Internalised_ligand_1000']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000986](https://www.ebi.ac.uk/biomodels/BIOMD0000000986)<br/><sup>Aubry1995 - Multi-compartment </sup></summary><sup>Aubry1995 - Multi-compartment model of fluid-phase endocytosis kinetics in Dictyostelium discoideum</sup></details>|<details><summary>pass</summary>[Aubry1995.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000986#Files)</details>|pass|<details><summary>pass</summary>[Aubry1995.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000986#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Aubry1995.sedml868042.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Aubry1995.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (99505). The following is the first warning at line 666, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'FITC_Dextran_max_uptake_rate * (1 - exp(-lambda_single_compartment_model_FITC_Dextran * time))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 484, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'FITC_Dextran_max_uptake_rate' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Aubry1995.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Aubry1995.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (99505). The following is the first warning at line 666, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'FITC_Dextran_max_uptake_rate * (1 - exp(-lambda_single_compartment_model_FITC_Dextran * time))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 484, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'FITC_Dextran_max_uptake_rate' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Aubry1995.sedml868042.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Aubry1995.sedml` is invalid.\n - Data generator `FITC_Dextran_single_compartment_model_1_task1` is invalid.\n - Variable `p1_FITC_Dextran_single_compartment_model_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FITC_Dextran_single_compartment_model']` are not defined. No namespaces are defined for the target.\n - Data generator `Pyranine_single_compartment_model_2_task1` is invalid.\n - Variable `p1_Pyranine_single_compartment_model_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Pyranine_single_compartment_model']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_FITC_Dextran_3_task1` is invalid.\n - Variable `p1_Total_FITC_Dextran_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_FITC_Dextran']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Pyranine_4_task1` is invalid.\n - Variable `p1_Total_Pyranine_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Pyranine']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Aubry1995.sedml62713.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Aubry1995.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (99505). The following is the first warning at line 666, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'FITC_Dextran_max_uptake_rate * (1 - exp(-lambda_single_compartment_model_FITC_Dextran * time))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 484, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'FITC_Dextran_max_uptake_rate' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Aubry1995.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Aubry1995.xml` has warnings.\r\n - 13 warnings of type SBML unit consistency (99505). The following is the first warning at line 666, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'FITC_Dextran_max_uptake_rate * (1 - exp(-lambda_single_compartment_model_FITC_Dextran * time))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 12 warnings of type Modeling practice (80701). The following is the first warning at line 484, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'FITC_Dextran_max_uptake_rate' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Aubry1995.sedml62713.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Aubry1995.sedml` is invalid.\n - Data generator `FITC_Dextran_single_compartment_model_1_task1` is invalid.\n - Variable `p1_FITC_Dextran_single_compartment_model_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FITC_Dextran_single_compartment_model']` are not defined. No namespaces are defined for the target.\n - Data generator `Pyranine_single_compartment_model_2_task1` is invalid.\n - Variable `p1_Pyranine_single_compartment_model_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Pyranine_single_compartment_model']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_FITC_Dextran_3_task1` is invalid.\n - Variable `p1_Total_FITC_Dextran_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_FITC_Dextran']` are not defined. No namespaces are defined for the target.\n - Data generator `Total_Pyranine_4_task1` is invalid.\n - Variable `p1_Total_Pyranine_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_Pyranine']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000987](https://www.ebi.ac.uk/biomodels/BIOMD0000000987)<br/><sup>Aubry1995 - Nine-compartment m</sup></summary><sup>Aubry1995 - Nine-compartment model of fluid-phase endocytosis kinetics in Dictyostelium discoideum</sup></details>|<details><summary>pass</summary>[Aubry1995.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000987#Files)</details>|pass|<details><summary>pass</summary>[Aubry1995.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000987#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Aubry1995.sedml402761.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Aubry1995.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 737, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Dictyostelium_discoideum_AX2 * lambda_compartment_1 * FITC_Dextran_three_compartment_model_compartment_1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 518, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lambda_compartment_1' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Aubry1995.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Aubry1995.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 737, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Dictyostelium_discoideum_AX2 * lambda_compartment_1 * FITC_Dextran_three_compartment_model_compartment_1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 518, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lambda_compartment_1' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Aubry1995.sedml402761.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Aubry1995.sedml` is invalid.\n - Data generator `Total_FITC_Dextran_1_task1` is invalid.\n - Variable `p1_Total_FITC_Dextran_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_FITC_Dextran']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Aubry1995.sedml951962.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Aubry1995.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 737, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Dictyostelium_discoideum_AX2 * lambda_compartment_1 * FITC_Dextran_three_compartment_model_compartment_1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 518, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lambda_compartment_1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Aubry1995.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Aubry1995.xml` has warnings.\r\n - 10 warnings of type SBML unit consistency (99505). The following is the first warning at line 737, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Dictyostelium_discoideum_AX2 * lambda_compartment_1 * FITC_Dextran_three_compartment_model_compartment_1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 518, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'lambda_compartment_1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Aubry1995.sedml951962.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Aubry1995.sedml` is invalid.\n - Data generator `Total_FITC_Dextran_1_task1` is invalid.\n - Variable `p1_Total_FITC_Dextran_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Total_FITC_Dextran']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000988](https://www.ebi.ac.uk/biomodels/BIOMD0000000988)<br/><sup>Westerhoff2020 - systems biolo</sup></summary><sup>Westerhoff2020 - systems biology model of the coronavirus pandemic 2020</sup></details>|<details><summary>FAIL</summary>[Westerhoff2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000988#Files)</details>|FAIL|<details><summary>pass</summary>[Westerhoff2020.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000988#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Westerhoff2020.sedml194758.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/Westerhoff2020.sedml194758.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Westerhoff2020.sedml` is invalid.\n - Model `model` is invalid.\n - The model file `Westerhoff2020.xml` is invalid.\n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout' cannot be used in this document.\n \n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout_L2' cannot be used in this document.\n \n - General SBML conformance (10102) at line 3, column 2: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L2V4 Section 4.1\n Element 'listOfLayouts' is not part of the definition of SBML Level 2 Version 4.\n \n - Data generator `dead_corona_perc_0_1_task1` is invalid.\n - Variable `p5_dead_corona_perc_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='dead_corona_perc_0']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Westerhoff2020.sedml852028.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Westerhoff2020.sedml852028.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Westerhoff2020.sedml` is invalid.\n - Model `model` is invalid.\n - The model file `Westerhoff2020.xml` is invalid.\n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout' cannot be used in this document.\n \n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout_L2' cannot be used in this document.\n \n - General SBML conformance (10102) at line 3, column 2: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L2V4 Section 4.1\n Element 'listOfLayouts' is not part of the definition of SBML Level 2 Version 4.\n \n - Data generator `dead_corona_perc_0_1_task1` is invalid.\n - Variable `p5_dead_corona_perc_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='dead_corona_perc_0']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000989](https://www.ebi.ac.uk/biomodels/BIOMD0000000989)<br/><sup>Strasen2018 - TGFb SMAD Signal</sup></summary><sup>Strasen2018 - TGFb SMAD Signalling - Dose dependent dynamics upon TGFb stimulation</sup></details>|<details><summary>pass</summary>[MODEL1712050001.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000989#Files)</details>|FAIL|<details><summary>FAIL</summary>[MODEL1712050001.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000989#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/MODEL1712050001.sedml321170.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1712050001.sedml` has warnings.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/MODEL1712050001.sedml321170.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1712050001.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmp8birxk2o/./model1.xml` is not a file.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='TGFb_LIGAND_Dose']` are not defined. No namespaces are defined for the target.\n - Data generator `NUC_CYT_SMAD2_1_task1` is invalid.\n - Variable `p1_NUC_CYT_SMAD2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='NUC_CYT_SMAD2']` are not defined. No namespaces are defined for the target.\n - Data generator `NUC_CYT_SMAD2_1_task2` is invalid.\n - Variable `p1_NUC_CYT_SMAD2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='NUC_CYT_SMAD2']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1712050001.sedml54902.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1712050001.sedml` has warnings.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1712050001.sedml54902.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1712050001.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmptshpikxo/./model1.xml` is not a file.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='TGFb_LIGAND_Dose']` are not defined. No namespaces are defined for the target.\n - Data generator `NUC_CYT_SMAD2_1_task1` is invalid.\n - Variable `p1_NUC_CYT_SMAD2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='NUC_CYT_SMAD2']` are not defined. No namespaces are defined for the target.\n - Data generator `NUC_CYT_SMAD2_1_task2` is invalid.\n - Variable `p1_NUC_CYT_SMAD2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='NUC_CYT_SMAD2']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000990](https://www.ebi.ac.uk/biomodels/BIOMD0000000990)<br/><sup>Strasen2018 - TGFb SMAD Signal</sup></summary><sup>Strasen2018 - TGFb SMAD Signalling - Degradation of 25pM ligand (TGFb)</sup></details>|<details><summary>pass</summary>[MODEL1712050002.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000990#Files)</details>|FAIL|<details><summary>FAIL</summary>[MODEL1712050002.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000990#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/MODEL1712050002.sedml399136.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1712050002.sedml` has warnings.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/MODEL1712050002.sedml399136.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1712050002.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmp7_78bms8/./model1.xml` is not a file.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='TGFb_LIGAND_Dose']` are not defined. No namespaces are defined for the target.\n - Data generator `TGFb_1_task1` is invalid.\n - Variable `p1_TGFb_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='TGFb']` are not defined. No namespaces are defined for the target.\n - Data generator `TGFb_1_task2` is invalid.\n - Variable `p1_TGFb_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='TGFb']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1712050002.sedml783882.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1712050002.sedml` has warnings.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1712050002.sedml783882.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1712050002.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmpofo31nkl/./model1.xml` is not a file.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='TGFb_LIGAND_Dose']` are not defined. No namespaces are defined for the target.\n - Data generator `TGFb_1_task1` is invalid.\n - Variable `p1_TGFb_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='TGFb']` are not defined. No namespaces are defined for the target.\n - Data generator `TGFb_1_task2` is invalid.\n - Variable `p1_TGFb_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='TGFb']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000991](https://www.ebi.ac.uk/biomodels/BIOMD0000000991)<br/><sup>Okuonghae2020 - SEAIR model of</sup></summary><sup>Okuonghae2020 - SEAIR model of COVID-19 transmission in Lagos, Nigeria</sup></details>|<details><summary>pass</summary>[Okuonghae2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000991#Files)</details>|FAIL|<details><summary>pass</summary>[Okuonghae2020.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000991#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Okuonghae2020.sedml750255.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Okuonghae2020.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 990, column 6:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'alpha * asymptomatic + symptomatic' in the math element of the <assignmentRule> can only act on variables with the same units.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 926, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'exposed_start' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 1176, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'exposed_to_asymptopmatic') are second (exponent = -2, multiplier = 86400, scale = 0), item (exponent = 1, multiplier = 1, scale = 0).\r\n - 7 warnings of type Modeling practice (80701). The following is the first warning at line 583, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'transmission_rate_effective' does not have a 'units' attribute.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - `task2`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task2`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Okuonghae2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Okuonghae2020.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 990, column 6:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'alpha * asymptomatic + symptomatic' in the math element of the <assignmentRule> can only act on variables with the same units.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 926, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'exposed_start' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 1176, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'exposed_to_asymptopmatic') are second (exponent = -2, multiplier = 86400, scale = 0), item (exponent = 1, multiplier = 1, scale = 0).\r\n - 7 warnings of type Modeling practice (80701). The following is the first warning at line 583, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'transmission_rate_effective' does not have a 'units' attribute.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - `task2`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task2`\x1b[0m\r\n\r\n`/root/in/Okuonghae2020.sedml750255.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Okuonghae2020.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='eps']` are not defined. No namespaces are defined for the target.\n - Output `plot_1_task1` is invalid.\n - Plot must have at least one curve.\n - Output `plot_2_task1` is invalid.\n - Plot must have at least one curve.\n - Output `plot_3_task2` is invalid.\n - Plot must have at least one curve.\n - Output `plot_4_task2` is invalid.\n - Plot must have at least one curve.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Okuonghae2020.sedml275811.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Okuonghae2020.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 990, column 6:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'alpha * asymptomatic + symptomatic' in the math element of the <assignmentRule> can only act on variables with the same units.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 926, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'exposed_start' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 1176, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'exposed_to_asymptopmatic') are second (exponent = -2, multiplier = 86400, scale = 0), item (exponent = 1, multiplier = 1, scale = 0).\r\n - 7 warnings of type Modeling practice (80701). The following is the first warning at line 583, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'transmission_rate_effective' does not have a 'units' attribute.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - `task2`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task2`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Okuonghae2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Okuonghae2020.xml` has warnings.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 990, column 6:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'alpha * asymptomatic + symptomatic' in the math element of the <assignmentRule> can only act on variables with the same units.\r\n - 9 warnings of type SBML unit consistency (99505). The following is the first warning at line 926, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'exposed_start' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 1 warning of type SBML unit consistency (10541). The following is the first warning at line 1176, column 8:\r\n - The units of the 'math' formula in a <kineticLaw> definition are expected to be the equivalent of _substance per time_.\r\n Reference: L2V4 Section 4.13.5\r\n Expected units are item (exponent = 1, multiplier = 1, scale = 0), second (exponent = -1, multiplier = 86400, scale = 0) but the units returned by the <math> expression in the <kineticLaw> (from the <reaction> with id 'exposed_to_asymptopmatic') are second (exponent = -2, multiplier = 86400, scale = 0), item (exponent = 1, multiplier = 1, scale = 0).\r\n - 7 warnings of type Modeling practice (80701). The following is the first warning at line 583, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'transmission_rate_effective' does not have a 'units' attribute.\r\n - The following tasks do not contribute to outputs:\r\n - `task1`\r\n - `task2`\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task2`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Okuonghae2020.sedml275811.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Okuonghae2020.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='eps']` are not defined. No namespaces are defined for the target.\n - Output `plot_1_task1` is invalid.\n - Plot must have at least one curve.\n - Output `plot_2_task1` is invalid.\n - Plot must have at least one curve.\n - Output `plot_3_task2` is invalid.\n - Plot must have at least one curve.\n - Output `plot_4_task2` is invalid.\n - Plot must have at least one curve.\n"```</details>|
|<details><summary>[BIOMD0000000994](https://www.ebi.ac.uk/biomodels/BIOMD0000000994)<br/><sup>Strasen2018 - TGFb SMAD Signal</sup></summary><sup>Strasen2018 - TGFb SMAD Signalling - Restimulation with 5pM TGFb at 3hr</sup></details>|<details><summary>pass</summary>[MODEL1712050003.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000994#Files)</details>|FAIL|<details><summary>pass</summary>[MODEL1712050003.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000994#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1712050003.sedml857208.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050003.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5897, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 92 warnings of type SBML unit consistency (99505). The following is the first warning at line 3320, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed * k_in_1) / (k_in_1 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed * k_in_1 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 52 warnings of type Modeling practice (80701). The following is the first warning at line 2538, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K_mran' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1712050003.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050003.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5897, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 92 warnings of type SBML unit consistency (99505). The following is the first warning at line 3320, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed * k_in_1) / (k_in_1 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed * k_in_1 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 52 warnings of type Modeling practice (80701). The following is the first warning at line 2538, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K_mran' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/MODEL1712050003.sedml857208.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1712050003.sedml` is invalid.\n - Data generator `NUC_CYT_SMAD2_1_task1` is invalid.\n - Variable `p1_NUC_CYT_SMAD2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='NUC_CYT_SMAD2']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1712050003.sedml654643.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050003.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5897, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 92 warnings of type SBML unit consistency (99505). The following is the first warning at line 3320, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed * k_in_1) / (k_in_1 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed * k_in_1 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 52 warnings of type Modeling practice (80701). The following is the first warning at line 2538, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K_mran' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1712050003.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050003.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5897, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 92 warnings of type SBML unit consistency (99505). The following is the first warning at line 3320, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed * k_in_1) / (k_in_1 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed * k_in_1 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 52 warnings of type Modeling practice (80701). The following is the first warning at line 2538, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K_mran' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1712050003.sedml654643.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1712050003.sedml` is invalid.\n - Data generator `NUC_CYT_SMAD2_1_task1` is invalid.\n - Variable `p1_NUC_CYT_SMAD2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='NUC_CYT_SMAD2']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000995](https://www.ebi.ac.uk/biomodels/BIOMD0000000995)<br/><sup>Strasen2018 - TGFb SMAD Signal</sup></summary><sup>Strasen2018 - TGFb SMAD Signalling - Restimulation with 5pM TGFb at 8hr</sup></details>|<details><summary>pass</summary>[MODEL1712050004.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000995#Files)</details>|FAIL|<details><summary>pass</summary>[MODEL1712050004.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000995#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1712050004.sedml262711.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050004.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5888, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 92 warnings of type SBML unit consistency (99505). The following is the first warning at line 3318, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed * k_in_1) / (k_in_1 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed * k_in_1 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 52 warnings of type Modeling practice (80701). The following is the first warning at line 2536, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K_mran' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1712050004.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050004.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5888, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 92 warnings of type SBML unit consistency (99505). The following is the first warning at line 3318, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed * k_in_1) / (k_in_1 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed * k_in_1 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 52 warnings of type Modeling practice (80701). The following is the first warning at line 2536, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K_mran' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/MODEL1712050004.sedml262711.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1712050004.sedml` is invalid.\n - Data generator `NUC_CYT_SMAD2_1_task1` is invalid.\n - Variable `p1_NUC_CYT_SMAD2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='NUC_CYT_SMAD2']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1712050004.sedml29319.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050004.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5888, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 92 warnings of type SBML unit consistency (99505). The following is the first warning at line 3318, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed * k_in_1) / (k_in_1 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed * k_in_1 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 52 warnings of type Modeling practice (80701). The following is the first warning at line 2536, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K_mran' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1712050004.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050004.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5888, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 92 warnings of type SBML unit consistency (99505). The following is the first warning at line 3318, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed * k_in_1) / (k_in_1 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed * k_in_1 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 52 warnings of type Modeling practice (80701). The following is the first warning at line 2536, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K_mran' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1712050004.sedml29319.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1712050004.sedml` is invalid.\n - Data generator `NUC_CYT_SMAD2_1_task1` is invalid.\n - Variable `p1_NUC_CYT_SMAD2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='NUC_CYT_SMAD2']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000996](https://www.ebi.ac.uk/biomodels/BIOMD0000000996)<br/><sup>Strasen2018 - TGFb SMAD Signal</sup></summary><sup>Strasen2018 - TGFb SMAD Signalling - Restimulation with 100pM TGFb at 6hr</sup></details>|<details><summary>pass</summary>[MODEL1712050005.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000996#Files)</details>|FAIL|<details><summary>pass</summary>[MODEL1712050005.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000996#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1712050005.sedml407171.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050005.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5888, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 92 warnings of type SBML unit consistency (99505). The following is the first warning at line 3318, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed * k_in_1) / (k_in_1 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed * k_in_1 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 52 warnings of type Modeling practice (80701). The following is the first warning at line 2536, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K_mran' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1712050005.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050005.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5888, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 92 warnings of type SBML unit consistency (99505). The following is the first warning at line 3318, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed * k_in_1) / (k_in_1 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed * k_in_1 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 52 warnings of type Modeling practice (80701). The following is the first warning at line 2536, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K_mran' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/MODEL1712050005.sedml407171.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1712050005.sedml` is invalid.\n - Data generator `NUC_CYT_SMAD2_1_task1` is invalid.\n - Variable `p1_NUC_CYT_SMAD2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='NUC_CYT_SMAD2']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1712050005.sedml8402.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050005.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5888, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 92 warnings of type SBML unit consistency (99505). The following is the first warning at line 3318, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed * k_in_1) / (k_in_1 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed * k_in_1 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 52 warnings of type Modeling practice (80701). The following is the first warning at line 2536, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K_mran' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1712050005.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050005.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5888, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 92 warnings of type SBML unit consistency (99505). The following is the first warning at line 3318, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed * k_in_1) / (k_in_1 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed * k_in_1 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 52 warnings of type Modeling practice (80701). The following is the first warning at line 2536, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K_mran' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1712050005.sedml8402.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1712050005.sedml` is invalid.\n - Data generator `NUC_CYT_SMAD2_1_task1` is invalid.\n - Variable `p1_NUC_CYT_SMAD2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='NUC_CYT_SMAD2']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000997](https://www.ebi.ac.uk/biomodels/BIOMD0000000997)<br/><sup>Strasen2018 - TGFb SMAD Signal</sup></summary><sup>Strasen2018 - TGFb SMAD Signalling - DRB treatment</sup></details>|<details><summary>pass</summary>[MODEL1712050006.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000997#Files)</details>|FAIL|<details><summary>pass</summary>[MODEL1712050006.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000997#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1712050006.sedml952082.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050006.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5876, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 3311, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed * k_in_1) / (k_in_1 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed * k_in_1 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 51 warnings of type Modeling practice (80701). The following is the first warning at line 2544, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K_mran' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1712050006.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050006.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5876, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 3311, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed * k_in_1) / (k_in_1 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed * k_in_1 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 51 warnings of type Modeling practice (80701). The following is the first warning at line 2544, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K_mran' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/MODEL1712050006.sedml952082.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1712050006.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='DRB']` are not defined. No namespaces are defined for the target.\n - Data generator `NUC_CYT_SMAD2_1_task1` is invalid.\n - Variable `p1_NUC_CYT_SMAD2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='NUC_CYT_SMAD2']` are not defined. No namespaces are defined for the target.\n - Data generator `NUC_CYT_SMAD2_1_task2` is invalid.\n - Variable `p1_NUC_CYT_SMAD2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='NUC_CYT_SMAD2']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1712050006.sedml316096.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050006.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5876, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 3311, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed * k_in_1) / (k_in_1 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed * k_in_1 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 51 warnings of type Modeling practice (80701). The following is the first warning at line 2544, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K_mran' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1712050006.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050006.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5876, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 3311, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed * k_in_1) / (k_in_1 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed * k_in_1 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 51 warnings of type Modeling practice (80701). The following is the first warning at line 2544, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K_mran' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1712050006.sedml316096.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1712050006.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='DRB']` are not defined. No namespaces are defined for the target.\n - Data generator `NUC_CYT_SMAD2_1_task1` is invalid.\n - Variable `p1_NUC_CYT_SMAD2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='NUC_CYT_SMAD2']` are not defined. No namespaces are defined for the target.\n - Data generator `NUC_CYT_SMAD2_1_task2` is invalid.\n - Variable `p1_NUC_CYT_SMAD2_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='NUC_CYT_SMAD2']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000998](https://www.ebi.ac.uk/biomodels/BIOMD0000000998)<br/><sup>Strasen2018 - TGFb SMAD Signal</sup></summary><sup>Strasen2018 - TGFb SMAD Signalling Class 1</sup></details>|<details><summary>pass</summary>[MODEL1712050007.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000998#Files)</details>|FAIL|<details><summary>pass</summary>[MODEL1712050007.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000998#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1712050007.sedml4725.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050007.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5851, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 3258, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total_C1 * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed_C1 * k_in_R1_C1) / (k_in_R1_C1 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed_C1 * k_in_R1_C1 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 48 warnings of type Modeling practice (80701). The following is the first warning at line 2536, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K_mran' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1712050007.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050007.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5851, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 3258, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total_C1 * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed_C1 * k_in_R1_C1) / (k_in_R1_C1 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed_C1 * k_in_R1_C1 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 48 warnings of type Modeling practice (80701). The following is the first warning at line 2536, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K_mran' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/MODEL1712050007.sedml4725.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1712050007.sedml` is invalid.\n - Data generator `NUC_CYT_SMAD2_1_task1` is invalid.\n - Variable `p1_NUC_CYT_SMAD2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='NUC_CYT_SMAD2']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1712050007.sedml821971.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050007.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5851, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 3258, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total_C1 * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed_C1 * k_in_R1_C1) / (k_in_R1_C1 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed_C1 * k_in_R1_C1 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 48 warnings of type Modeling practice (80701). The following is the first warning at line 2536, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K_mran' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1712050007.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050007.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5851, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 3258, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total_C1 * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed_C1 * k_in_R1_C1) / (k_in_R1_C1 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed_C1 * k_in_R1_C1 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 48 warnings of type Modeling practice (80701). The following is the first warning at line 2536, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K_mran' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1712050007.sedml821971.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1712050007.sedml` is invalid.\n - Data generator `NUC_CYT_SMAD2_1_task1` is invalid.\n - Variable `p1_NUC_CYT_SMAD2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='NUC_CYT_SMAD2']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000000999](https://www.ebi.ac.uk/biomodels/BIOMD0000000999)<br/><sup>Strasen2018 - TGFb SMAD Signal</sup></summary><sup>Strasen2018 - TGFb SMAD Signalling Class 2</sup></details>|<details><summary>pass</summary>[MODEL1712050008.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000000999#Files)</details>|FAIL|<details><summary>pass</summary>[MODEL1712050008.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000000999#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1712050008.sedml907253.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050008.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5877, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 3284, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total_C2 * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed_C2 * k_in_R1_C2) / (k_in_R1_C2 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed_C2 * k_in_R1_C2 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 49 warnings of type Modeling practice (80701). The following is the first warning at line 2547, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'DRB' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1712050008.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050008.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5877, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 3284, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total_C2 * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed_C2 * k_in_R1_C2) / (k_in_R1_C2 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed_C2 * k_in_R1_C2 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 49 warnings of type Modeling practice (80701). The following is the first warning at line 2547, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'DRB' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/MODEL1712050008.sedml907253.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1712050008.sedml` is invalid.\n - Data generator `NUC_CYT_SMAD2_1_task1` is invalid.\n - Variable `p1_NUC_CYT_SMAD2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='NUC_CYT_SMAD2']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1712050008.sedml431281.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050008.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5877, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 3284, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total_C2 * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed_C2 * k_in_R1_C2) / (k_in_R1_C2 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed_C2 * k_in_R1_C2 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 49 warnings of type Modeling practice (80701). The following is the first warning at line 2547, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'DRB' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1712050008.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050008.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5877, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 3284, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total_C2 * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed_C2 * k_in_R1_C2) / (k_in_R1_C2 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed_C2 * k_in_R1_C2 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 49 warnings of type Modeling practice (80701). The following is the first warning at line 2547, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'DRB' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1712050008.sedml431281.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1712050008.sedml` is invalid.\n - Data generator `NUC_CYT_SMAD2_1_task1` is invalid.\n - Variable `p1_NUC_CYT_SMAD2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='NUC_CYT_SMAD2']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000001000](https://www.ebi.ac.uk/biomodels/BIOMD0000001000)<br/><sup>Strasen2018 - TGFb SMAD Signal</sup></summary><sup>Strasen2018 - TGFb SMAD Signalling Class 3</sup></details>|<details><summary>pass</summary>[MODEL1712050009.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001000#Files)</details>|FAIL|<details><summary>pass</summary>[MODEL1712050009.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001000#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1712050009.sedml413432.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050009.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5877, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 3284, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total_C3 * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed_C3 * k_in_R1_C3) / (k_in_R1_C3 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed_C3 * k_in_R1_C3 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 49 warnings of type Modeling practice (80701). The following is the first warning at line 2547, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'DRB' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1712050009.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050009.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5877, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 3284, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total_C3 * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed_C3 * k_in_R1_C3) / (k_in_R1_C3 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed_C3 * k_in_R1_C3 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 49 warnings of type Modeling practice (80701). The following is the first warning at line 2547, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'DRB' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/MODEL1712050009.sedml413432.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1712050009.sedml` is invalid.\n - Data generator `NUC_CYT_SMAD2_1_task1` is invalid.\n - Variable `p1_NUC_CYT_SMAD2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='NUC_CYT_SMAD2']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1712050009.sedml673034.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050009.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5877, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 3284, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total_C3 * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed_C3 * k_in_R1_C3) / (k_in_R1_C3 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed_C3 * k_in_R1_C3 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 49 warnings of type Modeling practice (80701). The following is the first warning at line 2547, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'DRB' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1712050009.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050009.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5877, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 3284, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total_C3 * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed_C3 * k_in_R1_C3) / (k_in_R1_C3 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed_C3 * k_in_R1_C3 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 49 warnings of type Modeling practice (80701). The following is the first warning at line 2547, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'DRB' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1712050009.sedml673034.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1712050009.sedml` is invalid.\n - Data generator `NUC_CYT_SMAD2_1_task1` is invalid.\n - Variable `p1_NUC_CYT_SMAD2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='NUC_CYT_SMAD2']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000001001](https://www.ebi.ac.uk/biomodels/BIOMD0000001001)<br/><sup>Strasen2018 - TGFb SMAD Signal</sup></summary><sup>Strasen2018 - TGFb SMAD Signalling Class 4</sup></details>|<details><summary>pass</summary>[MODEL1712050010.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001001#Files)</details>|FAIL|<details><summary>pass</summary>[MODEL1712050010.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001001#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1712050010.sedml965214.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050010.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5877, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 3284, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total_C4 * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed_C4 * k_in_R1_C4) / (k_in_R1_C4 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed_C4 * k_in_R1_C4 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 49 warnings of type Modeling practice (80701). The following is the first warning at line 2547, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'DRB' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1712050010.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050010.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5877, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 3284, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total_C4 * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed_C4 * k_in_R1_C4) / (k_in_R1_C4 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed_C4 * k_in_R1_C4 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 49 warnings of type Modeling practice (80701). The following is the first warning at line 2547, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'DRB' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/MODEL1712050010.sedml965214.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1712050010.sedml` is invalid.\n - Data generator `NUC_CYT_SMAD2_1_task1` is invalid.\n - Variable `p1_NUC_CYT_SMAD2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='NUC_CYT_SMAD2']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1712050010.sedml357570.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050010.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5877, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 3284, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total_C4 * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed_C4 * k_in_R1_C4) / (k_in_R1_C4 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed_C4 * k_in_R1_C4 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 49 warnings of type Modeling practice (80701). The following is the first warning at line 2547, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'DRB' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1712050010.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050010.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5877, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 3284, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total_C4 * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed_C4 * k_in_R1_C4) / (k_in_R1_C4 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed_C4 * k_in_R1_C4 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 49 warnings of type Modeling practice (80701). The following is the first warning at line 2547, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'DRB' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1712050010.sedml357570.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1712050010.sedml` is invalid.\n - Data generator `NUC_CYT_SMAD2_1_task1` is invalid.\n - Variable `p1_NUC_CYT_SMAD2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='NUC_CYT_SMAD2']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000001002](https://www.ebi.ac.uk/biomodels/BIOMD0000001002)<br/><sup>Strasen2018 - TGFb SMAD Signal</sup></summary><sup>Strasen2018 - TGFb SMAD Signalling Class 5</sup></details>|<details><summary>pass</summary>[MODEL1712050011.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001002#Files)</details>|FAIL|<details><summary>pass</summary>[MODEL1712050011.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001002#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL1712050011.sedml752145.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050011.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5858, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 3265, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total_C5 * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed_C5 * k_in_R1_C5) / (k_in_R1_C5 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed_C5 * k_in_R1_C5 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 48 warnings of type Modeling practice (80701). The following is the first warning at line 2543, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K_mran' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1712050011.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050011.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5858, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 3265, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total_C5 * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed_C5 * k_in_R1_C5) / (k_in_R1_C5 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed_C5 * k_in_R1_C5 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 48 warnings of type Modeling practice (80701). The following is the first warning at line 2543, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K_mran' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/MODEL1712050011.sedml752145.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1712050011.sedml` is invalid.\n - Data generator `NUC_CYT_SMAD2_1_task1` is invalid.\n - Variable `p1_NUC_CYT_SMAD2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='NUC_CYT_SMAD2']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL1712050011.sedml924950.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050011.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5858, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 3265, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total_C5 * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed_C5 * k_in_R1_C5) / (k_in_R1_C5 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed_C5 * k_in_R1_C5 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 48 warnings of type Modeling practice (80701). The following is the first warning at line 2543, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K_mran' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL1712050011.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `MODEL1712050011.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5858, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 3265, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total_C5 * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed_C5 * k_in_R1_C5) / (k_in_R1_C5 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed_C5 * k_in_R1_C5 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 48 warnings of type Modeling practice (80701). The following is the first warning at line 2543, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'K_mran' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL1712050011.sedml924950.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL1712050011.sedml` is invalid.\n - Data generator `NUC_CYT_SMAD2_1_task1` is invalid.\n - Variable `p1_NUC_CYT_SMAD2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='NUC_CYT_SMAD2']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000001003](https://www.ebi.ac.uk/biomodels/BIOMD0000001003)<br/><sup>Strasen2018 - TGFb SMAD Signal</sup></summary><sup>Strasen2018 - TGFb SMAD Signalling Class 6</sup></details>|<details><summary>pass</summary>[MODEL17120500012.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001003#Files)</details>|FAIL|<details><summary>pass</summary>[MODEL17120500012.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001003#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/MODEL17120500012.sedml48098.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `MODEL17120500012.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5877, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 3284, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total_C6 * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed_C6 * k_in_R1_C6) / (k_in_R1_C6 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed_C6 * k_in_R1_C6 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 49 warnings of type Modeling practice (80701). The following is the first warning at line 2547, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'DRB' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL17120500012.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `MODEL17120500012.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5877, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 3284, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total_C6 * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed_C6 * k_in_R1_C6) / (k_in_R1_C6 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed_C6 * k_in_R1_C6 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 49 warnings of type Modeling practice (80701). The following is the first warning at line 2547, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'DRB' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/MODEL17120500012.sedml48098.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL17120500012.sedml` is invalid.\n - Data generator `NUC_CYT_SMAD2_1_task1` is invalid.\n - Variable `p1_NUC_CYT_SMAD2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='NUC_CYT_SMAD2']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/MODEL17120500012.sedml521643.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `MODEL17120500012.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5877, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 3284, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total_C6 * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed_C6 * k_in_R1_C6) / (k_in_R1_C6 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed_C6 * k_in_R1_C6 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 49 warnings of type Modeling practice (80701). The following is the first warning at line 2547, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'DRB' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./MODEL17120500012.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `MODEL17120500012.xml` has warnings.\r\n - 2 warnings of type SBML unit consistency (10501). The following is the first warning at line 5877, column 8:\r\n - The formula 'pow(pS2_pS2_S4_n, hill_fact1)' in the math element of the <kineticLaw> contains a power that is not an integer and thus may produce invalid units.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 3284, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'R1_total_C6 * kdeg_R1 * (kdeg_R1 + index_k_out_1_relative_speed_C6 * k_in_R1_C6) / (k_in_R1_C6 * kdeg_R1 + pow(kdeg_R1, 2) + index_k_out_1_relative_speed_C6 * k_in_R1_C6 * kdeg_R1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 49 warnings of type Modeling practice (80701). The following is the first warning at line 2547, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'DRB' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/MODEL17120500012.sedml521643.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./MODEL17120500012.sedml` is invalid.\n - Data generator `NUC_CYT_SMAD2_1_task1` is invalid.\n - Variable `p1_NUC_CYT_SMAD2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='NUC_CYT_SMAD2']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000001004](https://www.ebi.ac.uk/biomodels/BIOMD0000001004)<br/><sup>Intosalmi2015 - Th17 core netw</sup></summary><sup>Intosalmi2015 - Th17 core network model</sup></details>|<details><summary>pass</summary>[Intosalmi2015.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001004#Files)</details>|pass|<details><summary>FAIL</summary>[Intosalmi2015.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001004#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Intosalmi2015.sedml52366.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/Intosalmi2015.sedml52366.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Intosalmi2015.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmpi8incrv4/./model1.xml` is not a file.\n - Data generator `IL6ext_1_task1` is invalid.\n - Variable `p1_IL6ext_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IL6ext']` are not defined. No namespaces are defined for the target.\n - Data generator `IL6int_2_task1` is invalid.\n - Variable `p1_IL6int_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IL6int']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT3mRNA_3_task1` is invalid.\n - Variable `p1_STAT3mRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT3mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT3prot_star_4_task1` is invalid.\n - Variable `p1_STAT3prot_star_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT3prot_star']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT3prot_5_task1` is invalid.\n - Variable `p1_STAT3prot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT3prot']` are not defined. No namespaces are defined for the target.\n - Data generator `TGFbext_6_task1` is invalid.\n - Variable `p1_TGFbext_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='TGFbext']` are not defined. No namespaces are defined for the target.\n - Data generator `TGFbint_7_task1` is invalid.\n - Variable `p1_TGFbint_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='TGFbint']` are not defined. No namespaces are defined for the target.\n - Data generator `RORgtmRNA_8_task1` is invalid.\n - Variable `p1_RORgtmRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='RORgtmRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `FOXP3prot_star_9_task1` is invalid.\n - Variable `p1_FOXP3prot_star_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FOXP3prot_star']` are not defined. No namespaces are defined for the target.\n - Data generator `FOXP3mRNA_10_task1` is invalid.\n - Variable `p1_FOXP3mRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FOXP3mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT3mRNA_1_task1` is invalid.\n - Variable `p2_STAT3mRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT3mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `RORgtmRNA_1_task1` is invalid.\n - Variable `p3_RORgtmRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='RORgtmRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `FOXP3mRNA_1_task1` is invalid.\n - Variable `p4_FOXP3mRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FOXP3mRNA']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Intosalmi2015.sedml859037.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Intosalmi2015.sedml859037.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Intosalmi2015.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmplw3yoaq_/./model1.xml` is not a file.\n - Data generator `IL6ext_1_task1` is invalid.\n - Variable `p1_IL6ext_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IL6ext']` are not defined. No namespaces are defined for the target.\n - Data generator `IL6int_2_task1` is invalid.\n - Variable `p1_IL6int_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='IL6int']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT3mRNA_3_task1` is invalid.\n - Variable `p1_STAT3mRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT3mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT3prot_star_4_task1` is invalid.\n - Variable `p1_STAT3prot_star_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT3prot_star']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT3prot_5_task1` is invalid.\n - Variable `p1_STAT3prot_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT3prot']` are not defined. No namespaces are defined for the target.\n - Data generator `TGFbext_6_task1` is invalid.\n - Variable `p1_TGFbext_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='TGFbext']` are not defined. No namespaces are defined for the target.\n - Data generator `TGFbint_7_task1` is invalid.\n - Variable `p1_TGFbint_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='TGFbint']` are not defined. No namespaces are defined for the target.\n - Data generator `RORgtmRNA_8_task1` is invalid.\n - Variable `p1_RORgtmRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='RORgtmRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `FOXP3prot_star_9_task1` is invalid.\n - Variable `p1_FOXP3prot_star_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FOXP3prot_star']` are not defined. No namespaces are defined for the target.\n - Data generator `FOXP3mRNA_10_task1` is invalid.\n - Variable `p1_FOXP3mRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FOXP3mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `STAT3mRNA_1_task1` is invalid.\n - Variable `p2_STAT3mRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='STAT3mRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `RORgtmRNA_1_task1` is invalid.\n - Variable `p3_RORgtmRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='RORgtmRNA']` are not defined. No namespaces are defined for the target.\n - Data generator `FOXP3mRNA_1_task1` is invalid.\n - Variable `p4_FOXP3mRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FOXP3mRNA']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000001005](https://www.ebi.ac.uk/biomodels/BIOMD0000001005)<br/><sup>Bae2017 - Mathematical analysi</sup></summary><sup>Bae2017 - Mathematical analysis of circadian disruption and metabolic re-entrainment of hepatic gluconeogenesis</sup></details>|<details><summary>pass</summary>[Bae2018.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001005#Files)</details>|FAIL|<details><summary>pass</summary>[Bae2018.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001005#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Bae2018.sedml702487.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Bae2018.xml` has warnings.\r\n - 2 warnings of type SBO term consistency (10703). The following is the first warning at line 3754, column 6:\r\n - The value of the \'sboTerm\' attribute on a <parameter> is expected to be an SBO identifier (http://www.biomodels.net/SBO/) referring to a quantitative parameter defined in SBO (i.e., terms derived from SBO:0000002, "quantitative systems description parameter"). \r\n Reference: L2V4 Sections 4.9.5 and 5\r\n SBO term \'SBO:0000475\' on the <parameter> is not in the appropriate branch.\r\n - 7 warnings of type SBO term consistency (10707). The following is the first warning at line 4964, column 6:\r\n - The value of the \'sboTerm\' attribute on a <reaction> is expected to be an SBO identifier (http://www.biomodels.net/SBO/) referring to an occurring entity representation defined in SBO (i.e., terms derived from SBO:0000231, "occurring entity representation"). \r\n Reference: L2V4 Sections 4.13.1 and 5\r\n SBO term \'SBO:0000560\' on the <reaction> is not in the appropriate branch.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 5932, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula \'Feed - feed2\' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 81 warnings of type SBML unit consistency (99505). The following is the first warning at line 3812, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression \'piecewise(1, and(geq(piecewise(time - 24 * ceil(time / 24), xor(lt(time, 0), lt(24, 0)), time - 24 * floor(time / 24)), 6), leq(piecewise(time - 24 * ceil(time / 24), xor(lt(time, 0), lt(24, 0)), time - 24 * floor(time / 24)), 18)), 0)\' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 131 warnings of type Modeling practice (80701). The following is the first warning at line 4111, column 12:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id \'KP1\' does not have a \'units\' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 456.0\r\n - Output end time: 480.0\r\n - Number of steps: 96\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Bae2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Bae2018.xml` has warnings.\r\n - 2 warnings of type SBO term consistency (10703). The following is the first warning at line 3754, column 6:\r\n - The value of the \'sboTerm\' attribute on a <parameter> is expected to be an SBO identifier (http://www.biomodels.net/SBO/) referring to a quantitative parameter defined in SBO (i.e., terms derived from SBO:0000002, "quantitative systems description parameter"). \r\n Reference: L2V4 Sections 4.9.5 and 5\r\n SBO term \'SBO:0000475\' on the <parameter> is not in the appropriate branch.\r\n - 7 warnings of type SBO term consistency (10707). The following is the first warning at line 4964, column 6:\r\n - The value of the \'sboTerm\' attribute on a <reaction> is expected to be an SBO identifier (http://www.biomodels.net/SBO/) referring to an occurring entity representation defined in SBO (i.e., terms derived from SBO:0000231, "occurring entity representation"). \r\n Reference: L2V4 Sections 4.13.1 and 5\r\n SBO term \'SBO:0000560\' on the <reaction> is not in the appropriate branch.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 5932, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula \'Feed - feed2\' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 81 warnings of type SBML unit consistency (99505). The following is the first warning at line 3812, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression \'piecewise(1, and(geq(piecewise(time - 24 * ceil(time / 24), xor(lt(time, 0), lt(24, 0)), time - 24 * floor(time / 24)), 6), leq(piecewise(time - 24 * ceil(time / 24), xor(lt(time, 0), lt(24, 0)), time - 24 * floor(time / 24)), 18)), 0)\' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 131 warnings of type Modeling practice (80701). The following is the first warning at line 4111, column 12:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id \'KP1\' does not have a \'units\' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 456.0\r\n - Output end time: 480.0\r\n - Number of steps: 96\x1b[0m\r\n\r\n`/root/in/Bae2018.sedml702487.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Bae2018.sedml` is invalid.\n - Data generator `CRH_1_task1` is invalid.\n - Variable `p1_CRH_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'CRH\']` are not defined. No namespaces are defined for the target.\n - Data generator `ACTH_2_task1` is invalid.\n - Variable `p1_ACTH_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'ACTH\']` are not defined. No namespaces are defined for the target.\n - Data generator `F_HPA_3_task1` is invalid.\n - Variable `p1_F_HPA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'F_HPA\']` are not defined. No namespaces are defined for the target.\n - Data generator `F_Cell_4_task1` is invalid.\n - Variable `p1_F_Cell_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'F_Cell\']` are not defined. No namespaces are defined for the target.\n - Data generator `y1_5_task1` is invalid.\n - Variable `p1_y1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'y1\']` are not defined. No namespaces are defined for the target.\n - Data generator `y2_6_task1` is invalid.\n - Variable `p1_y2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'y2\']` are not defined. No namespaces are defined for the target.\n - Data generator `y3_7_task1` is invalid.\n - Variable `p1_y3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'y3\']` are not defined. No namespaces are defined for the target.\n - Data generator `y4_8_task1` is invalid.\n - Variable `p1_y4_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'y4\']` are not defined. No namespaces are defined for the target.\n - Data generator `y5_9_task1` is invalid.\n - Variable `p1_y5_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'y5\']` are not defined. No namespaces are defined for the target.\n - Data generator `y6_10_task1` is invalid.\n - Variable `p1_y6_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'y6\']` are not defined. No namespaces are defined for the target.\n - Data generator `y7_11_task1` is invalid.\n - Variable `p1_y7_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'y7\']` are not defined. No namespaces are defined for the target.\n - Data generator `NAD_12_task1` is invalid.\n - Variable `p1_NAD_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'NAD\']` are not defined. No namespaces are defined for the target.\n - Data generator `NMN_13_task1` is invalid.\n - Variable `p1_NMN_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'NMN\']` are not defined. No namespaces are defined for the target.\n - Data generator `NAM_14_task1` is invalid.\n - Variable `p1_NAM_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'NAM\']` are not defined. No namespaces are defined for the target.\n - Data generator `EntF_15_task1` is invalid.\n - Variable `p1_EntF_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'EntF\']` are not defined. No namespaces are defined for the target.\n - Data generator `SIRT1_16_task1` is invalid.\n - Variable `p1_SIRT1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'SIRT1\']` are not defined. No namespaces are defined for the target.\n - Data generator `y8_17_task1` is invalid.\n - Variable `p1_y8_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'y8\']` are not defined. No namespaces are defined for the target.\n - Data generator `NAMPT_18_task1` is invalid.\n - Variable `p1_NAMPT_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'NAMPT\']` are not defined. No namespaces are defined for the target.\n - Data generator `feed2_19_task1` is invalid.\n - Variable `p1_feed2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'feed2\']` are not defined. No namespaces are defined for the target.\n - Data generator `feed3_20_task1` is invalid.\n - Variable `p1_feed3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'feed3\']` are not defined. No namespaces are defined for the target.\n - Data generator `PGC1amRNA_21_task1` is invalid.\n - Variable `p1_PGC1amRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'PGC1amRNA\']` are not defined. No namespaces are defined for the target.\n - Data generator `PGC1a_22_task1` is invalid.\n - Variable `p1_PGC1a_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'PGC1a\']` are not defined. No namespaces are defined for the target.\n - Data generator `PGC1aN_23_task1` is invalid.\n - Variable `p1_PGC1aN_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'PGC1aN\']` are not defined. No namespaces are defined for the target.\n - Data generator `PGC1aNact_24_task1` is invalid.\n - Variable `p1_PGC1aNact_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'PGC1aNact\']` are not defined. No namespaces are defined for the target.\n - Data generator `FOXO1_25_task1` is invalid.\n - Variable `p1_FOXO1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'FOXO1\']` are not defined. No namespaces are defined for the target.\n - Data generator `PckG6p_26_task1` is invalid.\n - Variable `p1_PckG6p_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'PckG6p\']` are not defined. No namespaces are defined for the target.\n - Data generator `Phpa_27_task1` is invalid.\n - Variable `p1_Phpa_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'Phpa\']` are not defined. No namespaces are defined for the target.\n - Data generator `MR_28_task1` is invalid.\n - Variable `p1_MR_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'MR\']` are not defined. No namespaces are defined for the target.\n - Data generator `GR_Cell_29_task1` is invalid.\n - Variable `p1_GR_Cell_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'GR_Cell\']` are not defined. No namespaces are defined for the target.\n - Data generator `FGR_Cell_30_task1` is invalid.\n - Variable `p1_FGR_Cell_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'FGR_Cell\']` are not defined. No namespaces are defined for the target.\n - Data generator `FMR_31_task1` is invalid.\n - Variable `p1_FMR_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'FMR\']` are not defined. No namespaces are defined for the target.\n - Data generator `FMRN_32_task1` is invalid.\n - Variable `p1_FMRN_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'FMRN\']` are not defined. No namespaces are defined for the target.\n - Data generator `FGRN_Cell_33_task1` is invalid.\n - Variable `p1_FGRN_Cell_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'FGRN_Cell\']` are not defined. No namespaces are defined for the target.\n - Data generator `P_34_task1` is invalid.\n - Variable `p1_P_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'P\']` are not defined. No namespaces are defined for the target.\n - Data generator `mRNAP_35_task1` is invalid.\n - Variable `p1_mRNAP_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'mRNAP\']` are not defined. No namespaces are defined for the target.\n - Data generator `mRNARP_36_task1` is invalid.\n - Variable `p1_mRNARP_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'mRNARP\']` are not defined. No namespaces are defined for the target.\n - Data generator `Rp_37_task1` is invalid.\n - Variable `p1_Rp_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'Rp\']` are not defined. No namespaces are defined for the target.\n - Data generator `PR_38_task1` is invalid.\n - Variable `p1_PR_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'PR\']` are not defined. No namespaces are defined for the target.\n - Data generator `GR_HPA_39_task1` is invalid.\n - Variable `p1_GR_HPA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'GR_HPA\']` are not defined. No namespaces are defined for the target.\n - Data generator `FGR_HPA_40_task1` is invalid.\n - Variable `p1_FGR_HPA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'FGR_HPA\']` are not defined. No namespaces are defined for the target.\n - Data generator `FGRN_HPA_41_task1` is invalid.\n - Variable `p1_FGRN_HPA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'FGRN_HPA\']` are not defined. No namespaces are defined for the target.\n - Data generator `L_42_task1` is invalid.\n - Variable `p1_L_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id=\'L\']` are not defined. No namespaces are defined for the target.\n - Data generator `Feed_43_task1` is invalid.\n - Variable `p1_Feed_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id=\'Feed\']` are not defined. No namespaces are defined for the target.\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Bae2018.sedml611274.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Bae2018.xml` has warnings.\r\n - 2 warnings of type SBO term consistency (10703). The following is the first warning at line 3754, column 6:\r\n - The value of the \'sboTerm\' attribute on a <parameter> is expected to be an SBO identifier (http://www.biomodels.net/SBO/) referring to a quantitative parameter defined in SBO (i.e., terms derived from SBO:0000002, "quantitative systems description parameter"). \r\n Reference: L2V4 Sections 4.9.5 and 5\r\n SBO term \'SBO:0000475\' on the <parameter> is not in the appropriate branch.\r\n - 7 warnings of type SBO term consistency (10707). The following is the first warning at line 4964, column 6:\r\n - The value of the \'sboTerm\' attribute on a <reaction> is expected to be an SBO identifier (http://www.biomodels.net/SBO/) referring to an occurring entity representation defined in SBO (i.e., terms derived from SBO:0000231, "occurring entity representation"). \r\n Reference: L2V4 Sections 4.13.1 and 5\r\n SBO term \'SBO:0000560\' on the <reaction> is not in the appropriate branch.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 5932, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula \'Feed - feed2\' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 81 warnings of type SBML unit consistency (99505). The following is the first warning at line 3812, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression \'piecewise(1, and(geq(piecewise(time - 24 * ceil(time / 24), xor(lt(time, 0), lt(24, 0)), time - 24 * floor(time / 24)), 6), leq(piecewise(time - 24 * ceil(time / 24), xor(lt(time, 0), lt(24, 0)), time - 24 * floor(time / 24)), 18)), 0)\' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 131 warnings of type Modeling practice (80701). The following is the first warning at line 4111, column 12:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id \'KP1\' does not have a \'units\' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 456.0\r\n - Output end time: 480.0\r\n - Number of steps: 96\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Bae2018.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Bae2018.xml` has warnings.\r\n - 2 warnings of type SBO term consistency (10703). The following is the first warning at line 3754, column 6:\r\n - The value of the \'sboTerm\' attribute on a <parameter> is expected to be an SBO identifier (http://www.biomodels.net/SBO/) referring to a quantitative parameter defined in SBO (i.e., terms derived from SBO:0000002, "quantitative systems description parameter"). \r\n Reference: L2V4 Sections 4.9.5 and 5\r\n SBO term \'SBO:0000475\' on the <parameter> is not in the appropriate branch.\r\n - 7 warnings of type SBO term consistency (10707). The following is the first warning at line 4964, column 6:\r\n - The value of the \'sboTerm\' attribute on a <reaction> is expected to be an SBO identifier (http://www.biomodels.net/SBO/) referring to an occurring entity representation defined in SBO (i.e., terms derived from SBO:0000231, "occurring entity representation"). \r\n Reference: L2V4 Sections 4.13.1 and 5\r\n SBO term \'SBO:0000560\' on the <reaction> is not in the appropriate branch.\r\n - 1 warning of type SBML unit consistency (10501). The following is the first warning at line 5932, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula \'Feed - feed2\' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 81 warnings of type SBML unit consistency (99505). The following is the first warning at line 3812, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression \'piecewise(1, and(geq(piecewise(time - 24 * ceil(time / 24), xor(lt(time, 0), lt(24, 0)), time - 24 * floor(time / 24)), 6), leq(piecewise(time - 24 * ceil(time / 24), xor(lt(time, 0), lt(24, 0)), time - 24 * floor(time / 24)), 18)), 0)\' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 131 warnings of type Modeling practice (80701). The following is the first warning at line 4111, column 12:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id \'KP1\' does not have a \'units\' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 456.0\r\n - Output end time: 480.0\r\n - Number of steps: 96\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Bae2018.sedml611274.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Bae2018.sedml` is invalid.\n - Data generator `CRH_1_task1` is invalid.\n - Variable `p1_CRH_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'CRH\']` are not defined. No namespaces are defined for the target.\n - Data generator `ACTH_2_task1` is invalid.\n - Variable `p1_ACTH_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'ACTH\']` are not defined. No namespaces are defined for the target.\n - Data generator `F_HPA_3_task1` is invalid.\n - Variable `p1_F_HPA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'F_HPA\']` are not defined. No namespaces are defined for the target.\n - Data generator `F_Cell_4_task1` is invalid.\n - Variable `p1_F_Cell_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'F_Cell\']` are not defined. No namespaces are defined for the target.\n - Data generator `y1_5_task1` is invalid.\n - Variable `p1_y1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'y1\']` are not defined. No namespaces are defined for the target.\n - Data generator `y2_6_task1` is invalid.\n - Variable `p1_y2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'y2\']` are not defined. No namespaces are defined for the target.\n - Data generator `y3_7_task1` is invalid.\n - Variable `p1_y3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'y3\']` are not defined. No namespaces are defined for the target.\n - Data generator `y4_8_task1` is invalid.\n - Variable `p1_y4_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'y4\']` are not defined. No namespaces are defined for the target.\n - Data generator `y5_9_task1` is invalid.\n - Variable `p1_y5_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'y5\']` are not defined. No namespaces are defined for the target.\n - Data generator `y6_10_task1` is invalid.\n - Variable `p1_y6_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'y6\']` are not defined. No namespaces are defined for the target.\n - Data generator `y7_11_task1` is invalid.\n - Variable `p1_y7_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'y7\']` are not defined. No namespaces are defined for the target.\n - Data generator `NAD_12_task1` is invalid.\n - Variable `p1_NAD_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'NAD\']` are not defined. No namespaces are defined for the target.\n - Data generator `NMN_13_task1` is invalid.\n - Variable `p1_NMN_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'NMN\']` are not defined. No namespaces are defined for the target.\n - Data generator `NAM_14_task1` is invalid.\n - Variable `p1_NAM_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'NAM\']` are not defined. No namespaces are defined for the target.\n - Data generator `EntF_15_task1` is invalid.\n - Variable `p1_EntF_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'EntF\']` are not defined. No namespaces are defined for the target.\n - Data generator `SIRT1_16_task1` is invalid.\n - Variable `p1_SIRT1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'SIRT1\']` are not defined. No namespaces are defined for the target.\n - Data generator `y8_17_task1` is invalid.\n - Variable `p1_y8_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'y8\']` are not defined. No namespaces are defined for the target.\n - Data generator `NAMPT_18_task1` is invalid.\n - Variable `p1_NAMPT_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'NAMPT\']` are not defined. No namespaces are defined for the target.\n - Data generator `feed2_19_task1` is invalid.\n - Variable `p1_feed2_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'feed2\']` are not defined. No namespaces are defined for the target.\n - Data generator `feed3_20_task1` is invalid.\n - Variable `p1_feed3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'feed3\']` are not defined. No namespaces are defined for the target.\n - Data generator `PGC1amRNA_21_task1` is invalid.\n - Variable `p1_PGC1amRNA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'PGC1amRNA\']` are not defined. No namespaces are defined for the target.\n - Data generator `PGC1a_22_task1` is invalid.\n - Variable `p1_PGC1a_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'PGC1a\']` are not defined. No namespaces are defined for the target.\n - Data generator `PGC1aN_23_task1` is invalid.\n - Variable `p1_PGC1aN_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'PGC1aN\']` are not defined. No namespaces are defined for the target.\n - Data generator `PGC1aNact_24_task1` is invalid.\n - Variable `p1_PGC1aNact_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'PGC1aNact\']` are not defined. No namespaces are defined for the target.\n - Data generator `FOXO1_25_task1` is invalid.\n - Variable `p1_FOXO1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'FOXO1\']` are not defined. No namespaces are defined for the target.\n - Data generator `PckG6p_26_task1` is invalid.\n - Variable `p1_PckG6p_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'PckG6p\']` are not defined. No namespaces are defined for the target.\n - Data generator `Phpa_27_task1` is invalid.\n - Variable `p1_Phpa_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'Phpa\']` are not defined. No namespaces are defined for the target.\n - Data generator `MR_28_task1` is invalid.\n - Variable `p1_MR_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'MR\']` are not defined. No namespaces are defined for the target.\n - Data generator `GR_Cell_29_task1` is invalid.\n - Variable `p1_GR_Cell_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'GR_Cell\']` are not defined. No namespaces are defined for the target.\n - Data generator `FGR_Cell_30_task1` is invalid.\n - Variable `p1_FGR_Cell_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'FGR_Cell\']` are not defined. No namespaces are defined for the target.\n - Data generator `FMR_31_task1` is invalid.\n - Variable `p1_FMR_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'FMR\']` are not defined. No namespaces are defined for the target.\n - Data generator `FMRN_32_task1` is invalid.\n - Variable `p1_FMRN_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'FMRN\']` are not defined. No namespaces are defined for the target.\n - Data generator `FGRN_Cell_33_task1` is invalid.\n - Variable `p1_FGRN_Cell_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'FGRN_Cell\']` are not defined. No namespaces are defined for the target.\n - Data generator `P_34_task1` is invalid.\n - Variable `p1_P_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'P\']` are not defined. No namespaces are defined for the target.\n - Data generator `mRNAP_35_task1` is invalid.\n - Variable `p1_mRNAP_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'mRNAP\']` are not defined. No namespaces are defined for the target.\n - Data generator `mRNARP_36_task1` is invalid.\n - Variable `p1_mRNARP_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'mRNARP\']` are not defined. No namespaces are defined for the target.\n - Data generator `Rp_37_task1` is invalid.\n - Variable `p1_Rp_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'Rp\']` are not defined. No namespaces are defined for the target.\n - Data generator `PR_38_task1` is invalid.\n - Variable `p1_PR_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'PR\']` are not defined. No namespaces are defined for the target.\n - Data generator `GR_HPA_39_task1` is invalid.\n - Variable `p1_GR_HPA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'GR_HPA\']` are not defined. No namespaces are defined for the target.\n - Data generator `FGR_HPA_40_task1` is invalid.\n - Variable `p1_FGR_HPA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'FGR_HPA\']` are not defined. No namespaces are defined for the target.\n - Data generator `FGRN_HPA_41_task1` is invalid.\n - Variable `p1_FGRN_HPA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'FGRN_HPA\']` are not defined. No namespaces are defined for the target.\n - Data generator `L_42_task1` is invalid.\n - Variable `p1_L_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id=\'L\']` are not defined. No namespaces are defined for the target.\n - Data generator `Feed_43_task1` is invalid.\n - Variable `p1_Feed_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id=\'Feed\']` are not defined. No namespaces are defined for the target.\n'```</details>|
|<details><summary>[BIOMD0000001006](https://www.ebi.ac.uk/biomodels/BIOMD0000001006)<br/><sup>Ciliberto2005 - Steady states </sup></summary><sup>Ciliberto2005 - Steady states and oscillations in the p53/Mdm2 network</sup></details>|<details><summary>pass</summary>[Ciliberto2005.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001006#Files)</details>|pass|<details><summary>pass</summary>[Ciliberto2005.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001006#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Ciliberto2005.sedml612550.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Ciliberto2005.xml` has warnings.\r\n - 30 warnings of type SBML unit consistency (99505). The following is the first warning at line 1325, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'vratio' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 27 warnings of type Modeling practice (80701). The following is the first warning at line 928, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Kf' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Ciliberto2005.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Ciliberto2005.xml` has warnings.\r\n - 30 warnings of type SBML unit consistency (99505). The following is the first warning at line 1325, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'vratio' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 27 warnings of type Modeling practice (80701). The following is the first warning at line 928, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Kf' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Ciliberto2005.sedml612550.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Ciliberto2005.sedml` is invalid.\n - Data generator `Mdm2_p_nuc_1_task1` is invalid.\n - Variable `p2_Mdm2_p_nuc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Mdm2_p_nuc']` are not defined. No namespaces are defined for the target.\n - Data generator `p53_total_2_task1` is invalid.\n - Variable `p2_p53_total_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='p53_total']` are not defined. No namespaces are defined for the target.\n - Data generator `Mdm2_cyt_1_task1` is invalid.\n - Variable `p3_Mdm2_cyt_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Mdm2_cyt']` are not defined. No namespaces are defined for the target.\n - Data generator `Mdm2_p_cyt_2_task1` is invalid.\n - Variable `p3_Mdm2_p_cyt_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Mdm2_p_cyt']` are not defined. No namespaces are defined for the target.\n - Data generator `DNA_damage_1_task1` is invalid.\n - Variable `p5_DNA_damage_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='DNA_damage']` are not defined. No namespaces are defined for the target.\n - Data generator `kd2_0_1_task1` is invalid.\n - Variable `p6_kd2_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='kd2_0']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Ciliberto2005.sedml663692.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Ciliberto2005.xml` has warnings.\r\n - 30 warnings of type SBML unit consistency (99505). The following is the first warning at line 1325, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'vratio' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 27 warnings of type Modeling practice (80701). The following is the first warning at line 928, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Kf' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Ciliberto2005.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Ciliberto2005.xml` has warnings.\r\n - 30 warnings of type SBML unit consistency (99505). The following is the first warning at line 1325, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'vratio' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 27 warnings of type Modeling practice (80701). The following is the first warning at line 928, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Kf' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Ciliberto2005.sedml663692.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Ciliberto2005.sedml` is invalid.\n - Data generator `Mdm2_p_nuc_1_task1` is invalid.\n - Variable `p2_Mdm2_p_nuc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Mdm2_p_nuc']` are not defined. No namespaces are defined for the target.\n - Data generator `p53_total_2_task1` is invalid.\n - Variable `p2_p53_total_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='p53_total']` are not defined. No namespaces are defined for the target.\n - Data generator `Mdm2_cyt_1_task1` is invalid.\n - Variable `p3_Mdm2_cyt_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Mdm2_cyt']` are not defined. No namespaces are defined for the target.\n - Data generator `Mdm2_p_cyt_2_task1` is invalid.\n - Variable `p3_Mdm2_p_cyt_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Mdm2_p_cyt']` are not defined. No namespaces are defined for the target.\n - Data generator `DNA_damage_1_task1` is invalid.\n - Variable `p5_DNA_damage_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='DNA_damage']` are not defined. No namespaces are defined for the target.\n - Data generator `kd2_0_1_task1` is invalid.\n - Variable `p6_kd2_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='kd2_0']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000001007](https://www.ebi.ac.uk/biomodels/BIOMD0000001007)<br/><sup>Zhang2007 - Mechanism of DNA d</sup></summary><sup>Zhang2007 - Mechanism of DNA damage response (Model1)</sup></details>|<details><summary>pass</summary>[Zhang2007_M1_lowDD.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001007#Files)</details>|pass|<details><summary>MultipleSEDMLs</summary>['Zhang2007_M1_highDD.sedml', 'Zhang2007_M1_lowDD.sedml']</details>|NA|NA|NA|
|<details><summary>[BIOMD0000001008](https://www.ebi.ac.uk/biomodels/BIOMD0000001008)<br/><sup>Scaramellini1997 - Two-recepto</sup></summary><sup>Scaramellini1997 - Two-receptor:One-transducer (2R1T) model for analysis of interactions between agonists</sup></details>|<details><summary>pass</summary>[Scaramellini1997.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001008#Files)</details>|pass|<details><summary>pass</summary>[Scaramellini1997.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001008#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Scaramellini1997.sedml446027.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Scaramellini1997.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 491, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'm1 * pow(Agonist_1, n1) / (pow(K1, n1) + pow(Agonist_1, n1))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 339, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'm1' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 1.0\r\n - Number of steps: 1\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task2`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Scaramellini1997.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Scaramellini1997.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 491, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'm1 * pow(Agonist_1, n1) / (pow(K1, n1) + pow(Agonist_1, n1))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 339, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'm1' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 1.0\r\n - Number of steps: 1\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task2`\x1b[0m\r\n\r\n`/root/in/Scaramellini1997.sedml446027.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Scaramellini1997.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Agonist_1']` are not defined. No namespaces are defined for the target.\n - Change 2 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Agonist_2']` are not defined. No namespaces are defined for the target.\n - The child ranges of repeated tasks must be at least as long as the main range of their parent repeated tasks. Range `range1` is shorter than its main range, 4 < 10000001.\n - The child ranges of repeated tasks must be at least as long as the main range of their parent repeated tasks. Range `range1` is shorter than its main range, 4 < 10000001.\n - Data generator `Effect_1_task1` is invalid.\n - Variable `p3_Effect_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Effect']` are not defined. No namespaces are defined for the target.\n - Data generator `Agonist_1_1_task1` is invalid.\n - Variable `p3_Agonist_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Agonist_1']` are not defined. No namespaces are defined for the target.\n - Data generator `Effect_1_task2` is invalid.\n - Variable `p3_Effect_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Effect']` are not defined. No namespaces are defined for the target.\n - Data generator `Agonist_1_1_task2` is invalid.\n - Variable `p3_Agonist_1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Agonist_1']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Scaramellini1997.sedml170901.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Scaramellini1997.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 491, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'm1 * pow(Agonist_1, n1) / (pow(K1, n1) + pow(Agonist_1, n1))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 339, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'm1' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 1.0\r\n - Number of steps: 1\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task2`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Scaramellini1997.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Scaramellini1997.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 491, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'm1 * pow(Agonist_1, n1) / (pow(K1, n1) + pow(Agonist_1, n1))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 10 warnings of type Modeling practice (80701). The following is the first warning at line 339, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'm1' does not have a 'units' attribute.\r\n - Simulation `sim1` has warnings.\r\n - The time course simulation has an unusual number of steps. Check that the parameters of the time course are correct.\r\n - Initial time: 0.0\r\n - Output start time: 0.0\r\n - Output end time: 1.0\r\n - Number of steps: 1\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task2`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Scaramellini1997.sedml170901.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Scaramellini1997.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Agonist_1']` are not defined. No namespaces are defined for the target.\n - Change 2 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Agonist_2']` are not defined. No namespaces are defined for the target.\n - The child ranges of repeated tasks must be at least as long as the main range of their parent repeated tasks. Range `range1` is shorter than its main range, 4 < 10000001.\n - The child ranges of repeated tasks must be at least as long as the main range of their parent repeated tasks. Range `range1` is shorter than its main range, 4 < 10000001.\n - Data generator `Effect_1_task1` is invalid.\n - Variable `p3_Effect_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Effect']` are not defined. No namespaces are defined for the target.\n - Data generator `Agonist_1_1_task1` is invalid.\n - Variable `p3_Agonist_1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Agonist_1']` are not defined. No namespaces are defined for the target.\n - Data generator `Effect_1_task2` is invalid.\n - Variable `p3_Effect_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Effect']` are not defined. No namespaces are defined for the target.\n - Data generator `Agonist_1_1_task2` is invalid.\n - Variable `p3_Agonist_1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Agonist_1']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000001009](https://www.ebi.ac.uk/biomodels/BIOMD0000001009)<br/><sup>Zhang2007 - Mechanism of DNA d</sup></summary><sup>Zhang2007 - Mechanism of DNA damage response (Model2)</sup></details>|<details><summary>pass</summary>[Zhang2007_M2_low_dose.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001009#Files)</details>|pass|<details><summary>FAIL</summary>[Zhang2007_M2_high_dose.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001009#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Zhang2007_M2_high_dose.sedml42408.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/Zhang2007_M2_high_dose.sedml42408.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Zhang2007_M2_high_dose.sedml` is invalid.\n - Model `model2` is invalid.\n - The model file `model2.xml` is invalid.\n - `/tmp/tmpa_dvq3yb/./model2.xml` is not a file.\n - Data generator `MDM2nuc_1_task1` is invalid.\n - Variable `p1_MDM2nuc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='MDM2nuc']` are not defined. No namespaces are defined for the target.\n - Data generator `p53_total_2_task1` is invalid.\n - Variable `p1_p53_total_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='p53_total']` are not defined. No namespaces are defined for the target.\n - Data generator `DNAdamage_1_task1` is invalid.\n - Variable `p2_DNAdamage_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='DNAdamage']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Zhang2007_M2_high_dose.sedml26745.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Zhang2007_M2_high_dose.sedml26745.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Zhang2007_M2_high_dose.sedml` is invalid.\n - Model `model2` is invalid.\n - The model file `model2.xml` is invalid.\n - `/tmp/tmpzu6y28t2/./model2.xml` is not a file.\n - Data generator `MDM2nuc_1_task1` is invalid.\n - Variable `p1_MDM2nuc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='MDM2nuc']` are not defined. No namespaces are defined for the target.\n - Data generator `p53_total_2_task1` is invalid.\n - Variable `p1_p53_total_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='p53_total']` are not defined. No namespaces are defined for the target.\n - Data generator `DNAdamage_1_task1` is invalid.\n - Variable `p2_DNAdamage_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='DNAdamage']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000001010](https://www.ebi.ac.uk/biomodels/BIOMD0000001010)<br/><sup>Zhang2007 - Mechanism of DNA d</sup></summary><sup>Zhang2007 - Mechanism of DNA damage response (Model3)</sup></details>|<details><summary>pass</summary>[Zhang2007_M3_low_DD.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001010#Files)</details>|pass|<details><summary>MultipleSEDMLs</summary>['Zhang2007_M3_low_DD.sedml', 'Zhang2007_M3_high_DD.sedml']</details>|NA|NA|NA|
|<details><summary>[BIOMD0000001011](https://www.ebi.ac.uk/biomodels/BIOMD0000001011)<br/><sup>Leon-Triana2020 - CAR T-cell t</sup></summary><sup>Leon-Triana2020 - CAR T-cell therapy in B-cell acute lymphoblastic leukaemia</sup></details>|<details><summary>pass</summary>[Leon-Triana2020 (eqs 4).xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001011#Files)</details>|FAIL|<details><summary>pass</summary>[Leon-Triana2020 (eqs 4).sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001011#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Leon-Triana2020 (eqs 4).sedml461357.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 2: b'usage: biosimulators-tellurium [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-tellurium: error: unrecognized arguments: (eqs 4).sedml461357.omex\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Leon-Triana2020 (eqs 4).sedml58596.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 2: b'usage: biosimulators-copasi [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-copasi: error: unrecognized arguments: (eqs 4).sedml58596.omex\n'```</details>|
|<details><summary>[BIOMD0000001012](https://www.ebi.ac.uk/biomodels/BIOMD0000001012)<br/><sup>Leon-Triana2020 - CAR T-cell t</sup></summary><sup>Leon-Triana2020 - CAR T-cell therapy in B-cell acute lymphoblastic leukaemia with contribution from immature B cells</sup></details>|<details><summary>pass</summary>[Leon-Triana2020 (eqs 3).xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001012#Files)</details>|FAIL|<details><summary>pass</summary>[Leon-Triana2020 (eqs 3).sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001012#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Leon-Triana2020 (eqs 3).sedml125255.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 2: b'usage: biosimulators-tellurium [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-tellurium: error: unrecognized arguments: (eqs 3).sedml125255.omex\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Leon-Triana2020 (eqs 3).sedml636931.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 2: b'usage: biosimulators-copasi [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-copasi: error: unrecognized arguments: (eqs 3).sedml636931.omex\n'```</details>|
|<details><summary>[BIOMD0000001013](https://www.ebi.ac.uk/biomodels/BIOMD0000001013)<br/><sup>Leon-Triana2021 - Competition </sup></summary><sup>Leon-Triana2021 - Competition between tumour cells and single-target CAR T-cells</sup></details>|<details><summary>pass</summary>[Leon-Triana2021 (eqs 1 and 2).xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001013#Files)</details>|FAIL|<details><summary>MultipleSEDMLs</summary>['Leon-Triana2021 (eqs 1 and 2) - fig 4(a).sedml', 'Leon-Triana2021 (eqs 1 and 2) - fig 1 (a, c).sedml', 'Leon-Triana2021 (eqs 1 and 2) - fig 3(b).sedml', 'Leon-Triana2021 (eqs 1 and 2) - fig 3(a).sedml', 'Leon-Triana2021 (eqs 1 and 2) - fig 5(a).sedml', 'Leon-Triana2021 (eqs 1 and 2) - fig 1 (b, d).sedml', 'Leon-Triana2021 (eqs 1 and 2) - fig 4(c).sedml', 'Leon-Triana2021 (eqs 1 and 2) - fig 4(b).sedml']</details>|NA|NA|NA|
|<details><summary>[BIOMD0000001014](https://www.ebi.ac.uk/biomodels/BIOMD0000001014)<br/><sup>Leon-Triana2021 - Competition </sup></summary><sup>Leon-Triana2021 - Competition between tumour cells and dual-target CAR T-cells</sup></details>|<details><summary>pass</summary>[Leon-Triana2021 (eqs 3-6).xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001014#Files)</details>|FAIL|<details><summary>pass</summary>[Leon-Triana2021 (eqs 3-6).sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001014#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Leon-Triana2021 (eqs 3-6).sedml972108.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 2: b'usage: biosimulators-tellurium [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-tellurium: error: unrecognized arguments: (eqs 3-6).sedml972108.omex\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Leon-Triana2021 (eqs 3-6).sedml251157.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 2: b'usage: biosimulators-copasi [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-copasi: error: unrecognized arguments: (eqs 3-6).sedml251157.omex\n'```</details>|
|<details><summary>[BIOMD0000001015](https://www.ebi.ac.uk/biomodels/BIOMD0000001015)<br/><sup>Jarrah2014 - mathematical mode</sup></summary><sup>Jarrah2014 - mathematical model of the immune response in muscle degeneration and subsequent regeneration in Duchenne muscular dystrophy in mdx mice</sup></details>|<details><summary>pass</summary>[Jarrah2014.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001015#Files)</details>|pass|<details><summary>pass</summary>[Jarrah2014.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001015#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Jarrah2014.sedml183381.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Jarrah2014.xml` has warnings.\r\n - 20 warnings of type SBML unit consistency (99508). The following is the first warning at line 670, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'k1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 20 warnings of type SBML unit consistency (20702). The following is the first warning at line 670, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'k1' does not have a 'units' attribute.\r\n - 13 warnings of type SBML unit consistency (99505). The following is the first warning at line 836, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'dC * C0' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 20 warnings of type Modeling practice (80701). The following is the first warning at line 670, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Jarrah2014.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Jarrah2014.xml` has warnings.\r\n - 20 warnings of type SBML unit consistency (99508). The following is the first warning at line 670, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'k1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 20 warnings of type SBML unit consistency (20702). The following is the first warning at line 670, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'k1' does not have a 'units' attribute.\r\n - 13 warnings of type SBML unit consistency (99505). The following is the first warning at line 836, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'dC * C0' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 20 warnings of type Modeling practice (80701). The following is the first warning at line 670, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n\r\n`/root/in/Jarrah2014.sedml183381.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Jarrah2014.sedml` is invalid.\n - Data generator `C_1_task1` is invalid.\n - Variable `p1_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C']` are not defined. No namespaces are defined for the target.\n - Data generator `D_1_task1` is invalid.\n - Variable `p2_D_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='D']` are not defined. No namespaces are defined for the target.\n - Data generator `H_1_task1` is invalid.\n - Variable `p3_H_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H']` are not defined. No namespaces are defined for the target.\n - Data generator `M_1_task1` is invalid.\n - Variable `p4_M_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M']` are not defined. No namespaces are defined for the target.\n - Data generator `N_1_task1` is invalid.\n - Variable `p5_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n - Data generator `R_1_task1` is invalid.\n - Variable `p6_R_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='R']` are not defined. No namespaces are defined for the target.\n - Data generator `H_1_task12` is invalid.\n - Variable `p7_H_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H']` are not defined. No namespaces are defined for the target.\n - Data generator `D_2_task1` is invalid.\n - Variable `p7_D_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='D']` are not defined. No namespaces are defined for the target.\n - Data generator `M_3_task1` is invalid.\n - Variable `p7_M_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M']` are not defined. No namespaces are defined for the target.\n - Data generator `C_4_task1` is invalid.\n - Variable `p7_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C']` are not defined. No namespaces are defined for the target.\n - Data generator `N_5_task1` is invalid.\n - Variable `p7_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n - Data generator `R_6_task1` is invalid.\n - Variable `p7_R_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='R']` are not defined. No namespaces are defined for the target.\n - Data generator `bH_7_task1` is invalid.\n - Variable `p7_bH_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='bH']` are not defined. No namespaces are defined for the target.\n - Data generator `bC_8_task1` is invalid.\n - Variable `p7_bC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='bC']` are not defined. No namespaces are defined for the target.\n - Data generator `bM_9_task1` is invalid.\n - Variable `p7_bM_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='bM']` are not defined. No namespaces are defined for the target.\n - Data generator `alpha_10_task1` is invalid.\n - Variable `p7_alpha_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Jarrah2014.sedml371860.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Jarrah2014.xml` has warnings.\r\n - 20 warnings of type SBML unit consistency (99508). The following is the first warning at line 670, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'k1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 20 warnings of type SBML unit consistency (20702). The following is the first warning at line 670, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'k1' does not have a 'units' attribute.\r\n - 13 warnings of type SBML unit consistency (99505). The following is the first warning at line 836, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'dC * C0' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 20 warnings of type Modeling practice (80701). The following is the first warning at line 670, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Jarrah2014.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Jarrah2014.xml` has warnings.\r\n - 20 warnings of type SBML unit consistency (99508). The following is the first warning at line 670, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'k1' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 20 warnings of type SBML unit consistency (20702). The following is the first warning at line 670, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'k1' does not have a 'units' attribute.\r\n - 13 warnings of type SBML unit consistency (99505). The following is the first warning at line 836, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'dC * C0' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 20 warnings of type Modeling practice (80701). The following is the first warning at line 670, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Jarrah2014.sedml371860.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Jarrah2014.sedml` is invalid.\n - Data generator `C_1_task1` is invalid.\n - Variable `p1_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C']` are not defined. No namespaces are defined for the target.\n - Data generator `D_1_task1` is invalid.\n - Variable `p2_D_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='D']` are not defined. No namespaces are defined for the target.\n - Data generator `H_1_task1` is invalid.\n - Variable `p3_H_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H']` are not defined. No namespaces are defined for the target.\n - Data generator `M_1_task1` is invalid.\n - Variable `p4_M_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M']` are not defined. No namespaces are defined for the target.\n - Data generator `N_1_task1` is invalid.\n - Variable `p5_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n - Data generator `R_1_task1` is invalid.\n - Variable `p6_R_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='R']` are not defined. No namespaces are defined for the target.\n - Data generator `H_1_task12` is invalid.\n - Variable `p7_H_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='H']` are not defined. No namespaces are defined for the target.\n - Data generator `D_2_task1` is invalid.\n - Variable `p7_D_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='D']` are not defined. No namespaces are defined for the target.\n - Data generator `M_3_task1` is invalid.\n - Variable `p7_M_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='M']` are not defined. No namespaces are defined for the target.\n - Data generator `C_4_task1` is invalid.\n - Variable `p7_C_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C']` are not defined. No namespaces are defined for the target.\n - Data generator `N_5_task1` is invalid.\n - Variable `p7_N_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` are not defined. No namespaces are defined for the target.\n - Data generator `R_6_task1` is invalid.\n - Variable `p7_R_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='R']` are not defined. No namespaces are defined for the target.\n - Data generator `bH_7_task1` is invalid.\n - Variable `p7_bH_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='bH']` are not defined. No namespaces are defined for the target.\n - Data generator `bC_8_task1` is invalid.\n - Variable `p7_bC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='bC']` are not defined. No namespaces are defined for the target.\n - Data generator `bM_9_task1` is invalid.\n - Variable `p7_bM_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='bM']` are not defined. No namespaces are defined for the target.\n - Data generator `alpha_10_task1` is invalid.\n - Variable `p7_alpha_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='alpha']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000001016](https://www.ebi.ac.uk/biomodels/BIOMD0000001016)<br/><sup>Bakshi2020 - Truncated minimal</sup></summary><sup>Bakshi2020 - Truncated minimal model of alternative pathway of complement system</sup></details>|<details><summary>pass</summary>[Bakshi2020 truncated minimal model.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001016#Files)</details>|FAIL|<details><summary>FAIL</summary>[Bakshi2020 truncated minimal model.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001016#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Bakshi2020 truncated minimal model.sedml349416.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 2: b'usage: biosimulators-tellurium [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-tellurium: error: unrecognized arguments: truncated minimal model.sedml349416.omex\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Bakshi2020 truncated minimal model.sedml610600.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 2: b'usage: biosimulators-copasi [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-copasi: error: unrecognized arguments: truncated minimal model.sedml610600.omex\n'```</details>|
|<details><summary>[BIOMD0000001017](https://www.ebi.ac.uk/biomodels/BIOMD0000001017)<br/><sup>Bakshi2020 - Minimal model of </sup></summary><sup>Bakshi2020 - Minimal model of alternative pathway of complement system</sup></details>|<details><summary>pass</summary>[Bakshi2020 minimal model.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001017#Files)</details>|FAIL|<details><summary>pass</summary>[Bakshi2020 minimal model.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001017#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Bakshi2020 minimal model.sedml772320.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 2: b'usage: biosimulators-tellurium [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-tellurium: error: unrecognized arguments: minimal model.sedml772320.omex\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Bakshi2020 minimal model.sedml379001.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 2: b'usage: biosimulators-copasi [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-copasi: error: unrecognized arguments: minimal model.sedml379001.omex\n'```</details>|
|<details><summary>[BIOMD0000001018](https://www.ebi.ac.uk/biomodels/BIOMD0000001018)<br/><sup>Bakshi2020 - Properdin model o</sup></summary><sup>Bakshi2020 - Properdin model of alternative pathway of complement system</sup></details>|<details><summary>pass</summary>[Bakshi2020 properdin model.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001018#Files)</details>|FAIL|<details><summary>FAIL</summary>[Bakshi2020 properdin model.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001018#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Bakshi2020 properdin model.sedml398672.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 2: b'usage: biosimulators-tellurium [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-tellurium: error: unrecognized arguments: properdin model.sedml398672.omex\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Bakshi2020 properdin model.sedml512926.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 2: b'usage: biosimulators-copasi [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-copasi: error: unrecognized arguments: properdin model.sedml512926.omex\n'```</details>|
|<details><summary>[BIOMD0000001019](https://www.ebi.ac.uk/biomodels/BIOMD0000001019)<br/><sup>Barros2021 - CARTmath, Mathema</sup></summary><sup>Barros2021 - CARTmath, Mathematical Model of CAR-T Immunotherapy in HDLM-2 cell line</sup></details>|<details><summary>pass</summary>[Barros2021_HDLM2.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001019#Files)</details>|FAIL|<details><summary>FAIL</summary>[Barros2021_HDLM2.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001019#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Barros2021_HDLM2.sedml957440.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/Barros2021_HDLM2.sedml957440.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Barros2021_HDLM2.sedml` is invalid.\n - Model `model3` is invalid.\n - The model file `model3.xml` is invalid.\n - `/tmp/tmpoxaps0oz/./model3.xml` is not a file.\n - Data generator `Cm_1_task1` is invalid.\n - Variable `p1_Cm_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cm']` are not defined. No namespaces are defined for the target.\n - Data generator `Ct_2_task1` is invalid.\n - Variable `p1_Ct_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Ct']` are not defined. No namespaces are defined for the target.\n - Data generator `T_3_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Barros2021_HDLM2.sedml609238.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Barros2021_HDLM2.sedml609238.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Barros2021_HDLM2.sedml` is invalid.\n - Model `model3` is invalid.\n - The model file `model3.xml` is invalid.\n - `/tmp/tmpt_vh980y/./model3.xml` is not a file.\n - Data generator `Cm_1_task1` is invalid.\n - Variable `p1_Cm_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cm']` are not defined. No namespaces are defined for the target.\n - Data generator `Ct_2_task1` is invalid.\n - Variable `p1_Ct_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Ct']` are not defined. No namespaces are defined for the target.\n - Data generator `T_3_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000001020](https://www.ebi.ac.uk/biomodels/BIOMD0000001020)<br/><sup>Barros2021 - CARTmath, Mathema</sup></summary><sup>Barros2021 - CARTmath, Mathematical Model of CAR-T Immunotherapy in Raji Cell Line</sup></details>|<details><summary>pass</summary>[Barros2021_RAJI.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001020#Files)</details>|FAIL|<details><summary>FAIL</summary>[Barros2021_RAJI.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001020#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Barros2021_RAJI.sedml629980.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/Barros2021_RAJI.sedml629980.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Barros2021_RAJI.sedml` is invalid.\n - Model `model4` is invalid.\n - The model file `model4.xml` is invalid.\n - `/tmp/tmpy77x0iuv/./model4.xml` is not a file.\n - Data generator `Cm_1_task1` is invalid.\n - Variable `p1_Cm_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cm']` are not defined. No namespaces are defined for the target.\n - Data generator `Ct_2_task1` is invalid.\n - Variable `p1_Ct_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Ct']` are not defined. No namespaces are defined for the target.\n - Data generator `T_3_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Barros2021_RAJI.sedml519613.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Barros2021_RAJI.sedml519613.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Barros2021_RAJI.sedml` is invalid.\n - Model `model4` is invalid.\n - The model file `model4.xml` is invalid.\n - `/tmp/tmpc6yfl5m6/./model4.xml` is not a file.\n - Data generator `Cm_1_task1` is invalid.\n - Variable `p1_Cm_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cm']` are not defined. No namespaces are defined for the target.\n - Data generator `Ct_2_task1` is invalid.\n - Variable `p1_Ct_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Ct']` are not defined. No namespaces are defined for the target.\n - Data generator `T_3_task1` is invalid.\n - Variable `p1_T_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000001021](https://www.ebi.ac.uk/biomodels/BIOMD0000001021)<br/><sup>Lavigne2021 - Non-spatial mode</sup></summary><sup>Lavigne2021 - Non-spatial model of viral infection dynamics and interferon response of well-mixed viral infection</sup></details>|<details><summary>pass</summary>[Lavigne2021 - ODE model.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001021#Files)</details>|FAIL|<details><summary>MultipleSEDMLs</summary>['Lavigne2021 - ODE model (paracrine only condition).sedml', 'Lavigne2021 - ODE model (with IFN condition).sedml', 'Lavigne2021 - ODE model (no IFN condition).sedml']</details>|NA|NA|NA|
|<details><summary>[BIOMD0000001022](https://www.ebi.ac.uk/biomodels/BIOMD0000001022)<br/><sup>Creemers2021 - Tumor-immune dy</sup></summary><sup>Creemers2021 - Tumor-immune dynamics and implications on immunotherapy responses</sup></details>|<details><summary>pass</summary>[Creemers2021.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001022#Files)</details>|FAIL|<details><summary>MultipleSEDMLs</summary>['Creemers2021 - Fig 2(B) inset 1.sedml', 'Creemers2021 - Fig 2(A) inset 1.sedml', 'Creemers2021 - Fig 2(A) inset 2.sedml', 'Creemers2021 - Fig 1(C).sedml', 'Creemers2021 - Fig 2(B) inset 2.sedml', 'Creemers2021 - Fig 1(B).sedml', 'Creemers2021 - Fig 1(D).sedml']</details>|NA|NA|NA|
|<details><summary>[BIOMD0000001023](https://www.ebi.ac.uk/biomodels/BIOMD0000001023)<br/><sup>Alharbi2020 - An ODE-based mod</sup></summary><sup>Alharbi2020 - An ODE-based model of the dynamics of tumor cell progression and its effects on normal cell growth and immune system functionality</sup></details>|<details><summary>pass</summary>[Alharbi2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001023#Files)</details>|pass|<details><summary>pass</summary>[Alharbi2020 - Fig 5.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001023#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Alharbi2020 - Fig 5.sedml789792.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 2: b'usage: biosimulators-tellurium [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-tellurium: error: unrecognized arguments: - Fig 5.sedml789792.omex\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Alharbi2020 - Fig 5.sedml646895.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 2: b'usage: biosimulators-copasi [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-copasi: error: unrecognized arguments: - Fig 5.sedml646895.omex\n'```</details>|
|<details><summary>[BIOMD0000001024](https://www.ebi.ac.uk/biomodels/BIOMD0000001024)<br/><sup>Chaudhury2020 - Lotka-Volterra</sup></summary><sup>Chaudhury2020 - Lotka-Volterra mathematical model of CAR-T cell and tumour kinetics</sup></details>|<details><summary>pass</summary>[Chaudhury2020 Eq 4-5.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001024#Files)</details>|FAIL|<details><summary>pass</summary>[Chaudhury2020 Eq 4-5 - Fig 3(C) simulation.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001024#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Chaudhury2020 Eq 4-5 - Fig 3(C) simulation.sedml961588.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 2: b'usage: biosimulators-tellurium [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-tellurium: error: unrecognized arguments: Eq 4-5 - Fig 3(C) simulation.sedml961588.omex\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Chaudhury2020 Eq 4-5 - Fig 3(C) simulation.sedml101495.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 2: b'usage: biosimulators-copasi [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-copasi: error: unrecognized arguments: Eq 4-5 - Fig 3(C) simulation.sedml101495.omex\n'```</details>|
|<details><summary>[BIOMD0000001025](https://www.ebi.ac.uk/biomodels/BIOMD0000001025)<br/><sup>Chaudhury2020 - EC50 expansion</sup></summary><sup>Chaudhury2020 - EC50 expansion and killing mathematical model of CAR-T cell and tumour kinetics</sup></details>|<details><summary>pass</summary>[Chaudhury2020 Eq 7-9.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001025#Files)</details>|pass|<details><summary>MultipleSEDMLs</summary>['Chaudhury2020 Eq 7-9 - Fig 3(B) simulation.sedml', 'Chaudhury2020 Eq 7-9 - Fig 3(A) simulation.sedml', 'Chaudhury2020 Eq 7-9 - Fig 4(B) simulation.sedml', 'Chaudhury2020 Eq 7-9 - Fig 4(A) simulation.sedml']</details>|NA|NA|NA|
|<details><summary>[BIOMD0000001026](https://www.ebi.ac.uk/biomodels/BIOMD0000001026)<br/><sup>Kurlovics2021 - Metformin part</sup></summary><sup>Kurlovics2021 - Metformin partitioning between plasma and RBC with independent Kin and Kout coefficients</sup></details>|<details><summary>pass</summary>[Kurlovics2021.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001026#Files)</details>|pass|<details><summary>FAIL</summary>[Kurlovics2021.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001026#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Kurlovics2021.sedml956302.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/Kurlovics2021.sedml956302.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Kurlovics2021.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmpvhrpj79p/./model1.xml` is not a file.\n - Data generator `Mpl_1_task1` is invalid.\n - Variable `p2_Mpl_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Mpl']` are not defined. No namespaces are defined for the target.\n - Data generator `Mrbc_2_task1` is invalid.\n - Variable `p2_Mrbc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Mrbc']` are not defined. No namespaces are defined for the target.\n - Data generator `Vin_1_task1` is invalid.\n - Variable `p3_Vin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='Vin']` are not defined. No namespaces are defined for the target.\n - Data generator `Vout_2_task1` is invalid.\n - Variable `p3_Vout_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='Vout']` are not defined. No namespaces are defined for the target.\n - Data generator `Summary_flux_to_RBC_3_task1` is invalid.\n - Variable `p3_Summary_flux_to_RBC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Summary_flux_to_RBC']` are not defined. No namespaces are defined for the target.\n - Output `plot_1_task1` is invalid.\n - Plot must have at least one curve.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Kurlovics2021.sedml272589.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Kurlovics2021.sedml272589.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Kurlovics2021.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmpqtox6n94/./model1.xml` is not a file.\n - Data generator `Mpl_1_task1` is invalid.\n - Variable `p2_Mpl_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Mpl']` are not defined. No namespaces are defined for the target.\n - Data generator `Mrbc_2_task1` is invalid.\n - Variable `p2_Mrbc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Mrbc']` are not defined. No namespaces are defined for the target.\n - Data generator `Vin_1_task1` is invalid.\n - Variable `p3_Vin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='Vin']` are not defined. No namespaces are defined for the target.\n - Data generator `Vout_2_task1` is invalid.\n - Variable `p3_Vout_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='Vout']` are not defined. No namespaces are defined for the target.\n - Data generator `Summary_flux_to_RBC_3_task1` is invalid.\n - Variable `p3_Summary_flux_to_RBC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Summary_flux_to_RBC']` are not defined. No namespaces are defined for the target.\n - Output `plot_1_task1` is invalid.\n - Plot must have at least one curve.\n"```</details>|
|<details><summary>[BIOMD0000001027](https://www.ebi.ac.uk/biomodels/BIOMD0000001027)<br/><sup>Zake2021 - PBPK model of metfo</sup></summary><sup>Zake2021 - PBPK model of metformin in mice: single dose peroral</sup></details>|<details><summary>pass</summary>[Zake2021_Metformin+Mice+PO.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001027#Files)</details>|pass|<details><summary>pass</summary>[Zake2021_Metformin+Mice+PO.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001027#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Zake2021_Metformin+Mice+PO.sedml821918.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Zake2021_Metformin+Mice+PO.xml` has warnings.\r\n - 66 warnings of type SBML unit consistency (99508). The following is the first warning at line 1733, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'Ktp_Liver' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 66 warnings of type SBML unit consistency (20702). The following is the first warning at line 1733, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'Ktp_Liver' does not have a 'units' attribute.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 2595, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression '1000000 * Metformin_Dose_in_Plasma_in_mg / (129.16 * PlasmaVenous) * PlasmaVenous' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 83 warnings of type Modeling practice (80701). The following is the first warning at line 1733, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Ktp_Liver' does not have a 'units' attribute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_5_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_6_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Zake2021_Metformin+Mice+PO.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Zake2021_Metformin+Mice+PO.xml` has warnings.\r\n - 66 warnings of type SBML unit consistency (99508). The following is the first warning at line 1733, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'Ktp_Liver' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 66 warnings of type SBML unit consistency (20702). The following is the first warning at line 1733, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'Ktp_Liver' does not have a 'units' attribute.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 2595, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression '1000000 * Metformin_Dose_in_Plasma_in_mg / (129.16 * PlasmaVenous) * PlasmaVenous' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 83 warnings of type Modeling practice (80701). The following is the first warning at line 1733, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Ktp_Liver' does not have a 'units' attribute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_5_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_6_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/Zake2021_Metformin+Mice+PO.sedml821918.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Zake2021_Metformin+Mice+PO.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Ktp_IntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mAdipose_1_task1` is invalid.\n - Variable `p1_mAdipose_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mBrain_2_task1` is invalid.\n - Variable `p1_mBrain_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mFeces_3_task1` is invalid.\n - Variable `p1_mFeces_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mHeart_4_task1` is invalid.\n - Variable `p1_mHeart_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineEnterocytes_5_task1` is invalid.\n - Variable `p1_mIntestineEnterocytes_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineLumen_6_task1` is invalid.\n - Variable `p1_mIntestineLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineSum_7_task1` is invalid.\n - Variable `p1_mIntestineSum_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mIntestineSum']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineVascular_8_task1` is invalid.\n - Variable `p1_mIntestineVascular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyPlasma_9_task1` is invalid.\n - Variable `p1_mKidneyPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneySum_10_task1` is invalid.\n - Variable `p1_mKidneySum_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mKidneySum']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTissue_11_task1` is invalid.\n - Variable `p1_mKidneyTissue_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTissue']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTubular_12_task1` is invalid.\n - Variable `p1_mKidneyTubular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mLiver_13_task1` is invalid.\n - Variable `p1_mLiver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mLung_14_task1` is invalid.\n - Variable `p1_mLung_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mMuscle_15_task1` is invalid.\n - Variable `p1_mMuscle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaArterial_16_task1` is invalid.\n - Variable `p1_mPlasmaArterial_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaVenous_17_task1` is invalid.\n - Variable `p1_mPlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mPortalVein_18_task1` is invalid.\n - Variable `p1_mPortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mRemainder_19_task1` is invalid.\n - Variable `p1_mRemainder_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomach_20_task1` is invalid.\n - Variable `p1_mStomach_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomachLumen_21_task1` is invalid.\n - Variable `p1_mStomachLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mUrine_22_task1` is invalid.\n - Variable `p1_mUrine_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mUrine']` are not defined. No namespaces are defined for the target.\n - Data generator `_01__StomachLumen_____IntestineLumen_1_task1` is invalid.\n - Variable `p2__01__StomachLumen_____IntestineLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_01__StomachLumen_____IntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `_02__IntestineLumen____Feces_2_task1` is invalid.\n - Variable `p2__02__IntestineLumen____Feces_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_02__IntestineLumen____Feces']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_1__PlasmaArterial____IntestineVascular_3_task1` is invalid.\n - Variable `p2__03_1__PlasmaArterial____IntestineVascular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_1__PlasmaArterial____IntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_2__IntestineLumen____IntestineEnterocytes__PMAT_OCT3_4_task1` is invalid.\n - Variable `p2__03_2__IntestineLumen____IntestineEnterocytes__PMAT_OCT3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_2__IntestineLumen____IntestineEnterocytes__PMAT_OCT3']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_3__IntestineEnterocytes____IntestineVascular__OCT1_5_task1` is invalid.\n - Variable `p2__03_3__IntestineEnterocytes____IntestineVascular__OCT1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_3__IntestineEnterocytes____IntestineVascular__OCT1']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_4__IntestineLumen____IntestineVascular__Saturable_6_task1` is invalid.\n - Variable `p2__03_4__IntestineLumen____IntestineVascular__Saturable_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_4__IntestineLumen____IntestineVascular__Saturable']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_5__IntestineVascular____PortalVein_7_task1` is invalid.\n - Variable `p2__03_5__IntestineVascular____PortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_5__IntestineVascular____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_6__IntestineLumen____IntestineEnterocytes__Diffusion_8_task1` is invalid.\n - Variable `p2__03_6__IntestineLumen____IntestineEnterocytes__Diffusion_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_6__IntestineLumen____IntestineEnterocytes__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_7__IntestineLumen____IntestineVascular__Diffusion_9_task1` is invalid.\n - Variable `p2__03_7__IntestineLumen____IntestineVascular__Diffusion_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_7__IntestineLumen____IntestineVascular__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_04__PlasmaArterial____PortalVein_10_task1` is invalid.\n - Variable `p2__04__PlasmaArterial____PortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_04__PlasmaArterial____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_1__PlasmaArterial____Stomach_11_task1` is invalid.\n - Variable `p2__05_1__PlasmaArterial____Stomach_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_1__PlasmaArterial____Stomach']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_2__Stomach____PortalVein_12_task1` is invalid.\n - Variable `p2__05_2__Stomach____PortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_2__Stomach____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_1__PlasmaArterial____Liver_13_task1` is invalid.\n - Variable `p2__06_1__PlasmaArterial____Liver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_1__PlasmaArterial____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_2__PortalVein____Liver_14_task1` is invalid.\n - Variable `p2__06_2__PortalVein____Liver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_2__PortalVein____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_3__Liver____PlasmaVenous_15_task1` is invalid.\n - Variable `p2__06_3__Liver____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_3__Liver____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_1__PlasmaVenous____Lung_16_task1` is invalid.\n - Variable `p2__07_1__PlasmaVenous____Lung_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_1__PlasmaVenous____Lung']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_2__Lung____PlasmaArterial_17_task1` is invalid.\n - Variable `p2__07_2__Lung____PlasmaArterial_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_2__Lung____PlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_1__PlasmaArterial____Brain_18_task1` is invalid.\n - Variable `p2__08_1__PlasmaArterial____Brain_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_1__PlasmaArterial____Brain']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_2__Brain____PlasmaVenous_19_task1` is invalid.\n - Variable `p2__08_2__Brain____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_2__Brain____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_1__PlasmaArterial____Muscle_20_task1` is invalid.\n - Variable `p2__09_1__PlasmaArterial____Muscle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_1__PlasmaArterial____Muscle']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_2__Muscle____PlasmaVenous_21_task1` is invalid.\n - Variable `p2__09_2__Muscle____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_2__Muscle____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_1__PlasmaArterial____Adipose_22_task1` is invalid.\n - Variable `p2__10_1__PlasmaArterial____Adipose_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_1__PlasmaArterial____Adipose']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_2__Adipose____PlasmaVenous_23_task1` is invalid.\n - Variable `p2__10_2__Adipose____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_2__Adipose____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_1__PlasmaArterial____Heart_24_task1` is invalid.\n - Variable `p2__11_1__PlasmaArterial____Heart_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_1__PlasmaArterial____Heart']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_2__Heart____PlasmaVenous_25_task1` is invalid.\n - Variable `p2__11_2__Heart____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_2__Heart____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_1__PlasmaArterial____Remainder_26_task1` is invalid.\n - Variable `p2__12_1__PlasmaArterial____Remainder_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_1__PlasmaArterial____Remainder']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_2__Remainder____PlasmaVenous_27_task1` is invalid.\n - Variable `p2__12_2__Remainder____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_2__Remainder____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_1__PlasmaArterial____KidneyPlasma_28_task1` is invalid.\n - Variable `p2__13_1__PlasmaArterial____KidneyPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_1__PlasmaArterial____KidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_2__KidneyPlasma____PlasmaVenous_29_task1` is invalid.\n - Variable `p2__13_2__KidneyPlasma____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_2__KidneyPlasma____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_3__KidneyPlasma____KidneyTubular_30_task1` is invalid.\n - Variable `p2__13_3__KidneyPlasma____KidneyTubular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_3__KidneyPlasma____KidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_4__KidneyPlasma____KidneyTissue_31_task1` is invalid.\n - Variable `p2__13_4__KidneyPlasma____KidneyTissue_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_4__KidneyPlasma____KidneyTissue']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_5__KidneyTissue____KidneyTubular_32_task1` is invalid.\n - Variable `p2__13_5__KidneyTissue____KidneyTubular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_5__KidneyTissue____KidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_6__KidneyTubular____UrineExternal_33_task1` is invalid.\n - Variable `p2__13_6__KidneyTubular____UrineExternal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_6__KidneyTubular____UrineExternal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgAdipose_1_task1` is invalid.\n - Variable `p3_mgAdipose_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mgBrain_2_task1` is invalid.\n - Variable `p3_mgBrain_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mgFeces_3_task1` is invalid.\n - Variable `p3_mgFeces_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mgHeart_4_task1` is invalid.\n - Variable `p3_mgHeart_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineEnterocytes_5_task1` is invalid.\n - Variable `p3_mgIntestineEnterocytes_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineLumen_6_task1` is invalid.\n - Variable `p3_mgIntestineLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineTotal_7_task1` is invalid.\n - Variable `p3_mgIntestineTotal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineVascular_8_task1` is invalid.\n - Variable `p3_mgIntestineVascular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyPlasma_9_task1` is invalid.\n - Variable `p3_mgKidneyPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTissues_10_task1` is invalid.\n - Variable `p3_mgKidneyTissues_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTissues']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTotal_11_task1` is invalid.\n - Variable `p3_mgKidneyTotal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTubular_12_task1` is invalid.\n - Variable `p3_mgKidneyTubular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLiver_13_task1` is invalid.\n - Variable `p3_mgLiver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLung_14_task1` is invalid.\n - Variable `p3_mgLung_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mgMuscle_15_task1` is invalid.\n - Variable `p3_mgMuscle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPlasmaArterial_16_task1` is invalid.\n - Variable `p3_mgPlasmaArterial_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPlasmaVenous_17_task1` is invalid.\n - Variable `p3_mgPlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPortalVein_18_task1` is invalid.\n - Variable `p3_mgPortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mgRemainder_19_task1` is invalid.\n - Variable `p3_mgRemainder_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomach_20_task1` is invalid.\n - Variable `p3_mgStomach_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomachLumen_21_task1` is invalid.\n - Variable `p3_mgStomachLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgUrine_22_task1` is invalid.\n - Variable `p3_mgUrine_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgUrine']` are not defined. No namespaces are defined for the target.\n - Data generator `mAdipose_1_task2` is invalid.\n - Variable `p1_mAdipose_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mBrain_2_task2` is invalid.\n - Variable `p1_mBrain_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mFeces_3_task2` is invalid.\n - Variable `p1_mFeces_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mHeart_4_task2` is invalid.\n - Variable `p1_mHeart_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineEnterocytes_5_task2` is invalid.\n - Variable `p1_mIntestineEnterocytes_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineLumen_6_task2` is invalid.\n - Variable `p1_mIntestineLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineSum_7_task2` is invalid.\n - Variable `p1_mIntestineSum_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mIntestineSum']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineVascular_8_task2` is invalid.\n - Variable `p1_mIntestineVascular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyPlasma_9_task2` is invalid.\n - Variable `p1_mKidneyPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneySum_10_task2` is invalid.\n - Variable `p1_mKidneySum_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mKidneySum']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTissue_11_task2` is invalid.\n - Variable `p1_mKidneyTissue_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTissue']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTubular_12_task2` is invalid.\n - Variable `p1_mKidneyTubular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mLiver_13_task2` is invalid.\n - Variable `p1_mLiver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mLung_14_task2` is invalid.\n - Variable `p1_mLung_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mMuscle_15_task2` is invalid.\n - Variable `p1_mMuscle_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaArterial_16_task2` is invalid.\n - Variable `p1_mPlasmaArterial_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaVenous_17_task2` is invalid.\n - Variable `p1_mPlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mPortalVein_18_task2` is invalid.\n - Variable `p1_mPortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mRemainder_19_task2` is invalid.\n - Variable `p1_mRemainder_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomach_20_task2` is invalid.\n - Variable `p1_mStomach_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomachLumen_21_task2` is invalid.\n - Variable `p1_mStomachLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mUrine_22_task2` is invalid.\n - Variable `p1_mUrine_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mUrine']` are not defined. No namespaces are defined for the target.\n - Data generator `_01__StomachLumen_____IntestineLumen_1_task2` is invalid.\n - Variable `p2__01__StomachLumen_____IntestineLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_01__StomachLumen_____IntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `_02__IntestineLumen____Feces_2_task2` is invalid.\n - Variable `p2__02__IntestineLumen____Feces_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_02__IntestineLumen____Feces']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_1__PlasmaArterial____IntestineVascular_3_task2` is invalid.\n - Variable `p2__03_1__PlasmaArterial____IntestineVascular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_1__PlasmaArterial____IntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_2__IntestineLumen____IntestineEnterocytes__PMAT_OCT3_4_task2` is invalid.\n - Variable `p2__03_2__IntestineLumen____IntestineEnterocytes__PMAT_OCT3_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_2__IntestineLumen____IntestineEnterocytes__PMAT_OCT3']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_3__IntestineEnterocytes____IntestineVascular__OCT1_5_task2` is invalid.\n - Variable `p2__03_3__IntestineEnterocytes____IntestineVascular__OCT1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_3__IntestineEnterocytes____IntestineVascular__OCT1']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_4__IntestineLumen____IntestineVascular__Saturable_6_task2` is invalid.\n - Variable `p2__03_4__IntestineLumen____IntestineVascular__Saturable_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_4__IntestineLumen____IntestineVascular__Saturable']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_5__IntestineVascular____PortalVein_7_task2` is invalid.\n - Variable `p2__03_5__IntestineVascular____PortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_5__IntestineVascular____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_6__IntestineLumen____IntestineEnterocytes__Diffusion_8_task2` is invalid.\n - Variable `p2__03_6__IntestineLumen____IntestineEnterocytes__Diffusion_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_6__IntestineLumen____IntestineEnterocytes__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_7__IntestineLumen____IntestineVascular__Diffusion_9_task2` is invalid.\n - Variable `p2__03_7__IntestineLumen____IntestineVascular__Diffusion_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_7__IntestineLumen____IntestineVascular__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_04__PlasmaArterial____PortalVein_10_task2` is invalid.\n - Variable `p2__04__PlasmaArterial____PortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_04__PlasmaArterial____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_1__PlasmaArterial____Stomach_11_task2` is invalid.\n - Variable `p2__05_1__PlasmaArterial____Stomach_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_1__PlasmaArterial____Stomach']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_2__Stomach____PortalVein_12_task2` is invalid.\n - Variable `p2__05_2__Stomach____PortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_2__Stomach____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_1__PlasmaArterial____Liver_13_task2` is invalid.\n - Variable `p2__06_1__PlasmaArterial____Liver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_1__PlasmaArterial____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_2__PortalVein____Liver_14_task2` is invalid.\n - Variable `p2__06_2__PortalVein____Liver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_2__PortalVein____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_3__Liver____PlasmaVenous_15_task2` is invalid.\n - Variable `p2__06_3__Liver____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_3__Liver____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_1__PlasmaVenous____Lung_16_task2` is invalid.\n - Variable `p2__07_1__PlasmaVenous____Lung_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_1__PlasmaVenous____Lung']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_2__Lung____PlasmaArterial_17_task2` is invalid.\n - Variable `p2__07_2__Lung____PlasmaArterial_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_2__Lung____PlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_1__PlasmaArterial____Brain_18_task2` is invalid.\n - Variable `p2__08_1__PlasmaArterial____Brain_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_1__PlasmaArterial____Brain']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_2__Brain____PlasmaVenous_19_task2` is invalid.\n - Variable `p2__08_2__Brain____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_2__Brain____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_1__PlasmaArterial____Muscle_20_task2` is invalid.\n - Variable `p2__09_1__PlasmaArterial____Muscle_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_1__PlasmaArterial____Muscle']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_2__Muscle____PlasmaVenous_21_task2` is invalid.\n - Variable `p2__09_2__Muscle____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_2__Muscle____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_1__PlasmaArterial____Adipose_22_task2` is invalid.\n - Variable `p2__10_1__PlasmaArterial____Adipose_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_1__PlasmaArterial____Adipose']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_2__Adipose____PlasmaVenous_23_task2` is invalid.\n - Variable `p2__10_2__Adipose____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_2__Adipose____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_1__PlasmaArterial____Heart_24_task2` is invalid.\n - Variable `p2__11_1__PlasmaArterial____Heart_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_1__PlasmaArterial____Heart']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_2__Heart____PlasmaVenous_25_task2` is invalid.\n - Variable `p2__11_2__Heart____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_2__Heart____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_1__PlasmaArterial____Remainder_26_task2` is invalid.\n - Variable `p2__12_1__PlasmaArterial____Remainder_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_1__PlasmaArterial____Remainder']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_2__Remainder____PlasmaVenous_27_task2` is invalid.\n - Variable `p2__12_2__Remainder____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_2__Remainder____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_1__PlasmaArterial____KidneyPlasma_28_task2` is invalid.\n - Variable `p2__13_1__PlasmaArterial____KidneyPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_1__PlasmaArterial____KidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_2__KidneyPlasma____PlasmaVenous_29_task2` is invalid.\n - Variable `p2__13_2__KidneyPlasma____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_2__KidneyPlasma____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_3__KidneyPlasma____KidneyTubular_30_task2` is invalid.\n - Variable `p2__13_3__KidneyPlasma____KidneyTubular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_3__KidneyPlasma____KidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_4__KidneyPlasma____KidneyTissue_31_task2` is invalid.\n - Variable `p2__13_4__KidneyPlasma____KidneyTissue_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_4__KidneyPlasma____KidneyTissue']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_5__KidneyTissue____KidneyTubular_32_task2` is invalid.\n - Variable `p2__13_5__KidneyTissue____KidneyTubular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_5__KidneyTissue____KidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_6__KidneyTubular____UrineExternal_33_task2` is invalid.\n - Variable `p2__13_6__KidneyTubular____UrineExternal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_6__KidneyTubular____UrineExternal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgAdipose_1_task2` is invalid.\n - Variable `p3_mgAdipose_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mgBrain_2_task2` is invalid.\n - Variable `p3_mgBrain_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mgFeces_3_task2` is invalid.\n - Variable `p3_mgFeces_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mgHeart_4_task2` is invalid.\n - Variable `p3_mgHeart_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineEnterocytes_5_task2` is invalid.\n - Variable `p3_mgIntestineEnterocytes_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineLumen_6_task2` is invalid.\n - Variable `p3_mgIntestineLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineTotal_7_task2` is invalid.\n - Variable `p3_mgIntestineTotal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineVascular_8_task2` is invalid.\n - Variable `p3_mgIntestineVascular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyPlasma_9_task2` is invalid.\n - Variable `p3_mgKidneyPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTissues_10_task2` is invalid.\n - Variable `p3_mgKidneyTissues_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTissues']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTotal_11_task2` is invalid.\n - Variable `p3_mgKidneyTotal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTubular_12_task2` is invalid.\n - Variable `p3_mgKidneyTubular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLiver_13_task2` is invalid.\n - Variable `p3_mgLiver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLung_14_task2` is invalid.\n - Variable `p3_mgLung_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mgMuscle_15_task2` is invalid.\n - Variable `p3_mgMuscle_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPlasmaArterial_16_task2` is invalid.\n - Variable `p3_mgPlasmaArterial_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPlasmaVenous_17_task2` is invalid.\n - Variable `p3_mgPlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPortalVein_18_task2` is invalid.\n - Variable `p3_mgPortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mgRemainder_19_task2` is invalid.\n - Variable `p3_mgRemainder_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomach_20_task2` is invalid.\n - Variable `p3_mgStomach_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomachLumen_21_task2` is invalid.\n - Variable `p3_mgStomachLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgUrine_22_task2` is invalid.\n - Variable `p3_mgUrine_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgUrine']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Zake2021_Metformin+Mice+PO.sedml493948.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Zake2021_Metformin+Mice+PO.xml` has warnings.\r\n - 66 warnings of type SBML unit consistency (99508). The following is the first warning at line 1733, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'Ktp_Liver' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 66 warnings of type SBML unit consistency (20702). The following is the first warning at line 1733, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'Ktp_Liver' does not have a 'units' attribute.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 2595, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression '1000000 * Metformin_Dose_in_Plasma_in_mg / (129.16 * PlasmaVenous) * PlasmaVenous' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 83 warnings of type Modeling practice (80701). The following is the first warning at line 1733, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Ktp_Liver' does not have a 'units' attribute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_5_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_6_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Zake2021_Metformin+Mice+PO.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Zake2021_Metformin+Mice+PO.xml` has warnings.\r\n - 66 warnings of type SBML unit consistency (99508). The following is the first warning at line 1733, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'Ktp_Liver' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 66 warnings of type SBML unit consistency (20702). The following is the first warning at line 1733, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'Ktp_Liver' does not have a 'units' attribute.\r\n - 78 warnings of type SBML unit consistency (99505). The following is the first warning at line 2595, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression '1000000 * Metformin_Dose_in_Plasma_in_mg / (129.16 * PlasmaVenous) * PlasmaVenous' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 83 warnings of type Modeling practice (80701). The following is the first warning at line 1733, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Ktp_Liver' does not have a 'units' attribute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_5_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_6_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Zake2021_Metformin+Mice+PO.sedml493948.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Zake2021_Metformin+Mice+PO.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Ktp_IntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mAdipose_1_task1` is invalid.\n - Variable `p1_mAdipose_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mBrain_2_task1` is invalid.\n - Variable `p1_mBrain_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mFeces_3_task1` is invalid.\n - Variable `p1_mFeces_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mHeart_4_task1` is invalid.\n - Variable `p1_mHeart_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineEnterocytes_5_task1` is invalid.\n - Variable `p1_mIntestineEnterocytes_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineLumen_6_task1` is invalid.\n - Variable `p1_mIntestineLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineSum_7_task1` is invalid.\n - Variable `p1_mIntestineSum_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mIntestineSum']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineVascular_8_task1` is invalid.\n - Variable `p1_mIntestineVascular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyPlasma_9_task1` is invalid.\n - Variable `p1_mKidneyPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneySum_10_task1` is invalid.\n - Variable `p1_mKidneySum_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mKidneySum']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTissue_11_task1` is invalid.\n - Variable `p1_mKidneyTissue_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTissue']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTubular_12_task1` is invalid.\n - Variable `p1_mKidneyTubular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mLiver_13_task1` is invalid.\n - Variable `p1_mLiver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mLung_14_task1` is invalid.\n - Variable `p1_mLung_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mMuscle_15_task1` is invalid.\n - Variable `p1_mMuscle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaArterial_16_task1` is invalid.\n - Variable `p1_mPlasmaArterial_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaVenous_17_task1` is invalid.\n - Variable `p1_mPlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mPortalVein_18_task1` is invalid.\n - Variable `p1_mPortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mRemainder_19_task1` is invalid.\n - Variable `p1_mRemainder_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomach_20_task1` is invalid.\n - Variable `p1_mStomach_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomachLumen_21_task1` is invalid.\n - Variable `p1_mStomachLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mUrine_22_task1` is invalid.\n - Variable `p1_mUrine_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mUrine']` are not defined. No namespaces are defined for the target.\n - Data generator `_01__StomachLumen_____IntestineLumen_1_task1` is invalid.\n - Variable `p2__01__StomachLumen_____IntestineLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_01__StomachLumen_____IntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `_02__IntestineLumen____Feces_2_task1` is invalid.\n - Variable `p2__02__IntestineLumen____Feces_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_02__IntestineLumen____Feces']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_1__PlasmaArterial____IntestineVascular_3_task1` is invalid.\n - Variable `p2__03_1__PlasmaArterial____IntestineVascular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_1__PlasmaArterial____IntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_2__IntestineLumen____IntestineEnterocytes__PMAT_OCT3_4_task1` is invalid.\n - Variable `p2__03_2__IntestineLumen____IntestineEnterocytes__PMAT_OCT3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_2__IntestineLumen____IntestineEnterocytes__PMAT_OCT3']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_3__IntestineEnterocytes____IntestineVascular__OCT1_5_task1` is invalid.\n - Variable `p2__03_3__IntestineEnterocytes____IntestineVascular__OCT1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_3__IntestineEnterocytes____IntestineVascular__OCT1']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_4__IntestineLumen____IntestineVascular__Saturable_6_task1` is invalid.\n - Variable `p2__03_4__IntestineLumen____IntestineVascular__Saturable_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_4__IntestineLumen____IntestineVascular__Saturable']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_5__IntestineVascular____PortalVein_7_task1` is invalid.\n - Variable `p2__03_5__IntestineVascular____PortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_5__IntestineVascular____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_6__IntestineLumen____IntestineEnterocytes__Diffusion_8_task1` is invalid.\n - Variable `p2__03_6__IntestineLumen____IntestineEnterocytes__Diffusion_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_6__IntestineLumen____IntestineEnterocytes__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_7__IntestineLumen____IntestineVascular__Diffusion_9_task1` is invalid.\n - Variable `p2__03_7__IntestineLumen____IntestineVascular__Diffusion_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_7__IntestineLumen____IntestineVascular__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_04__PlasmaArterial____PortalVein_10_task1` is invalid.\n - Variable `p2__04__PlasmaArterial____PortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_04__PlasmaArterial____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_1__PlasmaArterial____Stomach_11_task1` is invalid.\n - Variable `p2__05_1__PlasmaArterial____Stomach_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_1__PlasmaArterial____Stomach']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_2__Stomach____PortalVein_12_task1` is invalid.\n - Variable `p2__05_2__Stomach____PortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_2__Stomach____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_1__PlasmaArterial____Liver_13_task1` is invalid.\n - Variable `p2__06_1__PlasmaArterial____Liver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_1__PlasmaArterial____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_2__PortalVein____Liver_14_task1` is invalid.\n - Variable `p2__06_2__PortalVein____Liver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_2__PortalVein____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_3__Liver____PlasmaVenous_15_task1` is invalid.\n - Variable `p2__06_3__Liver____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_3__Liver____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_1__PlasmaVenous____Lung_16_task1` is invalid.\n - Variable `p2__07_1__PlasmaVenous____Lung_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_1__PlasmaVenous____Lung']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_2__Lung____PlasmaArterial_17_task1` is invalid.\n - Variable `p2__07_2__Lung____PlasmaArterial_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_2__Lung____PlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_1__PlasmaArterial____Brain_18_task1` is invalid.\n - Variable `p2__08_1__PlasmaArterial____Brain_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_1__PlasmaArterial____Brain']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_2__Brain____PlasmaVenous_19_task1` is invalid.\n - Variable `p2__08_2__Brain____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_2__Brain____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_1__PlasmaArterial____Muscle_20_task1` is invalid.\n - Variable `p2__09_1__PlasmaArterial____Muscle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_1__PlasmaArterial____Muscle']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_2__Muscle____PlasmaVenous_21_task1` is invalid.\n - Variable `p2__09_2__Muscle____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_2__Muscle____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_1__PlasmaArterial____Adipose_22_task1` is invalid.\n - Variable `p2__10_1__PlasmaArterial____Adipose_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_1__PlasmaArterial____Adipose']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_2__Adipose____PlasmaVenous_23_task1` is invalid.\n - Variable `p2__10_2__Adipose____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_2__Adipose____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_1__PlasmaArterial____Heart_24_task1` is invalid.\n - Variable `p2__11_1__PlasmaArterial____Heart_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_1__PlasmaArterial____Heart']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_2__Heart____PlasmaVenous_25_task1` is invalid.\n - Variable `p2__11_2__Heart____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_2__Heart____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_1__PlasmaArterial____Remainder_26_task1` is invalid.\n - Variable `p2__12_1__PlasmaArterial____Remainder_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_1__PlasmaArterial____Remainder']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_2__Remainder____PlasmaVenous_27_task1` is invalid.\n - Variable `p2__12_2__Remainder____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_2__Remainder____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_1__PlasmaArterial____KidneyPlasma_28_task1` is invalid.\n - Variable `p2__13_1__PlasmaArterial____KidneyPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_1__PlasmaArterial____KidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_2__KidneyPlasma____PlasmaVenous_29_task1` is invalid.\n - Variable `p2__13_2__KidneyPlasma____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_2__KidneyPlasma____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_3__KidneyPlasma____KidneyTubular_30_task1` is invalid.\n - Variable `p2__13_3__KidneyPlasma____KidneyTubular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_3__KidneyPlasma____KidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_4__KidneyPlasma____KidneyTissue_31_task1` is invalid.\n - Variable `p2__13_4__KidneyPlasma____KidneyTissue_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_4__KidneyPlasma____KidneyTissue']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_5__KidneyTissue____KidneyTubular_32_task1` is invalid.\n - Variable `p2__13_5__KidneyTissue____KidneyTubular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_5__KidneyTissue____KidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_6__KidneyTubular____UrineExternal_33_task1` is invalid.\n - Variable `p2__13_6__KidneyTubular____UrineExternal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_6__KidneyTubular____UrineExternal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgAdipose_1_task1` is invalid.\n - Variable `p3_mgAdipose_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mgBrain_2_task1` is invalid.\n - Variable `p3_mgBrain_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mgFeces_3_task1` is invalid.\n - Variable `p3_mgFeces_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mgHeart_4_task1` is invalid.\n - Variable `p3_mgHeart_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineEnterocytes_5_task1` is invalid.\n - Variable `p3_mgIntestineEnterocytes_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineLumen_6_task1` is invalid.\n - Variable `p3_mgIntestineLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineTotal_7_task1` is invalid.\n - Variable `p3_mgIntestineTotal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineVascular_8_task1` is invalid.\n - Variable `p3_mgIntestineVascular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyPlasma_9_task1` is invalid.\n - Variable `p3_mgKidneyPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTissues_10_task1` is invalid.\n - Variable `p3_mgKidneyTissues_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTissues']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTotal_11_task1` is invalid.\n - Variable `p3_mgKidneyTotal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTubular_12_task1` is invalid.\n - Variable `p3_mgKidneyTubular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLiver_13_task1` is invalid.\n - Variable `p3_mgLiver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLung_14_task1` is invalid.\n - Variable `p3_mgLung_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mgMuscle_15_task1` is invalid.\n - Variable `p3_mgMuscle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPlasmaArterial_16_task1` is invalid.\n - Variable `p3_mgPlasmaArterial_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPlasmaVenous_17_task1` is invalid.\n - Variable `p3_mgPlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPortalVein_18_task1` is invalid.\n - Variable `p3_mgPortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mgRemainder_19_task1` is invalid.\n - Variable `p3_mgRemainder_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomach_20_task1` is invalid.\n - Variable `p3_mgStomach_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomachLumen_21_task1` is invalid.\n - Variable `p3_mgStomachLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgUrine_22_task1` is invalid.\n - Variable `p3_mgUrine_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgUrine']` are not defined. No namespaces are defined for the target.\n - Data generator `mAdipose_1_task2` is invalid.\n - Variable `p1_mAdipose_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mBrain_2_task2` is invalid.\n - Variable `p1_mBrain_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mFeces_3_task2` is invalid.\n - Variable `p1_mFeces_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mHeart_4_task2` is invalid.\n - Variable `p1_mHeart_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineEnterocytes_5_task2` is invalid.\n - Variable `p1_mIntestineEnterocytes_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineLumen_6_task2` is invalid.\n - Variable `p1_mIntestineLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineSum_7_task2` is invalid.\n - Variable `p1_mIntestineSum_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mIntestineSum']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineVascular_8_task2` is invalid.\n - Variable `p1_mIntestineVascular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyPlasma_9_task2` is invalid.\n - Variable `p1_mKidneyPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneySum_10_task2` is invalid.\n - Variable `p1_mKidneySum_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mKidneySum']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTissue_11_task2` is invalid.\n - Variable `p1_mKidneyTissue_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTissue']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTubular_12_task2` is invalid.\n - Variable `p1_mKidneyTubular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mLiver_13_task2` is invalid.\n - Variable `p1_mLiver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mLung_14_task2` is invalid.\n - Variable `p1_mLung_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mMuscle_15_task2` is invalid.\n - Variable `p1_mMuscle_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaArterial_16_task2` is invalid.\n - Variable `p1_mPlasmaArterial_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaVenous_17_task2` is invalid.\n - Variable `p1_mPlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mPortalVein_18_task2` is invalid.\n - Variable `p1_mPortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mRemainder_19_task2` is invalid.\n - Variable `p1_mRemainder_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomach_20_task2` is invalid.\n - Variable `p1_mStomach_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomachLumen_21_task2` is invalid.\n - Variable `p1_mStomachLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mUrine_22_task2` is invalid.\n - Variable `p1_mUrine_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mUrine']` are not defined. No namespaces are defined for the target.\n - Data generator `_01__StomachLumen_____IntestineLumen_1_task2` is invalid.\n - Variable `p2__01__StomachLumen_____IntestineLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_01__StomachLumen_____IntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `_02__IntestineLumen____Feces_2_task2` is invalid.\n - Variable `p2__02__IntestineLumen____Feces_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_02__IntestineLumen____Feces']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_1__PlasmaArterial____IntestineVascular_3_task2` is invalid.\n - Variable `p2__03_1__PlasmaArterial____IntestineVascular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_1__PlasmaArterial____IntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_2__IntestineLumen____IntestineEnterocytes__PMAT_OCT3_4_task2` is invalid.\n - Variable `p2__03_2__IntestineLumen____IntestineEnterocytes__PMAT_OCT3_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_2__IntestineLumen____IntestineEnterocytes__PMAT_OCT3']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_3__IntestineEnterocytes____IntestineVascular__OCT1_5_task2` is invalid.\n - Variable `p2__03_3__IntestineEnterocytes____IntestineVascular__OCT1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_3__IntestineEnterocytes____IntestineVascular__OCT1']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_4__IntestineLumen____IntestineVascular__Saturable_6_task2` is invalid.\n - Variable `p2__03_4__IntestineLumen____IntestineVascular__Saturable_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_4__IntestineLumen____IntestineVascular__Saturable']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_5__IntestineVascular____PortalVein_7_task2` is invalid.\n - Variable `p2__03_5__IntestineVascular____PortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_5__IntestineVascular____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_6__IntestineLumen____IntestineEnterocytes__Diffusion_8_task2` is invalid.\n - Variable `p2__03_6__IntestineLumen____IntestineEnterocytes__Diffusion_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_6__IntestineLumen____IntestineEnterocytes__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_7__IntestineLumen____IntestineVascular__Diffusion_9_task2` is invalid.\n - Variable `p2__03_7__IntestineLumen____IntestineVascular__Diffusion_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_7__IntestineLumen____IntestineVascular__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_04__PlasmaArterial____PortalVein_10_task2` is invalid.\n - Variable `p2__04__PlasmaArterial____PortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_04__PlasmaArterial____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_1__PlasmaArterial____Stomach_11_task2` is invalid.\n - Variable `p2__05_1__PlasmaArterial____Stomach_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_1__PlasmaArterial____Stomach']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_2__Stomach____PortalVein_12_task2` is invalid.\n - Variable `p2__05_2__Stomach____PortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_2__Stomach____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_1__PlasmaArterial____Liver_13_task2` is invalid.\n - Variable `p2__06_1__PlasmaArterial____Liver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_1__PlasmaArterial____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_2__PortalVein____Liver_14_task2` is invalid.\n - Variable `p2__06_2__PortalVein____Liver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_2__PortalVein____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_3__Liver____PlasmaVenous_15_task2` is invalid.\n - Variable `p2__06_3__Liver____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_3__Liver____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_1__PlasmaVenous____Lung_16_task2` is invalid.\n - Variable `p2__07_1__PlasmaVenous____Lung_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_1__PlasmaVenous____Lung']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_2__Lung____PlasmaArterial_17_task2` is invalid.\n - Variable `p2__07_2__Lung____PlasmaArterial_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_2__Lung____PlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_1__PlasmaArterial____Brain_18_task2` is invalid.\n - Variable `p2__08_1__PlasmaArterial____Brain_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_1__PlasmaArterial____Brain']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_2__Brain____PlasmaVenous_19_task2` is invalid.\n - Variable `p2__08_2__Brain____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_2__Brain____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_1__PlasmaArterial____Muscle_20_task2` is invalid.\n - Variable `p2__09_1__PlasmaArterial____Muscle_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_1__PlasmaArterial____Muscle']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_2__Muscle____PlasmaVenous_21_task2` is invalid.\n - Variable `p2__09_2__Muscle____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_2__Muscle____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_1__PlasmaArterial____Adipose_22_task2` is invalid.\n - Variable `p2__10_1__PlasmaArterial____Adipose_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_1__PlasmaArterial____Adipose']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_2__Adipose____PlasmaVenous_23_task2` is invalid.\n - Variable `p2__10_2__Adipose____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_2__Adipose____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_1__PlasmaArterial____Heart_24_task2` is invalid.\n - Variable `p2__11_1__PlasmaArterial____Heart_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_1__PlasmaArterial____Heart']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_2__Heart____PlasmaVenous_25_task2` is invalid.\n - Variable `p2__11_2__Heart____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_2__Heart____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_1__PlasmaArterial____Remainder_26_task2` is invalid.\n - Variable `p2__12_1__PlasmaArterial____Remainder_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_1__PlasmaArterial____Remainder']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_2__Remainder____PlasmaVenous_27_task2` is invalid.\n - Variable `p2__12_2__Remainder____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_2__Remainder____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_1__PlasmaArterial____KidneyPlasma_28_task2` is invalid.\n - Variable `p2__13_1__PlasmaArterial____KidneyPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_1__PlasmaArterial____KidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_2__KidneyPlasma____PlasmaVenous_29_task2` is invalid.\n - Variable `p2__13_2__KidneyPlasma____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_2__KidneyPlasma____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_3__KidneyPlasma____KidneyTubular_30_task2` is invalid.\n - Variable `p2__13_3__KidneyPlasma____KidneyTubular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_3__KidneyPlasma____KidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_4__KidneyPlasma____KidneyTissue_31_task2` is invalid.\n - Variable `p2__13_4__KidneyPlasma____KidneyTissue_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_4__KidneyPlasma____KidneyTissue']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_5__KidneyTissue____KidneyTubular_32_task2` is invalid.\n - Variable `p2__13_5__KidneyTissue____KidneyTubular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_5__KidneyTissue____KidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_6__KidneyTubular____UrineExternal_33_task2` is invalid.\n - Variable `p2__13_6__KidneyTubular____UrineExternal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_6__KidneyTubular____UrineExternal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgAdipose_1_task2` is invalid.\n - Variable `p3_mgAdipose_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mgBrain_2_task2` is invalid.\n - Variable `p3_mgBrain_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mgFeces_3_task2` is invalid.\n - Variable `p3_mgFeces_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mgHeart_4_task2` is invalid.\n - Variable `p3_mgHeart_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineEnterocytes_5_task2` is invalid.\n - Variable `p3_mgIntestineEnterocytes_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineLumen_6_task2` is invalid.\n - Variable `p3_mgIntestineLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineTotal_7_task2` is invalid.\n - Variable `p3_mgIntestineTotal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineVascular_8_task2` is invalid.\n - Variable `p3_mgIntestineVascular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyPlasma_9_task2` is invalid.\n - Variable `p3_mgKidneyPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTissues_10_task2` is invalid.\n - Variable `p3_mgKidneyTissues_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTissues']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTotal_11_task2` is invalid.\n - Variable `p3_mgKidneyTotal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTubular_12_task2` is invalid.\n - Variable `p3_mgKidneyTubular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLiver_13_task2` is invalid.\n - Variable `p3_mgLiver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLung_14_task2` is invalid.\n - Variable `p3_mgLung_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mgMuscle_15_task2` is invalid.\n - Variable `p3_mgMuscle_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPlasmaArterial_16_task2` is invalid.\n - Variable `p3_mgPlasmaArterial_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPlasmaVenous_17_task2` is invalid.\n - Variable `p3_mgPlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPortalVein_18_task2` is invalid.\n - Variable `p3_mgPortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mgRemainder_19_task2` is invalid.\n - Variable `p3_mgRemainder_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomach_20_task2` is invalid.\n - Variable `p3_mgStomach_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomachLumen_21_task2` is invalid.\n - Variable `p3_mgStomachLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgUrine_22_task2` is invalid.\n - Variable `p3_mgUrine_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgUrine']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000001028](https://www.ebi.ac.uk/biomodels/BIOMD0000001028)<br/><sup>Zake2021 - PBPK model of metfo</sup></summary><sup>Zake2021 - PBPK model of metformin in humans, single PO dose</sup></details>|<details><summary>pass</summary>[Zake2021_Metformin+Human+single+PO+dose.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001028#Files)</details>|pass|<details><summary>pass</summary>[Zake2021_Metformin+Human+single+PO+dose.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001028#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Zake2021_Metformin+Human+single+PO+dose.sedml762786.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Zake2021_Metformin+Human+single+PO+dose.xml` has warnings.\r\n - 81 warnings of type SBML unit consistency (99508). The following is the first warning at line 1819, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'Ktp_Liver' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 81 warnings of type SBML unit consistency (20702). The following is the first warning at line 1819, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'Ktp_Liver' does not have a 'units' attribute.\r\n - 92 warnings of type SBML unit consistency (99505). The following is the first warning at line 2981, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Intestine_Coefficient' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 91 warnings of type Modeling practice (80701). The following is the first warning at line 1819, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Ktp_Liver' does not have a 'units' attribute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_5_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_6_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Zake2021_Metformin+Human+single+PO+dose.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Zake2021_Metformin+Human+single+PO+dose.xml` has warnings.\r\n - 81 warnings of type SBML unit consistency (99508). The following is the first warning at line 1819, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'Ktp_Liver' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 81 warnings of type SBML unit consistency (20702). The following is the first warning at line 1819, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'Ktp_Liver' does not have a 'units' attribute.\r\n - 92 warnings of type SBML unit consistency (99505). The following is the first warning at line 2981, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Intestine_Coefficient' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 91 warnings of type Modeling practice (80701). The following is the first warning at line 1819, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Ktp_Liver' does not have a 'units' attribute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_5_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_6_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/Zake2021_Metformin+Human+single+PO+dose.sedml762786.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Zake2021_Metformin+Human+single+PO+dose.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Metformin_Dose_in_Lumen_in_mg']` are not defined. No namespaces are defined for the target.\n - Data generator `mgAdipose_1_task1` is invalid.\n - Variable `p1_mgAdipose_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mgArterialPlasma_2_task1` is invalid.\n - Variable `p1_mgArterialPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgArterialPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mgBrain_3_task1` is invalid.\n - Variable `p1_mgBrain_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mgEnterocytes_4_task1` is invalid.\n - Variable `p1_mgEnterocytes_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mgFeces_5_task1` is invalid.\n - Variable `p1_mgFeces_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mgHeart_6_task1` is invalid.\n - Variable `p1_mgHeart_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineLumen_7_task1` is invalid.\n - Variable `p1_mgIntestineLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineTotal_8_task1` is invalid.\n - Variable `p1_mgIntestineTotal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineVascular_9_task1` is invalid.\n - Variable `p1_mgIntestineVascular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTubular_10_task1` is invalid.\n - Variable `p1_mgKidneyTubular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyPlasma_11_task1` is invalid.\n - Variable `p1_mgKidneyPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTissues_12_task1` is invalid.\n - Variable `p1_mgKidneyTissues_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTissues']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTotal_13_task1` is invalid.\n - Variable `p1_mgKidneyTotal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLiver_14_task1` is invalid.\n - Variable `p1_mgLiver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLung_15_task1` is invalid.\n - Variable `p1_mgLung_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mgMuscle_16_task1` is invalid.\n - Variable `p1_mgMuscle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPortalVein_17_task1` is invalid.\n - Variable `p1_mgPortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mgRBC_18_task1` is invalid.\n - Variable `p1_mgRBC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgRBC']` are not defined. No namespaces are defined for the target.\n - Data generator `mgRemainder_19_task1` is invalid.\n - Variable `p1_mgRemainder_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomach_20_task1` is invalid.\n - Variable `p1_mgStomach_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomachLumen_21_task1` is invalid.\n - Variable `p1_mgStomachLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgUrineSum_22_task1` is invalid.\n - Variable `p1_mgUrineSum_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgUrineSum']` are not defined. No namespaces are defined for the target.\n - Data generator `mgVenousPlasma_23_task1` is invalid.\n - Variable `p1_mgVenousPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgVenousPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mAdipose_1_task1` is invalid.\n - Variable `p2_mAdipose_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mBrain_2_task1` is invalid.\n - Variable `p2_mBrain_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mEnterocytes_3_task1` is invalid.\n - Variable `p2_mEnterocytes_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mFeces_4_task1` is invalid.\n - Variable `p2_mFeces_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mHeart_5_task1` is invalid.\n - Variable `p2_mHeart_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineLumen_6_task1` is invalid.\n - Variable `p2_mIntestineLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineTotal_7_task1` is invalid.\n - Variable `p2_mIntestineTotal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mIntestineTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineVascular_8_task1` is invalid.\n - Variable `p2_mIntestineVascular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyPlasma_9_task1` is invalid.\n - Variable `p2_mKidneyPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTotal_10_task1` is invalid.\n - Variable `p2_mKidneyTotal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mKidneyTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTissue_11_task1` is invalid.\n - Variable `p2_mKidneyTissue_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTissue']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTubular_12_task1` is invalid.\n - Variable `p2_mKidneyTubular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mLiver_13_task1` is invalid.\n - Variable `p2_mLiver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mLung_14_task1` is invalid.\n - Variable `p2_mLung_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mMuscle_15_task1` is invalid.\n - Variable `p2_mMuscle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaArterial_16_task1` is invalid.\n - Variable `p2_mPlasmaArterial_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaVenous_17_task1` is invalid.\n - Variable `p2_mPlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mPortalVein_18_task1` is invalid.\n - Variable `p2_mPortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mRBC_19_task1` is invalid.\n - Variable `p2_mRBC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mRBC']` are not defined. No namespaces are defined for the target.\n - Data generator `mRemainder_20_task1` is invalid.\n - Variable `p2_mRemainder_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomach_21_task1` is invalid.\n - Variable `p2_mStomach_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomachLumen_22_task1` is invalid.\n - Variable `p2_mStomachLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mUrineExternal_23_task1` is invalid.\n - Variable `p2_mUrineExternal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mUrineExternal']` are not defined. No namespaces are defined for the target.\n - Data generator `_01__StomachLumen_____IntestineLumen_1_task1` is invalid.\n - Variable `p3__01__StomachLumen_____IntestineLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_01__StomachLumen_____IntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `_02__IntestineLumen____Feces_2_task1` is invalid.\n - Variable `p3__02__IntestineLumen____Feces_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_02__IntestineLumen____Feces']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_1__PlasmaArterial____IntestineVascular_3_task1` is invalid.\n - Variable `p3__03_1__PlasmaArterial____IntestineVascular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_1__PlasmaArterial____IntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_2__IntestineLumen____Enterocytes__PMAT_OCT3_4_task1` is invalid.\n - Variable `p3__03_2__IntestineLumen____Enterocytes__PMAT_OCT3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_2__IntestineLumen____Enterocytes__PMAT_OCT3']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_3__Enterocytes____IntestineVascular__OCT1_5_task1` is invalid.\n - Variable `p3__03_3__Enterocytes____IntestineVascular__OCT1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_3__Enterocytes____IntestineVascular__OCT1']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_4__IntestineLumen____IntestineVascular__Saturable_0_6_task1` is invalid.\n - Variable `p3__03_4__IntestineLumen____IntestineVascular__Saturable_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_4__IntestineLumen____IntestineVascular__Saturable_0']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_5__IntestineVascular____PortalVein_7_task1` is invalid.\n - Variable `p3__03_5__IntestineVascular____PortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_5__IntestineVascular____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_6__IntestineLumen____Enterocytes__Diffusion_8_task1` is invalid.\n - Variable `p3__03_6__IntestineLumen____Enterocytes__Diffusion_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_6__IntestineLumen____Enterocytes__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_7__IntestineLumen____IntestineVascular__Diffusion_9_task1` is invalid.\n - Variable `p3__03_7__IntestineLumen____IntestineVascular__Diffusion_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_7__IntestineLumen____IntestineVascular__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_04__PlasmaArterial____PortalVein_10_task1` is invalid.\n - Variable `p3__04__PlasmaArterial____PortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_04__PlasmaArterial____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_1__PlasmaArterial____Stomach_11_task1` is invalid.\n - Variable `p3__05_1__PlasmaArterial____Stomach_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_1__PlasmaArterial____Stomach']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_2__Stomach____PortalVein_12_task1` is invalid.\n - Variable `p3__05_2__Stomach____PortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_2__Stomach____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_1__PlasmaArterial____Liver_13_task1` is invalid.\n - Variable `p3__06_1__PlasmaArterial____Liver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_1__PlasmaArterial____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_2__PortalVein____Liver_14_task1` is invalid.\n - Variable `p3__06_2__PortalVein____Liver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_2__PortalVein____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_3__Liver____PlasmaVenous_15_task1` is invalid.\n - Variable `p3__06_3__Liver____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_3__Liver____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_1__PlasmaVenous____Lung_16_task1` is invalid.\n - Variable `p3__07_1__PlasmaVenous____Lung_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_1__PlasmaVenous____Lung']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_2__Lung____PlasmaArterial_17_task1` is invalid.\n - Variable `p3__07_2__Lung____PlasmaArterial_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_2__Lung____PlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_1__PlasmaArterial____Brain_18_task1` is invalid.\n - Variable `p3__08_1__PlasmaArterial____Brain_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_1__PlasmaArterial____Brain']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_2__Brain____PlasmaVenous_19_task1` is invalid.\n - Variable `p3__08_2__Brain____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_2__Brain____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_1__PlasmaArterial____Muscle_20_task1` is invalid.\n - Variable `p3__09_1__PlasmaArterial____Muscle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_1__PlasmaArterial____Muscle']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_2__Muscle____PlasmaVenous_21_task1` is invalid.\n - Variable `p3__09_2__Muscle____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_2__Muscle____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_1__PlasmaArterial____Adipose_22_task1` is invalid.\n - Variable `p3__10_1__PlasmaArterial____Adipose_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_1__PlasmaArterial____Adipose']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_2__Adipose____PlasmaVenous_23_task1` is invalid.\n - Variable `p3__10_2__Adipose____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_2__Adipose____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_1__PlasmaArterial____Heart_24_task1` is invalid.\n - Variable `p3__11_1__PlasmaArterial____Heart_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_1__PlasmaArterial____Heart']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_2__Heart____PlasmaVenous_25_task1` is invalid.\n - Variable `p3__11_2__Heart____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_2__Heart____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_1__PlasmaArterial____Remainder_26_task1` is invalid.\n - Variable `p3__12_1__PlasmaArterial____Remainder_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_1__PlasmaArterial____Remainder']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_2__Remainder____PlasmaVenous_27_task1` is invalid.\n - Variable `p3__12_2__Remainder____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_2__Remainder____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_1__PlasmaArterial____KidneyPlasma_28_task1` is invalid.\n - Variable `p3__13_1__PlasmaArterial____KidneyPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_1__PlasmaArterial____KidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_2__KidneyPlasma____PlasmaVenous_29_task1` is invalid.\n - Variable `p3__13_2__KidneyPlasma____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_2__KidneyPlasma____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_3__PlasmaArterial____KidneyTubular_30_task1` is invalid.\n - Variable `p3__13_3__PlasmaArterial____KidneyTubular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_3__PlasmaArterial____KidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_4__KidneyPlasma____KidneyTissue_0_31_task1` is invalid.\n - Variable `p3__13_4__KidneyPlasma____KidneyTissue_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_4__KidneyPlasma____KidneyTissue_0']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_5__KidneyTissue____KidneyTubular_0_32_task1` is invalid.\n - Variable `p3__13_5__KidneyTissue____KidneyTubular_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_5__KidneyTissue____KidneyTubular_0']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_6__KidneyTubular____UrineExternal_33_task1` is invalid.\n - Variable `p3__13_6__KidneyTubular____UrineExternal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_6__KidneyTubular____UrineExternal']` are not defined. No namespaces are defined for the target.\n - Data generator `_14_1__PlasmaVenous____RBC_34_task1` is invalid.\n - Variable `p3__14_1__PlasmaVenous____RBC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_14_1__PlasmaVenous____RBC']` are not defined. No namespaces are defined for the target.\n - Data generator `_14_2__RBC____PlasmaVenous_35_task1` is invalid.\n - Variable `p3__14_2__RBC____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_14_2__RBC____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mgAdipose_1_task2` is invalid.\n - Variable `p1_mgAdipose_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mgArterialPlasma_2_task2` is invalid.\n - Variable `p1_mgArterialPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgArterialPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mgBrain_3_task2` is invalid.\n - Variable `p1_mgBrain_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mgEnterocytes_4_task2` is invalid.\n - Variable `p1_mgEnterocytes_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mgFeces_5_task2` is invalid.\n - Variable `p1_mgFeces_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mgHeart_6_task2` is invalid.\n - Variable `p1_mgHeart_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineLumen_7_task2` is invalid.\n - Variable `p1_mgIntestineLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineTotal_8_task2` is invalid.\n - Variable `p1_mgIntestineTotal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineVascular_9_task2` is invalid.\n - Variable `p1_mgIntestineVascular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTubular_10_task2` is invalid.\n - Variable `p1_mgKidneyTubular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyPlasma_11_task2` is invalid.\n - Variable `p1_mgKidneyPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTissues_12_task2` is invalid.\n - Variable `p1_mgKidneyTissues_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTissues']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTotal_13_task2` is invalid.\n - Variable `p1_mgKidneyTotal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLiver_14_task2` is invalid.\n - Variable `p1_mgLiver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLung_15_task2` is invalid.\n - Variable `p1_mgLung_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mgMuscle_16_task2` is invalid.\n - Variable `p1_mgMuscle_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPortalVein_17_task2` is invalid.\n - Variable `p1_mgPortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mgRBC_18_task2` is invalid.\n - Variable `p1_mgRBC_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgRBC']` are not defined. No namespaces are defined for the target.\n - Data generator `mgRemainder_19_task2` is invalid.\n - Variable `p1_mgRemainder_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomach_20_task2` is invalid.\n - Variable `p1_mgStomach_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomachLumen_21_task2` is invalid.\n - Variable `p1_mgStomachLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgUrineSum_22_task2` is invalid.\n - Variable `p1_mgUrineSum_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgUrineSum']` are not defined. No namespaces are defined for the target.\n - Data generator `mgVenousPlasma_23_task2` is invalid.\n - Variable `p1_mgVenousPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgVenousPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mAdipose_1_task2` is invalid.\n - Variable `p2_mAdipose_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mBrain_2_task2` is invalid.\n - Variable `p2_mBrain_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mEnterocytes_3_task2` is invalid.\n - Variable `p2_mEnterocytes_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mFeces_4_task2` is invalid.\n - Variable `p2_mFeces_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mHeart_5_task2` is invalid.\n - Variable `p2_mHeart_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineLumen_6_task2` is invalid.\n - Variable `p2_mIntestineLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineTotal_7_task2` is invalid.\n - Variable `p2_mIntestineTotal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mIntestineTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineVascular_8_task2` is invalid.\n - Variable `p2_mIntestineVascular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyPlasma_9_task2` is invalid.\n - Variable `p2_mKidneyPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTotal_10_task2` is invalid.\n - Variable `p2_mKidneyTotal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mKidneyTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTissue_11_task2` is invalid.\n - Variable `p2_mKidneyTissue_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTissue']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTubular_12_task2` is invalid.\n - Variable `p2_mKidneyTubular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mLiver_13_task2` is invalid.\n - Variable `p2_mLiver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mLung_14_task2` is invalid.\n - Variable `p2_mLung_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mMuscle_15_task2` is invalid.\n - Variable `p2_mMuscle_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaArterial_16_task2` is invalid.\n - Variable `p2_mPlasmaArterial_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaVenous_17_task2` is invalid.\n - Variable `p2_mPlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mPortalVein_18_task2` is invalid.\n - Variable `p2_mPortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mRBC_19_task2` is invalid.\n - Variable `p2_mRBC_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mRBC']` are not defined. No namespaces are defined for the target.\n - Data generator `mRemainder_20_task2` is invalid.\n - Variable `p2_mRemainder_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomach_21_task2` is invalid.\n - Variable `p2_mStomach_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomachLumen_22_task2` is invalid.\n - Variable `p2_mStomachLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mUrineExternal_23_task2` is invalid.\n - Variable `p2_mUrineExternal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mUrineExternal']` are not defined. No namespaces are defined for the target.\n - Data generator `_01__StomachLumen_____IntestineLumen_1_task2` is invalid.\n - Variable `p3__01__StomachLumen_____IntestineLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_01__StomachLumen_____IntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `_02__IntestineLumen____Feces_2_task2` is invalid.\n - Variable `p3__02__IntestineLumen____Feces_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_02__IntestineLumen____Feces']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_1__PlasmaArterial____IntestineVascular_3_task2` is invalid.\n - Variable `p3__03_1__PlasmaArterial____IntestineVascular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_1__PlasmaArterial____IntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_2__IntestineLumen____Enterocytes__PMAT_OCT3_4_task2` is invalid.\n - Variable `p3__03_2__IntestineLumen____Enterocytes__PMAT_OCT3_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_2__IntestineLumen____Enterocytes__PMAT_OCT3']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_3__Enterocytes____IntestineVascular__OCT1_5_task2` is invalid.\n - Variable `p3__03_3__Enterocytes____IntestineVascular__OCT1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_3__Enterocytes____IntestineVascular__OCT1']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_4__IntestineLumen____IntestineVascular__Saturable_0_6_task2` is invalid.\n - Variable `p3__03_4__IntestineLumen____IntestineVascular__Saturable_0_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_4__IntestineLumen____IntestineVascular__Saturable_0']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_5__IntestineVascular____PortalVein_7_task2` is invalid.\n - Variable `p3__03_5__IntestineVascular____PortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_5__IntestineVascular____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_6__IntestineLumen____Enterocytes__Diffusion_8_task2` is invalid.\n - Variable `p3__03_6__IntestineLumen____Enterocytes__Diffusion_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_6__IntestineLumen____Enterocytes__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_7__IntestineLumen____IntestineVascular__Diffusion_9_task2` is invalid.\n - Variable `p3__03_7__IntestineLumen____IntestineVascular__Diffusion_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_7__IntestineLumen____IntestineVascular__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_04__PlasmaArterial____PortalVein_10_task2` is invalid.\n - Variable `p3__04__PlasmaArterial____PortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_04__PlasmaArterial____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_1__PlasmaArterial____Stomach_11_task2` is invalid.\n - Variable `p3__05_1__PlasmaArterial____Stomach_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_1__PlasmaArterial____Stomach']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_2__Stomach____PortalVein_12_task2` is invalid.\n - Variable `p3__05_2__Stomach____PortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_2__Stomach____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_1__PlasmaArterial____Liver_13_task2` is invalid.\n - Variable `p3__06_1__PlasmaArterial____Liver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_1__PlasmaArterial____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_2__PortalVein____Liver_14_task2` is invalid.\n - Variable `p3__06_2__PortalVein____Liver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_2__PortalVein____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_3__Liver____PlasmaVenous_15_task2` is invalid.\n - Variable `p3__06_3__Liver____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_3__Liver____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_1__PlasmaVenous____Lung_16_task2` is invalid.\n - Variable `p3__07_1__PlasmaVenous____Lung_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_1__PlasmaVenous____Lung']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_2__Lung____PlasmaArterial_17_task2` is invalid.\n - Variable `p3__07_2__Lung____PlasmaArterial_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_2__Lung____PlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_1__PlasmaArterial____Brain_18_task2` is invalid.\n - Variable `p3__08_1__PlasmaArterial____Brain_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_1__PlasmaArterial____Brain']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_2__Brain____PlasmaVenous_19_task2` is invalid.\n - Variable `p3__08_2__Brain____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_2__Brain____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_1__PlasmaArterial____Muscle_20_task2` is invalid.\n - Variable `p3__09_1__PlasmaArterial____Muscle_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_1__PlasmaArterial____Muscle']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_2__Muscle____PlasmaVenous_21_task2` is invalid.\n - Variable `p3__09_2__Muscle____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_2__Muscle____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_1__PlasmaArterial____Adipose_22_task2` is invalid.\n - Variable `p3__10_1__PlasmaArterial____Adipose_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_1__PlasmaArterial____Adipose']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_2__Adipose____PlasmaVenous_23_task2` is invalid.\n - Variable `p3__10_2__Adipose____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_2__Adipose____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_1__PlasmaArterial____Heart_24_task2` is invalid.\n - Variable `p3__11_1__PlasmaArterial____Heart_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_1__PlasmaArterial____Heart']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_2__Heart____PlasmaVenous_25_task2` is invalid.\n - Variable `p3__11_2__Heart____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_2__Heart____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_1__PlasmaArterial____Remainder_26_task2` is invalid.\n - Variable `p3__12_1__PlasmaArterial____Remainder_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_1__PlasmaArterial____Remainder']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_2__Remainder____PlasmaVenous_27_task2` is invalid.\n - Variable `p3__12_2__Remainder____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_2__Remainder____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_1__PlasmaArterial____KidneyPlasma_28_task2` is invalid.\n - Variable `p3__13_1__PlasmaArterial____KidneyPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_1__PlasmaArterial____KidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_2__KidneyPlasma____PlasmaVenous_29_task2` is invalid.\n - Variable `p3__13_2__KidneyPlasma____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_2__KidneyPlasma____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_3__PlasmaArterial____KidneyTubular_30_task2` is invalid.\n - Variable `p3__13_3__PlasmaArterial____KidneyTubular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_3__PlasmaArterial____KidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_4__KidneyPlasma____KidneyTissue_0_31_task2` is invalid.\n - Variable `p3__13_4__KidneyPlasma____KidneyTissue_0_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_4__KidneyPlasma____KidneyTissue_0']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_5__KidneyTissue____KidneyTubular_0_32_task2` is invalid.\n - Variable `p3__13_5__KidneyTissue____KidneyTubular_0_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_5__KidneyTissue____KidneyTubular_0']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_6__KidneyTubular____UrineExternal_33_task2` is invalid.\n - Variable `p3__13_6__KidneyTubular____UrineExternal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_6__KidneyTubular____UrineExternal']` are not defined. No namespaces are defined for the target.\n - Data generator `_14_1__PlasmaVenous____RBC_34_task2` is invalid.\n - Variable `p3__14_1__PlasmaVenous____RBC_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_14_1__PlasmaVenous____RBC']` are not defined. No namespaces are defined for the target.\n - Data generator `_14_2__RBC____PlasmaVenous_35_task2` is invalid.\n - Variable `p3__14_2__RBC____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_14_2__RBC____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Zake2021_Metformin+Human+single+PO+dose.sedml599150.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Zake2021_Metformin+Human+single+PO+dose.xml` has warnings.\r\n - 81 warnings of type SBML unit consistency (99508). The following is the first warning at line 1819, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'Ktp_Liver' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 81 warnings of type SBML unit consistency (20702). The following is the first warning at line 1819, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'Ktp_Liver' does not have a 'units' attribute.\r\n - 92 warnings of type SBML unit consistency (99505). The following is the first warning at line 2981, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Intestine_Coefficient' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 91 warnings of type Modeling practice (80701). The following is the first warning at line 1819, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Ktp_Liver' does not have a 'units' attribute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_5_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_6_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Zake2021_Metformin+Human+single+PO+dose.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Zake2021_Metformin+Human+single+PO+dose.xml` has warnings.\r\n - 81 warnings of type SBML unit consistency (99508). The following is the first warning at line 1819, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'Ktp_Liver' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 81 warnings of type SBML unit consistency (20702). The following is the first warning at line 1819, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'Ktp_Liver' does not have a 'units' attribute.\r\n - 92 warnings of type SBML unit consistency (99505). The following is the first warning at line 2981, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Intestine_Coefficient' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 91 warnings of type Modeling practice (80701). The following is the first warning at line 1819, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Ktp_Liver' does not have a 'units' attribute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_5_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_6_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Zake2021_Metformin+Human+single+PO+dose.sedml599150.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Zake2021_Metformin+Human+single+PO+dose.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Metformin_Dose_in_Lumen_in_mg']` are not defined. No namespaces are defined for the target.\n - Data generator `mgAdipose_1_task1` is invalid.\n - Variable `p1_mgAdipose_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mgArterialPlasma_2_task1` is invalid.\n - Variable `p1_mgArterialPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgArterialPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mgBrain_3_task1` is invalid.\n - Variable `p1_mgBrain_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mgEnterocytes_4_task1` is invalid.\n - Variable `p1_mgEnterocytes_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mgFeces_5_task1` is invalid.\n - Variable `p1_mgFeces_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mgHeart_6_task1` is invalid.\n - Variable `p1_mgHeart_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineLumen_7_task1` is invalid.\n - Variable `p1_mgIntestineLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineTotal_8_task1` is invalid.\n - Variable `p1_mgIntestineTotal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineVascular_9_task1` is invalid.\n - Variable `p1_mgIntestineVascular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTubular_10_task1` is invalid.\n - Variable `p1_mgKidneyTubular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyPlasma_11_task1` is invalid.\n - Variable `p1_mgKidneyPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTissues_12_task1` is invalid.\n - Variable `p1_mgKidneyTissues_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTissues']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTotal_13_task1` is invalid.\n - Variable `p1_mgKidneyTotal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLiver_14_task1` is invalid.\n - Variable `p1_mgLiver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLung_15_task1` is invalid.\n - Variable `p1_mgLung_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mgMuscle_16_task1` is invalid.\n - Variable `p1_mgMuscle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPortalVein_17_task1` is invalid.\n - Variable `p1_mgPortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mgRBC_18_task1` is invalid.\n - Variable `p1_mgRBC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgRBC']` are not defined. No namespaces are defined for the target.\n - Data generator `mgRemainder_19_task1` is invalid.\n - Variable `p1_mgRemainder_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomach_20_task1` is invalid.\n - Variable `p1_mgStomach_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomachLumen_21_task1` is invalid.\n - Variable `p1_mgStomachLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgUrineSum_22_task1` is invalid.\n - Variable `p1_mgUrineSum_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgUrineSum']` are not defined. No namespaces are defined for the target.\n - Data generator `mgVenousPlasma_23_task1` is invalid.\n - Variable `p1_mgVenousPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgVenousPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mAdipose_1_task1` is invalid.\n - Variable `p2_mAdipose_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mBrain_2_task1` is invalid.\n - Variable `p2_mBrain_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mEnterocytes_3_task1` is invalid.\n - Variable `p2_mEnterocytes_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mFeces_4_task1` is invalid.\n - Variable `p2_mFeces_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mHeart_5_task1` is invalid.\n - Variable `p2_mHeart_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineLumen_6_task1` is invalid.\n - Variable `p2_mIntestineLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineTotal_7_task1` is invalid.\n - Variable `p2_mIntestineTotal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mIntestineTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineVascular_8_task1` is invalid.\n - Variable `p2_mIntestineVascular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyPlasma_9_task1` is invalid.\n - Variable `p2_mKidneyPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTotal_10_task1` is invalid.\n - Variable `p2_mKidneyTotal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mKidneyTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTissue_11_task1` is invalid.\n - Variable `p2_mKidneyTissue_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTissue']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTubular_12_task1` is invalid.\n - Variable `p2_mKidneyTubular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mLiver_13_task1` is invalid.\n - Variable `p2_mLiver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mLung_14_task1` is invalid.\n - Variable `p2_mLung_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mMuscle_15_task1` is invalid.\n - Variable `p2_mMuscle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaArterial_16_task1` is invalid.\n - Variable `p2_mPlasmaArterial_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaVenous_17_task1` is invalid.\n - Variable `p2_mPlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mPortalVein_18_task1` is invalid.\n - Variable `p2_mPortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mRBC_19_task1` is invalid.\n - Variable `p2_mRBC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mRBC']` are not defined. No namespaces are defined for the target.\n - Data generator `mRemainder_20_task1` is invalid.\n - Variable `p2_mRemainder_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomach_21_task1` is invalid.\n - Variable `p2_mStomach_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomachLumen_22_task1` is invalid.\n - Variable `p2_mStomachLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mUrineExternal_23_task1` is invalid.\n - Variable `p2_mUrineExternal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mUrineExternal']` are not defined. No namespaces are defined for the target.\n - Data generator `_01__StomachLumen_____IntestineLumen_1_task1` is invalid.\n - Variable `p3__01__StomachLumen_____IntestineLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_01__StomachLumen_____IntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `_02__IntestineLumen____Feces_2_task1` is invalid.\n - Variable `p3__02__IntestineLumen____Feces_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_02__IntestineLumen____Feces']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_1__PlasmaArterial____IntestineVascular_3_task1` is invalid.\n - Variable `p3__03_1__PlasmaArterial____IntestineVascular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_1__PlasmaArterial____IntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_2__IntestineLumen____Enterocytes__PMAT_OCT3_4_task1` is invalid.\n - Variable `p3__03_2__IntestineLumen____Enterocytes__PMAT_OCT3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_2__IntestineLumen____Enterocytes__PMAT_OCT3']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_3__Enterocytes____IntestineVascular__OCT1_5_task1` is invalid.\n - Variable `p3__03_3__Enterocytes____IntestineVascular__OCT1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_3__Enterocytes____IntestineVascular__OCT1']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_4__IntestineLumen____IntestineVascular__Saturable_0_6_task1` is invalid.\n - Variable `p3__03_4__IntestineLumen____IntestineVascular__Saturable_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_4__IntestineLumen____IntestineVascular__Saturable_0']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_5__IntestineVascular____PortalVein_7_task1` is invalid.\n - Variable `p3__03_5__IntestineVascular____PortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_5__IntestineVascular____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_6__IntestineLumen____Enterocytes__Diffusion_8_task1` is invalid.\n - Variable `p3__03_6__IntestineLumen____Enterocytes__Diffusion_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_6__IntestineLumen____Enterocytes__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_7__IntestineLumen____IntestineVascular__Diffusion_9_task1` is invalid.\n - Variable `p3__03_7__IntestineLumen____IntestineVascular__Diffusion_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_7__IntestineLumen____IntestineVascular__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_04__PlasmaArterial____PortalVein_10_task1` is invalid.\n - Variable `p3__04__PlasmaArterial____PortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_04__PlasmaArterial____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_1__PlasmaArterial____Stomach_11_task1` is invalid.\n - Variable `p3__05_1__PlasmaArterial____Stomach_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_1__PlasmaArterial____Stomach']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_2__Stomach____PortalVein_12_task1` is invalid.\n - Variable `p3__05_2__Stomach____PortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_2__Stomach____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_1__PlasmaArterial____Liver_13_task1` is invalid.\n - Variable `p3__06_1__PlasmaArterial____Liver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_1__PlasmaArterial____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_2__PortalVein____Liver_14_task1` is invalid.\n - Variable `p3__06_2__PortalVein____Liver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_2__PortalVein____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_3__Liver____PlasmaVenous_15_task1` is invalid.\n - Variable `p3__06_3__Liver____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_3__Liver____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_1__PlasmaVenous____Lung_16_task1` is invalid.\n - Variable `p3__07_1__PlasmaVenous____Lung_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_1__PlasmaVenous____Lung']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_2__Lung____PlasmaArterial_17_task1` is invalid.\n - Variable `p3__07_2__Lung____PlasmaArterial_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_2__Lung____PlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_1__PlasmaArterial____Brain_18_task1` is invalid.\n - Variable `p3__08_1__PlasmaArterial____Brain_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_1__PlasmaArterial____Brain']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_2__Brain____PlasmaVenous_19_task1` is invalid.\n - Variable `p3__08_2__Brain____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_2__Brain____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_1__PlasmaArterial____Muscle_20_task1` is invalid.\n - Variable `p3__09_1__PlasmaArterial____Muscle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_1__PlasmaArterial____Muscle']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_2__Muscle____PlasmaVenous_21_task1` is invalid.\n - Variable `p3__09_2__Muscle____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_2__Muscle____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_1__PlasmaArterial____Adipose_22_task1` is invalid.\n - Variable `p3__10_1__PlasmaArterial____Adipose_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_1__PlasmaArterial____Adipose']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_2__Adipose____PlasmaVenous_23_task1` is invalid.\n - Variable `p3__10_2__Adipose____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_2__Adipose____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_1__PlasmaArterial____Heart_24_task1` is invalid.\n - Variable `p3__11_1__PlasmaArterial____Heart_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_1__PlasmaArterial____Heart']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_2__Heart____PlasmaVenous_25_task1` is invalid.\n - Variable `p3__11_2__Heart____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_2__Heart____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_1__PlasmaArterial____Remainder_26_task1` is invalid.\n - Variable `p3__12_1__PlasmaArterial____Remainder_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_1__PlasmaArterial____Remainder']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_2__Remainder____PlasmaVenous_27_task1` is invalid.\n - Variable `p3__12_2__Remainder____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_2__Remainder____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_1__PlasmaArterial____KidneyPlasma_28_task1` is invalid.\n - Variable `p3__13_1__PlasmaArterial____KidneyPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_1__PlasmaArterial____KidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_2__KidneyPlasma____PlasmaVenous_29_task1` is invalid.\n - Variable `p3__13_2__KidneyPlasma____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_2__KidneyPlasma____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_3__PlasmaArterial____KidneyTubular_30_task1` is invalid.\n - Variable `p3__13_3__PlasmaArterial____KidneyTubular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_3__PlasmaArterial____KidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_4__KidneyPlasma____KidneyTissue_0_31_task1` is invalid.\n - Variable `p3__13_4__KidneyPlasma____KidneyTissue_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_4__KidneyPlasma____KidneyTissue_0']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_5__KidneyTissue____KidneyTubular_0_32_task1` is invalid.\n - Variable `p3__13_5__KidneyTissue____KidneyTubular_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_5__KidneyTissue____KidneyTubular_0']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_6__KidneyTubular____UrineExternal_33_task1` is invalid.\n - Variable `p3__13_6__KidneyTubular____UrineExternal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_6__KidneyTubular____UrineExternal']` are not defined. No namespaces are defined for the target.\n - Data generator `_14_1__PlasmaVenous____RBC_34_task1` is invalid.\n - Variable `p3__14_1__PlasmaVenous____RBC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_14_1__PlasmaVenous____RBC']` are not defined. No namespaces are defined for the target.\n - Data generator `_14_2__RBC____PlasmaVenous_35_task1` is invalid.\n - Variable `p3__14_2__RBC____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_14_2__RBC____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mgAdipose_1_task2` is invalid.\n - Variable `p1_mgAdipose_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mgArterialPlasma_2_task2` is invalid.\n - Variable `p1_mgArterialPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgArterialPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mgBrain_3_task2` is invalid.\n - Variable `p1_mgBrain_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mgEnterocytes_4_task2` is invalid.\n - Variable `p1_mgEnterocytes_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mgFeces_5_task2` is invalid.\n - Variable `p1_mgFeces_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mgHeart_6_task2` is invalid.\n - Variable `p1_mgHeart_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineLumen_7_task2` is invalid.\n - Variable `p1_mgIntestineLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineTotal_8_task2` is invalid.\n - Variable `p1_mgIntestineTotal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineVascular_9_task2` is invalid.\n - Variable `p1_mgIntestineVascular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTubular_10_task2` is invalid.\n - Variable `p1_mgKidneyTubular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyPlasma_11_task2` is invalid.\n - Variable `p1_mgKidneyPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTissues_12_task2` is invalid.\n - Variable `p1_mgKidneyTissues_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTissues']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTotal_13_task2` is invalid.\n - Variable `p1_mgKidneyTotal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLiver_14_task2` is invalid.\n - Variable `p1_mgLiver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLung_15_task2` is invalid.\n - Variable `p1_mgLung_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mgMuscle_16_task2` is invalid.\n - Variable `p1_mgMuscle_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPortalVein_17_task2` is invalid.\n - Variable `p1_mgPortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mgRBC_18_task2` is invalid.\n - Variable `p1_mgRBC_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgRBC']` are not defined. No namespaces are defined for the target.\n - Data generator `mgRemainder_19_task2` is invalid.\n - Variable `p1_mgRemainder_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomach_20_task2` is invalid.\n - Variable `p1_mgStomach_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomachLumen_21_task2` is invalid.\n - Variable `p1_mgStomachLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgUrineSum_22_task2` is invalid.\n - Variable `p1_mgUrineSum_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgUrineSum']` are not defined. No namespaces are defined for the target.\n - Data generator `mgVenousPlasma_23_task2` is invalid.\n - Variable `p1_mgVenousPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgVenousPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mAdipose_1_task2` is invalid.\n - Variable `p2_mAdipose_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mBrain_2_task2` is invalid.\n - Variable `p2_mBrain_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mEnterocytes_3_task2` is invalid.\n - Variable `p2_mEnterocytes_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mFeces_4_task2` is invalid.\n - Variable `p2_mFeces_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mHeart_5_task2` is invalid.\n - Variable `p2_mHeart_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineLumen_6_task2` is invalid.\n - Variable `p2_mIntestineLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineTotal_7_task2` is invalid.\n - Variable `p2_mIntestineTotal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mIntestineTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineVascular_8_task2` is invalid.\n - Variable `p2_mIntestineVascular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyPlasma_9_task2` is invalid.\n - Variable `p2_mKidneyPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTotal_10_task2` is invalid.\n - Variable `p2_mKidneyTotal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mKidneyTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTissue_11_task2` is invalid.\n - Variable `p2_mKidneyTissue_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTissue']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTubular_12_task2` is invalid.\n - Variable `p2_mKidneyTubular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mLiver_13_task2` is invalid.\n - Variable `p2_mLiver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mLung_14_task2` is invalid.\n - Variable `p2_mLung_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mMuscle_15_task2` is invalid.\n - Variable `p2_mMuscle_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaArterial_16_task2` is invalid.\n - Variable `p2_mPlasmaArterial_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaVenous_17_task2` is invalid.\n - Variable `p2_mPlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mPortalVein_18_task2` is invalid.\n - Variable `p2_mPortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mRBC_19_task2` is invalid.\n - Variable `p2_mRBC_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mRBC']` are not defined. No namespaces are defined for the target.\n - Data generator `mRemainder_20_task2` is invalid.\n - Variable `p2_mRemainder_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomach_21_task2` is invalid.\n - Variable `p2_mStomach_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomachLumen_22_task2` is invalid.\n - Variable `p2_mStomachLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mUrineExternal_23_task2` is invalid.\n - Variable `p2_mUrineExternal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mUrineExternal']` are not defined. No namespaces are defined for the target.\n - Data generator `_01__StomachLumen_____IntestineLumen_1_task2` is invalid.\n - Variable `p3__01__StomachLumen_____IntestineLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_01__StomachLumen_____IntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `_02__IntestineLumen____Feces_2_task2` is invalid.\n - Variable `p3__02__IntestineLumen____Feces_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_02__IntestineLumen____Feces']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_1__PlasmaArterial____IntestineVascular_3_task2` is invalid.\n - Variable `p3__03_1__PlasmaArterial____IntestineVascular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_1__PlasmaArterial____IntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_2__IntestineLumen____Enterocytes__PMAT_OCT3_4_task2` is invalid.\n - Variable `p3__03_2__IntestineLumen____Enterocytes__PMAT_OCT3_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_2__IntestineLumen____Enterocytes__PMAT_OCT3']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_3__Enterocytes____IntestineVascular__OCT1_5_task2` is invalid.\n - Variable `p3__03_3__Enterocytes____IntestineVascular__OCT1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_3__Enterocytes____IntestineVascular__OCT1']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_4__IntestineLumen____IntestineVascular__Saturable_0_6_task2` is invalid.\n - Variable `p3__03_4__IntestineLumen____IntestineVascular__Saturable_0_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_4__IntestineLumen____IntestineVascular__Saturable_0']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_5__IntestineVascular____PortalVein_7_task2` is invalid.\n - Variable `p3__03_5__IntestineVascular____PortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_5__IntestineVascular____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_6__IntestineLumen____Enterocytes__Diffusion_8_task2` is invalid.\n - Variable `p3__03_6__IntestineLumen____Enterocytes__Diffusion_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_6__IntestineLumen____Enterocytes__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_7__IntestineLumen____IntestineVascular__Diffusion_9_task2` is invalid.\n - Variable `p3__03_7__IntestineLumen____IntestineVascular__Diffusion_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_7__IntestineLumen____IntestineVascular__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_04__PlasmaArterial____PortalVein_10_task2` is invalid.\n - Variable `p3__04__PlasmaArterial____PortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_04__PlasmaArterial____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_1__PlasmaArterial____Stomach_11_task2` is invalid.\n - Variable `p3__05_1__PlasmaArterial____Stomach_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_1__PlasmaArterial____Stomach']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_2__Stomach____PortalVein_12_task2` is invalid.\n - Variable `p3__05_2__Stomach____PortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_2__Stomach____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_1__PlasmaArterial____Liver_13_task2` is invalid.\n - Variable `p3__06_1__PlasmaArterial____Liver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_1__PlasmaArterial____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_2__PortalVein____Liver_14_task2` is invalid.\n - Variable `p3__06_2__PortalVein____Liver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_2__PortalVein____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_3__Liver____PlasmaVenous_15_task2` is invalid.\n - Variable `p3__06_3__Liver____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_3__Liver____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_1__PlasmaVenous____Lung_16_task2` is invalid.\n - Variable `p3__07_1__PlasmaVenous____Lung_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_1__PlasmaVenous____Lung']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_2__Lung____PlasmaArterial_17_task2` is invalid.\n - Variable `p3__07_2__Lung____PlasmaArterial_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_2__Lung____PlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_1__PlasmaArterial____Brain_18_task2` is invalid.\n - Variable `p3__08_1__PlasmaArterial____Brain_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_1__PlasmaArterial____Brain']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_2__Brain____PlasmaVenous_19_task2` is invalid.\n - Variable `p3__08_2__Brain____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_2__Brain____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_1__PlasmaArterial____Muscle_20_task2` is invalid.\n - Variable `p3__09_1__PlasmaArterial____Muscle_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_1__PlasmaArterial____Muscle']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_2__Muscle____PlasmaVenous_21_task2` is invalid.\n - Variable `p3__09_2__Muscle____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_2__Muscle____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_1__PlasmaArterial____Adipose_22_task2` is invalid.\n - Variable `p3__10_1__PlasmaArterial____Adipose_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_1__PlasmaArterial____Adipose']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_2__Adipose____PlasmaVenous_23_task2` is invalid.\n - Variable `p3__10_2__Adipose____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_2__Adipose____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_1__PlasmaArterial____Heart_24_task2` is invalid.\n - Variable `p3__11_1__PlasmaArterial____Heart_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_1__PlasmaArterial____Heart']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_2__Heart____PlasmaVenous_25_task2` is invalid.\n - Variable `p3__11_2__Heart____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_2__Heart____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_1__PlasmaArterial____Remainder_26_task2` is invalid.\n - Variable `p3__12_1__PlasmaArterial____Remainder_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_1__PlasmaArterial____Remainder']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_2__Remainder____PlasmaVenous_27_task2` is invalid.\n - Variable `p3__12_2__Remainder____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_2__Remainder____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_1__PlasmaArterial____KidneyPlasma_28_task2` is invalid.\n - Variable `p3__13_1__PlasmaArterial____KidneyPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_1__PlasmaArterial____KidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_2__KidneyPlasma____PlasmaVenous_29_task2` is invalid.\n - Variable `p3__13_2__KidneyPlasma____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_2__KidneyPlasma____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_3__PlasmaArterial____KidneyTubular_30_task2` is invalid.\n - Variable `p3__13_3__PlasmaArterial____KidneyTubular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_3__PlasmaArterial____KidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_4__KidneyPlasma____KidneyTissue_0_31_task2` is invalid.\n - Variable `p3__13_4__KidneyPlasma____KidneyTissue_0_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_4__KidneyPlasma____KidneyTissue_0']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_5__KidneyTissue____KidneyTubular_0_32_task2` is invalid.\n - Variable `p3__13_5__KidneyTissue____KidneyTubular_0_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_5__KidneyTissue____KidneyTubular_0']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_6__KidneyTubular____UrineExternal_33_task2` is invalid.\n - Variable `p3__13_6__KidneyTubular____UrineExternal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_6__KidneyTubular____UrineExternal']` are not defined. No namespaces are defined for the target.\n - Data generator `_14_1__PlasmaVenous____RBC_34_task2` is invalid.\n - Variable `p3__14_1__PlasmaVenous____RBC_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_14_1__PlasmaVenous____RBC']` are not defined. No namespaces are defined for the target.\n - Data generator `_14_2__RBC____PlasmaVenous_35_task2` is invalid.\n - Variable `p3__14_2__RBC____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_14_2__RBC____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000001029](https://www.ebi.ac.uk/biomodels/BIOMD0000001029)<br/><sup>Zake2021 - PBPK model of metfo</sup></summary><sup>Zake2021 - PBPK model of metformin in humans, eight PO administrations with 12h interval</sup></details>|<details><summary>pass</summary>[Zake2021_Metformin+Human+multiple+PO+dose.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001029#Files)</details>|pass|<details><summary>pass</summary>[Zake2021_Metformin+Human+multiple+PO+dose.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001029#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Zake2021_Metformin+Human+multiple+PO+dose.sedml847075.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Zake2021_Metformin+Human+multiple+PO+dose.xml` has warnings.\r\n - 81 warnings of type SBML unit consistency (99508). The following is the first warning at line 1805, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'Ktp_Liver' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 81 warnings of type SBML unit consistency (20702). The following is the first warning at line 1805, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'Ktp_Liver' does not have a 'units' attribute.\r\n - 98 warnings of type SBML unit consistency (99505). The following is the first warning at line 2967, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Intestine_Coefficient' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 91 warnings of type Modeling practice (80701). The following is the first warning at line 1805, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Ktp_Liver' does not have a 'units' attribute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_5_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_6_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Zake2021_Metformin+Human+multiple+PO+dose.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Zake2021_Metformin+Human+multiple+PO+dose.xml` has warnings.\r\n - 81 warnings of type SBML unit consistency (99508). The following is the first warning at line 1805, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'Ktp_Liver' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 81 warnings of type SBML unit consistency (20702). The following is the first warning at line 1805, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'Ktp_Liver' does not have a 'units' attribute.\r\n - 98 warnings of type SBML unit consistency (99505). The following is the first warning at line 2967, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Intestine_Coefficient' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 91 warnings of type Modeling practice (80701). The following is the first warning at line 1805, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Ktp_Liver' does not have a 'units' attribute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_5_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_6_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/Zake2021_Metformin+Human+multiple+PO+dose.sedml847075.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Zake2021_Metformin+Human+multiple+PO+dose.sedml` is invalid.\n - Simulation `sim1` is invalid.\n - Output start time -12.0 must be at least the initial time 0.0.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Metformin_Dose_in_Lumen_in_mg']` are not defined. No namespaces are defined for the target.\n - Data generator `mgAdipose_1_task1` is invalid.\n - Variable `p1_mgAdipose_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mgArterialPlasma_2_task1` is invalid.\n - Variable `p1_mgArterialPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgArterialPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mgBrain_3_task1` is invalid.\n - Variable `p1_mgBrain_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mgEnterocytes_4_task1` is invalid.\n - Variable `p1_mgEnterocytes_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mgFeces_5_task1` is invalid.\n - Variable `p1_mgFeces_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mgHeart_6_task1` is invalid.\n - Variable `p1_mgHeart_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineLumen_7_task1` is invalid.\n - Variable `p1_mgIntestineLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineTotal_8_task1` is invalid.\n - Variable `p1_mgIntestineTotal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineVascular_9_task1` is invalid.\n - Variable `p1_mgIntestineVascular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTubular_10_task1` is invalid.\n - Variable `p1_mgKidneyTubular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyPlasma_11_task1` is invalid.\n - Variable `p1_mgKidneyPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTissues_12_task1` is invalid.\n - Variable `p1_mgKidneyTissues_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTissues']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTotal_13_task1` is invalid.\n - Variable `p1_mgKidneyTotal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLiver_14_task1` is invalid.\n - Variable `p1_mgLiver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLung_15_task1` is invalid.\n - Variable `p1_mgLung_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mgMuscle_16_task1` is invalid.\n - Variable `p1_mgMuscle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPortalVein_17_task1` is invalid.\n - Variable `p1_mgPortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mgRBC_18_task1` is invalid.\n - Variable `p1_mgRBC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgRBC']` are not defined. No namespaces are defined for the target.\n - Data generator `mgRemainder_19_task1` is invalid.\n - Variable `p1_mgRemainder_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomach_20_task1` is invalid.\n - Variable `p1_mgStomach_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomachLumen_21_task1` is invalid.\n - Variable `p1_mgStomachLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgUrineSum_22_task1` is invalid.\n - Variable `p1_mgUrineSum_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgUrineSum']` are not defined. No namespaces are defined for the target.\n - Data generator `mgVenousPlasma_23_task1` is invalid.\n - Variable `p1_mgVenousPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgVenousPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mAdipose_1_task1` is invalid.\n - Variable `p2_mAdipose_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mBrain_2_task1` is invalid.\n - Variable `p2_mBrain_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mEnterocytes_3_task1` is invalid.\n - Variable `p2_mEnterocytes_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mFeces_4_task1` is invalid.\n - Variable `p2_mFeces_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mHeart_5_task1` is invalid.\n - Variable `p2_mHeart_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineLumen_6_task1` is invalid.\n - Variable `p2_mIntestineLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineTotal_7_task1` is invalid.\n - Variable `p2_mIntestineTotal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mIntestineTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineVascular_8_task1` is invalid.\n - Variable `p2_mIntestineVascular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyPlasma_9_task1` is invalid.\n - Variable `p2_mKidneyPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTotal_10_task1` is invalid.\n - Variable `p2_mKidneyTotal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mKidneyTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTissue_11_task1` is invalid.\n - Variable `p2_mKidneyTissue_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTissue']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTubular_12_task1` is invalid.\n - Variable `p2_mKidneyTubular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mLiver_13_task1` is invalid.\n - Variable `p2_mLiver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mLung_14_task1` is invalid.\n - Variable `p2_mLung_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mMuscle_15_task1` is invalid.\n - Variable `p2_mMuscle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaArterial_16_task1` is invalid.\n - Variable `p2_mPlasmaArterial_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaVenous_17_task1` is invalid.\n - Variable `p2_mPlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mPortalVein_18_task1` is invalid.\n - Variable `p2_mPortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mRBC_19_task1` is invalid.\n - Variable `p2_mRBC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mRBC']` are not defined. No namespaces are defined for the target.\n - Data generator `mRemainder_20_task1` is invalid.\n - Variable `p2_mRemainder_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomach_21_task1` is invalid.\n - Variable `p2_mStomach_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomachLumen_22_task1` is invalid.\n - Variable `p2_mStomachLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mUrineExternal_23_task1` is invalid.\n - Variable `p2_mUrineExternal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mUrineExternal']` are not defined. No namespaces are defined for the target.\n - Data generator `_01__StomachLumen_____IntestineLumen_1_task1` is invalid.\n - Variable `p3__01__StomachLumen_____IntestineLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_01__StomachLumen_____IntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `_02__IntestineLumen____Feces_2_task1` is invalid.\n - Variable `p3__02__IntestineLumen____Feces_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_02__IntestineLumen____Feces']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_1__PlasmaArterial____IntestineVascular_3_task1` is invalid.\n - Variable `p3__03_1__PlasmaArterial____IntestineVascular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_1__PlasmaArterial____IntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_2__IntestineLumen____Enterocytes__PMAT_OCT3_4_task1` is invalid.\n - Variable `p3__03_2__IntestineLumen____Enterocytes__PMAT_OCT3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_2__IntestineLumen____Enterocytes__PMAT_OCT3']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_3__Enterocytes____IntestineVascular__OCT1_5_task1` is invalid.\n - Variable `p3__03_3__Enterocytes____IntestineVascular__OCT1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_3__Enterocytes____IntestineVascular__OCT1']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_4__IntestineLumen____IntestineVascular__Saturable_0_6_task1` is invalid.\n - Variable `p3__03_4__IntestineLumen____IntestineVascular__Saturable_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_4__IntestineLumen____IntestineVascular__Saturable_0']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_5__IntestineVascular____PortalVein_7_task1` is invalid.\n - Variable `p3__03_5__IntestineVascular____PortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_5__IntestineVascular____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_6__IntestineLumen____Enterocytes__Diffusion_8_task1` is invalid.\n - Variable `p3__03_6__IntestineLumen____Enterocytes__Diffusion_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_6__IntestineLumen____Enterocytes__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_7__IntestineLumen____IntestineVascular__Diffusion_9_task1` is invalid.\n - Variable `p3__03_7__IntestineLumen____IntestineVascular__Diffusion_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_7__IntestineLumen____IntestineVascular__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_04__PlasmaArterial____PortalVein_10_task1` is invalid.\n - Variable `p3__04__PlasmaArterial____PortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_04__PlasmaArterial____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_1__PlasmaArterial____Stomach_11_task1` is invalid.\n - Variable `p3__05_1__PlasmaArterial____Stomach_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_1__PlasmaArterial____Stomach']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_2__Stomach____PortalVein_12_task1` is invalid.\n - Variable `p3__05_2__Stomach____PortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_2__Stomach____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_1__PlasmaArterial____Liver_13_task1` is invalid.\n - Variable `p3__06_1__PlasmaArterial____Liver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_1__PlasmaArterial____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_2__PortalVein____Liver_14_task1` is invalid.\n - Variable `p3__06_2__PortalVein____Liver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_2__PortalVein____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_3__Liver____PlasmaVenous_15_task1` is invalid.\n - Variable `p3__06_3__Liver____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_3__Liver____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_1__PlasmaVenous____Lung_16_task1` is invalid.\n - Variable `p3__07_1__PlasmaVenous____Lung_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_1__PlasmaVenous____Lung']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_2__Lung____PlasmaArterial_17_task1` is invalid.\n - Variable `p3__07_2__Lung____PlasmaArterial_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_2__Lung____PlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_1__PlasmaArterial____Brain_18_task1` is invalid.\n - Variable `p3__08_1__PlasmaArterial____Brain_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_1__PlasmaArterial____Brain']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_2__Brain____PlasmaVenous_19_task1` is invalid.\n - Variable `p3__08_2__Brain____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_2__Brain____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_1__PlasmaArterial____Muscle_20_task1` is invalid.\n - Variable `p3__09_1__PlasmaArterial____Muscle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_1__PlasmaArterial____Muscle']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_2__Muscle____PlasmaVenous_21_task1` is invalid.\n - Variable `p3__09_2__Muscle____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_2__Muscle____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_1__PlasmaArterial____Adipose_22_task1` is invalid.\n - Variable `p3__10_1__PlasmaArterial____Adipose_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_1__PlasmaArterial____Adipose']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_2__Adipose____PlasmaVenous_23_task1` is invalid.\n - Variable `p3__10_2__Adipose____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_2__Adipose____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_1__PlasmaArterial____Heart_24_task1` is invalid.\n - Variable `p3__11_1__PlasmaArterial____Heart_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_1__PlasmaArterial____Heart']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_2__Heart____PlasmaVenous_25_task1` is invalid.\n - Variable `p3__11_2__Heart____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_2__Heart____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_1__PlasmaArterial____Remainder_26_task1` is invalid.\n - Variable `p3__12_1__PlasmaArterial____Remainder_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_1__PlasmaArterial____Remainder']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_2__Remainder____PlasmaVenous_27_task1` is invalid.\n - Variable `p3__12_2__Remainder____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_2__Remainder____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_1__PlasmaArterial____KidneyPlasma_28_task1` is invalid.\n - Variable `p3__13_1__PlasmaArterial____KidneyPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_1__PlasmaArterial____KidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_2__KidneyPlasma____PlasmaVenous_29_task1` is invalid.\n - Variable `p3__13_2__KidneyPlasma____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_2__KidneyPlasma____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_3__PlasmaArterial____KidneyTubular_30_task1` is invalid.\n - Variable `p3__13_3__PlasmaArterial____KidneyTubular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_3__PlasmaArterial____KidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_4__KidneyPlasma____KidneyTissue_0_31_task1` is invalid.\n - Variable `p3__13_4__KidneyPlasma____KidneyTissue_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_4__KidneyPlasma____KidneyTissue_0']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_5__KidneyTissue____KidneyTubular_0_32_task1` is invalid.\n - Variable `p3__13_5__KidneyTissue____KidneyTubular_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_5__KidneyTissue____KidneyTubular_0']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_6__KidneyTubular____UrineExternal_33_task1` is invalid.\n - Variable `p3__13_6__KidneyTubular____UrineExternal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_6__KidneyTubular____UrineExternal']` are not defined. No namespaces are defined for the target.\n - Data generator `_14_1__PlasmaVenous____RBC_34_task1` is invalid.\n - Variable `p3__14_1__PlasmaVenous____RBC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_14_1__PlasmaVenous____RBC']` are not defined. No namespaces are defined for the target.\n - Data generator `_14_2__RBC____PlasmaVenous_35_task1` is invalid.\n - Variable `p3__14_2__RBC____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_14_2__RBC____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mgAdipose_1_task2` is invalid.\n - Variable `p1_mgAdipose_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mgArterialPlasma_2_task2` is invalid.\n - Variable `p1_mgArterialPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgArterialPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mgBrain_3_task2` is invalid.\n - Variable `p1_mgBrain_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mgEnterocytes_4_task2` is invalid.\n - Variable `p1_mgEnterocytes_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mgFeces_5_task2` is invalid.\n - Variable `p1_mgFeces_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mgHeart_6_task2` is invalid.\n - Variable `p1_mgHeart_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineLumen_7_task2` is invalid.\n - Variable `p1_mgIntestineLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineTotal_8_task2` is invalid.\n - Variable `p1_mgIntestineTotal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineVascular_9_task2` is invalid.\n - Variable `p1_mgIntestineVascular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTubular_10_task2` is invalid.\n - Variable `p1_mgKidneyTubular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyPlasma_11_task2` is invalid.\n - Variable `p1_mgKidneyPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTissues_12_task2` is invalid.\n - Variable `p1_mgKidneyTissues_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTissues']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTotal_13_task2` is invalid.\n - Variable `p1_mgKidneyTotal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLiver_14_task2` is invalid.\n - Variable `p1_mgLiver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLung_15_task2` is invalid.\n - Variable `p1_mgLung_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mgMuscle_16_task2` is invalid.\n - Variable `p1_mgMuscle_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPortalVein_17_task2` is invalid.\n - Variable `p1_mgPortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mgRBC_18_task2` is invalid.\n - Variable `p1_mgRBC_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgRBC']` are not defined. No namespaces are defined for the target.\n - Data generator `mgRemainder_19_task2` is invalid.\n - Variable `p1_mgRemainder_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomach_20_task2` is invalid.\n - Variable `p1_mgStomach_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomachLumen_21_task2` is invalid.\n - Variable `p1_mgStomachLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgUrineSum_22_task2` is invalid.\n - Variable `p1_mgUrineSum_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgUrineSum']` are not defined. No namespaces are defined for the target.\n - Data generator `mgVenousPlasma_23_task2` is invalid.\n - Variable `p1_mgVenousPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgVenousPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mAdipose_1_task2` is invalid.\n - Variable `p2_mAdipose_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mBrain_2_task2` is invalid.\n - Variable `p2_mBrain_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mEnterocytes_3_task2` is invalid.\n - Variable `p2_mEnterocytes_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mFeces_4_task2` is invalid.\n - Variable `p2_mFeces_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mHeart_5_task2` is invalid.\n - Variable `p2_mHeart_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineLumen_6_task2` is invalid.\n - Variable `p2_mIntestineLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineTotal_7_task2` is invalid.\n - Variable `p2_mIntestineTotal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mIntestineTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineVascular_8_task2` is invalid.\n - Variable `p2_mIntestineVascular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyPlasma_9_task2` is invalid.\n - Variable `p2_mKidneyPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTotal_10_task2` is invalid.\n - Variable `p2_mKidneyTotal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mKidneyTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTissue_11_task2` is invalid.\n - Variable `p2_mKidneyTissue_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTissue']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTubular_12_task2` is invalid.\n - Variable `p2_mKidneyTubular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mLiver_13_task2` is invalid.\n - Variable `p2_mLiver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mLung_14_task2` is invalid.\n - Variable `p2_mLung_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mMuscle_15_task2` is invalid.\n - Variable `p2_mMuscle_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaArterial_16_task2` is invalid.\n - Variable `p2_mPlasmaArterial_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaVenous_17_task2` is invalid.\n - Variable `p2_mPlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mPortalVein_18_task2` is invalid.\n - Variable `p2_mPortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mRBC_19_task2` is invalid.\n - Variable `p2_mRBC_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mRBC']` are not defined. No namespaces are defined for the target.\n - Data generator `mRemainder_20_task2` is invalid.\n - Variable `p2_mRemainder_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomach_21_task2` is invalid.\n - Variable `p2_mStomach_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomachLumen_22_task2` is invalid.\n - Variable `p2_mStomachLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mUrineExternal_23_task2` is invalid.\n - Variable `p2_mUrineExternal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mUrineExternal']` are not defined. No namespaces are defined for the target.\n - Data generator `_01__StomachLumen_____IntestineLumen_1_task2` is invalid.\n - Variable `p3__01__StomachLumen_____IntestineLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_01__StomachLumen_____IntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `_02__IntestineLumen____Feces_2_task2` is invalid.\n - Variable `p3__02__IntestineLumen____Feces_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_02__IntestineLumen____Feces']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_1__PlasmaArterial____IntestineVascular_3_task2` is invalid.\n - Variable `p3__03_1__PlasmaArterial____IntestineVascular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_1__PlasmaArterial____IntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_2__IntestineLumen____Enterocytes__PMAT_OCT3_4_task2` is invalid.\n - Variable `p3__03_2__IntestineLumen____Enterocytes__PMAT_OCT3_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_2__IntestineLumen____Enterocytes__PMAT_OCT3']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_3__Enterocytes____IntestineVascular__OCT1_5_task2` is invalid.\n - Variable `p3__03_3__Enterocytes____IntestineVascular__OCT1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_3__Enterocytes____IntestineVascular__OCT1']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_4__IntestineLumen____IntestineVascular__Saturable_0_6_task2` is invalid.\n - Variable `p3__03_4__IntestineLumen____IntestineVascular__Saturable_0_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_4__IntestineLumen____IntestineVascular__Saturable_0']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_5__IntestineVascular____PortalVein_7_task2` is invalid.\n - Variable `p3__03_5__IntestineVascular____PortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_5__IntestineVascular____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_6__IntestineLumen____Enterocytes__Diffusion_8_task2` is invalid.\n - Variable `p3__03_6__IntestineLumen____Enterocytes__Diffusion_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_6__IntestineLumen____Enterocytes__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_7__IntestineLumen____IntestineVascular__Diffusion_9_task2` is invalid.\n - Variable `p3__03_7__IntestineLumen____IntestineVascular__Diffusion_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_7__IntestineLumen____IntestineVascular__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_04__PlasmaArterial____PortalVein_10_task2` is invalid.\n - Variable `p3__04__PlasmaArterial____PortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_04__PlasmaArterial____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_1__PlasmaArterial____Stomach_11_task2` is invalid.\n - Variable `p3__05_1__PlasmaArterial____Stomach_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_1__PlasmaArterial____Stomach']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_2__Stomach____PortalVein_12_task2` is invalid.\n - Variable `p3__05_2__Stomach____PortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_2__Stomach____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_1__PlasmaArterial____Liver_13_task2` is invalid.\n - Variable `p3__06_1__PlasmaArterial____Liver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_1__PlasmaArterial____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_2__PortalVein____Liver_14_task2` is invalid.\n - Variable `p3__06_2__PortalVein____Liver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_2__PortalVein____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_3__Liver____PlasmaVenous_15_task2` is invalid.\n - Variable `p3__06_3__Liver____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_3__Liver____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_1__PlasmaVenous____Lung_16_task2` is invalid.\n - Variable `p3__07_1__PlasmaVenous____Lung_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_1__PlasmaVenous____Lung']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_2__Lung____PlasmaArterial_17_task2` is invalid.\n - Variable `p3__07_2__Lung____PlasmaArterial_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_2__Lung____PlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_1__PlasmaArterial____Brain_18_task2` is invalid.\n - Variable `p3__08_1__PlasmaArterial____Brain_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_1__PlasmaArterial____Brain']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_2__Brain____PlasmaVenous_19_task2` is invalid.\n - Variable `p3__08_2__Brain____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_2__Brain____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_1__PlasmaArterial____Muscle_20_task2` is invalid.\n - Variable `p3__09_1__PlasmaArterial____Muscle_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_1__PlasmaArterial____Muscle']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_2__Muscle____PlasmaVenous_21_task2` is invalid.\n - Variable `p3__09_2__Muscle____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_2__Muscle____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_1__PlasmaArterial____Adipose_22_task2` is invalid.\n - Variable `p3__10_1__PlasmaArterial____Adipose_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_1__PlasmaArterial____Adipose']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_2__Adipose____PlasmaVenous_23_task2` is invalid.\n - Variable `p3__10_2__Adipose____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_2__Adipose____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_1__PlasmaArterial____Heart_24_task2` is invalid.\n - Variable `p3__11_1__PlasmaArterial____Heart_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_1__PlasmaArterial____Heart']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_2__Heart____PlasmaVenous_25_task2` is invalid.\n - Variable `p3__11_2__Heart____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_2__Heart____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_1__PlasmaArterial____Remainder_26_task2` is invalid.\n - Variable `p3__12_1__PlasmaArterial____Remainder_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_1__PlasmaArterial____Remainder']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_2__Remainder____PlasmaVenous_27_task2` is invalid.\n - Variable `p3__12_2__Remainder____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_2__Remainder____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_1__PlasmaArterial____KidneyPlasma_28_task2` is invalid.\n - Variable `p3__13_1__PlasmaArterial____KidneyPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_1__PlasmaArterial____KidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_2__KidneyPlasma____PlasmaVenous_29_task2` is invalid.\n - Variable `p3__13_2__KidneyPlasma____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_2__KidneyPlasma____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_3__PlasmaArterial____KidneyTubular_30_task2` is invalid.\n - Variable `p3__13_3__PlasmaArterial____KidneyTubular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_3__PlasmaArterial____KidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_4__KidneyPlasma____KidneyTissue_0_31_task2` is invalid.\n - Variable `p3__13_4__KidneyPlasma____KidneyTissue_0_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_4__KidneyPlasma____KidneyTissue_0']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_5__KidneyTissue____KidneyTubular_0_32_task2` is invalid.\n - Variable `p3__13_5__KidneyTissue____KidneyTubular_0_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_5__KidneyTissue____KidneyTubular_0']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_6__KidneyTubular____UrineExternal_33_task2` is invalid.\n - Variable `p3__13_6__KidneyTubular____UrineExternal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_6__KidneyTubular____UrineExternal']` are not defined. No namespaces are defined for the target.\n - Data generator `_14_1__PlasmaVenous____RBC_34_task2` is invalid.\n - Variable `p3__14_1__PlasmaVenous____RBC_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_14_1__PlasmaVenous____RBC']` are not defined. No namespaces are defined for the target.\n - Data generator `_14_2__RBC____PlasmaVenous_35_task2` is invalid.\n - Variable `p3__14_2__RBC____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_14_2__RBC____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Zake2021_Metformin+Human+multiple+PO+dose.sedml576446.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Zake2021_Metformin+Human+multiple+PO+dose.xml` has warnings.\r\n - 81 warnings of type SBML unit consistency (99508). The following is the first warning at line 1805, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'Ktp_Liver' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 81 warnings of type SBML unit consistency (20702). The following is the first warning at line 1805, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'Ktp_Liver' does not have a 'units' attribute.\r\n - 98 warnings of type SBML unit consistency (99505). The following is the first warning at line 2967, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Intestine_Coefficient' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 91 warnings of type Modeling practice (80701). The following is the first warning at line 1805, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Ktp_Liver' does not have a 'units' attribute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_5_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_6_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Zake2021_Metformin+Human+multiple+PO+dose.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Zake2021_Metformin+Human+multiple+PO+dose.xml` has warnings.\r\n - 81 warnings of type SBML unit consistency (99508). The following is the first warning at line 1805, column 6:\r\n - In situations where a mathematical expression refers to a compartment, species or parameter, it is necessary to know the units of the object to establish unit consistency. In models where the units of an object have not been declared, libSBML does not yet have the functionality to accurately verify the consistency of the units in mathematical expressions referring to that object. \r\n The units of the <parameter> 'Ktp_Liver' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 81 warnings of type SBML unit consistency (20702). The following is the first warning at line 1805, column 6:\r\n - If the attribute 'units' on a given Parameter object has not been set, then the unit of measurement associated with that parameter's value is undefined.\r\n Reference: L3V1 Section 4.7.3\r\n The <parameter> with id 'Ktp_Liver' does not have a 'units' attribute.\r\n - 98 warnings of type SBML unit consistency (99505). The following is the first warning at line 2967, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Intestine_Coefficient' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 91 warnings of type Modeling practice (80701). The following is the first warning at line 1805, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Ktp_Liver' does not have a 'units' attribute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_5_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_6_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Zake2021_Metformin+Human+multiple+PO+dose.sedml576446.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Zake2021_Metformin+Human+multiple+PO+dose.sedml` is invalid.\n - Simulation `sim1` is invalid.\n - Output start time -12.0 must be at least the initial time 0.0.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Metformin_Dose_in_Lumen_in_mg']` are not defined. No namespaces are defined for the target.\n - Data generator `mgAdipose_1_task1` is invalid.\n - Variable `p1_mgAdipose_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mgArterialPlasma_2_task1` is invalid.\n - Variable `p1_mgArterialPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgArterialPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mgBrain_3_task1` is invalid.\n - Variable `p1_mgBrain_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mgEnterocytes_4_task1` is invalid.\n - Variable `p1_mgEnterocytes_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mgFeces_5_task1` is invalid.\n - Variable `p1_mgFeces_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mgHeart_6_task1` is invalid.\n - Variable `p1_mgHeart_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineLumen_7_task1` is invalid.\n - Variable `p1_mgIntestineLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineTotal_8_task1` is invalid.\n - Variable `p1_mgIntestineTotal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineVascular_9_task1` is invalid.\n - Variable `p1_mgIntestineVascular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTubular_10_task1` is invalid.\n - Variable `p1_mgKidneyTubular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyPlasma_11_task1` is invalid.\n - Variable `p1_mgKidneyPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTissues_12_task1` is invalid.\n - Variable `p1_mgKidneyTissues_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTissues']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTotal_13_task1` is invalid.\n - Variable `p1_mgKidneyTotal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLiver_14_task1` is invalid.\n - Variable `p1_mgLiver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLung_15_task1` is invalid.\n - Variable `p1_mgLung_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mgMuscle_16_task1` is invalid.\n - Variable `p1_mgMuscle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPortalVein_17_task1` is invalid.\n - Variable `p1_mgPortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mgRBC_18_task1` is invalid.\n - Variable `p1_mgRBC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgRBC']` are not defined. No namespaces are defined for the target.\n - Data generator `mgRemainder_19_task1` is invalid.\n - Variable `p1_mgRemainder_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomach_20_task1` is invalid.\n - Variable `p1_mgStomach_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomachLumen_21_task1` is invalid.\n - Variable `p1_mgStomachLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgUrineSum_22_task1` is invalid.\n - Variable `p1_mgUrineSum_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgUrineSum']` are not defined. No namespaces are defined for the target.\n - Data generator `mgVenousPlasma_23_task1` is invalid.\n - Variable `p1_mgVenousPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgVenousPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mAdipose_1_task1` is invalid.\n - Variable `p2_mAdipose_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mBrain_2_task1` is invalid.\n - Variable `p2_mBrain_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mEnterocytes_3_task1` is invalid.\n - Variable `p2_mEnterocytes_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mFeces_4_task1` is invalid.\n - Variable `p2_mFeces_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mHeart_5_task1` is invalid.\n - Variable `p2_mHeart_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineLumen_6_task1` is invalid.\n - Variable `p2_mIntestineLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineTotal_7_task1` is invalid.\n - Variable `p2_mIntestineTotal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mIntestineTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineVascular_8_task1` is invalid.\n - Variable `p2_mIntestineVascular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyPlasma_9_task1` is invalid.\n - Variable `p2_mKidneyPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTotal_10_task1` is invalid.\n - Variable `p2_mKidneyTotal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mKidneyTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTissue_11_task1` is invalid.\n - Variable `p2_mKidneyTissue_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTissue']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTubular_12_task1` is invalid.\n - Variable `p2_mKidneyTubular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mLiver_13_task1` is invalid.\n - Variable `p2_mLiver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mLung_14_task1` is invalid.\n - Variable `p2_mLung_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mMuscle_15_task1` is invalid.\n - Variable `p2_mMuscle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaArterial_16_task1` is invalid.\n - Variable `p2_mPlasmaArterial_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaVenous_17_task1` is invalid.\n - Variable `p2_mPlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mPortalVein_18_task1` is invalid.\n - Variable `p2_mPortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mRBC_19_task1` is invalid.\n - Variable `p2_mRBC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mRBC']` are not defined. No namespaces are defined for the target.\n - Data generator `mRemainder_20_task1` is invalid.\n - Variable `p2_mRemainder_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomach_21_task1` is invalid.\n - Variable `p2_mStomach_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomachLumen_22_task1` is invalid.\n - Variable `p2_mStomachLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mUrineExternal_23_task1` is invalid.\n - Variable `p2_mUrineExternal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mUrineExternal']` are not defined. No namespaces are defined for the target.\n - Data generator `_01__StomachLumen_____IntestineLumen_1_task1` is invalid.\n - Variable `p3__01__StomachLumen_____IntestineLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_01__StomachLumen_____IntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `_02__IntestineLumen____Feces_2_task1` is invalid.\n - Variable `p3__02__IntestineLumen____Feces_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_02__IntestineLumen____Feces']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_1__PlasmaArterial____IntestineVascular_3_task1` is invalid.\n - Variable `p3__03_1__PlasmaArterial____IntestineVascular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_1__PlasmaArterial____IntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_2__IntestineLumen____Enterocytes__PMAT_OCT3_4_task1` is invalid.\n - Variable `p3__03_2__IntestineLumen____Enterocytes__PMAT_OCT3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_2__IntestineLumen____Enterocytes__PMAT_OCT3']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_3__Enterocytes____IntestineVascular__OCT1_5_task1` is invalid.\n - Variable `p3__03_3__Enterocytes____IntestineVascular__OCT1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_3__Enterocytes____IntestineVascular__OCT1']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_4__IntestineLumen____IntestineVascular__Saturable_0_6_task1` is invalid.\n - Variable `p3__03_4__IntestineLumen____IntestineVascular__Saturable_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_4__IntestineLumen____IntestineVascular__Saturable_0']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_5__IntestineVascular____PortalVein_7_task1` is invalid.\n - Variable `p3__03_5__IntestineVascular____PortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_5__IntestineVascular____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_6__IntestineLumen____Enterocytes__Diffusion_8_task1` is invalid.\n - Variable `p3__03_6__IntestineLumen____Enterocytes__Diffusion_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_6__IntestineLumen____Enterocytes__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_7__IntestineLumen____IntestineVascular__Diffusion_9_task1` is invalid.\n - Variable `p3__03_7__IntestineLumen____IntestineVascular__Diffusion_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_7__IntestineLumen____IntestineVascular__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_04__PlasmaArterial____PortalVein_10_task1` is invalid.\n - Variable `p3__04__PlasmaArterial____PortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_04__PlasmaArterial____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_1__PlasmaArterial____Stomach_11_task1` is invalid.\n - Variable `p3__05_1__PlasmaArterial____Stomach_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_1__PlasmaArterial____Stomach']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_2__Stomach____PortalVein_12_task1` is invalid.\n - Variable `p3__05_2__Stomach____PortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_2__Stomach____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_1__PlasmaArterial____Liver_13_task1` is invalid.\n - Variable `p3__06_1__PlasmaArterial____Liver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_1__PlasmaArterial____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_2__PortalVein____Liver_14_task1` is invalid.\n - Variable `p3__06_2__PortalVein____Liver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_2__PortalVein____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_3__Liver____PlasmaVenous_15_task1` is invalid.\n - Variable `p3__06_3__Liver____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_3__Liver____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_1__PlasmaVenous____Lung_16_task1` is invalid.\n - Variable `p3__07_1__PlasmaVenous____Lung_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_1__PlasmaVenous____Lung']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_2__Lung____PlasmaArterial_17_task1` is invalid.\n - Variable `p3__07_2__Lung____PlasmaArterial_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_2__Lung____PlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_1__PlasmaArterial____Brain_18_task1` is invalid.\n - Variable `p3__08_1__PlasmaArterial____Brain_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_1__PlasmaArterial____Brain']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_2__Brain____PlasmaVenous_19_task1` is invalid.\n - Variable `p3__08_2__Brain____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_2__Brain____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_1__PlasmaArterial____Muscle_20_task1` is invalid.\n - Variable `p3__09_1__PlasmaArterial____Muscle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_1__PlasmaArterial____Muscle']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_2__Muscle____PlasmaVenous_21_task1` is invalid.\n - Variable `p3__09_2__Muscle____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_2__Muscle____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_1__PlasmaArterial____Adipose_22_task1` is invalid.\n - Variable `p3__10_1__PlasmaArterial____Adipose_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_1__PlasmaArterial____Adipose']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_2__Adipose____PlasmaVenous_23_task1` is invalid.\n - Variable `p3__10_2__Adipose____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_2__Adipose____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_1__PlasmaArterial____Heart_24_task1` is invalid.\n - Variable `p3__11_1__PlasmaArterial____Heart_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_1__PlasmaArterial____Heart']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_2__Heart____PlasmaVenous_25_task1` is invalid.\n - Variable `p3__11_2__Heart____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_2__Heart____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_1__PlasmaArterial____Remainder_26_task1` is invalid.\n - Variable `p3__12_1__PlasmaArterial____Remainder_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_1__PlasmaArterial____Remainder']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_2__Remainder____PlasmaVenous_27_task1` is invalid.\n - Variable `p3__12_2__Remainder____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_2__Remainder____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_1__PlasmaArterial____KidneyPlasma_28_task1` is invalid.\n - Variable `p3__13_1__PlasmaArterial____KidneyPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_1__PlasmaArterial____KidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_2__KidneyPlasma____PlasmaVenous_29_task1` is invalid.\n - Variable `p3__13_2__KidneyPlasma____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_2__KidneyPlasma____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_3__PlasmaArterial____KidneyTubular_30_task1` is invalid.\n - Variable `p3__13_3__PlasmaArterial____KidneyTubular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_3__PlasmaArterial____KidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_4__KidneyPlasma____KidneyTissue_0_31_task1` is invalid.\n - Variable `p3__13_4__KidneyPlasma____KidneyTissue_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_4__KidneyPlasma____KidneyTissue_0']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_5__KidneyTissue____KidneyTubular_0_32_task1` is invalid.\n - Variable `p3__13_5__KidneyTissue____KidneyTubular_0_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_5__KidneyTissue____KidneyTubular_0']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_6__KidneyTubular____UrineExternal_33_task1` is invalid.\n - Variable `p3__13_6__KidneyTubular____UrineExternal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_6__KidneyTubular____UrineExternal']` are not defined. No namespaces are defined for the target.\n - Data generator `_14_1__PlasmaVenous____RBC_34_task1` is invalid.\n - Variable `p3__14_1__PlasmaVenous____RBC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_14_1__PlasmaVenous____RBC']` are not defined. No namespaces are defined for the target.\n - Data generator `_14_2__RBC____PlasmaVenous_35_task1` is invalid.\n - Variable `p3__14_2__RBC____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_14_2__RBC____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mgAdipose_1_task2` is invalid.\n - Variable `p1_mgAdipose_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mgArterialPlasma_2_task2` is invalid.\n - Variable `p1_mgArterialPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgArterialPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mgBrain_3_task2` is invalid.\n - Variable `p1_mgBrain_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mgEnterocytes_4_task2` is invalid.\n - Variable `p1_mgEnterocytes_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mgFeces_5_task2` is invalid.\n - Variable `p1_mgFeces_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mgHeart_6_task2` is invalid.\n - Variable `p1_mgHeart_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineLumen_7_task2` is invalid.\n - Variable `p1_mgIntestineLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineTotal_8_task2` is invalid.\n - Variable `p1_mgIntestineTotal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineVascular_9_task2` is invalid.\n - Variable `p1_mgIntestineVascular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTubular_10_task2` is invalid.\n - Variable `p1_mgKidneyTubular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyPlasma_11_task2` is invalid.\n - Variable `p1_mgKidneyPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTissues_12_task2` is invalid.\n - Variable `p1_mgKidneyTissues_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTissues']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTotal_13_task2` is invalid.\n - Variable `p1_mgKidneyTotal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLiver_14_task2` is invalid.\n - Variable `p1_mgLiver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLung_15_task2` is invalid.\n - Variable `p1_mgLung_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mgMuscle_16_task2` is invalid.\n - Variable `p1_mgMuscle_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPortalVein_17_task2` is invalid.\n - Variable `p1_mgPortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mgRBC_18_task2` is invalid.\n - Variable `p1_mgRBC_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgRBC']` are not defined. No namespaces are defined for the target.\n - Data generator `mgRemainder_19_task2` is invalid.\n - Variable `p1_mgRemainder_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomach_20_task2` is invalid.\n - Variable `p1_mgStomach_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomachLumen_21_task2` is invalid.\n - Variable `p1_mgStomachLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgUrineSum_22_task2` is invalid.\n - Variable `p1_mgUrineSum_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgUrineSum']` are not defined. No namespaces are defined for the target.\n - Data generator `mgVenousPlasma_23_task2` is invalid.\n - Variable `p1_mgVenousPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgVenousPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mAdipose_1_task2` is invalid.\n - Variable `p2_mAdipose_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mBrain_2_task2` is invalid.\n - Variable `p2_mBrain_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mEnterocytes_3_task2` is invalid.\n - Variable `p2_mEnterocytes_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mFeces_4_task2` is invalid.\n - Variable `p2_mFeces_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mHeart_5_task2` is invalid.\n - Variable `p2_mHeart_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineLumen_6_task2` is invalid.\n - Variable `p2_mIntestineLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineTotal_7_task2` is invalid.\n - Variable `p2_mIntestineTotal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mIntestineTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineVascular_8_task2` is invalid.\n - Variable `p2_mIntestineVascular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyPlasma_9_task2` is invalid.\n - Variable `p2_mKidneyPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTotal_10_task2` is invalid.\n - Variable `p2_mKidneyTotal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mKidneyTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTissue_11_task2` is invalid.\n - Variable `p2_mKidneyTissue_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTissue']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTubular_12_task2` is invalid.\n - Variable `p2_mKidneyTubular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mLiver_13_task2` is invalid.\n - Variable `p2_mLiver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mLung_14_task2` is invalid.\n - Variable `p2_mLung_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mMuscle_15_task2` is invalid.\n - Variable `p2_mMuscle_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaArterial_16_task2` is invalid.\n - Variable `p2_mPlasmaArterial_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaVenous_17_task2` is invalid.\n - Variable `p2_mPlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mPortalVein_18_task2` is invalid.\n - Variable `p2_mPortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mRBC_19_task2` is invalid.\n - Variable `p2_mRBC_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mRBC']` are not defined. No namespaces are defined for the target.\n - Data generator `mRemainder_20_task2` is invalid.\n - Variable `p2_mRemainder_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomach_21_task2` is invalid.\n - Variable `p2_mStomach_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomachLumen_22_task2` is invalid.\n - Variable `p2_mStomachLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mUrineExternal_23_task2` is invalid.\n - Variable `p2_mUrineExternal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mUrineExternal']` are not defined. No namespaces are defined for the target.\n - Data generator `_01__StomachLumen_____IntestineLumen_1_task2` is invalid.\n - Variable `p3__01__StomachLumen_____IntestineLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_01__StomachLumen_____IntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `_02__IntestineLumen____Feces_2_task2` is invalid.\n - Variable `p3__02__IntestineLumen____Feces_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_02__IntestineLumen____Feces']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_1__PlasmaArterial____IntestineVascular_3_task2` is invalid.\n - Variable `p3__03_1__PlasmaArterial____IntestineVascular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_1__PlasmaArterial____IntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_2__IntestineLumen____Enterocytes__PMAT_OCT3_4_task2` is invalid.\n - Variable `p3__03_2__IntestineLumen____Enterocytes__PMAT_OCT3_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_2__IntestineLumen____Enterocytes__PMAT_OCT3']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_3__Enterocytes____IntestineVascular__OCT1_5_task2` is invalid.\n - Variable `p3__03_3__Enterocytes____IntestineVascular__OCT1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_3__Enterocytes____IntestineVascular__OCT1']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_4__IntestineLumen____IntestineVascular__Saturable_0_6_task2` is invalid.\n - Variable `p3__03_4__IntestineLumen____IntestineVascular__Saturable_0_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_4__IntestineLumen____IntestineVascular__Saturable_0']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_5__IntestineVascular____PortalVein_7_task2` is invalid.\n - Variable `p3__03_5__IntestineVascular____PortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_5__IntestineVascular____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_6__IntestineLumen____Enterocytes__Diffusion_8_task2` is invalid.\n - Variable `p3__03_6__IntestineLumen____Enterocytes__Diffusion_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_6__IntestineLumen____Enterocytes__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_7__IntestineLumen____IntestineVascular__Diffusion_9_task2` is invalid.\n - Variable `p3__03_7__IntestineLumen____IntestineVascular__Diffusion_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_7__IntestineLumen____IntestineVascular__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_04__PlasmaArterial____PortalVein_10_task2` is invalid.\n - Variable `p3__04__PlasmaArterial____PortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_04__PlasmaArterial____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_1__PlasmaArterial____Stomach_11_task2` is invalid.\n - Variable `p3__05_1__PlasmaArterial____Stomach_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_1__PlasmaArterial____Stomach']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_2__Stomach____PortalVein_12_task2` is invalid.\n - Variable `p3__05_2__Stomach____PortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_2__Stomach____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_1__PlasmaArterial____Liver_13_task2` is invalid.\n - Variable `p3__06_1__PlasmaArterial____Liver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_1__PlasmaArterial____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_2__PortalVein____Liver_14_task2` is invalid.\n - Variable `p3__06_2__PortalVein____Liver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_2__PortalVein____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_3__Liver____PlasmaVenous_15_task2` is invalid.\n - Variable `p3__06_3__Liver____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_3__Liver____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_1__PlasmaVenous____Lung_16_task2` is invalid.\n - Variable `p3__07_1__PlasmaVenous____Lung_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_1__PlasmaVenous____Lung']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_2__Lung____PlasmaArterial_17_task2` is invalid.\n - Variable `p3__07_2__Lung____PlasmaArterial_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_2__Lung____PlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_1__PlasmaArterial____Brain_18_task2` is invalid.\n - Variable `p3__08_1__PlasmaArterial____Brain_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_1__PlasmaArterial____Brain']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_2__Brain____PlasmaVenous_19_task2` is invalid.\n - Variable `p3__08_2__Brain____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_2__Brain____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_1__PlasmaArterial____Muscle_20_task2` is invalid.\n - Variable `p3__09_1__PlasmaArterial____Muscle_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_1__PlasmaArterial____Muscle']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_2__Muscle____PlasmaVenous_21_task2` is invalid.\n - Variable `p3__09_2__Muscle____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_2__Muscle____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_1__PlasmaArterial____Adipose_22_task2` is invalid.\n - Variable `p3__10_1__PlasmaArterial____Adipose_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_1__PlasmaArterial____Adipose']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_2__Adipose____PlasmaVenous_23_task2` is invalid.\n - Variable `p3__10_2__Adipose____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_2__Adipose____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_1__PlasmaArterial____Heart_24_task2` is invalid.\n - Variable `p3__11_1__PlasmaArterial____Heart_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_1__PlasmaArterial____Heart']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_2__Heart____PlasmaVenous_25_task2` is invalid.\n - Variable `p3__11_2__Heart____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_2__Heart____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_1__PlasmaArterial____Remainder_26_task2` is invalid.\n - Variable `p3__12_1__PlasmaArterial____Remainder_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_1__PlasmaArterial____Remainder']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_2__Remainder____PlasmaVenous_27_task2` is invalid.\n - Variable `p3__12_2__Remainder____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_2__Remainder____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_1__PlasmaArterial____KidneyPlasma_28_task2` is invalid.\n - Variable `p3__13_1__PlasmaArterial____KidneyPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_1__PlasmaArterial____KidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_2__KidneyPlasma____PlasmaVenous_29_task2` is invalid.\n - Variable `p3__13_2__KidneyPlasma____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_2__KidneyPlasma____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_3__PlasmaArterial____KidneyTubular_30_task2` is invalid.\n - Variable `p3__13_3__PlasmaArterial____KidneyTubular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_3__PlasmaArterial____KidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_4__KidneyPlasma____KidneyTissue_0_31_task2` is invalid.\n - Variable `p3__13_4__KidneyPlasma____KidneyTissue_0_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_4__KidneyPlasma____KidneyTissue_0']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_5__KidneyTissue____KidneyTubular_0_32_task2` is invalid.\n - Variable `p3__13_5__KidneyTissue____KidneyTubular_0_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_5__KidneyTissue____KidneyTubular_0']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_6__KidneyTubular____UrineExternal_33_task2` is invalid.\n - Variable `p3__13_6__KidneyTubular____UrineExternal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_6__KidneyTubular____UrineExternal']` are not defined. No namespaces are defined for the target.\n - Data generator `_14_1__PlasmaVenous____RBC_34_task2` is invalid.\n - Variable `p3__14_1__PlasmaVenous____RBC_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_14_1__PlasmaVenous____RBC']` are not defined. No namespaces are defined for the target.\n - Data generator `_14_2__RBC____PlasmaVenous_35_task2` is invalid.\n - Variable `p3__14_2__RBC____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_14_2__RBC____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000001030](https://www.ebi.ac.uk/biomodels/BIOMD0000001030)<br/><sup>Sontag2017 - Dynamic model of </sup></summary><sup>Sontag2017 - Dynamic model of immune responses to antigen presentation by tumor or pathogen</sup></details>|<details><summary>pass</summary>[Sontag2017.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001030#Files)</details>|FAIL|<details><summary>MultipleSEDMLs</summary>['Sontag2017 - Fig 1D shifted ramp simulation.sedml', 'Sontag2017 - Fig 1D shifted exponential simulation.sedml', 'Sontag2017 - Fig 1D new constant value simulation.sedml']</details>|NA|NA|NA|
|<details><summary>[BIOMD0000001031](https://www.ebi.ac.uk/biomodels/BIOMD0000001031)<br/><sup>Al-Tuwairqi2020 - Dynamics of </sup></summary><sup>Al-Tuwairqi2020 - Dynamics of cancer virotherapy - Phase I treatment</sup></details>|<details><summary>pass</summary>[Al-Tuwairqi2020 - Phase I virotherapy.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001031#Files)</details>|FAIL|<details><summary>MultipleSEDMLs</summary>['Al-Tuwairqi2020 - Fig2a simulation.sedml', 'Al-Tuwairqi2020 - Fig2b simulation.sedml']</details>|NA|NA|NA|
|<details><summary>[BIOMD0000001032](https://www.ebi.ac.uk/biomodels/BIOMD0000001032)<br/><sup>Al-Tuwairqi2020 - Dynamics of </sup></summary><sup>Al-Tuwairqi2020 - Dynamics of cancer radiovirotherapy - Phase II treatment</sup></details>|<details><summary>pass</summary>[Al-Tuwairqi2020 - Phase II radiovirotherapy treatment.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001032#Files)</details>|FAIL|<details><summary>FAIL</summary>[Al-Tuwairqi2020 - Fig8 simulation.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001032#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Al-Tuwairqi2020 - Fig8 simulation.sedml674641.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 2: b'usage: biosimulators-tellurium [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-tellurium: error: unrecognized arguments: - Fig8 simulation.sedml674641.omex\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Al-Tuwairqi2020 - Fig8 simulation.sedml596442.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 2: b'usage: biosimulators-copasi [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-copasi: error: unrecognized arguments: - Fig8 simulation.sedml596442.omex\n'```</details>|
|<details><summary>[BIOMD0000001033](https://www.ebi.ac.uk/biomodels/BIOMD0000001033)<br/><sup>Almuallem2020 - Virus-macropha</sup></summary><sup>Almuallem2020 - Virus-macrophage-tumour interactions in oncolytic viral therapies</sup></details>|<details><summary>pass</summary>[Almuallem2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001033#Files)</details>|pass|<details><summary>pass</summary>[Almuallem2020 - Fig6.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001033#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Almuallem2020 - Fig6.sedml824223.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 2: b'usage: biosimulators-tellurium [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-tellurium: error: unrecognized arguments: - Fig6.sedml824223.omex\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Almuallem2020 - Fig6.sedml707907.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 2: b'usage: biosimulators-copasi [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-copasi: error: unrecognized arguments: - Fig6.sedml707907.omex\n'```</details>|
|<details><summary>[BIOMD0000001034](https://www.ebi.ac.uk/biomodels/BIOMD0000001034)<br/><sup>Bunimovich-Mendrazitsky2007 - </sup></summary><sup>Bunimovich-Mendrazitsky2007 - Mathematical model of BCG immunotherapy</sup></details>|<details><summary>pass</summary>[Bunimovich-Mendrazitsky2007.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001034#Files)</details>|pass|<details><summary>pass</summary>[Bunimovich-Mendrazitsky2007 - Fig4.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001034#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Bunimovich-Mendrazitsky2007 - Fig4.sedml917423.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 2: b'usage: biosimulators-tellurium [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-tellurium: error: unrecognized arguments: - Fig4.sedml917423.omex\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Bunimovich-Mendrazitsky2007 - Fig4.sedml96296.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 2: b'usage: biosimulators-copasi [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-copasi: error: unrecognized arguments: - Fig4.sedml96296.omex\n'```</details>|
|<details><summary>[BIOMD0000001035](https://www.ebi.ac.uk/biomodels/BIOMD0000001035)<br/><sup>Al-Tuwairqi2020 - Dynamics of </sup></summary><sup>Al-Tuwairqi2020 - Dynamics of cancer virotherapy with immune response</sup></details>|<details><summary>pass</summary>[Al-Tuwairqi2020 - Virotherapy.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001035#Files)</details>|pass|<details><summary>MultipleSEDMLs</summary>['Al-Tuwairqi2020 - Fig2(a) simulation.sedml', 'Al-Tuwairqi2020 - Fig7(a) simulation.sedml', 'Al-Tuwairqi2020 - Fig7(b) simulation.sedml', 'Al-Tuwairqi2020 - Fig2(c) simulation.sedml', 'Al-Tuwairqi2020 - Fig2(b) simulation.sedml']</details>|NA|NA|NA|
|<details><summary>[BIOMD0000001036](https://www.ebi.ac.uk/biomodels/BIOMD0000001036)<br/><sup>Cappuccio2007 - Tumor-immune s</sup></summary><sup>Cappuccio2007 - Tumor-immune system interactions and determination of the optimal therapeutic protocol in immunotherapy</sup></details>|<details><summary>pass</summary>[Cappuccio2007.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001036#Files)</details>|pass|<details><summary>FAIL</summary>[Cappuccio2007 - Fig1.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001036#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Cappuccio2007 - Fig1.sedml513247.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 2: b'usage: biosimulators-tellurium [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-tellurium: error: unrecognized arguments: - Fig1.sedml513247.omex\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Cappuccio2007 - Fig1.sedml234970.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 2: b'usage: biosimulators-copasi [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-copasi: error: unrecognized arguments: - Fig1.sedml234970.omex\n'```</details>|
|<details><summary>[BIOMD0000001037](https://www.ebi.ac.uk/biomodels/BIOMD0000001037)<br/><sup>Alharbi2019 - Tumor-normal mod</sup></summary><sup>Alharbi2019 - Tumor-normal model (TNM) of the development of tumor cells and their impact on normal cell dynamics</sup></details>|<details><summary>pass</summary>[Alharbi2019 TNM.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001037#Files)</details>|pass|<details><summary>FAIL</summary>[Alharbi2019 TNM - Fig10.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001037#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Alharbi2019 TNM - Fig10.sedml676409.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 2: b'usage: biosimulators-tellurium [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-tellurium: error: unrecognized arguments: TNM - Fig10.sedml676409.omex\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Alharbi2019 TNM - Fig10.sedml935670.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 2: b'usage: biosimulators-copasi [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-copasi: error: unrecognized arguments: TNM - Fig10.sedml935670.omex\n'```</details>|
|<details><summary>[BIOMD0000001038](https://www.ebi.ac.uk/biomodels/BIOMD0000001038)<br/><sup>Alharbi2019 - Tumor-normal-vit</sup></summary><sup>Alharbi2019 - Tumor-normal-vitamins model (TNVM) of the effects of vitamins on delaying the growth of tumor cells</sup></details>|<details><summary>pass</summary>[Alharbi2019 TNVM.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001038#Files)</details>|pass|<details><summary>FAIL</summary>[Alharbi2019 TNVM - Fig11.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001038#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Alharbi2019 TNVM - Fig11.sedml923468.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 2: b'usage: biosimulators-tellurium [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-tellurium: error: unrecognized arguments: TNVM - Fig11.sedml923468.omex\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Alharbi2019 TNVM - Fig11.sedml104712.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 2: b'usage: biosimulators-copasi [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-copasi: error: unrecognized arguments: TNVM - Fig11.sedml104712.omex\n'```</details>|
|<details><summary>[BIOMD0000001039](https://www.ebi.ac.uk/biomodels/BIOMD0000001039)<br/><sup>Zake2021 - PBPK model of metfo</sup></summary><sup>Zake2021 - PBPK model of metformin in mice: single dose intavenous</sup></details>|<details><summary>pass</summary>[Zake2021_Metformin+Mice+IV.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001039#Files)</details>|pass|<details><summary>FAIL</summary>[Zake2021_Metformin+Mice+IV.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001039#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Zake2021_Metformin+Mice+IV.sedml750198.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Output `plot_5_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_6_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_7_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_8_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Zake2021_Metformin+Mice+IV.sedml` has warnings.\r\n - Output `plot_5_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_6_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_7_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_8_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/Zake2021_Metformin+Mice+IV.sedml750198.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Zake2021_Metformin+Mice+IV.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmpfhu163kv/./model1.xml` is not a file.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Ktp_IntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mAdipose_1_task1` is invalid.\n - Variable `p1_mAdipose_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mBrain_2_task1` is invalid.\n - Variable `p1_mBrain_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mFeces_3_task1` is invalid.\n - Variable `p1_mFeces_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mHeart_4_task1` is invalid.\n - Variable `p1_mHeart_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineEnterocytes_5_task1` is invalid.\n - Variable `p1_mIntestineEnterocytes_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineLumen_6_task1` is invalid.\n - Variable `p1_mIntestineLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineSum_7_task1` is invalid.\n - Variable `p1_mIntestineSum_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mIntestineSum']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineVascular_8_task1` is invalid.\n - Variable `p1_mIntestineVascular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyPlasma_9_task1` is invalid.\n - Variable `p1_mKidneyPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneySum_10_task1` is invalid.\n - Variable `p1_mKidneySum_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mKidneySum']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTissue_11_task1` is invalid.\n - Variable `p1_mKidneyTissue_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTissue']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTubular_12_task1` is invalid.\n - Variable `p1_mKidneyTubular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mLiver_13_task1` is invalid.\n - Variable `p1_mLiver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mLung_14_task1` is invalid.\n - Variable `p1_mLung_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mMuscle_15_task1` is invalid.\n - Variable `p1_mMuscle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaArterial_16_task1` is invalid.\n - Variable `p1_mPlasmaArterial_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaVenous_17_task1` is invalid.\n - Variable `p1_mPlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mPortalVein_18_task1` is invalid.\n - Variable `p1_mPortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mRemainder_19_task1` is invalid.\n - Variable `p1_mRemainder_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomach_20_task1` is invalid.\n - Variable `p1_mStomach_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomachLumen_21_task1` is invalid.\n - Variable `p1_mStomachLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mUrine_22_task1` is invalid.\n - Variable `p1_mUrine_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mUrine']` are not defined. No namespaces are defined for the target.\n - Data generator `_01__StomachLumen_____IntestineLumen_1_task1` is invalid.\n - Variable `p2__01__StomachLumen_____IntestineLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_01__StomachLumen_____IntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `_02__IntestineLumen____Feces_2_task1` is invalid.\n - Variable `p2__02__IntestineLumen____Feces_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_02__IntestineLumen____Feces']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_1__PlasmaArterial____IntestineVascular_3_task1` is invalid.\n - Variable `p2__03_1__PlasmaArterial____IntestineVascular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_1__PlasmaArterial____IntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_2__IntestineLumen____IntestineEnterocytes__PMAT_OCT3_4_task1` is invalid.\n - Variable `p2__03_2__IntestineLumen____IntestineEnterocytes__PMAT_OCT3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_2__IntestineLumen____IntestineEnterocytes__PMAT_OCT3']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_3__IntestineEnterocytes____IntestineVascular__OCT1_5_task1` is invalid.\n - Variable `p2__03_3__IntestineEnterocytes____IntestineVascular__OCT1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_3__IntestineEnterocytes____IntestineVascular__OCT1']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_4__IntestineLumen____IntestineVascular__Saturable_6_task1` is invalid.\n - Variable `p2__03_4__IntestineLumen____IntestineVascular__Saturable_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_4__IntestineLumen____IntestineVascular__Saturable']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_5__IntestineVascular____PortalVein_7_task1` is invalid.\n - Variable `p2__03_5__IntestineVascular____PortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_5__IntestineVascular____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_6__IntestineLumen____IntestineEnterocytes__Diffusion_8_task1` is invalid.\n - Variable `p2__03_6__IntestineLumen____IntestineEnterocytes__Diffusion_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_6__IntestineLumen____IntestineEnterocytes__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_7__IntestineLumen____IntestineVascular__Diffusion_9_task1` is invalid.\n - Variable `p2__03_7__IntestineLumen____IntestineVascular__Diffusion_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_7__IntestineLumen____IntestineVascular__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_04__PlasmaArterial____PortalVein_10_task1` is invalid.\n - Variable `p2__04__PlasmaArterial____PortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_04__PlasmaArterial____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_1__PlasmaArterial____Stomach_11_task1` is invalid.\n - Variable `p2__05_1__PlasmaArterial____Stomach_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_1__PlasmaArterial____Stomach']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_2__Stomach____PortalVein_12_task1` is invalid.\n - Variable `p2__05_2__Stomach____PortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_2__Stomach____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_1__PlasmaArterial____Liver_13_task1` is invalid.\n - Variable `p2__06_1__PlasmaArterial____Liver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_1__PlasmaArterial____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_2__PortalVein____Liver_14_task1` is invalid.\n - Variable `p2__06_2__PortalVein____Liver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_2__PortalVein____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_3__Liver____PlasmaVenous_15_task1` is invalid.\n - Variable `p2__06_3__Liver____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_3__Liver____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_1__PlasmaVenous____Lung_16_task1` is invalid.\n - Variable `p2__07_1__PlasmaVenous____Lung_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_1__PlasmaVenous____Lung']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_2__Lung____PlasmaArterial_17_task1` is invalid.\n - Variable `p2__07_2__Lung____PlasmaArterial_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_2__Lung____PlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_1__PlasmaArterial____Brain_18_task1` is invalid.\n - Variable `p2__08_1__PlasmaArterial____Brain_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_1__PlasmaArterial____Brain']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_2__Brain____PlasmaVenous_19_task1` is invalid.\n - Variable `p2__08_2__Brain____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_2__Brain____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_1__PlasmaArterial____Muscle_20_task1` is invalid.\n - Variable `p2__09_1__PlasmaArterial____Muscle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_1__PlasmaArterial____Muscle']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_2__Muscle____PlasmaVenous_21_task1` is invalid.\n - Variable `p2__09_2__Muscle____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_2__Muscle____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_1__PlasmaArterial____Adipose_22_task1` is invalid.\n - Variable `p2__10_1__PlasmaArterial____Adipose_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_1__PlasmaArterial____Adipose']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_2__Adipose____PlasmaVenous_23_task1` is invalid.\n - Variable `p2__10_2__Adipose____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_2__Adipose____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_1__PlasmaArterial____Heart_24_task1` is invalid.\n - Variable `p2__11_1__PlasmaArterial____Heart_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_1__PlasmaArterial____Heart']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_2__Heart____PlasmaVenous_25_task1` is invalid.\n - Variable `p2__11_2__Heart____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_2__Heart____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_1__PlasmaArterial____Remainder_26_task1` is invalid.\n - Variable `p2__12_1__PlasmaArterial____Remainder_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_1__PlasmaArterial____Remainder']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_2__Remainder____PlasmaVenous_27_task1` is invalid.\n - Variable `p2__12_2__Remainder____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_2__Remainder____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_1__PlasmaArterial____KidneyPlasma_28_task1` is invalid.\n - Variable `p2__13_1__PlasmaArterial____KidneyPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_1__PlasmaArterial____KidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_2__KidneyPlasma____PlasmaVenous_29_task1` is invalid.\n - Variable `p2__13_2__KidneyPlasma____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_2__KidneyPlasma____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_3__KidneyPlasma____KidneyTubular_30_task1` is invalid.\n - Variable `p2__13_3__KidneyPlasma____KidneyTubular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_3__KidneyPlasma____KidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_4__KidneyPlasma____KidneyTissue_31_task1` is invalid.\n - Variable `p2__13_4__KidneyPlasma____KidneyTissue_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_4__KidneyPlasma____KidneyTissue']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_5__KidneyTissue____KidneyTubular_32_task1` is invalid.\n - Variable `p2__13_5__KidneyTissue____KidneyTubular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_5__KidneyTissue____KidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_6__KidneyTubular____UrineExternal_33_task1` is invalid.\n - Variable `p2__13_6__KidneyTubular____UrineExternal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_6__KidneyTubular____UrineExternal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgAdipose_1_task1` is invalid.\n - Variable `p3_mgAdipose_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mgBrain_2_task1` is invalid.\n - Variable `p3_mgBrain_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mgFeces_3_task1` is invalid.\n - Variable `p3_mgFeces_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mgHeart_4_task1` is invalid.\n - Variable `p3_mgHeart_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineEnterocytes_5_task1` is invalid.\n - Variable `p3_mgIntestineEnterocytes_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineLumen_6_task1` is invalid.\n - Variable `p3_mgIntestineLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineTotal_7_task1` is invalid.\n - Variable `p3_mgIntestineTotal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineVascular_8_task1` is invalid.\n - Variable `p3_mgIntestineVascular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyPlasma_9_task1` is invalid.\n - Variable `p3_mgKidneyPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTissues_10_task1` is invalid.\n - Variable `p3_mgKidneyTissues_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTissues']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTotal_11_task1` is invalid.\n - Variable `p3_mgKidneyTotal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTubular_12_task1` is invalid.\n - Variable `p3_mgKidneyTubular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLiver_13_task1` is invalid.\n - Variable `p3_mgLiver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLung_14_task1` is invalid.\n - Variable `p3_mgLung_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mgMuscle_15_task1` is invalid.\n - Variable `p3_mgMuscle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPlasmaArterial_16_task1` is invalid.\n - Variable `p3_mgPlasmaArterial_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPlasmaVenous_17_task1` is invalid.\n - Variable `p3_mgPlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPortalVein_18_task1` is invalid.\n - Variable `p3_mgPortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mgRemainder_19_task1` is invalid.\n - Variable `p3_mgRemainder_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomach_20_task1` is invalid.\n - Variable `p3_mgStomach_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomachLumen_21_task1` is invalid.\n - Variable `p3_mgStomachLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgUrine_22_task1` is invalid.\n - Variable `p3_mgUrine_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgUrine']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaVenous_1_task1` is invalid.\n - Variable `p4_mPlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineSum_2_task1` is invalid.\n - Variable `p4_mIntestineSum_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mIntestineSum']` are not defined. No namespaces are defined for the target.\n - Data generator `mLiver_3_task1` is invalid.\n - Variable `p4_mLiver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomach_4_task1` is invalid.\n - Variable `p4_mStomach_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mAdipose_1_task2` is invalid.\n - Variable `p1_mAdipose_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mBrain_2_task2` is invalid.\n - Variable `p1_mBrain_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mFeces_3_task2` is invalid.\n - Variable `p1_mFeces_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mHeart_4_task2` is invalid.\n - Variable `p1_mHeart_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineEnterocytes_5_task2` is invalid.\n - Variable `p1_mIntestineEnterocytes_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineLumen_6_task2` is invalid.\n - Variable `p1_mIntestineLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineSum_7_task2` is invalid.\n - Variable `p1_mIntestineSum_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mIntestineSum']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineVascular_8_task2` is invalid.\n - Variable `p1_mIntestineVascular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyPlasma_9_task2` is invalid.\n - Variable `p1_mKidneyPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneySum_10_task2` is invalid.\n - Variable `p1_mKidneySum_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mKidneySum']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTissue_11_task2` is invalid.\n - Variable `p1_mKidneyTissue_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTissue']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTubular_12_task2` is invalid.\n - Variable `p1_mKidneyTubular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mLiver_13_task2` is invalid.\n - Variable `p1_mLiver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mLung_14_task2` is invalid.\n - Variable `p1_mLung_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mMuscle_15_task2` is invalid.\n - Variable `p1_mMuscle_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaArterial_16_task2` is invalid.\n - Variable `p1_mPlasmaArterial_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaVenous_17_task2` is invalid.\n - Variable `p1_mPlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mPortalVein_18_task2` is invalid.\n - Variable `p1_mPortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mRemainder_19_task2` is invalid.\n - Variable `p1_mRemainder_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomach_20_task2` is invalid.\n - Variable `p1_mStomach_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomachLumen_21_task2` is invalid.\n - Variable `p1_mStomachLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mUrine_22_task2` is invalid.\n - Variable `p1_mUrine_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mUrine']` are not defined. No namespaces are defined for the target.\n - Data generator `_01__StomachLumen_____IntestineLumen_1_task2` is invalid.\n - Variable `p2__01__StomachLumen_____IntestineLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_01__StomachLumen_____IntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `_02__IntestineLumen____Feces_2_task2` is invalid.\n - Variable `p2__02__IntestineLumen____Feces_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_02__IntestineLumen____Feces']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_1__PlasmaArterial____IntestineVascular_3_task2` is invalid.\n - Variable `p2__03_1__PlasmaArterial____IntestineVascular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_1__PlasmaArterial____IntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_2__IntestineLumen____IntestineEnterocytes__PMAT_OCT3_4_task2` is invalid.\n - Variable `p2__03_2__IntestineLumen____IntestineEnterocytes__PMAT_OCT3_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_2__IntestineLumen____IntestineEnterocytes__PMAT_OCT3']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_3__IntestineEnterocytes____IntestineVascular__OCT1_5_task2` is invalid.\n - Variable `p2__03_3__IntestineEnterocytes____IntestineVascular__OCT1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_3__IntestineEnterocytes____IntestineVascular__OCT1']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_4__IntestineLumen____IntestineVascular__Saturable_6_task2` is invalid.\n - Variable `p2__03_4__IntestineLumen____IntestineVascular__Saturable_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_4__IntestineLumen____IntestineVascular__Saturable']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_5__IntestineVascular____PortalVein_7_task2` is invalid.\n - Variable `p2__03_5__IntestineVascular____PortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_5__IntestineVascular____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_6__IntestineLumen____IntestineEnterocytes__Diffusion_8_task2` is invalid.\n - Variable `p2__03_6__IntestineLumen____IntestineEnterocytes__Diffusion_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_6__IntestineLumen____IntestineEnterocytes__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_7__IntestineLumen____IntestineVascular__Diffusion_9_task2` is invalid.\n - Variable `p2__03_7__IntestineLumen____IntestineVascular__Diffusion_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_7__IntestineLumen____IntestineVascular__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_04__PlasmaArterial____PortalVein_10_task2` is invalid.\n - Variable `p2__04__PlasmaArterial____PortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_04__PlasmaArterial____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_1__PlasmaArterial____Stomach_11_task2` is invalid.\n - Variable `p2__05_1__PlasmaArterial____Stomach_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_1__PlasmaArterial____Stomach']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_2__Stomach____PortalVein_12_task2` is invalid.\n - Variable `p2__05_2__Stomach____PortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_2__Stomach____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_1__PlasmaArterial____Liver_13_task2` is invalid.\n - Variable `p2__06_1__PlasmaArterial____Liver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_1__PlasmaArterial____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_2__PortalVein____Liver_14_task2` is invalid.\n - Variable `p2__06_2__PortalVein____Liver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_2__PortalVein____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_3__Liver____PlasmaVenous_15_task2` is invalid.\n - Variable `p2__06_3__Liver____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_3__Liver____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_1__PlasmaVenous____Lung_16_task2` is invalid.\n - Variable `p2__07_1__PlasmaVenous____Lung_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_1__PlasmaVenous____Lung']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_2__Lung____PlasmaArterial_17_task2` is invalid.\n - Variable `p2__07_2__Lung____PlasmaArterial_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_2__Lung____PlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_1__PlasmaArterial____Brain_18_task2` is invalid.\n - Variable `p2__08_1__PlasmaArterial____Brain_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_1__PlasmaArterial____Brain']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_2__Brain____PlasmaVenous_19_task2` is invalid.\n - Variable `p2__08_2__Brain____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_2__Brain____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_1__PlasmaArterial____Muscle_20_task2` is invalid.\n - Variable `p2__09_1__PlasmaArterial____Muscle_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_1__PlasmaArterial____Muscle']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_2__Muscle____PlasmaVenous_21_task2` is invalid.\n - Variable `p2__09_2__Muscle____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_2__Muscle____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_1__PlasmaArterial____Adipose_22_task2` is invalid.\n - Variable `p2__10_1__PlasmaArterial____Adipose_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_1__PlasmaArterial____Adipose']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_2__Adipose____PlasmaVenous_23_task2` is invalid.\n - Variable `p2__10_2__Adipose____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_2__Adipose____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_1__PlasmaArterial____Heart_24_task2` is invalid.\n - Variable `p2__11_1__PlasmaArterial____Heart_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_1__PlasmaArterial____Heart']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_2__Heart____PlasmaVenous_25_task2` is invalid.\n - Variable `p2__11_2__Heart____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_2__Heart____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_1__PlasmaArterial____Remainder_26_task2` is invalid.\n - Variable `p2__12_1__PlasmaArterial____Remainder_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_1__PlasmaArterial____Remainder']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_2__Remainder____PlasmaVenous_27_task2` is invalid.\n - Variable `p2__12_2__Remainder____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_2__Remainder____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_1__PlasmaArterial____KidneyPlasma_28_task2` is invalid.\n - Variable `p2__13_1__PlasmaArterial____KidneyPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_1__PlasmaArterial____KidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_2__KidneyPlasma____PlasmaVenous_29_task2` is invalid.\n - Variable `p2__13_2__KidneyPlasma____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_2__KidneyPlasma____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_3__KidneyPlasma____KidneyTubular_30_task2` is invalid.\n - Variable `p2__13_3__KidneyPlasma____KidneyTubular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_3__KidneyPlasma____KidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_4__KidneyPlasma____KidneyTissue_31_task2` is invalid.\n - Variable `p2__13_4__KidneyPlasma____KidneyTissue_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_4__KidneyPlasma____KidneyTissue']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_5__KidneyTissue____KidneyTubular_32_task2` is invalid.\n - Variable `p2__13_5__KidneyTissue____KidneyTubular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_5__KidneyTissue____KidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_6__KidneyTubular____UrineExternal_33_task2` is invalid.\n - Variable `p2__13_6__KidneyTubular____UrineExternal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_6__KidneyTubular____UrineExternal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgAdipose_1_task2` is invalid.\n - Variable `p3_mgAdipose_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mgBrain_2_task2` is invalid.\n - Variable `p3_mgBrain_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mgFeces_3_task2` is invalid.\n - Variable `p3_mgFeces_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mgHeart_4_task2` is invalid.\n - Variable `p3_mgHeart_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineEnterocytes_5_task2` is invalid.\n - Variable `p3_mgIntestineEnterocytes_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineLumen_6_task2` is invalid.\n - Variable `p3_mgIntestineLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineTotal_7_task2` is invalid.\n - Variable `p3_mgIntestineTotal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineVascular_8_task2` is invalid.\n - Variable `p3_mgIntestineVascular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyPlasma_9_task2` is invalid.\n - Variable `p3_mgKidneyPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTissues_10_task2` is invalid.\n - Variable `p3_mgKidneyTissues_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTissues']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTotal_11_task2` is invalid.\n - Variable `p3_mgKidneyTotal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTubular_12_task2` is invalid.\n - Variable `p3_mgKidneyTubular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLiver_13_task2` is invalid.\n - Variable `p3_mgLiver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLung_14_task2` is invalid.\n - Variable `p3_mgLung_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mgMuscle_15_task2` is invalid.\n - Variable `p3_mgMuscle_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPlasmaArterial_16_task2` is invalid.\n - Variable `p3_mgPlasmaArterial_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPlasmaVenous_17_task2` is invalid.\n - Variable `p3_mgPlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPortalVein_18_task2` is invalid.\n - Variable `p3_mgPortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mgRemainder_19_task2` is invalid.\n - Variable `p3_mgRemainder_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomach_20_task2` is invalid.\n - Variable `p3_mgStomach_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomachLumen_21_task2` is invalid.\n - Variable `p3_mgStomachLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgUrine_22_task2` is invalid.\n - Variable `p3_mgUrine_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgUrine']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaVenous_1_task2` is invalid.\n - Variable `p4_mPlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineSum_2_task2` is invalid.\n - Variable `p4_mIntestineSum_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mIntestineSum']` are not defined. No namespaces are defined for the target.\n - Data generator `mLiver_3_task2` is invalid.\n - Variable `p4_mLiver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomach_4_task2` is invalid.\n - Variable `p4_mStomach_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomach']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Zake2021_Metformin+Mice+IV.sedml530686.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Output `plot_5_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_6_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_7_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_8_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Zake2021_Metformin+Mice+IV.sedml` has warnings.\r\n - Output `plot_5_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_6_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_7_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_8_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Zake2021_Metformin+Mice+IV.sedml530686.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Zake2021_Metformin+Mice+IV.sedml` is invalid.\n - Model `model1` is invalid.\n - The model file `model1.xml` is invalid.\n - `/tmp/tmppo7cy4t_/./model1.xml` is not a file.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Ktp_IntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mAdipose_1_task1` is invalid.\n - Variable `p1_mAdipose_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mBrain_2_task1` is invalid.\n - Variable `p1_mBrain_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mFeces_3_task1` is invalid.\n - Variable `p1_mFeces_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mHeart_4_task1` is invalid.\n - Variable `p1_mHeart_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineEnterocytes_5_task1` is invalid.\n - Variable `p1_mIntestineEnterocytes_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineLumen_6_task1` is invalid.\n - Variable `p1_mIntestineLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineSum_7_task1` is invalid.\n - Variable `p1_mIntestineSum_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mIntestineSum']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineVascular_8_task1` is invalid.\n - Variable `p1_mIntestineVascular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyPlasma_9_task1` is invalid.\n - Variable `p1_mKidneyPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneySum_10_task1` is invalid.\n - Variable `p1_mKidneySum_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mKidneySum']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTissue_11_task1` is invalid.\n - Variable `p1_mKidneyTissue_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTissue']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTubular_12_task1` is invalid.\n - Variable `p1_mKidneyTubular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mLiver_13_task1` is invalid.\n - Variable `p1_mLiver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mLung_14_task1` is invalid.\n - Variable `p1_mLung_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mMuscle_15_task1` is invalid.\n - Variable `p1_mMuscle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaArterial_16_task1` is invalid.\n - Variable `p1_mPlasmaArterial_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaVenous_17_task1` is invalid.\n - Variable `p1_mPlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mPortalVein_18_task1` is invalid.\n - Variable `p1_mPortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mRemainder_19_task1` is invalid.\n - Variable `p1_mRemainder_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomach_20_task1` is invalid.\n - Variable `p1_mStomach_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomachLumen_21_task1` is invalid.\n - Variable `p1_mStomachLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mUrine_22_task1` is invalid.\n - Variable `p1_mUrine_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mUrine']` are not defined. No namespaces are defined for the target.\n - Data generator `_01__StomachLumen_____IntestineLumen_1_task1` is invalid.\n - Variable `p2__01__StomachLumen_____IntestineLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_01__StomachLumen_____IntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `_02__IntestineLumen____Feces_2_task1` is invalid.\n - Variable `p2__02__IntestineLumen____Feces_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_02__IntestineLumen____Feces']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_1__PlasmaArterial____IntestineVascular_3_task1` is invalid.\n - Variable `p2__03_1__PlasmaArterial____IntestineVascular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_1__PlasmaArterial____IntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_2__IntestineLumen____IntestineEnterocytes__PMAT_OCT3_4_task1` is invalid.\n - Variable `p2__03_2__IntestineLumen____IntestineEnterocytes__PMAT_OCT3_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_2__IntestineLumen____IntestineEnterocytes__PMAT_OCT3']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_3__IntestineEnterocytes____IntestineVascular__OCT1_5_task1` is invalid.\n - Variable `p2__03_3__IntestineEnterocytes____IntestineVascular__OCT1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_3__IntestineEnterocytes____IntestineVascular__OCT1']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_4__IntestineLumen____IntestineVascular__Saturable_6_task1` is invalid.\n - Variable `p2__03_4__IntestineLumen____IntestineVascular__Saturable_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_4__IntestineLumen____IntestineVascular__Saturable']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_5__IntestineVascular____PortalVein_7_task1` is invalid.\n - Variable `p2__03_5__IntestineVascular____PortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_5__IntestineVascular____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_6__IntestineLumen____IntestineEnterocytes__Diffusion_8_task1` is invalid.\n - Variable `p2__03_6__IntestineLumen____IntestineEnterocytes__Diffusion_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_6__IntestineLumen____IntestineEnterocytes__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_7__IntestineLumen____IntestineVascular__Diffusion_9_task1` is invalid.\n - Variable `p2__03_7__IntestineLumen____IntestineVascular__Diffusion_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_7__IntestineLumen____IntestineVascular__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_04__PlasmaArterial____PortalVein_10_task1` is invalid.\n - Variable `p2__04__PlasmaArterial____PortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_04__PlasmaArterial____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_1__PlasmaArterial____Stomach_11_task1` is invalid.\n - Variable `p2__05_1__PlasmaArterial____Stomach_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_1__PlasmaArterial____Stomach']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_2__Stomach____PortalVein_12_task1` is invalid.\n - Variable `p2__05_2__Stomach____PortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_2__Stomach____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_1__PlasmaArterial____Liver_13_task1` is invalid.\n - Variable `p2__06_1__PlasmaArterial____Liver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_1__PlasmaArterial____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_2__PortalVein____Liver_14_task1` is invalid.\n - Variable `p2__06_2__PortalVein____Liver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_2__PortalVein____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_3__Liver____PlasmaVenous_15_task1` is invalid.\n - Variable `p2__06_3__Liver____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_3__Liver____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_1__PlasmaVenous____Lung_16_task1` is invalid.\n - Variable `p2__07_1__PlasmaVenous____Lung_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_1__PlasmaVenous____Lung']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_2__Lung____PlasmaArterial_17_task1` is invalid.\n - Variable `p2__07_2__Lung____PlasmaArterial_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_2__Lung____PlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_1__PlasmaArterial____Brain_18_task1` is invalid.\n - Variable `p2__08_1__PlasmaArterial____Brain_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_1__PlasmaArterial____Brain']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_2__Brain____PlasmaVenous_19_task1` is invalid.\n - Variable `p2__08_2__Brain____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_2__Brain____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_1__PlasmaArterial____Muscle_20_task1` is invalid.\n - Variable `p2__09_1__PlasmaArterial____Muscle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_1__PlasmaArterial____Muscle']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_2__Muscle____PlasmaVenous_21_task1` is invalid.\n - Variable `p2__09_2__Muscle____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_2__Muscle____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_1__PlasmaArterial____Adipose_22_task1` is invalid.\n - Variable `p2__10_1__PlasmaArterial____Adipose_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_1__PlasmaArterial____Adipose']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_2__Adipose____PlasmaVenous_23_task1` is invalid.\n - Variable `p2__10_2__Adipose____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_2__Adipose____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_1__PlasmaArterial____Heart_24_task1` is invalid.\n - Variable `p2__11_1__PlasmaArterial____Heart_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_1__PlasmaArterial____Heart']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_2__Heart____PlasmaVenous_25_task1` is invalid.\n - Variable `p2__11_2__Heart____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_2__Heart____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_1__PlasmaArterial____Remainder_26_task1` is invalid.\n - Variable `p2__12_1__PlasmaArterial____Remainder_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_1__PlasmaArterial____Remainder']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_2__Remainder____PlasmaVenous_27_task1` is invalid.\n - Variable `p2__12_2__Remainder____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_2__Remainder____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_1__PlasmaArterial____KidneyPlasma_28_task1` is invalid.\n - Variable `p2__13_1__PlasmaArterial____KidneyPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_1__PlasmaArterial____KidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_2__KidneyPlasma____PlasmaVenous_29_task1` is invalid.\n - Variable `p2__13_2__KidneyPlasma____PlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_2__KidneyPlasma____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_3__KidneyPlasma____KidneyTubular_30_task1` is invalid.\n - Variable `p2__13_3__KidneyPlasma____KidneyTubular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_3__KidneyPlasma____KidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_4__KidneyPlasma____KidneyTissue_31_task1` is invalid.\n - Variable `p2__13_4__KidneyPlasma____KidneyTissue_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_4__KidneyPlasma____KidneyTissue']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_5__KidneyTissue____KidneyTubular_32_task1` is invalid.\n - Variable `p2__13_5__KidneyTissue____KidneyTubular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_5__KidneyTissue____KidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_6__KidneyTubular____UrineExternal_33_task1` is invalid.\n - Variable `p2__13_6__KidneyTubular____UrineExternal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_6__KidneyTubular____UrineExternal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgAdipose_1_task1` is invalid.\n - Variable `p3_mgAdipose_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mgBrain_2_task1` is invalid.\n - Variable `p3_mgBrain_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mgFeces_3_task1` is invalid.\n - Variable `p3_mgFeces_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mgHeart_4_task1` is invalid.\n - Variable `p3_mgHeart_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineEnterocytes_5_task1` is invalid.\n - Variable `p3_mgIntestineEnterocytes_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineLumen_6_task1` is invalid.\n - Variable `p3_mgIntestineLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineTotal_7_task1` is invalid.\n - Variable `p3_mgIntestineTotal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineVascular_8_task1` is invalid.\n - Variable `p3_mgIntestineVascular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyPlasma_9_task1` is invalid.\n - Variable `p3_mgKidneyPlasma_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTissues_10_task1` is invalid.\n - Variable `p3_mgKidneyTissues_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTissues']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTotal_11_task1` is invalid.\n - Variable `p3_mgKidneyTotal_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTubular_12_task1` is invalid.\n - Variable `p3_mgKidneyTubular_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLiver_13_task1` is invalid.\n - Variable `p3_mgLiver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLung_14_task1` is invalid.\n - Variable `p3_mgLung_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mgMuscle_15_task1` is invalid.\n - Variable `p3_mgMuscle_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPlasmaArterial_16_task1` is invalid.\n - Variable `p3_mgPlasmaArterial_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPlasmaVenous_17_task1` is invalid.\n - Variable `p3_mgPlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPortalVein_18_task1` is invalid.\n - Variable `p3_mgPortalVein_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mgRemainder_19_task1` is invalid.\n - Variable `p3_mgRemainder_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomach_20_task1` is invalid.\n - Variable `p3_mgStomach_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomachLumen_21_task1` is invalid.\n - Variable `p3_mgStomachLumen_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgUrine_22_task1` is invalid.\n - Variable `p3_mgUrine_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgUrine']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaVenous_1_task1` is invalid.\n - Variable `p4_mPlasmaVenous_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineSum_2_task1` is invalid.\n - Variable `p4_mIntestineSum_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mIntestineSum']` are not defined. No namespaces are defined for the target.\n - Data generator `mLiver_3_task1` is invalid.\n - Variable `p4_mLiver_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomach_4_task1` is invalid.\n - Variable `p4_mStomach_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mAdipose_1_task2` is invalid.\n - Variable `p1_mAdipose_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mBrain_2_task2` is invalid.\n - Variable `p1_mBrain_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mFeces_3_task2` is invalid.\n - Variable `p1_mFeces_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mHeart_4_task2` is invalid.\n - Variable `p1_mHeart_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineEnterocytes_5_task2` is invalid.\n - Variable `p1_mIntestineEnterocytes_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineLumen_6_task2` is invalid.\n - Variable `p1_mIntestineLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineSum_7_task2` is invalid.\n - Variable `p1_mIntestineSum_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mIntestineSum']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineVascular_8_task2` is invalid.\n - Variable `p1_mIntestineVascular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyPlasma_9_task2` is invalid.\n - Variable `p1_mKidneyPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneySum_10_task2` is invalid.\n - Variable `p1_mKidneySum_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mKidneySum']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTissue_11_task2` is invalid.\n - Variable `p1_mKidneyTissue_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTissue']` are not defined. No namespaces are defined for the target.\n - Data generator `mKidneyTubular_12_task2` is invalid.\n - Variable `p1_mKidneyTubular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mLiver_13_task2` is invalid.\n - Variable `p1_mLiver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mLung_14_task2` is invalid.\n - Variable `p1_mLung_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mMuscle_15_task2` is invalid.\n - Variable `p1_mMuscle_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaArterial_16_task2` is invalid.\n - Variable `p1_mPlasmaArterial_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaVenous_17_task2` is invalid.\n - Variable `p1_mPlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mPortalVein_18_task2` is invalid.\n - Variable `p1_mPortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mRemainder_19_task2` is invalid.\n - Variable `p1_mRemainder_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomach_20_task2` is invalid.\n - Variable `p1_mStomach_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomachLumen_21_task2` is invalid.\n - Variable `p1_mStomachLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mUrine_22_task2` is invalid.\n - Variable `p1_mUrine_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mUrine']` are not defined. No namespaces are defined for the target.\n - Data generator `_01__StomachLumen_____IntestineLumen_1_task2` is invalid.\n - Variable `p2__01__StomachLumen_____IntestineLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_01__StomachLumen_____IntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `_02__IntestineLumen____Feces_2_task2` is invalid.\n - Variable `p2__02__IntestineLumen____Feces_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_02__IntestineLumen____Feces']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_1__PlasmaArterial____IntestineVascular_3_task2` is invalid.\n - Variable `p2__03_1__PlasmaArterial____IntestineVascular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_1__PlasmaArterial____IntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_2__IntestineLumen____IntestineEnterocytes__PMAT_OCT3_4_task2` is invalid.\n - Variable `p2__03_2__IntestineLumen____IntestineEnterocytes__PMAT_OCT3_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_2__IntestineLumen____IntestineEnterocytes__PMAT_OCT3']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_3__IntestineEnterocytes____IntestineVascular__OCT1_5_task2` is invalid.\n - Variable `p2__03_3__IntestineEnterocytes____IntestineVascular__OCT1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_3__IntestineEnterocytes____IntestineVascular__OCT1']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_4__IntestineLumen____IntestineVascular__Saturable_6_task2` is invalid.\n - Variable `p2__03_4__IntestineLumen____IntestineVascular__Saturable_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_4__IntestineLumen____IntestineVascular__Saturable']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_5__IntestineVascular____PortalVein_7_task2` is invalid.\n - Variable `p2__03_5__IntestineVascular____PortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_5__IntestineVascular____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_6__IntestineLumen____IntestineEnterocytes__Diffusion_8_task2` is invalid.\n - Variable `p2__03_6__IntestineLumen____IntestineEnterocytes__Diffusion_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_6__IntestineLumen____IntestineEnterocytes__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_03_7__IntestineLumen____IntestineVascular__Diffusion_9_task2` is invalid.\n - Variable `p2__03_7__IntestineLumen____IntestineVascular__Diffusion_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_03_7__IntestineLumen____IntestineVascular__Diffusion']` are not defined. No namespaces are defined for the target.\n - Data generator `_04__PlasmaArterial____PortalVein_10_task2` is invalid.\n - Variable `p2__04__PlasmaArterial____PortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_04__PlasmaArterial____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_1__PlasmaArterial____Stomach_11_task2` is invalid.\n - Variable `p2__05_1__PlasmaArterial____Stomach_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_1__PlasmaArterial____Stomach']` are not defined. No namespaces are defined for the target.\n - Data generator `_05_2__Stomach____PortalVein_12_task2` is invalid.\n - Variable `p2__05_2__Stomach____PortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_05_2__Stomach____PortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_1__PlasmaArterial____Liver_13_task2` is invalid.\n - Variable `p2__06_1__PlasmaArterial____Liver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_1__PlasmaArterial____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_2__PortalVein____Liver_14_task2` is invalid.\n - Variable `p2__06_2__PortalVein____Liver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_2__PortalVein____Liver']` are not defined. No namespaces are defined for the target.\n - Data generator `_06_3__Liver____PlasmaVenous_15_task2` is invalid.\n - Variable `p2__06_3__Liver____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_06_3__Liver____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_1__PlasmaVenous____Lung_16_task2` is invalid.\n - Variable `p2__07_1__PlasmaVenous____Lung_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_1__PlasmaVenous____Lung']` are not defined. No namespaces are defined for the target.\n - Data generator `_07_2__Lung____PlasmaArterial_17_task2` is invalid.\n - Variable `p2__07_2__Lung____PlasmaArterial_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_07_2__Lung____PlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_1__PlasmaArterial____Brain_18_task2` is invalid.\n - Variable `p2__08_1__PlasmaArterial____Brain_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_1__PlasmaArterial____Brain']` are not defined. No namespaces are defined for the target.\n - Data generator `_08_2__Brain____PlasmaVenous_19_task2` is invalid.\n - Variable `p2__08_2__Brain____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_08_2__Brain____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_1__PlasmaArterial____Muscle_20_task2` is invalid.\n - Variable `p2__09_1__PlasmaArterial____Muscle_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_1__PlasmaArterial____Muscle']` are not defined. No namespaces are defined for the target.\n - Data generator `_09_2__Muscle____PlasmaVenous_21_task2` is invalid.\n - Variable `p2__09_2__Muscle____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_09_2__Muscle____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_1__PlasmaArterial____Adipose_22_task2` is invalid.\n - Variable `p2__10_1__PlasmaArterial____Adipose_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_1__PlasmaArterial____Adipose']` are not defined. No namespaces are defined for the target.\n - Data generator `_10_2__Adipose____PlasmaVenous_23_task2` is invalid.\n - Variable `p2__10_2__Adipose____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_10_2__Adipose____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_1__PlasmaArterial____Heart_24_task2` is invalid.\n - Variable `p2__11_1__PlasmaArterial____Heart_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_1__PlasmaArterial____Heart']` are not defined. No namespaces are defined for the target.\n - Data generator `_11_2__Heart____PlasmaVenous_25_task2` is invalid.\n - Variable `p2__11_2__Heart____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_11_2__Heart____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_1__PlasmaArterial____Remainder_26_task2` is invalid.\n - Variable `p2__12_1__PlasmaArterial____Remainder_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_1__PlasmaArterial____Remainder']` are not defined. No namespaces are defined for the target.\n - Data generator `_12_2__Remainder____PlasmaVenous_27_task2` is invalid.\n - Variable `p2__12_2__Remainder____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_12_2__Remainder____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_1__PlasmaArterial____KidneyPlasma_28_task2` is invalid.\n - Variable `p2__13_1__PlasmaArterial____KidneyPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_1__PlasmaArterial____KidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_2__KidneyPlasma____PlasmaVenous_29_task2` is invalid.\n - Variable `p2__13_2__KidneyPlasma____PlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_2__KidneyPlasma____PlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_3__KidneyPlasma____KidneyTubular_30_task2` is invalid.\n - Variable `p2__13_3__KidneyPlasma____KidneyTubular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_3__KidneyPlasma____KidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_4__KidneyPlasma____KidneyTissue_31_task2` is invalid.\n - Variable `p2__13_4__KidneyPlasma____KidneyTissue_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_4__KidneyPlasma____KidneyTissue']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_5__KidneyTissue____KidneyTubular_32_task2` is invalid.\n - Variable `p2__13_5__KidneyTissue____KidneyTubular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_5__KidneyTissue____KidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `_13_6__KidneyTubular____UrineExternal_33_task2` is invalid.\n - Variable `p2__13_6__KidneyTubular____UrineExternal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='_13_6__KidneyTubular____UrineExternal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgAdipose_1_task2` is invalid.\n - Variable `p3_mgAdipose_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgAdipose']` are not defined. No namespaces are defined for the target.\n - Data generator `mgBrain_2_task2` is invalid.\n - Variable `p3_mgBrain_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgBrain']` are not defined. No namespaces are defined for the target.\n - Data generator `mgFeces_3_task2` is invalid.\n - Variable `p3_mgFeces_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgFeces']` are not defined. No namespaces are defined for the target.\n - Data generator `mgHeart_4_task2` is invalid.\n - Variable `p3_mgHeart_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgHeart']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineEnterocytes_5_task2` is invalid.\n - Variable `p3_mgIntestineEnterocytes_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineEnterocytes']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineLumen_6_task2` is invalid.\n - Variable `p3_mgIntestineLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineTotal_7_task2` is invalid.\n - Variable `p3_mgIntestineTotal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgIntestineVascular_8_task2` is invalid.\n - Variable `p3_mgIntestineVascular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgIntestineVascular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyPlasma_9_task2` is invalid.\n - Variable `p3_mgKidneyPlasma_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyPlasma']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTissues_10_task2` is invalid.\n - Variable `p3_mgKidneyTissues_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTissues']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTotal_11_task2` is invalid.\n - Variable `p3_mgKidneyTotal_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTotal']` are not defined. No namespaces are defined for the target.\n - Data generator `mgKidneyTubular_12_task2` is invalid.\n - Variable `p3_mgKidneyTubular_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgKidneyTubular']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLiver_13_task2` is invalid.\n - Variable `p3_mgLiver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mgLung_14_task2` is invalid.\n - Variable `p3_mgLung_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgLung']` are not defined. No namespaces are defined for the target.\n - Data generator `mgMuscle_15_task2` is invalid.\n - Variable `p3_mgMuscle_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgMuscle']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPlasmaArterial_16_task2` is invalid.\n - Variable `p3_mgPlasmaArterial_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPlasmaArterial']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPlasmaVenous_17_task2` is invalid.\n - Variable `p3_mgPlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mgPortalVein_18_task2` is invalid.\n - Variable `p3_mgPortalVein_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgPortalVein']` are not defined. No namespaces are defined for the target.\n - Data generator `mgRemainder_19_task2` is invalid.\n - Variable `p3_mgRemainder_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgRemainder']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomach_20_task2` is invalid.\n - Variable `p3_mgStomach_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomach']` are not defined. No namespaces are defined for the target.\n - Data generator `mgStomachLumen_21_task2` is invalid.\n - Variable `p3_mgStomachLumen_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgStomachLumen']` are not defined. No namespaces are defined for the target.\n - Data generator `mgUrine_22_task2` is invalid.\n - Variable `p3_mgUrine_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mgUrine']` are not defined. No namespaces are defined for the target.\n - Data generator `mPlasmaVenous_1_task2` is invalid.\n - Variable `p4_mPlasmaVenous_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mPlasmaVenous']` are not defined. No namespaces are defined for the target.\n - Data generator `mIntestineSum_2_task2` is invalid.\n - Variable `p4_mIntestineSum_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='mIntestineSum']` are not defined. No namespaces are defined for the target.\n - Data generator `mLiver_3_task2` is invalid.\n - Variable `p4_mLiver_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mLiver']` are not defined. No namespaces are defined for the target.\n - Data generator `mStomach_4_task2` is invalid.\n - Variable `p4_mStomach_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='mStomach']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000001040](https://www.ebi.ac.uk/biomodels/BIOMD0000001040)<br/><sup>Kurlovics2021 - Metformin part</sup></summary><sup>Kurlovics2021 - Metformin partitioning from plasma to RBC,  single coefficient</sup></details>|<details><summary>pass</summary>[Kurlovics2021_single.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001040#Files)</details>|pass|<details><summary>FAIL</summary>[Kurlovics2021_single.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001040#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/Kurlovics2021_single.sedml989257.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n\r\n`/root/in/Kurlovics2021_single.sedml989257.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Kurlovics2021_single.sedml` is invalid.\n - Model `model2` is invalid.\n - The model file `model2.xml` is invalid.\n - `/tmp/tmpa70hc5sb/./model2.xml` is not a file.\n - Data generator `Mpl_1_task1` is invalid.\n - Variable `p2_Mpl_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Mpl']` are not defined. No namespaces are defined for the target.\n - Data generator `Mrbc_2_task1` is invalid.\n - Variable `p2_Mrbc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Mrbc']` are not defined. No namespaces are defined for the target.\n - Data generator `Vin_1_task1` is invalid.\n - Variable `p3_Vin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='Vin']` are not defined. No namespaces are defined for the target.\n - Data generator `Vout_2_task1` is invalid.\n - Variable `p3_Vout_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='Vout']` are not defined. No namespaces are defined for the target.\n - Data generator `Summary_flux_to_RBC_3_task1` is invalid.\n - Variable `p3_Summary_flux_to_RBC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Summary_flux_to_RBC']` are not defined. No namespaces are defined for the target.\n - Output `plot_1_task1` is invalid.\n - Plot must have at least one curve.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Kurlovics2021_single.sedml235429.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Kurlovics2021_single.sedml235429.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Kurlovics2021_single.sedml` is invalid.\n - Model `model2` is invalid.\n - The model file `model2.xml` is invalid.\n - `/tmp/tmp7neiis3a/./model2.xml` is not a file.\n - Data generator `Mpl_1_task1` is invalid.\n - Variable `p2_Mpl_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Mpl']` are not defined. No namespaces are defined for the target.\n - Data generator `Mrbc_2_task1` is invalid.\n - Variable `p2_Mrbc_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Mrbc']` are not defined. No namespaces are defined for the target.\n - Data generator `Vin_1_task1` is invalid.\n - Variable `p3_Vin_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='Vin']` are not defined. No namespaces are defined for the target.\n - Data generator `Vout_2_task1` is invalid.\n - Variable `p3_Vout_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfReactions/sbml:reaction[@id='Vout']` are not defined. No namespaces are defined for the target.\n - Data generator `Summary_flux_to_RBC_3_task1` is invalid.\n - Variable `p3_Summary_flux_to_RBC_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Summary_flux_to_RBC']` are not defined. No namespaces are defined for the target.\n - Output `plot_1_task1` is invalid.\n - Plot must have at least one curve.\n"```</details>|
|<details><summary>[BIOMD0000001041](https://www.ebi.ac.uk/biomodels/BIOMD0000001041)<br/><sup>Kimmel2021 - T cell competitio</sup></summary><sup>Kimmel2021 - T cell competition and stochastic extinction events in CAR T cell therapy</sup></details>|<details><summary>pass</summary>[Kimmel2021.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001041#Files)</details>|FAIL|<details><summary>pass</summary>[Kimmel2021 - Fig2(a) and (b).sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001041#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Kimmel2021 - Fig2(a) and (b).sedml782754.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 2: b'usage: biosimulators-tellurium [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-tellurium: error: unrecognized arguments: - Fig2(a) and (b).sedml782754.omex\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Kimmel2021 - Fig2(a) and (b).sedml352325.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 2: b'usage: biosimulators-copasi [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-copasi: error: unrecognized arguments: - Fig2(a) and (b).sedml352325.omex\n'```</details>|
|<details><summary>[BIOMD0000001042](https://www.ebi.ac.uk/biomodels/BIOMD0000001042)<br/><sup>Makhlouf2020 - No treatment mo</sup></summary><sup>Makhlouf2020 - No treatment model of the role of CD4 T cells in tumor-immune interactions</sup></details>|<details><summary>pass</summary>[Makhlouf2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001042#Files)</details>|FAIL|<details><summary>MultipleSEDMLs</summary>['Makhlouf2020 - Fig1(a).sedml', 'Makhlouf2020 - Fig1(b).sedml', 'Makhlouf2020 - Fig1(c).sedml']</details>|NA|NA|NA|
|<details><summary>[BIOMD0000001043](https://www.ebi.ac.uk/biomodels/BIOMD0000001043)<br/><sup>Wodarz2001 - Viruses as antitu</sup></summary><sup>Wodarz2001 - Viruses as antitumor weapons</sup></details>|<details><summary>pass</summary>[Wodarz2001.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001043#Files)</details>|pass|<details><summary>MultipleSEDMLs</summary>['Wodarz2001 - Fig3A non-cytotoxic.sedml', 'Wodarz2001 - Fig3A more cytotoxic.sedml']</details>|NA|NA|NA|
|<details><summary>[BIOMD0000001044](https://www.ebi.ac.uk/biomodels/BIOMD0000001044)<br/><sup>Csikasz-Nagy2006 - Mammalian C</sup></summary><sup>Csikasz-Nagy2006 - Mammalian Cell Cycle model</sup></details>|<details><summary>FAIL</summary>[Csikasz-Nagy2006.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001044#Files)</details>|FAIL|<details><summary>pass</summary>[Csikasz-Nagy2006.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001044#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Csikasz-Nagy2006.sedml240532.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Csikasz-Nagy2006.sedml` has warnings.\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n\r\n`/root/in/Csikasz-Nagy2006.sedml240532.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Csikasz-Nagy2006.sedml` is invalid.\n - Model `model` is invalid.\n - The model file `Csikasz-Nagy2006.xml` is invalid.\n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout' cannot be used in this document.\n \n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout_L2' cannot be used in this document.\n \n - General SBML conformance (10102) at line 3, column 2: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L2V4 Section 4.1\n Element 'listOfLayouts' is not part of the definition of SBML Level 2 Version 4.\n \n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Mass']` are not defined. No namespaces are defined for the target.\n - Data generator `Cdh1_1_task1` is invalid.\n - Variable `p2_Cdh1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cdh1']` are not defined. No namespaces are defined for the target.\n - Data generator `CycA_2_task1` is invalid.\n - Variable `p2_CycA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CycA']` are not defined. No namespaces are defined for the target.\n - Data generator `CycB_3_task1` is invalid.\n - Variable `p2_CycB_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CycB']` are not defined. No namespaces are defined for the target.\n - Data generator `CycE_4_task1` is invalid.\n - Variable `p2_CycE_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CycE']` are not defined. No namespaces are defined for the target.\n - Data generator `Mass_5_task1` is invalid.\n - Variable `p2_Mass_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Mass']` are not defined. No namespaces are defined for the target.\n - Data generator `CKIT_6_task1` is invalid.\n - Variable `p2_CKIT_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CKIT']` are not defined. No namespaces are defined for the target.\n - Data generator `CycB_1_task1` is invalid.\n - Variable `p3_CycB_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CycB']` are not defined. No namespaces are defined for the target.\n - Data generator `Mass_1_task1` is invalid.\n - Variable `p3_Mass_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Mass']` are not defined. No namespaces are defined for the target.\n - Data generator `Cdh1_1_task2` is invalid.\n - Variable `p2_Cdh1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cdh1']` are not defined. No namespaces are defined for the target.\n - Data generator `CycA_2_task2` is invalid.\n - Variable `p2_CycA_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CycA']` are not defined. No namespaces are defined for the target.\n - Data generator `CycB_3_task2` is invalid.\n - Variable `p2_CycB_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CycB']` are not defined. No namespaces are defined for the target.\n - Data generator `CycE_4_task2` is invalid.\n - Variable `p2_CycE_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CycE']` are not defined. No namespaces are defined for the target.\n - Data generator `Mass_5_task2` is invalid.\n - Variable `p2_Mass_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Mass']` are not defined. No namespaces are defined for the target.\n - Data generator `CKIT_6_task2` is invalid.\n - Variable `p2_CKIT_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CKIT']` are not defined. No namespaces are defined for the target.\n - Data generator `CycB_1_task2` is invalid.\n - Variable `p3_CycB_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CycB']` are not defined. No namespaces are defined for the target.\n - Data generator `Mass_1_task2` is invalid.\n - Variable `p3_Mass_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Mass']` are not defined. No namespaces are defined for the target.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Csikasz-Nagy2006.sedml194572.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Csikasz-Nagy2006.sedml` has warnings.\r\n - Output `plot_3_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - Output `plot_4_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Csikasz-Nagy2006.sedml194572.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Csikasz-Nagy2006.sedml` is invalid.\n - Model `model` is invalid.\n - The model file `Csikasz-Nagy2006.xml` is invalid.\n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout' cannot be used in this document.\n \n - General SBML conformance (10103) at line 2, column 0: An SBML XML document must conform to the XML Schema for the corresponding SBML Level, Version and Release. The XML Schema for SBML defines the basic SBML object structure, the data types used by those objects, and the order in which the objects may appear in an SBML document.\n Reference: L2V4 Section 4.1\n The L3 package 'layout_L2' cannot be used in this document.\n \n - General SBML conformance (10102) at line 3, column 2: An SBML XML document must not contain undefined elements or attributes in the SBML namespace. Documents containing unknown elements or attributes placed in the SBML namespace do not conform to the SBML specification.\n Reference: L2V4 Section 4.1\n Element 'listOfLayouts' is not part of the definition of SBML Level 2 Version 4.\n \n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Mass']` are not defined. No namespaces are defined for the target.\n - Data generator `Cdh1_1_task1` is invalid.\n - Variable `p2_Cdh1_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cdh1']` are not defined. No namespaces are defined for the target.\n - Data generator `CycA_2_task1` is invalid.\n - Variable `p2_CycA_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CycA']` are not defined. No namespaces are defined for the target.\n - Data generator `CycB_3_task1` is invalid.\n - Variable `p2_CycB_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CycB']` are not defined. No namespaces are defined for the target.\n - Data generator `CycE_4_task1` is invalid.\n - Variable `p2_CycE_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CycE']` are not defined. No namespaces are defined for the target.\n - Data generator `Mass_5_task1` is invalid.\n - Variable `p2_Mass_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Mass']` are not defined. No namespaces are defined for the target.\n - Data generator `CKIT_6_task1` is invalid.\n - Variable `p2_CKIT_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CKIT']` are not defined. No namespaces are defined for the target.\n - Data generator `CycB_1_task1` is invalid.\n - Variable `p3_CycB_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CycB']` are not defined. No namespaces are defined for the target.\n - Data generator `Mass_1_task1` is invalid.\n - Variable `p3_Mass_task1` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Mass']` are not defined. No namespaces are defined for the target.\n - Data generator `Cdh1_1_task2` is invalid.\n - Variable `p2_Cdh1_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cdh1']` are not defined. No namespaces are defined for the target.\n - Data generator `CycA_2_task2` is invalid.\n - Variable `p2_CycA_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CycA']` are not defined. No namespaces are defined for the target.\n - Data generator `CycB_3_task2` is invalid.\n - Variable `p2_CycB_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CycB']` are not defined. No namespaces are defined for the target.\n - Data generator `CycE_4_task2` is invalid.\n - Variable `p2_CycE_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CycE']` are not defined. No namespaces are defined for the target.\n - Data generator `Mass_5_task2` is invalid.\n - Variable `p2_Mass_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Mass']` are not defined. No namespaces are defined for the target.\n - Data generator `CKIT_6_task2` is invalid.\n - Variable `p2_CKIT_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CKIT']` are not defined. No namespaces are defined for the target.\n - Data generator `CycB_1_task2` is invalid.\n - Variable `p3_CycB_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CycB']` are not defined. No namespaces are defined for the target.\n - Data generator `Mass_1_task2` is invalid.\n - Variable `p3_Mass_task2` is invalid.\n - One or more namespaces required for target `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Mass']` are not defined. No namespaces are defined for the target.\n"```</details>|
|<details><summary>[BIOMD0000001045](https://www.ebi.ac.uk/biomodels/BIOMD0000001045)<br/><sup>Smith&Moore2004 - The SIR mode</sup></summary><sup>Smith&Moore2004 - The SIR model for the spread of HongKong Flu</sup></details>|<details><summary>pass</summary>[Smith and moore2004-the SIR Model for Spread of Disease.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001045#Files)</details>|pass|<details><summary>pass</summary>[Smith and moore2004-the SIR Model for Spread of Disease.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001045#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Smith and moore2004-the SIR Model for Spread of Disease.sedml349461.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 2: b'usage: biosimulators-tellurium [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-tellurium: error: unrecognized arguments: and moore2004-the SIR Model for Spread of Disease.sedml349461.omex\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Smith and moore2004-the SIR Model for Spread of Disease.sedml21120.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 2: b'usage: biosimulators-copasi [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-copasi: error: unrecognized arguments: and moore2004-the SIR Model for Spread of Disease.sedml21120.omex\n'```</details>|
|<details><summary>[BIOMD0000001046](https://www.ebi.ac.uk/biomodels/BIOMD0000001046)<br/><sup>Raman2005 - Mycolic acid pathw</sup></summary><sup>Raman2005 - Mycolic acid pathway of M. tuberculosis</sup></details>|<details><summary>pass</summary>[Raman2005_MAP.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001046#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000001047](https://www.ebi.ac.uk/biomodels/BIOMD0000001047)<br/><sup>Collier1996 - Delta Notch inte</sup></summary><sup>Collier1996 - Delta Notch intercellular signalling and lateral inhibition</sup></details>|<details><summary>pass</summary>[delta_notch.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001047#Files)</details>|pass|<details><summary>pass</summary>[delta_notch.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001047#Files)</details>|pass|<details><summary>other</summary>```Command '-i /root/in/delta_notch.sedml322861.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: SedmlFeatureNotSupportedWarning:\r\n\r\n\x1b[33m`/tmp/tmpjads7_3c/./delta_notch.sedml` is encoded using L1V4. Only a few features introduced after L1V3 are supported. Most newer features, such as finer stylistic control of plots and parameter estimation, are not yet supported.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `delta_notch.xml` has warnings.\r\n - 4 warnings of type SBO term consistency (10703). The following is the first warning at line 424, column 6:\r\n - The value of the \'sboTerm\' attribute on a <parameter> is expected to be an SBO identifier (http://www.biomodels.net/SBO/) referring to a quantitative parameter defined in SBO (i.e., terms derived from SBO:0000002, "quantitative systems description parameter"). \r\n Reference: L2V4 Sections 4.9.5 and 5\r\n SBO term \'SBO:0000064\' on the <parameter> is not in the appropriate branch.\r\n - 16 warnings of type SBML unit consistency (99505). The following is the first warning at line 604, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression \'a\' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 14 warnings of type Modeling practice (80701). The following is the first warning at line 410, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id \'a\' does not have a \'units\' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./delta_notch.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `delta_notch.xml` has warnings.\r\n - 4 warnings of type SBO term consistency (10703). The following is the first warning at line 424, column 6:\r\n - The value of the \'sboTerm\' attribute on a <parameter> is expected to be an SBO identifier (http://www.biomodels.net/SBO/) referring to a quantitative parameter defined in SBO (i.e., terms derived from SBO:0000002, "quantitative systems description parameter"). \r\n Reference: L2V4 Sections 4.9.5 and 5\r\n SBO term \'SBO:0000064\' on the <parameter> is not in the appropriate branch.\r\n - 16 warnings of type SBML unit consistency (99505). The following is the first warning at line 604, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression \'a\' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 14 warnings of type Modeling practice (80701). The following is the first warning at line 410, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id \'a\' does not have a \'units\' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n`/root/in/delta_notch.sedml322861.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./delta_notch.sedml` is invalid.\n - Data generator `delta1_1_task1` is invalid.\n - Variable `p1_delta1_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'delta1\']` does not match any elements of model `model`.\n - Data generator `delta2_2_task1` is invalid.\n - Variable `p1_delta2_2_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'delta2\']` does not match any elements of model `model`.\n - Data generator `notch1_3_task1` is invalid.\n - Variable `p1_notch1_3_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'notch1\']` does not match any elements of model `model`.\n - Data generator `notch2_4_task1` is invalid.\n - Variable `p1_notch2_4_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'notch2\']` does not match any elements of model `model`.\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/delta_notch.sedml881325.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b'/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: SedmlFeatureNotSupportedWarning: \x1b[33m`/tmp/tmplrbfepxe/./delta_notch.sedml` is encoded using L1V4. Only a few features introduced after L1V3 are supported. Most newer features, such as finer stylistic control of plots and parameter estimation, are not yet supported.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `delta_notch.xml` has warnings.\r\n - 4 warnings of type SBO term consistency (10703). The following is the first warning at line 424, column 6:\r\n - The value of the \'sboTerm\' attribute on a <parameter> is expected to be an SBO identifier (http://www.biomodels.net/SBO/) referring to a quantitative parameter defined in SBO (i.e., terms derived from SBO:0000002, "quantitative systems description parameter"). \r\n Reference: L2V4 Sections 4.9.5 and 5\r\n SBO term \'SBO:0000064\' on the <parameter> is not in the appropriate branch.\r\n - 16 warnings of type SBML unit consistency (99505). The following is the first warning at line 604, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression \'a\' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 14 warnings of type Modeling practice (80701). The following is the first warning at line 410, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id \'a\' does not have a \'units\' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./delta_notch.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `delta_notch.xml` has warnings.\r\n - 4 warnings of type SBO term consistency (10703). The following is the first warning at line 424, column 6:\r\n - The value of the \'sboTerm\' attribute on a <parameter> is expected to be an SBO identifier (http://www.biomodels.net/SBO/) referring to a quantitative parameter defined in SBO (i.e., terms derived from SBO:0000002, "quantitative systems description parameter"). \r\n Reference: L2V4 Sections 4.9.5 and 5\r\n SBO term \'SBO:0000064\' on the <parameter> is not in the appropriate branch.\r\n - 16 warnings of type SBML unit consistency (99505). The following is the first warning at line 604, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression \'a\' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 14 warnings of type Modeling practice (80701). The following is the first warning at line 410, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id \'a\' does not have a \'units\' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/delta_notch.sedml881325.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./delta_notch.sedml` is invalid.\n - Data generator `delta1_1_task1` is invalid.\n - Variable `p1_delta1_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'delta1\']` does not match any elements of model `model`.\n - Data generator `delta2_2_task1` is invalid.\n - Variable `p1_delta2_2_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'delta2\']` does not match any elements of model `model`.\n - Data generator `notch1_3_task1` is invalid.\n - Variable `p1_notch1_3_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'notch1\']` does not match any elements of model `model`.\n - Data generator `notch2_4_task1` is invalid.\n - Variable `p1_notch2_4_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id=\'notch2\']` does not match any elements of model `model`.\n'```</details>|
|<details><summary>[BIOMD0000001048](https://www.ebi.ac.uk/biomodels/BIOMD0000001048)<br/><sup>Siddhartha2002 - Kinetic model</sup></summary><sup>Siddhartha2002 - Kinetic modelling of cancer therapies</sup></details>|<details><summary>pass</summary>[cancer_therapy.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001048#Files)</details>|pass|<details><summary>pass</summary>[cancer_therapy.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001048#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/cancer_therapy.sedml671017.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: SedmlFeatureNotSupportedWarning:\r\n\r\n\x1b[33m`/tmp/tmps_fi88qn/./cancer_therapy.sedml` is encoded using L1V4. Only a few features introduced after L1V3 are supported. Most newer features, such as finer stylistic control of plots and parameter estimation, are not yet supported.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `cancer_therapy.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 700, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Tumor * Mass_action_2(b, kf1, Tplas, Ttum, kr1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 417, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./cancer_therapy.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `cancer_therapy.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 700, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Tumor * Mass_action_2(b, kf1, Tplas, Ttum, kr1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 417, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n`/root/in/cancer_therapy.sedml671017.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./cancer_therapy.sedml` is invalid.\n - Data generator `Ttum_1_task1` is invalid.\n - Variable `p1_Ttum_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Ttum']` does not match any elements of model `model`.\n - Data generator `Tnew_2_task1` is invalid.\n - Variable `p1_Tnew_2_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Tnew']` does not match any elements of model `model`.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/cancer_therapy.sedml455892.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: SedmlFeatureNotSupportedWarning: \x1b[33m`/tmp/tmpkmcv9myt/./cancer_therapy.sedml` is encoded using L1V4. Only a few features introduced after L1V3 are supported. Most newer features, such as finer stylistic control of plots and parameter estimation, are not yet supported.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `cancer_therapy.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 700, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Tumor * Mass_action_2(b, kf1, Tplas, Ttum, kr1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 417, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./cancer_therapy.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `cancer_therapy.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (99505). The following is the first warning at line 700, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Tumor * Mass_action_2(b, kf1, Tplas, Ttum, kr1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 417, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'a' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/cancer_therapy.sedml455892.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./cancer_therapy.sedml` is invalid.\n - Data generator `Ttum_1_task1` is invalid.\n - Variable `p1_Ttum_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Ttum']` does not match any elements of model `model`.\n - Data generator `Tnew_2_task1` is invalid.\n - Variable `p1_Tnew_2_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Tnew']` does not match any elements of model `model`.\n"```</details>|
|<details><summary>[BIOMD0000001052](https://www.ebi.ac.uk/biomodels/BIOMD0000001052)<br/><sup>Alharbi2020 - Tumor and immune</sup></summary><sup>Alharbi2020 - Tumor and immune system competition</sup></details>|<details><summary>pass</summary>[Tumor_cells_immune_system_competition.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001052#Files)</details>|pass|<details><summary>pass</summary>[Tumor_cells_immune_system_competition.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001052#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Tumor_cells_immune_system_competition.sedml776574.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: SedmlFeatureNotSupportedWarning:\r\n\r\n\x1b[33m`/tmp/tmpvnemyfz8/./Tumor_cells_immune_system_competition.sedml` is encoded using L1V4. Only a few features introduced after L1V3 are supported. Most newer features, such as finer stylistic control of plots and parameter estimation, are not yet supported.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Tumor_cells_immune_system_competition.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 929, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Tissue * function_for_reaction_1(r, N)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 16 warnings of type Modeling practice (80701). The following is the first warning at line 582, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Tumor_cells_immune_system_competition.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Tumor_cells_immune_system_competition.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 929, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Tissue * function_for_reaction_1(r, N)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 16 warnings of type Modeling practice (80701). The following is the first warning at line 582, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n`/root/in/Tumor_cells_immune_system_competition.sedml776574.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Tumor_cells_immune_system_competition.sedml` is invalid.\n - Data generator `I_1_task1` is invalid.\n - Variable `p1_I_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` does not match any elements of model `model`.\n - Data generator `N_2_task1` is invalid.\n - Variable `p1_N_2_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` does not match any elements of model `model`.\n - Data generator `T_3_task1` is invalid.\n - Variable `p1_T_3_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` does not match any elements of model `model`.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Tumor_cells_immune_system_competition.sedml204950.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: SedmlFeatureNotSupportedWarning: \x1b[33m`/tmp/tmp1w70ih4x/./Tumor_cells_immune_system_competition.sedml` is encoded using L1V4. Only a few features introduced after L1V3 are supported. Most newer features, such as finer stylistic control of plots and parameter estimation, are not yet supported.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Tumor_cells_immune_system_competition.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 929, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Tissue * function_for_reaction_1(r, N)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 16 warnings of type Modeling practice (80701). The following is the first warning at line 582, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Tumor_cells_immune_system_competition.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Tumor_cells_immune_system_competition.xml` has warnings.\r\n - 14 warnings of type SBML unit consistency (99505). The following is the first warning at line 929, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Tissue * function_for_reaction_1(r, N)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 16 warnings of type Modeling practice (80701). The following is the first warning at line 582, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'r' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Tumor_cells_immune_system_competition.sedml204950.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Tumor_cells_immune_system_competition.sedml` is invalid.\n - Data generator `I_1_task1` is invalid.\n - Variable `p1_I_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='I']` does not match any elements of model `model`.\n - Data generator `N_2_task1` is invalid.\n - Variable `p1_N_2_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='N']` does not match any elements of model `model`.\n - Data generator `T_3_task1` is invalid.\n - Variable `p1_T_3_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='T']` does not match any elements of model `model`.\n"```</details>|
|<details><summary>[BIOMD0000001053](https://www.ebi.ac.uk/biomodels/BIOMD0000001053)<br/><sup>Garde2020 - metabolic oscillat</sup></summary><sup>Garde2020 - metabolic oscillations in Bacillus subtilis biofilms</sup></details>|<details><summary>pass</summary>[Garde2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001053#Files)</details>|pass|<details><summary>pass</summary>[Garde2020.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001053#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Garde2020.sedml974874.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: SedmlFeatureNotSupportedWarning:\r\n\r\n\x1b[33m`/tmp/tmpbmo1giti/./Garde2020.sedml` is encoded using L1V4. Only a few features introduced after L1V3 are supported. Most newer features, such as finer stylistic control of plots and parameter estimation, are not yet supported.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Garde2020.sedml` has warnings.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n`/root/in/Garde2020.sedml974874.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Garde2020.sedml` is invalid.\n - Data generator `Gp_1_task1` is invalid.\n - Variable `p1_Gp_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Gp']` does not match any elements of model `model`.\n - Data generator `Ai_2_task1` is invalid.\n - Variable `p1_Ai_2_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Ai']` does not match any elements of model `model`.\n - Data generator `Am_3_task1` is invalid.\n - Variable `p1_Am_3_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Am']` does not match any elements of model `model`.\n - Data generator `Ap_4_task1` is invalid.\n - Variable `p1_Ap_4_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Ap']` does not match any elements of model `model`.\n - Data generator `Gm_5_task1` is invalid.\n - Variable `p1_Gm_5_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Gm']` does not match any elements of model `model`.\n - Data generator `Gi_6_task1` is invalid.\n - Variable `p1_Gi_6_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Gi']` does not match any elements of model `model`.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Garde2020.sedml252811.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: SedmlFeatureNotSupportedWarning: \x1b[33m`/tmp/tmpnlkc5o4o/./Garde2020.sedml` is encoded using L1V4. Only a few features introduced after L1V3 are supported. Most newer features, such as finer stylistic control of plots and parameter estimation, are not yet supported.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Garde2020.sedml` has warnings.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Garde2020.sedml252811.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Garde2020.sedml` is invalid.\n - Data generator `Gp_1_task1` is invalid.\n - Variable `p1_Gp_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Gp']` does not match any elements of model `model`.\n - Data generator `Ai_2_task1` is invalid.\n - Variable `p1_Ai_2_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Ai']` does not match any elements of model `model`.\n - Data generator `Am_3_task1` is invalid.\n - Variable `p1_Am_3_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Am']` does not match any elements of model `model`.\n - Data generator `Ap_4_task1` is invalid.\n - Variable `p1_Ap_4_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Ap']` does not match any elements of model `model`.\n - Data generator `Gm_5_task1` is invalid.\n - Variable `p1_Gm_5_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Gm']` does not match any elements of model `model`.\n - Data generator `Gi_6_task1` is invalid.\n - Variable `p1_Gi_6_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Gi']` does not match any elements of model `model`.\n"```</details>|
|[BIOMD0000001054](https://www.ebi.ac.uk/biomodels/BIOMD0000001054)<br/><sup>Pearce2021 - Fibrin Polymerization</sup>|<details><summary>pass</summary>[fibrin_polymerization.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001054#Files)</details>|pass|<details><summary>pass</summary>[fibrin_polymerization.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001054#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/fibrin_polymerization.sedml953275.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: SedmlFeatureNotSupportedWarning:\r\n\r\n\x1b[33m`/tmp/tmp_n689af4/./fibrin_polymerization.sedml` is encoded using L1V4. Only a few features introduced after L1V3 are supported. Most newer features, such as finer stylistic control of plots and parameter estimation, are not yet supported.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `fibrin_polymerization.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (99505). The following is the first warning at line 702, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Endothelium * k1 * Fbna * Thb' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 406, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./fibrin_polymerization.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `fibrin_polymerization.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (99505). The following is the first warning at line 702, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Endothelium * k1 * Fbna * Thb' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 406, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n`/root/in/fibrin_polymerization.sedml953275.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./fibrin_polymerization.sedml` is invalid.\n - Data generator `FM_1_task1` is invalid.\n - Variable `p1_FM_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FM']` does not match any elements of model `model`.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/fibrin_polymerization.sedml37467.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: SedmlFeatureNotSupportedWarning: \x1b[33m`/tmp/tmp8yuoyzqf/./fibrin_polymerization.sedml` is encoded using L1V4. Only a few features introduced after L1V3 are supported. Most newer features, such as finer stylistic control of plots and parameter estimation, are not yet supported.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `fibrin_polymerization.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (99505). The following is the first warning at line 702, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Endothelium * k1 * Fbna * Thb' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 406, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./fibrin_polymerization.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `fibrin_polymerization.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (99505). The following is the first warning at line 702, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Endothelium * k1 * Fbna * Thb' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 11 warnings of type Modeling practice (80701). The following is the first warning at line 406, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/fibrin_polymerization.sedml37467.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./fibrin_polymerization.sedml` is invalid.\n - Data generator `FM_1_task1` is invalid.\n - Variable `p1_FM_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='FM']` does not match any elements of model `model`.\n"```</details>|
|<details><summary>[BIOMD0000001055](https://www.ebi.ac.uk/biomodels/BIOMD0000001055)<br/><sup>Jeon2018 - Enzyme clustering i</sup></summary><sup>Jeon2018 - Enzyme clustering in Glucose metabolism</sup></details>|<details><summary>pass</summary>[Enzyme_clustering_in glucose_metabolism.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001055#Files)</details>|pass|<details><summary>pass</summary>[Enzyme_clustering_in glucose_metabolism.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001055#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Enzyme_clustering_in glucose_metabolism.sedml312036.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 2: b'usage: biosimulators-tellurium [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-tellurium: error: unrecognized arguments: glucose_metabolism.sedml312036.omex\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Enzyme_clustering_in glucose_metabolism.sedml551420.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 2: b'usage: biosimulators-copasi [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-copasi: error: unrecognized arguments: glucose_metabolism.sedml551420.omex\n'```</details>|
|<details><summary>[BIOMD0000001056](https://www.ebi.ac.uk/biomodels/BIOMD0000001056)<br/><sup>Chulian2021 - feedback signall</sup></summary><sup>Chulian2021 - feedback signalling in B lymphopoeisis</sup></details>|<details><summary>pass</summary>[chulian2021.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001056#Files)</details>|pass|<details><summary>pass</summary>[chulian2021.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001056#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/chulian2021.sedml615780.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: SedmlFeatureNotSupportedWarning:\r\n\r\n\x1b[33m`/tmp/tmp3rp3xr5q/./chulian2021.sedml` is encoded using L1V4. Only a few features introduced after L1V3 are supported. Most newer features, such as finer stylistic control of plots and parameter estimation, are not yet supported.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `chulian2021.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 441, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'k' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type Modeling practice (80701). The following is the first warning at line 250, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 's1' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./chulian2021.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `chulian2021.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 441, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'k' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type Modeling practice (80701). The following is the first warning at line 250, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 's1' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n`/root/in/chulian2021.sedml615780.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./chulian2021.sedml` is invalid.\n - Data generator `C1_1_task1` is invalid.\n - Variable `p1_C1_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C1']` does not match any elements of model `model`.\n - Data generator `C2_2_task1` is invalid.\n - Variable `p1_C2_2_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C2']` does not match any elements of model `model`.\n - Data generator `C3_3_task1` is invalid.\n - Variable `p1_C3_3_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C3']` does not match any elements of model `model`.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/chulian2021.sedml999347.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: SedmlFeatureNotSupportedWarning: \x1b[33m`/tmp/tmpsh9p_7l9/./chulian2021.sedml` is encoded using L1V4. Only a few features introduced after L1V3 are supported. Most newer features, such as finer stylistic control of plots and parameter estimation, are not yet supported.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `chulian2021.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 441, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'k' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type Modeling practice (80701). The following is the first warning at line 250, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 's1' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./chulian2021.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `chulian2021.xml` has warnings.\r\n - 8 warnings of type SBML unit consistency (99505). The following is the first warning at line 441, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'k' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 9 warnings of type Modeling practice (80701). The following is the first warning at line 250, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 's1' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/chulian2021.sedml999347.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./chulian2021.sedml` is invalid.\n - Data generator `C1_1_task1` is invalid.\n - Variable `p1_C1_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C1']` does not match any elements of model `model`.\n - Data generator `C2_2_task1` is invalid.\n - Variable `p1_C2_2_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C2']` does not match any elements of model `model`.\n - Data generator `C3_3_task1` is invalid.\n - Variable `p1_C3_3_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='C3']` does not match any elements of model `model`.\n"```</details>|
|[BIOMD0000001057](https://www.ebi.ac.uk/biomodels/BIOMD0000001057)<br/><sup>Nikolov2020 - p53-miR34 model</sup>|<details><summary>pass</summary>[Nikolov2020.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001057#Files)</details>|pass|<details><summary>pass</summary>[Nikolov2020.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001057#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Nikolov2020.sedml665905.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: SedmlFeatureNotSupportedWarning:\r\n\r\n\x1b[33m`/tmp/tmpz7oobm0x/./Nikolov2020.sedml` is encoded using L1V4. Only a few features introduced after L1V3 are supported. Most newer features, such as finer stylistic control of plots and parameter estimation, are not yet supported.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Nikolov2020.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 480, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'cell * Constant_flux__irreversible(k1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type Modeling practice (80701). The following is the first warning at line 292, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Nikolov2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Nikolov2020.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 480, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'cell * Constant_flux__irreversible(k1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type Modeling practice (80701). The following is the first warning at line 292, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n`/root/in/Nikolov2020.sedml665905.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Nikolov2020.sedml` is invalid.\n - Data generator `TFp53_1_task1` is invalid.\n - Variable `p1_TFp53_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='TFp53']` does not match any elements of model `model`.\n - Data generator `miR34a_1_task1` is invalid.\n - Variable `p2_miR34a_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='miR34a']` does not match any elements of model `model`.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Nikolov2020.sedml510072.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: SedmlFeatureNotSupportedWarning: \x1b[33m`/tmp/tmpzbxyy42n/./Nikolov2020.sedml` is encoded using L1V4. Only a few features introduced after L1V3 are supported. Most newer features, such as finer stylistic control of plots and parameter estimation, are not yet supported.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Nikolov2020.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 480, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'cell * Constant_flux__irreversible(k1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type Modeling practice (80701). The following is the first warning at line 292, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Nikolov2020.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Nikolov2020.xml` has warnings.\r\n - 5 warnings of type SBML unit consistency (99505). The following is the first warning at line 480, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'cell * Constant_flux__irreversible(k1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 7 warnings of type Modeling practice (80701). The following is the first warning at line 292, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Nikolov2020.sedml510072.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Nikolov2020.sedml` is invalid.\n - Data generator `TFp53_1_task1` is invalid.\n - Variable `p1_TFp53_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='TFp53']` does not match any elements of model `model`.\n - Data generator `miR34a_1_task1` is invalid.\n - Variable `p2_miR34a_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='miR34a']` does not match any elements of model `model`.\n"```</details>|
|<details><summary>[BIOMD0000001058](https://www.ebi.ac.uk/biomodels/BIOMD0000001058)<br/><sup>Novak2022 - Mitotic kinase osc</sup></summary><sup>Novak2022 - Mitotic kinase oscillation</sup></details>|<details><summary>pass</summary>[Novak2022.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001058#Files)</details>|pass|<details><summary>pass</summary>[Novak2022.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001058#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Novak2022.sedml632726.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: SedmlFeatureNotSupportedWarning:\r\n\r\n\x1b[33m`/tmp/tmp_l6fv4d7/./Novak2022.sedml` is encoded using L1V4. Only a few features introduced after L1V3 are supported. Most newer features, such as finer stylistic control of plots and parameter estimation, are not yet supported.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Novak2022.xml` has warnings.\r\n - 30 warnings of type SBML unit consistency (99505). The following is the first warning at line 2091, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Jmbf' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 48 warnings of type Modeling practice (80701). The following is the first warning at line 987, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kscln' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Novak2022.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Novak2022.xml` has warnings.\r\n - 30 warnings of type SBML unit consistency (99505). The following is the first warning at line 2091, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Jmbf' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 48 warnings of type Modeling practice (80701). The following is the first warning at line 987, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kscln' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n`/root/in/Novak2022.sedml632726.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Novak2022.sedml` is invalid.\n - Data generator `Cln_1_task1` is invalid.\n - Variable `p1_Cln_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cln']` does not match any elements of model `model`.\n - Data generator `Cdh1_2_task1` is invalid.\n - Variable `p1_Cdh1_2_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cdh1']` does not match any elements of model `model`.\n - Data generator `Cdc14_3_task1` is invalid.\n - Variable `p1_Cdc14_3_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cdc14']` does not match any elements of model `model`.\n - Data generator `ClbM_4_task1` is invalid.\n - Variable `p1_ClbM_4_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='ClbM']` does not match any elements of model `model`.\n - Data generator `ClbS_1_task1` is invalid.\n - Variable `p2_ClbS_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='ClbS']` does not match any elements of model `model`.\n - Data generator `MBF_2_task1` is invalid.\n - Variable `p2_MBF_2_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='MBF']` does not match any elements of model `model`.\n - Data generator `Nrm1t_3_task1` is invalid.\n - Variable `p2_Nrm1t_3_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Nrm1t']` does not match any elements of model `model`.\n - Data generator `Polo_4_task1` is invalid.\n - Variable `p2_Polo_4_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Polo']` does not match any elements of model `model`.\n - Data generator `Sic1t_5_task1` is invalid.\n - Variable `p2_Sic1t_5_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Sic1t']` does not match any elements of model `model`.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Novak2022.sedml595775.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: SedmlFeatureNotSupportedWarning: \x1b[33m`/tmp/tmpk1ztu_gk/./Novak2022.sedml` is encoded using L1V4. Only a few features introduced after L1V3 are supported. Most newer features, such as finer stylistic control of plots and parameter estimation, are not yet supported.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Novak2022.xml` has warnings.\r\n - 30 warnings of type SBML unit consistency (99505). The following is the first warning at line 2091, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Jmbf' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 48 warnings of type Modeling practice (80701). The following is the first warning at line 987, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kscln' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Novak2022.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Novak2022.xml` has warnings.\r\n - 30 warnings of type SBML unit consistency (99505). The following is the first warning at line 2091, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Jmbf' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 48 warnings of type Modeling practice (80701). The following is the first warning at line 987, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kscln' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Novak2022.sedml595775.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Novak2022.sedml` is invalid.\n - Data generator `Cln_1_task1` is invalid.\n - Variable `p1_Cln_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cln']` does not match any elements of model `model`.\n - Data generator `Cdh1_2_task1` is invalid.\n - Variable `p1_Cdh1_2_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cdh1']` does not match any elements of model `model`.\n - Data generator `Cdc14_3_task1` is invalid.\n - Variable `p1_Cdc14_3_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Cdc14']` does not match any elements of model `model`.\n - Data generator `ClbM_4_task1` is invalid.\n - Variable `p1_ClbM_4_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='ClbM']` does not match any elements of model `model`.\n - Data generator `ClbS_1_task1` is invalid.\n - Variable `p2_ClbS_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='ClbS']` does not match any elements of model `model`.\n - Data generator `MBF_2_task1` is invalid.\n - Variable `p2_MBF_2_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='MBF']` does not match any elements of model `model`.\n - Data generator `Nrm1t_3_task1` is invalid.\n - Variable `p2_Nrm1t_3_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Nrm1t']` does not match any elements of model `model`.\n - Data generator `Polo_4_task1` is invalid.\n - Variable `p2_Polo_4_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Polo']` does not match any elements of model `model`.\n - Data generator `Sic1t_5_task1` is invalid.\n - Variable `p2_Sic1t_5_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Sic1t']` does not match any elements of model `model`.\n"```</details>|
|[BIOMD0000001059](https://www.ebi.ac.uk/biomodels/BIOMD0000001059)<br/><sup>Stucki2005 - caspase-3 metabolism</sup>|<details><summary>pass</summary>[Stucki2005.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001059#Files)</details>|pass|<details><summary>pass</summary>[Stucki2005.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001059#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Stucki2005.sedml832335.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: SedmlFeatureNotSupportedWarning:\r\n\r\n\x1b[33m`/tmp/tmpn4a9rxq9/./Stucki2005.sedml` is encoded using L1V4. Only a few features introduced after L1V3 are supported. Most newer features, such as finer stylistic control of plots and parameter estimation, are not yet supported.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Stucki2005.xml` has warnings.\r\n - 31 warnings of type SBML unit consistency (99505). The following is the first warning at line 1072, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Cytoplasm * Constant_flux__irreversible(k1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 23 warnings of type Modeling practice (80701). The following is the first warning at line 516, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Stucki2005.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Stucki2005.xml` has warnings.\r\n - 31 warnings of type SBML unit consistency (99505). The following is the first warning at line 1072, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Cytoplasm * Constant_flux__irreversible(k1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 23 warnings of type Modeling practice (80701). The following is the first warning at line 516, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n`/root/in/Stucki2005.sedml832335.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Stucki2005.sedml` is invalid.\n - Data generator `c3_1_task1` is invalid.\n - Variable `p1_c3_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='c3']` does not match any elements of model `model`.\n - Data generator `iap_2_task1` is invalid.\n - Variable `p1_iap_2_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='iap']` does not match any elements of model `model`.\n - Data generator `survivin_3_task1` is invalid.\n - Variable `p1_survivin_3_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='survivin']` does not match any elements of model `model`.\n - Data generator `cascade_4_task1` is invalid.\n - Variable `p1_cascade_4_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='cascade']` does not match any elements of model `model`.\n - Data generator `iapc3_5_task1` is invalid.\n - Variable `p1_iapc3_5_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='iapc3']` does not match any elements of model `model`.\n - Data generator `iapsmac_6_task1` is invalid.\n - Variable `p1_iapsmac_6_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='iapsmac']` does not match any elements of model `model`.\n - Data generator `sursmac_7_task1` is invalid.\n - Variable `p1_sursmac_7_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='sursmac']` does not match any elements of model `model`.\n - Data generator `smac_8_task1` is invalid.\n - Variable `p1_smac_8_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='smac']` does not match any elements of model `model`.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Stucki2005.sedml995705.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: SedmlFeatureNotSupportedWarning: \x1b[33m`/tmp/tmp7_fed7pc/./Stucki2005.sedml` is encoded using L1V4. Only a few features introduced after L1V3 are supported. Most newer features, such as finer stylistic control of plots and parameter estimation, are not yet supported.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Stucki2005.xml` has warnings.\r\n - 31 warnings of type SBML unit consistency (99505). The following is the first warning at line 1072, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Cytoplasm * Constant_flux__irreversible(k1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 23 warnings of type Modeling practice (80701). The following is the first warning at line 516, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Stucki2005.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Stucki2005.xml` has warnings.\r\n - 31 warnings of type SBML unit consistency (99505). The following is the first warning at line 1072, column 8:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <kineticLaw> <math> expression 'Cytoplasm * Constant_flux__irreversible(k1)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 23 warnings of type Modeling practice (80701). The following is the first warning at line 516, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'k1' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Stucki2005.sedml995705.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Stucki2005.sedml` is invalid.\n - Data generator `c3_1_task1` is invalid.\n - Variable `p1_c3_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='c3']` does not match any elements of model `model`.\n - Data generator `iap_2_task1` is invalid.\n - Variable `p1_iap_2_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='iap']` does not match any elements of model `model`.\n - Data generator `survivin_3_task1` is invalid.\n - Variable `p1_survivin_3_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='survivin']` does not match any elements of model `model`.\n - Data generator `cascade_4_task1` is invalid.\n - Variable `p1_cascade_4_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='cascade']` does not match any elements of model `model`.\n - Data generator `iapc3_5_task1` is invalid.\n - Variable `p1_iapc3_5_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='iapc3']` does not match any elements of model `model`.\n - Data generator `iapsmac_6_task1` is invalid.\n - Variable `p1_iapsmac_6_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='iapsmac']` does not match any elements of model `model`.\n - Data generator `sursmac_7_task1` is invalid.\n - Variable `p1_sursmac_7_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='sursmac']` does not match any elements of model `model`.\n - Data generator `smac_8_task1` is invalid.\n - Variable `p1_smac_8_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='smac']` does not match any elements of model `model`.\n"```</details>|
|[BIOMD0000001060](https://www.ebi.ac.uk/biomodels/BIOMD0000001060)<br/><sup>Frank2021 - Macrophage polarization</sup>|<details><summary>pass</summary>[Frank2021.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001060#Files)</details>|pass|<details><summary>pass</summary>[Frank2021.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001060#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Frank2021.sedml197871.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: SedmlFeatureNotSupportedWarning:\r\n\r\n\x1b[33m`/tmp/tmpove_feai/./Frank2021.sedml` is encoded using L1V4. Only a few features introduced after L1V3 are supported. Most newer features, such as finer stylistic control of plots and parameter estimation, are not yet supported.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Frank2021.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (99505). The following is the first warning at line 787, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'pow(x1, n1) / (pow(x1, n1) + pow(k1, n1))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 20 warnings of type Modeling practice (80701). The following is the first warning at line 305, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'H_plus_x1' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Frank2021.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Frank2021.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (99505). The following is the first warning at line 787, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'pow(x1, n1) / (pow(x1, n1) + pow(k1, n1))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 20 warnings of type Modeling practice (80701). The following is the first warning at line 305, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'H_plus_x1' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n`/root/in/Frank2021.sedml197871.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Frank2021.sedml` is invalid.\n - Data generator `x1_1_task1` is invalid.\n - Variable `p1_x1_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x1']` does not match any elements of model `model`.\n - Data generator `x2_2_task1` is invalid.\n - Variable `p1_x2_2_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x2']` does not match any elements of model `model`.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Frank2021.sedml623483.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: SedmlFeatureNotSupportedWarning: \x1b[33m`/tmp/tmpyzd_q3qi/./Frank2021.sedml` is encoded using L1V4. Only a few features introduced after L1V3 are supported. Most newer features, such as finer stylistic control of plots and parameter estimation, are not yet supported.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Frank2021.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (99505). The following is the first warning at line 787, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'pow(x1, n1) / (pow(x1, n1) + pow(k1, n1))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 20 warnings of type Modeling practice (80701). The following is the first warning at line 305, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'H_plus_x1' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Frank2021.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Frank2021.xml` has warnings.\r\n - 11 warnings of type SBML unit consistency (99505). The following is the first warning at line 787, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'pow(x1, n1) / (pow(x1, n1) + pow(k1, n1))' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 20 warnings of type Modeling practice (80701). The following is the first warning at line 305, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'H_plus_x1' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Frank2021.sedml623483.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Frank2021.sedml` is invalid.\n - Data generator `x1_1_task1` is invalid.\n - Variable `p1_x1_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x1']` does not match any elements of model `model`.\n - Data generator `x2_2_task1` is invalid.\n - Variable `p1_x2_2_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='x2']` does not match any elements of model `model`.\n"```</details>|
|<details><summary>[BIOMD0000001061](https://www.ebi.ac.uk/biomodels/BIOMD0000001061)<br/><sup>Canto-Encalada2022-FBA of simu</sup></summary><sup>Canto-Encalada2022-FBA of simultaneous degradation of ammonia and pollutants</sup></details>|<details><summary>pass</summary>[Canto-Encalada2022-FBA+of+simultaneous+degradation+of+ammonia+and+pollutants.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001061#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000001062](https://www.ebi.ac.uk/biomodels/BIOMD0000001062)<br/><sup>Kim2021 - Development of a Gen</sup></summary><sup>Kim2021 - Development of a Genome-Scale Metabolic Model and Phenome Analysis of the Probiotic Escherichia coli Strain Nissle 1917</sup></details>|<details><summary>pass</summary>[iDK1463.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001062#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000001063](https://www.ebi.ac.uk/biomodels/BIOMD0000001063)<br/><sup>Lu2019 - Genome scale metaboli</sup></summary><sup>Lu2019 - Genome scale metabolic model for Saccharomyces cerevisiae - yeastGEM8.5.0</sup></details>|<details><summary>pass</summary>[yeast-GEM.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001063#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000001064](https://www.ebi.ac.uk/biomodels/BIOMD0000001064)<br/><sup>Kim2011 - the genome scale rec</sup></summary><sup>Kim2011 - the genome scale reconstruction of the Vibrio vulnificus metabolic network - VvuMBEL943</sup></details>|<details><summary>pass</summary>[Kim2011_VvuMBEL943.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001064#Files)</details>|pass|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000001065](https://www.ebi.ac.uk/biomodels/BIOMD0000001065)<br/><sup>vonDassow2000 - Segment Polari</sup></summary><sup>vonDassow2000 - Segment Polarity Network model on 1x4 grid of cells</sup></details>|<details><summary>FAIL</summary>[vonDassow2000_1x4.timecourse1.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001065#Files)</details>|FAIL|NoSEDML|NA|NA|NA|
|<details><summary>[BIOMD0000001066](https://www.ebi.ac.uk/biomodels/BIOMD0000001066)<br/><sup>Chowell2022 - Random Forest mo</sup></summary><sup>Chowell2022 - Random Forest model to predict efficacy of immune checkpoint blockade across multiple cancer patient cohorts</sup></details>|<details><summary>NonSBML</summary>Open Neural Network Exchange:[{'name': 'RF16.onnx', 'fileSize': '1889619'}]</details>|NA|NA|NA|NA|NA|
|<details><summary>[BIOMD0000001067](https://www.ebi.ac.uk/biomodels/BIOMD0000001067)<br/><sup>Deshpande2019 - Random Forest </sup></summary><sup>Deshpande2019 - Random Forest model to predict long non-coding RNAs from coding RNAs in Zea Mays plant transcriptomic data</sup></details>|<details><summary>NonSBML</summary>Open Neural Network Exchange:[{'name': 'zmays_trained_model.onnx', 'fileSize': '35521903'}]</details>|NA|NA|NA|NA|NA|
|<details><summary>[BIOMD0000001068](https://www.ebi.ac.uk/biomodels/BIOMD0000001068)<br/><sup>Nassar2022 - Metagenomics Clas</sup></summary><sup>Nassar2022 - Metagenomics Classification Task for Scientific Literature Text</sup></details>|<details><summary>NonSBML</summary>Open Neural Network Exchange:[{'name': 'doc2vec-model.onnx', 'fileSize': '865713'}]</details>|NA|NA|NA|NA|NA|
|<details><summary>[BIOMD0000001069](https://www.ebi.ac.uk/biomodels/BIOMD0000001069)<br/><sup>Alam2019 - Machine learning ap</sup></summary><sup>Alam2019 - Machine learning approach of automatic identification and counting of blood cells</sup></details>|<details><summary>NonSBML</summary>Open Neural Network Exchange:[{'name': 'saved_model.onnx', 'fileSize': '63132297'}]</details>|NA|NA|NA|NA|NA|
|<details><summary>[BIOMD0000001070](https://www.ebi.ac.uk/biomodels/BIOMD0000001070)<br/><sup>Kong2022 - Conditional Antibod</sup></summary><sup>Kong2022 - Conditional Antibody Design as 3D Equivariant Graph Translation</sup></details>|<details><summary>NonSBML</summary>Other:[{'name': 'mean.Dockerfile', 'fileSize': '799'}]</details>|NA|NA|NA|NA|NA|
|<details><summary>[BIOMD0000001071](https://www.ebi.ac.uk/biomodels/BIOMD0000001071)<br/><sup>Wang2022 - Scaffolding protein</sup></summary><sup>Wang2022 - Scaffolding protein functional sites using deep learning</sup></details>|<details><summary>NonSBML</summary>Other:[{'name': 'Dockerfile.txt', 'fileSize': '2612'}]</details>|NA|NA|NA|NA|NA|
|<details><summary>[BIOMD0000001072](https://www.ebi.ac.uk/biomodels/BIOMD0000001072)<br/><sup>Phillips2013 - physiologically</sup></summary><sup>Phillips2013 - physiologically based modeling explaining Mammalian rest/activity patterns</sup></details>|<details><summary>pass</summary>[Phillips2013.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001072#Files)</details>|FAIL|<details><summary>pass</summary>[Phillips2013.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001072#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Phillips2013.sedml160979.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: SedmlFeatureNotSupportedWarning:\r\n\r\n\x1b[33m`/tmp/tmpslfb_qus/./Phillips2013.sedml` is encoded using L1V4. Only a few features introduced after L1V3 are supported. Most newer features, such as finer stylistic control of plots and parameter estimation, are not yet supported.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Phillips2013.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (10501). The following is the first warning at line 1328, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'theta - V_m' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 37 warnings of type SBML unit consistency (99505). The following is the first warning at line 962, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression '12 / pi * 3600' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 22 warnings of type Modeling practice (80701). The following is the first warning at line 846, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Theta' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task2`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Phillips2013.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Phillips2013.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (10501). The following is the first warning at line 1328, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'theta - V_m' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 37 warnings of type SBML unit consistency (99505). The following is the first warning at line 962, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression '12 / pi * 3600' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 22 warnings of type Modeling practice (80701). The following is the first warning at line 846, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Theta' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task2`\x1b[0m\r\n\r\n`/root/in/Phillips2013.sedml160979.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Phillips2013.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='v_md']` does not match any elements of model `model`.\n - Data generator `Qm_1_task1` is invalid.\n - Variable `p1_Qm_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Qm']` does not match any elements of model `model`.\n - Data generator `Qm_1_task2` is invalid.\n - Variable `p1_Qm_1_task2` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Qm']` does not match any elements of model `model`.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Phillips2013.sedml724947.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: SedmlFeatureNotSupportedWarning: \x1b[33m`/tmp/tmp9u3evui6/./Phillips2013.sedml` is encoded using L1V4. Only a few features introduced after L1V3 are supported. Most newer features, such as finer stylistic control of plots and parameter estimation, are not yet supported.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Phillips2013.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (10501). The following is the first warning at line 1328, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'theta - V_m' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 37 warnings of type SBML unit consistency (99505). The following is the first warning at line 962, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression '12 / pi * 3600' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 22 warnings of type Modeling practice (80701). The following is the first warning at line 846, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Theta' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task2`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Phillips2013.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Phillips2013.xml` has warnings.\r\n - 6 warnings of type SBML unit consistency (10501). The following is the first warning at line 1328, column 8:\r\n - The units of the expressions used as arguments to a function call are expected to match the units expected for the arguments of that function. \r\n Reference: L2V4 Section 3.4\r\n The formula 'theta - V_m' in the math element of the <kineticLaw> can only act on variables with the same units.\r\n - 37 warnings of type SBML unit consistency (99505). The following is the first warning at line 962, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression '12 / pi * 3600' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 22 warnings of type Modeling practice (80701). The following is the first warning at line 846, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'Theta' does not have a 'units' attribute.\r\n - Output `plot_2_task2` has warnings.\r\n - Some simulation tools will not be able to generate this output because it uses data from repeated tasks, which some simulation tools do not have the capability to execute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\r\n - `time_task2`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Phillips2013.sedml724947.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Phillips2013.sedml` is invalid.\n - Task `task2` is invalid.\n - Changes are invalid.\n - Change 1 is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='v_md']` does not match any elements of model `model`.\n - Data generator `Qm_1_task1` is invalid.\n - Variable `p1_Qm_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Qm']` does not match any elements of model `model`.\n - Data generator `Qm_1_task2` is invalid.\n - Variable `p1_Qm_1_task2` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Qm']` does not match any elements of model `model`.\n"```</details>|
|<details><summary>[BIOMD0000001073](https://www.ebi.ac.uk/biomodels/BIOMD0000001073)<br/><sup>Patterson2022 - Tumour mutatio</sup></summary><sup>Patterson2022 - Tumour mutation data driven Random Forest model to predict immune checkpoint inhibitor therapy benefit in metastatic melanoma</sup></details>|<details><summary>NonSBML</summary>Open Neural Network Exchange:[{'name': 'rforest_ROLP.onnx', 'fileSize': '80528'}]</details>|NA|NA|NA|NA|NA|
|<details><summary>[BIOMD0000001074](https://www.ebi.ac.uk/biomodels/BIOMD0000001074)<br/><sup>Liu2023 - Predicting the effic</sup></summary><sup>Liu2023 - Predicting the efficacy of immune checkpoint inhibitors monotherapy in advanced non-small cell lung cancer: a machine learning method based on multidimensional data</sup></details>|<details><summary>NonSBML</summary>Open Neural Network Exchange:[{'name': 'rf_ici.onnx', 'fileSize': '37866'}]</details>|NA|NA|NA|NA|NA|
|<details><summary>[BIOMD0000001075](https://www.ebi.ac.uk/biomodels/BIOMD0000001075)<br/><sup>Sammut2022 - Multi-omic machin</sup></summary><sup>Sammut2022 - Multi-omic machine learning model to predict pathological complete response for breast cancer neoadjuvant therapy</sup></details>|<details><summary>NonSBML</summary>Other:[{'name': 'reproduced_trained_models.zip', 'fileSize': '87786133'}]</details>|NA|NA|NA|NA|NA|
|<details><summary>[BIOMD0000001076](https://www.ebi.ac.uk/biomodels/BIOMD0000001076)<br/><sup>Lpez-Corts2020 - Prediction </sup></summary><sup>Lpez-Corts2020 - Prediction of Breast Cancer (BC) proteins involved in cancer immunotherapy using molecular descriptors and Multi Layer Perceptron (MLP) neural network</sup></details>|<details><summary>NonSBML</summary>Open Neural Network Exchange:[{'name': 'MLP_model3.onnx', 'fileSize': '25518'}]</details>|NA|NA|NA|NA|NA|
|<details><summary>[BIOMD0000001077](https://www.ebi.ac.uk/biomodels/BIOMD0000001077)<br/><sup>Adlung2021 - Cell-to-cell vari</sup></summary><sup>Adlung2021 - Cell-to-cell variability in JAK2/STAT5 pathway</sup></details>|<details><summary>pass</summary>[Adlung2021 _model_jakstat_pa.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001077#Files)</details>|pass|<details><summary>pass</summary>[Adlung2021 _model_jakstat_pa.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001077#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Adlung2021 _model_jakstat_pa.sedml862042.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 2: b'usage: biosimulators-tellurium [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-tellurium: error: unrecognized arguments: _model_jakstat_pa.sedml862042.omex\n'```</details>|<details><summary>other</summary>```Command '-i /root/in/Adlung2021 _model_jakstat_pa.sedml75114.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 2: b'usage: biosimulators-copasi [-h] [-d] [-q] -i ARCHIVE [-o OUT_DIR] [-v]\nbiosimulators-copasi: error: unrecognized arguments: _model_jakstat_pa.sedml75114.omex\n'```</details>|
|[BIOMD0000001078](https://www.ebi.ac.uk/biomodels/BIOMD0000001078)<br/><sup>Hammaren-Geissen2022_PPToP_Model12</sup>|<details><summary>pass</summary>[Hammaren-Geissen2022_PPToP_Model12_withParameterSets.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001078#Files)</details>|pass|<details><summary>pass</summary>[Hammaren-Geissen2022_PPToP_Model12_withParameterSets.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001078#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/Hammaren-Geissen2022_PPToP_Model12_withParameterSets.sedml487807.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: SedmlFeatureNotSupportedWarning:\r\n\r\n\x1b[33m`/tmp/tmpy_me3_m1/./Hammaren-Geissen2022_PPToP_Model12_withParameterSets.sedml` is encoded using L1V4. Only a few features introduced after L1V3 are supported. Most newer features, such as finer stylistic control of plots and parameter estimation, are not yet supported.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Hammaren-Geissen2022_PPToP_Model12_withParameterSets.xml` has warnings.\r\n - 23 warnings of type SBML unit consistency (99505). The following is the first warning at line 314, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Pu_total' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 20 warnings of type Modeling practice (80701). The following is the first warning at line 264, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kdegu' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Hammaren-Geissen2022_PPToP_Model12_withParameterSets.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Hammaren-Geissen2022_PPToP_Model12_withParameterSets.xml` has warnings.\r\n - 23 warnings of type SBML unit consistency (99505). The following is the first warning at line 314, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Pu_total' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 20 warnings of type Modeling practice (80701). The following is the first warning at line 264, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kdegu' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n`/root/in/Hammaren-Geissen2022_PPToP_Model12_withParameterSets.sedml487807.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Hammaren-Geissen2022_PPToP_Model12_withParameterSets.sedml` is invalid.\n - Data generator `Pu_1_task1` is invalid.\n - Variable `p1_Pu_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Pu']` does not match any elements of model `model`.\n - Data generator `Pp_2_task1` is invalid.\n - Variable `p1_Pp_2_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Pp']` does not match any elements of model `model`.\n - Data generator `Pu_new_3_task1` is invalid.\n - Variable `p1_Pu_new_3_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Pu_new']` does not match any elements of model `model`.\n - Data generator `Pp_new_4_task1` is invalid.\n - Variable `p1_Pp_new_4_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Pp_new']` does not match any elements of model `model`.\n - Data generator `Pu_old_clearance_1_task1` is invalid.\n - Variable `p2_Pu_old_clearance_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Pu_old_clearance']` does not match any elements of model `model`.\n - Data generator `Pp_old_clearance_2_task1` is invalid.\n - Variable `p2_Pp_old_clearance_2_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Pp_old_clearance']` does not match any elements of model `model`.\n - Data generator `P_old_clearance_3_task1` is invalid.\n - Variable `p2_P_old_clearance_3_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='P_old_clearance']` does not match any elements of model `model`.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/Hammaren-Geissen2022_PPToP_Model12_withParameterSets.sedml423741.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: SedmlFeatureNotSupportedWarning: \x1b[33m`/tmp/tmp7vf73yvf/./Hammaren-Geissen2022_PPToP_Model12_withParameterSets.sedml` is encoded using L1V4. Only a few features introduced after L1V3 are supported. Most newer features, such as finer stylistic control of plots and parameter estimation, are not yet supported.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `Hammaren-Geissen2022_PPToP_Model12_withParameterSets.xml` has warnings.\r\n - 23 warnings of type SBML unit consistency (99505). The following is the first warning at line 314, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Pu_total' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 20 warnings of type Modeling practice (80701). The following is the first warning at line 264, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kdegu' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./Hammaren-Geissen2022_PPToP_Model12_withParameterSets.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `Hammaren-Geissen2022_PPToP_Model12_withParameterSets.xml` has warnings.\r\n - 23 warnings of type SBML unit consistency (99505). The following is the first warning at line 314, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <initialAssignment> <math> expression 'Pu_total' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 20 warnings of type Modeling practice (80701). The following is the first warning at line 264, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'kdegu' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/Hammaren-Geissen2022_PPToP_Model12_withParameterSets.sedml423741.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./Hammaren-Geissen2022_PPToP_Model12_withParameterSets.sedml` is invalid.\n - Data generator `Pu_1_task1` is invalid.\n - Variable `p1_Pu_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Pu']` does not match any elements of model `model`.\n - Data generator `Pp_2_task1` is invalid.\n - Variable `p1_Pp_2_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Pp']` does not match any elements of model `model`.\n - Data generator `Pu_new_3_task1` is invalid.\n - Variable `p1_Pu_new_3_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Pu_new']` does not match any elements of model `model`.\n - Data generator `Pp_new_4_task1` is invalid.\n - Variable `p1_Pp_new_4_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='Pp_new']` does not match any elements of model `model`.\n - Data generator `Pu_old_clearance_1_task1` is invalid.\n - Variable `p2_Pu_old_clearance_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Pu_old_clearance']` does not match any elements of model `model`.\n - Data generator `Pp_old_clearance_2_task1` is invalid.\n - Variable `p2_Pp_old_clearance_2_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Pp_old_clearance']` does not match any elements of model `model`.\n - Data generator `P_old_clearance_3_task1` is invalid.\n - Variable `p2_P_old_clearance_3_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='P_old_clearance']` does not match any elements of model `model`.\n"```</details>|
|<details><summary>[BIOMD0000001079](https://www.ebi.ac.uk/biomodels/BIOMD0000001079)<br/><sup>DeBoeck2021 - Modular approach</sup></summary><sup>DeBoeck2021 - Modular approach to modeling the cell cycle, simple cell cycle model</sup></details>|<details><summary>pass</summary>[DeBoeck2021_cellcycle_bistableapc.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001079#Files)</details>|pass|<details><summary>pass</summary>[DeBoeck2021_cellcycle_bistableapc.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001079#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/DeBoeck2021_cellcycle_bistableapc.sedml993610.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: SedmlFeatureNotSupportedWarning:\r\n\r\n\x1b[33m`/tmp/tmp8wukralz/./DeBoeck2021_cellcycle_bistableapc.sedml` is encoded using L1V4. Only a few features introduced after L1V3 are supported. Most newer features, such as finer stylistic control of plots and parameter estimation, are not yet supported.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `DeBoeck2021_cellcycle_bistableapc.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 308, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <rateRule> <math> expression '1 / epsilon * (pow(CycCdk, n) / (pow(K * Xi(APC, alpha, r), n) + pow(CycCdk, n)) - APC)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 276, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'ks' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./DeBoeck2021_cellcycle_bistableapc.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `DeBoeck2021_cellcycle_bistableapc.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 308, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <rateRule> <math> expression '1 / epsilon * (pow(CycCdk, n) / (pow(K * Xi(APC, alpha, r), n) + pow(CycCdk, n)) - APC)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 276, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'ks' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n`/root/in/DeBoeck2021_cellcycle_bistableapc.sedml993610.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./DeBoeck2021_cellcycle_bistableapc.sedml` is invalid.\n - Data generator `CycCdk_1_task1` is invalid.\n - Variable `p1_CycCdk_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CycCdk']` does not match any elements of model `model`.\n - Data generator `APC_2_task1` is invalid.\n - Variable `p1_APC_2_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='APC']` does not match any elements of model `model`.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/DeBoeck2021_cellcycle_bistableapc.sedml182960.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: SedmlFeatureNotSupportedWarning: \x1b[33m`/tmp/tmpvreojty7/./DeBoeck2021_cellcycle_bistableapc.sedml` is encoded using L1V4. Only a few features introduced after L1V3 are supported. Most newer features, such as finer stylistic control of plots and parameter estimation, are not yet supported.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `DeBoeck2021_cellcycle_bistableapc.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 308, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <rateRule> <math> expression '1 / epsilon * (pow(CycCdk, n) / (pow(K * Xi(APC, alpha, r), n) + pow(CycCdk, n)) - APC)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 276, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'ks' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./DeBoeck2021_cellcycle_bistableapc.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `DeBoeck2021_cellcycle_bistableapc.xml` has warnings.\r\n - 3 warnings of type SBML unit consistency (99505). The following is the first warning at line 308, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <rateRule> <math> expression '1 / epsilon * (pow(CycCdk, n) / (pow(K * Xi(APC, alpha, r), n) + pow(CycCdk, n)) - APC)' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 8 warnings of type Modeling practice (80701). The following is the first warning at line 276, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'ks' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/DeBoeck2021_cellcycle_bistableapc.sedml182960.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./DeBoeck2021_cellcycle_bistableapc.sedml` is invalid.\n - Data generator `CycCdk_1_task1` is invalid.\n - Variable `p1_CycCdk_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CycCdk']` does not match any elements of model `model`.\n - Data generator `APC_2_task1` is invalid.\n - Variable `p1_APC_2_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='APC']` does not match any elements of model `model`.\n"```</details>|
|<details><summary>[BIOMD0000001080](https://www.ebi.ac.uk/biomodels/BIOMD0000001080)<br/><sup>DeBoeck2021 - Modular approach</sup></summary><sup>DeBoeck2021 - Modular approach to modeling the cell cycle, 5 ODE model with 3 bistable switches</sup></details>|<details><summary>pass</summary>[DeBoeck2021_cellcycle_threeswitches.xml](https://www.ebi.ac.uk/biomodels/BIOMD0000001080#Files)</details>|pass|<details><summary>pass</summary>[DeBoeck2021_cellcycle_threeswitches.sedml](https://www.ebi.ac.uk/biomodels/BIOMD0000001080#Files)</details>|FAIL|<details><summary>other</summary>```Command '-i /root/in/DeBoeck2021_cellcycle_threeswitches.sedml461219.omex -o /root/out' in image 'ghcr.io/biosimulators/tellurium' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: SedmlFeatureNotSupportedWarning:\r\n\r\n\x1b[33m`/tmp/tmpb0q8dbjw/./DeBoeck2021_cellcycle_threeswitches.sedml` is encoded using L1V4. Only a few features introduced after L1V3 are supported. Most newer features, such as finer stylistic control of plots and parameter estimation, are not yet supported.\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `DeBoeck2021_cellcycle_threeswitches.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 461, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'Cdk / 44' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 25 warnings of type Modeling practice (80701). The following is the first warning at line 368, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'd_syn' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning:\r\n\r\n\x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./DeBoeck2021_cellcycle_threeswitches.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `DeBoeck2021_cellcycle_threeswitches.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 461, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'Cdk / 44' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 25 warnings of type Modeling practice (80701). The following is the first warning at line 368, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'd_syn' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n\r\n`/root/in/DeBoeck2021_cellcycle_threeswitches.sedml461219.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./DeBoeck2021_cellcycle_threeswitches.sedml` is invalid.\n - Data generator `CycD_1_task1` is invalid.\n - Variable `p1_CycD_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CycD']` does not match any elements of model `model`.\n - Data generator `E2F_1_task1` is invalid.\n - Variable `p1_E2F_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='E2F']` does not match any elements of model `model`.\n - Data generator `CycB_1_task1` is invalid.\n - Variable `p2_CycB_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CycB']` does not match any elements of model `model`.\n - Data generator `Cdk_1_task1` is invalid.\n - Variable `p2_Cdk_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Cdk']` does not match any elements of model `model`.\n - Data generator `APC_3_task1` is invalid.\n - Variable `p3_APC_3_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='APC']` does not match any elements of model `model`.\n - Data generator `CycD_max_6_task1` is invalid.\n - Variable `p3_CycD_max_6_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='CycD_max']` does not match any elements of model `model`.\n - Data generator `CycB_max_7_task1` is invalid.\n - Variable `p3_CycB_max_7_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='CycB_max']` does not match any elements of model `model`.\n - Data generator `APC_max_8_task1` is invalid.\n - Variable `p3_APC_max_8_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='APC_max']` does not match any elements of model `model`.\n - Data generator `E2F_max_9_task1` is invalid.\n - Variable `p3_E2F_max_9_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='E2F_max']` does not match any elements of model `model`.\n - Data generator `Cdk_max_1_task1` is invalid.\n - Variable `p8_Cdk_max_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Cdk_max']` does not match any elements of model `model`.\n"```</details>|<details><summary>other</summary>```Command '-i /root/in/DeBoeck2021_cellcycle_threeswitches.sedml896398.omex -o /root/out' in image 'ghcr.io/biosimulators/copasi' returned non-zero exit status 1: b"/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: SedmlFeatureNotSupportedWarning: \x1b[33m`/tmp/tmphd8nlycj/./DeBoeck2021_cellcycle_threeswitches.sedml` is encoded using L1V4. Only a few features introduced after L1V3 are supported. Most newer features, such as finer stylistic control of plots and parameter estimation, are not yet supported.\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe SED document is potentially incorrect.\r\n - Model `model` has warnings.\r\n - The model file `DeBoeck2021_cellcycle_threeswitches.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 461, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'Cdk / 44' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 25 warnings of type Modeling practice (80701). The following is the first warning at line 368, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'd_syn' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n/usr/local/lib/python3.9/site-packages/biosimulators_utils/warnings.py:31: BioSimulatorsWarning: \x1b[33mThe COMBINE/OMEX archive has warnings.\r\n - OMEX manifests should not contain content entries for themselves.\r\n - The SED-ML file at location `./DeBoeck2021_cellcycle_threeswitches.sedml` has warnings.\r\n - Model `model` has warnings.\r\n - The model file `DeBoeck2021_cellcycle_threeswitches.xml` has warnings.\r\n - 12 warnings of type SBML unit consistency (99505). The following is the first warning at line 461, column 6:\r\n - In situations where a mathematical expression contains literal numbers or parameters whose units have not been declared, it is not possible to verify accurately the consistency of the units in the expression. \r\n The units of the <assignmentRule> <math> expression 'Cdk / 44' cannot be fully checked. Unit consistency reported as either no errors or further unit errors related to this object may not be accurate.\r\n - 25 warnings of type Modeling practice (80701). The following is the first warning at line 368, column 6:\r\n - As a principle of best modeling practice, the units of a <parameter> should be declared rather than be left undefined. Doing so improves the ability of software to check the consistency of units and helps make it easier to detect potential errors in models.\r\n The <parameter> with the id 'd_syn' does not have a 'units' attribute.\r\n - The following data generators do not contribute to outputs:\r\n - `time_task1`\x1b[0m\r\n warnings.warn(termcolor.colored(message, Colors.warning.value), category)\r\n`/root/in/DeBoeck2021_cellcycle_threeswitches.sedml896398.omex` is not a valid COMBINE/OMEX archive.\n - The SED-ML file at location `./DeBoeck2021_cellcycle_threeswitches.sedml` is invalid.\n - Data generator `CycD_1_task1` is invalid.\n - Variable `p1_CycD_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CycD']` does not match any elements of model `model`.\n - Data generator `E2F_1_task1` is invalid.\n - Variable `p1_E2F_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='E2F']` does not match any elements of model `model`.\n - Data generator `CycB_1_task1` is invalid.\n - Variable `p2_CycB_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfSpecies/sbml:species[@id='CycB']` does not match any elements of model `model`.\n - Data generator `Cdk_1_task1` is invalid.\n - Variable `p2_Cdk_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Cdk']` does not match any elements of model `model`.\n - Data generator `APC_3_task1` is invalid.\n - Variable `p3_APC_3_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='APC']` does not match any elements of model `model`.\n - Data generator `CycD_max_6_task1` is invalid.\n - Variable `p3_CycD_max_6_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='CycD_max']` does not match any elements of model `model`.\n - Data generator `CycB_max_7_task1` is invalid.\n - Variable `p3_CycB_max_7_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='CycB_max']` does not match any elements of model `model`.\n - Data generator `APC_max_8_task1` is invalid.\n - Variable `p3_APC_max_8_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='APC_max']` does not match any elements of model `model`.\n - Data generator `E2F_max_9_task1` is invalid.\n - Variable `p3_E2F_max_9_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='E2F_max']` does not match any elements of model `model`.\n - Data generator `Cdk_max_1_task1` is invalid.\n - Variable `p8_Cdk_max_1_task1` is invalid.\n - XPath `/sbml:sbml/sbml:model/sbml:listOfParameters/sbml:parameter[@id='Cdk_max']` does not match any elements of model `model`.\n"```</details>|
